{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Postprocessing of the ExTRI2 pipeline results to create the ExTRI2 resource"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This notebook was used to determine the rules to use for renormalisation & discard of sentences. Sentences were extracted from the ExTRI2 resource and checked manually, to determine how to handle each category. \n",
    "\n",
    "Furthermore, the **run postprocessing.py** section is a self-contained section to create the post-processed ExTRI2 file from the file obtained from the pipeline."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "<h3>Table of contents</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "[Postprocessing of the ExTRI2 pipeline results to create the ExTRI2 resource](#Postprocessing-of-the-ExTRI2-pipeline-results-to-create-the-ExTRI2-resource)\n",
       "- [Run postprocessing.py](#Run-postprocessing.py)\n",
       "- [Setup](#Setup)\n",
       "- [Postprocessing](#Postprocessing)\n",
       "  - [AP1 & NFKB](#AP1-&-NFKB)\n",
       "  - [Initial exploration](#Initial-exploration)\n",
       "  - [Create sets of sentences to check](#Create-sets-of-sentences-to-check)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "__import__('sys').path.append('../common/'); __import__('notebook_utils').table_of_contents('postprocessing_checkings.ipynb')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "vscode": {
     "languageId": "plaintext"
    }
   },
   "source": [
    "This notebook will now only be used for the normalisation of the results."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run postprocessing.py\n",
    "Self-contained cell to run postprocessing.py"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "### POSTPROCESSING TRI_df\n",
      "We got 6706 different TFs and 26196 different TGs from sentences labelled as TRI\n",
      "Retrieving from Entrez...\n",
      "\n",
      "4967 sentences are dropped as their TG is not normalised\n",
      "\n",
      "38287 rows (4.23%) will have its TF renormalized to NFKB\n",
      "6329 rows (0.70%) will be dropped as the TG corresponds to NFKB\n",
      "9003 rows (1.00%) will have its TF renormalized to AP1\n",
      "1858 rows (0.21%) will be dropped as the TG corresponds to AP1\n",
      "Breakdown by NCBI Symbol saved in ../../data/postprocessing/tables/AP1_NFKB_breakdown.tsv\n",
      "Number of renormalized sentences and normalization:\n",
      "4827\t0.54%\tp21 is normalized to CDKN1A\n",
      "1922\t0.21%\tp53-ps is normalized to its respective p53 symbol\n",
      "\n",
      "Number of discarded sentences and percentage from total (896330 sentences) and reasoning:\n",
      "2556\t0.29%\tTheir TF contains -AS[1-3]\n",
      "673\t0.08%\tTheir TF are circRNAs\n",
      "952\t0.11%\tTheir TF (NLRP3) is followed by inflammasome but normalised to NLRP3\n",
      "1875\t0.21%\tTheir TG (NLRP3) is followed by inflammasome but normalised to NLRP3\n",
      "1088\t0.12%\tTheir TG is followed by pathway/signalling/axis/program\n",
      "2037\t0.23%\tMDM2-TP53 pair, which is always a PPI\n",
      "1554\t0.17%\tTF is CD(4|8A|8B|74|34) positive (indicative of a cell, not a gene)\n",
      "22\t0.00%\tTG is CD(4|8A|8B|74|34) positive (indicative of a cell, not a gene)\n",
      "724\t0.08%\tPreposition of incorrectly identified as a gene\n",
      "7368\t0.82%\tThey contain translation in them (found to only be correct 40% of the time)\n",
      "41646\t4.65%\tThey are autoregulations (found to only be correct ~10% of the time)\n",
      "\n",
      "Discarding from the NCBI_Gene_ID-to_HGNC_ID mapping 2570 Human Gene IDs out of 25879 as they map to multiple HGNCs\n",
      "Fetching 1092 missing human gene symbols from Entrez...\n",
      "No orthologs in      17184 rows (2.06%). Some missing in 17968 rows (2.15%)\n",
      "No HGNCs in          17262 rows (2.07%). Some missing in 18046 rows (2.16%)\n",
      "No human symbols in  17184 rows (2.06%). Some missing in 17968 rows (2.15%)\n",
      "\n",
      "### POSTPROCESSING nonTRI_df\n",
      "We got 4284 different TFs and 10938 different TGs from sentences labelled as TRI\n",
      "Retrieving from Entrez...\n",
      "\n",
      "476 sentences are dropped as their TG is not normalised\n",
      "\n",
      "2065 rows (2.78%) will have its TF renormalized to NFKB\n",
      "1221 rows (1.65%) will be dropped as the TG corresponds to NFKB\n",
      "315 rows (0.43%) will have its TF renormalized to AP1\n",
      "265 rows (0.36%) will be dropped as the TG corresponds to AP1\n",
      "Breakdown by NCBI Symbol saved in ../../data/postprocessing/tables/AP1_NFKB_breakdown.tsv\n",
      "Number of renormalized sentences and normalization:\n",
      "83\t0.11%\tp21 is normalized to CDKN1A\n",
      "125\t0.17%\tp53-ps is normalized to its respective p53 symbol\n",
      "\n",
      "Number of discarded sentences and percentage from total (72663 sentences) and reasoning:\n",
      "60\t0.08%\tTheir TF contains -AS[1-3]\n",
      "12\t0.02%\tTheir TF are circRNAs\n",
      "99\t0.14%\tTheir TF (NLRP3) is followed by inflammasome but normalised to NLRP3\n",
      "23\t0.03%\tTheir TG (NLRP3) is followed by inflammasome but normalised to NLRP3\n",
      "1554\t2.14%\tTheir TG is followed by pathway/signalling/axis/program\n",
      "82\t0.11%\tMDM2-TP53 pair, which is always a PPI\n",
      "311\t0.43%\tTF is CD(4|8A|8B|74|34) positive (indicative of a cell, not a gene)\n",
      "245\t0.34%\tTG is CD(4|8A|8B|74|34) positive (indicative of a cell, not a gene)\n",
      "82\t0.11%\tPreposition of incorrectly identified as a gene\n",
      "665\t0.92%\tThey contain translation in them (found to only be correct 40% of the time)\n",
      "15020\t20.67%\tThey are autoregulations (found to only be correct ~10% of the time)\n",
      "\n",
      "Discarding from the NCBI_Gene_ID-to_HGNC_ID mapping 2570 Human Gene IDs out of 25879 as they map to multiple HGNCs\n",
      "Fetching 730 missing human gene symbols from Entrez...\n",
      "No orthologs in      909 rows (1.67%). Some missing in 955 rows (1.75%)\n",
      "No HGNCs in          914 rows (1.68%). Some missing in 960 rows (1.76%)\n",
      "No human symbols in  909 rows (1.67%). Some missing in 955 rows (1.75%)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import sys\n",
    "sys.path.append('../common')\n",
    "sys.path.append('../../')\n",
    "from scripts.postprocessing.postprocessing import *\n",
    "main()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Setup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "from IPython.display import display, HTML\n",
    "import matplotlib.pyplot as plt\n",
    "import itertools\n",
    "import re\n",
    "\n",
    "## Custom functions\n",
    "import sys\n",
    "\n",
    "sys.path.append('../common')\n",
    "sys.path.append('../../')\n",
    "\n",
    "from notebook_utils import table_of_contents, table_from_dict, h3, h4, h5, md, bold\n",
    "from renormalisations import *\n",
    "from postprocessing import *\n",
    "pd.set_option('display.max_colwidth', 20)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#SentenceID</th>\n",
       "      <th>Sentence</th>\n",
       "      <th>TF</th>\n",
       "      <th>TG</th>\n",
       "      <th>TF Id</th>\n",
       "      <th>TG Id</th>\n",
       "      <th>TF offset</th>\n",
       "      <th>Gene offset</th>\n",
       "      <th>TRI score</th>\n",
       "      <th>Valid</th>\n",
       "      <th>...</th>\n",
       "      <th>TG Symbol</th>\n",
       "      <th>TG TaxID</th>\n",
       "      <th>TF_type</th>\n",
       "      <th>renormalisation</th>\n",
       "      <th>TF_human_Id</th>\n",
       "      <th>TF_human_symbol</th>\n",
       "      <th>TF_HGNC_Id</th>\n",
       "      <th>TG_human_Id</th>\n",
       "      <th>TG_human_symbol</th>\n",
       "      <th>TG_HGNC_Id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PMID:35388756:pu...</td>\n",
       "      <td>Chip-IP results ...</td>\n",
       "      <td>CHCHD2</td>\n",
       "      <td>GNPTG</td>\n",
       "      <td>51142</td>\n",
       "      <td>84572</td>\n",
       "      <td>1366</td>\n",
       "      <td>1458</td>\n",
       "      <td>0.992724397527042</td>\n",
       "      <td>Valid</td>\n",
       "      <td>...</td>\n",
       "      <td>GNPTG</td>\n",
       "      <td>9606</td>\n",
       "      <td>coTF</td>\n",
       "      <td></td>\n",
       "      <td>51142</td>\n",
       "      <td>CHCHD2</td>\n",
       "      <td>HGNC:21645</td>\n",
       "      <td>84572</td>\n",
       "      <td>GNPTG</td>\n",
       "      <td>HGNC:23026</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PMID:26808438:pu...</td>\n",
       "      <td>Among multiple C...</td>\n",
       "      <td>ChREBP</td>\n",
       "      <td>Mid1ip1,Txnip</td>\n",
       "      <td>51085</td>\n",
       "      <td>58526;10628</td>\n",
       "      <td>919</td>\n",
       "      <td>1050</td>\n",
       "      <td>0.9927056483604216</td>\n",
       "      <td>Valid</td>\n",
       "      <td>...</td>\n",
       "      <td>MID1IP1;TXNIP</td>\n",
       "      <td>9606</td>\n",
       "      <td>dbTF</td>\n",
       "      <td></td>\n",
       "      <td>51085</td>\n",
       "      <td>MLXIPL</td>\n",
       "      <td>HGNC:12744</td>\n",
       "      <td>58526;10628</td>\n",
       "      <td>MID1IP1;TXNIP</td>\n",
       "      <td>HGNC:20715;HGNC:...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           #SentenceID             Sentence      TF             TG  TF Id  \\\n",
       "0  PMID:35388756:pu...  Chip-IP results ...  CHCHD2          GNPTG  51142   \n",
       "1  PMID:26808438:pu...  Among multiple C...  ChREBP  Mid1ip1,Txnip  51085   \n",
       "\n",
       "         TG Id TF offset Gene offset           TRI score  Valid  ...  \\\n",
       "0        84572      1366        1458   0.992724397527042  Valid  ...   \n",
       "1  58526;10628       919        1050  0.9927056483604216  Valid  ...   \n",
       "\n",
       "       TG Symbol TG TaxID TF_type renormalisation TF_human_Id TF_human_symbol  \\\n",
       "0          GNPTG     9606    coTF                       51142          CHCHD2   \n",
       "1  MID1IP1;TXNIP     9606    dbTF                       51085          MLXIPL   \n",
       "\n",
       "   TF_HGNC_Id  TG_human_Id TG_human_symbol           TG_HGNC_Id  \n",
       "0  HGNC:21645        84572           GNPTG           HGNC:23026  \n",
       "1  HGNC:12744  58526;10628   MID1IP1;TXNIP  HGNC:20715;HGNC:...  \n",
       "\n",
       "[2 rows x 26 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene_ID</th>\n",
       "      <th>human_gene_ID</th>\n",
       "      <th>TaxID</th>\n",
       "      <th>gene_symbol</th>\n",
       "      <th>human_gene_symbol</th>\n",
       "      <th>unique_human_gene_ID</th>\n",
       "      <th>unique_human_gene_symbol</th>\n",
       "      <th>HGNC_ID</th>\n",
       "      <th>unique_HGNC_ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>100009600</td>\n",
       "      <td>100125288</td>\n",
       "      <td>10090</td>\n",
       "      <td>Zglp1</td>\n",
       "      <td>ZGLP1</td>\n",
       "      <td>100125288</td>\n",
       "      <td>ZGLP1</td>\n",
       "      <td>HGNC:37245</td>\n",
       "      <td>HGNC:37245</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>100033459</td>\n",
       "      <td>None</td>\n",
       "      <td>10090</td>\n",
       "      <td>Ifi208</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     Gene_ID human_gene_ID  TaxID gene_symbol human_gene_symbol  \\\n",
       "0  100009600     100125288  10090       Zglp1             ZGLP1   \n",
       "1  100033459          None  10090      Ifi208              None   \n",
       "\n",
       "  unique_human_gene_ID unique_human_gene_symbol     HGNC_ID unique_HGNC_ID  \n",
       "0            100125288                ZGLP1      HGNC:37245     HGNC:37245  \n",
       "1                 None                 None            None           None  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Checkings on the processed final TRI df\n",
    "config = load_config()\n",
    "final_TRI_df = load_df(config['final_ExTRI2_p'])\n",
    "\n",
    "# TODO - I updated this so that it uses Ensembl instead of HCOP. Ensure that any analysis done with that is still valid.\n",
    "orthologs_df = load_df(config['orthologs_final_p'])\n",
    "\n",
    "display(final_TRI_df[:2])\n",
    "display(orthologs_df[:2])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Postprocessing"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### HGNC orthologs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene_ID</th>\n",
       "      <th>human_gene_ID</th>\n",
       "      <th>TaxID</th>\n",
       "      <th>gene_symbol</th>\n",
       "      <th>human_gene_symbol</th>\n",
       "      <th>unique_human_gene_ID</th>\n",
       "      <th>unique_human_gene_symbol</th>\n",
       "      <th>HGNC_ID</th>\n",
       "      <th>unique_HGNC_ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>100009600</td>\n",
       "      <td>100125288</td>\n",
       "      <td>10090</td>\n",
       "      <td>Zglp1</td>\n",
       "      <td>ZGLP1</td>\n",
       "      <td>100125288</td>\n",
       "      <td>ZGLP1</td>\n",
       "      <td>HGNC:37245</td>\n",
       "      <td>HGNC:37245</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>100033459</td>\n",
       "      <td>None</td>\n",
       "      <td>10090</td>\n",
       "      <td>Ifi208</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     Gene_ID human_gene_ID  TaxID gene_symbol human_gene_symbol  \\\n",
       "0  100009600     100125288  10090       Zglp1             ZGLP1   \n",
       "1  100033459          None  10090      Ifi208              None   \n",
       "\n",
       "  unique_human_gene_ID unique_human_gene_symbol     HGNC_ID unique_HGNC_ID  \n",
       "0            100125288                ZGLP1      HGNC:37245     HGNC:37245  \n",
       "1                 None                 None            None           None  "
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ExTRI2_df = final_TRI_df.copy()\n",
    "orthologs_df[:2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "All gene IDs in ExTRI2: 24541    \n",
      "Missing gene symbols: 0\n",
      "\n",
      "One-to-one human gene mapping: 23690\n",
      "One-to-many human gene mapping: 270\n",
      "- With a unique_human_gene_ID assigned: 242 (28 unassigned)\n",
      "- Assigned through exact match: 189 (53 through 1st family member rule)\n",
      "Missing human gene mapping: 581\n",
      "Missing HGNC IDs: 586\n",
      "\n",
      "Number of orthologs to check: 81\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>TaxID</th>\n",
       "      <th>Gene_ID</th>\n",
       "      <th>gene_symbol</th>\n",
       "      <th>unique_human_gene_symbol</th>\n",
       "      <th>human_gene_symbol</th>\n",
       "      <th>human_gene_ID</th>\n",
       "      <th>HGNC_ID</th>\n",
       "      <th>unique_human_gene_ID</th>\n",
       "      <th>unique_HGNC_ID</th>\n",
       "      <th>TF_count</th>\n",
       "      <th>TG_count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10090</td>\n",
       "      <td>171281</td>\n",
       "      <td>Acot3</td>\n",
       "      <td>ACOT1</td>\n",
       "      <td>ACOT2;ACOT1</td>\n",
       "      <td>10965;641371</td>\n",
       "      <td>HGNC:18431;HGNC:...</td>\n",
       "      <td>641371</td>\n",
       "      <td>HGNC:33128</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10090</td>\n",
       "      <td>217698</td>\n",
       "      <td>Acot5</td>\n",
       "      <td>ACOT1</td>\n",
       "      <td>ACOT2;ACOT1</td>\n",
       "      <td>10965;641371</td>\n",
       "      <td>HGNC:18431;HGNC:...</td>\n",
       "      <td>641371</td>\n",
       "      <td>HGNC:33128</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10090</td>\n",
       "      <td>233799</td>\n",
       "      <td>Acsm2</td>\n",
       "      <td>ACSM2A</td>\n",
       "      <td>ACSM2A;ACSM2B</td>\n",
       "      <td>123876;348158</td>\n",
       "      <td>HGNC:32017;HGNC:...</td>\n",
       "      <td>123876</td>\n",
       "      <td>HGNC:32017</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10090</td>\n",
       "      <td>11522</td>\n",
       "      <td>Adh1</td>\n",
       "      <td>ADH1A</td>\n",
       "      <td>ADH1B;ADH1A;ADH1C</td>\n",
       "      <td>125;124;126</td>\n",
       "      <td>HGNC:250;HGNC:24...</td>\n",
       "      <td>124</td>\n",
       "      <td>HGNC:249</td>\n",
       "      <td>0</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10090</td>\n",
       "      <td>11648</td>\n",
       "      <td>Akp3</td>\n",
       "      <td>None</td>\n",
       "      <td>ALPP;ALPG;ALPI</td>\n",
       "      <td>250;251;248</td>\n",
       "      <td>HGNC:439;HGNC:44...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10090</td>\n",
       "      <td>223672</td>\n",
       "      <td>Apol9a</td>\n",
       "      <td>APOL1</td>\n",
       "      <td>APOL1;APOL3;APOL...</td>\n",
       "      <td>8542;80833;80832...</td>\n",
       "      <td>HGNC:618;HGNC:14...</td>\n",
       "      <td>8542</td>\n",
       "      <td>HGNC:618</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10090</td>\n",
       "      <td>20305</td>\n",
       "      <td>Ccl6</td>\n",
       "      <td>CCL15</td>\n",
       "      <td>CCL15;CCL23</td>\n",
       "      <td>6359;6368</td>\n",
       "      <td>HGNC:10613;HGNC:...</td>\n",
       "      <td>6359</td>\n",
       "      <td>HGNC:10613</td>\n",
       "      <td>0</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10090</td>\n",
       "      <td>20308</td>\n",
       "      <td>Ccl9</td>\n",
       "      <td>CCL15</td>\n",
       "      <td>CCL15;CCL23</td>\n",
       "      <td>6359;6368</td>\n",
       "      <td>HGNC:10613;HGNC:...</td>\n",
       "      <td>6359</td>\n",
       "      <td>HGNC:10613</td>\n",
       "      <td>0</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10090</td>\n",
       "      <td>12526</td>\n",
       "      <td>Cd8b1</td>\n",
       "      <td>CD8B</td>\n",
       "      <td>CD8B2;CD8B</td>\n",
       "      <td>927;926</td>\n",
       "      <td>HGNC:1708;HGNC:1707</td>\n",
       "      <td>926</td>\n",
       "      <td>HGNC:1707</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10090</td>\n",
       "      <td>12593</td>\n",
       "      <td>Cdyl</td>\n",
       "      <td>None</td>\n",
       "      <td>CDY2B;CDY2A;CDY1...</td>\n",
       "      <td>203611;9426;2531...</td>\n",
       "      <td>HGNC:23921;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>18</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10116</td>\n",
       "      <td>24895</td>\n",
       "      <td>Cyp2a2</td>\n",
       "      <td>CYP2A13</td>\n",
       "      <td>CYP2A7;CYP2A6;CY...</td>\n",
       "      <td>1549;1548;1553</td>\n",
       "      <td>HGNC:2611;HGNC:2...</td>\n",
       "      <td>1553</td>\n",
       "      <td>HGNC:2608</td>\n",
       "      <td>0</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10116</td>\n",
       "      <td>24299</td>\n",
       "      <td>Cyp2a3</td>\n",
       "      <td>CYP2A13</td>\n",
       "      <td>CYP2A7;CYP2A6;CY...</td>\n",
       "      <td>1549;1548;1553</td>\n",
       "      <td>HGNC:2611;HGNC:2...</td>\n",
       "      <td>1553</td>\n",
       "      <td>HGNC:2608</td>\n",
       "      <td>0</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10090</td>\n",
       "      <td>13086</td>\n",
       "      <td>Cyp2a4</td>\n",
       "      <td>CYP2A13</td>\n",
       "      <td>CYP2A7;CYP2A6;CY...</td>\n",
       "      <td>1549;1548;1553</td>\n",
       "      <td>HGNC:2611;HGNC:2...</td>\n",
       "      <td>1553</td>\n",
       "      <td>HGNC:2608</td>\n",
       "      <td>0</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10090</td>\n",
       "      <td>13087</td>\n",
       "      <td>Cyp2a5</td>\n",
       "      <td>CYP2A13</td>\n",
       "      <td>CYP2A7;CYP2A6;CY...</td>\n",
       "      <td>1549;1548;1553</td>\n",
       "      <td>HGNC:2611;HGNC:2...</td>\n",
       "      <td>1553</td>\n",
       "      <td>HGNC:2608</td>\n",
       "      <td>0</td>\n",
       "      <td>134</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10090</td>\n",
       "      <td>13112</td>\n",
       "      <td>Cyp3a11</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>1576</td>\n",
       "      <td>HGNC:2637</td>\n",
       "      <td>0</td>\n",
       "      <td>281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10090</td>\n",
       "      <td>13113</td>\n",
       "      <td>Cyp3a13</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>1576</td>\n",
       "      <td>HGNC:2637</td>\n",
       "      <td>0</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10090</td>\n",
       "      <td>13114</td>\n",
       "      <td>Cyp3a16</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>1576</td>\n",
       "      <td>HGNC:2637</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10116</td>\n",
       "      <td>266682</td>\n",
       "      <td>Cyp3a2</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>1576</td>\n",
       "      <td>HGNC:2637</td>\n",
       "      <td>0</td>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10116</td>\n",
       "      <td>25642</td>\n",
       "      <td>Cyp3a23-3a1</td>\n",
       "      <td>None</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>65</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10090</td>\n",
       "      <td>56388</td>\n",
       "      <td>Cyp3a25</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>1576</td>\n",
       "      <td>HGNC:2637</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10090</td>\n",
       "      <td>53973</td>\n",
       "      <td>Cyp3a41a</td>\n",
       "      <td>None</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10090</td>\n",
       "      <td>337924</td>\n",
       "      <td>Cyp3a44</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>1576</td>\n",
       "      <td>HGNC:2637</td>\n",
       "      <td>0</td>\n",
       "      <td>23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10116</td>\n",
       "      <td>170509</td>\n",
       "      <td>Cyp3a62</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>1576</td>\n",
       "      <td>HGNC:2637</td>\n",
       "      <td>0</td>\n",
       "      <td>43</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10116</td>\n",
       "      <td>171352</td>\n",
       "      <td>Cyp3a9</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A43;CYP3A5;C...</td>\n",
       "      <td>64816;1577;1576;...</td>\n",
       "      <td>HGNC:17450;HGNC:...</td>\n",
       "      <td>1576</td>\n",
       "      <td>HGNC:2637</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10116</td>\n",
       "      <td>50549</td>\n",
       "      <td>Cyp4a1</td>\n",
       "      <td>CYP4A11</td>\n",
       "      <td>CYP4A22;CYP4F11;...</td>\n",
       "      <td>284541;57834;157...</td>\n",
       "      <td>HGNC:20575;HGNC:...</td>\n",
       "      <td>1579</td>\n",
       "      <td>HGNC:2642</td>\n",
       "      <td>0</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10090</td>\n",
       "      <td>13117</td>\n",
       "      <td>Cyp4a10</td>\n",
       "      <td>CYP4A11</td>\n",
       "      <td>CYP4A22;CYP4F11;...</td>\n",
       "      <td>284541;57834;157...</td>\n",
       "      <td>HGNC:20575;HGNC:...</td>\n",
       "      <td>1579</td>\n",
       "      <td>HGNC:2642</td>\n",
       "      <td>0</td>\n",
       "      <td>73</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10090</td>\n",
       "      <td>666168</td>\n",
       "      <td>Cyp4a31</td>\n",
       "      <td>CYP4A11</td>\n",
       "      <td>CYP4A22;CYP4F11;...</td>\n",
       "      <td>284541;57834;157...</td>\n",
       "      <td>HGNC:20575;HGNC:...</td>\n",
       "      <td>1579</td>\n",
       "      <td>HGNC:2642</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10090</td>\n",
       "      <td>100040843</td>\n",
       "      <td>Cyp4a32</td>\n",
       "      <td>CYP4A11</td>\n",
       "      <td>CYP4A22;CYP4F11;...</td>\n",
       "      <td>284541;57834;157...</td>\n",
       "      <td>HGNC:20575;HGNC:...</td>\n",
       "      <td>1579</td>\n",
       "      <td>HGNC:2642</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10090</td>\n",
       "      <td>68009</td>\n",
       "      <td>Defa20</td>\n",
       "      <td>DEFA1</td>\n",
       "      <td>DEFA6;DEFA5;DEFA...</td>\n",
       "      <td>1671;1670;1669;7...</td>\n",
       "      <td>HGNC:2765;HGNC:2...</td>\n",
       "      <td>1667</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10090</td>\n",
       "      <td>442835</td>\n",
       "      <td>Defb22</td>\n",
       "      <td>DEFB104A</td>\n",
       "      <td>DEFB121;DEFB134;...</td>\n",
       "      <td>245934;613211;14...</td>\n",
       "      <td>HGNC:18101;HGNC:...</td>\n",
       "      <td>140596</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    TaxID    Gene_ID  gene_symbol unique_human_gene_symbol  \\\n",
       "0   10090     171281        Acot3                ACOT1       \n",
       "1   10090     217698        Acot5                ACOT1       \n",
       "2   10090     233799        Acsm2               ACSM2A       \n",
       "3   10090      11522         Adh1                ADH1A       \n",
       "4   10090      11648         Akp3                 None       \n",
       "5   10090     223672       Apol9a                APOL1       \n",
       "6   10090      20305         Ccl6                CCL15       \n",
       "7   10090      20308         Ccl9                CCL15       \n",
       "8   10090      12526        Cd8b1                 CD8B       \n",
       "9   10090      12593         Cdyl                 None       \n",
       "10  10116      24895       Cyp2a2              CYP2A13       \n",
       "11  10116      24299       Cyp2a3              CYP2A13       \n",
       "12  10090      13086       Cyp2a4              CYP2A13       \n",
       "13  10090      13087       Cyp2a5              CYP2A13       \n",
       "14  10090      13112      Cyp3a11               CYP3A4       \n",
       "15  10090      13113      Cyp3a13               CYP3A4       \n",
       "16  10090      13114      Cyp3a16               CYP3A4       \n",
       "17  10116     266682       Cyp3a2               CYP3A4       \n",
       "18  10116      25642  Cyp3a23-3a1                 None       \n",
       "19  10090      56388      Cyp3a25               CYP3A4       \n",
       "20  10090      53973     Cyp3a41a                 None       \n",
       "21  10090     337924      Cyp3a44               CYP3A4       \n",
       "22  10116     170509      Cyp3a62               CYP3A4       \n",
       "23  10116     171352       Cyp3a9               CYP3A4       \n",
       "24  10116      50549       Cyp4a1              CYP4A11       \n",
       "25  10090      13117      Cyp4a10              CYP4A11       \n",
       "26  10090     666168      Cyp4a31              CYP4A11       \n",
       "27  10090  100040843      Cyp4a32              CYP4A11       \n",
       "28  10090      68009       Defa20                DEFA1       \n",
       "29  10090     442835       Defb22             DEFB104A       \n",
       "\n",
       "      human_gene_symbol        human_gene_ID              HGNC_ID  \\\n",
       "0           ACOT2;ACOT1         10965;641371  HGNC:18431;HGNC:...   \n",
       "1           ACOT2;ACOT1         10965;641371  HGNC:18431;HGNC:...   \n",
       "2         ACSM2A;ACSM2B        123876;348158  HGNC:32017;HGNC:...   \n",
       "3     ADH1B;ADH1A;ADH1C          125;124;126  HGNC:250;HGNC:24...   \n",
       "4        ALPP;ALPG;ALPI          250;251;248  HGNC:439;HGNC:44...   \n",
       "5   APOL1;APOL3;APOL...  8542;80833;80832...  HGNC:618;HGNC:14...   \n",
       "6           CCL15;CCL23            6359;6368  HGNC:10613;HGNC:...   \n",
       "7           CCL15;CCL23            6359;6368  HGNC:10613;HGNC:...   \n",
       "8            CD8B2;CD8B              927;926  HGNC:1708;HGNC:1707   \n",
       "9   CDY2B;CDY2A;CDY1...  203611;9426;2531...  HGNC:23921;HGNC:...   \n",
       "10  CYP2A7;CYP2A6;CY...       1549;1548;1553  HGNC:2611;HGNC:2...   \n",
       "11  CYP2A7;CYP2A6;CY...       1549;1548;1553  HGNC:2611;HGNC:2...   \n",
       "12  CYP2A7;CYP2A6;CY...       1549;1548;1553  HGNC:2611;HGNC:2...   \n",
       "13  CYP2A7;CYP2A6;CY...       1549;1548;1553  HGNC:2611;HGNC:2...   \n",
       "14  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "15  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "16  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "17  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "18  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "19  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "20  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "21  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "22  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "23  CYP3A43;CYP3A5;C...  64816;1577;1576;...  HGNC:17450;HGNC:...   \n",
       "24  CYP4A22;CYP4F11;...  284541;57834;157...  HGNC:20575;HGNC:...   \n",
       "25  CYP4A22;CYP4F11;...  284541;57834;157...  HGNC:20575;HGNC:...   \n",
       "26  CYP4A22;CYP4F11;...  284541;57834;157...  HGNC:20575;HGNC:...   \n",
       "27  CYP4A22;CYP4F11;...  284541;57834;157...  HGNC:20575;HGNC:...   \n",
       "28  DEFA6;DEFA5;DEFA...  1671;1670;1669;7...  HGNC:2765;HGNC:2...   \n",
       "29  DEFB121;DEFB134;...  245934;613211;14...  HGNC:18101;HGNC:...   \n",
       "\n",
       "   unique_human_gene_ID unique_HGNC_ID  TF_count  TG_count  \n",
       "0                641371     HGNC:33128         0         1  \n",
       "1                641371     HGNC:33128         0         1  \n",
       "2                123876     HGNC:32017         0         1  \n",
       "3                   124       HGNC:249         0         9  \n",
       "4                  None           None         0         1  \n",
       "5                  8542       HGNC:618         0         1  \n",
       "6                  6359     HGNC:10613         0        10  \n",
       "7                  6359     HGNC:10613         0        20  \n",
       "8                   926      HGNC:1707         0         1  \n",
       "9                  None           None        18         3  \n",
       "10                 1553      HGNC:2608         0         7  \n",
       "11                 1553      HGNC:2608         0         8  \n",
       "12                 1553      HGNC:2608         0        20  \n",
       "13                 1553      HGNC:2608         0       134  \n",
       "14                 1576      HGNC:2637         0       281  \n",
       "15                 1576      HGNC:2637         0         5  \n",
       "16                 1576      HGNC:2637         0         1  \n",
       "17                 1576      HGNC:2637         0        34  \n",
       "18                 None           None         0        65  \n",
       "19                 1576      HGNC:2637         0         1  \n",
       "20                 None           None         0        23  \n",
       "21                 1576      HGNC:2637         0        23  \n",
       "22                 1576      HGNC:2637         0        43  \n",
       "23                 1576      HGNC:2637         0         2  \n",
       "24                 1579      HGNC:2642         0        31  \n",
       "25                 1579      HGNC:2642         0        73  \n",
       "26                 1579      HGNC:2642         0         1  \n",
       "27                 1579      HGNC:2642         0         1  \n",
       "28                 1667           None         0         2  \n",
       "29               140596           None         0         0  "
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# CHECKINGS ON ORTHOLOGS_DF\n",
    "\n",
    "# Unique human mappings\n",
    "m = orthologs_df['human_gene_ID'].str.contains(';')\n",
    "m1 = m & (orthologs_df['unique_human_gene_ID'] != 'None')\n",
    "m2 = m1 & (orthologs_df['unique_human_gene_symbol'].str.upper() == orthologs_df['gene_symbol'].str.upper())\n",
    "\n",
    "print(f\"\"\"\\\n",
    "All gene IDs in ExTRI2: {len(orthologs_df)}    \n",
    "Missing gene symbols: {orthologs_df['gene_symbol'].str.contains('None').sum()}\n",
    "\n",
    "One-to-one human gene mapping: {(~m & (orthologs_df['human_gene_ID'] != 'None')).sum()}\n",
    "One-to-many human gene mapping: {m.sum()}\n",
    "- With a unique_human_gene_ID assigned: {m1.sum()} ({(m & ~m1).sum()} unassigned)\n",
    "- Assigned through exact match: {m2.sum()} ({(m1 & ~m2).sum()} through 1st family member rule)\n",
    "Missing human gene mapping: {(orthologs_df['human_gene_ID'] == 'None').sum()}\n",
    "Missing HGNC IDs: {(orthologs_df['HGNC_ID'] == 'None').sum()}\n",
    "\"\"\")\n",
    "\n",
    "# Precompute counts in ExTRI2_df\n",
    "tf_counts = ExTRI2_df['TF Symbol'].value_counts()\n",
    "tg_counts = ExTRI2_df['TG Symbol'].value_counts()\n",
    "\n",
    "# Map counts\n",
    "orthologs_df['TF_count'] = orthologs_df['gene_symbol'].map(tf_counts).fillna(0).astype(int)\n",
    "orthologs_df['TG_count'] = orthologs_df['gene_symbol'].map(tg_counts).fillna(0).astype(int)\n",
    "\n",
    "# Save orthologs with unique human ID either unassigned or assigned through 1st family member rule\n",
    "m = (orthologs_df['human_gene_ID'].str.contains(';')) & (orthologs_df['unique_human_gene_symbol'].str.upper() != orthologs_df['gene_symbol'].str.upper())\n",
    "\n",
    "cols_first = ['TaxID', 'Gene_ID', 'gene_symbol', 'unique_human_gene_symbol', 'human_gene_symbol', 'human_gene_ID', 'HGNC_ID']\n",
    "orthologs_to_check = orthologs_df[m][cols_first + [c for c in orthologs_df.columns if c not in cols_first]].sort_values(by=['gene_symbol', 'TaxID']).reset_index(drop=True)\n",
    "print(f\"Number of orthologs to check: {len(orthologs_to_check)}\")\n",
    "\n",
    "orthologs_to_check.to_csv(config['data_p'] + 'validation/orthologs_to_check.tsv', sep='\\t', index=False)\n",
    "orthologs_to_check[:30]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>TaxID</th>\n",
       "      <th>Gene_ID</th>\n",
       "      <th>gene_symbol</th>\n",
       "      <th>unique_human_gene_symbol</th>\n",
       "      <th>human_gene_symbol</th>\n",
       "      <th>human_gene_ID</th>\n",
       "      <th>HGNC_ID</th>\n",
       "      <th>unique_HGNC_ID</th>\n",
       "      <th>TF_count</th>\n",
       "      <th>TG_count</th>\n",
       "      <th>unique_human_gene_ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10090</td>\n",
       "      <td>171281</td>\n",
       "      <td>Acot3</td>\n",
       "      <td>ACOT1</td>\n",
       "      <td>ACOT2;ACOT1</td>\n",
       "      <td>10965;641371</td>\n",
       "      <td>HGNC:18431;HGNC:33128</td>\n",
       "      <td>HGNC:33128</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>641371</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10090</td>\n",
       "      <td>217698</td>\n",
       "      <td>Acot5</td>\n",
       "      <td>ACOT1</td>\n",
       "      <td>ACOT2;ACOT1</td>\n",
       "      <td>10965;641371</td>\n",
       "      <td>HGNC:18431;HGNC:33128</td>\n",
       "      <td>HGNC:33128</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>641371</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10090</td>\n",
       "      <td>233799</td>\n",
       "      <td>Acsm2</td>\n",
       "      <td>ACSM2A</td>\n",
       "      <td>ACSM2B;ACSM2A</td>\n",
       "      <td>348158;123876</td>\n",
       "      <td>HGNC:30931;HGNC:32017</td>\n",
       "      <td>HGNC:32017</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>123876</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10090</td>\n",
       "      <td>11522</td>\n",
       "      <td>Adh1</td>\n",
       "      <td>ADH1A</td>\n",
       "      <td>ADH1B;ADH1C;ADH1A</td>\n",
       "      <td>125;126;124</td>\n",
       "      <td>HGNC:250;HGNC:251;HGNC:249</td>\n",
       "      <td>HGNC:249</td>\n",
       "      <td>0</td>\n",
       "      <td>9</td>\n",
       "      <td>124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10090</td>\n",
       "      <td>11648</td>\n",
       "      <td>Akp3</td>\n",
       "      <td>None</td>\n",
       "      <td>ALPP;ALPI;ALPG</td>\n",
       "      <td>250;248;251</td>\n",
       "      <td>HGNC:439;HGNC:437;HGNC:441</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10090</td>\n",
       "      <td>223672</td>\n",
       "      <td>Apol9a</td>\n",
       "      <td>APOL1</td>\n",
       "      <td>APOL3;APOL1;APOL2;APOL4</td>\n",
       "      <td>80833;8542;23780;80832</td>\n",
       "      <td>HGNC:14868;HGNC:618;HGNC:619;HGNC:14867</td>\n",
       "      <td>HGNC:618</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>8542</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>10090</td>\n",
       "      <td>20305</td>\n",
       "      <td>Ccl6</td>\n",
       "      <td>CCL15</td>\n",
       "      <td>CCL23;CCL15</td>\n",
       "      <td>6368;6359</td>\n",
       "      <td>HGNC:10622;HGNC:10613</td>\n",
       "      <td>HGNC:10613</td>\n",
       "      <td>0</td>\n",
       "      <td>10</td>\n",
       "      <td>6359</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>10090</td>\n",
       "      <td>20308</td>\n",
       "      <td>Ccl9</td>\n",
       "      <td>CCL15</td>\n",
       "      <td>CCL23;CCL15</td>\n",
       "      <td>6368;6359</td>\n",
       "      <td>HGNC:10622;HGNC:10613</td>\n",
       "      <td>HGNC:10613</td>\n",
       "      <td>0</td>\n",
       "      <td>20</td>\n",
       "      <td>6359</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>10090</td>\n",
       "      <td>12526</td>\n",
       "      <td>Cd8b1</td>\n",
       "      <td>CD8B</td>\n",
       "      <td>CD8B;CD8B2</td>\n",
       "      <td>926;927</td>\n",
       "      <td>HGNC:1707;HGNC:1708</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>926</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10090</td>\n",
       "      <td>12593</td>\n",
       "      <td>Cdyl</td>\n",
       "      <td>None</td>\n",
       "      <td>CDY1B;CDY2B;CDY2A;CDY1</td>\n",
       "      <td>253175;203611;9426;9085</td>\n",
       "      <td>HGNC:23920;HGNC:23921;HGNC:1810;HGNC...</td>\n",
       "      <td>HGNC:1809</td>\n",
       "      <td>18</td>\n",
       "      <td>3</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10116</td>\n",
       "      <td>24895</td>\n",
       "      <td>Cyp2a2</td>\n",
       "      <td>CYP2A13</td>\n",
       "      <td>CYP2A7;CYP2A6;CYP2A13</td>\n",
       "      <td>1549;1548;1553</td>\n",
       "      <td>HGNC:2611;HGNC:2610;HGNC:2608</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>7</td>\n",
       "      <td>1553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>10116</td>\n",
       "      <td>24299</td>\n",
       "      <td>Cyp2a3</td>\n",
       "      <td>CYP2A13</td>\n",
       "      <td>CYP2A7;CYP2A6;CYP2A13</td>\n",
       "      <td>1549;1548;1553</td>\n",
       "      <td>HGNC:2611;HGNC:2610;HGNC:2608</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>8</td>\n",
       "      <td>1553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>10090</td>\n",
       "      <td>13086</td>\n",
       "      <td>Cyp2a4</td>\n",
       "      <td>CYP2A13</td>\n",
       "      <td>CYP2A7;CYP2A6;CYP2A13</td>\n",
       "      <td>1549;1548;1553</td>\n",
       "      <td>HGNC:2611;HGNC:2610;HGNC:2608</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>20</td>\n",
       "      <td>1553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>10090</td>\n",
       "      <td>13087</td>\n",
       "      <td>Cyp2a5</td>\n",
       "      <td>CYP2A13</td>\n",
       "      <td>CYP2A7;CYP2A6;CYP2A13</td>\n",
       "      <td>1549;1548;1553</td>\n",
       "      <td>HGNC:2611;HGNC:2610;HGNC:2608</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>134</td>\n",
       "      <td>1553</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>10090</td>\n",
       "      <td>13112</td>\n",
       "      <td>Cyp3a11</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>281</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>10090</td>\n",
       "      <td>13113</td>\n",
       "      <td>Cyp3a13</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>5</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10090</td>\n",
       "      <td>13114</td>\n",
       "      <td>Cyp3a16</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>10116</td>\n",
       "      <td>266682</td>\n",
       "      <td>Cyp3a2</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>34</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>10116</td>\n",
       "      <td>25642</td>\n",
       "      <td>Cyp3a23-3a1</td>\n",
       "      <td>None</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>65</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>10090</td>\n",
       "      <td>56388</td>\n",
       "      <td>Cyp3a25</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>10090</td>\n",
       "      <td>53973</td>\n",
       "      <td>Cyp3a41a</td>\n",
       "      <td>None</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>23</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>10090</td>\n",
       "      <td>337924</td>\n",
       "      <td>Cyp3a44</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>23</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>10116</td>\n",
       "      <td>170509</td>\n",
       "      <td>Cyp3a62</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>43</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>10116</td>\n",
       "      <td>171352</td>\n",
       "      <td>Cyp3a9</td>\n",
       "      <td>CYP3A4</td>\n",
       "      <td>CYP3A5;CYP3A43;CYP3A4;CYP3A7</td>\n",
       "      <td>1577;64816;1576;1551</td>\n",
       "      <td>HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>1576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>10116</td>\n",
       "      <td>50549</td>\n",
       "      <td>Cyp4a1</td>\n",
       "      <td>CYP4A11</td>\n",
       "      <td>CYP4A11;CYP4F12;CYP4F11;CYP4A22</td>\n",
       "      <td>1579;66002;57834;284541</td>\n",
       "      <td>HGNC:2642;HGNC:18857;HGNC:13265;HGNC...</td>\n",
       "      <td>HGNC:2642</td>\n",
       "      <td>0</td>\n",
       "      <td>31</td>\n",
       "      <td>1579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>10090</td>\n",
       "      <td>13117</td>\n",
       "      <td>Cyp4a10</td>\n",
       "      <td>CYP4A11</td>\n",
       "      <td>CYP4A11;CYP4F12;CYP4F11;CYP4A22</td>\n",
       "      <td>1579;66002;57834;284541</td>\n",
       "      <td>HGNC:2642;HGNC:18857;HGNC:13265;HGNC...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>73</td>\n",
       "      <td>1579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>10090</td>\n",
       "      <td>666168</td>\n",
       "      <td>Cyp4a31</td>\n",
       "      <td>CYP4A11</td>\n",
       "      <td>CYP4A11;CYP4F12;CYP4F11;CYP4A22</td>\n",
       "      <td>1579;66002;57834;284541</td>\n",
       "      <td>HGNC:2642;HGNC:18857;HGNC:13265;HGNC...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>10090</td>\n",
       "      <td>100040843</td>\n",
       "      <td>Cyp4a32</td>\n",
       "      <td>CYP4A11</td>\n",
       "      <td>CYP4A11;CYP4F12;CYP4F11;CYP4A22</td>\n",
       "      <td>1579;66002;57834;284541</td>\n",
       "      <td>HGNC:2642;HGNC:18857;HGNC:13265;HGNC...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1579</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10090</td>\n",
       "      <td>68009</td>\n",
       "      <td>Defa20</td>\n",
       "      <td>DEFA1</td>\n",
       "      <td>DEFA5;DEFA1;DEFA3;DEFA1B;DEFA4;DEFA6</td>\n",
       "      <td>1670;1667;1668;728358;1669;1671</td>\n",
       "      <td>HGNC:2764;None;None;HGNC:33596;HGNC:...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>1667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>10090</td>\n",
       "      <td>442835</td>\n",
       "      <td>Defb22</td>\n",
       "      <td>DEFB104A</td>\n",
       "      <td>DEFB121;DEFB108B;DEFB127;LOC12490674...</td>\n",
       "      <td>245934;245911;140850;124906746;50361...</td>\n",
       "      <td>HGNC:18101;HGNC:29966;HGNC:16206;Non...</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>140596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>10090</td>\n",
       "      <td>27358</td>\n",
       "      <td>Defb3</td>\n",
       "      <td>DEFB4A</td>\n",
       "      <td>DEFB4A;LOC124906748;DEFB4B</td>\n",
       "      <td>1673;124906748;100289462</td>\n",
       "      <td>None;HGNC:30193;None</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>5</td>\n",
       "      <td>1673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>10090</td>\n",
       "      <td>56519</td>\n",
       "      <td>Defb4</td>\n",
       "      <td>DEFB4A</td>\n",
       "      <td>DEFB4A;LOC124906748;DEFB4B</td>\n",
       "      <td>1673;124906748;100289462</td>\n",
       "      <td>None;HGNC:30193;None</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>5</td>\n",
       "      <td>1673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>10116</td>\n",
       "      <td>64389</td>\n",
       "      <td>Defb4</td>\n",
       "      <td>DEFB4A</td>\n",
       "      <td>DEFB4A;LOC124906748;DEFB4B</td>\n",
       "      <td>1673;124906748;100289462</td>\n",
       "      <td>None;HGNC:30193;None</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>5</td>\n",
       "      <td>1673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10116</td>\n",
       "      <td>100361909</td>\n",
       "      <td>Ear1</td>\n",
       "      <td>None</td>\n",
       "      <td>RNASE3;RNASE2</td>\n",
       "      <td>6037;6036</td>\n",
       "      <td>HGNC:10046;HGNC:10045</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>10090</td>\n",
       "      <td>93725</td>\n",
       "      <td>Ear10</td>\n",
       "      <td>None</td>\n",
       "      <td>RNASE3;RNASE2</td>\n",
       "      <td>6037;6036</td>\n",
       "      <td>HGNC:10046;HGNC:10045</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>10090</td>\n",
       "      <td>13587</td>\n",
       "      <td>Ear2</td>\n",
       "      <td>None</td>\n",
       "      <td>RNASE3;RNASE2</td>\n",
       "      <td>6037;6036</td>\n",
       "      <td>HGNC:10046;HGNC:10045</td>\n",
       "      <td>None</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10090</td>\n",
       "      <td>14131</td>\n",
       "      <td>Fcgr3</td>\n",
       "      <td>FCGR2A</td>\n",
       "      <td>FCGR2B;FCGR2A</td>\n",
       "      <td>2213;2212</td>\n",
       "      <td>HGNC:3618;HGNC:3616</td>\n",
       "      <td>HGNC:3616</td>\n",
       "      <td>0</td>\n",
       "      <td>6</td>\n",
       "      <td>2212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10090</td>\n",
       "      <td>246256</td>\n",
       "      <td>Fcgr4</td>\n",
       "      <td>FCGR3A</td>\n",
       "      <td>LOC124905743;FCGR3A;FCGR3B</td>\n",
       "      <td>124905743;2214;2215</td>\n",
       "      <td>HGNC:3620;HGNC:3619;HGNC:3620</td>\n",
       "      <td>HGNC:3619</td>\n",
       "      <td>0</td>\n",
       "      <td>4</td>\n",
       "      <td>2214</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>10090</td>\n",
       "      <td>14599</td>\n",
       "      <td>Gh</td>\n",
       "      <td>GH1</td>\n",
       "      <td>CSH2;CSHL1;GH1;GH2;CSH1</td>\n",
       "      <td>1443;1444;2688;2689;1442</td>\n",
       "      <td>HGNC:2441;None;HGNC:4261;HGNC:4262;H...</td>\n",
       "      <td>HGNC:4261</td>\n",
       "      <td>0</td>\n",
       "      <td>103</td>\n",
       "      <td>2688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>10090</td>\n",
       "      <td>15081</td>\n",
       "      <td>H3f3b</td>\n",
       "      <td>None</td>\n",
       "      <td>H3Y2;H3Y1</td>\n",
       "      <td>340096;391769</td>\n",
       "      <td>HGNC:43734;HGNC:43735</td>\n",
       "      <td>None</td>\n",
       "      <td>4</td>\n",
       "      <td>4</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    TaxID    Gene_ID  gene_symbol unique_human_gene_symbol  \\\n",
       "0   10090     171281        Acot3                    ACOT1   \n",
       "1   10090     217698        Acot5                    ACOT1   \n",
       "2   10090     233799        Acsm2                   ACSM2A   \n",
       "3   10090      11522         Adh1                    ADH1A   \n",
       "4   10090      11648         Akp3                     None   \n",
       "5   10090     223672       Apol9a                    APOL1   \n",
       "6   10090      20305         Ccl6                    CCL15   \n",
       "7   10090      20308         Ccl9                    CCL15   \n",
       "8   10090      12526        Cd8b1                     CD8B   \n",
       "9   10090      12593         Cdyl                     None   \n",
       "10  10116      24895       Cyp2a2                  CYP2A13   \n",
       "11  10116      24299       Cyp2a3                  CYP2A13   \n",
       "12  10090      13086       Cyp2a4                  CYP2A13   \n",
       "13  10090      13087       Cyp2a5                  CYP2A13   \n",
       "14  10090      13112      Cyp3a11                   CYP3A4   \n",
       "15  10090      13113      Cyp3a13                   CYP3A4   \n",
       "16  10090      13114      Cyp3a16                   CYP3A4   \n",
       "17  10116     266682       Cyp3a2                   CYP3A4   \n",
       "18  10116      25642  Cyp3a23-3a1                     None   \n",
       "19  10090      56388      Cyp3a25                   CYP3A4   \n",
       "20  10090      53973     Cyp3a41a                     None   \n",
       "21  10090     337924      Cyp3a44                   CYP3A4   \n",
       "22  10116     170509      Cyp3a62                   CYP3A4   \n",
       "23  10116     171352       Cyp3a9                   CYP3A4   \n",
       "24  10116      50549       Cyp4a1                  CYP4A11   \n",
       "25  10090      13117      Cyp4a10                  CYP4A11   \n",
       "26  10090     666168      Cyp4a31                  CYP4A11   \n",
       "27  10090  100040843      Cyp4a32                  CYP4A11   \n",
       "28  10090      68009       Defa20                    DEFA1   \n",
       "29  10090     442835       Defb22                 DEFB104A   \n",
       "30  10090      27358        Defb3                   DEFB4A   \n",
       "31  10090      56519        Defb4                   DEFB4A   \n",
       "32  10116      64389        Defb4                   DEFB4A   \n",
       "33  10116  100361909         Ear1                     None   \n",
       "34  10090      93725        Ear10                     None   \n",
       "35  10090      13587         Ear2                     None   \n",
       "36  10090      14131        Fcgr3                   FCGR2A   \n",
       "37  10090     246256        Fcgr4                   FCGR3A   \n",
       "38  10090      14599           Gh                      GH1   \n",
       "39  10090      15081        H3f3b                     None   \n",
       "\n",
       "                          human_gene_symbol  \\\n",
       "0                               ACOT2;ACOT1   \n",
       "1                               ACOT2;ACOT1   \n",
       "2                             ACSM2B;ACSM2A   \n",
       "3                         ADH1B;ADH1C;ADH1A   \n",
       "4                            ALPP;ALPI;ALPG   \n",
       "5                   APOL3;APOL1;APOL2;APOL4   \n",
       "6                               CCL23;CCL15   \n",
       "7                               CCL23;CCL15   \n",
       "8                                CD8B;CD8B2   \n",
       "9                    CDY1B;CDY2B;CDY2A;CDY1   \n",
       "10                    CYP2A7;CYP2A6;CYP2A13   \n",
       "11                    CYP2A7;CYP2A6;CYP2A13   \n",
       "12                    CYP2A7;CYP2A6;CYP2A13   \n",
       "13                    CYP2A7;CYP2A6;CYP2A13   \n",
       "14             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "15             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "16             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "17             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "18             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "19             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "20             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "21             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "22             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "23             CYP3A5;CYP3A43;CYP3A4;CYP3A7   \n",
       "24          CYP4A11;CYP4F12;CYP4F11;CYP4A22   \n",
       "25          CYP4A11;CYP4F12;CYP4F11;CYP4A22   \n",
       "26          CYP4A11;CYP4F12;CYP4F11;CYP4A22   \n",
       "27          CYP4A11;CYP4F12;CYP4F11;CYP4A22   \n",
       "28     DEFA5;DEFA1;DEFA3;DEFA1B;DEFA4;DEFA6   \n",
       "29  DEFB121;DEFB108B;DEFB127;LOC12490674...   \n",
       "30               DEFB4A;LOC124906748;DEFB4B   \n",
       "31               DEFB4A;LOC124906748;DEFB4B   \n",
       "32               DEFB4A;LOC124906748;DEFB4B   \n",
       "33                            RNASE3;RNASE2   \n",
       "34                            RNASE3;RNASE2   \n",
       "35                            RNASE3;RNASE2   \n",
       "36                            FCGR2B;FCGR2A   \n",
       "37               LOC124905743;FCGR3A;FCGR3B   \n",
       "38                  CSH2;CSHL1;GH1;GH2;CSH1   \n",
       "39                                H3Y2;H3Y1   \n",
       "\n",
       "                              human_gene_ID  \\\n",
       "0                              10965;641371   \n",
       "1                              10965;641371   \n",
       "2                             348158;123876   \n",
       "3                               125;126;124   \n",
       "4                               250;248;251   \n",
       "5                    80833;8542;23780;80832   \n",
       "6                                 6368;6359   \n",
       "7                                 6368;6359   \n",
       "8                                   926;927   \n",
       "9                   253175;203611;9426;9085   \n",
       "10                           1549;1548;1553   \n",
       "11                           1549;1548;1553   \n",
       "12                           1549;1548;1553   \n",
       "13                           1549;1548;1553   \n",
       "14                     1577;64816;1576;1551   \n",
       "15                     1577;64816;1576;1551   \n",
       "16                     1577;64816;1576;1551   \n",
       "17                     1577;64816;1576;1551   \n",
       "18                     1577;64816;1576;1551   \n",
       "19                     1577;64816;1576;1551   \n",
       "20                     1577;64816;1576;1551   \n",
       "21                     1577;64816;1576;1551   \n",
       "22                     1577;64816;1576;1551   \n",
       "23                     1577;64816;1576;1551   \n",
       "24                  1579;66002;57834;284541   \n",
       "25                  1579;66002;57834;284541   \n",
       "26                  1579;66002;57834;284541   \n",
       "27                  1579;66002;57834;284541   \n",
       "28          1670;1667;1668;728358;1669;1671   \n",
       "29  245934;245911;140850;124906746;50361...   \n",
       "30                 1673;124906748;100289462   \n",
       "31                 1673;124906748;100289462   \n",
       "32                 1673;124906748;100289462   \n",
       "33                                6037;6036   \n",
       "34                                6037;6036   \n",
       "35                                6037;6036   \n",
       "36                                2213;2212   \n",
       "37                      124905743;2214;2215   \n",
       "38                 1443;1444;2688;2689;1442   \n",
       "39                            340096;391769   \n",
       "\n",
       "                                    HGNC_ID unique_HGNC_ID  TF_count  \\\n",
       "0                     HGNC:18431;HGNC:33128     HGNC:33128         0   \n",
       "1                     HGNC:18431;HGNC:33128     HGNC:33128         0   \n",
       "2                     HGNC:30931;HGNC:32017     HGNC:32017         0   \n",
       "3                HGNC:250;HGNC:251;HGNC:249       HGNC:249         0   \n",
       "4                HGNC:439;HGNC:437;HGNC:441           None         0   \n",
       "5   HGNC:14868;HGNC:618;HGNC:619;HGNC:14867       HGNC:618         0   \n",
       "6                     HGNC:10622;HGNC:10613     HGNC:10613         0   \n",
       "7                     HGNC:10622;HGNC:10613     HGNC:10613         0   \n",
       "8                       HGNC:1707;HGNC:1708           None         0   \n",
       "9   HGNC:23920;HGNC:23921;HGNC:1810;HGNC...      HGNC:1809        18   \n",
       "10            HGNC:2611;HGNC:2610;HGNC:2608           None         0   \n",
       "11            HGNC:2611;HGNC:2610;HGNC:2608           None         0   \n",
       "12            HGNC:2611;HGNC:2610;HGNC:2608           None         0   \n",
       "13            HGNC:2611;HGNC:2610;HGNC:2608           None         0   \n",
       "14  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "15  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "16  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "17  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "18  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "19  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "20  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "21  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "22  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "23  HGNC:2638;HGNC:17450;HGNC:2637;HGNC:...           None         0   \n",
       "24  HGNC:2642;HGNC:18857;HGNC:13265;HGNC...      HGNC:2642         0   \n",
       "25  HGNC:2642;HGNC:18857;HGNC:13265;HGNC...           None         0   \n",
       "26  HGNC:2642;HGNC:18857;HGNC:13265;HGNC...           None         0   \n",
       "27  HGNC:2642;HGNC:18857;HGNC:13265;HGNC...           None         0   \n",
       "28  HGNC:2764;None;None;HGNC:33596;HGNC:...           None         0   \n",
       "29  HGNC:18101;HGNC:29966;HGNC:16206;Non...           None         0   \n",
       "30                     None;HGNC:30193;None           None         0   \n",
       "31                     None;HGNC:30193;None           None         0   \n",
       "32                     None;HGNC:30193;None           None         0   \n",
       "33                    HGNC:10046;HGNC:10045           None         0   \n",
       "34                    HGNC:10046;HGNC:10045           None         0   \n",
       "35                    HGNC:10046;HGNC:10045           None         0   \n",
       "36                      HGNC:3618;HGNC:3616      HGNC:3616         0   \n",
       "37            HGNC:3620;HGNC:3619;HGNC:3620      HGNC:3619         0   \n",
       "38  HGNC:2441;None;HGNC:4261;HGNC:4262;H...      HGNC:4261         0   \n",
       "39                    HGNC:43734;HGNC:43735           None         4   \n",
       "\n",
       "    TG_count unique_human_gene_ID  \n",
       "0          1               641371  \n",
       "1          1               641371  \n",
       "2          1               123876  \n",
       "3          9                  124  \n",
       "4          1                 None  \n",
       "5          1                 8542  \n",
       "6         10                 6359  \n",
       "7         20                 6359  \n",
       "8          1                  926  \n",
       "9          3                 None  \n",
       "10         7                 1553  \n",
       "11         8                 1553  \n",
       "12        20                 1553  \n",
       "13       134                 1553  \n",
       "14       281                 1576  \n",
       "15         5                 1576  \n",
       "16         1                 1576  \n",
       "17        34                 1576  \n",
       "18        65                 None  \n",
       "19         1                 1576  \n",
       "20        23                 None  \n",
       "21        23                 1576  \n",
       "22        43                 1576  \n",
       "23         2                 1576  \n",
       "24        31                 1579  \n",
       "25        73                 1579  \n",
       "26         1                 1579  \n",
       "27         1                 1579  \n",
       "28         2                 1667  \n",
       "29         0               140596  \n",
       "30         5                 1673  \n",
       "31         5                 1673  \n",
       "32         5                 1673  \n",
       "33         1                 None  \n",
       "34         2                 None  \n",
       "35         1                 None  \n",
       "36         6                 2212  \n",
       "37         4                 2214  \n",
       "38       103                 2688  \n",
       "39         4                 None  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def get_unique_human_symbol_index(row):\n",
    "    '''\n",
    "    Get the index of the unique human gene symbol following these rules:\n",
    "    1) Return exact lowercase match, if any\n",
    "    2) First gene family member (e.g. ACSM2A for ACSM2; if multiple, take the one with the smallest numeric suffix)\n",
    "    3) If no match, return None\n",
    "    '''\n",
    "    # Get rodent symbol & human symbols\n",
    "    rodent_symbol = row['gene_symbol'].upper()\n",
    "    human_symbols = row['human_gene_symbol'].upper().split(\";\")\n",
    "\n",
    "    # If there's only one symbol, return index 0\n",
    "    if len(human_symbols) == 1:\n",
    "        return 0\n",
    "\n",
    "    # 1) Exact match (case-insensitive)\n",
    "    for i, c in enumerate(human_symbols):\n",
    "        if c == rodent_symbol:\n",
    "            return i\n",
    "\n",
    "    # 2) Apply first gene family member rule:\n",
    "    # Assumption: gene names are formed by \"[A-Z]+[0-9]*[A-Z]?\"\n",
    "    # Extract family stem and number (e.g. ADH1 -> stem=ADH, number=1)\n",
    "    m = re.match(r'^([A-Z]+)(\\d+)?([A-Z]?)(\\d+)?$', rodent_symbol)\n",
    "    stem = m.group(1) if m else rodent_symbol\n",
    "    num = m.group(2) if (m and m.group(2)) else None\n",
    "    letter = m.group(3) if (m and m.group(3)) else None\n",
    "    num2 = m.group(4) if (m and m.group(4)) else None\n",
    "\n",
    "    # Get human symbols that start with the same stem\n",
    "    candidates = [(i, hs) for i, hs in enumerate(human_symbols) if hs.startswith(stem)]\n",
    "    if not candidates:\n",
    "        return None\n",
    "    \n",
    "    # Prefer candidates that match the stem\n",
    "    for i, hs in candidates:\n",
    "        if hs == stem:\n",
    "            return i\n",
    "        \n",
    "    # If there is a number, prefer same numeric family (e.g. ADH1A over ADH2A)\n",
    "    if num:\n",
    "        same_family = [(i, hs) for i, hs in candidates if hs.startswith(stem + num)]\n",
    "\n",
    "        if same_family:\n",
    "            # Prefer exact stem+num (if present), else smallest suffix\n",
    "            exact = [i for i, hs in same_family if hs == stem + num]\n",
    "            if exact:\n",
    "                return exact[0]\n",
    "            same_family.sort(key=lambda x: x[1])\n",
    "            return same_family[0][0]\n",
    "        \n",
    "    # Otherwise, just pick smallest lexicographic suffix (first family member)\n",
    "    candidates.sort(key=lambda x: x[1])\n",
    "    return candidates[0][0]\n",
    "\n",
    "def assign_unique_human_fields(row):\n",
    "    '''Helper function to assign unique fields based on index'''\n",
    "    idx = get_unique_human_symbol_index(row)\n",
    "\n",
    "    if idx is None:\n",
    "        return pd.Series({\n",
    "            'unique_human_gene_ID': 'None',\n",
    "            'unique_human_gene_symbol':  'None',\n",
    "        })\n",
    "\n",
    "    return pd.Series({\n",
    "        'unique_human_gene_ID': row['human_gene_ID'].split(';')[idx],\n",
    "        'unique_human_gene_symbol': row['human_gene_symbol'].split(';')[idx],\n",
    "    })\n",
    "\n",
    "orthologs_df = orthologs_df.drop(columns=['unique_human_gene_ID', 'unique_human_gene_symbol'])\n",
    "orthologs_df = orthologs_df.join(orthologs_df.apply(assign_unique_human_fields, axis=1))\n",
    "\n",
    "m = (orthologs_df['human_gene_ID'].str.contains(';')) & (orthologs_df['unique_human_gene_symbol'].str.upper() != orthologs_df['gene_symbol'].str.upper())\n",
    "cols_first = ['TaxID', 'Gene_ID', 'gene_symbol', 'unique_human_gene_symbol', 'human_gene_symbol', 'human_gene_ID', 'HGNC_ID']\n",
    "orthologs_to_check = orthologs_df[m][cols_first + [c for c in orthologs_df.columns if c not in cols_first]].sort_values(by=['gene_symbol', 'TaxID']).reset_index(drop=True)\n",
    "\n",
    "pd.set_option('display.max_colwidth', 40)\n",
    "orthologs_to_check[:40]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "TF_type     \n",
       "dbTF|coTF       5\n",
       "coTF|dbTF       1\n",
       "ll_coTF|coTF    1\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>TF_human_Id</th>\n",
       "      <th>TF Symbol</th>\n",
       "      <th>TF_type</th>\n",
       "      <th>TF TaxID</th>\n",
       "      <th>TF_human_symbol</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>10196</td>\n",
       "      <td>PRMT3|Prmt3</td>\n",
       "      <td>dbTF|coTF</td>\n",
       "      <td>9606|10090|10116</td>\n",
       "      <td>PRMT3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>10270</td>\n",
       "      <td>AKAP8|Akap8</td>\n",
       "      <td>dbTF|coTF</td>\n",
       "      <td>9606|10116</td>\n",
       "      <td>AKAP8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>608</th>\n",
       "      <td>1810</td>\n",
       "      <td>DR1|Dr1</td>\n",
       "      <td>dbTF|coTF</td>\n",
       "      <td>9606|10090|10116</td>\n",
       "      <td>DR1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1764</th>\n",
       "      <td>434</td>\n",
       "      <td>a|ASIP</td>\n",
       "      <td>ll_coTF|coTF</td>\n",
       "      <td>10090|9606</td>\n",
       "      <td>ASIP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1902</th>\n",
       "      <td>4855</td>\n",
       "      <td>NOTCH4|Notch4</td>\n",
       "      <td>dbTF|coTF</td>\n",
       "      <td>9606|10116</td>\n",
       "      <td>NOTCH4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2379</th>\n",
       "      <td>56252</td>\n",
       "      <td>YLPM1|Ylpm1</td>\n",
       "      <td>coTF|dbTF</td>\n",
       "      <td>9606|10090</td>\n",
       "      <td>YLPM1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2648</th>\n",
       "      <td>63925</td>\n",
       "      <td>Zfp335|ZNF335</td>\n",
       "      <td>dbTF|coTF</td>\n",
       "      <td>10090|9606|10116</td>\n",
       "      <td>ZNF335</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     TF_human_Id      TF Symbol       TF_type          TF TaxID  \\\n",
       "53         10196    PRMT3|Prmt3     dbTF|coTF  9606|10090|10116   \n",
       "72         10270    AKAP8|Akap8     dbTF|coTF        9606|10116   \n",
       "608         1810        DR1|Dr1     dbTF|coTF  9606|10090|10116   \n",
       "1764         434         a|ASIP  ll_coTF|coTF        10090|9606   \n",
       "1902        4855  NOTCH4|Notch4     dbTF|coTF        9606|10116   \n",
       "2379       56252    YLPM1|Ylpm1     coTF|dbTF        9606|10090   \n",
       "2648       63925  Zfp335|ZNF335     dbTF|coTF  10090|9606|10116   \n",
       "\n",
       "     TF_human_symbol  \n",
       "53             PRMT3  \n",
       "72             AKAP8  \n",
       "608              DR1  \n",
       "1764            ASIP  \n",
       "1902          NOTCH4  \n",
       "2379           YLPM1  \n",
       "2648          ZNF335  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Rows in ExTRI2 affected: 322\n"
     ]
    }
   ],
   "source": [
    "## CHECK THE DISTRUBTION OF TF TYPES PER HUMAN MAPPING\n",
    "# Check whether any TF_human_symbol has multiple TF types from different rodent orthologs \n",
    "agg_funcs = {\n",
    "    'TF Symbol': lambda x: '|'.join(x.unique()),\n",
    "    'TF_type': lambda x: '|'.join(x.unique()),\n",
    "    'TF TaxID': lambda x: '|'.join(x.unique()),\n",
    "    'TF_human_symbol': lambda x: '|'.join(x.unique()),\n",
    "}\n",
    "\n",
    "final_TRI_grouped = ExTRI2_df.groupby('TF_human_Id', as_index=False).agg(agg_funcs)\n",
    "m = final_TRI_grouped['TF_human_Id'].str.contains(';') | final_TRI_grouped['TF_type'].isin(['coTF', 'dbTF', 'll_coTF']) | (final_TRI_grouped['TF_human_Id'] == 'None')\n",
    "display(final_TRI_grouped[~m][['TF_type']].value_counts())\n",
    "display(final_TRI_grouped[~m])\n",
    "print(f\"Rows in ExTRI2 affected: {ExTRI2_df['TF_human_Id'].isin(final_TRI_grouped[~m]['TF_human_Id']).sum()}\")\n",
    "# TODO - Clashes in a few sentences. Decide with Astrid how to solve this problem"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# TODO - Modify \"get_NCBI_TF_IDs.ipynb\", where I used it to create the TF table\n",
    "# And understand how that table was created, and update it too. That will take a while too, probably, yay."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### AP1 & NFKB"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "AP1 and NFKB are dimers, and as such don't have neither a NCBI EntrezID, nor a HGNC symbol. PubTator normalizes them to one of their monomers. Therefore, in `postprocessing.py`, we\n",
    "* Find all dimers incorrectly normalized to monomers using regex\n",
    "* Change the TF metadata to AP1/NFKB. Delete the TG instances (a TG can't be a dimer)\n",
    "* Save a summary of the results and affected sentences in `data/postprocessing/tables`"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Initial exploration"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "### POSTPROCESSING TRI_df\n",
      "We got 6706 different TFs and 26196 different TGs from sentences labelled as TRI\n",
      "Retrieving from Entrez...\n",
      "\n",
      "4967 sentences are dropped as their TG is not normalised\n",
      "\n",
      "38287 rows (4.23%) will have its TF renormalized to NFKB\n",
      "6327 rows (0.70%) will be dropped as the TG corresponds to NFKB\n",
      "9003 rows (1.00%) will have its TF renormalized to AP1\n",
      "1858 rows (0.21%) will be dropped as the TG corresponds to AP1\n",
      "Breakdown by NCBI Symbol saved in ../../data/postprocessing/tables/AP1_NFKB_breakdown.tsv\n"
     ]
    }
   ],
   "source": [
    "# POSTPROCESSING BEFORE RENORMALISATION & DISCARDING WERE IMPLEMENTED\n",
    "def half_postprocess(ExTRI2_df: pd.DataFrame, TRI_sents: bool, config: dict) -> pd.DataFrame:\n",
    "    '''same as postprocess but before the renormalisation & discarding'''\n",
    "\n",
    "    df_type = 'TRI' if TRI_sents else 'nonTRI'\n",
    "    print(f'### POSTPROCESSING {df_type}_df')\n",
    "\n",
    "    # Retrieve Symbol & TaxID from Entrez\n",
    "    save_Symbol_TaxID_dict(ExTRI2_df, config[f'EntrezID_to_Symbol_{df_type}_p'])\n",
    "\n",
    "    # Filter & add metadata\n",
    "    if TRI_sents:\n",
    "        remove_duplicates(ExTRI2_df)\n",
    "    ExTRI2_df = add_symbols_TaxID(ExTRI2_df, config[f'EntrezID_to_Symbol_{df_type}_p'])\n",
    "    add_TF_type(ExTRI2_df, config)\n",
    "    ExTRI2_df = drop_GTFs(ExTRI2_df)\n",
    "    ExTRI2_df = remove_other_species(ExTRI2_df, TaxID)\n",
    "\n",
    "    # Fix AP1 & NFKB normalisations\n",
    "    ExTRI2_df = fix_NFKB_AP1(ExTRI2_df, config)\n",
    "\n",
    "    return ExTRI2_df\n",
    "\n",
    "config = load_config()\n",
    "\n",
    "# Load raw dataframe\n",
    "TRI_df = load_preprocess_df(config['raw_TRI_p'])\n",
    "\n",
    "# Postprocess (without renormalisation/discarding)\n",
    "TRI_df = half_postprocess(TRI_df, TRI_sents=True,  config=config)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AP1 TF Unique Entities: 155\n",
      "NFKB TF Unique Entities: 115\n"
     ]
    }
   ],
   "source": [
    "# GET NUMBER OF UNIQUE ENTITY NAMES\n",
    "ExTRI2_df = TRI_df\n",
    "\n",
    "m_AP1 = ExTRI2_df['TF Symbol'].str.contains('|'.join(('FOS', 'JUN')), case=False)\n",
    "\n",
    "NFKB_symbols = {'NFKB1', 'NFKB2', 'RELA', 'RELB'}\n",
    "m_NFKB = ExTRI2_df['TF Symbol'].str.upper().isin(NFKB_symbols)\n",
    "\n",
    "print(\"AP1 TF Unique Entities:\", len(ExTRI2_df[m_AP1]['TF'].unique()))\n",
    "# print(ExTRI2_df[m_AP1]['TF'].unique())\n",
    "print(\"NFKB TF Unique Entities:\", len(ExTRI2_df[m_NFKB]['TF'].unique()))\n",
    "# print(ExTRI2_df[m_NFKB]['TF'].unique())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# POST-PROCESSING FUNCTIONS\n",
    "def print_symbol_counts_side_by_side(m_template, max_counts = 10):\n",
    "    'Print symbol counts of m_template for TF & TG'\n",
    "    results = []\n",
    "    for T in ('TF', 'TG'):\n",
    "        m = eval(m_template.replace('{T}', T))\n",
    "        T_lines = TRI_df[m][f'{T} Symbol'].value_counts()[:max_counts].to_string().split('\\n')\n",
    "        results.append(T_lines)\n",
    "\n",
    "    # Print the two tables side by side\n",
    "    for tf_line, tg_line in zip(*results):\n",
    "        print(f\"{tf_line:<35} {tg_line}\")\n",
    "\n",
    "def print_dubious_pairs_TFTGcounts_side_by_side(dubious_pairs):\n",
    "    '''Print counts of TF&TG of 3 different symbols side by side'''\n",
    "    all_tables = []\n",
    "    for p in dubious_pairs:\n",
    "        results = []\n",
    "        for T in ('TF', 'TG'):\n",
    "            m = TRI_df[f'{T} Symbol'].isin([';'.join((p[0], p[1])), ';'.join((p[1], p[0]))])\n",
    "            T_counts = TRI_df[m][f'{T}'].value_counts().rename(f'{T} count')[:10]\n",
    "            results.append(T_counts)\n",
    "\n",
    "        # Merge the TF and TG counts on the same index\n",
    "        merged_df = pd.concat(results, axis=1).fillna(0).astype(int)\n",
    "        all_tables.append(merged_df)\n",
    "\n",
    "    # Convert each table to a string and split by lines\n",
    "    table_strings = [table.to_string().split('\\n') for table in all_tables]\n",
    "\n",
    "    # Use itertools.zip_longest to handle tables with different lengths\n",
    "    for lines in itertools.zip_longest(*table_strings, fillvalue=''):\n",
    "        # Print each line of the three tables side by side\n",
    "        print(f\"{lines[0]:<40} {lines[1]:<40} {lines[2]}\")\n",
    "\n",
    "def print_TF_TG_counts_side_by_side(title, m_template, sep=40):\n",
    "    bold(title)\n",
    "    counts = []\n",
    "    for T in ('TF', 'TG'):\n",
    "        m = eval(m_template)\n",
    "        T_lines = TRI_df[m][[f'{T}', f'{T} Symbol']].value_counts().to_string().split('\\n')\n",
    "        counts.append(T_lines)\n",
    "    \n",
    "    for tf_line, tg_line in itertools.zip_longest(*counts, fillvalue=''):\n",
    "        print(f\"{tf_line:<{sep}} {tg_line}\")\n",
    "    print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "<b>Entities normalized to +1 ID</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "25644 (2.86%) entities are normalized to more than 1 ID.<br>We revise those that appear more than 100 times further:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF Symbol                           TG Symbol\n",
      "MAPK3;MAPK1            5347         MAPK3;MAPK1       1425\n",
      "Mapk3;Mapk1            2798         Mapk3;Mapk1        618\n",
      "MAP2K1;MAP2K2           717         MMP2;MMP9          395\n",
      "SMAD2;SMAD3             481         SMAD2;SMAD3        253\n",
      "MAPK8;MAPK9             311         Smad2;Smad3        171\n",
      "Map2k1;Map2k2           303         CDK4;CDK6           96\n",
      "Smad2;Smad3             222         Mmp2;Mmp9           89\n",
      "EWSR1;FLI1              216         HSD11B1;RNU1-1      66\n",
      "ABL1;BCR                169         MIR143;MIR145       66\n",
      "BCR;ABL1                152         CASP3;CASP7         63\n",
      "CREBBP;EP300            147         MAP2K1;MAP2K2       57\n",
      "OIP5-AS1;OIP5;PTGDR     144         NKX2-5;NKX3-1       54\n",
      "SMAD1;SMAD5;SMAD9       128         CASP3;CASP9         42\n",
      "MAPK1;MAPK3             111         Ifna;Ifnb1          38\n",
      "HDAC1;HDAC2             104         EWSR1;FLI1          36\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "From those, 3 seem suspicious and are investigated further: ABL1;BCR, FLI1;EWSR1, MMP2;MMP9"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "           TF count  TG count                         TF count  TG count                                                   TF count  TG count\n",
      "BCR/ABL         126         9            EWS-FLI1          142        16          MMP-2/9                                         0        99\n",
      "BCR-ABL1        108        18            EWS/FLI1           43         6          MMP-2/-9                                        0        68\n",
      "BCR/ABL1         34         6            EWS/FLI-1          12        11          MMP2/9                                          0        65\n",
      "Bcr/Abl          26         4            EWS::FLI1           9         2          MMP-2 and -9                                    0        40\n",
      "BCR::ABL1        10         3            EWSR1-FLI1          9         0          matrix metalloproteinase-2 and -9               0        16\n",
      "bcr/abl           8         6            EWSR1::FLI1         7         1          matrix metalloproteinase-2/9                    0        11\n",
      "BCR::ABL          5         0                                                     matrix metalloproteinase-2/-9                   0         9\n",
      "Bcr/abl           3         0                                                     matrix metalloproteinases 2 and 9               0         9\n",
      "BCR/abl           1         0                                                     matrix metalloproteinase (MMP)-2 and -9         0         7\n",
      "                                                                                  matrix metalloproteinase 2 and 9                0         6\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "Through further manual investigation of the sentences, we have determined that:\n",
       "* FL1;EWSR1 & ABL1;BCR are fusion genes. They are correct TFs but must be discarded as TGs.\n",
       "* TG = MMP9;MMP2 entities indicate that the TF regulates both genes.\n",
       "   \n",
       "One TG sentence example for each case (first two will be discarded)\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ABL1;BCR:\t In this report, we show evidence that [TF] transcription factor stringently controls the expression of [TG], which can strategically be targeted by our novel RUNX inhibitor, Chb-M'.\n",
      "FLI1;EWSR1:\t [TF] knockdown showed a reduced cell growth and transcriptional activity of [TG].\n",
      "MMP2;MMP9:\t Finally, nimbolide suppressed the nuclear translocation of p65/p50 and DNA binding of [TF], which is an important transcription factor for controlling [TG] and VEGF gene expression.\n"
     ]
    }
   ],
   "source": [
    "### ENTITIES NORMALIZED TO +1 ID\n",
    "bold(\"Entities normalized to +1 ID\")\n",
    "m = (TRI_df['TF Symbol'].str.upper().str.contains(';')) | TRI_df['TG Symbol'].str.upper().str.contains(';')\n",
    "md(f\"{m.sum()} ({m.sum() / len(m):.2%}) entities are normalized to more than 1 ID.<br>We revise those that appear more than 100 times further:\")\n",
    "m_template = \"TRI_df['{T} Symbol'].str.contains(';')\"\n",
    "print_symbol_counts_side_by_side(m_template, max_counts=15)\n",
    "\n",
    "dubious_pairs = [('ABL1', 'BCR'), ('FLI1','EWSR1'), ('MMP2','MMP9')]\n",
    "\n",
    "md(f'From those, 3 seem suspicious and are investigated further: {\", \".join((\";\".join(p) for p in dubious_pairs))}')\n",
    "\n",
    "print_dubious_pairs_TFTGcounts_side_by_side(dubious_pairs)\n",
    "md('''\\\n",
    "Through further manual investigation of the sentences, we have determined that:\n",
    "* FL1;EWSR1 & ABL1;BCR are fusion genes. They are correct TFs but must be discarded as TGs.\n",
    "* TG = MMP9;MMP2 entities indicate that the TF regulates both genes.\n",
    "   \n",
    "One TG sentence example for each case (first two will be discarded)\n",
    "''')\n",
    "\n",
    "for p in dubious_pairs:\n",
    "    pairs = [';'.join(p) for p in itertools.permutations(p)]\n",
    "    print(f\"{pairs[0]}:\\t\", TRI_df[TRI_df['TG Symbol'].isin(pairs)].sample(1)['Sentence'].values[0])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Create sets of sentences to check"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "sents_to_check_path = config['data_p'] + 'validation/sents_to_check.tsv'\n",
    "sents_to_check_2_path = config['data_p'] + 'validation/sents_to_check_2.tsv'\n",
    "sents_to_check_of_path = config['data_p'] + 'validation/sents_to_check_of.tsv'\n",
    "sents_to_check_CDX_path = config['data_p'] + 'validation/sents_to_check_CDX.tsv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "<b>Sentences with 'p21' not normalised to 'CDKN1A':</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF   TF Symbol                           TG   TG Symbol\n",
      "p21  Tceal1       14                     p21  H3P16        4542\n",
      "     TCEAL1        1                          Kras          232\n",
      "                                              TCEAL1         29\n",
      "                                              Tceal1         25\n",
      "                                              Tpt1            1\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "<b>Sentences with 'p53' not normalised to 'TP53':</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF   TF Symbol                           TG   TG Symbol\n",
      "p53  Trp53        195                    p53  Trp53-ps     1615\n",
      "                                              p53-ps        240\n",
      "                                              Trp53          90\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "<b>Sentences with MDM2-TP53 pairs must be removed: they're always a PPI.</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF Symbol  TG Symbol\n",
      "MDM2       TP53         1609\n",
      "Mdm2       TP53           23\n",
      "MDM2;MDM4  TP53            3\n",
      "MDM2       TP53BP2         1 \n",
      "\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "<b>Sentences with 'MET' not normalised to 'MET':</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF   TF Symbol                           TG   TG Symbol\n",
      "MET  SLTM         494                    MET  SLTM         392\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "<b>Sentences with 'CD\\d' :</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF            TF Symbol                  TG             TG Symbol\n",
      "CD34          CD34         1704          CD34           CD34         206\n",
      "              Cd34          113          CD4            CD4          162\n",
      "CD34(         Cd34            5                         Cd4           82\n",
      "              CD34            2          Cd4            CD4           46\n",
      "CD4CD25FoxP3  FOXP3           2          CD34           Cd34          40\n",
      "cd34          CD34            2          CD74           CD74          27\n",
      "CD34+         CD34            1          CD8alpha       Cd8a          17\n",
      "CD34Exo       CD34            1          Cd4            Cd4           12\n",
      "CD34LC        CD34            1          Cd74           Cd74           9\n",
      "CD34brCD38    CD34;CD38       1          CD74           Cd74           8\n",
      "CD4.Ezh2      Cd4;Ezh2        1          Cd8a           CD8A           5\n",
      "                                         CD8a           Cd8a           5\n",
      "                                         CD8alpha       CD8A           4\n",
      "                                         CD8A           CD8A           3\n",
      "                                         CD8B           CD8B           2\n",
      "                                         CD8beta        CD8B           2\n",
      "                                         CD8a           CD8A           1\n",
      "                                         CD8b           Cd8b1          1\n",
      "                                         Cd34           CD34           1\n",
      "                                                        Cd34           1\n",
      "                                         CD4;IFN-gamma  Cd4;Ifng       1\n",
      "                                         CD4 receptor   CD4            1\n",
      "                                         Cd74           CD74           1\n",
      "                                         CD34EGFP       Cd34           1\n",
      "                                         CD34 antigen   CD34           1\n",
      "                                         Cd8alpha       CD8A           1\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "<b>Entities normalised to +1 IDs: ABL1;BCR</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF        TF Symbol                      TG       TG Symbol\n",
      "BCR/ABL   ABL1;BCR     126               BCR/ABL  ABL1;BCR     9\n",
      "Bcr/Abl   ABL1;BCR      26               bcr/abl  ABL1;BCR     6\n",
      "bcr/abl   ABL1;BCR       8               Bcr/Abl  ABL1;BCR     4\n",
      "BCR::ABL  ABL1;BCR       5               \n",
      "Bcr/abl   ABL1;BCR       3               \n",
      "BCR/abl   ABL1;BCR       1               \n",
      "\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "<b>\n",
       "Autoregulation:</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "4.8% of sentences show autoregulation: TF & TG are the same. Most popular:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF Symbol  TG Symbol\n",
      "TP53       TP53         1978\n",
      "VEGFA      VEGFA        1108\n",
      "EGFR       EGFR          852\n",
      "MYC        MYC           719\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "<b>\n",
       "Translation instead of gene expression</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7479 sentences contain 'translat' in them and should be checked\n"
     ]
    }
   ],
   "source": [
    "# PREPARE SENTECES TO CHECK\n",
    "def add_to_sents_to_check(sents_to_check: list, m_template: str, issue: str) -> list:\n",
    "    T = 'TF'\n",
    "    m = eval(m_template)\n",
    "    T = 'TG'\n",
    "    m |= eval(m_template)\n",
    "    df_m = TRI_df[m].copy()\n",
    "    df_m['issue'] = issue\n",
    "    sents_to_check.append(df_m)\n",
    "\n",
    "sents_to_check = []\n",
    "\n",
    "t = \"Sentences with 'p21' not normalised to 'CDKN1A':\"\n",
    "m_template = \"(TRI_df[f'{T}'] == 'p21') & (TRI_df[f'{T} Symbol'].str.upper() != 'CDKN1A')\"\n",
    "print_TF_TG_counts_side_by_side(t, m_template)\n",
    "add_to_sents_to_check(sents_to_check, m_template, 'p21-CDKN1A')\n",
    "\n",
    "\n",
    "t = \"Sentences with 'p53' not normalised to 'TP53':\"\n",
    "m_template = \"(TRI_df[f'{T}'] == 'p53') & (TRI_df[f'{T} Symbol'].str.upper() != 'TP53')\"\n",
    "print_TF_TG_counts_side_by_side(t, m_template)\n",
    "add_to_sents_to_check(sents_to_check, m_template, 'p53-TP53')\n",
    "\n",
    "bold(f\"Sentences with MDM2-TP53 pairs must be removed: they're always a PPI.\")\n",
    "m = TRI_df['TF Symbol'].str.upper().str.contains('MDM2')\n",
    "m &= TRI_df['TG Symbol'].str.upper().str.contains('TP53')\n",
    "print(TRI_df[m][['TF Symbol', 'TG Symbol']].value_counts().to_string(), '\\n')\n",
    "\n",
    "\n",
    "t = \"Sentences with 'MET' not normalised to 'MET':\"\n",
    "m_template = \"(TRI_df[f'{T}'] == 'MET') & (TRI_df[f'{T} Symbol'].str.upper() != 'MET')\"\n",
    "print_TF_TG_counts_side_by_side(t, m_template)\n",
    "add_to_sents_to_check(sents_to_check, m_template, 'MET')\n",
    "\n",
    "\n",
    "t = \"Sentences with 'CD\\d' :\"\n",
    "m_template = \"TRI_df[f'{T}'].str.upper().str.contains(r'^CD(?:4|8A|8B|74|34)(?!\\d)')\"\n",
    "print_TF_TG_counts_side_by_side(t, m_template)\n",
    "add_to_sents_to_check(sents_to_check, m_template, 'CD*')\n",
    "\n",
    "# Joined NCBI IDs to check\n",
    "t = \"Entities normalised to +1 IDs: ABL1;BCR\"\n",
    "m_template = \"TRI_df[f'{T} Symbol'] == 'ABL1;BCR'\"\n",
    "print_TF_TG_counts_side_by_side(t, m_template)\n",
    "add_to_sents_to_check(sents_to_check, m_template, 'ABL1;BCR')\n",
    "\n",
    "\n",
    "bold(f\"\\nAutoregulation:\")\n",
    "m = TRI_df['TF Symbol'].str.upper() == TRI_df['TG Symbol'].str.upper()\n",
    "md(f\"{m.sum() / len(TRI_df):.1%} of sentences show autoregulation: TF & TG are the same. Most popular:\")\n",
    "print(TRI_df[m][['TF Symbol', 'TG Symbol']].value_counts()[:4].to_string())\n",
    "\n",
    "# Those are potentially commonly wrong. Prepare a set of 300 sentences for validation purposes\n",
    "df_m = TRI_df[m].sample(n=300)\n",
    "df_m['issue'] = 'Autoregulation'\n",
    "sents_to_check.append(df_m)\n",
    "\n",
    "bold('\\nTranslation instead of gene expression')\n",
    "m = TRI_df['Sentence'].str.lower().str.contains('translat')\n",
    "print(f\"{m.sum()} sentences contain 'translat' in them and should be checked\")\n",
    "df_m = TRI_df[m].sample(n=100)\n",
    "df_m['issue'] = 'Translate'\n",
    "sents_to_check.append(df_m)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "# CREATE EXCEL\n",
    "def extract_context(sentence, token='[TF]', window=4, how='both'):\n",
    "    '''Get the last and next 4 words from the token'''\n",
    "    # Split the sentence by spaces\n",
    "    words = sentence.split()\n",
    "\n",
    "    # Find the index of the word that contains '[TF]' or its variations\n",
    "    index = [i for i, word in enumerate(words) if token in word][0]\n",
    "\n",
    "    # Extract the 4 words before and after the token, handling boundaries\n",
    "    start = max(0, index) if how=='right' else max(0, index - window)\n",
    "    end = min(len(words), index + 1) if how=='left' else min(len(words), index + window + 1)\n",
    "\n",
    "\n",
    "    # Join the extracted context words back into a string\n",
    "    return ' '.join(['...'] + words[start:end] + ['...'])\n",
    "\n",
    "sents_to_check = pd.concat(sents_to_check)\n",
    "\n",
    "cols_to_keep = ['issue', 'TF', 'TF Symbol', 'TG', 'TG Symbol', 'Sentence',  '#SentenceID', \n",
    "                'TF Id', 'TG Id', 'MoR', 'TF TaxID',  'TG TaxID', 'TF_type', 'issue']\n",
    "\n",
    "sents_to_check = sents_to_check[cols_to_keep]\n",
    "\n",
    "for T in ('TF', 'TG'):\n",
    "    sents_to_check[f'{T}_context'] = sents_to_check['Sentence'].apply(lambda x: extract_context(x, token=f'{T}'))\n",
    "    sents_to_check[f'{T}_left_context'] = sents_to_check['Sentence'].apply(lambda x: extract_context(x, token=f'{T}', how='left'))\n",
    "    sents_to_check[f'{T}_right_context'] = sents_to_check['Sentence'].apply(lambda x: extract_context(x, token=f'{T}', how='right'))\n",
    "\n",
    "sents_to_check.to_csv(sents_to_check_path, sep='\\t', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "<b>\n",
       "dbTF Autoregulation:</b>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "2.1% of sentences show autoregulation: TF & TG are the same. Most popular:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TF Symbol  TG Symbol\n",
      "TP53       TP53         1978\n",
      "MYC        MYC           719\n",
      "HIF1A      HIF1A         478\n",
      "ESR1       ESR1          434\n",
      "results saved in ../../data/validation/sents_to_check_2.tsv\n"
     ]
    }
   ],
   "source": [
    "# PREPARE 2nd SET OF SENTENCES TO CHECK\n",
    "bold(f\"\\ndbTF Autoregulation:\")\n",
    "\n",
    "# Get a set of 300 sentences to check with autoregulation (dbTF)#\n",
    "# Previous set contained a lot of coTF sentences. We want to check the number of false positives in dbTF-specific autoregulation.\n",
    "m = TRI_df['TF Symbol'].str.upper() == TRI_df['TG Symbol'].str.upper()\n",
    "m &= TRI_df['TF_type'] == 'dbTF'\n",
    "md(f\"{m.sum() / len(TRI_df):.1%} of sentences show autoregulation: TF & TG are the same. Most popular:\")\n",
    "print(TRI_df[m][['TF Symbol', 'TG Symbol']].value_counts()[:4].to_string())\n",
    "df_m = TRI_df[m].sample(n=300)\n",
    "df_m['issue'] = 'dbTF_autoregulation'\n",
    "\n",
    "# We will only check dbTF autoregulation\n",
    "sents_to_check_2 = df_m\n",
    "tab_cols = (\"issue\t#SentenceID\tTF\tTF Symbol\tTG\tTG Symbol\tSentence\tTF Id\tTG Id\tTF offset\tGene offset\tTRI score\tValid\tMoR scores\tMoR\tPMID\tPMID+Sent+TRI_Id\tTF TaxID\tTG TaxID\tTF_type\")\n",
    "sents_to_check_2 = sents_to_check_2[tab_cols.split(\"\\t\")]\n",
    "\n",
    "sents_to_check_2.to_csv(sents_to_check_2_path, sep='\\t', index=False)\n",
    "print(f\"results saved in {sents_to_check_2_path}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "TG        TG Symbol\n",
       "CD44      CD44         944\n",
       "CD133     PROM1        350\n",
       "CD40      CD40         306\n",
       "CD86      CD86         232\n",
       "          Cd86         180\n",
       "                      ... \n",
       "CD40L.    Cd40lg         1\n",
       "CD41b     ITGA2B         1\n",
       "CD42a     GP9            1\n",
       "CD44High  Cd44           1\n",
       "cd59      CD59           1\n",
       "Name: count, Length: 491, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "9319"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# PREPARE 3rd SET OF SENTENCES TO CHECK\n",
    "m = final_TRI_df['TG'] == 'of'\n",
    "m |= final_TRI_df['TF'] == 'of'\n",
    "sents_to_check_of = final_TRI_df[m].sort_values(by=['TF', 'TG'])\n",
    "sents_to_check_of.to_csv(sents_to_check_of_path, sep='\\t', index=False)\n",
    "\n",
    "m = final_TRI_df['TG'].str.upper().str.contains(\"^CD[0-9]\")\n",
    "m &= final_TRI_df['Sentence'].str.contains(\"\\[TG\\] ?\")\n",
    "\n",
    "display(final_TRI_df[m][['TG', 'TG Symbol']].value_counts())\n",
    "m.sum()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "m = final_TRI_df['TG'].str.upper().str.contains(\"^CD[0-9]\")\n",
    "final_TRI_df[m].to_csv(sents_to_check_CDX_path, sep='\\t', index=False)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".general_env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
