{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "df4c0c1a-3b10-451f-8884-d231ec89b734",
   "metadata": {},
   "source": [
    "# Update raw data and NTNU dataset"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "302e995d-7420-47a1-a0e8-91cc46c6fbfc",
   "metadata": {},
   "source": [
    "modify `original_tri_sentences.tsv` to obtain `tri_sentences.tsv` with the updated annotations, including:\n",
    "* The re-annotations\n",
    "* Negation sentences from the extended NTNU dataset\n",
    "\n",
    "After running this notebook, `make_train_data.ipynb` must also be run, which converts the created `tri_sentences.tsv` into the datasets used as input for training the TRI and MoR classifiers."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "e5a66afe",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "<h3>Table of contents</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "[Update raw data and NTNU dataset](#Update-raw-data-and-NTNU-dataset)\n",
       "- [Setup](#Setup)\n",
       "- [Clean the dataset](#Clean-the-dataset)\n",
       "  - [Modify joined sentences](#Modify-joined-sentences)\n",
       "- [Reannotations](#Reannotations)\n",
       "- [Save the dataset](#Save-the-dataset)\n",
       "- [Old code - Add negative sentences](#Old-code---Add-negative-sentences)\n",
       "  - [Enhance with negations from the extended NTNU dataset](#Enhance-with-negations-from-the-extended-NTNU-dataset)"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "__import__('sys').path.append('../common/'); __import__('notebook_utils').table_of_contents('update_tri_sentences.ipynb')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3bd45d45",
   "metadata": {},
   "source": [
    "## Setup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "1b82aa45",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Imports\n",
    "from IPython.display import display, HTML, display_html\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "from sklearn.model_selection import train_test_split, KFold\n",
    "import matplotlib.pyplot as plt\n",
    "import spacy\n",
    "from tqdm import tqdm\n",
    "\n",
    "# My functions\n",
    "import sys\n",
    "sys.path.append('../common/') \n",
    "from analysis_utils import prettify_plots\n",
    "from notebook_utils import table_from_dict, md, h3, h4, highlight_words\n",
    "\n",
    "prettify_plots()\n",
    "\n",
    "# %pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.5.4/en_core_sci_md-0.5.4.tar.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "49e4c84f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# PATHS  & LOAD DATASETS\n",
    "# Inputs\n",
    "data_path = '../../data/external/'\n",
    "original_data_path = data_path + 'original_tri_sentences.tsv'\n",
    "NTNU_extended_path = data_path + 'NTNU_extended.tsv'\n",
    "\n",
    "dataset_improvement_path = '../../data/dataset_improvement/'\n",
    "iter_1_2_path      = dataset_improvement_path + 'to_reannotate/iter1_iter2_worst_predictions_validated.txt'\n",
    "iter_3_path        = dataset_improvement_path + 'to_reannotate/iter3_worst_preds_AL_ver2.txt'\n",
    "negations_to_add_path = dataset_improvement_path + 'negations_NTNU.tsv'\n",
    "\n",
    "# Load datasets\n",
    "original_data   = pd.read_csv(original_data_path, sep='\\t', header=1, index_col=None, dtype='str')\n",
    "iter_1_2        = pd.read_csv(iter_1_2_path, sep='\\t', header=0, index_col=None, dtype='str')\n",
    "iter_3          = pd.read_csv(iter_3_path, sep='\\t', header=0, index_col=None, dtype='str')\n",
    "NTNU_extended   = pd.read_csv(NTNU_extended_path, sep='\\t', header=1, index_col=None, dtype='str')\n",
    "negations_to_add   = pd.read_csv(negations_to_add_path, sep='\\t', header=0, index_col=None, dtype='str')\n",
    "\n",
    "\n",
    "# Outputs\n",
    "splitted_sentences_path = dataset_improvement_path + 'to_reannotate/splitted_sentences_ids.txt'\n",
    "\n",
    "classifiers_data_path = '../../classifiers_training/data/'\n",
    "training_data_path  = classifiers_data_path + 'tri_sentences.tsv'\n",
    "NTNU_dataset_path = '../../results/NTNU_dataset.tsv'"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "27c12411",
   "metadata": {},
   "source": [
    "## Clean the dataset"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "87b79f86",
   "metadata": {},
   "source": [
    "There are a few duplicated sentences: same PMID and sentence number, but different TF or TG ID. Some also differ in their TRI/MoR annotations. They will be removed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "6d0f85b9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<h4>Information on duplicated rows</h4><table><tr><td><b>Total number of sentences</b></td><td>22135</td></tr><tr><td><b>Unique sentences</b></td><td>21763</td></tr><tr><td><b>Duplicated sentences</b></td><td>735 (3.3%)</td></tr><tr><td><b>.. where TRI or MoR differs</b></td><td>278 (1.3%)</td></tr><tr><td><b>Unique duplicated sentences</b></td><td>363</td></tr><tr><td><b>.. where TRI or MOR differs</b></td><td>137</td></tr></table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Two sets of duplicated sentences where either Label or Type differ:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#TRI ID</th>\n",
       "      <th>Sentence</th>\n",
       "      <th>TF</th>\n",
       "      <th>TG</th>\n",
       "      <th>Label</th>\n",
       "      <th>Type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>20620</th>\n",
       "      <td>21056029:3:NFKB:CCND1</td>\n",
       "      <td>-catenin activated transcription from the [TG]...</td>\n",
       "      <td>p65</td>\n",
       "      <td>cyclin D1</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10925</th>\n",
       "      <td>21056029:3:RELA:CCND1</td>\n",
       "      <td>-catenin activated transcription from the [TG]...</td>\n",
       "      <td>p65</td>\n",
       "      <td>cyclin D1</td>\n",
       "      <td>true</td>\n",
       "      <td>REPRESSION</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12854</th>\n",
       "      <td>21720709:0:TCF7L2:AKT1</td>\n",
       "      <td>-catenin/[TF] complex transcriptionally regula...</td>\n",
       "      <td>Tcf-4</td>\n",
       "      <td>AKT1</td>\n",
       "      <td>true</td>\n",
       "      <td>UNDEFINED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19541</th>\n",
       "      <td>21720709:0:TCF4:AKT1</td>\n",
       "      <td>-catenin/[TF] complex transcriptionally regula...</td>\n",
       "      <td>Tcf-4</td>\n",
       "      <td>AKT1</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                      #TRI ID  \\\n",
       "20620   21056029:3:NFKB:CCND1   \n",
       "10925   21056029:3:RELA:CCND1   \n",
       "12854  21720709:0:TCF7L2:AKT1   \n",
       "19541    21720709:0:TCF4:AKT1   \n",
       "\n",
       "                                                Sentence     TF         TG  \\\n",
       "20620  -catenin activated transcription from the [TG]...    p65  cyclin D1   \n",
       "10925  -catenin activated transcription from the [TG]...    p65  cyclin D1   \n",
       "12854  -catenin/[TF] complex transcriptionally regula...  Tcf-4       AKT1   \n",
       "19541  -catenin/[TF] complex transcriptionally regula...  Tcf-4       AKT1   \n",
       "\n",
       "       Label        Type  \n",
       "20620  false         NaN  \n",
       "10925   true  REPRESSION  \n",
       "12854   true   UNDEFINED  \n",
       "19541  false         NaN  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# SHOW NUMBER OF DUPLICATE ROWS\n",
    "unique_rows = original_data['Sentence'].unique()\n",
    "duplicated_rows = original_data[original_data['Sentence'].duplicated(keep=False)]\n",
    "unique_duplicated_rows = duplicated_rows.drop_duplicates(subset='Sentence')\n",
    "duplicates_with_diff_label_or_type = original_data.groupby('Sentence').filter(lambda x: x['Label'].nunique() > 1 or x['Type'].nunique() > 1)\n",
    "\n",
    "table_from_dict('Information on duplicated rows', {\n",
    "    'Total number of sentences': len(original_data),\n",
    "    'Unique sentences': len(unique_rows),\n",
    "    'Duplicated sentences': f'{len(duplicated_rows)} ({round(len(duplicated_rows)/len(original_data)*100,1)}%)',\n",
    "    '.. where TRI or MoR differs': f'{len(duplicates_with_diff_label_or_type)} ({round(len(duplicates_with_diff_label_or_type)/len(original_data)*100,1)}%)',\n",
    "    'Unique duplicated sentences': len(unique_duplicated_rows),\n",
    "    '.. where TRI or MOR differs': len(duplicates_with_diff_label_or_type['Sentence'].unique()),\n",
    "}, heading='h4')\n",
    "\n",
    "md('Two sets of duplicated sentences where either Label or Type differ:')\n",
    "display(duplicates_with_diff_label_or_type.sort_values(by='Sentence').head(4))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "e352814c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "463 sentences are duplicated but have the same Label & MoR. Only the first is kept (234 are dropped)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "275 sentences are duplicated and differ on their Label or MoR. They're all dropped."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# DROP DUPLICATES\n",
    "# Drop duplicates in which Sentence, TRI and MoR are the same\n",
    "m1 = original_data.duplicated(subset=['Sentence', 'Label', 'Type'], keep=False)\n",
    "m2 = original_data.duplicated(subset=['Sentence', 'Label', 'Type'], keep='first')\n",
    "md(f\"{m1.sum()} sentences are duplicated but have the same Label & MoR. Only the first is kept ({m2.sum()} are dropped).\")\n",
    "original_data = original_data.drop_duplicates(subset=['Sentence', 'Label', 'Type'], keep='first')\n",
    "\n",
    "# Drop rows where Sentence is duplicated, but Label and/or Type differ\n",
    "m = original_data.duplicated(subset=['Sentence'], keep=False)\n",
    "md(f\"{m.sum()} sentences are duplicated and differ on their Label or MoR. They're all dropped.\")\n",
    "original_data = original_data[~m]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1aab1c40",
   "metadata": {},
   "source": [
    "### Modify joined sentences"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "870b2ef3",
   "metadata": {},
   "source": [
    "The splitter we will use for the pipeline is `en_core_sci_md`. We will now make sure that the dataset we use for training is separated according to their standards. \n",
    "\n",
    "As explained in `scripts/preprocessing/spacy_splitter_analysis.ipynb`, the spacy splitter makes some mistakes, namely separating a sentence in 2 sometimes when a \".\" is combined by a lowercase letter, or when there is no \".\". To avoid these cases, we use a rule-based method to re-merge those sentences, defined in the `merge_sentences()` function.\n",
    "\n",
    "We pass all sentences in the dataset through the splitter, obtaining a 0.3% of cases where the sentences were incorrectly splitted. They can be divided into 2 groups:\n",
    "* **[TF] and [TG] are in the same sentence:** Those sentences have been revised, and are included in the `iter_1_2` dataset. They will be updated.\n",
    "* **[TF] and [TG] are in separate sentences:** Those sentences will be dropped, as they are no longer valid."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "076c1655",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/cdt_wsl/git/ExTRI2_gits/ExTRI2/.general_env/lib/python3.10/site-packages/spacy/language.py:2195: FutureWarning: Possible set union at position 6328\n",
      "  deserializers[\"tokenizer\"] = lambda p: self.tokenizer.from_disk(  # type: ignore[union-attr]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 1min 53s, sys: 3.84 s, total: 1min 57s\n",
      "Wall time: 1min 57s\n"
     ]
    },
    {
     "data": {
      "text/markdown": [
       "28 rows (0.13%) have [TF] and [TG] in separate sentences. They are discarded.<br>Example:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#TRI ID</th>\n",
       "      <th>Sentence</th>\n",
       "      <th>Label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>4132</th>\n",
       "      <td>12525489:0:activator protein 1:tissue inhibitors of metalloproteinases</td>\n",
       "      <td>The comparative role of <strong>[TF]</strong> and Smad factors in the regulation of Timp-1 and MMP-1 gene expression by transforming growth factor-beta 1.<br> The balance between matrix metalloproteinases (MMPs) and their inhibitors, the <strong>[TG]</strong> (TIMPs), is pivotal in the remodeling of extracellular matrix.<br></td>\n",
       "      <td>false</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4133</th>\n",
       "      <td>12525489:0:activator protein 1:TIMPs</td>\n",
       "      <td>The comparative role of <strong>[TF]</strong> and Smad factors in the regulation of Timp-1 and MMP-1 gene expression by transforming growth factor-beta 1.<br> The balance between matrix metalloproteinases (MMPs) and their inhibitors, the tissue inhibitors of metalloproteinases (<strong>[TG]</strong>), is pivotal in the remodeling of extracellular matrix.<br></td>\n",
       "      <td>false</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "26 rows (0.12%) have [TF] and [TG] in the same sentence. The correct sentence is kept.<br>Example:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#TRI ID</th>\n",
       "      <th>Sentence</th>\n",
       "      <th>Label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>1655749:0:ATF-1:cAMP-dependent protein kinase A</td>\n",
       "      <td>The cAMP-regulated enhancer-binding protein <strong>[TF]</strong> activates transcription in response to <strong>[TG]</strong>.<br> Many promoters respond transcriptionally to elevated levels of cAMP through the cAMP-responsive enhancer (CRE).<br></td>\n",
       "      <td>false</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4128</th>\n",
       "      <td>12525489:0:activator protein 1:Smad</td>\n",
       "      <td>The comparative role of <strong>[TF]</strong> and <strong>[TG]</strong> factors in the regulation of Timp-1 and MMP-1 gene expression by transforming growth factor-beta 1.<br> The balance between matrix metalloproteinases (MMPs) and their inhibitors, the tissue inhibitors of metalloproteinases (TIMPs), is pivotal in the remodeling of extracellular matrix.<br></td>\n",
       "      <td>false</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "nlp = spacy.load(\"en_core_sci_md\")\n",
    "\n",
    "def merge_sentences(doc):\n",
    "    merged_sentences = []\n",
    "    temp_sentence = \"\"\n",
    "\n",
    "    sentences = [i for i in doc.sents]  # Convert generator to a list for easier handling\n",
    "    for i, sentence in enumerate(sentences):\n",
    "        current_text = sentence.text.strip()\n",
    "        temp_sentence += current_text + \" \"\n",
    "\n",
    "        # Separate to a new sentence if: \n",
    "        #     it is the last sentence, or\n",
    "        #     The sentence ends with \". \" and next sentence starts with uppercase\n",
    "        if i == len(sentences) - 1 or ( current_text.endswith(('.', '!', '?')) \n",
    "                                       and sentences[i + 1].text.strip()[0].isupper()\n",
    "                                       and sentences[i + 1].start_char == sentence.end_char + 1\n",
    "                                      ):\n",
    "            merged_sentences.append(temp_sentence.strip())\n",
    "            temp_sentence = \"\"\n",
    "    return merged_sentences\n",
    "\n",
    "\n",
    "# Use spaCy's pipe for efficient batch processing\n",
    "sentences = original_data['Sentence'].tolist()  # Convert column to list for efficient processing\n",
    "%time docs = list(nlp.pipe(sentences))\n",
    "\n",
    "# Get list of IDs to keep and to discard:\n",
    "discard_ids = []\n",
    "keep_ids = {}\n",
    "for doc, id in zip(docs, original_data['#TRI ID']):\n",
    "    merged_sents = merge_sentences(doc)\n",
    "    if len(merged_sents) > 1:\n",
    "        for sentence in merged_sents:\n",
    "            if '[TF]' in sentence:\n",
    "                if '[TG]' in sentence:\n",
    "                    # TF and TG are together. Keep the sentence\n",
    "                    keep_ids[id] = sentence\n",
    "                    pass\n",
    "                else:\n",
    "                    # TF and TG are in separate sentences. Discard.\n",
    "                    discard_ids.append(id)\n",
    "\n",
    "# Show examples of discarded rows:\n",
    "md(f\"{len(discard_ids)} rows ({len(discard_ids) / len(original_data):.2%}) have [TF] and [TG] in separate sentences. They are discarded.<br>Example:\")\n",
    "m = original_data['#TRI ID'].isin(discard_ids)\n",
    "df = original_data[m][['#TRI ID', 'Sentence', 'Label']]\n",
    "df['Sentence'] = df['Sentence'].apply(lambda x: highlight_words(x, ['[TF]', '[TG]']))\n",
    "display(HTML(f'{df[:2].to_html(escape=False)}'))                                    \n",
    "\n",
    "md(f'{len(keep_ids)} rows ({len(keep_ids) / len(original_data):.2%}) have [TF] and [TG] in the same sentence. The correct sentence is kept.<br>Example:')\n",
    "m = original_data['#TRI ID'].isin(keep_ids)\n",
    "df = original_data[m][['#TRI ID', 'Sentence', 'Label']]\n",
    "df['Sentence'] = df['Sentence'].apply(lambda x: highlight_words(x, ['[TF]', '[TG]']))\n",
    "display(HTML(f'{df[:2].to_html(escape=False)}'))      \n",
    "\n",
    "# Save the ids of the splitted sentences to keep to be reannotated:\n",
    "with open(splitted_sentences_path, 'w') as f:\n",
    "    f.writelines([id+'\\n' for id in keep_ids.keys()])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "6cd15cf1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "We discard 54 rows because they have been split\n"
     ]
    }
   ],
   "source": [
    "# DROP SPLIT SENTENCES\n",
    "print(f\"We discard {len(discard_ids) + len(keep_ids)} rows because they have been split\")\n",
    "original_data = original_data[~original_data['#TRI ID'].isin(discard_ids)]\n",
    "\n",
    "original_data = original_data[~original_data['#TRI ID'].isin(keep_ids.keys())]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8714d624",
   "metadata": {},
   "source": [
    "## Reannotations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "44d2fef0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "<h4>ACTIVATION & REPRESSION</h4>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "There are 5 sentences that have been noted as <b>both ACTIVATION and REPRESSION</b><br>\n",
       "Ideally, they should have a BOTH label, but as we don't have enough data, we'll convert them into UNDEFINED.<br>\n",
       "Resulting rows:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#TRI ID</th>\n",
       "      <th>Sentence</th>\n",
       "      <th>Label</th>\n",
       "      <th>MoR</th>\n",
       "      <th>Valid?</th>\n",
       "      <th>true_label</th>\n",
       "      <th>true_MoR</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1578</th>\n",
       "      <td>8994186:6:NR4A1:POMC</td>\n",
       "      <td>Finally, we provide evidence that the nurr1/[T...</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>ACTIVATION</td>\n",
       "      <td>F</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>UNDEFINED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1581</th>\n",
       "      <td>9514889:7:NFKB:PTGS2</td>\n",
       "      <td>These results suggest that the [TF] site is in...</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>ACTIVATION</td>\n",
       "      <td>F</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>UNDEFINED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1915</th>\n",
       "      <td>9256061:7:TFEC:HMOX1</td>\n",
       "      <td>By transient coexpression assays, we showed th...</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>UNDEFINED</td>\n",
       "      <td>T</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>UNDEFINED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1918</th>\n",
       "      <td>9819390:0:TLX1:ALDH1A1</td>\n",
       "      <td>The T-cell oncogenic protein [TF] activates [T...</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>UNDEFINED</td>\n",
       "      <td>T</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>UNDEFINED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1941</th>\n",
       "      <td>15660126:4:NFKB:SLC1A2</td>\n",
       "      <td>Herein, we demonstrate that both TNFalpha-medi...</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>UNDEFINED</td>\n",
       "      <td>T</td>\n",
       "      <td>TRUE</td>\n",
       "      <td>UNDEFINED</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                     #TRI ID  \\\n",
       "1578    8994186:6:NR4A1:POMC   \n",
       "1581    9514889:7:NFKB:PTGS2   \n",
       "1915    9256061:7:TFEC:HMOX1   \n",
       "1918  9819390:0:TLX1:ALDH1A1   \n",
       "1941  15660126:4:NFKB:SLC1A2   \n",
       "\n",
       "                                               Sentence Label         MoR  \\\n",
       "1578  Finally, we provide evidence that the nurr1/[T...  TRUE  ACTIVATION   \n",
       "1581  These results suggest that the [TF] site is in...  TRUE  ACTIVATION   \n",
       "1915  By transient coexpression assays, we showed th...  TRUE   UNDEFINED   \n",
       "1918  The T-cell oncogenic protein [TF] activates [T...  TRUE   UNDEFINED   \n",
       "1941  Herein, we demonstrate that both TNFalpha-medi...  TRUE   UNDEFINED   \n",
       "\n",
       "     Valid? true_label   true_MoR  \n",
       "1578      F       TRUE  UNDEFINED  \n",
       "1581      F       TRUE  UNDEFINED  \n",
       "1915      T       TRUE  UNDEFINED  \n",
       "1918      T       TRUE  UNDEFINED  \n",
       "1941      T       TRUE  UNDEFINED  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "<h4>Suboptimal</h4>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "There are 160 sentences that are suboptimal and will be converted to 'False' for training"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# PREPROCESS ITER DATASETS\n",
    "\n",
    "# Remove empty rows & columns\n",
    "iter_1_2.dropna(how='all', inplace=True)\n",
    "iter_3.dropna(subset = iter_3.columns.difference(['Problem with:']), how='all', inplace=True)\n",
    "iter_3.drop(columns=['Unnamed: 17'], inplace=True)\n",
    "\n",
    "## Fix columns/cells\n",
    "# If 'Iter' is neither 1 nor 2, it belongs to the Splitter.\n",
    "iter_1_2.loc[iter_1_2['Iter'].isna(), 'Iter'] = 'Spliter'\n",
    "iter_3['Iter'] = '3'\n",
    "# These specific cells were incorrectly annotated\n",
    "m = iter_1_2['#TRI ID'] == '24276245:6:E2F1:ABCG2'\n",
    "iter_1_2.loc[m, 'true_MoR'] = 'UNDEFINED'\n",
    "m = iter_1_2['#TRI ID'] == '21813016:5:CTCF:CDKN1A'\n",
    "iter_1_2.loc[m, 'true_label'] = 'TRUE'\n",
    "\n",
    "## Join the 'Problem_with' in one only row\n",
    "columns_to_join = ['Problem with:', 'Unnamed: 10']\n",
    "iter_1_2['Problem_with'] = iter_1_2.apply(lambda row: '|'.join(sorted(row[col] for col in columns_to_join if not pd.isna(row[col]))), axis=1)\n",
    "iter_1_2.drop(columns=columns_to_join, inplace=True)\n",
    "\n",
    "columns_to_join = ['Problem with:', 'Unnamed: 11', 'Suboptimal?']\n",
    "iter_3['Problem_with'] = iter_3.apply(lambda row: '|'.join(sorted([row[col] for col in columns_to_join if not pd.isna(row[col])])), axis=1)\n",
    "iter_3.drop(columns=columns_to_join, inplace=True)\n",
    "\n",
    "## Remove duplicates (annotated both in Iteration 1 and 2)\n",
    "iter_1_2.drop_duplicates(subset=['#TRI ID', 'Sentence', 'Label', 'MoR'], keep='first', inplace=True)\n",
    "\n",
    "## Convert 'both activation & repression' into UNDEFINED\n",
    "m = iter_1_2['Explanation'] == 'BOTH activation AND repression'\n",
    "h4('ACTIVATION & REPRESSION')\n",
    "md(f'''There are {m.sum()} sentences that have been noted as <b>both ACTIVATION and REPRESSION</b><br>\n",
    "Ideally, they should have a BOTH label, but as we don't have enough data, we'll convert them into UNDEFINED.<br>\n",
    "Resulting rows:''')\n",
    "iter_1_2.loc[m, 'Valid?'] = iter_1_2.loc[m, 'MoR'].apply(lambda MoR: 'T' if MoR == 'UNDEFINED' else 'F')\n",
    "iter_1_2.loc[m, 'true_label'] = 'TRUE'\n",
    "iter_1_2.loc[m, 'true_MoR'] = 'UNDEFINED'\n",
    "display(iter_1_2[m][['#TRI ID', 'Sentence', 'Label', 'MoR', 'Valid?', 'true_label', 'true_MoR']])\n",
    "\n",
    "# Join iter 1,2,3\n",
    "validated_sents = pd.concat([iter_1_2, iter_3], axis=0)\n",
    "# Drop duplicates between iters 2 & 3 (keep Iteration 3)\n",
    "validated_sents = validated_sents.sort_values(by='Iter').drop_duplicates(subset=['#TRI ID', 'Sentence', 'Label', 'MoR'], keep='last')\n",
    "\n",
    "# Remove validated rows that have been split\n",
    "validated_sents = validated_sents[~validated_sents['#TRI ID'].isin(keep_ids.keys())]\n",
    "\n",
    "# SUBOPTIMAL SENTENCES\n",
    "h4(\"Suboptimal\")\n",
    "# Suboptimal sentences are only modified for the training dataset, not the NTNU dataset.\n",
    "# Create a copy of the validated sentences for the NTNU\n",
    "validated_sents_for_NTNU = validated_sents.copy()\n",
    "\n",
    "# For the training dataset, modify the suboptimal sentences to \"Label\" = \"False\"\n",
    "m_suboptimal = validated_sents['Problem_with'].str.contains('Suboptimal')\n",
    "md(f\"There are {m_suboptimal.sum()} sentences that are suboptimal and will be converted to 'False' for training\")\n",
    "\n",
    "validated_sents.loc[m_suboptimal, 'Valid?'] = np.where(validated_sents.loc[m_suboptimal, 'Label'] == 'FALSE', 'T', 'F')\n",
    "validated_sents.loc[m_suboptimal, 'true_label'] = 'FALSE'\n",
    "validated_sents.loc[m_suboptimal, 'true_MoR'] = np.nan\n",
    "\n",
    "\n",
    "## Assertions\n",
    "assert validated_sents.duplicated(subset=['#TRI ID']).sum() == 0\n",
    "assert all(iter_3.columns == iter_1_2.columns), 'Iter 1_2 and Iter_3 have different columns'\n",
    "assert validated_sents['Valid?'].isna().sum() == 0, 'Some rows have no Valid? value. Check the data.'\n",
    "assert ((validated_sents['Valid?'] == 'F') & (validated_sents['true_label'].isna())).sum() == 0\n",
    "assert ((validated_sents['true_label'] == 'TRUE') & (validated_sents['true_MoR'].isna())).sum() == 0\n",
    "assert ((validated_sents['true_label'] == 'FALSE') & ~(validated_sents['true_MoR'].isna())).sum() == 0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "3a8dea62",
   "metadata": {},
   "outputs": [],
   "source": [
    "# UPDATE TRAINING DATASET\n",
    "\n",
    "def update_training_dataset(original_data, validated_sents):\n",
    "    '''\n",
    "    Update the training dataset with validated sentences.\n",
    "    '''\n",
    "\n",
    "    updated_data = original_data.copy()\n",
    "\n",
    "    # Get data to update in the same format as the training dataset\n",
    "    m = validated_sents['Valid?'] == 'F'\n",
    "    to_update = validated_sents[m]\n",
    "    to_update.loc[:, 'true_label'] = to_update['true_label'].replace({'FALSE': 'false', 'TRUE': 'true'})\n",
    "\n",
    "    # Create mappings\n",
    "    label_map = to_update.set_index('#TRI ID')['true_label']\n",
    "    MoR_map   = to_update.set_index('#TRI ID')['true_MoR']\n",
    "\n",
    "    # Apply mappings to update 'Label' and 'Type' where '#TRI ID' matches\n",
    "    mask = updated_data['#TRI ID'].isin(to_update['#TRI ID'])\n",
    "    updated_data.loc[mask, 'Label'] = updated_data.loc[mask, '#TRI ID'].map(label_map)\n",
    "    updated_data.loc[mask, 'Type']  = updated_data.loc[mask, '#TRI ID'].map(MoR_map)\n",
    "\n",
    "    # Create column to keep track of validated rows\n",
    "    updated_data['validated?'] = updated_data['#TRI ID'].isin(validated_sents['#TRI ID'])\n",
    "\n",
    "    return updated_data\n",
    "\n",
    "# Update training data\n",
    "training_data = update_training_dataset(original_data, validated_sents)\n",
    "\n",
    "# Update the NTNU dataset\n",
    "NTNU_dataset = update_training_dataset(original_data, validated_sents_for_NTNU)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "6804fdc5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "We discard 43 sentences as duplicates, 14 sentences where labels differ, and 14 unvalid sentences where the valid version has different labels.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#TRI ID</th>\n",
       "      <th>Sentence</th>\n",
       "      <th>TF</th>\n",
       "      <th>TG</th>\n",
       "      <th>Label</th>\n",
       "      <th>Type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>16373364:0:Ets1:GATA-3</td>\n",
       "      <td>A role for Ets1, synergizing with AP-1 and GAT...</td>\n",
       "      <td>Ets1</td>\n",
       "      <td>GATA-3</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>16373364:0:Ets1:IL-5</td>\n",
       "      <td>A role for Ets1, synergizing with AP-1 and GAT...</td>\n",
       "      <td>Ets1</td>\n",
       "      <td>IL-5</td>\n",
       "      <td>true</td>\n",
       "      <td>UNDEFINED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>16373364:0:Ets1:AP-1</td>\n",
       "      <td>A role for Ets1, synergizing with AP-1 and GAT...</td>\n",
       "      <td>Ets1</td>\n",
       "      <td>AP-1</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>16373364:0:GATA-3:Ets1</td>\n",
       "      <td>A role for Ets1, synergizing with AP-1 and GAT...</td>\n",
       "      <td>GATA-3</td>\n",
       "      <td>Ets1</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>16373364:0:GATA-3:IL-5</td>\n",
       "      <td>A role for Ets1, synergizing with AP-1 and GAT...</td>\n",
       "      <td>GATA-3</td>\n",
       "      <td>IL-5</td>\n",
       "      <td>true</td>\n",
       "      <td>UNDEFINED</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                  #TRI ID                                           Sentence  \\\n",
       "0  16373364:0:Ets1:GATA-3  A role for Ets1, synergizing with AP-1 and GAT...   \n",
       "1    16373364:0:Ets1:IL-5  A role for Ets1, synergizing with AP-1 and GAT...   \n",
       "2    16373364:0:Ets1:AP-1  A role for Ets1, synergizing with AP-1 and GAT...   \n",
       "3  16373364:0:GATA-3:Ets1  A role for Ets1, synergizing with AP-1 and GAT...   \n",
       "4  16373364:0:GATA-3:IL-5  A role for Ets1, synergizing with AP-1 and GAT...   \n",
       "\n",
       "       TF      TG  Label       Type  \n",
       "0    Ets1  GATA-3  false        NaN  \n",
       "1    Ets1    IL-5   true  UNDEFINED  \n",
       "2    Ets1    AP-1  false        NaN  \n",
       "3  GATA-3    Ets1  false        NaN  \n",
       "4  GATA-3    IL-5   true  UNDEFINED  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# CREATE THE NTNU DATASET\n",
    "\n",
    "# Return to the original sentence\n",
    "NTNU_dataset['Sentence'] = NTNU_dataset.apply(lambda row: row['Sentence'].replace('[TF]', row['TF']), axis=1)\n",
    "NTNU_dataset['Sentence'] = NTNU_dataset.apply(lambda row: row['Sentence'].replace('[TG]', row['TG']), axis=1)\n",
    "\n",
    "# When the mask is removed, there are some duplicated sentences. They are handled here:\n",
    "\n",
    "# 1) Discard complete duplicates\n",
    "m1 = NTNU_dataset[['Sentence', 'TF', 'TG', 'Label', 'Type']].duplicated()\n",
    "NTNU_dataset = NTNU_dataset[~m1]\n",
    "\n",
    "# 2) If both are validated or unvalidated, discard\n",
    "m2 = NTNU_dataset[['Sentence', 'TF', 'TG', 'validated?']].duplicated(keep=False)\n",
    "NTNU_dataset = NTNU_dataset[~m2]\n",
    "\n",
    "\n",
    "# 3) If Labels differ, keep the validated version\n",
    "m3 = NTNU_dataset[['Sentence', 'TF', 'TG']].duplicated(keep=False) & ~NTNU_dataset['validated?']\n",
    "NTNU_dataset = NTNU_dataset[~m3]\n",
    "\n",
    "# Discard 'validated?' column\n",
    "NTNU_dataset = NTNU_dataset.drop(columns=['validated?'])\n",
    "\n",
    "print(f\"We discard {m1.sum()} sentences as duplicates, {m2.sum()} sentences where labels differ, and {m3.sum()} unvalid sentences where the valid version has different labels.\")\n",
    "NTNU_dataset.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "be46c955",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "2024/3086 (66%) reannotated sentences have changed their labels"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Issues due to TRI label"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Valid?  Label  true_label\n",
       "F       FALSE  TRUE          1067\n",
       "        TRUE   TRUE           485\n",
       "               FALSE          472\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Erroneous sentences due to suboptimal TRI"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Valid?  Label  true_label\n",
       "F       TRUE   FALSE         149\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Other problems encountered:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "            All data        Non valid rows \n",
      "PPI         19              0              \n",
      "dir-gene    27              0              \n",
      "dir-syntax  3050            149            \n",
      "Suboptimal  160             149            \n",
      "negation    60              0              \n",
      "Splitter    11              0              \n"
     ]
    }
   ],
   "source": [
    "# SHOW STATISTICS\n",
    "\n",
    "# Sentences reannotated\n",
    "m = validated_sents['Valid?'] == 'F'\n",
    "md(f'{m.sum()}/{len(m)} ({m.sum() / len(m):.0%}) reannotated sentences have changed their labels')\n",
    "\n",
    "# TRI - Non-TRI issues\n",
    "md(\"Issues due to TRI label\")\n",
    "m = validated_sents['Valid?'] == 'F'\n",
    "display(validated_sents[m][[\"Valid?\", \"Label\", \"true_label\"]].value_counts(dropna=False))\n",
    "\n",
    "md(\"Erroneous sentences due to suboptimal TRI\")\n",
    "m = validated_sents['Problem_with'].str.contains('Suboptimal') & (validated_sents['Valid?'] == 'F')\n",
    "display(validated_sents[m][[\"Valid?\", \"Label\", \"true_label\"]].value_counts(dropna=False))\n",
    "\n",
    "\n",
    "md('Other problems encountered:')\n",
    "problems = set(p for unique in validated_sents['Problem_with'].unique() for p in unique.split('|'))\n",
    "problems.discard('')\n",
    "print(f\"{'':<11} {'All data':<15} {'Non valid rows':<15}\")\n",
    "for problem in problems:\n",
    "    in_all_data  = validated_sents['Problem_with'].str.contains(problem).sum()\n",
    "    in_incorrect = validated_sents[m]['Problem_with'].str.contains(problem).sum()\n",
    "    print(f\"{problem:<11} {in_all_data:<15} {in_incorrect:<15}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "25a1a608",
   "metadata": {},
   "source": [
    "## Save the dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "8d1b31df",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Ensure data has expected values\n",
    "assert set(training_data['Label']).issubset({'false', 'true'})\n",
    "assert set(training_data['Type']).issubset({np.nan, 'UNDEFINED', 'ACTIVATION', 'REPRESSION'})\n",
    "assert ((training_data['Label'] == 'False') & ~(training_data['Type'].isna())).sum() == 0\n",
    "\n",
    "# Convert it to False and True\n",
    "training_data['Label'] = training_data['Label'].map({'false': False, 'true': True})\n",
    "NTNU_dataset['Label'] = NTNU_dataset['Label'].map({'false': 'Not TRI', 'true': 'TRI'})\n",
    "\n",
    "# Save the datasets\n",
    "training_data.to_csv(training_data_path, sep='\\t')\n",
    "NTNU_dataset.to_csv(NTNU_dataset_path, sep='\\t', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "26076f9f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "NTNU dataset, 21501 sentences. 11521 are TRI."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Label    Type      \n",
       "Not TRI  NaN           9980\n",
       "TRI      ACTIVATION    5395\n",
       "         UNDEFINED     4073\n",
       "         REPRESSION    2053\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Training data, 21572 sentences"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Label  Type      \n",
       "False  NaN           10169\n",
       "True   ACTIVATION     5344\n",
       "       UNDEFINED      4044\n",
       "       REPRESSION     2015\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "md(f\"NTNU dataset, {len(NTNU_dataset)} sentences. {(NTNU_dataset['Label'] == 'TRI').sum()} are TRI.\")\n",
    "display(NTNU_dataset[['Label', 'Type']].value_counts(dropna=False))\n",
    "\n",
    "md(f\"Training data, {len(training_data)} sentences\")\n",
    "display(training_data[['Label', 'Type']].value_counts(dropna=False))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "84951ad3",
   "metadata": {},
   "source": [
    "## Old code - Add negative sentences\n",
    "We tried to add negative sentences to the dataset in an attempt for the classifier to flag them as negative. It did not give good results, so we abandoned the idea."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "de13c9ff",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "<h4>Negations</h4>"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "There are 60 sentences that are negations, considered as 'True'"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# NEGATIONS - Only mentioned, kept as positive\n",
    "h4('Negations')\n",
    "m_negations =  validated_sents['Problem_with'].str.contains('negation')\n",
    "md(f\"There are {m_negations.sum()} sentences that are negations, considered as 'True'\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f0fe1f1b",
   "metadata": {},
   "source": [
    "### Enhance with negations from the extended NTNU dataset"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bf720de4",
   "metadata": {},
   "source": [
    "`original_tri_sentences.tsv` is a refined subset of the NTNU dataset, a set of 40K validated sentences from ExTRI. Between other things, it did not include sentences marked as 'negations'. Without those, the model doesn't learn that sentences such as:\n",
    "> TF has been observed to have no effect on TG\n",
    "Are marked as 'Valid' by the model. \n",
    "\n",
    "Thus, some of the negation sentences from the original NTNU dataset are recovered to enhance training data with negations."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "9932c60f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unique PMIDs 8917\n",
      "Unique PMIDs 20165\n",
      "Unique PMIDs 14254\n",
      "Difference: 2614\n",
      "Difference: 7951\n",
      "18986 25260\n"
     ]
    }
   ],
   "source": [
    "NTNU_extended_salvable = NTNU_extended[~NTNU_extended['Sentence'].isna()]\n",
    "\n",
    "len(NTNU_dataset), len(NTNU_extended)\n",
    "\n",
    "NTNU_dataset['PMID'] = NTNU_dataset['#TRI ID'].apply(lambda x: x.split(':')[0])\n",
    "\n",
    "NTNU_extended['PMID'] = NTNU_extended['#TRI'].apply(lambda x: x.split(':')[0])\n",
    "\n",
    "\n",
    "for df in [NTNU_dataset, NTNU_extended, NTNU_extended_salvable]:\n",
    "    print(\"Unique PMIDs\", len(df['PMID'].unique()))\n",
    "\n",
    "print(\"Difference:\", len(set(NTNU_dataset['PMID']).difference(set(NTNU_extended_salvable['PMID']))))\n",
    "print(\"Difference:\", len(set(NTNU_extended_salvable['PMID']).difference(set(NTNU_dataset['PMID']))))\n",
    "\n",
    "print(NTNU_dataset['PMID'].isin(NTNU_extended['PMID']).sum(), NTNU_extended['PMID'].isin(NTNU_dataset['PMID']).sum())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9217c2be",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "29d3edcb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#TRI</th>\n",
       "      <th>Valid</th>\n",
       "      <th>Sign</th>\n",
       "      <th>Negation</th>\n",
       "      <th>QC</th>\n",
       "      <th>Comment</th>\n",
       "      <th>Source file</th>\n",
       "      <th>Sentence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>10026114:5:ETV4:GGT1</td>\n",
       "      <td>true</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>true</td>\n",
       "      <td>---{\"transcription factor (associated gene nam...</td>\n",
       "      <td>AGS_24h_transcriptome_QCed_curation_sept21_cor...</td>\n",
       "      <td>Using transient transfection, PEA3 activated t...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>10411295:1:ESR1:CKB</td>\n",
       "      <td>true</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>true</td>\n",
       "      <td>---{\"transcription factor (associated gene nam...</td>\n",
       "      <td>AGS_24h_transcriptome_QCed_curation_sept21_cor...</td>\n",
       "      <td>ECC-1 endometrial cancer cells express estroge...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>10631113:0:SP1:TNXB</td>\n",
       "      <td>true</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>true</td>\n",
       "      <td>---{\"transcription factor (associated gene nam...</td>\n",
       "      <td>AGS_24h_transcriptome_QCed_curation_sept21_cor...</td>\n",
       "      <td>Transcription factor Sp1 activates the express...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>11397003:11:GATA5:MYL3</td>\n",
       "      <td>true</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>true</td>\n",
       "      <td>---{\"transcription factor (associated gene nam...</td>\n",
       "      <td>AGS_24h_transcriptome_QCed_curation_sept21_cor...</td>\n",
       "      <td>Moreover, overexpression of gata5 in Zoep muta...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>11877413:5:NFIC:CEL</td>\n",
       "      <td>true</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>true</td>\n",
       "      <td>---{\"transcription factor (associated gene nam...</td>\n",
       "      <td>AGS_24h_transcriptome_QCed_curation_sept21_cor...</td>\n",
       "      <td>During development of the mouse mammary gland,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47105</th>\n",
       "      <td>12832543:8:JUN:AKT1</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "      <td>false</td>\n",
       "      <td>false</td>\n",
       "      <td>---{\"transcription factor\":[\"JUN\"],\"target gen...</td>\n",
       "      <td>unsigned_curated_subset_May_2023.xlsx</td>\n",
       "      <td>We conclude that the combination of ATF3 and c...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47107</th>\n",
       "      <td>7753624:4:JUN:ATP6AP1</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "      <td>false</td>\n",
       "      <td>false</td>\n",
       "      <td>---{\"transcription factor\":[\"JUN\"],\"target gen...</td>\n",
       "      <td>unsigned_curated_subset_May_2023.xlsx</td>\n",
       "      <td>Studies of the regulatory activities of the re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47108</th>\n",
       "      <td>14764703:7:JUN:BTK</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "      <td>false</td>\n",
       "      <td>false</td>\n",
       "      <td>---{\"transcription factor\":[\"JUN\"],\"target gen...</td>\n",
       "      <td>unsigned_curated_subset_May_2023.xlsx</td>\n",
       "      <td>In addition, NADA inhibited both binding to DN...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47109</th>\n",
       "      <td>16365438:5:JUN:BTK</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "      <td>false</td>\n",
       "      <td>false</td>\n",
       "      <td>---{\"transcription factor\":[\"JUN\"],\"target gen...</td>\n",
       "      <td>unsigned_curated_subset_May_2023.xlsx</td>\n",
       "      <td>This inhibition was partially reverted by over...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47110</th>\n",
       "      <td>8683110:9:JUN:BTK</td>\n",
       "      <td>false</td>\n",
       "      <td>NaN</td>\n",
       "      <td>false</td>\n",
       "      <td>false</td>\n",
       "      <td>---{\"transcription factor\":[\"JUN\"],\"target gen...</td>\n",
       "      <td>unsigned_curated_subset_May_2023.xlsx</td>\n",
       "      <td>These results suggest that oxidative signals c...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>16485 rows  8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         #TRI  Valid Sign Negation     QC  \\\n",
       "33       10026114:5:ETV4:GGT1   true    +      NaN   true   \n",
       "36        10411295:1:ESR1:CKB   true    +      NaN   true   \n",
       "37        10631113:0:SP1:TNXB   true    +      NaN   true   \n",
       "40     11397003:11:GATA5:MYL3   true    +      NaN   true   \n",
       "44        11877413:5:NFIC:CEL   true    +      NaN   true   \n",
       "...                       ...    ...  ...      ...    ...   \n",
       "47105     12832543:8:JUN:AKT1  false  NaN    false  false   \n",
       "47107   7753624:4:JUN:ATP6AP1  false  NaN    false  false   \n",
       "47108      14764703:7:JUN:BTK  false  NaN    false  false   \n",
       "47109      16365438:5:JUN:BTK  false  NaN    false  false   \n",
       "47110       8683110:9:JUN:BTK  false  NaN    false  false   \n",
       "\n",
       "                                                 Comment  \\\n",
       "33     ---{\"transcription factor (associated gene nam...   \n",
       "36     ---{\"transcription factor (associated gene nam...   \n",
       "37     ---{\"transcription factor (associated gene nam...   \n",
       "40     ---{\"transcription factor (associated gene nam...   \n",
       "44     ---{\"transcription factor (associated gene nam...   \n",
       "...                                                  ...   \n",
       "47105  ---{\"transcription factor\":[\"JUN\"],\"target gen...   \n",
       "47107  ---{\"transcription factor\":[\"JUN\"],\"target gen...   \n",
       "47108  ---{\"transcription factor\":[\"JUN\"],\"target gen...   \n",
       "47109  ---{\"transcription factor\":[\"JUN\"],\"target gen...   \n",
       "47110  ---{\"transcription factor\":[\"JUN\"],\"target gen...   \n",
       "\n",
       "                                             Source file  \\\n",
       "33     AGS_24h_transcriptome_QCed_curation_sept21_cor...   \n",
       "36     AGS_24h_transcriptome_QCed_curation_sept21_cor...   \n",
       "37     AGS_24h_transcriptome_QCed_curation_sept21_cor...   \n",
       "40     AGS_24h_transcriptome_QCed_curation_sept21_cor...   \n",
       "44     AGS_24h_transcriptome_QCed_curation_sept21_cor...   \n",
       "...                                                  ...   \n",
       "47105              unsigned_curated_subset_May_2023.xlsx   \n",
       "47107              unsigned_curated_subset_May_2023.xlsx   \n",
       "47108              unsigned_curated_subset_May_2023.xlsx   \n",
       "47109              unsigned_curated_subset_May_2023.xlsx   \n",
       "47110              unsigned_curated_subset_May_2023.xlsx   \n",
       "\n",
       "                                                Sentence  \n",
       "33     Using transient transfection, PEA3 activated t...  \n",
       "36     ECC-1 endometrial cancer cells express estroge...  \n",
       "37     Transcription factor Sp1 activates the express...  \n",
       "40     Moreover, overexpression of gata5 in Zoep muta...  \n",
       "44     During development of the mouse mammary gland,...  \n",
       "...                                                  ...  \n",
       "47105  We conclude that the combination of ATF3 and c...  \n",
       "47107  Studies of the regulatory activities of the re...  \n",
       "47108  In addition, NADA inhibited both binding to DN...  \n",
       "47109  This inhibition was partially reverted by over...  \n",
       "47110  These results suggest that oxidative signals c...  \n",
       "\n",
       "[16485 rows x 8 columns]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "NTNU_dataset['Sentence'].isin(NTNU_extended['Sentence']).sum(), NTNU_extended['Sentence'].isin(NTNU_dataset['Sentence']).sum()\n",
    "\n",
    "m = ~NTNU_extended['Sentence'].isin(NTNU_dataset['Sentence']) & ~NTNU_extended['Sentence'].isna()\n",
    "NTNU_extended[m]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f2527801",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG most likely MYH10 (nonmuscle-type myosin heavy chain B)---{\"tf associated gene name\":[\"MEF2A\"],\"tg associated gene name\":[\"MYH14\"],\"tf text\":[\"MEF2A\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[\"x\"],\"comment\":[\"TG most likely MYH10 (nonmuscle-type myosin heavy chain B)\"],\"sentence\":[\"Furthermore\",\"Ang II induced the promoter activity of the nonmuscle-type myosin heavy chain B gene\",\"which we used as a marker of the dedifferentiated state of VSMCs\",\"and this increase was inhibited by calcineurin inhibitors but not by the dominant-negative mutants of MEF2A or mitogen-activated protein kinase kinase 6.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[true],\"QC\":[null]}\n",
      "TG most likely MYH10 (nonmuscle-type myosin heavy chain B)---{\"tf associated gene name\":[\"MEF2A\"],\"tg associated gene name\":[\"MYH14\"],\"tf text\":[\"MEF2A\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[\"x\"],\"comment\":[\"TG most likely MYH10 (nonmuscle-type myosin heavy chain B)\"],\"sentence\":[\"Furthermore\",\"Ang II induced the promoter activity of the nonmuscle-type myosin heavy chain B gene\",\"which we used as a marker of the dedifferentiated state of VSMCs\",\"and this increase was inhibited by calcineurin inhibitors but not by the dominant-negative mutants of MEF2A or mitogen-activated protein kinase kinase 6.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MEF2A\"],\"target gene (associated gene name)\":[\"MYH14\"],\"valid\":[\"0\"],\"sign\":[\"n\"],\"negation\":[],\"qc\":[\"1\"],\"tffusion\":[],\"sentence\":[\"Furthermore\",\"Ang II induced the promoter activity of the nonmuscle-type myosin heavy chain B gene\",\"which we used as a marker of the dedifferentiated state of VSMCs\",\"and this increase was inhibited by calcineurin inhibitors but not by the dominant-negative mutants of MEF2A or mitogen-activated protein kinase kinase 6.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"MEF2A\"],\"target gene (associated gene name)\":[\"MYH14\"],\"valid\":[\"1\"],\"sign\":[\"n\"],\"negation\":[],\"qc\":[\"1\"],\"tffusion\":[],\"sentence\":[\"Furthermore\",\"Ang II induced the promoter activity of the nonmuscle-type myosin heavy chain B gene\",\"which we used as a marker of the dedifferentiated state of VSMCs\",\"and this increase was inhibited by calcineurin inhibitors but not by the dominant-negative mutants of MEF2A or mitogen-activated protein kinase kinase 6.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 336 (char 335)\n",
      "{\"transcription factor (associated gene name)\":[\"BACH1\"],\"target gene (associated gene name)\":[\"TXNRD1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In contrast\",\"thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BACH1\"],\"tg (associated gene name)\":[\"TXNRD1\"],\"tf text\":[\"BACH1\"],\"tg text\":[\"TXNRD1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast, thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"BACH1\"],\"tg associated gene name\":[\"TXNRD1\"],\"tf text\":[\"BACH1\"],\"tg text\":[\"TXNRD1\"],\"1\":[\"u\"],\"negation\":[\"x\"],\"not true\":[],\"comment\":[],\"sentence\":[\"In contrast\",\"thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"BACH1\"],\"target gene (associated gene name)\":[\"TXNRD1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"v\"],\"negation\":[\"x\"],\"sentence\":[\"In contrast\",\"thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.131734\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"84\"],\"sentence pair density\":[\"0.00425531914893617\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"BACH1\"],\"target gene (associated gene name)\":[\"TXNRD1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In contrast\",\"thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BACH1\"],\"tg (associated gene name)\":[\"TXNRD1\"],\"tf text\":[\"BACH1\"],\"tg text\":[\"TXNRD1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast, thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"BACH1\"],\"target gene (associated gene name)\":[\"TXNRD1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"x\"],\"qc\":[\"1\"],\"tffusion\":[],\"sentence\":[\"In contrast\",\"thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"BACH1\"],\"target gene (associated gene name)\":[\"TXNRD1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"x\"],\"qc\":[\"1\"],\"tffusion\":[],\"sentence\":[\"In contrast\",\"thioredoxin reductase 1 (TXNRD1) is regulated by NRF2 but not by BACH1.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs (both enzymes) not named---{\"tf associated gene name\":[\"FOSB\"],\"tg associated gene name\":[\"ELL2\"],\"tf text\":[\"FOSB\"],\"tg text\":[\"ELL2\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"TGs (both enzymes) not named\"],\"sentence\":[\"BHLHB2\",\"BTG2\",\"and SALL1 decreased expression of both enzymes\",\"whereas CITED2\",\"EGR2\",\"ELL2\",\"FOS\",\"FOSB\",\"HDAC5\",\"MAFF\",\"MITF\",\"NFIL3\",\"NR4A1\",\"NR4A2\",\"NR4A3\",\"PER1\",\"and VDR increased expression for both enzymes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TGs (both enzymes) not named---{\"tf associated gene name\":[\"FOSB\"],\"tg associated gene name\":[\"ELL2\"],\"tf text\":[\"FOSB\"],\"tg text\":[\"ELL2\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"TGs (both enzymes) not named\"],\"sentence\":[\"BHLHB2\",\"BTG2\",\"and SALL1 decreased expression of both enzymes\",\"whereas CITED2\",\"EGR2\",\"ELL2\",\"FOS\",\"FOSB\",\"HDAC5\",\"MAFF\",\"MITF\",\"NFIL3\",\"NR4A1\",\"NR4A2\",\"NR4A3\",\"PER1\",\"and VDR increased expression for both enzymes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"ELL2\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[\"BHLHB2\",\"BTG2\",\"and SALL1 decreased expression of both enzymes\",\"whereas CITED2\",\"EGR2\",\"ELL2\",\"FOS\",\"FOSB\",\"HDAC5\",\"MAFF\",\"MITF\",\"NFIL3\",\"NR4A1\",\"NR4A2\",\"NR4A3\",\"PER1\",\"and VDR increased expression for both enzymes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"ELL2\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"BHLHB2\",\"BTG2\",\"and SALL1 decreased expression of both enzymes\",\"whereas CITED2\",\"EGR2\",\"ELL2\",\"FOS\",\"FOSB\",\"HDAC5\",\"MAFF\",\"MITF\",\"NFIL3\",\"NR4A1\",\"NR4A2\",\"NR4A3\",\"PER1\",\"and VDR increased expression for both enzymes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "viral TG (reactivation from latency to lytic replication)---{\"tf associated gene name\":[\"JDP2\"],\"tg associated gene name\":[\"HEPHL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"viral TG (reactivation from latency to lytic replication)\"],\"sentence\":[\"These results indicate that JDP2 plays a role as a repressor of Zp and that its replacement by CREB/ATF/AP-1 at ZII is crucial to triggering reactivation from latency to lytic replication.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JDP2\"],\"tg (associated gene name)\":[\"HEPHL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that JDP2 plays a role as a repressor of Zp and that its replacement by CREB/ATF/AP-1 at ZII is crucial to triggering reactivation from latency to lytic replication.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "viral TG (reactivation from latency to lytic replication)---{\"tf associated gene name\":[\"JDP2\"],\"tg associated gene name\":[\"HEPHL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"viral TG (reactivation from latency to lytic replication)\"],\"sentence\":[\"These results indicate that JDP2 plays a role as a repressor of Zp and that its replacement by CREB/ATF/AP-1 at ZII is crucial to triggering reactivation from latency to lytic replication.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JDP2\"],\"tg (associated gene name)\":[\"HEPHL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that JDP2 plays a role as a repressor of Zp and that its replacement by CREB/ATF/AP-1 at ZII is crucial to triggering reactivation from latency to lytic replication.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"JDP2\"],\"target gene (associated gene name)\":[\"HEPHL1\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"These results indicate that JDP2 plays a role as a repressor of Zp and that its replacement by CREB/ATF/AP-1 at ZII is crucial to triggering reactivation from latency to lytic replication.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is ZID---{\"tf associated gene name\":[\"MEF2A\"],\"tg associated gene name\":[\"HEPHL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is ZID\"],\"sentence\":[\"The binding of MEF2 transcription factor to ZID was characterized in detail and linked functionally to Zp promoter activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is ZID---{\"tf associated gene name\":[\"MEF2A\"],\"tg associated gene name\":[\"HEPHL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is ZID\"],\"sentence\":[\"The binding of MEF2 transcription factor to ZID was characterized in detail and linked functionally to Zp promoter activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MEF2A\"],\"target gene (associated gene name)\":[\"HEPHL1\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"The binding of MEF2 transcription factor to ZID was characterized in detail and linked functionally to Zp promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"MYH14\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We also provide the first evidence that Eos can regulate expression of additional myosin genes (Myosin 1 and ??-Myosin) via the miR-499/Sox6 pathway.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"MYH14\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We also provide the first evidence that Eos can regulate expression of additional myosin genes (Myosin 1 and ??-Myosin) via the miR-499/Sox6 pathway.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"MLXIP\"],\"target gene (associated gene name)\":[\"MYH14\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"We also provide the first evidence that Eos can regulate expression of additional myosin genes (Myosin 1 and -Myosin) via the miR-499/Sox6 pathway.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Extra data: line 1 column 1194 (char 1193)\n",
      "{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"ATP8A2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"This review summarizes our current knowledge and understanding of the elements involved in HIV-1 transcriptional reactivation: (1) the site of integration; (2) the transcription factor NF-kappaB\",\"which is induced by proinflammatory cytokines (such as TNFalpha) and binds to two kappaB sites in the HIV-1 promoter region; (3) the specific remodeling of a single nucleosome (called nuc-1 and located immediately downstream of the HIV-1 transcription start site under latency conditions) upon activation of the HIV-1 promoter; (4) post-translational acetylation of histones and of non-histone proteins (following treatment with deacetylases inhibitors\",\"which induce viral transcription and nuc-1 remodeling); and (5) the viral trans-activator Tat\",\"which promotes transcription by mediating the recruitment to the HIV-1 promoter of histone-modifying enzymes and ATP-dependent chromatin remodeling complexes required for nucleosome disruption and transcriptional processivity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HIV-1 TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"ATP8A2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"HIV-1 TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This review summarizes our current knowledge and understanding of the elements involved in HIV-1 transcriptional reactivation: (1) the site of integration; (2) the transcription factor NF-kappaB, which is induced by proinflammatory cytokines (such as TNFalpha) and binds to two kappaB sites in the HIV-1 promoter region; (3) the specific remodeling of a single nucleosome (called nuc-1 and located immediately downstream of the HIV-1 transcription start site under latency conditions) upon activation of the HIV-1 promoter; (4) post-translational acetylation of histones and of non-histone proteins (following treatment with deacetylases inhibitors, which induce viral transcription and nuc-1 remodeling); and (5) the viral trans-activator Tat, which promotes transcription by mediating the recruitment to the HIV-1 promoter of histone-modifying enzymes and ATP-dependent chromatin remodeling complexes required for nucleosome disruption and transcriptional processivity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"ATP8A2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"This review summarizes our current knowledge and understanding of the elements involved in HIV-1 transcriptional reactivation: (1) the site of integration; (2) the transcription factor NF-kappaB\",\"which is induced by proinflammatory cytokines (such as TNFalpha) and binds to two kappaB sites in the HIV-1 promoter region; (3) the specific remodeling of a single nucleosome (called nuc-1 and located immediately downstream of the HIV-1 transcription start site under latency conditions) upon activation of the HIV-1 promoter; (4) post-translational acetylation of histones and of non-histone proteins (following treatment with deacetylases inhibitors\",\"which induce viral transcription and nuc-1 remodeling); and (5) the viral trans-activator Tat\",\"which promotes transcription by mediating the recruitment to the HIV-1 promoter of histone-modifying enzymes and ATP-dependent chromatin remodeling complexes required for nucleosome disruption and transcriptional processivity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HIV-1 TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"ATP8A2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"HIV-1 TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This review summarizes our current knowledge and understanding of the elements involved in HIV-1 transcriptional reactivation: (1) the site of integration; (2) the transcription factor NF-kappaB, which is induced by proinflammatory cytokines (such as TNFalpha) and binds to two kappaB sites in the HIV-1 promoter region; (3) the specific remodeling of a single nucleosome (called nuc-1 and located immediately downstream of the HIV-1 transcription start site under latency conditions) upon activation of the HIV-1 promoter; (4) post-translational acetylation of histones and of non-histone proteins (following treatment with deacetylases inhibitors, which induce viral transcription and nuc-1 remodeling); and (5) the viral trans-activator Tat, which promotes transcription by mediating the recruitment to the HIV-1 promoter of histone-modifying enzymes and ATP-dependent chromatin remodeling complexes required for nucleosome disruption and transcriptional processivity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"ATP8A2\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"This review summarizes our current knowledge and understanding of the elements involved in HIV-1 transcriptional reactivation: (1) the site of integration; (2) the transcription factor NF-kappaB\",\"which is induced by proinflammatory cytokines (such as TNFalpha) and binds to two kappaB sites in the HIV-1 promoter region; (3) the specific remodeling of a single nucleosome (called nuc-1 and located immediately downstream of the HIV-1 transcription start site under latency conditions) upon activation of the HIV-1 promoter; (4) post-translational acetylation of histones and of non-histone proteins (following treatment with deacetylases inhibitors\",\"which induce viral transcription and nuc-1 remodeling); and (5) the viral trans-activator Tat\",\"which promotes transcription by mediating the recruitment to the HIV-1 promoter of histone-modifying enzymes and ATP-dependent chromatin remodeling complexes required for nucleosome disruption and transcriptional processivity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is MMTV (mammary tumor virus?)---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"FAM89B\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is MMTV (mammary tumor virus?)\"],\"sentence\":[\"E1A\",\"p65\",\"and GR bind the MMTV promoter\",\"and chromatin remodeling enhances binding on both repressed and activated promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"FAM89B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"E1A, p65, and GR bind the MMTV promoter, and chromatin remodeling enhances binding on both repressed and activated promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is MMTV (mammary tumor virus?)---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"FAM89B\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is MMTV (mammary tumor virus?)\"],\"sentence\":[\"E1A\",\"p65\",\"and GR bind the MMTV promoter\",\"and chromatin remodeling enhances binding on both repressed and activated promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"FAM89B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"E1A, p65, and GR bind the MMTV promoter, and chromatin remodeling enhances binding on both repressed and activated promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"NR3C1\"],\"target gene (associated gene name)\":[\"FAM89B\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"E1A\",\"p65\",\"and GR bind the MMTV promoter\",\"and chromatin remodeling enhances binding on both repressed and activated promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is glucagon---{\"tf associated gene name\":[\"PAX6\"],\"tg associated gene name\":[\"AIF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is glucagon\"],\"sentence\":[\"In this report\",\"we show that Pax6 additionally binds the glucagon gene promoter G1-element and forms a transcriptionally active complex with another homeodomain protein\",\"Cdx2/3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|G1 enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX6\"],\"tg (associated gene name)\":[\"AIF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"G1 enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this report, we show that Pax6 additionally binds the glucagon gene promoter G1-element and forms a transcriptionally active complex with another homeodomain protein, Cdx2/3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is glucagon---{\"tf associated gene name\":[\"PAX6\"],\"tg associated gene name\":[\"AIF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is glucagon\"],\"sentence\":[\"In this report\",\"we show that Pax6 additionally binds the glucagon gene promoter G1-element and forms a transcriptionally active complex with another homeodomain protein\",\"Cdx2/3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|G1 enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX6\"],\"tg (associated gene name)\":[\"AIF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"G1 enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this report, we show that Pax6 additionally binds the glucagon gene promoter G1-element and forms a transcriptionally active complex with another homeodomain protein, Cdx2/3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"PAX6\"],\"target gene (associated gene name)\":[\"AIF1\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"In this report\",\"we show that Pax6 additionally binds the glucagon gene promoter G1-element and forms a transcriptionally active complex with another homeodomain protein\",\"Cdx2/3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 2 (char 1)\n",
      " TRE for other pair (increased SULT1C expression through an FXR-independent mechanism)---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"SULT1C4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\" TRE for other pair (increased SULT1C expression through an FXR-independent mechanism)\"],\"sentence\":[\"Expression of AhR\",\"LXR??\",\"LXR??\",\"PPAR??\",\"PPAR??\",\"PXR\",\"and VDR was confirmed in LS180 cells using quantitative reverse-transcription polymerase chain reaction; however\",\"FXR expression was negligible\",\"suggesting that GW4064 increased SULT1C expression through an FXR-independent mechanism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"SULT1C4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression of AhR, LXR, LXR, PPAR, PPAR, PXR, and VDR was confirmed in LS180 cells using quantitative reverse-transcription polymerase chain reaction; however, FXR expression was negligible, suggesting that GW4064 increased SULT1C expression through an FXR-independent mechanism.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      " TRE for other pair (increased SULT1C expression through an FXR-independent mechanism)---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"SULT1C4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\" TRE for other pair (increased SULT1C expression through an FXR-independent mechanism)\"],\"sentence\":[\"Expression of AhR\",\"LXR??\",\"LXR??\",\"PPAR??\",\"PPAR??\",\"PXR\",\"and VDR was confirmed in LS180 cells using quantitative reverse-transcription polymerase chain reaction; however\",\"FXR expression was negligible\",\"suggesting that GW4064 increased SULT1C expression through an FXR-independent mechanism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"SULT1C4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression of AhR, LXR, LXR, PPAR, PPAR, PXR, and VDR was confirmed in LS180 cells using quantitative reverse-transcription polymerase chain reaction; however, FXR expression was negligible, suggesting that GW4064 increased SULT1C expression through an FXR-independent mechanism.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"PPARA\"],\"target gene (associated gene name)\":[\"SULT1C4\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"Expression of AhR\",\"LXR\",\"LXR\",\"PPAR\",\"PPAR\",\"PXR\",\"and VDR was confirmed in LS180 cells using quantitative reverse-transcription polymerase chain reaction; however\",\"FXR expression was negligible\",\"suggesting that GW4064 increased SULT1C expression through an FXR-independent mechanism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is EF-1 alpha---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"NEDD9\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is EF-1 alpha\"],\"sentence\":[\"These results indicate that Sp1 or its related factor cooperatively enhances the expression of the EF-1 alpha gene in the 5' flanking region.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|EL1 alpha?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"NEDD9\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"EL1 alpha?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that Sp1 or its related factor cooperatively enhances the expression of the EF-1 alpha gene in the 5' flanking region.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is EF-1 alpha---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"NEDD9\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is EF-1 alpha\"],\"sentence\":[\"These results indicate that Sp1 or its related factor cooperatively enhances the expression of the EF-1 alpha gene in the 5' flanking region.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|EL1 alpha?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"NEDD9\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"EL1 alpha?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that Sp1 or its related factor cooperatively enhances the expression of the EF-1 alpha gene in the 5' flanking region.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"NEDD9\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"These results indicate that Sp1 or its related factor cooperatively enhances the expression of the EF-1 alpha gene in the 5' flanking region.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is STAT3---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"NOSTRIN\"],\"tf text\":[],\"tg text\":[\"NOSTRIN\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is STAT3\"],\"sentence\":[\"Furthermore\",\"we demonstrate that activation of blunted signal transducers and activators of transcription 3 (STAT3) in the absence of PECAM-1 results in decreased NOSTRIN expression via direct binding of the signal transducers and activators of transcription 3 to the NOSTRIN promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF STAT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"NOSTRIN\"],\"tf text\":[],\"tg text\":[\"NOSTRIN\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF STAT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, we demonstrate that activation of blunted signal transducers and activators of transcription 3 (STAT3) in the absence of PECAM-1 results in decreased NOSTRIN expression via direct binding of the signal transducers and activators of transcription 3 to the NOSTRIN promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is STAT3---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"NOSTRIN\"],\"tf text\":[],\"tg text\":[\"NOSTRIN\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is STAT3\"],\"sentence\":[\"Furthermore\",\"we demonstrate that activation of blunted signal transducers and activators of transcription 3 (STAT3) in the absence of PECAM-1 results in decreased NOSTRIN expression via direct binding of the signal transducers and activators of transcription 3 to the NOSTRIN promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF STAT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"NOSTRIN\"],\"tf text\":[],\"tg text\":[\"NOSTRIN\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF STAT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, we demonstrate that activation of blunted signal transducers and activators of transcription 3 (STAT3) in the absence of PECAM-1 results in decreased NOSTRIN expression via direct binding of the signal transducers and activators of transcription 3 to the NOSTRIN promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"SPI1\"],\"target gene (associated gene name)\":[\"NOSTRIN\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"Furthermore\",\"we demonstrate that activation of blunted signal transducers and activators of transcription 3 (STAT3) in the absence of PECAM-1 results in decreased NOSTRIN expression via direct binding of the signal transducers and activators of transcription 3 to the NOSTRIN promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Extra data: line 1 column 227 (char 226)\n",
      "{\"transcription factor (associated gene name)\":[\"TCF7\"],\"target gene (associated gene name)\":[\"NCR1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF7\"],\"tg (associated gene name)\":[\"NCR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this study, we demonstrate that T cell factor 1 (TCF-1; encoded by Tcf7), a transcription factor also important for NK and T cell differentiation, is expressed by multiple innate lymphoid cell (ILC) subsets, including GATA3(+) nuocytes (ILC2) and NKp46(+) ILCs (ILC3), which confer protection against lung and intestinal inflammation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TCF7\"],\"tg associated gene name\":[\"NCR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In this study\",\"we demonstrate that T cell factor 1 (TCF-1; encoded by Tcf7)\",\"a transcription factor also important for NK and T cell differentiation\",\"is expressed by multiple innate lymphoid cell (ILC) subsets\",\"including GATA3(+) nuocytes (ILC2) and NKp46(+) ILCs (ILC3)\",\"which confer protection against lung and intestinal inflammation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TCF7\"],\"target gene (associated gene name)\":[\"NCR1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF7\"],\"tg (associated gene name)\":[\"NCR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this study, we demonstrate that T cell factor 1 (TCF-1; encoded by Tcf7), a transcription factor also important for NK and T cell differentiation, is expressed by multiple innate lymphoid cell (ILC) subsets, including GATA3(+) nuocytes (ILC2) and NKp46(+) ILCs (ILC3), which confer protection against lung and intestinal inflammation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"TCF7\"],\"target gene (associated gene name)\":[\"NCR1\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"In this study\",\"we demonstrate that T cell factor 1 (TCF-1; encoded by Tcf7)\",\"a transcription factor also important for NK and T cell differentiation\",\"is expressed by multiple innate lymphoid cell (ILC) subsets\",\"including GATA3(+) nuocytes (ILC2) and NKp46(+) ILCs (ILC3)\",\"which confer protection against lung and intestinal inflammation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unsure about TF- and TG-normalization---{\"tf associated gene name\":[\"YBX3\"],\"tg associated gene name\":[\"PDSS1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"unsure about TF- and TG-normalization\"],\"sentence\":[\"Our results suggest that the dps promoter is recognized by both sigma 70-holoenzyme and sigma S-holoenzyme\",\"since OxyR acts through sigma 70 and the starts of the OxyR- and sigma S-dependent transcripts are identical.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YBX3\"],\"tg (associated gene name)\":[\"PDSS1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that the dps promoter is recognized by both sigma 70-holoenzyme and sigma S-holoenzyme, since OxyR acts through sigma 70 and the starts of the OxyR- and sigma S-dependent transcripts are identical.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "unsure about TF- and TG-normalization---{\"tf associated gene name\":[\"YBX3\"],\"tg associated gene name\":[\"PDSS1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"unsure about TF- and TG-normalization\"],\"sentence\":[\"Our results suggest that the dps promoter is recognized by both sigma 70-holoenzyme and sigma S-holoenzyme\",\"since OxyR acts through sigma 70 and the starts of the OxyR- and sigma S-dependent transcripts are identical.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YBX3\"],\"tg (associated gene name)\":[\"PDSS1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that the dps promoter is recognized by both sigma 70-holoenzyme and sigma S-holoenzyme, since OxyR acts through sigma 70 and the starts of the OxyR- and sigma S-dependent transcripts are identical.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"YBX3\"],\"target gene (associated gene name)\":[\"PDSS1\"],\"valid\":[\"0\"],\"qc\":[\"0\"],\"sentence\":[\"Our results suggest that the dps promoter is recognized by both sigma 70-holoenzyme and sigma S-holoenzyme\",\"since OxyR acts through sigma 70 and the starts of the OxyR- and sigma S-dependent transcripts are identical.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Extra data: line 1 column 411 (char 410)\n",
      "{\"transcription factor (associated gene name)\":[\"POU3F2\"],\"target gene (associated gene name)\":[\"MITF\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"However\",\"recently the paired box transcription factor PAX3 was shown to transcriptionally activate POU3F2/BRN2\",\"leading to direct repression of MITF expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|PAX3 is TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU3F2\"],\"tg (associated gene name)\":[\"MITF\"],\"tf text\":[\"POU3F2\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"PAX3 is TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, recently the paired box transcription factor PAX3 was shown to transcriptionally activate POU3F2/BRN2, leading to direct repression of MITF expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"POU3F2\"],\"tg associated gene name\":[\"MITF\"],\"tf text\":[\"POU3F2\"],\"tg text\":[],\"1\":[\"r\"],\"0\":[],\"negation\":[\"x\"],\"comment\":[],\"sentence\":[\"However\",\"recently the paired box transcription factor PAX3 was shown to transcriptionally activate POU3F2/BRN2\",\"leading to direct repression of MITF expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"POU3F2\"],\"target gene (associated gene name)\":[\"MITF\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"However\",\"recently the paired box transcription factor PAX3 was shown to transcriptionally activate POU3F2/BRN2\",\"leading to direct repression of MITF expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|PAX3 is TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU3F2\"],\"tg (associated gene name)\":[\"MITF\"],\"tf text\":[\"POU3F2\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"PAX3 is TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, recently the paired box transcription factor PAX3 was shown to transcriptionally activate POU3F2/BRN2, leading to direct repression of MITF expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 410 (char 409)\n",
      "{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Pretreatment of monocyte with a RhoA inhibitor\",\"C3 transferase from Clostridium botulinum\",\"effectively blocked fMLP-induced NF-kappaB activation as well as interleukin-1beta gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Pretreatment of monocyte with a RhoA inhibitor, C3 transferase from Clostridium botulinum, effectively blocked fMLP-induced NF-kappaB activation as well as interleukin-1beta gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Pretreatment of monocyte with a RhoA inhibitor\",\"C3 transferase from Clostridium botulinum\",\"effectively blocked fMLP-induced NF-kappaB activation as well as interleukin-1beta gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Pretreatment of monocyte with a RhoA inhibitor, C3 transferase from Clostridium botulinum, effectively blocked fMLP-induced NF-kappaB activation as well as interleukin-1beta gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 880 (char 879)\n",
      "{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"EGR1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Real-time PCR verified photochemical up-regulation of the HSP family genes\",\"as well as down-regulation of EGR-1 at 2-10h post-treatment\",\"suggesting that the HSP (particularly HSP70)\",\"in addition to the microarray-identified metallothioneins\",\"but not the EGR-1 promoters\",\"could be relevant promoter candidates for transcriptional targeting via PCI.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.69138\"],\"pmid counts\":[\"92\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"347\"],\"sentence pair density\":[\"0.0189873417721519\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|complicated---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"EGR1\"],\"tf text\":[\"EGR-1\"],\"tg text\":[\"EGR-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"complicated\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Real-time PCR verified photochemical up-regulation of the HSP family genes, as well as down-regulation of EGR-1 at 2-10h post-treatment, suggesting that the HSP (particularly HSP70), in addition to the microarray-identified metallothioneins, but not the EGR-1 promoters, could be relevant promoter candidates for transcriptional targeting via PCI.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"EGR1\"],\"tf text\":[\"EGR-1\"],\"tg text\":[\"EGR-1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Real-time PCR verified photochemical up-regulation of the HSP family genes\",\"as well as down-regulation of EGR-1 at 2-10h post-treatment\",\"suggesting that the HSP (particularly HSP70)\",\"in addition to the microarray-identified metallothioneins\",\"but not the EGR-1 promoters\",\"could be relevant promoter candidates for transcriptional targeting via PCI.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"EGR1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Real-time PCR verified photochemical up-regulation of the HSP family genes\",\"as well as down-regulation of EGR-1 at 2-10h post-treatment\",\"suggesting that the HSP (particularly HSP70)\",\"in addition to the microarray-identified metallothioneins\",\"but not the EGR-1 promoters\",\"could be relevant promoter candidates for transcriptional targeting via PCI.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.69138\"],\"pmid counts\":[\"92\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"347\"],\"sentence pair density\":[\"0.0189873417721519\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|complicated---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"EGR1\"],\"tf text\":[\"EGR-1\"],\"tg text\":[\"EGR-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"complicated\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Real-time PCR verified photochemical up-regulation of the HSP family genes, as well as down-regulation of EGR-1 at 2-10h post-treatment, suggesting that the HSP (particularly HSP70), in addition to the microarray-identified metallothioneins, but not the EGR-1 promoters, could be relevant promoter candidates for transcriptional targeting via PCI.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 599 (char 598)\n",
      "{\"tf associated gene name\":[\"NCOA1\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The androgen and androgen receptor (AR)-regulated gene expression plays important roles in normal prostate and prostate cancer development\",\"and AR transcriptional control of genes is mediated by transcriptional coactivators\",\"including the three members of the steroid receptor coactivator (SRC) family\",\"SRC-1 (NCOA1)\",\"SRC-2 (TIF2/GRIP1/NCOA2) and SRC-3 (AIB1\",\"ACTR/RAC3/NCOA3).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|only TF part---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA1\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"only TF part\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The androgen and androgen receptor (AR)-regulated gene expression plays important roles in normal prostate and prostate cancer development, and AR transcriptional control of genes is mediated by transcriptional coactivators, including the three members of the steroid receptor coactivator (SRC) family, SRC-1 (NCOA1), SRC-2 (TIF2/GRIP1/NCOA2) and SRC-3 (AIB1, ACTR/RAC3/NCOA3).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NCOA1\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The androgen and androgen receptor (AR)-regulated gene expression plays important roles in normal prostate and prostate cancer development\",\"and AR transcriptional control of genes is mediated by transcriptional coactivators\",\"including the three members of the steroid receptor coactivator (SRC) family\",\"SRC-1 (NCOA1)\",\"SRC-2 (TIF2/GRIP1/NCOA2) and SRC-3 (AIB1\",\"ACTR/RAC3/NCOA3).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|only TF part---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA1\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"only TF part\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The androgen and androgen receptor (AR)-regulated gene expression plays important roles in normal prostate and prostate cancer development, and AR transcriptional control of genes is mediated by transcriptional coactivators, including the three members of the steroid receptor coactivator (SRC) family, SRC-1 (NCOA1), SRC-2 (TIF2/GRIP1/NCOA2) and SRC-3 (AIB1, ACTR/RAC3/NCOA3).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "FNL normalization correct---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"WAF1\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"FNL normalization correct\"],\"sentence\":[\"HDAC4 interacts with Sp1\",\"binds and reduces histone H3 acetylation at the Sp1/Sp3 binding site-rich p21(WAF1/Cip1) proximal promoter\",\"suggesting a key role for Sp1 in HDAC4-mediated repression of p21(WAF1/Cip1).\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "FNL normalization correct---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"WAF1\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"FNL normalization correct\"],\"sentence\":[\"HDAC4 interacts with Sp1\",\"binds and reduces histone H3 acetylation at the Sp1/Sp3 binding site-rich p21(WAF1/Cip1) proximal promoter\",\"suggesting a key role for Sp1 in HDAC4-mediated repression of p21(WAF1/Cip1).\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"CDKN1A\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"HDAC4 interacts with Sp1\",\"binds and reduces histone H3 acetylation at the Sp1/Sp3 binding site-rich p21(WAF1/Cip1) proximal promoter\",\"suggesting a key role for Sp1 in HDAC4-mediated repression of p21(WAF1/Cip1).\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"WAF1\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"HDAC4 interacts with Sp1, binds and reduces histone H3 acetylation at the Sp1/Sp3 binding site-rich p21(WAF1/Cip1) proximal promoter, suggesting a key role for Sp1 in HDAC4-mediated repression of p21(WAF1/Cip1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "named TG not detected---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"named TG not detected\"],\"sentence\":[\"By dissecting its promoters with progressive deletion analysis\",\"we identified the sequence between -106 and -49 in the alpha proximal promoter as the minimal v-Src-responsive element\",\"which contains E- and GC-boxes bound by USF1 and Sp1/Sp3\",\"respectively.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP3\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By dissecting its promoters with progressive deletion analysis, we identified the sequence between -106 and -49 in the alpha proximal promoter as the minimal v-Src-responsive element, which contains E- and GC-boxes bound by USF1 and Sp1/Sp3, respectively.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "named TG not detected---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"named TG not detected\"],\"sentence\":[\"By dissecting its promoters with progressive deletion analysis\",\"we identified the sequence between -106 and -49 in the alpha proximal promoter as the minimal v-Src-responsive element\",\"which contains E- and GC-boxes bound by USF1 and Sp1/Sp3\",\"respectively.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP3\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By dissecting its promoters with progressive deletion analysis, we identified the sequence between -106 and -49 in the alpha proximal promoter as the minimal v-Src-responsive element, which contains E- and GC-boxes bound by USF1 and Sp1/Sp3, respectively.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 291 (char 290)\n",
      "{\"tf associated gene name\":[\"STAT5B\"],\"tg associated gene name\":[\"EPO\"],\"tf text\":[\"STAT5B\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In contrast\",\"Epo stimulation activated both STAT5A and STAT5B.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5B\"],\"tg (associated gene name)\":[\"EPO\"],\"tf text\":[\"STAT5B\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast, Epo stimulation activated both STAT5A and STAT5B.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"STAT5B\"],\"tg associated gene name\":[\"EPO\"],\"tf text\":[\"STAT5B\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In contrast\",\"Epo stimulation activated both STAT5A and STAT5B.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5B\"],\"tg (associated gene name)\":[\"EPO\"],\"tf text\":[\"STAT5B\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast, Epo stimulation activated both STAT5A and STAT5B.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 554 (char 553)\n",
      "{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"NR1I2\"],\"tf text\":[],\"tg text\":[\"PXR\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"To delineate the mechanisms underlying PXR-mediated suppression of bile acid biosynthesis\",\"we examined the functional cross-talk between human PXR and HNF-4\",\"a key hepatic activator of genes involved in bile acid biosynthesis including the cholesterol 7-alpha hydroxylase (CYP7A1) and sterol 12-alpha hydroxylase (CYP8B1) genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HNF-4---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"NR1I2\"],\"tf text\":[],\"tg text\":[\"PXR\"],\"consensus\":[\"N\"],\"martin_comments\":[\"HNF-4\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To delineate the mechanisms underlying PXR-mediated suppression of bile acid biosynthesis, we examined the functional cross-talk between human PXR and HNF-4, a key hepatic activator of genes involved in bile acid biosynthesis including the cholesterol 7-alpha hydroxylase (CYP7A1) and sterol 12-alpha hydroxylase (CYP8B1) genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"NR1I2\"],\"tf text\":[],\"tg text\":[\"PXR\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"To delineate the mechanisms underlying PXR-mediated suppression of bile acid biosynthesis\",\"we examined the functional cross-talk between human PXR and HNF-4\",\"a key hepatic activator of genes involved in bile acid biosynthesis including the cholesterol 7-alpha hydroxylase (CYP7A1) and sterol 12-alpha hydroxylase (CYP8B1) genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HNF-4---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"NR1I2\"],\"tf text\":[],\"tg text\":[\"PXR\"],\"consensus\":[\"N\"],\"martin_comments\":[\"HNF-4\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To delineate the mechanisms underlying PXR-mediated suppression of bile acid biosynthesis, we examined the functional cross-talk between human PXR and HNF-4, a key hepatic activator of genes involved in bile acid biosynthesis including the cholesterol 7-alpha hydroxylase (CYP7A1) and sterol 12-alpha hydroxylase (CYP8B1) genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 457 (char 456)\n",
      "{\"tf associated gene name\":[\"HOXA10\"],\"tg associated gene name\":[\"PBX1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of HOXA10 increased the expression of osteoblast-related genes\",\"osteoblast differentiation and mineralization; expression of PBX1 impaired osteogenic commitment of pluripotent cells and the differentiation of osteoblasts.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HOXA10\"],\"tg (associated gene name)\":[\"PBX1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of HOXA10 increased the expression of osteoblast-related genes, osteoblast differentiation and mineralization; expression of PBX1 impaired osteogenic commitment of pluripotent cells and the differentiation of osteoblasts.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"HOXA10\"],\"tg associated gene name\":[\"PBX1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of HOXA10 increased the expression of osteoblast-related genes\",\"osteoblast differentiation and mineralization; expression of PBX1 impaired osteogenic commitment of pluripotent cells and the differentiation of osteoblasts.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HOXA10\"],\"tg (associated gene name)\":[\"PBX1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of HOXA10 increased the expression of osteoblast-related genes, osteoblast differentiation and mineralization; expression of PBX1 impaired osteogenic commitment of pluripotent cells and the differentiation of osteoblasts.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG---{\"tf associated gene name\":[\"SIX6\"],\"tg associated gene name\":[\"RHD\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG\"],\"sentence\":[\"We show that expression of Six6 is dramatically increased during GnRH neuronal maturation and that overexpression of Six6 induces GnRH transcription in neuronal cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong tg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SIX6\"],\"tg (associated gene name)\":[\"RHD\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong tg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that expression of Six6 is dramatically increased during GnRH neuronal maturation and that overexpression of Six6 induces GnRH transcription in neuronal cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG---{\"tf associated gene name\":[\"SIX6\"],\"tg associated gene name\":[\"RHD\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG\"],\"sentence\":[\"We show that expression of Six6 is dramatically increased during GnRH neuronal maturation and that overexpression of Six6 induces GnRH transcription in neuronal cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong tg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SIX6\"],\"tg (associated gene name)\":[\"RHD\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong tg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that expression of Six6 is dramatically increased during GnRH neuronal maturation and that overexpression of Six6 induces GnRH transcription in neuronal cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 229 (char 228)\n",
      "{\"transcription factor (associated gene name)\":[\"HNF1A\"],\"target gene (associated gene name)\":[\"NR2C1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|TG wrong---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF1A\"],\"tg (associated gene name)\":[\"NR2C1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG wrong\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transactivation of the rat CYP2C13 gene promoter involves HNF-1, HNF-3, and members of the orphan receptor subfamily.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"HNF1A\"],\"tg associated gene name\":[\"NR2C1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Transactivation of the rat CYP2C13 gene promoter involves HNF-1\",\"HNF-3\",\"and members of the orphan receptor subfamily.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"HNF1A\"],\"target gene (associated gene name)\":[\"NR2C1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|TG wrong---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF1A\"],\"tg (associated gene name)\":[\"NR2C1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG wrong\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transactivation of the rat CYP2C13 gene promoter involves HNF-1, HNF-3, and members of the orphan receptor subfamily.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no named TG---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"SLU7\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no named TG\"],\"sentence\":[\"This complex formation between Stat5 and the glucocorticoid receptor diminishes the glucocorticoid response of a GRE-containing promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no named TG---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"SLU7\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no named TG\"],\"sentence\":[\"This complex formation between Stat5 and the glucocorticoid receptor diminishes the glucocorticoid response of a GRE-containing promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"This complex formation between Stat5 and the glucocorticoid receptor diminishes the glucocorticoid response of a GRE-containing promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CNS1 upstram of TF JUN---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"GPS1\"],\"tf text\":[],\"tg text\":[\"CSN1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"CNS1 upstram of TF JUN\"],\"sentence\":[\"Here we show that CSN1 can inhibit phosphorylation of proto-oncogene c-Jun product and repress c-Jun dependent transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "CNS1 upstram of TF JUN---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"GPS1\"],\"tf text\":[],\"tg text\":[\"CSN1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"CNS1 upstram of TF JUN\"],\"sentence\":[\"Here we show that CSN1 can inhibit phosphorylation of proto-oncogene c-Jun product and repress c-Jun dependent transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"GPS1\"],\"tf text\":[],\"tg text\":[\"CSN1\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we show that CSN1 can inhibit phosphorylation of proto-oncogene c-Jun product and repress c-Jun dependent transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "VEGF possible TG|bu unclear---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"KRAS\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"VEGF possible TG\",\"bu unclear\"],\"sentence\":[\"We propose that Nox1 mediates oncogenic Ras-induced upregulation of VEGF and angiogenesis by activating Sp1 through Ras-ERK-dependent phosphorylation of Sp1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "VEGF possible TG|bu unclear---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"KRAS\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"VEGF possible TG\",\"bu unclear\"],\"sentence\":[\"We propose that Nox1 mediates oncogenic Ras-induced upregulation of VEGF and angiogenesis by activating Sp1 through Ras-ERK-dependent phosphorylation of Sp1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"KRAS\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"We propose that Nox1 mediates oncogenic Ras-induced upregulation of VEGF and angiogenesis by activating Sp1 through Ras-ERK-dependent phosphorylation of Sp1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 407 (char 406)\n",
      "{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"RELA\"],\"tf text\":[\"estrogen receptor\"],\"tg text\":[\"NF-kappaB p65\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Promoter activity induced by transfection of both NF-kappaB p65 subunit and NF-IL6 was decreased by 45% by estradiol (E2)-estrogen receptor (ER) complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|promoter undefined---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"RELA\"],\"tf text\":[\"estrogen receptor\"],\"tg text\":[\"NF-kappaB p65\"],\"consensus\":[\"N\"],\"martin_comments\":[\"promoter undefined\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Promoter activity induced by transfection of both NF-kappaB p65 subunit and NF-IL6 was decreased by 45% by estradiol (E2)-estrogen receptor (ER) complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"RELA\"],\"tf text\":[\"estrogen receptor\"],\"tg text\":[\"NF-kappaB p65\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Promoter activity induced by transfection of both NF-kappaB p65 subunit and NF-IL6 was decreased by 45% by estradiol (E2)-estrogen receptor (ER) complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|promoter undefined---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"RELA\"],\"tf text\":[\"estrogen receptor\"],\"tg text\":[\"NF-kappaB p65\"],\"consensus\":[\"N\"],\"martin_comments\":[\"promoter undefined\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Promoter activity induced by transfection of both NF-kappaB p65 subunit and NF-IL6 was decreased by 45% by estradiol (E2)-estrogen receptor (ER) complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "FNL-normalization TG correct---{\"tf associated gene name\":[\"RXRA\"],\"tg associated gene name\":[\"TXNIP\"],\"tf text\":[\"retinoid X receptor alpha\"],\"tg text\":[\"VDUP-1\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"FNL-normalization TG correct\"],\"sentence\":[\"The interference of PPARalpha/retinoid X receptor alpha with the AP-1 transcription factor elements c-Jun/c-Fos resulted in the inhibition of AP-1 binding and down-regulation of the VDUP-1 gene expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "FNL-normalization TG correct---{\"tf associated gene name\":[\"RXRA\"],\"tg associated gene name\":[\"TXNIP\"],\"tf text\":[\"retinoid X receptor alpha\"],\"tg text\":[\"VDUP-1\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"FNL-normalization TG correct\"],\"sentence\":[\"The interference of PPARalpha/retinoid X receptor alpha with the AP-1 transcription factor elements c-Jun/c-Fos resulted in the inhibition of AP-1 binding and down-regulation of the VDUP-1 gene expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RXRA\"],\"target gene (associated gene name)\":[\"TXNIP\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The interference of PPARalpha/retinoid X receptor alpha with the AP-1 transcription factor elements c-Jun/c-Fos resulted in the inhibition of AP-1 binding and down-regulation of the VDUP-1 gene expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|should be AP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RXRA\"],\"tg (associated gene name)\":[\"TXNIP\"],\"tf text\":[\"retinoid X receptor alpha\"],\"tg text\":[\"VDUP-1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"should be AP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The interference of PPARalpha/retinoid X receptor alpha with the AP-1 transcription factor elements c-Jun/c-Fos resulted in the inhibition of AP-1 binding and down-regulation of the VDUP-1 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "FNL correct TF normalization---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[\"E47\"],\"tg text\":[\"insulin\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"FNL correct TF normalization\"],\"sentence\":[\"Relative contribution of PDX-1\",\"MafA and E47/beta2 to the regulation of the human insulin promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "FNL correct TF normalization---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[\"E47\"],\"tg text\":[\"insulin\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"FNL correct TF normalization\"],\"sentence\":[\"Relative contribution of PDX-1\",\"MafA and E47/beta2 to the regulation of the human insulin promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TCF3\"],\"target gene (associated gene name)\":[\"INS\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Relative contribution of PDX-1\",\"MafA and E47/beta2 to the regulation of the human insulin promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"E47\"],\"tg text\":[\"insulin\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Relative contribution of PDX-1, MafA and E47/beta2 to the regulation of the human insulin promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CBFbeta-SMMHC can possibly be normalized to CEBZ|however more likely CBF which is not DbTF ---{\"tf associated gene name\":[\"CEBPZ\"],\"tg associated gene name\":[\"CDK4\"],\"tf text\":[\"CBF\"],\"tg text\":[\"cdk4\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"CBFbeta-SMMHC can possibly be normalized to CEBZ\",\"however more likely CBF which is not DbTF \"],\"sentence\":[\"Inhibition of CBF activities by KRAB-AML1-ER or CBFbeta-SMMHC rapidly reduced endogenous cdk4 mRNA levels\",\"even in cells proliferating at or near control rates as a result of exogenous cdk4 expression.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "CBFbeta-SMMHC can possibly be normalized to CEBZ|however more likely CBF which is not DbTF ---{\"tf associated gene name\":[\"CEBPZ\"],\"tg associated gene name\":[\"CDK4\"],\"tf text\":[\"CBF\"],\"tg text\":[\"cdk4\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"CBFbeta-SMMHC can possibly be normalized to CEBZ\",\"however more likely CBF which is not DbTF \"],\"sentence\":[\"Inhibition of CBF activities by KRAB-AML1-ER or CBFbeta-SMMHC rapidly reduced endogenous cdk4 mRNA levels\",\"even in cells proliferating at or near control rates as a result of exogenous cdk4 expression.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CEBPZ\"],\"target gene (associated gene name)\":[\"CDK4\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Inhibition of CBF activities by KRAB-AML1-ER or CBFbeta-SMMHC rapidly reduced endogenous cdk4 mRNA levels\",\"even in cells proliferating at or near control rates as a result of exogenous cdk4 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPZ\"],\"tg (associated gene name)\":[\"CDK4\"],\"tf text\":[\"CBF\"],\"tg text\":[\"cdk4\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of CBF activities by KRAB-AML1-ER or CBFbeta-SMMHC rapidly reduced endogenous cdk4 mRNA levels, even in cells proliferating at or near control rates as a result of exogenous cdk4 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 434 (char 433)\n",
      "{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[\"c-Myc\"],\"tg text\":[\"p21\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"However\",\"mutational analysis of the p21 promoter and the use of a Gal4-Sp1 chimeric protein strongly suggest that c-Myc affects Sp1 transcriptional activity but not the binding of Sp1 to the p21 promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|sp1 is tf---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[\"c-Myc\"],\"tg text\":[\"p21\"],\"consensus\":[\"N\"],\"martin_comments\":[\"sp1 is tf\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, mutational analysis of the p21 promoter and the use of a Gal4-Sp1 chimeric protein strongly suggest that c-Myc affects Sp1 transcriptional activity but not the binding of Sp1 to the p21 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[\"c-Myc\"],\"tg text\":[\"p21\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"However\",\"mutational analysis of the p21 promoter and the use of a Gal4-Sp1 chimeric protein strongly suggest that c-Myc affects Sp1 transcriptional activity but not the binding of Sp1 to the p21 promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|sp1 is tf---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[\"c-Myc\"],\"tg text\":[\"p21\"],\"consensus\":[\"N\"],\"martin_comments\":[\"sp1 is tf\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, mutational analysis of the p21 promoter and the use of a Gal4-Sp1 chimeric protein strongly suggest that c-Myc affects Sp1 transcriptional activity but not the binding of Sp1 to the p21 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no named TG---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"CHKA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no named TG\"],\"sentence\":[\"Furthermore\",\"TLR4-dependent repression of inflammatory response genes by C-K was mediated through the disruption of p65/interferon regulatory factor complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no named TG---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"CHKA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no named TG\"],\"sentence\":[\"Furthermore\",\"TLR4-dependent repression of inflammatory response genes by C-K was mediated through the disruption of p65/interferon regulatory factor complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"CHKA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, TLR4-dependent repression of inflammatory response genes by C-K was mediated through the disruption of p65/interferon regulatory factor complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "found no normalizeable TG---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"SLC6A2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"found no normalizeable TG\"],\"sentence\":[\"Previously\",\"we reported that Ets1 enhanced and Net repressed CTalpha transcription by binding at the Ets binding site (-49/-47) in the Pcyt1a promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|tg?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"SLC6A2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"tg?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Previously, we reported that Ets1 enhanced and Net repressed CTalpha transcription by binding at the Ets binding site (-49/-47) in the Pcyt1a promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "found no normalizeable TG---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"SLC6A2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"found no normalizeable TG\"],\"sentence\":[\"Previously\",\"we reported that Ets1 enhanced and Net repressed CTalpha transcription by binding at the Ets binding site (-49/-47) in the Pcyt1a promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|tg?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"SLC6A2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"tg?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Previously, we reported that Ets1 enhanced and Net repressed CTalpha transcription by binding at the Ets binding site (-49/-47) in the Pcyt1a promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "provided \"ER??\" is correctly normalized---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CYP2C9\"],\"tf text\":[],\"tg text\":[\"CYP2C9\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"provided \\\"ER??\\\" is correctly normalized\"],\"sentence\":[\"Taken together\",\"these results indicate that ER?? and its ligands play an important role in the regulation of CYP2C9 expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "provided \"ER??\" is correctly normalized---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CYP2C9\"],\"tf text\":[],\"tg text\":[\"CYP2C9\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"provided \\\"ER??\\\" is correctly normalized\"],\"sentence\":[\"Taken together\",\"these results indicate that ER?? and its ligands play an important role in the regulation of CYP2C9 expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"CYP2C9\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Taken together\",\"these results indicate that ER and its ligands play an important role in the regulation of CYP2C9 expression.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"CYP2C9\"],\"tf text\":[],\"tg text\":[\"CYP2C9\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, these results indicate that ER and its ligands play an important role in the regulation of CYP2C9 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 595 (char 594)\n",
      "{\"tf associated gene name\":[\"NR5A1\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Tandem copies of regulatory elements that bind early growth response protein 1 (Egr-1) and steroidogenic factor 1 (SF-1) are located in the proximal region of the LHbeta promoter and make essential contributions to its activity as well as mediate responsiveness to GNRH: Located between these tandem elements is a single site capable of binding the homeodomain protein Pitx1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|Lgbeta?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR5A1\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"Lgbeta?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Tandem copies of regulatory elements that bind early growth response protein 1 (Egr-1) and steroidogenic factor 1 (SF-1) are located in the proximal region of the LHbeta promoter and make essential contributions to its activity as well as mediate responsiveness to GNRH: Located between these tandem elements is a single site capable of binding the homeodomain protein Pitx1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR5A1\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Tandem copies of regulatory elements that bind early growth response protein 1 (Egr-1) and steroidogenic factor 1 (SF-1) are located in the proximal region of the LHbeta promoter and make essential contributions to its activity as well as mediate responsiveness to GNRH: Located between these tandem elements is a single site capable of binding the homeodomain protein Pitx1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|Lgbeta?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR5A1\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"Lgbeta?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Tandem copies of regulatory elements that bind early growth response protein 1 (Egr-1) and steroidogenic factor 1 (SF-1) are located in the proximal region of the LHbeta promoter and make essential contributions to its activity as well as mediate responsiveness to GNRH: Located between these tandem elements is a single site capable of binding the homeodomain protein Pitx1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 487 (char 486)\n",
      "{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"CDX2\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"CDX2\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Furthermore\",\"we evaluated OCT-1 binding by electrophoretic mobility shift assay and activation of the CDX2 promoter by co-transfecting a CDX2 promoter/reporter construct with an OCT-1 expression vector in two gastric carcinoma cell lines\",\"GP220 and MKN45.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|OCT1 tf?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"CDX2\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"CDX2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"OCT1 tf?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, we evaluated OCT-1 binding by electrophoretic mobility shift assay and activation of the CDX2 promoter by co-transfecting a CDX2 promoter/reporter construct with an OCT-1 expression vector in two gastric carcinoma cell lines, GP220 and MKN45.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"CDX2\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"CDX2\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Furthermore\",\"we evaluated OCT-1 binding by electrophoretic mobility shift assay and activation of the CDX2 promoter by co-transfecting a CDX2 promoter/reporter construct with an OCT-1 expression vector in two gastric carcinoma cell lines\",\"GP220 and MKN45.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|OCT1 tf?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"CDX2\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"CDX2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"OCT1 tf?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, we evaluated OCT-1 binding by electrophoretic mobility shift assay and activation of the CDX2 promoter by co-transfecting a CDX2 promoter/reporter construct with an OCT-1 expression vector in two gastric carcinoma cell lines, GP220 and MKN45.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 500 (char 499)\n",
      "{\"tf associated gene name\":[\"TBPL1\"],\"tg associated gene name\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"s\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We present evidence that two such loci (stdE and stdF) are involved in Dam-dependent control of Salmonella SPI-1: in a Dam(-) background\",\"deletion of stdE or stdF suppresses SPI-1 repression; in a Dam(+) background\",\"constitutive expression of StdE and/or StdF represses SPI-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SPI-1 = tg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBPL1\"],\"tg (associated gene name)\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SPI-1 = tg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We present evidence that two such loci (stdE and stdF) are involved in Dam-dependent control of Salmonella SPI-1: in a Dam(-) background, deletion of stdE or stdF suppresses SPI-1 repression; in a Dam(+) background, constitutive expression of StdE and/or StdF represses SPI-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TBPL1\"],\"tg associated gene name\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"s\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We present evidence that two such loci (stdE and stdF) are involved in Dam-dependent control of Salmonella SPI-1: in a Dam(-) background\",\"deletion of stdE or stdF suppresses SPI-1 repression; in a Dam(+) background\",\"constitutive expression of StdE and/or StdF represses SPI-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SPI-1 = tg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBPL1\"],\"tg (associated gene name)\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SPI-1 = tg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We present evidence that two such loci (stdE and stdF) are involved in Dam-dependent control of Salmonella SPI-1: in a Dam(-) background, deletion of stdE or stdF suppresses SPI-1 repression; in a Dam(+) background, constitutive expression of StdE and/or StdF represses SPI-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 386 (char 385)\n",
      "{\"tf associated gene name\":[\"TCF23\"],\"tg associated gene name\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Two independent PCAF HAT mutants\",\"which knock out acetyltransferase enzyme activity\",\"activate Tax transactivation to approximately the same level as wild-type PCAF.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|PCAF=TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF23\"],\"tg (associated gene name)\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PCAF=TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Two independent PCAF HAT mutants, which knock out acetyltransferase enzyme activity, activate Tax transactivation to approximately the same level as wild-type PCAF.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TCF23\"],\"tg associated gene name\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Two independent PCAF HAT mutants\",\"which knock out acetyltransferase enzyme activity\",\"activate Tax transactivation to approximately the same level as wild-type PCAF.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|PCAF=TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF23\"],\"tg (associated gene name)\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PCAF=TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Two independent PCAF HAT mutants, which knock out acetyltransferase enzyme activity, activate Tax transactivation to approximately the same level as wild-type PCAF.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 368 (char 367)\n",
      "{\"tf associated gene name\":[\"PBX1\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The interferon regulatory factor-1 (IRF-1) gene is both an immediate-early G1 phase gene and an S phase gene inducible by PRL in rat Nb2 T lymphocytes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|WRONG TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PBX1\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"WRONG TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The interferon regulatory factor-1 (IRF-1) gene is both an immediate-early G1 phase gene and an S phase gene inducible by PRL in rat Nb2 T lymphocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PBX1\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The interferon regulatory factor-1 (IRF-1) gene is both an immediate-early G1 phase gene and an S phase gene inducible by PRL in rat Nb2 T lymphocytes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|WRONG TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PBX1\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"WRONG TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The interferon regulatory factor-1 (IRF-1) gene is both an immediate-early G1 phase gene and an S phase gene inducible by PRL in rat Nb2 T lymphocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 362 (char 361)\n",
      "{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Western blot analysis showed that treatment with ATRA increased C/EBPalpha and beta expression\",\"but decreased that of C/EBPdelta.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|NO TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"NO TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Western blot analysis showed that treatment with ATRA increased C/EBPalpha and beta expression, but decreased that of C/EBPdelta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Western blot analysis showed that treatment with ATRA increased C/EBPalpha and beta expression\",\"but decreased that of C/EBPdelta.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|NO TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"NO TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Western blot analysis showed that treatment with ATRA increased C/EBPalpha and beta expression, but decreased that of C/EBPdelta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 351 (char 350)\n",
      "{\"tf associated gene name\":[\"ELK4\"],\"tg associated gene name\":[\"EGF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Net-b complexes\",\"as well as Sap1 and Elk1\",\"are induced by epidermal growth factor (EGF) stimulation of cells cultured in low serum.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|NO TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELK4\"],\"tg (associated gene name)\":[\"EGF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"NO TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Net-b complexes, as well as Sap1 and Elk1, are induced by epidermal growth factor (EGF) stimulation of cells cultured in low serum.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ELK4\"],\"tg associated gene name\":[\"EGF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Net-b complexes\",\"as well as Sap1 and Elk1\",\"are induced by epidermal growth factor (EGF) stimulation of cells cultured in low serum.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|NO TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELK4\"],\"tg (associated gene name)\":[\"EGF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"NO TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Net-b complexes, as well as Sap1 and Elk1, are induced by epidermal growth factor (EGF) stimulation of cells cultured in low serum.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "translation---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"EIF3E\"],\"tf text\":[],\"tg text\":[\"Int6\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"translation\"],\"sentence\":[\"These findings suggest that eIF3e/Int6 is a tumor suppressor; however\",\"a recent study has reported that a reduction of eIF3e/Int6 expression in breast cancer cells leads to reduced translation of oncogenes\",\"suggesting that eIF3e/Int6 may in fact have an oncogenic role in breast cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "translation---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"EIF3E\"],\"tf text\":[],\"tg text\":[\"Int6\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"translation\"],\"sentence\":[\"These findings suggest that eIF3e/Int6 is a tumor suppressor; however\",\"a recent study has reported that a reduction of eIF3e/Int6 expression in breast cancer cells leads to reduced translation of oncogenes\",\"suggesting that eIF3e/Int6 may in fact have an oncogenic role in breast cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SSRP1\"],\"tg (associated gene name)\":[\"EIF3E\"],\"tf text\":[],\"tg text\":[\"Int6\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"These findings suggest that eIF3e/Int6 is a tumor suppressor; however, a recent study has reported that a reduction of eIF3e/Int6 expression in breast cancer cells leads to reduced translation of oncogenes, suggesting that eIF3e/Int6 may in fact have an oncogenic role in breast cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "target is PKA-response---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"PRKACA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"\\\"target\\\" is PKA-response\"],\"sentence\":[\"In this manuscript we show 1) that the Pit-1beta isoform not only fails to block PKA activation\",\"but is\",\"in fact\",\"a superior mediator of the PKA response; 2) that the PKA response requires intact POU-specific and POU-homeo domains of Pit-1; and 3) that Oct-1\",\"but not thyrotroph embryonic factor\",\"functions as a Pit-1-interacting factor to mediate an optimal PKA response.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|POU1F1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SSRP1\"],\"tg (associated gene name)\":[\"PRKACA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"POU1F1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this manuscript we show 1) that the Pit-1beta isoform not only fails to block PKA activation, but is, in fact, a superior mediator of the PKA response; 2) that the PKA response requires intact POU-specific and POU-homeo domains of Pit-1; and 3) that Oct-1, but not thyrotroph embryonic factor, functions as a Pit-1-interacting factor to mediate an optimal PKA response.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "target is PKA-response---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"PRKACA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"\\\"target\\\" is PKA-response\"],\"sentence\":[\"In this manuscript we show 1) that the Pit-1beta isoform not only fails to block PKA activation\",\"but is\",\"in fact\",\"a superior mediator of the PKA response; 2) that the PKA response requires intact POU-specific and POU-homeo domains of Pit-1; and 3) that Oct-1\",\"but not thyrotroph embryonic factor\",\"functions as a Pit-1-interacting factor to mediate an optimal PKA response.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|POU1F1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SSRP1\"],\"tg (associated gene name)\":[\"PRKACA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"POU1F1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this manuscript we show 1) that the Pit-1beta isoform not only fails to block PKA activation, but is, in fact, a superior mediator of the PKA response; 2) that the PKA response requires intact POU-specific and POU-homeo domains of Pit-1; and 3) that Oct-1, but not thyrotroph embryonic factor, functions as a Pit-1-interacting factor to mediate an optimal PKA response.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 426 (char 425)\n",
      "{\"transcription factor (associated gene name)\":[\"NR0B2\"],\"target gene (associated gene name)\":[\"SLC10A1\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of the farnesoid X receptor-inducible transcriptional repressor small heterodimer partner also suppressed the GR/dexamethasone-activation of the hNTCP promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|TF=FXR?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR0B2\"],\"tg (associated gene name)\":[\"SLC10A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"TF=FXR?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of the farnesoid X receptor-inducible transcriptional repressor small heterodimer partner also suppressed the GR/dexamethasone-activation of the hNTCP promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR0B2\"],\"tg associated gene name\":[\"SLC10A1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of the farnesoid X receptor-inducible transcriptional repressor small heterodimer partner also suppressed the GR/dexamethasone-activation of the hNTCP promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR0B2\"],\"target gene (associated gene name)\":[\"SLC10A1\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of the farnesoid X receptor-inducible transcriptional repressor small heterodimer partner also suppressed the GR/dexamethasone-activation of the hNTCP promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|TF=FXR?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR0B2\"],\"tg (associated gene name)\":[\"SLC10A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"TF=FXR?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of the farnesoid X receptor-inducible transcriptional repressor small heterodimer partner also suppressed the GR/dexamethasone-activation of the hNTCP promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "check what the symbols with a lot of ? mean - maybe correct---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"check what the symbols with a lot of ? mean - maybe correct\"],\"sentence\":[\"In response to LPS\",\"I??B?? is recruited to the IL-1?? promoter forming a complex with the NF-??B subunits RelA/c-Rel required for IL-1?? transcription.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|LPS?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"LPS?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In response to LPS, IB is recruited to the IL-1 promoter forming a complex with the NF-B subunits RelA/c-Rel required for IL-1 transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "check what the symbols with a lot of ? mean - maybe correct---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"check what the symbols with a lot of ? mean - maybe correct\"],\"sentence\":[\"In response to LPS\",\"I??B?? is recruited to the IL-1?? promoter forming a complex with the NF-??B subunits RelA/c-Rel required for IL-1?? transcription.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|LPS?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"LPS?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In response to LPS, IB is recruited to the IL-1 promoter forming a complex with the NF-B subunits RelA/c-Rel required for IL-1 transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is \"erythroid genes\"|not normalizeable---{\"tf associated gene name\":[\"JDP2\"],\"tg associated gene name\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is \\\"erythroid genes\\\"\",\"not normalizeable\"],\"sentence\":[\"Chromatin immunoprecipitation assays in human CD34(+) cells supported a TIF1gamma-dependent recruitment of positive elongation factors to erythroid genes to promote transcription elongation by counteracting Pol II pausing.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is \"erythroid genes\"|not normalizeable---{\"tf associated gene name\":[\"JDP2\"],\"tg associated gene name\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is \\\"erythroid genes\\\"\",\"not normalizeable\"],\"sentence\":[\"Chromatin immunoprecipitation assays in human CD34(+) cells supported a TIF1gamma-dependent recruitment of positive elongation factors to erythroid genes to promote transcription elongation by counteracting Pol II pausing.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JDP2\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Chromatin immunoprecipitation assays in human CD34(+) cells supported a TIF1gamma-dependent recruitment of positive elongation factors to erythroid genes to promote transcription elongation by counteracting Pol II pausing.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 482 (char 481)\n",
      "{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"IL1B\"],\"tf text\":[],\"tg text\":[\"IL-1beta\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Indeed\",\"both NaBu and IL-1beta led to increased recruitment of NF-kappa B p65\",\"C/EBPbeta\",\"and C/EBP delta\",\"and decreased NF-kappa B p50 and C/EBP alpha DNA-binding to the proximal SAA2 promoter\",\"as assessed by chromatin immunoprecipitation assays.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[],\"tg text\":[\"IL-1beta\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Indeed, both NaBu and IL-1beta led to increased recruitment of NF-kappa B p65, C/EBPbeta, and C/EBP delta, and decreased NF-kappa B p50 and C/EBP alpha DNA-binding to the proximal SAA2 promoter, as assessed by chromatin immunoprecipitation assays.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"IL1B\"],\"tf text\":[],\"tg text\":[\"IL-1beta\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Indeed\",\"both NaBu and IL-1beta led to increased recruitment of NF-kappa B p65\",\"C/EBPbeta\",\"and C/EBP delta\",\"and decreased NF-kappa B p50 and C/EBP alpha DNA-binding to the proximal SAA2 promoter\",\"as assessed by chromatin immunoprecipitation assays.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[],\"tg text\":[\"IL-1beta\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Indeed, both NaBu and IL-1beta led to increased recruitment of NF-kappa B p65, C/EBPbeta, and C/EBP delta, and decreased NF-kappa B p50 and C/EBP alpha DNA-binding to the proximal SAA2 promoter, as assessed by chromatin immunoprecipitation assays.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 329 (char 328)\n",
      "{\"tf associated gene name\":[\"ETS2\"],\"tg associated gene name\":[\"PROM1\"],\"tf text\":[\"Ets-2\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Ets-2 and c-Myb act independently in regulating expression of the hematopoietic stem cell antigen CD34.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CD34 TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS2\"],\"tg (associated gene name)\":[\"PROM1\"],\"tf text\":[\"Ets-2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CD34 TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Ets-2 and c-Myb act independently in regulating expression of the hematopoietic stem cell antigen CD34.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ETS2\"],\"tg associated gene name\":[\"PROM1\"],\"tf text\":[\"Ets-2\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Ets-2 and c-Myb act independently in regulating expression of the hematopoietic stem cell antigen CD34.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CD34 TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS2\"],\"tg (associated gene name)\":[\"PROM1\"],\"tf text\":[\"Ets-2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CD34 TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Ets-2 and c-Myb act independently in regulating expression of the hematopoietic stem cell antigen CD34.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 381 (char 380)\n",
      "{\"tf associated gene name\":[\"ETV4\"],\"tg associated gene name\":[\"IL8\"],\"tf text\":[\"PEA3\"],\"tg text\":[\"IL-8\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of ER?? or PEA3 could up-regulate IL-8 promoter activity\",\"mRNA and secretion; silencing of ER?? or PEA3 decreased IL-8 mRNA and secretion.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|hypothesis---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETV4\"],\"tg (associated gene name)\":[\"IL8\"],\"tf text\":[\"PEA3\"],\"tg text\":[\"IL-8\"],\"consensus\":[\"V\"],\"martin_comments\":[\"hypothesis\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of ER or PEA3 could up-regulate IL-8 promoter activity, mRNA and secretion; silencing of ER or PEA3 decreased IL-8 mRNA and secretion.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ETV4\"],\"tg associated gene name\":[\"IL8\"],\"tf text\":[\"PEA3\"],\"tg text\":[\"IL-8\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of ER?? or PEA3 could up-regulate IL-8 promoter activity\",\"mRNA and secretion; silencing of ER?? or PEA3 decreased IL-8 mRNA and secretion.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|hypothesis---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETV4\"],\"tg (associated gene name)\":[\"IL8\"],\"tf text\":[\"PEA3\"],\"tg text\":[\"IL-8\"],\"consensus\":[\"V\"],\"martin_comments\":[\"hypothesis\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of ER or PEA3 could up-regulate IL-8 promoter activity, mRNA and secretion; silencing of ER or PEA3 decreased IL-8 mRNA and secretion.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 481 (char 480)\n",
      "{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[\"signal transducer and activator of transcription 3\"],\"tg text\":[\"estrogen receptor\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In estrogen receptor (ER)-negative MCF-10A cells\",\"recombinant APN promoted lobuloalveolar differentiation by inhibiting oncogenic signal transducer and activator of transcription 3 activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF and TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"ESR1\"],\"tf text\":[\"signal transducer and activator of transcription 3\"],\"tg text\":[\"estrogen receptor\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF and TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In estrogen receptor (ER)-negative MCF-10A cells, recombinant APN promoted lobuloalveolar differentiation by inhibiting oncogenic signal transducer and activator of transcription 3 activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[\"signal transducer and activator of transcription 3\"],\"tg text\":[\"estrogen receptor\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In estrogen receptor (ER)-negative MCF-10A cells\",\"recombinant APN promoted lobuloalveolar differentiation by inhibiting oncogenic signal transducer and activator of transcription 3 activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF and TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"ESR1\"],\"tf text\":[\"signal transducer and activator of transcription 3\"],\"tg text\":[\"estrogen receptor\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF and TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In estrogen receptor (ER)-negative MCF-10A cells, recombinant APN promoted lobuloalveolar differentiation by inhibiting oncogenic signal transducer and activator of transcription 3 activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG most likely not correctly normalized---{\"tf associated gene name\":[\"HSF2\"],\"tg associated gene name\":[\"IL12A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG most likely not correctly normalized\"],\"sentence\":[\"In conclusion\",\"HSF2 affects cellular populations that assist in radial migration and directly regulates the expression of p35\",\"a crucial actor of radial neuronal migration.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG most likely not correctly normalized---{\"tf associated gene name\":[\"HSF2\"],\"tg associated gene name\":[\"IL12A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG most likely not correctly normalized\"],\"sentence\":[\"In conclusion\",\"HSF2 affects cellular populations that assist in radial migration and directly regulates the expression of p35\",\"a crucial actor of radial neuronal migration.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"HSF2\"],\"target gene (associated gene name)\":[\"IL12A\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In conclusion\",\"HSF2 affects cellular populations that assist in radial migration and directly regulates the expression of p35\",\"a crucial actor of radial neuronal migration.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HSF2\"],\"tg (associated gene name)\":[\"IL12A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"In conclusion, HSF2 affects cellular populations that assist in radial migration and directly regulates the expression of p35, a crucial actor of radial neuronal migration.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 599 (char 598)\n",
      "{\"tf associated gene name\":[\"NCOR2\"],\"tg associated gene name\":[\"NCOR2\"],\"tf text\":[\"SMRT\"],\"tg text\":[\"SMRT\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Through direct phosphorylation of the corepressor silencing mediator for retinoic and thyroid hormone receptors (SMRT)\",\"Akt stabilized RAR/SMRT interaction\",\"leading to an increased tethering of SMRT to the RARbeta2 promoter\",\"decreased histone acetylation\",\"down-regulation of the RARbeta2 expression\",\"and impaired cellular differentiation in response to retinoid.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOR2\"],\"tg (associated gene name)\":[\"NCOR2\"],\"tf text\":[\"SMRT\"],\"tg text\":[\"SMRT\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Through direct phosphorylation of the corepressor silencing mediator for retinoic and thyroid hormone receptors (SMRT), Akt stabilized RAR/SMRT interaction, leading to an increased tethering of SMRT to the RARbeta2 promoter, decreased histone acetylation, down-regulation of the RARbeta2 expression, and impaired cellular differentiation in response to retinoid.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NCOR2\"],\"tg associated gene name\":[\"NCOR2\"],\"tf text\":[\"SMRT\"],\"tg text\":[\"SMRT\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Through direct phosphorylation of the corepressor silencing mediator for retinoic and thyroid hormone receptors (SMRT)\",\"Akt stabilized RAR/SMRT interaction\",\"leading to an increased tethering of SMRT to the RARbeta2 promoter\",\"decreased histone acetylation\",\"down-regulation of the RARbeta2 expression\",\"and impaired cellular differentiation in response to retinoid.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOR2\"],\"tg (associated gene name)\":[\"NCOR2\"],\"tf text\":[\"SMRT\"],\"tg text\":[\"SMRT\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Through direct phosphorylation of the corepressor silencing mediator for retinoic and thyroid hormone receptors (SMRT), Akt stabilized RAR/SMRT interaction, leading to an increased tethering of SMRT to the RARbeta2 promoter, decreased histone acetylation, down-regulation of the RARbeta2 expression, and impaired cellular differentiation in response to retinoid.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not identified correctly---{\"tf associated gene name\":[\"ASCL1\"],\"tg associated gene name\":[\"ASCL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not identified correctly\"],\"sentence\":[\"Our results suggest that the demethylase activity of Jmjd3 and its mediator Hes1 for Mash1 promoter binding are both required for Jmjd3 enhanced efficient expression of Mash1 gene in the early stage of RA-induced neuronal differentiation of P19 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF  ---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ASCL1\"],\"tg (associated gene name)\":[\"ASCL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF  \"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that the demethylase activity of Jmjd3 and its mediator Hes1 for Mash1 promoter binding are both required for Jmjd3 enhanced efficient expression of Mash1 gene in the early stage of RA-induced neuronal differentiation of P19 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF not identified correctly---{\"tf associated gene name\":[\"ASCL1\"],\"tg associated gene name\":[\"ASCL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not identified correctly\"],\"sentence\":[\"Our results suggest that the demethylase activity of Jmjd3 and its mediator Hes1 for Mash1 promoter binding are both required for Jmjd3 enhanced efficient expression of Mash1 gene in the early stage of RA-induced neuronal differentiation of P19 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF  ---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ASCL1\"],\"tg (associated gene name)\":[\"ASCL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF  \"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that the demethylase activity of Jmjd3 and its mediator Hes1 for Mash1 promoter binding are both required for Jmjd3 enhanced efficient expression of Mash1 gene in the early stage of RA-induced neuronal differentiation of P19 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 317 (char 316)\n",
      "{\"tf associated gene name\":[\"NR4A2\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Transcriptional cooperation of FGFR1 with Nurr1 was confirmed on isolated Nurr1-binding elements.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A2\"],\"tg (associated gene name)\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional cooperation of FGFR1 with Nurr1 was confirmed on isolated Nurr1-binding elements.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR4A2\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Transcriptional cooperation of FGFR1 with Nurr1 was confirmed on isolated Nurr1-binding elements.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A2\"],\"tg (associated gene name)\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional cooperation of FGFR1 with Nurr1 was confirmed on isolated Nurr1-binding elements.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 328 (char 327)\n",
      "{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IFNA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Thus\",\"IFN-induced miR-21 expression is coregulated by STAT3 and NF-??B at the level of the miR-21 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IFNA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, IFN-induced miR-21 expression is coregulated by STAT3 and NF-B at the level of the miR-21 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IFNA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Thus\",\"IFN-induced miR-21 expression is coregulated by STAT3 and NF-??B at the level of the miR-21 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IFNA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, IFN-induced miR-21 expression is coregulated by STAT3 and NF-B at the level of the miR-21 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 424 (char 423)\n",
      "{\"tf associated gene name\":[\"FOXN1\"],\"tg associated gene name\":[\"IL4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Our study suggests that Mta2/NuRD is involved in modulating IL-4 and IFN-gamma expression in T cell immune responses\",\"and gene expression in non-T cells plays an important role in controlling autoimmunity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXN1\"],\"tg (associated gene name)\":[\"IL4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our study suggests that Mta2/NuRD is involved in modulating IL-4 and IFN-gamma expression in T cell immune responses, and gene expression in non-T cells plays an important role in controlling autoimmunity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOXN1\"],\"tg associated gene name\":[\"IL4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Our study suggests that Mta2/NuRD is involved in modulating IL-4 and IFN-gamma expression in T cell immune responses\",\"and gene expression in non-T cells plays an important role in controlling autoimmunity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXN1\"],\"tg (associated gene name)\":[\"IL4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our study suggests that Mta2/NuRD is involved in modulating IL-4 and IFN-gamma expression in T cell immune responses, and gene expression in non-T cells plays an important role in controlling autoimmunity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "hypothesis---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"GREB1\"],\"tf text\":[],\"tg text\":[\"GREB1\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"hypothesis\"],\"sentence\":[\"We tested whether there was interaction between STAT3 and ER\",\"which could repress GREB1 expression by immunoprecipitation assay.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "hypothesis---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"GREB1\"],\"tf text\":[],\"tg text\":[\"GREB1\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"hypothesis\"],\"sentence\":[\"We tested whether there was interaction between STAT3 and ER\",\"which could repress GREB1 expression by immunoprecipitation assay.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"GREB1\"],\"tf text\":[],\"tg text\":[\"GREB1\"],\"1\":[\"r\"],\"not true\":[],\"comment\":[],\"sentence\":[\"We tested whether there was interaction between STAT3 and ER\",\"which could repress GREB1 expression by immunoprecipitation assay.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"STAT3\"],\"target gene (associated gene name)\":[\"GREB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"We tested whether there was interaction between STAT3 and ER\",\"which could repress GREB1 expression by immunoprecipitation assay.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"4.0125\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"3\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"128\"],\"sentence pair density\":[\"0.0124223602484472\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|hypothesis---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"GREB1\"],\"tf text\":[],\"tg text\":[\"GREB1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"hypothesis\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We tested whether there was interaction between STAT3 and ER, which could repress GREB1 expression by immunoprecipitation assay.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 439 (char 438)\n",
      "{\"tf associated gene name\":[\"ID2\"],\"tg associated gene name\":[\"ID2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These studies demonstrate a role of Id2 in mediating Gfi-1 functions in B-cell and myeloid development and provide a direct link between Gfi-1 and the B-cell transcriptional network by its ability to repress Id2 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID2\"],\"tg (associated gene name)\":[\"ID2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These studies demonstrate a role of Id2 in mediating Gfi-1 functions in B-cell and myeloid development and provide a direct link between Gfi-1 and the B-cell transcriptional network by its ability to repress Id2 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ID2\"],\"tg associated gene name\":[\"ID2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These studies demonstrate a role of Id2 in mediating Gfi-1 functions in B-cell and myeloid development and provide a direct link between Gfi-1 and the B-cell transcriptional network by its ability to repress Id2 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID2\"],\"tg (associated gene name)\":[\"ID2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These studies demonstrate a role of Id2 in mediating Gfi-1 functions in B-cell and myeloid development and provide a direct link between Gfi-1 and the B-cell transcriptional network by its ability to repress Id2 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 316 (char 315)\n",
      "{\"transcription factor (associated gene name)\":[\"MSX2\"],\"target gene (associated gene name)\":[\"PKD1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"FGF2 promotes Msx2 stimulated PC-1 expression via Frs2/MAPK signaling.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MSX2\"],\"tg (associated gene name)\":[\"PKD1\"],\"tf text\":[\"Msx2\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"FGF2 promotes Msx2 stimulated PC-1 expression via Frs2/MAPK signaling.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MSX2\"],\"tg associated gene name\":[\"PKD1\"],\"tf text\":[\"Msx2\"],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"FGF2 promotes Msx2 stimulated PC-1 expression via Frs2/MAPK signaling.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MSX2\"],\"target gene (associated gene name)\":[\"PKD1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"FGF2 promotes Msx2 stimulated PC-1 expression via Frs2/MAPK signaling.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MSX2\"],\"tg (associated gene name)\":[\"PKD1\"],\"tf text\":[\"Msx2\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"FGF2 promotes Msx2 stimulated PC-1 expression via Frs2/MAPK signaling.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is MMP1---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is MMP1\"],\"sentence\":[\"Taken together\",\"these results indicate that p53 down-regulates MMP1 mainly by disrupting the communications between the transactivator AP-1 and the basal transcriptional complex\",\"which are partially mediated by p300.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, these results indicate that p53 down-regulates MMP1 mainly by disrupting the communications between the transactivator AP-1 and the basal transcriptional complex, which are partially mediated by p300.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is MMP1---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is MMP1\"],\"sentence\":[\"Taken together\",\"these results indicate that p53 down-regulates MMP1 mainly by disrupting the communications between the transactivator AP-1 and the basal transcriptional complex\",\"which are partially mediated by p300.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, these results indicate that p53 down-regulates MMP1 mainly by disrupting the communications between the transactivator AP-1 and the basal transcriptional complex, which are partially mediated by p300.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 428 (char 427)\n",
      "{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"PRKAA1\"],\"tf text\":[\"HNF4alpha\"],\"tg text\":[\"AMPK\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These results demonstrate that HNF4alpha is a downstream target of AMPK and raise the possibility that one of the effects of AMPK activation is reduced expression of HNF4alpha target genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF4A\"],\"tg (associated gene name)\":[\"PRKAA1\"],\"tf text\":[\"HNF4alpha\"],\"tg text\":[\"AMPK\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results demonstrate that HNF4alpha is a downstream target of AMPK and raise the possibility that one of the effects of AMPK activation is reduced expression of HNF4alpha target genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"PRKAA1\"],\"tf text\":[\"HNF4alpha\"],\"tg text\":[\"AMPK\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These results demonstrate that HNF4alpha is a downstream target of AMPK and raise the possibility that one of the effects of AMPK activation is reduced expression of HNF4alpha target genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF4A\"],\"tg (associated gene name)\":[\"PRKAA1\"],\"tf text\":[\"HNF4alpha\"],\"tg text\":[\"AMPK\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results demonstrate that HNF4alpha is a downstream target of AMPK and raise the possibility that one of the effects of AMPK activation is reduced expression of HNF4alpha target genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 503 (char 502)\n",
      "{\"tf associated gene name\":[\"GLI1\"],\"tg associated gene name\":[\"BHLHE41\"],\"tf text\":[\"GLI1\"],\"tg text\":[\"BHLHE41\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Consistent with a previous report that BHLHE41 suppresses the MLH1 promoter activity\",\"we found that the activation of GLI1 led to the BHLHE41-dependent suppression of MLH1\",\"and a double knockdown of GLI1 and GLI2 conversely increased the MLH1 protein in PDAC cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|so close---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI1\"],\"tg (associated gene name)\":[\"BHLHE41\"],\"tf text\":[\"GLI1\"],\"tg text\":[\"BHLHE41\"],\"consensus\":[\"N\"],\"martin_comments\":[\"so close\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consistent with a previous report that BHLHE41 suppresses the MLH1 promoter activity, we found that the activation of GLI1 led to the BHLHE41-dependent suppression of MLH1, and a double knockdown of GLI1 and GLI2 conversely increased the MLH1 protein in PDAC cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"GLI1\"],\"tg associated gene name\":[\"BHLHE41\"],\"tf text\":[\"GLI1\"],\"tg text\":[\"BHLHE41\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Consistent with a previous report that BHLHE41 suppresses the MLH1 promoter activity\",\"we found that the activation of GLI1 led to the BHLHE41-dependent suppression of MLH1\",\"and a double knockdown of GLI1 and GLI2 conversely increased the MLH1 protein in PDAC cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|so close---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI1\"],\"tg (associated gene name)\":[\"BHLHE41\"],\"tf text\":[\"GLI1\"],\"tg text\":[\"BHLHE41\"],\"consensus\":[\"N\"],\"martin_comments\":[\"so close\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consistent with a previous report that BHLHE41 suppresses the MLH1 promoter activity, we found that the activation of GLI1 led to the BHLHE41-dependent suppression of MLH1, and a double knockdown of GLI1 and GLI2 conversely increased the MLH1 protein in PDAC cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 395 (char 394)\n",
      "{\"tf associated gene name\":[\"RUNX2\"],\"tg associated gene name\":[\"CEBPG\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We show by mutational analysis that this synergism is mediated through the C/EBP-responsive element in the OC promoter and by a direct interaction between Runx2 and C/EBPbeta.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG BGLAP?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RUNX2\"],\"tg (associated gene name)\":[\"CEBPG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG BGLAP?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show by mutational analysis that this synergism is mediated through the C/EBP-responsive element in the OC promoter and by a direct interaction between Runx2 and C/EBPbeta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RUNX2\"],\"tg associated gene name\":[\"CEBPG\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We show by mutational analysis that this synergism is mediated through the C/EBP-responsive element in the OC promoter and by a direct interaction between Runx2 and C/EBPbeta.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG BGLAP?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RUNX2\"],\"tg (associated gene name)\":[\"CEBPG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG BGLAP?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show by mutational analysis that this synergism is mediated through the C/EBP-responsive element in the OC promoter and by a direct interaction between Runx2 and C/EBPbeta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 399 (char 398)\n",
      "{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"TFF1\"],\"tf text\":[\"PgR\"],\"tg text\":[\"TFF1\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Histone methyltransferase G9a is recruited to both genes and required for estrogen-induced expression of TFF1 but negatively regulates estrogen-induced expression of PgR.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"TFF1\"],\"tf text\":[\"PgR\"],\"tg text\":[\"TFF1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Histone methyltransferase G9a is recruited to both genes and required for estrogen-induced expression of TFF1 but negatively regulates estrogen-induced expression of PgR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"TFF1\"],\"tf text\":[\"PgR\"],\"tg text\":[\"TFF1\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Histone methyltransferase G9a is recruited to both genes and required for estrogen-induced expression of TFF1 but negatively regulates estrogen-induced expression of PgR.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"TFF1\"],\"tf text\":[\"PgR\"],\"tg text\":[\"TFF1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Histone methyltransferase G9a is recruited to both genes and required for estrogen-induced expression of TFF1 but negatively regulates estrogen-induced expression of PgR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 392 (char 391)\n",
      "{\"tf associated gene name\":[\"NKX2-5\"],\"tg associated gene name\":[\"T\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"By chromatin immunoprecipitation\",\"transcriptional enhancer factor-1\",\"myocyte enhancer factor-2\",\"NKX2.5\",\"GATA-4\",\"and Sp factors bind the cardiac troponin T promoter in vivo.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|T? TNNI3!---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX2-5\"],\"tg (associated gene name)\":[\"T\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"T? TNNI3!\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By chromatin immunoprecipitation, transcriptional enhancer factor-1, myocyte enhancer factor-2, NKX2.5, GATA-4, and Sp factors bind the cardiac troponin T promoter in vivo.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NKX2-5\"],\"tg associated gene name\":[\"T\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"By chromatin immunoprecipitation\",\"transcriptional enhancer factor-1\",\"myocyte enhancer factor-2\",\"NKX2.5\",\"GATA-4\",\"and Sp factors bind the cardiac troponin T promoter in vivo.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|T? TNNI3!---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX2-5\"],\"tg (associated gene name)\":[\"T\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"T? TNNI3!\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By chromatin immunoprecipitation, transcriptional enhancer factor-1, myocyte enhancer factor-2, NKX2.5, GATA-4, and Sp factors bind the cardiac troponin T promoter in vivo.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is GnRH---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"RHD\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is GnRH\"],\"sentence\":[\"Thus\",\"the tethering of GR to the GnRH distal nGRE\",\"by virtue of a direct or indirect association with DNA-bound Oct-1\",\"could play a role in hormone-dependent transcriptional repression of the GnRH gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is GnRH---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"RHD\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is GnRH\"],\"sentence\":[\"Thus\",\"the tethering of GR to the GnRH distal nGRE\",\"by virtue of a direct or indirect association with DNA-bound Oct-1\",\"could play a role in hormone-dependent transcriptional repression of the GnRH gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"RHD\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, the tethering of GR to the GnRH distal nGRE, by virtue of a direct or indirect association with DNA-bound Oct-1, could play a role in hormone-dependent transcriptional repression of the GnRH gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG=APC---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LEF1\"],\"tg (associated gene name)\":[\"HDAC1\"],\"tf text\":[\"Lymphoid enhancer factor-1\"],\"tg text\":[\"histone deacetylase 1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=APC\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Lymphoid enhancer factor-1 blocks adenomatous polyposis coli-mediated nuclear export and degradation of beta-catenin. Regulation by histone deacetylase 1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG=APC---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LEF1\"],\"tg (associated gene name)\":[\"HDAC1\"],\"tf text\":[\"Lymphoid enhancer factor-1\"],\"tg text\":[\"histone deacetylase 1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=APC\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Lymphoid enhancer factor-1 blocks adenomatous polyposis coli-mediated nuclear export and degradation of beta-catenin. Regulation by histone deacetylase 1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF probably not named---{\"tf associated gene name\":[\"CREBZF\"],\"tg associated gene name\":[\"SREBF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF probably not named\"],\"sentence\":[\"Furthermore\",\"UDCA treatment repressed T7-induced SREBP-1c\",\"FAS\",\"and ACC protein levels\",\"whereas knockdown of endogenous SMILE gene expression by adenovirus SMILE shRNA (Ad-shSMILE) significantly reversed UDCA-mediated repression of SREBP-1c\",\"FAS\",\"and ACC protein levels.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF probably not named---{\"tf associated gene name\":[\"CREBZF\"],\"tg associated gene name\":[\"SREBF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF probably not named\"],\"sentence\":[\"Furthermore\",\"UDCA treatment repressed T7-induced SREBP-1c\",\"FAS\",\"and ACC protein levels\",\"whereas knockdown of endogenous SMILE gene expression by adenovirus SMILE shRNA (Ad-shSMILE) significantly reversed UDCA-mediated repression of SREBP-1c\",\"FAS\",\"and ACC protein levels.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CREBZF\"],\"target gene (associated gene name)\":[\"SREBF1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"r\"],\"negation\":[],\"sentence\":[\"Furthermore\",\"UDCA treatment repressed T7-induced SREBP-1c\",\"FAS\",\"and ACC protein levels\",\"whereas knockdown of endogenous SMILE gene expression by adenovirus SMILE shRNA (Ad-shSMILE) significantly reversed UDCA-mediated repression of SREBP-1c\",\"FAS\",\"and ACC protein levels.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.346261\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"9\"],\"sentence length\":[\"270\"],\"sentence pair density\":[\"0.0136054421768707\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"valid\":[\"1\"],\"fin+l_ev+lu+tion\":[\"-\"],\"negation\":[],\"tf-fusion\":[],\"sentence\":[\"Furthermore, UDCA treatment repressed T7-induced SREBP-1c, FAS, and ACC protein levels, whereas knockdown of endogenous SMILE gene expression by adenovirus SMILE shRNA (Ad-shSMILE) significantly reversed UDCA-mediated repression of SREBP-1c, FAS, and ACC protein levels.\"],\"qc\":[\"1\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"CREBZF\"],\"target gene (associated gene name)\":[\"SREBF1\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Furthermore\",\"UDCA treatment repressed T7-induced SREBP-1c\",\"FAS\",\"and ACC protein levels\",\"whereas knockdown of endogenous SMILE gene expression by adenovirus SMILE shRNA (Ad-shSMILE) significantly reversed UDCA-mediated repression of SREBP-1c\",\"FAS\",\"and ACC protein levels.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREBZF\"],\"tg (associated gene name)\":[\"SREBF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, UDCA treatment repressed T7-induced SREBP-1c, FAS, and ACC protein levels, whereas knockdown of endogenous SMILE gene expression by adenovirus SMILE shRNA (Ad-shSMILE) significantly reversed UDCA-mediated repression of SREBP-1c, FAS, and ACC protein levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 318 (char 317)\n",
      "{\"transcription factor (associated gene name)\":[\"HIF1A\"],\"target gene (associated gene name)\":[\"ADM2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"As shown by these results\",\"HIF-1alpha enhances IMD promoter activity.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"ADM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As shown by these results, HIF-1alpha enhances IMD promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"ADM2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"As shown by these results\",\"HIF-1alpha enhances IMD promoter activity.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"HIF1A\"],\"target gene (associated gene name)\":[\"ADM2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"As shown by these results\",\"HIF-1alpha enhances IMD promoter activity.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"ADM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As shown by these results, HIF-1alpha enhances IMD promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 400 (char 399)\n",
      "{\"transcription factor (associated gene name)\":[\"EGR2\"],\"target gene (associated gene name)\":[\"GJB1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In agreement with this finding\",\"SOX10 and EGR2 mutants identified in patients with peripheral myelin defects fail to transactivate the Cx32 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[true]}---|indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR2\"],\"tg (associated gene name)\":[\"GJB1\"],\"tf text\":[\"EGR2\"],\"tg text\":[\"Cx32\"],\"consensus\":[\"V\"],\"martin_comments\":[\"indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In agreement with this finding, SOX10 and EGR2 mutants identified in patients with peripheral myelin defects fail to transactivate the Cx32 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"EGR2\"],\"tg associated gene name\":[\"GJB1\"],\"tf text\":[\"EGR2\"],\"tg text\":[\"Cx32\"],\"1\":[\"a\"],\"0\":[],\"negation\":[\"x\"],\"comment\":[],\"sentence\":[\"In agreement with this finding\",\"SOX10 and EGR2 mutants identified in patients with peripheral myelin defects fail to transactivate the Cx32 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"EGR2\"],\"target gene (associated gene name)\":[\"GJB1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In agreement with this finding\",\"SOX10 and EGR2 mutants identified in patients with peripheral myelin defects fail to transactivate the Cx32 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[true]}---|indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR2\"],\"tg (associated gene name)\":[\"GJB1\"],\"tf text\":[\"EGR2\"],\"tg text\":[\"Cx32\"],\"consensus\":[\"V\"],\"martin_comments\":[\"indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In agreement with this finding, SOX10 and EGR2 mutants identified in patients with peripheral myelin defects fail to transactivate the Cx32 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG=GCK---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=GCK\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The transcription factors HIF-1 and HNF-4 and the coactivator p300 are involved in insulin-regulated glucokinase gene expression via the phosphatidylinositol 3-kinase/protein kinase B pathway.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG=GCK---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=GCK\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The transcription factors HIF-1 and HNF-4 and the coactivator p300 are involved in insulin-regulated glucokinase gene expression via the phosphatidylinositol 3-kinase/protein kinase B pathway.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PBX1\"],\"tg (associated gene name)\":[\"IGFBP1\"],\"tf text\":[],\"tg text\":[\"IGFBP1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"When FOXO1A was overexpressed in human endometrial stromal cells, expression of IGFBP1, DCN, and PRL increased, whereas levels of TIMP3 and CNR1 decreased.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PBX1\"],\"tg (associated gene name)\":[\"IGFBP1\"],\"tf text\":[],\"tg text\":[\"IGFBP1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"When FOXO1A was overexpressed in human endometrial stromal cells, expression of IGFBP1, DCN, and PRL increased, whereas levels of TIMP3 and CNR1 decreased.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 341 (char 340)\n",
      "{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"ALB\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of c-Jun specifically repressed the rat AFP promoter but not the albumin promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"ALB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Overexpression of c-Jun specifically repressed the rat AFP promoter but not the albumin promoter.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"ALB\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of c-Jun specifically repressed the rat AFP promoter but not the albumin promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"ALB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Overexpression of c-Jun specifically repressed the rat AFP promoter but not the albumin promoter.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"Analysis of the regulation of YY1 localization in vitro demonstrated that polymerized gamma-actin sequestered EGFP-YY1 in the cytoplasm and that YY1 activation of c-myc promoter activity was inhibited by LIM kinase\",\"which increases actin polymerization.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Analysis of the regulation of YY1 localization in vitro demonstrated that polymerized gamma-actin sequestered EGFP-YY1 in the cytoplasm and that YY1 activation of c-myc promoter activity was inhibited by LIM kinase, which increases actin polymerization.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"Analysis of the regulation of YY1 localization in vitro demonstrated that polymerized gamma-actin sequestered EGFP-YY1 in the cytoplasm and that YY1 activation of c-myc promoter activity was inhibited by LIM kinase\",\"which increases actin polymerization.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Analysis of the regulation of YY1 localization in vitro demonstrated that polymerized gamma-actin sequestered EGFP-YY1 in the cytoplasm and that YY1 activation of c-myc promoter activity was inhibited by LIM kinase, which increases actin polymerization.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 414 (char 413)\n",
      "{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These findings suggest that high glucose in combination with stimulation by LPS plus IFN-gamma enhances iNOS expression\",\"and protein kinase C activation may be playing a role in this enhancement.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF missing---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF missing\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These findings suggest that high glucose in combination with stimulation by LPS plus IFN-gamma enhances iNOS expression, and protein kinase C activation may be playing a role in this enhancement.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These findings suggest that high glucose in combination with stimulation by LPS plus IFN-gamma enhances iNOS expression\",\"and protein kinase C activation may be playing a role in this enhancement.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF missing---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF missing\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These findings suggest that high glucose in combination with stimulation by LPS plus IFN-gamma enhances iNOS expression, and protein kinase C activation may be playing a role in this enhancement.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "chicken c-myc---{\"tf associated gene name\":[\"CTCF\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"chicken c-myc\"],\"sentence\":[\"Removal of 110 bp sequence binding both CTCF and Sp1-like proteins leads to a 4 to 8-fold increase in transcription of stably transfected c-myc fusion constructs in chicken embryonic fibroblasts\",\"suggesting that the CTCF is likely to be one of multiple nuclear factors involved in the transcriptional regulation of the chicken c-myc gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|chicken---{\"_\":[\"_\"],\"tf\":[\"CTCF\"],\"tg\":[\"MYC\"],\"validation\":[\"V\"],\"comment 1\":[\"chicken\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Removal of 110 bp sequence binding both CTCF and Sp1-like proteins leads to a 4 to 8-fold increase in transcription of stably transfected c-myc fusion constructs in chicken embryonic fibroblasts, suggesting that the CTCF is likely to be one of multiple nuclear factors involved in the transcriptional regulation of the chicken c-myc gene.\"],\"interaction score\":[\"2.473278\"],\"pmid counts\":[\"10\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "chicken c-myc---{\"tf associated gene name\":[\"CTCF\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"chicken c-myc\"],\"sentence\":[\"Removal of 110 bp sequence binding both CTCF and Sp1-like proteins leads to a 4 to 8-fold increase in transcription of stably transfected c-myc fusion constructs in chicken embryonic fibroblasts\",\"suggesting that the CTCF is likely to be one of multiple nuclear factors involved in the transcriptional regulation of the chicken c-myc gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|chicken---{\"_\":[\"_\"],\"tf\":[\"CTCF\"],\"tg\":[\"MYC\"],\"validation\":[\"V\"],\"comment 1\":[\"chicken\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Removal of 110 bp sequence binding both CTCF and Sp1-like proteins leads to a 4 to 8-fold increase in transcription of stably transfected c-myc fusion constructs in chicken embryonic fibroblasts, suggesting that the CTCF is likely to be one of multiple nuclear factors involved in the transcriptional regulation of the chicken c-myc gene.\"],\"interaction score\":[\"2.473278\"],\"pmid counts\":[\"10\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CTCF\"],\"target gene (associated gene name)\":[\"MYC\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Removal of 110 bp sequence binding both CTCF and Sp1-like proteins leads to a 4 to 8-fold increase in transcription of stably transfected c-myc fusion constructs in chicken embryonic fibroblasts\",\"suggesting that the CTCF is likely to be one of multiple nuclear factors involved in the transcriptional regulation of the chicken c-myc gene.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CTCF\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Removal of 110 bp sequence binding both CTCF and Sp1-like proteins leads to a 4 to 8-fold increase in transcription of stably transfected c-myc fusion constructs in chicken embryonic fibroblasts, suggesting that the CTCF is likely to be one of multiple nuclear factors involved in the transcriptional regulation of the chicken c-myc gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is miR-346---{\"tf associated gene name\":[\"HNF1A\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[\"TCF-1\"],\"tg text\":[\"CyclinD1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is miR-346\"],\"sentence\":[\"We further found that miR-346 overexpression activated the Wnt/??-catenin pathway and increased the expression of several downstream genes including CyclinD1\",\"c-Myc\",\"TCF-1 and LEF-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is miR-346---{\"tf associated gene name\":[\"HNF1A\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[\"TCF-1\"],\"tg text\":[\"CyclinD1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is miR-346\"],\"sentence\":[\"We further found that miR-346 overexpression activated the Wnt/??-catenin pathway and increased the expression of several downstream genes including CyclinD1\",\"c-Myc\",\"TCF-1 and LEF-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"HNF1A\"],\"target gene (associated gene name)\":[\"CCND1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF1A\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[\"TCF-1\"],\"tg text\":[\"CyclinD1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We further found that miR-346 overexpression activated the Wnt/-catenin pathway and increased the expression of several downstream genes including CyclinD1, c-Myc, TCF-1 and LEF-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"SMAD4\"],\"tg associated gene name\":[\"GDF9\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Therefore\",\"we generated two oocyte-specific conditional knockout models for the common SMAD\",\"Smad4\",\"using cre recombinase expression from either the zona pellucida 3 (Zp3) or Gdf9 promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"SMAD4\"],\"tg associated gene name\":[\"GDF9\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Therefore\",\"we generated two oocyte-specific conditional knockout models for the common SMAD\",\"Smad4\",\"using cre recombinase expression from either the zona pellucida 3 (Zp3) or Gdf9 promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD4\"],\"tg (associated gene name)\":[\"GDF9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Therefore, we generated two oocyte-specific conditional knockout models for the common SMAD, Smad4, using cre recombinase expression from either the zona pellucida 3 (Zp3) or Gdf9 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs are p21(Cip1/Waf1) cyclin-kinase inhibitor (p21 CKI)|GADD45|bax|and cyclin G---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"CHKA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TGs are p21(Cip1/Waf1) cyclin-kinase inhibitor (p21 CKI)\",\"GADD45\",\"bax\",\"and cyclin G\"],\"sentence\":[\"p53 target genes p21(Cip1/Waf1) cyclin-kinase inhibitor (p21 CKI)\",\"GADD45\",\"bax\",\"and cyclin G and genes affecting the redox state of the cells are implicated in p53 damage control responses.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TGs are p21(Cip1/Waf1) cyclin-kinase inhibitor (p21 CKI)|GADD45|bax|and cyclin G---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"CHKA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TGs are p21(Cip1/Waf1) cyclin-kinase inhibitor (p21 CKI)\",\"GADD45\",\"bax\",\"and cyclin G\"],\"sentence\":[\"p53 target genes p21(Cip1/Waf1) cyclin-kinase inhibitor (p21 CKI)\",\"GADD45\",\"bax\",\"and cyclin G and genes affecting the redox state of the cells are implicated in p53 damage control responses.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"CHKA\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"p53 target genes p21(Cip1/Waf1) cyclin-kinase inhibitor (p21 CKI)\",\"GADD45\",\"bax\",\"and cyclin G and genes affecting the redox state of the cells are implicated in p53 damage control responses.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"CHKA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"p53 target genes p21(Cip1/Waf1) cyclin-kinase inhibitor (p21 CKI), GADD45, bax, and cyclin G and genes affecting the redox state of the cells are implicated in p53 damage control responses.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|no TG found---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TG found\"],\"sentence\":[\"HIF-1alpha displaces the transcriptional activator Myc from Sp1 binding to repress MutSalpha expression in a p53-dependent manner; Sp1 serves as a molecular switch by recruiting HIF-1alpha to the gene promoter under hypoxia.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE|no TG found---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TG found\"],\"sentence\":[\"HIF-1alpha displaces the transcriptional activator Myc from Sp1 binding to repress MutSalpha expression in a p53-dependent manner; Sp1 serves as a molecular switch by recruiting HIF-1alpha to the gene promoter under hypoxia.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"MYC\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"HIF-1alpha displaces the transcriptional activator Myc from Sp1 binding to repress MutSalpha expression in a p53-dependent manner; Sp1 serves as a molecular switch by recruiting HIF-1alpha to the gene promoter under hypoxia.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.068552\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"13\"],\"sentence length\":[\"224\"],\"sentence pair density\":[\"0.00578034682080924\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|switched?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"switched?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HIF-1alpha displaces the transcriptional activator Myc from Sp1 binding to repress MutSalpha expression in a p53-dependent manner; Sp1 serves as a molecular switch by recruiting HIF-1alpha to the gene promoter under hypoxia.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is MiR-199a---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"SMAD4\"],\"tf text\":[],\"tg text\":[\"Smad4\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"\\\"TF\\\" is MiR-199a\"],\"sentence\":[\"MiR-199a\",\"one of the identified miRNAs\",\"was inversely correlated with Smad4 expression in various human cancer cell lines and gastric cancer tissues\",\"and repressed Smad4 expression and blocked canonical TGF-?? transcriptional responses in cell lines.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|miRNA based, only normalisation issue?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"SMAD4\"],\"tf text\":[],\"tg text\":[\"Smad4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"miRNA based, only normalisation issue?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MiR-199a, one of the identified miRNAs, was inversely correlated with Smad4 expression in various human cancer cell lines and gastric cancer tissues, and repressed Smad4 expression and blocked canonical TGF- transcriptional responses in cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is MiR-199a---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"SMAD4\"],\"tf text\":[],\"tg text\":[\"Smad4\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"\\\"TF\\\" is MiR-199a\"],\"sentence\":[\"MiR-199a\",\"one of the identified miRNAs\",\"was inversely correlated with Smad4 expression in various human cancer cell lines and gastric cancer tissues\",\"and repressed Smad4 expression and blocked canonical TGF-?? transcriptional responses in cell lines.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|miRNA based, only normalisation issue?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"SMAD4\"],\"tf text\":[],\"tg text\":[\"Smad4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"miRNA based, only normalisation issue?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MiR-199a, one of the identified miRNAs, was inversely correlated with Smad4 expression in various human cancer cell lines and gastric cancer tissues, and repressed Smad4 expression and blocked canonical TGF- transcriptional responses in cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TFs are H-2RIIBP and the natural region II factor|TG is vitellogenin A2---{\"tf associated gene name\":[\"RXRB\"],\"tg associated gene name\":[\"SGCA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TFs are H-2RIIBP and the natural region II factor\",\"TG is vitellogenin A2\"],\"sentence\":[\"Further analyses showed that both H-2RIIBP and the natural region II factor bind to the estrogen response element (ERE) of the vitellogenin A2 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RXRB\"],\"tg (associated gene name)\":[\"SGCA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Further analyses showed that both H-2RIIBP and the natural region II factor bind to the estrogen response element (ERE) of the vitellogenin A2 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TFs are H-2RIIBP and the natural region II factor|TG is vitellogenin A2---{\"tf associated gene name\":[\"RXRB\"],\"tg associated gene name\":[\"SGCA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TFs are H-2RIIBP and the natural region II factor\",\"TG is vitellogenin A2\"],\"sentence\":[\"Further analyses showed that both H-2RIIBP and the natural region II factor bind to the estrogen response element (ERE) of the vitellogenin A2 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RXRB\"],\"tg (associated gene name)\":[\"SGCA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Further analyses showed that both H-2RIIBP and the natural region II factor bind to the estrogen response element (ERE) of the vitellogenin A2 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear sentence|unclear whether GRE/TRE; MMTV promoter (viral)---{\"tf associated gene name\":[\"NCOA1\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[\"CBP\"],\"1\":[],\"negation\":[],\"not true\":[\"unclear\"],\"comment\":[\"unclear sentence\",\"unclear whether GRE/TRE; MMTV promoter (viral)\"],\"sentence\":[\"Although PR is recruited to the MMTV promoter equivalently in the G1 and S phases\",\"recruitment of SRC-1\",\"SRC-3\",\"and\",\"consequently\",\"CBP is reduced in G1 phase despite comparable expression levels of SRC-1 and SRC-3.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "unclear sentence|unclear whether GRE/TRE; MMTV promoter (viral)---{\"tf associated gene name\":[\"NCOA1\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[\"CBP\"],\"1\":[],\"negation\":[],\"not true\":[\"unclear\"],\"comment\":[\"unclear sentence\",\"unclear whether GRE/TRE; MMTV promoter (viral)\"],\"sentence\":[\"Although PR is recruited to the MMTV promoter equivalently in the G1 and S phases\",\"recruitment of SRC-1\",\"SRC-3\",\"and\",\"consequently\",\"CBP is reduced in G1 phase despite comparable expression levels of SRC-1 and SRC-3.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA1\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[\"CBP\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although PR is recruited to the MMTV promoter equivalently in the G1 and S phases, recruitment of SRC-1, SRC-3, and, consequently, CBP is reduced in G1 phase despite comparable expression levels of SRC-1 and SRC-3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "switched?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"PRMT2\"],\"tf text\":[\"E2F1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"switched?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In addition, PRMT2 silencing could enhance 17??-estradiol-induced proliferation by regulating E2F1 expression and E2F1-responsive genes in ER??-positive breast cancer cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "switched?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"PRMT2\"],\"tf text\":[\"E2F1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"switched?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In addition, PRMT2 silencing could enhance 17??-estradiol-induced proliferation by regulating E2F1 expression and E2F1-responsive genes in ER??-positive breast cancer cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 422 (char 421)\n",
      "{\"tf associated gene name\":[\"OVOL1\"],\"tg associated gene name\":[\"OVOL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"We present molecular evidence that c-myc expression is up-regulated in Ovol1-deficient suprabasal cells and that Ovol1 represses c-myc transcription by directly binding to its promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF=c-myc---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"OVOL1\"],\"tg (associated gene name)\":[\"OVOL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=c-myc\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We present molecular evidence that c-myc expression is up-regulated in Ovol1-deficient suprabasal cells and that Ovol1 represses c-myc transcription by directly binding to its promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"OVOL1\"],\"tg associated gene name\":[\"OVOL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"We present molecular evidence that c-myc expression is up-regulated in Ovol1-deficient suprabasal cells and that Ovol1 represses c-myc transcription by directly binding to its promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF=c-myc---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"OVOL1\"],\"tg (associated gene name)\":[\"OVOL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=c-myc\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We present molecular evidence that c-myc expression is up-regulated in Ovol1-deficient suprabasal cells and that Ovol1 represses c-myc transcription by directly binding to its promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 377 (char 376)\n",
      "{\"transcription factor (associated gene name)\":[\"SMAD3\"],\"target gene (associated gene name)\":[\"TNFSF13B\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Furthermore\",\"overexpression of DN-Smad3 abolished activin-induced BAFF expression at the transcriptional and protein levels.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|to indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD3\"],\"tg (associated gene name)\":[\"TNFSF13B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"to indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, overexpression of DN-Smad3 abolished activin-induced BAFF expression at the transcriptional and protein levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SMAD3\"],\"tg associated gene name\":[\"TNFSF13B\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Furthermore\",\"overexpression of DN-Smad3 abolished activin-induced BAFF expression at the transcriptional and protein levels.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SMAD3\"],\"target gene (associated gene name)\":[\"TNFSF13B\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Furthermore\",\"overexpression of DN-Smad3 abolished activin-induced BAFF expression at the transcriptional and protein levels.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|to indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD3\"],\"tg (associated gene name)\":[\"TNFSF13B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"to indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, overexpression of DN-Smad3 abolished activin-induced BAFF expression at the transcriptional and protein levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 378 (char 377)\n",
      "{\"transcription factor (associated gene name)\":[\"CEBPB\"],\"target gene (associated gene name)\":[\"RCAN1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of C/EBPbeta increased activity of both the endogenous mouse Rcan1-4 gene and a human RCAN1-4 luciferase reporter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|to indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"RCAN1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"to indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of C/EBPbeta increased activity of both the endogenous mouse Rcan1-4 gene and a human RCAN1-4 luciferase reporter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"RCAN1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of C/EBPbeta increased activity of both the endogenous mouse Rcan1-4 gene and a human RCAN1-4 luciferase reporter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CEBPB\"],\"target gene (associated gene name)\":[\"RCAN1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of C/EBPbeta increased activity of both the endogenous mouse Rcan1-4 gene and a human RCAN1-4 luciferase reporter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|to indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"RCAN1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"to indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of C/EBPbeta increased activity of both the endogenous mouse Rcan1-4 gene and a human RCAN1-4 luciferase reporter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not named|TG most likely wrongly normalized---{\"tf associated gene name\":[\"ZNF699\"],\"tg associated gene name\":[\"SMARCA4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF not named\",\"TG most likely wrongly normalized\"],\"sentence\":[\"Using HSP12\",\"HSP82 and SSA4 gene promoters as reporters\",\"we demonstrated that while inactivation of SNF2\",\"a critical ATPase of the SWI/SNF complex\",\"primarily affects the HSP12 promoter\",\"depletion of STH1- a SNF2 homolog from the RSC complex reduces histone displacement and abolishes the Pol II recruitment at all three promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|ZNF699?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF699\"],\"tg (associated gene name)\":[\"SMARCA4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"ZNF699?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using HSP12, HSP82 and SSA4 gene promoters as reporters, we demonstrated that while inactivation of SNF2, a critical ATPase of the SWI/SNF complex, primarily affects the HSP12 promoter, depletion of STH1- a SNF2 homolog from the RSC complex reduces histone displacement and abolishes the Pol II recruitment at all three promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF not named|TG most likely wrongly normalized---{\"tf associated gene name\":[\"ZNF699\"],\"tg associated gene name\":[\"SMARCA4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF not named\",\"TG most likely wrongly normalized\"],\"sentence\":[\"Using HSP12\",\"HSP82 and SSA4 gene promoters as reporters\",\"we demonstrated that while inactivation of SNF2\",\"a critical ATPase of the SWI/SNF complex\",\"primarily affects the HSP12 promoter\",\"depletion of STH1- a SNF2 homolog from the RSC complex reduces histone displacement and abolishes the Pol II recruitment at all three promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|ZNF699?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF699\"],\"tg (associated gene name)\":[\"SMARCA4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"ZNF699?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using HSP12, HSP82 and SSA4 gene promoters as reporters, we demonstrated that while inactivation of SNF2, a critical ATPase of the SWI/SNF complex, primarily affects the HSP12 promoter, depletion of STH1- a SNF2 homolog from the RSC complex reduces histone displacement and abolishes the Pol II recruitment at all three promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is NANOG---{\"tf associated gene name\":[\"FOXJ1\"],\"tg associated gene name\":[\"CAV1\"],\"tf text\":[\"FOXJ1\"],\"tg text\":[\"caveolin-1\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"TF is NANOG\"],\"sentence\":[\"Conversely\",\"ectopic NANOG overexpression enhanced ovarian cancer cell migration and invasion along with decreased E-cadherin\",\"caveolin-1\",\"FOXO1\",\"FOXO3a\",\"FOXJ1 and FOXB1 mRNA expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|if any TF: nanog---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXJ1\"],\"tg (associated gene name)\":[\"CAV1\"],\"tf text\":[\"FOXJ1\"],\"tg text\":[\"caveolin-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"if any TF: nanog\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Conversely, ectopic NANOG overexpression enhanced ovarian cancer cell migration and invasion along with decreased E-cadherin, caveolin-1, FOXO1, FOXO3a, FOXJ1 and FOXB1 mRNA expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is NANOG---{\"tf associated gene name\":[\"FOXJ1\"],\"tg associated gene name\":[\"CAV1\"],\"tf text\":[\"FOXJ1\"],\"tg text\":[\"caveolin-1\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"TF is NANOG\"],\"sentence\":[\"Conversely\",\"ectopic NANOG overexpression enhanced ovarian cancer cell migration and invasion along with decreased E-cadherin\",\"caveolin-1\",\"FOXO1\",\"FOXO3a\",\"FOXJ1 and FOXB1 mRNA expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|if any TF: nanog---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXJ1\"],\"tg (associated gene name)\":[\"CAV1\"],\"tf text\":[\"FOXJ1\"],\"tg text\":[\"caveolin-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"if any TF: nanog\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Conversely, ectopic NANOG overexpression enhanced ovarian cancer cell migration and invasion along with decreased E-cadherin, caveolin-1, FOXO1, FOXO3a, FOXJ1 and FOXB1 mRNA expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 369 (char 368)\n",
      "{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"B3GNT9\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Analysis of DNA-protein interactions by EMSA revealed that both ERalpha and TR (alpha1 and beta1) could bind to the EREs in the rat PPE gene promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|PPE?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"B3GNT9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PPE?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Analysis of DNA-protein interactions by EMSA revealed that both ERalpha and TR (alpha1 and beta1) could bind to the EREs in the rat PPE gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"B3GNT9\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Analysis of DNA-protein interactions by EMSA revealed that both ERalpha and TR (alpha1 and beta1) could bind to the EREs in the rat PPE gene promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|PPE?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"B3GNT9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PPE?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Analysis of DNA-protein interactions by EMSA revealed that both ERalpha and TR (alpha1 and beta1) could bind to the EREs in the rat PPE gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is thyrotropin|unclear whether TF correctly normalized---{\"tf associated gene name\":[\"POU3F3\"],\"tg associated gene name\":[\"MT-ND1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is thyrotropin\",\"unclear whether TF correctly normalized\"],\"sentence\":[\"Thyrotropin-releasing hormone-stimulated thyrotropin expression involves islet-brain-1/c-Jun N-terminal kinase interacting protein-1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|THYROTROPIN?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU3F3\"],\"tg (associated gene name)\":[\"MT-ND1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"THYROTROPIN?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thyrotropin-releasing hormone-stimulated thyrotropin expression involves islet-brain-1/c-Jun N-terminal kinase interacting protein-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is thyrotropin|unclear whether TF correctly normalized---{\"tf associated gene name\":[\"POU3F3\"],\"tg associated gene name\":[\"MT-ND1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is thyrotropin\",\"unclear whether TF correctly normalized\"],\"sentence\":[\"Thyrotropin-releasing hormone-stimulated thyrotropin expression involves islet-brain-1/c-Jun N-terminal kinase interacting protein-1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|THYROTROPIN?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU3F3\"],\"tg (associated gene name)\":[\"MT-ND1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"THYROTROPIN?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thyrotropin-releasing hormone-stimulated thyrotropin expression involves islet-brain-1/c-Jun N-terminal kinase interacting protein-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 412 (char 411)\n",
      "{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"BRCA1\"],\"tf text\":[\"p53\"],\"tg text\":[\"BRCA1\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Moreover\",\"p53-mediated stimulation of transcription by BRCA1 was completely abrogated by either a dominant-negative mutant of BRG1 or the cancer-causing deletion in exon 11 of BRCA1.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|COOPERATION BETWEEN TP53 AND BRCA1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"BRCA1\"],\"tf text\":[\"p53\"],\"tg text\":[\"BRCA1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"COOPERATION BETWEEN TP53 AND BRCA1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, p53-mediated stimulation of transcription by BRCA1 was completely abrogated by either a dominant-negative mutant of BRG1 or the cancer-causing deletion in exon 11 of BRCA1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"BRCA1\"],\"tf text\":[\"p53\"],\"tg text\":[\"BRCA1\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Moreover\",\"p53-mediated stimulation of transcription by BRCA1 was completely abrogated by either a dominant-negative mutant of BRG1 or the cancer-causing deletion in exon 11 of BRCA1.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|COOPERATION BETWEEN TP53 AND BRCA1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"BRCA1\"],\"tf text\":[\"p53\"],\"tg text\":[\"BRCA1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"COOPERATION BETWEEN TP53 AND BRCA1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, p53-mediated stimulation of transcription by BRCA1 was completely abrogated by either a dominant-negative mutant of BRG1 or the cancer-causing deletion in exon 11 of BRCA1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 453 (char 452)\n",
      "{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"MYD88\"],\"tf text\":[],\"tg text\":[\"MyD88\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"In transient cotransfection experiments\",\"dominant negative mutants of the IL-1 signaling pathway\",\"including MyD88\",\"IRAK\",\"IRAK2\",\"and TRAF6 inhibited both IL-1- and LPS-induced NF-kappaB-luciferase activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"MYD88\"],\"tf text\":[],\"tg text\":[\"MyD88\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In transient cotransfection experiments, dominant negative mutants of the IL-1 signaling pathway, including MyD88, IRAK, IRAK2, and TRAF6 inhibited both IL-1- and LPS-induced NF-kappaB-luciferase activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"MYD88\"],\"tf text\":[],\"tg text\":[\"MyD88\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"In transient cotransfection experiments\",\"dominant negative mutants of the IL-1 signaling pathway\",\"including MyD88\",\"IRAK\",\"IRAK2\",\"and TRAF6 inhibited both IL-1- and LPS-induced NF-kappaB-luciferase activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"MYD88\"],\"tf text\":[],\"tg text\":[\"MyD88\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In transient cotransfection experiments, dominant negative mutants of the IL-1 signaling pathway, including MyD88, IRAK, IRAK2, and TRAF6 inhibited both IL-1- and LPS-induced NF-kappaB-luciferase activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 493 (char 492)\n",
      "{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"LPIN1\"],\"tf text\":[],\"tg text\":[\"lipin1\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The results of the present study show that lipin1 activates PPAR?? by releasing co-repressors\",\"NCoR1 (nuclear receptor co-repressor 1) and SMRT (silencing mediator of retinoid and thyroid hormone receptor)\",\"from PPAR?? in the absence of the ligand rosiglitazone.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"LPIN1\"],\"tf text\":[],\"tg text\":[\"lipin1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results of the present study show that lipin1 activates PPAR by releasing co-repressors, NCoR1 (nuclear receptor co-repressor 1) and SMRT (silencing mediator of retinoid and thyroid hormone receptor), from PPAR in the absence of the ligand rosiglitazone.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"LPIN1\"],\"tf text\":[],\"tg text\":[\"lipin1\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The results of the present study show that lipin1 activates PPAR?? by releasing co-repressors\",\"NCoR1 (nuclear receptor co-repressor 1) and SMRT (silencing mediator of retinoid and thyroid hormone receptor)\",\"from PPAR?? in the absence of the ligand rosiglitazone.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"LPIN1\"],\"tf text\":[],\"tg text\":[\"lipin1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results of the present study show that lipin1 activates PPAR by releasing co-repressors, NCoR1 (nuclear receptor co-repressor 1) and SMRT (silencing mediator of retinoid and thyroid hormone receptor), from PPAR in the absence of the ligand rosiglitazone.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Sp1 but not Sp3 protein level and increased Sp1-motif binding activity were correlated with ras activation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Sp1 but not Sp3 protein level and increased Sp1-motif binding activity were correlated with ras activation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is BDNF---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"ERVW-1\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is BDNF\"],\"sentence\":[\"Using gene knockdown\",\"we found that CREB is required for the expression of BDNF that is regulated by env.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"ERVW-1\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using gene knockdown, we found that CREB is required for the expression of BDNF that is regulated by env.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is BDNF---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"ERVW-1\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is BDNF\"],\"sentence\":[\"Using gene knockdown\",\"we found that CREB is required for the expression of BDNF that is regulated by env.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"ERVW-1\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using gene knockdown, we found that CREB is required for the expression of BDNF that is regulated by env.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 381 (char 380)\n",
      "{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"TP53BP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"p202 modulates the pattern of gene expression by inhibiting the activity of various transcription factors including NF-kappaB\",\"c-Fos\",\"c-Jun\",\"E2F-1\",\"and p53.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"TP53BP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"p202 modulates the pattern of gene expression by inhibiting the activity of various transcription factors including NF-kappaB, c-Fos, c-Jun, E2F-1, and p53.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"TP53BP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"p202 modulates the pattern of gene expression by inhibiting the activity of various transcription factors including NF-kappaB\",\"c-Fos\",\"c-Jun\",\"E2F-1\",\"and p53.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"TP53BP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"p202 modulates the pattern of gene expression by inhibiting the activity of various transcription factors including NF-kappaB, c-Fos, c-Jun, E2F-1, and p53.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"TP53\"],\"tf text\":[],\"tg text\":[\"p53\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"Here\",\"using a well-studied IgH-MAR and another potential MAR from p53 promoter\",\"we demonstrate that MARs alone can act as promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|weird, MAR as promoter?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[],\"tg text\":[\"p53\"],\"consensus\":[\"N\"],\"martin_comments\":[\"weird, MAR as promoter?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here, using a well-studied IgH-MAR and another potential MAR from p53 promoter, we demonstrate that MARs alone can act as promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"TP53\"],\"tf text\":[],\"tg text\":[\"p53\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"Here\",\"using a well-studied IgH-MAR and another potential MAR from p53 promoter\",\"we demonstrate that MARs alone can act as promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|weird, MAR as promoter?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[],\"tg text\":[\"p53\"],\"consensus\":[\"N\"],\"martin_comments\":[\"weird, MAR as promoter?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here, using a well-studied IgH-MAR and another potential MAR from p53 promoter, we demonstrate that MARs alone can act as promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is PRNP---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"SP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is PRNP\"],\"sentence\":[\"The coordinate regulation of the bovine PRNP promoter suggests the two Sp1 binding site polymorphisms control Sp1 binding to the PRNP promoter and its activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"SP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The coordinate regulation of the bovine PRNP promoter suggests the two Sp1 binding site polymorphisms control Sp1 binding to the PRNP promoter and its activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is PRNP---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"SP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is PRNP\"],\"sentence\":[\"The coordinate regulation of the bovine PRNP promoter suggests the two Sp1 binding site polymorphisms control Sp1 binding to the PRNP promoter and its activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"SP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The coordinate regulation of the bovine PRNP promoter suggests the two Sp1 binding site polymorphisms control Sp1 binding to the PRNP promoter and its activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 390 (char 389)\n",
      "{\"transcription factor (associated gene name)\":[\"NR1I2\"],\"target gene (associated gene name)\":[\"KDR\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that MMP-1 promotes VEGFR2 expression and proliferation of endothelial cells through stimulation of PAR-1 and activation of NF-B.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|PAR1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1I2\"],\"tg (associated gene name)\":[\"KDR\"],\"tf text\":[],\"tg text\":[\"VEGFR2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"PAR1\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We conclude that MMP-1 promotes VEGFR2 expression and proliferation of endothelial cells through stimulation of PAR-1 and activation of NF-??B.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"NR1I2\"],\"target gene (associated gene name)\":[\"KDR\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that MMP-1 promotes VEGFR2 expression and proliferation of endothelial cells through stimulation of PAR-1 and activation of NF-B.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|PAR1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1I2\"],\"tg (associated gene name)\":[\"KDR\"],\"tf text\":[],\"tg text\":[\"VEGFR2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"PAR1\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We conclude that MMP-1 promotes VEGFR2 expression and proliferation of endothelial cells through stimulation of PAR-1 and activation of NF-??B.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 457 (char 456)\n",
      "{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"GNRH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Affected genes included transcription factors (Stat5b\",\"Arnt\",\"Ahr)\",\"sex steroid hormone receptors (Esr1\",\"Esr2\",\"Pgr\",\"Ar)\",\"steroidogenic enzymes (Sts\",\"Hsd17b8)\",\"growth factors (Igf1\",\"Tgfa)\",\"and neuropeptides (Kiss1\",\"Tac2\",\"Gnrh1).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF role---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"GNRH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF role\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Affected genes included transcription factors (Stat5b, Arnt, Ahr), sex steroid hormone receptors (Esr1, Esr2, Pgr, Ar), steroidogenic enzymes (Sts, Hsd17b8), growth factors (Igf1, Tgfa), and neuropeptides (Kiss1, Tac2, Gnrh1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"GNRH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Affected genes included transcription factors (Stat5b\",\"Arnt\",\"Ahr)\",\"sex steroid hormone receptors (Esr1\",\"Esr2\",\"Pgr\",\"Ar)\",\"steroidogenic enzymes (Sts\",\"Hsd17b8)\",\"growth factors (Igf1\",\"Tgfa)\",\"and neuropeptides (Kiss1\",\"Tac2\",\"Gnrh1).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF role---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"GNRH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF role\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Affected genes included transcription factors (Stat5b, Arnt, Ahr), sex steroid hormone receptors (Esr1, Esr2, Pgr, Ar), steroidogenic enzymes (Sts, Hsd17b8), growth factors (Igf1, Tgfa), and neuropeptides (Kiss1, Tac2, Gnrh1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "HIF1A seems to be TG and regulator seems to be ATP6V0C? not TRE?---{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"ATP6V0C\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"ATP6V0C\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"HIF1A seems to be TG and regulator seems to be ATP6V0C? not TRE?\"],\"sentence\":[\"ATP6V0C competes with von Hippel-Lindau protein in hypoxia-inducible factor 1alpha (HIF-1alpha) binding and mediates HIF-1alpha expression by bafilomycin A1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"ATP6V0C\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"ATP6V0C\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"ATP6V0C competes with von Hippel-Lindau protein in hypoxia-inducible factor 1alpha (HIF-1alpha) binding and mediates HIF-1alpha expression by bafilomycin A1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "HIF1A seems to be TG and regulator seems to be ATP6V0C? not TRE?---{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"ATP6V0C\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"ATP6V0C\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"HIF1A seems to be TG and regulator seems to be ATP6V0C? not TRE?\"],\"sentence\":[\"ATP6V0C competes with von Hippel-Lindau protein in hypoxia-inducible factor 1alpha (HIF-1alpha) binding and mediates HIF-1alpha expression by bafilomycin A1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"ATP6V0C\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"ATP6V0C\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"ATP6V0C competes with von Hippel-Lindau protein in hypoxia-inducible factor 1alpha (HIF-1alpha) binding and mediates HIF-1alpha expression by bafilomycin A1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"HDAC7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"o\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Inhibition of c-jun expression by HDAC7 is mediated at the transcriptional level by its direct association with the c-jun gene promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|FLIPPED---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"HDAC7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"FLIPPED\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of c-jun expression by HDAC7 is mediated at the transcriptional level by its direct association with the c-jun gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"HDAC7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"o\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Inhibition of c-jun expression by HDAC7 is mediated at the transcriptional level by its direct association with the c-jun gene promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|FLIPPED---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"HDAC7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"FLIPPED\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of c-jun expression by HDAC7 is mediated at the transcriptional level by its direct association with the c-jun gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"PRKAA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Pharmacological activation of AMPK at high glucose concentrations interferes with glucose repression of PPARalpha and PPARalpha target genes in INS-1E cells as well as in rat islets.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"PRKAA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Pharmacological activation of AMPK at high glucose concentrations interferes with glucose repression of PPARalpha and PPARalpha target genes in INS-1E cells as well as in rat islets.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"PRKAA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Pharmacological activation of AMPK at high glucose concentrations interferes with glucose repression of PPARalpha and PPARalpha target genes in INS-1E cells as well as in rat islets.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"PRKAA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Pharmacological activation of AMPK at high glucose concentrations interferes with glucose repression of PPARalpha and PPARalpha target genes in INS-1E cells as well as in rat islets.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "cell?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR2E1\"],\"tg (associated gene name)\":[\"CEL\"],\"tf text\":[\"TLX\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"cell?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"This review focuses on cell-intrinsic regulators, including orphan nuclear receptor TLX, polycomb transcriptional repressor Bmi1, high-mobility-group DNA binding protein Sox2, basic helix-loop-helix Hes genes, histone modifying enzymes and chromatin remodeling proteins, and small RNA modulators, as well as cell-extrinsic signaling molecules, such as Wnt, Notch, Sonic hedgehog (Shh), TGFalpha, EGF, and FGF.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "cell?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR2E1\"],\"tg (associated gene name)\":[\"CEL\"],\"tf text\":[\"TLX\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"cell?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"This review focuses on cell-intrinsic regulators, including orphan nuclear receptor TLX, polycomb transcriptional repressor Bmi1, high-mobility-group DNA binding protein Sox2, basic helix-loop-helix Hes genes, histone modifying enzymes and chromatin remodeling proteins, and small RNA modulators, as well as cell-extrinsic signaling molecules, such as Wnt, Notch, Sonic hedgehog (Shh), TGFalpha, EGF, and FGF.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 400 (char 399)\n",
      "{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"SIRT1\"],\"tf text\":[\"RelA\"],\"tg text\":[\"SIRT1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Acetylation of NF-kappaB [RelA (p65)] at Lys(310) enhances its transcriptional activity\",\"which is inhibited by SIRT1 deacetylase\",\"type III HDAC (histone deacetylase).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"SIRT1\"],\"tf text\":[\"RelA\"],\"tg text\":[\"SIRT1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Acetylation of NF-kappaB [RelA (p65)] at Lys(310) enhances its transcriptional activity, which is inhibited by SIRT1 deacetylase, type III HDAC (histone deacetylase).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"SIRT1\"],\"tf text\":[\"RelA\"],\"tg text\":[\"SIRT1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Acetylation of NF-kappaB [RelA (p65)] at Lys(310) enhances its transcriptional activity\",\"which is inhibited by SIRT1 deacetylase\",\"type III HDAC (histone deacetylase).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"SIRT1\"],\"tf text\":[\"RelA\"],\"tg text\":[\"SIRT1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Acetylation of NF-kappaB [RelA (p65)] at Lys(310) enhances its transcriptional activity, which is inhibited by SIRT1 deacetylase, type III HDAC (histone deacetylase).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 366 (char 365)\n",
      "{\"tf associated gene name\":[\"MYBL2\"],\"tg associated gene name\":[\"CDK2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Whereas wild-type B-Myb transactivation activity could not be potentiated by cyclin A/Cdk2 in NIH3T3 cells\",\"the truncated protein was hyperactive.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYBL2\"],\"tg (associated gene name)\":[\"CDK2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Whereas wild-type B-Myb transactivation activity could not be potentiated by cyclin A/Cdk2 in NIH3T3 cells, the truncated protein was hyperactive.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MYBL2\"],\"tg associated gene name\":[\"CDK2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Whereas wild-type B-Myb transactivation activity could not be potentiated by cyclin A/Cdk2 in NIH3T3 cells\",\"the truncated protein was hyperactive.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYBL2\"],\"tg (associated gene name)\":[\"CDK2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Whereas wild-type B-Myb transactivation activity could not be potentiated by cyclin A/Cdk2 in NIH3T3 cells, the truncated protein was hyperactive.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear TRE|TG  type II 3 beta HSD---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"SGCG\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n?\",\"unclear\"],\"comment\":[\"unclear TRE\",\"TG  type II 3 beta HSD\"],\"sentence\":[\"A Stat5 expression construct was cotransfected with a 3 beta HSD-chloramphenol acetyltransferase (CAT) reporter construct comprised of nucleotides -301-->+45 of the human type II 3 beta HSD promoter linked to the CAT reporter gene sequence.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SGCG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"SGCG\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SGCG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"A Stat5 expression construct was cotransfected with a 3 beta HSD-chloramphenol acetyltransferase (CAT) reporter construct comprised of nucleotides -301-->+45 of the human type II 3 beta HSD promoter linked to the CAT reporter gene sequence.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "unclear TRE|TG  type II 3 beta HSD---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"SGCG\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n?\",\"unclear\"],\"comment\":[\"unclear TRE\",\"TG  type II 3 beta HSD\"],\"sentence\":[\"A Stat5 expression construct was cotransfected with a 3 beta HSD-chloramphenol acetyltransferase (CAT) reporter construct comprised of nucleotides -301-->+45 of the human type II 3 beta HSD promoter linked to the CAT reporter gene sequence.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SGCG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"SGCG\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SGCG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"A Stat5 expression construct was cotransfected with a 3 beta HSD-chloramphenol acetyltransferase (CAT) reporter construct comprised of nucleotides -301-->+45 of the human type II 3 beta HSD promoter linked to the CAT reporter gene sequence.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "Uncertain normalization TF|also only element stated---{\"tf associated gene name\":[\"CEBPG\"],\"tg associated gene name\":[\"APOA2\"],\"tf text\":[],\"tg text\":[\"apoA-II\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"Uncertain normalization TF\",\"also only element stated\"],\"sentence\":[\"Despite the importance of HNF1\",\"HNF4\",\"NF1\",\"and C/EBP alpha in the regulation of numerous other target genes\",\"deletion of the HNF1\",\"NF1\",\"and HNF4 and several C/EBP binding sites did not drastically affect the hepatic transcription of the apoA-II gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "Uncertain normalization TF|also only element stated---{\"tf associated gene name\":[\"CEBPG\"],\"tg associated gene name\":[\"APOA2\"],\"tf text\":[],\"tg text\":[\"apoA-II\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"Uncertain normalization TF\",\"also only element stated\"],\"sentence\":[\"Despite the importance of HNF1\",\"HNF4\",\"NF1\",\"and C/EBP alpha in the regulation of numerous other target genes\",\"deletion of the HNF1\",\"NF1\",\"and HNF4 and several C/EBP binding sites did not drastically affect the hepatic transcription of the apoA-II gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CEBPG\"],\"target gene (associated gene name)\":[\"APOA2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Despite the importance of HNF1\",\"HNF4\",\"NF1\",\"and C/EBP alpha in the regulation of numerous other target genes\",\"deletion of the HNF1\",\"NF1\",\"and HNF4 and several C/EBP binding sites did not drastically affect the hepatic transcription of the apoA-II gene.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPG\"],\"tg (associated gene name)\":[\"APOA2\"],\"tf text\":[],\"tg text\":[\"apoA-II\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Despite the importance of HNF1, HNF4, NF1, and C/EBP alpha in the regulation of numerous other target genes, deletion of the HNF1, NF1, and HNF4 and several C/EBP binding sites did not drastically affect the hepatic transcription of the apoA-II gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 344 (char 343)\n",
      "{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"CDKN1B\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"As a gene expression regulator\",\"QKI overexpression increased p27\",\"while it decreased cyclin D1 and c-fos expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"CDKN1B\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As a gene expression regulator, QKI overexpression increased p27, while it decreased cyclin D1 and c-fos expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"CDKN1B\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"As a gene expression regulator\",\"QKI overexpression increased p27\",\"while it decreased cyclin D1 and c-fos expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"CDKN1B\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As a gene expression regulator, QKI overexpression increased p27, while it decreased cyclin D1 and c-fos expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"MYBL2\"],\"tg associated gene name\":[\"CDC25C\"],\"tf text\":[\"B-myb\"],\"tg text\":[\"cdc25C\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"We found that Mdm2 expression rescues the temperature-sensitive phenotype of tsBN462 cells\",\"as shown by activation of cell cycle-regulated gene promoters (B-myb\",\"cyclin A\",\"and cdc25C)\",\"increased cell growth and DNA synthesis\",\"and inhibition of apoptosis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYBL2\"],\"tg (associated gene name)\":[\"CDC25C\"],\"tf text\":[\"B-myb\"],\"tg text\":[\"cdc25C\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that Mdm2 expression rescues the temperature-sensitive phenotype of tsBN462 cells, as shown by activation of cell cycle-regulated gene promoters (B-myb, cyclin A, and cdc25C), increased cell growth and DNA synthesis, and inhibition of apoptosis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"MYBL2\"],\"tg associated gene name\":[\"CDC25C\"],\"tf text\":[\"B-myb\"],\"tg text\":[\"cdc25C\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"We found that Mdm2 expression rescues the temperature-sensitive phenotype of tsBN462 cells\",\"as shown by activation of cell cycle-regulated gene promoters (B-myb\",\"cyclin A\",\"and cdc25C)\",\"increased cell growth and DNA synthesis\",\"and inhibition of apoptosis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYBL2\"],\"tg (associated gene name)\":[\"CDC25C\"],\"tf text\":[\"B-myb\"],\"tg text\":[\"cdc25C\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that Mdm2 expression rescues the temperature-sensitive phenotype of tsBN462 cells, as shown by activation of cell cycle-regulated gene promoters (B-myb, cyclin A, and cdc25C), increased cell growth and DNA synthesis, and inhibition of apoptosis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is AP-2 alpha---{\"tf associated gene name\":[\"GLI2\"],\"tg associated gene name\":[\"TLR2\"],\"tf text\":[],\"tg text\":[\"Toll-like receptor 2\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is AP-2 alpha\"],\"sentence\":[\"Upregulation of Toll-like receptor 2 gene expression by acetylation of AP-2 alpha in THP-1 cells\",\"a human monocytic cell line.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|very iffy use of THP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI2\"],\"tg (associated gene name)\":[\"TLR2\"],\"tf text\":[],\"tg text\":[\"Toll-like receptor 2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"very iffy use of THP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Upregulation of Toll-like receptor 2 gene expression by acetylation of AP-2 alpha in THP-1 cells, a human monocytic cell line.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is AP-2 alpha---{\"tf associated gene name\":[\"GLI2\"],\"tg associated gene name\":[\"TLR2\"],\"tf text\":[],\"tg text\":[\"Toll-like receptor 2\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is AP-2 alpha\"],\"sentence\":[\"Upregulation of Toll-like receptor 2 gene expression by acetylation of AP-2 alpha in THP-1 cells\",\"a human monocytic cell line.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|very iffy use of THP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI2\"],\"tg (associated gene name)\":[\"TLR2\"],\"tf text\":[],\"tg text\":[\"Toll-like receptor 2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"very iffy use of THP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Upregulation of Toll-like receptor 2 gene expression by acetylation of AP-2 alpha in THP-1 cells, a human monocytic cell line.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is crystallin genes---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is crystallin genes\"],\"sentence\":[\"The transcriptional control of crystallin genes is characterized by the reiterative use of transcription factors required for the establishment of lens precursors in combination with more ubiquitously expressed factors (e.g. AP-1\",\"AP-2alpha\",\"CREB and USF) and recruitment of histone acetyltransferases (HATs) CBP and p300\",\"and chromatin remodeling complexes SWI/SNF and ISWI.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcriptional control of crystallin genes is characterized by the reiterative use of transcription factors required for the establishment of lens precursors in combination with more ubiquitously expressed factors (e.g. AP-1, AP-2alpha, CREB and USF) and recruitment of histone acetyltransferases (HATs) CBP and p300, and chromatin remodeling complexes SWI/SNF and ISWI.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is crystallin genes---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is crystallin genes\"],\"sentence\":[\"The transcriptional control of crystallin genes is characterized by the reiterative use of transcription factors required for the establishment of lens precursors in combination with more ubiquitously expressed factors (e.g. AP-1\",\"AP-2alpha\",\"CREB and USF) and recruitment of histone acetyltransferases (HATs) CBP and p300\",\"and chromatin remodeling complexes SWI/SNF and ISWI.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcriptional control of crystallin genes is characterized by the reiterative use of transcription factors required for the establishment of lens precursors in combination with more ubiquitously expressed factors (e.g. AP-1, AP-2alpha, CREB and USF) and recruitment of histone acetyltransferases (HATs) CBP and p300, and chromatin remodeling complexes SWI/SNF and ISWI.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is PAC-1R ---{\"tf associated gene name\":[\"KLF6\"],\"tg associated gene name\":[\"LHB\"],\"tf text\":[],\"tg text\":[\"LHbeta\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is PAC-1R \"],\"sentence\":[\"In addition\",\"increasing concentrations of PACAP stimulation augmented the promoter activities for both LHbeta and FSHbeta in PAC-1R overexpressing cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is PAC-1R ---{\"tf associated gene name\":[\"KLF6\"],\"tg associated gene name\":[\"LHB\"],\"tf text\":[],\"tg text\":[\"LHbeta\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is PAC-1R \"],\"sentence\":[\"In addition\",\"increasing concentrations of PACAP stimulation augmented the promoter activities for both LHbeta and FSHbeta in PAC-1R overexpressing cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"KLF6\"],\"target gene (associated gene name)\":[\"LHB\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In addition\",\"increasing concentrations of PACAP stimulation augmented the promoter activities for both LHbeta and FSHbeta in PAC-1R overexpressing cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hmm PAC-1R enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KLF6\"],\"tg (associated gene name)\":[\"LHB\"],\"tf text\":[],\"tg text\":[\"LHbeta\"],\"consensus\":[\"V\"],\"martin_comments\":[\"hmm PAC-1R enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In addition, increasing concentrations of PACAP stimulation augmented the promoter activities for both LHbeta and FSHbeta in PAC-1R overexpressing cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 656 (char 655)\n",
      "{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Using arsenic trioxide-responsive KU7 and non-responsive 253JB-V bladder cancer cells as models\",\"we show that in KU7 cells\",\"< or =5 microM arsenic trioxide decreased Sp1\",\"Sp3 and Sp4 and several Sp-dependent genes and responses including cyclin D1\",\"epidermal growth factor receptor\",\"bcl-2\",\"survivin and vascular endothelial growth factor\",\"whereas at concentrations up to 15 microM\",\"minimal effects were observed in 253JB-V cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|PAX3?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX3\"],\"tg (associated gene name)\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PAX3?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using arsenic trioxide-responsive KU7 and non-responsive 253JB-V bladder cancer cells as models, we show that in KU7 cells, < or =5 microM arsenic trioxide decreased Sp1, Sp3 and Sp4 and several Sp-dependent genes and responses including cyclin D1, epidermal growth factor receptor, bcl-2, survivin and vascular endothelial growth factor, whereas at concentrations up to 15 microM, minimal effects were observed in 253JB-V cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Using arsenic trioxide-responsive KU7 and non-responsive 253JB-V bladder cancer cells as models\",\"we show that in KU7 cells\",\"< or =5 microM arsenic trioxide decreased Sp1\",\"Sp3 and Sp4 and several Sp-dependent genes and responses including cyclin D1\",\"epidermal growth factor receptor\",\"bcl-2\",\"survivin and vascular endothelial growth factor\",\"whereas at concentrations up to 15 microM\",\"minimal effects were observed in 253JB-V cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|PAX3?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX3\"],\"tg (associated gene name)\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PAX3?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using arsenic trioxide-responsive KU7 and non-responsive 253JB-V bladder cancer cells as models, we show that in KU7 cells, < or =5 microM arsenic trioxide decreased Sp1, Sp3 and Sp4 and several Sp-dependent genes and responses including cyclin D1, epidermal growth factor receptor, bcl-2, survivin and vascular endothelial growth factor, whereas at concentrations up to 15 microM, minimal effects were observed in 253JB-V cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is IL-23---{\"tf associated gene name\":[\"REL\"],\"tg associated gene name\":[\"CDKN2A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is IL-23\"],\"sentence\":[\"We report\",\"in this study\",\"that murine dendritic cells deficient in c-Rel\",\"a member of the NF-kappaB family\",\"are severely compromised in their ability to transcribe the p19 gene\",\"one of the two genes that encode the IL-23 protein.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is IL-23---{\"tf associated gene name\":[\"REL\"],\"tg associated gene name\":[\"CDKN2A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is IL-23\"],\"sentence\":[\"We report\",\"in this study\",\"that murine dendritic cells deficient in c-Rel\",\"a member of the NF-kappaB family\",\"are severely compromised in their ability to transcribe the p19 gene\",\"one of the two genes that encode the IL-23 protein.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"REL\"],\"target gene (associated gene name)\":[\"CDKN2A\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"v\"],\"negation\":[],\"sentence\":[\"We report\",\"in this study\",\"that murine dendritic cells deficient in c-Rel\",\"a member of the NF-kappaB family\",\"are severely compromised in their ability to transcribe the p19 gene\",\"one of the two genes that encode the IL-23 protein.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"0.127205\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"3\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"229\"],\"sentence pair density\":[\"0.0072992700729927\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"REL\"],\"target gene (associated gene name)\":[\"CDKN2A\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We report\",\"in this study\",\"that murine dendritic cells deficient in c-Rel\",\"a member of the NF-kappaB family\",\"are severely compromised in their ability to transcribe the p19 gene\",\"one of the two genes that encode the IL-23 protein.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|formally maybe ok, but p19 codes IL23?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"REL\"],\"tg (associated gene name)\":[\"CDKN2A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"formally maybe ok, but p19 codes IL23?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We report, in this study, that murine dendritic cells deficient in c-Rel, a member of the NF-kappaB family, are severely compromised in their ability to transcribe the p19 gene, one of the two genes that encode the IL-23 protein.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "IRF-problem; no suc TF here---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"IRF-problem; no suc TF here\"],\"sentence\":[\"Knockdown of OGT reduced the c-Rel O-GlcNAcylation and c-Rel-p50 interaction in response to LPS\",\"but did not affect either the binding of c-Rel to the iNOS promoter or the transcriptional activity of c-Rel.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|LPS=IRF6?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"LPS=IRF6?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Knockdown of OGT reduced the c-Rel O-GlcNAcylation and c-Rel-p50 interaction in response to LPS, but did not affect either the binding of c-Rel to the iNOS promoter or the transcriptional activity of c-Rel.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "IRF-problem; no suc TF here---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"IRF-problem; no suc TF here\"],\"sentence\":[\"Knockdown of OGT reduced the c-Rel O-GlcNAcylation and c-Rel-p50 interaction in response to LPS\",\"but did not affect either the binding of c-Rel to the iNOS promoter or the transcriptional activity of c-Rel.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|LPS=IRF6?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"LPS=IRF6?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Knockdown of OGT reduced the c-Rel O-GlcNAcylation and c-Rel-p50 interaction in response to LPS, but did not affect either the binding of c-Rel to the iNOS promoter or the transcriptional activity of c-Rel.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Cart1---{\"tf associated gene name\":[\"ALX1\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"o\"],\"comment\":[\"TG is Cart1\"],\"sentence\":[\"In this report\",\"we study the functional involvement of p300/CBP\",\"coactivators with intrinsic histone acetyltransferase (HAT) activity\",\"in the transcriptional control of Cart1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ALX1\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this report, we study the functional involvement of p300/CBP, coactivators with intrinsic histone acetyltransferase (HAT) activity, in the transcriptional control of Cart1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Cart1---{\"tf associated gene name\":[\"ALX1\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"o\"],\"comment\":[\"TG is Cart1\"],\"sentence\":[\"In this report\",\"we study the functional involvement of p300/CBP\",\"coactivators with intrinsic histone acetyltransferase (HAT) activity\",\"in the transcriptional control of Cart1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ALX1\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this report, we study the functional involvement of p300/CBP, coactivators with intrinsic histone acetyltransferase (HAT) activity, in the transcriptional control of Cart1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is GL alpha---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is GL alpha\"],\"sentence\":[\"Overexpression of Stat1 diminished TGF-beta1-induced\",\"Smad3/4-and Runx3-mediated GL alpha promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|GL-alpha?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"GL-alpha?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of Stat1 diminished TGF-beta1-induced, Smad3/4-and Runx3-mediated GL alpha promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is GL alpha---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is GL alpha\"],\"sentence\":[\"Overexpression of Stat1 diminished TGF-beta1-induced\",\"Smad3/4-and Runx3-mediated GL alpha promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|GL-alpha?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"GL-alpha?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of Stat1 diminished TGF-beta1-induced, Smad3/4-and Runx3-mediated GL alpha promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is IHG-1---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"CTGF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is IHG-1\"],\"sentence\":[\"In the HK-2 proximal tubule cell line\",\"overexpression of IHG-1 increased TGF-beta1-stimulated expression of connective tissue growth factor and fibronectin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is IHG-1---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"CTGF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is IHG-1\"],\"sentence\":[\"In the HK-2 proximal tubule cell line\",\"overexpression of IHG-1 increased TGF-beta1-stimulated expression of connective tissue growth factor and fibronectin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXC1\"],\"target gene (associated gene name)\":[\"CTGF\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In the HK-2 proximal tubule cell line\",\"overexpression of IHG-1 increased TGF-beta1-stimulated expression of connective tissue growth factor and fibronectin.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|IHG1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"CTGF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"IHG1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In the HK-2 proximal tubule cell line, overexpression of IHG-1 increased TGF-beta1-stimulated expression of connective tissue growth factor and fibronectin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 329 (char 328)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"TNF\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"TNF- expression in Schwann cells is induced by LPS and NF-B-dependent pathways.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|too indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"v\"],\"martin_comments\":[\"too indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TNF- expression in Schwann cells is induced by LPS and NF-B-dependent pathways.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"TNF\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"TNF- expression in Schwann cells is induced by LPS and NF-B-dependent pathways.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|too indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"v\"],\"martin_comments\":[\"too indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TNF- expression in Schwann cells is induced by LPS and NF-B-dependent pathways.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 379 (char 378)\n",
      "{\"transcription factor (associated gene name)\":[\"AR\"],\"target gene (associated gene name)\":[\"EGFR\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Inhibition of both AR isoforms did not affect EGFR and ERBB2 transcript levels but decreased EGFR and increased ERBB2 protein levels.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"EGFR\"],\"tf text\":[],\"tg text\":[\"EGFR\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of both AR isoforms did not affect EGFR and ERBB2 transcript levels but decreased EGFR and increased ERBB2 protein levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"EGFR\"],\"tf text\":[],\"tg text\":[\"EGFR\"],\"1\":[\"u\"],\"0\":[],\"negation\":[\"x\"],\"comment\":[],\"sentence\":[\"Inhibition of both AR isoforms did not affect EGFR and ERBB2 transcript levels but decreased EGFR and increased ERBB2 protein levels.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AR\"],\"target gene (associated gene name)\":[\"EGFR\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Inhibition of both AR isoforms did not affect EGFR and ERBB2 transcript levels but decreased EGFR and increased ERBB2 protein levels.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"EGFR\"],\"tf text\":[],\"tg text\":[\"EGFR\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of both AR isoforms did not affect EGFR and ERBB2 transcript levels but decreased EGFR and increased ERBB2 protein levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "rac2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"rac2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"However, AP1 binding to the endogenous Rac2 promoter exhibits a lag of 5-9 h, which correlates with reduced histone H3-Lys9 methylation, increased histone H3 acetylation, and increased nuclease accessibility within the 135 bp Rac2 gene regulatory region.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "rac2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"rac2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"However, AP1 binding to the endogenous Rac2 promoter exhibits a lag of 5-9 h, which correlates with reduced histone H3-Lys9 methylation, increased histone H3 acetylation, and increased nuclease accessibility within the 135 bp Rac2 gene regulatory region.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 346 (char 345)\n",
      "{\"transcription factor (associated gene name)\":[\"FOXL2\"],\"target gene (associated gene name)\":[\"CYP19A1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Nevertheless\",\"whether FOXL2 is a direct activator of aromatase requires further investigation.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|?hypothesis---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXL2\"],\"tg (associated gene name)\":[\"CYP19A1\"],\"tf text\":[\"FOXL2\"],\"tg text\":[\"aromatase\"],\"consensus\":[\"v\"],\"martin_comments\":[\"?hypothesis\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Nevertheless, whether FOXL2 is a direct activator of aromatase requires further investigation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOXL2\"],\"tg associated gene name\":[\"CYP19A1\"],\"tf text\":[\"FOXL2\"],\"tg text\":[\"aromatase\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Nevertheless\",\"whether FOXL2 is a direct activator of aromatase requires further investigation.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXL2\"],\"target gene (associated gene name)\":[\"CYP19A1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Nevertheless\",\"whether FOXL2 is a direct activator of aromatase requires further investigation.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|?hypothesis---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXL2\"],\"tg (associated gene name)\":[\"CYP19A1\"],\"tf text\":[\"FOXL2\"],\"tg text\":[\"aromatase\"],\"consensus\":[\"v\"],\"martin_comments\":[\"?hypothesis\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Nevertheless, whether FOXL2 is a direct activator of aromatase requires further investigation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "NRL not stated as (potential) regulator---{\"tf associated gene name\":[\"NRL\"],\"tg associated gene name\":[\"NRL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"NRL not stated as (potential) regulator\"],\"sentence\":[\"To decipher the transcriptional regulatory mechanisms upstream of Nrl\",\"we identified putative cis-control elements in the Nrl promoter/enhancer region by examining cross-species sequence conservation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRL\"],\"tg (associated gene name)\":[\"NRL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To decipher the transcriptional regulatory mechanisms upstream of Nrl, we identified putative cis-control elements in the Nrl promoter/enhancer region by examining cross-species sequence conservation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "NRL not stated as (potential) regulator---{\"tf associated gene name\":[\"NRL\"],\"tg associated gene name\":[\"NRL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"NRL not stated as (potential) regulator\"],\"sentence\":[\"To decipher the transcriptional regulatory mechanisms upstream of Nrl\",\"we identified putative cis-control elements in the Nrl promoter/enhancer region by examining cross-species sequence conservation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRL\"],\"tg (associated gene name)\":[\"NRL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To decipher the transcriptional regulatory mechanisms upstream of Nrl, we identified putative cis-control elements in the Nrl promoter/enhancer region by examining cross-species sequence conservation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 284 (char 283)\n",
      "{\"tf associated gene name\":[\"SALL3\"],\"tg associated gene name\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Sall3 correlates with the expression of TH in mouse olfactory bulb.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|correlates' enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SALL3\"],\"tg (associated gene name)\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"correlates' enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sall3 correlates with the expression of TH in mouse olfactory bulb.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SALL3\"],\"tg associated gene name\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Sall3 correlates with the expression of TH in mouse olfactory bulb.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|correlates' enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SALL3\"],\"tg (associated gene name)\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"correlates' enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sall3 correlates with the expression of TH in mouse olfactory bulb.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 467 (char 466)\n",
      "{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Augmentation of CBX7 by knockdown of miR-182 expression\",\"in turn\",\"positively regulated the expression of E-cadherin\",\"a key protein involved in maintaining normal epithelial cell morphology\",\"which is commonly lost during neoplastic progression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CBX7?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CBX7?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Augmentation of CBX7 by knockdown of miR-182 expression, in turn, positively regulated the expression of E-cadherin, a key protein involved in maintaining normal epithelial cell morphology, which is commonly lost during neoplastic progression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Augmentation of CBX7 by knockdown of miR-182 expression\",\"in turn\",\"positively regulated the expression of E-cadherin\",\"a key protein involved in maintaining normal epithelial cell morphology\",\"which is commonly lost during neoplastic progression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CBX7?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CBX7?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Augmentation of CBX7 by knockdown of miR-182 expression, in turn, positively regulated the expression of E-cadherin, a key protein involved in maintaining normal epithelial cell morphology, which is commonly lost during neoplastic progression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 389 (char 388)\n",
      "{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[\"p53\"],\"tg text\":[\"Notch1-IC\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Transcriptional downregulation of p53-responsive genes by Notch1-IC was confirmed both by chromatin immunoprecipitation assay and Northern blot analysis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[\"p53\"],\"tg text\":[\"Notch1-IC\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional downregulation of p53-responsive genes by Notch1-IC was confirmed both by chromatin immunoprecipitation assay and Northern blot analysis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[\"p53\"],\"tg text\":[\"Notch1-IC\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Transcriptional downregulation of p53-responsive genes by Notch1-IC was confirmed both by chromatin immunoprecipitation assay and Northern blot analysis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[\"p53\"],\"tg text\":[\"Notch1-IC\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional downregulation of p53-responsive genes by Notch1-IC was confirmed both by chromatin immunoprecipitation assay and Northern blot analysis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is IL-2 ---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"NFASC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is IL-2 \"],\"sentence\":[\"These results suggest that in human T lymphocytes both Dex and CsA inhibited IL-2 gene transcription through interference with transcription factors AP-1 and NF-AT.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is IL-2 ---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"NFASC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is IL-2 \"],\"sentence\":[\"These results suggest that in human T lymphocytes both Dex and CsA inhibited IL-2 gene transcription through interference with transcription factors AP-1 and NF-AT.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"NFASC\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"These results suggest that in human T lymphocytes both Dex and CsA inhibited IL-2 gene transcription through interference with transcription factors AP-1 and NF-AT.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"0.253751\"],\"pmid counts\":[\"22\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"164\"],\"sentence pair density\":[\"0.034090909090909\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"NFASC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results suggest that in human T lymphocytes both Dex and CsA inhibited IL-2 gene transcription through interference with transcription factors AP-1 and NF-AT.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"CP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"MBD2, MBD3, and MeCP2 occupy SYBL1 promoter simultaneously, as revealed by sequential ChIP.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"CP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"MBD2, MBD3, and MeCP2 occupy SYBL1 promoter simultaneously, as revealed by sequential ChIP.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is alpha-myosin heavy chain---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"SLC3A2\"],\"tf text\":[\"Yin Yang 1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is alpha-myosin heavy chain\"],\"sentence\":[\"Yin Yang 1 is increased in human heart failure and represses the activity of the human alpha-myosin heavy chain promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is alpha-myosin heavy chain---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"SLC3A2\"],\"tf text\":[\"Yin Yang 1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is alpha-myosin heavy chain\"],\"sentence\":[\"Yin Yang 1 is increased in human heart failure and represses the activity of the human alpha-myosin heavy chain promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"YY1\"],\"target gene (associated gene name)\":[\"SLC3A2\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Yin Yang 1 is increased in human heart failure and represses the activity of the human alpha-myosin heavy chain promoter.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"4.260045\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"121\"],\"sentence pair density\":[\"0.00571428571428571\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"SLC3A2\"],\"tf text\":[\"Yin Yang 1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Yin Yang 1 is increased in human heart failure and represses the activity of the human alpha-myosin heavy chain promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 487 (char 486)\n",
      "{\"tf associated gene name\":[\"NR4A1\"],\"tg associated gene name\":[\"NR3C1\"],\"tf text\":[\"NGFI-B\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"Transcriptional antagonism as well as direct protein-protein interaction between NGFI-B and GR take place primarily via their respective DNA binding domains\",\"although DNA binding itself and the GR homodimerization interface are not involved.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A1\"],\"tg (associated gene name)\":[\"NR3C1\"],\"tf text\":[\"NGFI-B\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional antagonism as well as direct protein-protein interaction between NGFI-B and GR take place primarily via their respective DNA binding domains, although DNA binding itself and the GR homodimerization interface are not involved.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR4A1\"],\"tg associated gene name\":[\"NR3C1\"],\"tf text\":[\"NGFI-B\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"Transcriptional antagonism as well as direct protein-protein interaction between NGFI-B and GR take place primarily via their respective DNA binding domains\",\"although DNA binding itself and the GR homodimerization interface are not involved.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A1\"],\"tg (associated gene name)\":[\"NR3C1\"],\"tf text\":[\"NGFI-B\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional antagonism as well as direct protein-protein interaction between NGFI-B and GR take place primarily via their respective DNA binding domains, although DNA binding itself and the GR homodimerization interface are not involved.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 357 (char 356)\n",
      "{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"GATA2\"],\"tf text\":[],\"tg text\":[\"GATA-2\"],\"1\":[],\"negation\":[],\"not true\":[\"o\"],\"comment\":[],\"sentence\":[\"Ectopic GATA-2 expression causes the induction of AR transcript levels under androgen-depleted conditions (P<0.05).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"GATA2\"],\"tf text\":[],\"tg text\":[\"GATA-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Ectopic GATA-2 expression causes the induction of AR transcript levels under androgen-depleted conditions (P<0.05).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"GATA2\"],\"tf text\":[],\"tg text\":[\"GATA-2\"],\"1\":[],\"negation\":[],\"not true\":[\"o\"],\"comment\":[],\"sentence\":[\"Ectopic GATA-2 expression causes the induction of AR transcript levels under androgen-depleted conditions (P<0.05).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"GATA2\"],\"tf text\":[],\"tg text\":[\"GATA-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Ectopic GATA-2 expression causes the induction of AR transcript levels under androgen-depleted conditions (P<0.05).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 346 (char 345)\n",
      "{\"transcription factor (associated gene name)\":[\"MAFB\"],\"target gene (associated gene name)\":[\"C1QA\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"GATA-1 was found to inhibit the activity of the C1qa promoter with the GATA-1 motifs and MafB motifs.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|only TF motif---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAFB\"],\"tg (associated gene name)\":[\"C1QA\"],\"tf text\":[\"MafB\"],\"tg text\":[\"C1qa\"],\"consensus\":[\"v\"],\"martin_comments\":[\"only TF motif\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"GATA-1 was found to inhibit the activity of the C1qa promoter with the GATA-1 motifs and MafB motifs.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MAFB\"],\"tg associated gene name\":[\"C1QA\"],\"tf text\":[\"MafB\"],\"tg text\":[\"C1qa\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"GATA-1 was found to inhibit the activity of the C1qa promoter with the GATA-1 motifs and MafB motifs.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor\":[\"MAFB\"],\"target gene\":[\"C1QA\"],\"sentence\":[\"GATA-1 was found to inhibit the activity of the C1qa promoter with the GATA-1 motifs and MafB motifs.\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"fusion tf\":[],\"qc'ed\":[\"0\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"MAFB\"],\"target gene (associated gene name)\":[\"C1QA\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"GATA-1 was found to inhibit the activity of the C1qa promoter with the GATA-1 motifs and MafB motifs.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|only TF motif---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAFB\"],\"tg (associated gene name)\":[\"C1QA\"],\"tf text\":[\"MafB\"],\"tg text\":[\"C1qa\"],\"consensus\":[\"v\"],\"martin_comments\":[\"only TF motif\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"GATA-1 was found to inhibit the activity of the C1qa promoter with the GATA-1 motifs and MafB motifs.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain about normalization TF and TG---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"DLST\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about normalization TF and TG\"],\"sentence\":[\"The ability of mutant E4-ORF6/7 proteins to form the stable E2F-E2aE promoter complex in vitro correlated well with their ability to trans-activate E2 transcription in vivo.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|wrong TG, E2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"DLST\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, E2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The ability of mutant E4-ORF6/7 proteins to form the stable E2F-E2aE promoter complex in vitro correlated well with their ability to trans-activate E2 transcription in vivo.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "uncertain about normalization TF and TG---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"DLST\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about normalization TF and TG\"],\"sentence\":[\"The ability of mutant E4-ORF6/7 proteins to form the stable E2F-E2aE promoter complex in vitro correlated well with their ability to trans-activate E2 transcription in vivo.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|wrong TG, E2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"DLST\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, E2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The ability of mutant E4-ORF6/7 proteins to form the stable E2F-E2aE promoter complex in vitro correlated well with their ability to trans-activate E2 transcription in vivo.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"PAK2\"],\"tf text\":[\"USF-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We show that the affinity of recombinant USF-1 for DNA is greatly increased by treatment with active cyclin A2-p34(cdc2) or cyclin B1-p34(cdc2) complexes and that its interaction with DNA is dependent on p34(cdc2)-mediated phosphorylation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"PAK2\"],\"tf text\":[\"USF-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We show that the affinity of recombinant USF-1 for DNA is greatly increased by treatment with active cyclin A2-p34(cdc2) or cyclin B1-p34(cdc2) complexes and that its interaction with DNA is dependent on p34(cdc2)-mediated phosphorylation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"TBP\"],\"tg associated gene name\":[\"GTF2A1\"],\"tf text\":[],\"tg text\":[\"TFIIA\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"This stimulatory effect of PC4 on promoter escape required TFIIA and the TATA box binding protein-associated factor subunits of TFIID.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBP\"],\"tg (associated gene name)\":[\"GTF2A1\"],\"tf text\":[],\"tg text\":[\"TFIIA\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This stimulatory effect of PC4 on promoter escape required TFIIA and the TATA box binding protein-associated factor subunits of TFIID.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named---{\"tf associated gene name\":[\"TBP\"],\"tg associated gene name\":[\"GTF2A1\"],\"tf text\":[],\"tg text\":[\"TFIIA\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"This stimulatory effect of PC4 on promoter escape required TFIIA and the TATA box binding protein-associated factor subunits of TFIID.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBP\"],\"tg (associated gene name)\":[\"GTF2A1\"],\"tf text\":[],\"tg text\":[\"TFIIA\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This stimulatory effect of PC4 on promoter escape required TFIIA and the TATA box binding protein-associated factor subunits of TFIID.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Myc---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"HDAC3\"],\"tf text\":[\"YY1\"],\"tg text\":[\"HDAC3\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is Myc\"],\"sentence\":[\"We demonstrated that p300 and HDAC3 cooperate with the transcription factor YY1 at an upstream YY1 binding site and repress the Myc promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"HDAC3\"],\"tf text\":[\"YY1\"],\"tg text\":[\"HDAC3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrated that p300 and HDAC3 cooperate with the transcription factor YY1 at an upstream YY1 binding site and repress the Myc promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Myc---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"HDAC3\"],\"tf text\":[\"YY1\"],\"tg text\":[\"HDAC3\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is Myc\"],\"sentence\":[\"We demonstrated that p300 and HDAC3 cooperate with the transcription factor YY1 at an upstream YY1 binding site and repress the Myc promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"HDAC3\"],\"tf text\":[\"YY1\"],\"tg text\":[\"HDAC3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrated that p300 and HDAC3 cooperate with the transcription factor YY1 at an upstream YY1 binding site and repress the Myc promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is NFAT2---{\"tf associated gene name\":[\"FOXP3\"],\"tg associated gene name\":[\"T\"],\"tf text\":[\"FOXP3\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is NFAT2\"],\"sentence\":[\"We demonstrate that transcription from the NFAT2 promoter is significantly suppressed by FOXP3\",\"and NFAT2 protein expression is markedly diminished in activated CD4(+)CD25(+)FOXP3(+) T(REG) compared with CD4(+)CD25(-)FOXP3(-) T cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"T\"],\"tf text\":[\"FOXP3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrate that transcription from the NFAT2 promoter is significantly suppressed by FOXP3, and NFAT2 protein expression is markedly diminished in activated CD4(+)CD25(+)FOXP3(+) T(REG) compared with CD4(+)CD25(-)FOXP3(-) T cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is NFAT2---{\"tf associated gene name\":[\"FOXP3\"],\"tg associated gene name\":[\"T\"],\"tf text\":[\"FOXP3\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is NFAT2\"],\"sentence\":[\"We demonstrate that transcription from the NFAT2 promoter is significantly suppressed by FOXP3\",\"and NFAT2 protein expression is markedly diminished in activated CD4(+)CD25(+)FOXP3(+) T(REG) compared with CD4(+)CD25(-)FOXP3(-) T cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"T\"],\"tf text\":[\"FOXP3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrate that transcription from the NFAT2 promoter is significantly suppressed by FOXP3, and NFAT2 protein expression is markedly diminished in activated CD4(+)CD25(+)FOXP3(+) T(REG) compared with CD4(+)CD25(-)FOXP3(-) T cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"GREB1\"],\"tf text\":[\"ESR1\"],\"tg text\":[\"GREB1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The identification of ESR1, WWOX, GREB1, and FOXA1 as primary targets confirmed the role of TFAP2C in hormone response.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"GREB1\"],\"tf text\":[\"ESR1\"],\"tg text\":[\"GREB1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The identification of ESR1, WWOX, GREB1, and FOXA1 as primary targets confirmed the role of TFAP2C in hormone response.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 421 (char 420)\n",
      "{\"tf associated gene name\":[\"SMAD3\"],\"tg associated gene name\":[\"SMAD3\"],\"tf text\":[\"Smad3\"],\"tg text\":[\"Smad3\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"TGF-beta1 increased endogenous Smad3 ubiquitination\",\"and proteasome inhibitor treatment blocked TGF-beta1-mediated Smad3 down-regulation resulting in accumulation of ubiquitinated Smad3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD3\"],\"tg (associated gene name)\":[\"SMAD3\"],\"tf text\":[\"Smad3\"],\"tg text\":[\"Smad3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TGF-beta1 increased endogenous Smad3 ubiquitination, and proteasome inhibitor treatment blocked TGF-beta1-mediated Smad3 down-regulation resulting in accumulation of ubiquitinated Smad3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SMAD3\"],\"tg associated gene name\":[\"SMAD3\"],\"tf text\":[\"Smad3\"],\"tg text\":[\"Smad3\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"TGF-beta1 increased endogenous Smad3 ubiquitination\",\"and proteasome inhibitor treatment blocked TGF-beta1-mediated Smad3 down-regulation resulting in accumulation of ubiquitinated Smad3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD3\"],\"tg (associated gene name)\":[\"SMAD3\"],\"tf text\":[\"Smad3\"],\"tg text\":[\"Smad3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TGF-beta1 increased endogenous Smad3 ubiquitination, and proteasome inhibitor treatment blocked TGF-beta1-mediated Smad3 down-regulation resulting in accumulation of ubiquitinated Smad3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is AR---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"SIN3A\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is AR\"],\"sentence\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated\",\"p/CAF-containing coactivator assembly for a p53-interacting\",\"Groucho/TLE1-\",\"and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SIN3A? AP1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"SIN3A\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SIN3A? AP1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated, p/CAF-containing coactivator assembly for a p53-interacting, Groucho/TLE1-, and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is AR---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"SIN3A\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is AR\"],\"sentence\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated\",\"p/CAF-containing coactivator assembly for a p53-interacting\",\"Groucho/TLE1-\",\"and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SIN3A? AP1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"SIN3A\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SIN3A? AP1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated, p/CAF-containing coactivator assembly for a p53-interacting, Groucho/TLE1-, and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 593 (char 592)\n",
      "{\"transcription factor (associated gene name)\":[\"NRF1\"],\"target gene (associated gene name)\":[\"SLC7A11\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Thus\",\"stress-inducible expression of xCT is a two-step process: under homeostatic conditions\",\"Nrf1 effectively suppresses nonspecific transactivation of xCT\",\"but when cells encounter severe oxidative/electrophilic stress\",\"Nrf1 is displaced from an antioxidant response element (ARE) in the gene promoter while Nrf2 is recruited to the ARE.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|xCT=XCT?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRF1\"],\"tg (associated gene name)\":[\"SLC7A11\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"v\"],\"martin_comments\":[\"xCT=XCT?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Thus, stress-inducible expression of xCT is a two-step process: under homeostatic conditions, Nrf1 effectively suppresses nonspecific transactivation of xCT, but when cells encounter severe oxidative/electrophilic stress, Nrf1 is displaced from an antioxidant response element (ARE) in the gene promoter while Nrf2 is recruited to the ARE.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"NRF1\"],\"target gene (associated gene name)\":[\"SLC7A11\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"r\"],\"negation\":[],\"sentence\":[\"Thus\",\"stress-inducible expression of xCT is a two-step process: under homeostatic conditions\",\"Nrf1 effectively suppresses nonspecific transactivation of xCT\",\"but when cells encounter severe oxidative/electrophilic stress\",\"Nrf1 is displaced from an antioxidant response element (ARE) in the gene promoter while Nrf2 is recruited to the ARE.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"0.966224\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"339\"],\"sentence pair density\":[\"0.00628930817610062\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NRF1\"],\"target gene (associated gene name)\":[\"SLC7A11\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Thus\",\"stress-inducible expression of xCT is a two-step process: under homeostatic conditions\",\"Nrf1 effectively suppresses nonspecific transactivation of xCT\",\"but when cells encounter severe oxidative/electrophilic stress\",\"Nrf1 is displaced from an antioxidant response element (ARE) in the gene promoter while Nrf2 is recruited to the ARE.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|xCT=XCT?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRF1\"],\"tg (associated gene name)\":[\"SLC7A11\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"v\"],\"martin_comments\":[\"xCT=XCT?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Thus, stress-inducible expression of xCT is a two-step process: under homeostatic conditions, Nrf1 effectively suppresses nonspecific transactivation of xCT, but when cells encounter severe oxidative/electrophilic stress, Nrf1 is displaced from an antioxidant response element (ARE) in the gene promoter while Nrf2 is recruited to the ARE.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 540 (char 539)\n",
      "{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"MDM2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Transient transfection of this mutant into a p53 null cell line resulted in activation not only of the pig3 but also of the MDM2 gene compatible with the presence of constitutively expressed MDM2 transcripts initiated from the P2 promoter in the p53-C277Y hemizygous Ewing's sarcoma cell line.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|hmm, p53 cell line enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"v\"],\"martin_comments\":[\"hmm, p53 cell line enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transient transfection of this mutant into a p53 null cell line resulted in activation not only of the pig3 but also of the MDM2 gene compatible with the presence of constitutively expressed MDM2 transcripts initiated from the P2 promoter in the p53-C277Y hemizygous Ewing's sarcoma cell line.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Transient transfection of this mutant into a p53 null cell line resulted in activation not only of the pig3 but also of the MDM2 gene compatible with the presence of constitutively expressed MDM2 transcripts initiated from the P2 promoter in the p53-C277Y hemizygous Ewing's sarcoma cell line.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"MDM2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Transient transfection of this mutant into a p53 null cell line resulted in activation not only of the pig3 but also of the MDM2 gene compatible with the presence of constitutively expressed MDM2 transcripts initiated from the P2 promoter in the p53-C277Y hemizygous Ewing's sarcoma cell line.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|hmm, p53 cell line enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"v\"],\"martin_comments\":[\"hmm, p53 cell line enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transient transfection of this mutant into a p53 null cell line resulted in activation not only of the pig3 but also of the MDM2 gene compatible with the presence of constitutively expressed MDM2 transcripts initiated from the P2 promoter in the p53-C277Y hemizygous Ewing's sarcoma cell line.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 454 (char 453)\n",
      "{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"SCNN1A\"],\"tf text\":[],\"tg text\":[\"alpha-ENaC2\"],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Our results suggest that Sp1 and Sp3 are essential for alpha-ENaC2 transcription in lung epithelial cells and that dephosphorylation of the Sp transcription factors by PP1 suppresses alpha-ENaC2 expression.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX3\"],\"tg (associated gene name)\":[\"SCNN1A\"],\"tf text\":[],\"tg text\":[\"alpha-ENaC2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that Sp1 and Sp3 are essential for alpha-ENaC2 transcription in lung epithelial cells and that dephosphorylation of the Sp transcription factors by PP1 suppresses alpha-ENaC2 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"SCNN1A\"],\"tf text\":[],\"tg text\":[\"alpha-ENaC2\"],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Our results suggest that Sp1 and Sp3 are essential for alpha-ENaC2 transcription in lung epithelial cells and that dephosphorylation of the Sp transcription factors by PP1 suppresses alpha-ENaC2 expression.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX3\"],\"tg (associated gene name)\":[\"SCNN1A\"],\"tf text\":[],\"tg text\":[\"alpha-ENaC2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that Sp1 and Sp3 are essential for alpha-ENaC2 transcription in lung epithelial cells and that dephosphorylation of the Sp transcription factors by PP1 suppresses alpha-ENaC2 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Arx---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"LGI1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is Arx\"],\"sentence\":[\"We thus identified 14 Arx-target genes enriched (Cxcr7\",\"Meis1\",\"Ppap2a\",\"Slc 12a5\",\"Ets2\",\"Phlda1\",\"Egr1\",\"Igf1\",\"Lmo3\",\"Sema6\",\"Lgi1\",\"Alk\",\"Tgfb3\",\"and Napb) and 5 genes specifically down-regulated (Hmgn3\",\"Lmo1\",\"Ebf3\",\"Rasgef1b\",\"and Slit2) in cortical migrating neurons.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"LGI1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We thus identified 14 Arx-target genes enriched (Cxcr7, Meis1, Ppap2a, Slc 12a5, Ets2, Phlda1, Egr1, Igf1, Lmo3, Sema6, Lgi1, Alk, Tgfb3, and Napb) and 5 genes specifically down-regulated (Hmgn3, Lmo1, Ebf3, Rasgef1b, and Slit2) in cortical migrating neurons.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is Arx---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"LGI1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is Arx\"],\"sentence\":[\"We thus identified 14 Arx-target genes enriched (Cxcr7\",\"Meis1\",\"Ppap2a\",\"Slc 12a5\",\"Ets2\",\"Phlda1\",\"Egr1\",\"Igf1\",\"Lmo3\",\"Sema6\",\"Lgi1\",\"Alk\",\"Tgfb3\",\"and Napb) and 5 genes specifically down-regulated (Hmgn3\",\"Lmo1\",\"Ebf3\",\"Rasgef1b\",\"and Slit2) in cortical migrating neurons.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"LGI1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We thus identified 14 Arx-target genes enriched (Cxcr7, Meis1, Ppap2a, Slc 12a5, Ets2, Phlda1, Egr1, Igf1, Lmo3, Sema6, Lgi1, Alk, Tgfb3, and Napb) and 5 genes specifically down-regulated (Hmgn3, Lmo1, Ebf3, Rasgef1b, and Slit2) in cortical migrating neurons.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|as indicated by phrase: \"regulated indirectly by alterations in 5-HT neurotransmission\"---{\"tf associated gene name\":[\"FEV\"],\"tg associated gene name\":[\"HTR1A\"],\"tf text\":[\"Pet-1\"],\"tg text\":[\"5-HT1A receptor\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\",\"as indicated by phrase: \\\"regulated indirectly by alterations in 5-HT neurotransmission\\\"\"],\"sentence\":[\"These data indicate that\",\"in addition to transcriptional regulation by Pet-1 in raphe neurons\",\"5-HT1A receptor expression is regulated indirectly by alterations in 5-HT neurotransmission in a region-specific manner that together may contribute to the aggressive/anxiety phenotype observed in Pet-1 null mice.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE|as indicated by phrase: \"regulated indirectly by alterations in 5-HT neurotransmission\"---{\"tf associated gene name\":[\"FEV\"],\"tg associated gene name\":[\"HTR1A\"],\"tf text\":[\"Pet-1\"],\"tg text\":[\"5-HT1A receptor\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\",\"as indicated by phrase: \\\"regulated indirectly by alterations in 5-HT neurotransmission\\\"\"],\"sentence\":[\"These data indicate that\",\"in addition to transcriptional regulation by Pet-1 in raphe neurons\",\"5-HT1A receptor expression is regulated indirectly by alterations in 5-HT neurotransmission in a region-specific manner that together may contribute to the aggressive/anxiety phenotype observed in Pet-1 null mice.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FEV\"],\"target gene (associated gene name)\":[\"HTR1A\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These data indicate that\",\"in addition to transcriptional regulation by Pet-1 in raphe neurons\",\"5-HT1A receptor expression is regulated indirectly by alterations in 5-HT neurotransmission in a region-specific manner that together may contribute to the aggressive/anxiety phenotype observed in Pet-1 null mice.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FEV\"],\"tg (associated gene name)\":[\"HTR1A\"],\"tf text\":[\"Pet-1\"],\"tg text\":[\"5-HT1A receptor\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"These data indicate that, in addition to transcriptional regulation by Pet-1 in raphe neurons, 5-HT1A receptor expression is regulated indirectly by alterations in 5-HT neurotransmission in a region-specific manner that together may contribute to the aggressive/anxiety phenotype observed in Pet-1 null mice.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 229 (char 228)\n",
      "{\"transcription factor (associated gene name)\":[\"ESRRA\"],\"target gene (associated gene name)\":[\"INSL3\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|NR4A1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESRRA\"],\"tg (associated gene name)\":[\"INSL3\"],\"tf text\":[],\"tg text\":[\"INSL3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"NR4A1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In the present study, we have identified the orphan nuclear receptor NR4A1 as a novel regulator of INSL3 transcription in Leydig cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"ESRRA\"],\"target gene (associated gene name)\":[\"INSL3\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|NR4A1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESRRA\"],\"tg (associated gene name)\":[\"INSL3\"],\"tf text\":[],\"tg text\":[\"INSL3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"NR4A1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In the present study, we have identified the orphan nuclear receptor NR4A1 as a novel regulator of INSL3 transcription in Leydig cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 488 (char 487)\n",
      "{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"We showed that LEF1 overexpression in LNCaP cells resulted in increased AR expression and consequently enhanced growth and invasion ability\",\"whereas LEF1 knockdown in LNCaP-AI cells decreased AR expression and\",\"subsequently\",\"growth and invasion capacity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|AR=TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AR=TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We showed that LEF1 overexpression in LNCaP cells resulted in increased AR expression and consequently enhanced growth and invasion ability, whereas LEF1 knockdown in LNCaP-AI cells decreased AR expression and, subsequently, growth and invasion capacity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"We showed that LEF1 overexpression in LNCaP cells resulted in increased AR expression and consequently enhanced growth and invasion ability\",\"whereas LEF1 knockdown in LNCaP-AI cells decreased AR expression and\",\"subsequently\",\"growth and invasion capacity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|AR=TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AR=TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We showed that LEF1 overexpression in LNCaP cells resulted in increased AR expression and consequently enhanced growth and invasion ability, whereas LEF1 knockdown in LNCaP-AI cells decreased AR expression and, subsequently, growth and invasion capacity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "PPARG is TG|TF not named---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"UCHL1\"],\"tf text\":[],\"tg text\":[\"UCHL1\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"PPARG is TG\",\"TF not named\"],\"sentence\":[\"We further found that UCHL1 induced p53 accumulation and reduced MDM2 protein level\",\"and subsequently upregulated the expression of p21\",\"as well as cleavage of caspase3 and PARP\",\"but not in catalytic mutant UCHL1 C90S-expressed cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|parp?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"UCHL1\"],\"tf text\":[],\"tg text\":[\"UCHL1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"parp?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We further found that UCHL1 induced p53 accumulation and reduced MDM2 protein level, and subsequently upregulated the expression of p21, as well as cleavage of caspase3 and PARP, but not in catalytic mutant UCHL1 C90S-expressed cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "PPARG is TG|TF not named---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"UCHL1\"],\"tf text\":[],\"tg text\":[\"UCHL1\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"PPARG is TG\",\"TF not named\"],\"sentence\":[\"We further found that UCHL1 induced p53 accumulation and reduced MDM2 protein level\",\"and subsequently upregulated the expression of p21\",\"as well as cleavage of caspase3 and PARP\",\"but not in catalytic mutant UCHL1 C90S-expressed cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|parp?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"UCHL1\"],\"tf text\":[],\"tg text\":[\"UCHL1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"parp?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We further found that UCHL1 induced p53 accumulation and reduced MDM2 protein level, and subsequently upregulated the expression of p21, as well as cleavage of caspase3 and PARP, but not in catalytic mutant UCHL1 C90S-expressed cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"IFNG\"],\"tf text\":[\"Stat1\"],\"tg text\":[\"IFN-gamma\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Interestingly this site does not appear to bind Stat1 itself\",\"as shown by supershift analysis and a lack of response to IFN-gamma both at the DNA-binding and transcriptional level.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no tg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[\"Stat1\"],\"tg text\":[\"IFN-gamma\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no tg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Interestingly this site does not appear to bind Stat1 itself, as shown by supershift analysis and a lack of response to IFN-gamma both at the DNA-binding and transcriptional level.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"IFNG\"],\"tf text\":[\"Stat1\"],\"tg text\":[\"IFN-gamma\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Interestingly this site does not appear to bind Stat1 itself\",\"as shown by supershift analysis and a lack of response to IFN-gamma both at the DNA-binding and transcriptional level.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no tg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[\"Stat1\"],\"tg text\":[\"IFN-gamma\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no tg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Interestingly this site does not appear to bind Stat1 itself, as shown by supershift analysis and a lack of response to IFN-gamma both at the DNA-binding and transcriptional level.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"KEAP1\"],\"tf text\":[\"Nrf2\"],\"tg text\":[\"Kelch like-ECH-associated protein 1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In stress conditions, Nrf2 dissociates from its cytosolic inhibitor, Kelch like-ECH-associated protein 1 (Keap1), and moves to the nucleus to regulate the transcription of antioxidant genes including the catalytic subunit of glutamylcysteine ligase (GCLC), a rate-limiting reduced glutathione (GSH) biosynthesis enzyme.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"KEAP1\"],\"tf text\":[\"Nrf2\"],\"tg text\":[\"Kelch like-ECH-associated protein 1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In stress conditions, Nrf2 dissociates from its cytosolic inhibitor, Kelch like-ECH-associated protein 1 (Keap1), and moves to the nucleus to regulate the transcription of antioxidant genes including the catalytic subunit of glutamylcysteine ligase (GCLC), a rate-limiting reduced glutathione (GSH) biosynthesis enzyme.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|no TG---{\"tf associated gene name\":[\"GATA1\"],\"tg associated gene name\":[\"SPTA1\"],\"tf text\":[\"GATA-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TG\"],\"sentence\":[\"GATA-1 and the ets factor PU.1 have been reported to functionally antagonize one another in the regulation of erythroid versus myeloid gene transcription and development.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no specific TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA1\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[\"GATA-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no specific TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"GATA-1 and the ets factor PU.1 have been reported to functionally antagonize one another in the regulation of erythroid versus myeloid gene transcription and development.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE|no TG---{\"tf associated gene name\":[\"GATA1\"],\"tg associated gene name\":[\"SPTA1\"],\"tf text\":[\"GATA-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TG\"],\"sentence\":[\"GATA-1 and the ets factor PU.1 have been reported to functionally antagonize one another in the regulation of erythroid versus myeloid gene transcription and development.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no specific TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA1\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[\"GATA-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no specific TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"GATA-1 and the ets factor PU.1 have been reported to functionally antagonize one another in the regulation of erythroid versus myeloid gene transcription and development.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 460 (char 459)\n",
      "{\"tf associated gene name\":[\"NCOA3\"],\"tg associated gene name\":[\"SUMO1\"],\"tf text\":[\"AIB1\"],\"tg text\":[\"SUMO-1\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Although AIB1 is known to be covalently modified by SUMO-1\",\"which serves to regulate its stability and transcriptional activity\",\"the exact SUMO E3 ligase involved in its sumoylation has not been determined.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA3\"],\"tg (associated gene name)\":[\"SUMO1\"],\"tf text\":[\"AIB1\"],\"tg text\":[\"SUMO-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although AIB1 is known to be covalently modified by SUMO-1, which serves to regulate its stability and transcriptional activity, the exact SUMO E3 ligase involved in its sumoylation has not been determined.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NCOA3\"],\"tg associated gene name\":[\"SUMO1\"],\"tf text\":[\"AIB1\"],\"tg text\":[\"SUMO-1\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Although AIB1 is known to be covalently modified by SUMO-1\",\"which serves to regulate its stability and transcriptional activity\",\"the exact SUMO E3 ligase involved in its sumoylation has not been determined.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA3\"],\"tg (associated gene name)\":[\"SUMO1\"],\"tf text\":[\"AIB1\"],\"tg text\":[\"SUMO-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although AIB1 is known to be covalently modified by SUMO-1, which serves to regulate its stability and transcriptional activity, the exact SUMO E3 ligase involved in its sumoylation has not been determined.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "tg=ICAM1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"PSMA5\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"tg=ICAM1\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We observed that PKC-zeta also induced the TNF-alpha-induced NF-kappaB binding to the ICAM-1 promoter and the resultant ICAM-1 gene transcription.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "tg=ICAM1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"PSMA5\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"tg=ICAM1\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We observed that PKC-zeta also induced the TNF-alpha-induced NF-kappaB binding to the ICAM-1 promoter and the resultant ICAM-1 gene transcription.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Sp1 sites---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"CCND3\"],\"tf text\":[],\"tg text\":[\"cyclin D3\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is Sp1 sites\"],\"sentence\":[\"Taken together\",\"these results suggest that the activation of cyclin D3 transcription by HDAC inhibitor apicidin was mediated through Sp1 sites and pointed to the possible participation of PKC-delta.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Sp1 sites---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"CCND3\"],\"tf text\":[],\"tg text\":[\"cyclin D3\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is Sp1 sites\"],\"sentence\":[\"Taken together\",\"these results suggest that the activation of cyclin D3 transcription by HDAC inhibitor apicidin was mediated through Sp1 sites and pointed to the possible participation of PKC-delta.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"YY1\"],\"target gene (associated gene name)\":[\"CCND3\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Taken together\",\"these results suggest that the activation of cyclin D3 transcription by HDAC inhibitor apicidin was mediated through Sp1 sites and pointed to the possible participation of PKC-delta.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.310014\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"198\"],\"sentence pair density\":[\"0.0161943319838056\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|delta---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"CCND3\"],\"tf text\":[],\"tg text\":[\"cyclin D3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"delta\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, these results suggest that the activation of cyclin D3 transcription by HDAC inhibitor apicidin was mediated through Sp1 sites and pointed to the possible participation of PKC-delta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "association weak---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TWIST1\"],\"tg (associated gene name)\":[\"VIM\"],\"tf text\":[],\"tg text\":[\"vimentin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"association weak\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Triple-negative breast cancer cells such as MDA-MB-231 and Hs578T show a basal phenotype characterized by loss of E-cadherin expression and higher expression of mesenchymal markers such as N-cadherin and vimentin along with transcriptional repressors such as twist and snail.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "association weak---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TWIST1\"],\"tg (associated gene name)\":[\"VIM\"],\"tf text\":[],\"tg text\":[\"vimentin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"association weak\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Triple-negative breast cancer cells such as MDA-MB-231 and Hs578T show a basal phenotype characterized by loss of E-cadherin expression and higher expression of mesenchymal markers such as N-cadherin and vimentin along with transcriptional repressors such as twist and snail.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 441 (char 440)\n",
      "{\"tf associated gene name\":[\"FOXN1\"],\"tg associated gene name\":[\"PTHLH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Neoplastic transformation of normal rat embryo fibroblasts by a mutated p53 and an activated ras oncogene induces parathyroid hormone-related peptide gene expression and causes hypercalcemia in nude mice.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|FOXN1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXN1\"],\"tg (associated gene name)\":[\"PTHLH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"FOXN1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Neoplastic transformation of normal rat embryo fibroblasts by a mutated p53 and an activated ras oncogene induces parathyroid hormone-related peptide gene expression and causes hypercalcemia in nude mice.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOXN1\"],\"tg associated gene name\":[\"PTHLH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Neoplastic transformation of normal rat embryo fibroblasts by a mutated p53 and an activated ras oncogene induces parathyroid hormone-related peptide gene expression and causes hypercalcemia in nude mice.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|FOXN1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXN1\"],\"tg (associated gene name)\":[\"PTHLH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"FOXN1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Neoplastic transformation of normal rat embryo fibroblasts by a mutated p53 and an activated ras oncogene induces parathyroid hormone-related peptide gene expression and causes hypercalcemia in nude mice.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is mitochondrial HMG-CoA synthase---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is mitochondrial HMG-CoA synthase\"],\"sentence\":[\"In conclusion\",\"our data demonstrate that regulation of mitochondrial HMG-CoA synthase gene expression by fatty acids is mediated by PPAR\",\"supporting the hypothesis that PPAR has an important role at the transcriptional level in the regulation of lipid metabolism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In conclusion, our data demonstrate that regulation of mitochondrial HMG-CoA synthase gene expression by fatty acids is mediated by PPAR, supporting the hypothesis that PPAR has an important role at the transcriptional level in the regulation of lipid metabolism.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is mitochondrial HMG-CoA synthase---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is mitochondrial HMG-CoA synthase\"],\"sentence\":[\"In conclusion\",\"our data demonstrate that regulation of mitochondrial HMG-CoA synthase gene expression by fatty acids is mediated by PPAR\",\"supporting the hypothesis that PPAR has an important role at the transcriptional level in the regulation of lipid metabolism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In conclusion, our data demonstrate that regulation of mitochondrial HMG-CoA synthase gene expression by fatty acids is mediated by PPAR, supporting the hypothesis that PPAR has an important role at the transcriptional level in the regulation of lipid metabolism.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 679 (char 678)\n",
      "{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"MAPK1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Treating VSMCs with PD98059\",\"the ERK inhibitor\",\"inhibited ERK activation and KLF5 phosphorylation as well as the interaction between KLF5 and c-Jun.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.261592\"],\"pmid counts\":[\"13\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"148\"],\"sentence pair density\":[\"0.0087719298245614\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"MAPK1\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Treating VSMCs with PD98059, the ERK inhibitor, inhibited ERK activation and KLF5 phosphorylation as well as the interaction between KLF5 and c-Jun.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"MAPK1\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Treating VSMCs with PD98059\",\"the ERK inhibitor\",\"inhibited ERK activation and KLF5 phosphorylation as well as the interaction between KLF5 and c-Jun.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"MAPK1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Treating VSMCs with PD98059\",\"the ERK inhibitor\",\"inhibited ERK activation and KLF5 phosphorylation as well as the interaction between KLF5 and c-Jun.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.261592\"],\"pmid counts\":[\"13\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"148\"],\"sentence pair density\":[\"0.0087719298245614\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"MAPK1\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Treating VSMCs with PD98059, the ERK inhibitor, inhibited ERK activation and KLF5 phosphorylation as well as the interaction between KLF5 and c-Jun.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is COL2A1---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"SP3\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"Sp3\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"TG is COL2A1\"],\"sentence\":[\"Nevertheless\",\"IL-1 beta decreased the binding activity of both Sp1 and Sp3 to the 63-bp short COL2A1 promoter\",\"suggesting that the cytokine exerts a post-transcriptional regulatory mechanism on Sp1 and Sp3 gene expressions.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"SP3\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"Sp3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Nevertheless, IL-1 beta decreased the binding activity of both Sp1 and Sp3 to the 63-bp short COL2A1 promoter, suggesting that the cytokine exerts a post-transcriptional regulatory mechanism on Sp1 and Sp3 gene expressions.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is COL2A1---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"SP3\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"Sp3\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"TG is COL2A1\"],\"sentence\":[\"Nevertheless\",\"IL-1 beta decreased the binding activity of both Sp1 and Sp3 to the 63-bp short COL2A1 promoter\",\"suggesting that the cytokine exerts a post-transcriptional regulatory mechanism on Sp1 and Sp3 gene expressions.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"SP3\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"Sp3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Nevertheless, IL-1 beta decreased the binding activity of both Sp1 and Sp3 to the 63-bp short COL2A1 promoter, suggesting that the cytokine exerts a post-transcriptional regulatory mechanism on Sp1 and Sp3 gene expressions.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not clear statement that ESR1 (ER) regulates TG BSP ---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not clear statement that ESR1 (ER) regulates TG BSP \"],\"sentence\":[\"We found that siRNA-targeted depletion of a cohesin subunit\",\"RAD21\",\"decreased MYC expression in ER-positive (MCF7 and T47D) and ER-negative (MDA-MB-231) breast cancer cell lines.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that siRNA-targeted depletion of a cohesin subunit, RAD21, decreased MYC expression in ER-positive (MCF7 and T47D) and ER-negative (MDA-MB-231) breast cancer cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not clear statement that ESR1 (ER) regulates TG BSP ---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not clear statement that ESR1 (ER) regulates TG BSP \"],\"sentence\":[\"We found that siRNA-targeted depletion of a cohesin subunit\",\"RAD21\",\"decreased MYC expression in ER-positive (MCF7 and T47D) and ER-negative (MDA-MB-231) breast cancer cell lines.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that siRNA-targeted depletion of a cohesin subunit, RAD21, decreased MYC expression in ER-positive (MCF7 and T47D) and ER-negative (MDA-MB-231) breast cancer cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 491 (char 490)\n",
      "{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Persistence of signaling by the wild-type AR in therapy-resistant tumors is due to the increased receptor activity caused by cross talk of AR with multiple intracellular signaling cascades\",\"especially the growth factor activated MAP kinase/ERK and PI3 kinase/Akt pathways.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no direct Gene Expression mention---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no direct Gene Expression mention\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Persistence of signaling by the wild-type AR in therapy-resistant tumors is due to the increased receptor activity caused by cross talk of AR with multiple intracellular signaling cascades, especially the growth factor activated MAP kinase/ERK and PI3 kinase/Akt pathways.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Persistence of signaling by the wild-type AR in therapy-resistant tumors is due to the increased receptor activity caused by cross talk of AR with multiple intracellular signaling cascades\",\"especially the growth factor activated MAP kinase/ERK and PI3 kinase/Akt pathways.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no direct Gene Expression mention---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no direct Gene Expression mention\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Persistence of signaling by the wild-type AR in therapy-resistant tumors is due to the increased receptor activity caused by cross talk of AR with multiple intracellular signaling cascades, especially the growth factor activated MAP kinase/ERK and PI3 kinase/Akt pathways.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 552 (char 551)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"IL6\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"As NFB inhibitor and knockdown RelA expression greatly reduced constitutive AhR-induced IL-6 expression\",\"we hypothesized that AhR expression\",\"in the absence of exogenous ligand\",\"is able to modulate NFB activity and subsequently upregulate IL-6 expression\",\"thus promoting the development of lung AD.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|a bit idirect but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit idirect but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As NFB inhibitor and knockdown RelA expression greatly reduced constitutive AhR-induced IL-6 expression, we hypothesized that AhR expression, in the absence of exogenous ligand, is able to modulate NFB activity and subsequently upregulate IL-6 expression, thus promoting the development of lung AD.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"As NF??B inhibitor and knockdown RelA expression greatly reduced constitutive AhR-induced IL-6 expression\",\"we hypothesized that AhR expression\",\"in the absence of exogenous ligand\",\"is able to modulate NF??B activity and subsequently upregulate IL-6 expression\",\"thus promoting the development of lung AD.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"IL6\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"As NFB inhibitor and knockdown RelA expression greatly reduced constitutive AhR-induced IL-6 expression\",\"we hypothesized that AhR expression\",\"in the absence of exogenous ligand\",\"is able to modulate NFB activity and subsequently upregulate IL-6 expression\",\"thus promoting the development of lung AD.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|a bit idirect but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit idirect but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As NFB inhibitor and knockdown RelA expression greatly reduced constitutive AhR-induced IL-6 expression, we hypothesized that AhR expression, in the absence of exogenous ligand, is able to modulate NFB activity and subsequently upregulate IL-6 expression, thus promoting the development of lung AD.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"GDI1\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Here we present evidence that expression of E1A243 leads to relief of YY1-mediated repression by a disruption of the ATF/CREB-YY1 complex.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"GDI1\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Here we present evidence that expression of E1A243 leads to relief of YY1-mediated repression by a disruption of the ATF/CREB-YY1 complex.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 416 (char 415)\n",
      "{\"tf associated gene name\":[\"GLI1\"],\"tg associated gene name\":[\"ULK3\"],\"tf text\":[\"GLI1\"],\"tg text\":[\"ULK3\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"We demonstrate that ULK3 enhances endogenous and over-expressed GLI1 and GLI2 transcriptional activity in cultured cells\",\"as assessed by GLI-luciferase reporter assay.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI1\"],\"tg (associated gene name)\":[\"ULK3\"],\"tf text\":[\"GLI1\"],\"tg text\":[\"ULK3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrate that ULK3 enhances endogenous and over-expressed GLI1 and GLI2 transcriptional activity in cultured cells, as assessed by GLI-luciferase reporter assay.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"GLI1\"],\"tg associated gene name\":[\"ULK3\"],\"tf text\":[\"GLI1\"],\"tg text\":[\"ULK3\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"We demonstrate that ULK3 enhances endogenous and over-expressed GLI1 and GLI2 transcriptional activity in cultured cells\",\"as assessed by GLI-luciferase reporter assay.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI1\"],\"tg (associated gene name)\":[\"ULK3\"],\"tf text\":[\"GLI1\"],\"tg text\":[\"ULK3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrate that ULK3 enhances endogenous and over-expressed GLI1 and GLI2 transcriptional activity in cultured cells, as assessed by GLI-luciferase reporter assay.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG could also be normalized to   Carboxypeptidase O---{\"tf associated gene name\":[\"GATA1\"],\"tg associated gene name\":[\"CPOX\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG could also be normalized to   Carboxypeptidase O\"],\"sentence\":[\"These results indicate that GATA-1 plays an important role in regulation of transcription of the CPO gene in erythroid cells.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TG could also be normalized to   Carboxypeptidase O---{\"tf associated gene name\":[\"GATA1\"],\"tg associated gene name\":[\"CPOX\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG could also be normalized to   Carboxypeptidase O\"],\"sentence\":[\"These results indicate that GATA-1 plays an important role in regulation of transcription of the CPO gene in erythroid cells.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"GATA1\"],\"target gene (associated gene name)\":[\"CPOX\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results indicate that GATA-1 plays an important role in regulation of transcription of the CPO gene in erythroid cells.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA1\"],\"tg (associated gene name)\":[\"CPOX\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that GATA-1 plays an important role in regulation of transcription of the CPO gene in erythroid cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 388 (char 387)\n",
      "{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"RELA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The enhanced acetylation of p52 in the nuclei may operate to cause nuclear retention of p65 by increasing the p52/p65 interaction and preventing IkappaBalpha-p65 binding.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"RELA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The enhanced acetylation of p52 in the nuclei may operate to cause nuclear retention of p65 by increasing the p52/p65 interaction and preventing IkappaBalpha-p65 binding.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"RELA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The enhanced acetylation of p52 in the nuclei may operate to cause nuclear retention of p65 by increasing the p52/p65 interaction and preventing IkappaBalpha-p65 binding.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"RELA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The enhanced acetylation of p52 in the nuclei may operate to cause nuclear retention of p65 by increasing the p52/p65 interaction and preventing IkappaBalpha-p65 binding.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is  Hoxa-1|several different TGs---{\"tf associated gene name\":[\"GBX2\"],\"tg associated gene name\":[\"IBSP\"],\"tf text\":[\"Gbx2\"],\"tg text\":[\"BSP\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is  Hoxa-1\",\"several different TGs\"],\"sentence\":[\"We have characterized in detail these Hoxa1(-/-) ES cells by performing microarray analyses\",\"and by this technique we have identified a number of putative Hoxa-1 target genes\",\"including genes involved in bone development (e.g. Col1a1\",\"Postn/Osf2\",\"and the bone sialoprotein gene or BSP)\",\"genes that are expressed in the developing brain (e.g. Nnat\",\"Wnt3a\",\"BDNF\",\"RhoB\",\"and Gbx2)\",\"and genes involved in various cellular processes (e.g. M-RAS\",\"Sox17\",\"Cdkn2b\",\"LamA1\",\"Col4a1\",\"Foxa2\",\"Foxq1\",\"Klf5\",\"and Igf2).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF=HOXA---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GBX2\"],\"tg (associated gene name)\":[\"IBSP\"],\"tf text\":[\"Gbx2\"],\"tg text\":[\"BSP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=HOXA\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We have characterized in detail these Hoxa1(-/-) ES cells by performing microarray analyses, and by this technique we have identified a number of putative Hoxa-1 target genes, including genes involved in bone development (e.g. Col1a1, Postn/Osf2, and the bone sialoprotein gene or BSP), genes that are expressed in the developing brain (e.g. Nnat, Wnt3a, BDNF, RhoB, and Gbx2), and genes involved in various cellular processes (e.g. M-RAS, Sox17, Cdkn2b, LamA1, Col4a1, Foxa2, Foxq1, Klf5, and Igf2).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is  Hoxa-1|several different TGs---{\"tf associated gene name\":[\"GBX2\"],\"tg associated gene name\":[\"IBSP\"],\"tf text\":[\"Gbx2\"],\"tg text\":[\"BSP\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is  Hoxa-1\",\"several different TGs\"],\"sentence\":[\"We have characterized in detail these Hoxa1(-/-) ES cells by performing microarray analyses\",\"and by this technique we have identified a number of putative Hoxa-1 target genes\",\"including genes involved in bone development (e.g. Col1a1\",\"Postn/Osf2\",\"and the bone sialoprotein gene or BSP)\",\"genes that are expressed in the developing brain (e.g. Nnat\",\"Wnt3a\",\"BDNF\",\"RhoB\",\"and Gbx2)\",\"and genes involved in various cellular processes (e.g. M-RAS\",\"Sox17\",\"Cdkn2b\",\"LamA1\",\"Col4a1\",\"Foxa2\",\"Foxq1\",\"Klf5\",\"and Igf2).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF=HOXA---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GBX2\"],\"tg (associated gene name)\":[\"IBSP\"],\"tf text\":[\"Gbx2\"],\"tg text\":[\"BSP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=HOXA\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We have characterized in detail these Hoxa1(-/-) ES cells by performing microarray analyses, and by this technique we have identified a number of putative Hoxa-1 target genes, including genes involved in bone development (e.g. Col1a1, Postn/Osf2, and the bone sialoprotein gene or BSP), genes that are expressed in the developing brain (e.g. Nnat, Wnt3a, BDNF, RhoB, and Gbx2), and genes involved in various cellular processes (e.g. M-RAS, Sox17, Cdkn2b, LamA1, Col4a1, Foxa2, Foxq1, Klf5, and Igf2).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 398 (char 397)\n",
      "{\"tf associated gene name\":[\"NR1H3\"],\"tg associated gene name\":[\"FGF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"We concluded that FGF-1-induced apoE expression in astrocytes depends on LXRalpha being mediated by both LXRalpha expression and an LXRalpha ligand biosynthesis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|jumbled relations---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1H3\"],\"tg (associated gene name)\":[\"FGF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"jumbled relations\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We concluded that FGF-1-induced apoE expression in astrocytes depends on LXRalpha being mediated by both LXRalpha expression and an LXRalpha ligand biosynthesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR1H3\"],\"tg associated gene name\":[\"FGF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"We concluded that FGF-1-induced apoE expression in astrocytes depends on LXRalpha being mediated by both LXRalpha expression and an LXRalpha ligand biosynthesis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|jumbled relations---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1H3\"],\"tg (associated gene name)\":[\"FGF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"jumbled relations\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We concluded that FGF-1-induced apoE expression in astrocytes depends on LXRalpha being mediated by both LXRalpha expression and an LXRalpha ligand biosynthesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"NR4A1\"],\"tg associated gene name\":[\"ISCU\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"In contrast to HMR\",\"we found that substantial silencing of HML could be established without passage through S phase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"NR4A1\"],\"tg associated gene name\":[\"ISCU\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"In contrast to HMR\",\"we found that substantial silencing of HML could be established without passage through S phase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A1\"],\"tg (associated gene name)\":[\"ISCU\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast to HMR, we found that substantial silencing of HML could be established without passage through S phase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 456 (char 455)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"IL6\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In a series of co-transfection assays\",\"we found that 5alpha-dihydrotestosterone\",\"through the androgen receptor\",\"repressed activation of the interleukin-6 promoter\",\"in part\",\"by inhibiting NFkappaB activity.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"interleukin-6\"],\"consensus\":[\"V\"],\"martin_comments\":[\"too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In a series of co-transfection assays, we found that 5alpha-dihydrotestosterone, through the androgen receptor, repressed activation of the interleukin-6 promoter, in part, by inhibiting NFkappaB activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"interleukin-6\"],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"In a series of co-transfection assays\",\"we found that 5alpha-dihydrotestosterone\",\"through the androgen receptor\",\"repressed activation of the interleukin-6 promoter\",\"in part\",\"by inhibiting NFkappaB activity.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"IL6\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In a series of co-transfection assays\",\"we found that 5alpha-dihydrotestosterone\",\"through the androgen receptor\",\"repressed activation of the interleukin-6 promoter\",\"in part\",\"by inhibiting NFkappaB activity.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"interleukin-6\"],\"consensus\":[\"V\"],\"martin_comments\":[\"too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In a series of co-transfection assays, we found that 5alpha-dihydrotestosterone, through the androgen receptor, repressed activation of the interleukin-6 promoter, in part, by inhibiting NFkappaB activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 554 (char 553)\n",
      "{\"tf associated gene name\":[\"IRF3\"],\"tg associated gene name\":[\"PRKCB\"],\"tf text\":[\"IRF-3\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Furthermore\",\"reporter gene assays confirmed that PKCalpha regulates IFN-beta promoter activity\",\"since overexpression of dominant negative PKCalpha but not PKCbeta(I) repressed interferon regulatory factor 3 (IRF-3)-dependent but not NF-kappaB-mediated promoter activity upon TLR3 engagement in HEK 293 cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF role flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF3\"],\"tg (associated gene name)\":[\"PRKCB\"],\"tf text\":[\"IRF-3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF role flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, reporter gene assays confirmed that PKCalpha regulates IFN-beta promoter activity, since overexpression of dominant negative PKCalpha but not PKCbeta(I) repressed interferon regulatory factor 3 (IRF-3)-dependent but not NF-kappaB-mediated promoter activity upon TLR3 engagement in HEK 293 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"IRF3\"],\"tg associated gene name\":[\"PRKCB\"],\"tf text\":[\"IRF-3\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Furthermore\",\"reporter gene assays confirmed that PKCalpha regulates IFN-beta promoter activity\",\"since overexpression of dominant negative PKCalpha but not PKCbeta(I) repressed interferon regulatory factor 3 (IRF-3)-dependent but not NF-kappaB-mediated promoter activity upon TLR3 engagement in HEK 293 cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF role flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF3\"],\"tg (associated gene name)\":[\"PRKCB\"],\"tf text\":[\"IRF-3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF role flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, reporter gene assays confirmed that PKCalpha regulates IFN-beta promoter activity, since overexpression of dominant negative PKCalpha but not PKCbeta(I) repressed interferon regulatory factor 3 (IRF-3)-dependent but not NF-kappaB-mediated promoter activity upon TLR3 engagement in HEK 293 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 491 (char 490)\n",
      "{\"tf associated gene name\":[\"MYBL2\"],\"tg associated gene name\":[\"DHFR\"],\"tf text\":[\"MYBL2\"],\"tg text\":[\"DHFR\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Expression levels of TFDP1 and E2F1 correlated with those of seven transcriptional targets ( TYMS\",\"DHFR\",\"PCNA\",\"RRM1\",\"CCNE1\",\"CDC2\",\"and MYBL2) that play important roles in the G1/S transition\",\"and down-regulation of TFDP1 inhibited growth of Hep3B cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|only targets linked---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYBL2\"],\"tg (associated gene name)\":[\"DHFR\"],\"tf text\":[\"MYBL2\"],\"tg text\":[\"DHFR\"],\"consensus\":[\"N\"],\"martin_comments\":[\"only targets linked\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression levels of TFDP1 and E2F1 correlated with those of seven transcriptional targets ( TYMS, DHFR, PCNA, RRM1, CCNE1, CDC2, and MYBL2) that play important roles in the G1/S transition, and down-regulation of TFDP1 inhibited growth of Hep3B cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MYBL2\"],\"tg associated gene name\":[\"DHFR\"],\"tf text\":[\"MYBL2\"],\"tg text\":[\"DHFR\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Expression levels of TFDP1 and E2F1 correlated with those of seven transcriptional targets ( TYMS\",\"DHFR\",\"PCNA\",\"RRM1\",\"CCNE1\",\"CDC2\",\"and MYBL2) that play important roles in the G1/S transition\",\"and down-regulation of TFDP1 inhibited growth of Hep3B cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|only targets linked---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYBL2\"],\"tg (associated gene name)\":[\"DHFR\"],\"tf text\":[\"MYBL2\"],\"tg text\":[\"DHFR\"],\"consensus\":[\"N\"],\"martin_comments\":[\"only targets linked\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression levels of TFDP1 and E2F1 correlated with those of seven transcriptional targets ( TYMS, DHFR, PCNA, RRM1, CCNE1, CDC2, and MYBL2) that play important roles in the G1/S transition, and down-regulation of TFDP1 inhibited growth of Hep3B cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "E-cadherin-problem---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"E-cadherin-problem\"],\"sentence\":[\"The tissue closure defects in Grhl2 mutants are similar to that of AP-2?? null mutants and AP-2?? has been shown to bind to the promoter of E-cadherin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|the wrong cadherin (L)---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TFAP2A\"],\"tg (associated gene name)\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"the wrong cadherin (L)\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The tissue closure defects in Grhl2 mutants are similar to that of AP-2 null mutants and AP-2 has been shown to bind to the promoter of E-cadherin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "E-cadherin-problem---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"E-cadherin-problem\"],\"sentence\":[\"The tissue closure defects in Grhl2 mutants are similar to that of AP-2?? null mutants and AP-2?? has been shown to bind to the promoter of E-cadherin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|the wrong cadherin (L)---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TFAP2A\"],\"tg (associated gene name)\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"the wrong cadherin (L)\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The tissue closure defects in Grhl2 mutants are similar to that of AP-2 null mutants and AP-2 has been shown to bind to the promoter of E-cadherin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 359 (char 358)\n",
      "{\"tf associated gene name\":[\"T\"],\"tg associated gene name\":[\"TAT\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Moreover\",\"enforced expression of cyclin T2 inhibits cyclin T1-mediated enhancement of Tat in rodent cells and it represses Tat activity in human cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|cyclin T1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"T\"],\"tg (associated gene name)\":[\"TAT\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"cyclin T1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, enforced expression of cyclin T2 inhibits cyclin T1-mediated enhancement of Tat in rodent cells and it represses Tat activity in human cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"T\"],\"tg associated gene name\":[\"TAT\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Moreover\",\"enforced expression of cyclin T2 inhibits cyclin T1-mediated enhancement of Tat in rodent cells and it represses Tat activity in human cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|cyclin T1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"T\"],\"tg (associated gene name)\":[\"TAT\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"cyclin T1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, enforced expression of cyclin T2 inhibits cyclin T1-mediated enhancement of Tat in rodent cells and it represses Tat activity in human cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 440 (char 439)\n",
      "{\"tf associated gene name\":[\"SMAD4\"],\"tg associated gene name\":[\"CDK2\"],\"tf text\":[\"Smad4\"],\"tg text\":[\"CDK2\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Moreover\",\"CDK2 is the predominant CDK that phosphorylates Smad2 on T8 in myeloma cells\",\"leading to inhibition of Smad2-Smad4 association that precludes transcriptional regulation by Smad2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG not specified---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD4\"],\"tg (associated gene name)\":[\"CDK2\"],\"tf text\":[\"Smad4\"],\"tg text\":[\"CDK2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG not specified\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, CDK2 is the predominant CDK that phosphorylates Smad2 on T8 in myeloma cells, leading to inhibition of Smad2-Smad4 association that precludes transcriptional regulation by Smad2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SMAD4\"],\"tg associated gene name\":[\"CDK2\"],\"tf text\":[\"Smad4\"],\"tg text\":[\"CDK2\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Moreover\",\"CDK2 is the predominant CDK that phosphorylates Smad2 on T8 in myeloma cells\",\"leading to inhibition of Smad2-Smad4 association that precludes transcriptional regulation by Smad2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG not specified---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD4\"],\"tg (associated gene name)\":[\"CDK2\"],\"tf text\":[\"Smad4\"],\"tg text\":[\"CDK2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG not specified\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, CDK2 is the predominant CDK that phosphorylates Smad2 on T8 in myeloma cells, leading to inhibition of Smad2-Smad4 association that precludes transcriptional regulation by Smad2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is NAC (TF?)|Nrf2 is target (protein expression regulated)---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"NLRP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"regulator is NAC (TF?)\",\"Nrf2 is target (protein expression regulated)\"],\"sentence\":[\"We then identified seven putative Nrf2 antioxidant response elements on the CD36 gene promoter and showed that NAC inhibited curcumin-induced Nrf2 protein expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|cd36 TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"NLRP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"cd36 TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We then identified seven putative Nrf2 antioxidant response elements on the CD36 gene promoter and showed that NAC inhibited curcumin-induced Nrf2 protein expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "regulator is NAC (TF?)|Nrf2 is target (protein expression regulated)---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"NLRP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"regulator is NAC (TF?)\",\"Nrf2 is target (protein expression regulated)\"],\"sentence\":[\"We then identified seven putative Nrf2 antioxidant response elements on the CD36 gene promoter and showed that NAC inhibited curcumin-induced Nrf2 protein expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|cd36 TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"NLRP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"cd36 TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We then identified seven putative Nrf2 antioxidant response elements on the CD36 gene promoter and showed that NAC inhibited curcumin-induced Nrf2 protein expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ER-problem; other TFs other TGs---{\"tf associated gene name\":[\"ESR2\"],\"tg associated gene name\":[\"EZR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"ER-problem; other TFs other TGs\"],\"sentence\":[\"nuc-ErbB3 regulates the transcriptional activity of ezrin and HMGB1 promoters\",\"whereas inhibition of nuc-ErbB3 expression results in reduced myelination and altered distribution of ezrin in the nodes of Ranvier.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF correctly normalised?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR2\"],\"tg (associated gene name)\":[\"EZR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF correctly normalised?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"nuc-ErbB3 regulates the transcriptional activity of ezrin and HMGB1 promoters, whereas inhibition of nuc-ErbB3 expression results in reduced myelination and altered distribution of ezrin in the nodes of Ranvier.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "ER-problem; other TFs other TGs---{\"tf associated gene name\":[\"ESR2\"],\"tg associated gene name\":[\"EZR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"ER-problem; other TFs other TGs\"],\"sentence\":[\"nuc-ErbB3 regulates the transcriptional activity of ezrin and HMGB1 promoters\",\"whereas inhibition of nuc-ErbB3 expression results in reduced myelination and altered distribution of ezrin in the nodes of Ranvier.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF correctly normalised?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR2\"],\"tg (associated gene name)\":[\"EZR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF correctly normalised?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"nuc-ErbB3 regulates the transcriptional activity of ezrin and HMGB1 promoters, whereas inhibition of nuc-ErbB3 expression results in reduced myelination and altered distribution of ezrin in the nodes of Ranvier.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "SPI1 most likely not mentioned---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"PIAS1\"],\"tf text\":[],\"tg text\":[\"PIAS1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"SPI1 most likely not mentioned\"],\"sentence\":[\"RET/PTC-induced STAT1 transcriptional activation was not inhibited by suppressor of cytokine signaling-1 or -3\",\"or protein inhibitors of activated STAT3 [(protein inhibitor of activated STAT (PIAS3)]\",\"but PIAS1 strongly repressed the RET/PTC-induced transcriptional activity of STAT1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|relations flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"PIAS1\"],\"tf text\":[],\"tg text\":[\"PIAS1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"relations flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"RET/PTC-induced STAT1 transcriptional activation was not inhibited by suppressor of cytokine signaling-1 or -3, or protein inhibitors of activated STAT3 [(protein inhibitor of activated STAT (PIAS3)], but PIAS1 strongly repressed the RET/PTC-induced transcriptional activity of STAT1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "SPI1 most likely not mentioned---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"PIAS1\"],\"tf text\":[],\"tg text\":[\"PIAS1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"SPI1 most likely not mentioned\"],\"sentence\":[\"RET/PTC-induced STAT1 transcriptional activation was not inhibited by suppressor of cytokine signaling-1 or -3\",\"or protein inhibitors of activated STAT3 [(protein inhibitor of activated STAT (PIAS3)]\",\"but PIAS1 strongly repressed the RET/PTC-induced transcriptional activity of STAT1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|relations flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"PIAS1\"],\"tf text\":[],\"tg text\":[\"PIAS1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"relations flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"RET/PTC-induced STAT1 transcriptional activation was not inhibited by suppressor of cytokine signaling-1 or -3, or protein inhibitors of activated STAT3 [(protein inhibitor of activated STAT (PIAS3)], but PIAS1 strongly repressed the RET/PTC-induced transcriptional activity of STAT1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "epigenetic mechanism is stated (i.e. not TRE)---{\"tf associated gene name\":[\"GRHL2\"],\"tg associated gene name\":[\"TERT\"],\"tf text\":[\"GRHL2\"],\"tg text\":[\"hTERT\"],\"1\":[\"u?\"],\"negation\":[],\"not true\":[],\"comment\":[\"epigenetic mechanism is stated (i.e. not TRE)\"],\"sentence\":[\"These results indicate that GRHL2 regulates the hTERT expression through an epigenetic mechanism and controls the cellular life span.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "epigenetic mechanism is stated (i.e. not TRE)---{\"tf associated gene name\":[\"GRHL2\"],\"tg associated gene name\":[\"TERT\"],\"tf text\":[\"GRHL2\"],\"tg text\":[\"hTERT\"],\"1\":[\"u?\"],\"negation\":[],\"not true\":[],\"comment\":[\"epigenetic mechanism is stated (i.e. not TRE)\"],\"sentence\":[\"These results indicate that GRHL2 regulates the hTERT expression through an epigenetic mechanism and controls the cellular life span.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"GRHL2\"],\"target gene (associated gene name)\":[\"TERT\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results indicate that GRHL2 regulates the hTERT expression through an epigenetic mechanism and controls the cellular life span.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GRHL2\"],\"tg (associated gene name)\":[\"TERT\"],\"tf text\":[\"GRHL2\"],\"tg text\":[\"hTERT\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that GRHL2 regulates the hTERT expression through an epigenetic mechanism and controls the cellular life span.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 439 (char 438)\n",
      "{\"transcription factor (associated gene name)\":[\"NR3C1\"],\"target gene (associated gene name)\":[\"PLAU\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Our data suggest that inactivation of FOXO3a after GR activation is an important mechanism contributing to GC-mediated repression of uPA gene expression in breast epithelial and cancer cells.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|GR not clear as TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"PLAU\"],\"tf text\":[],\"tg text\":[\"uPA\"],\"consensus\":[\"V\"],\"martin_comments\":[\"GR not clear as TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our data suggest that inactivation of FOXO3a after GR activation is an important mechanism contributing to GC-mediated repression of uPA gene expression in breast epithelial and cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"PLAU\"],\"tf text\":[],\"tg text\":[\"uPA\"],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Our data suggest that inactivation of FOXO3a after GR activation is an important mechanism contributing to GC-mediated repression of uPA gene expression in breast epithelial and cancer cells.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR3C1\"],\"target gene (associated gene name)\":[\"PLAU\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"v\"],\"negation\":[],\"sentence\":[\"Our data suggest that inactivation of FOXO3a after GR activation is an important mechanism contributing to GC-mediated repression of uPA gene expression in breast epithelial and cancer cells.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.692105\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"191\"],\"sentence pair density\":[\"0.0129870129870129\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR3C1\"],\"target gene (associated gene name)\":[\"PLAU\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Our data suggest that inactivation of FOXO3a after GR activation is an important mechanism contributing to GC-mediated repression of uPA gene expression in breast epithelial and cancer cells.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|GR not clear as TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"PLAU\"],\"tf text\":[],\"tg text\":[\"uPA\"],\"consensus\":[\"V\"],\"martin_comments\":[\"GR not clear as TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our data suggest that inactivation of FOXO3a after GR activation is an important mechanism contributing to GC-mediated repression of uPA gene expression in breast epithelial and cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is E2---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"CCNA2\"],\"tf text\":[\"E2F-1\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is E2\"],\"sentence\":[\"The binding of E2F-1 to E2 promoter was found to be reduced by phosphorylation of E2F-1 by cyclin A-cdk2\",\"suggesting that phosphorylation of E2F-1 may induce shut off of gene expression at the transcriptional level.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|E2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"CCNA2\"],\"tf text\":[\"E2F-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"E2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The binding of E2F-1 to E2 promoter was found to be reduced by phosphorylation of E2F-1 by cyclin A-cdk2, suggesting that phosphorylation of E2F-1 may induce shut off of gene expression at the transcriptional level.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is E2---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"CCNA2\"],\"tf text\":[\"E2F-1\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is E2\"],\"sentence\":[\"The binding of E2F-1 to E2 promoter was found to be reduced by phosphorylation of E2F-1 by cyclin A-cdk2\",\"suggesting that phosphorylation of E2F-1 may induce shut off of gene expression at the transcriptional level.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|E2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"CCNA2\"],\"tf text\":[\"E2F-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"E2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The binding of E2F-1 to E2 promoter was found to be reduced by phosphorylation of E2F-1 by cyclin A-cdk2, suggesting that phosphorylation of E2F-1 may induce shut off of gene expression at the transcriptional level.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 443 (char 442)\n",
      "{\"tf associated gene name\":[\"SFPQ\"],\"tg associated gene name\":[\"PARK7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"It has been reported that DJ-1 is a neuroprotective transcriptional co-activator that sequesters a transcriptional co-repressor polypyrimidine tract-binding protein-associated splicing factor (PSF) from the TH gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG = TH---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SFPQ\"],\"tg (associated gene name)\":[\"PARK7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG = TH\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"It has been reported that DJ-1 is a neuroprotective transcriptional co-activator that sequesters a transcriptional co-repressor polypyrimidine tract-binding protein-associated splicing factor (PSF) from the TH gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SFPQ\"],\"tg associated gene name\":[\"PARK7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"It has been reported that DJ-1 is a neuroprotective transcriptional co-activator that sequesters a transcriptional co-repressor polypyrimidine tract-binding protein-associated splicing factor (PSF) from the TH gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG = TH---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SFPQ\"],\"tg (associated gene name)\":[\"PARK7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG = TH\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"It has been reported that DJ-1 is a neuroprotective transcriptional co-activator that sequesters a transcriptional co-repressor polypyrimidine tract-binding protein-associated splicing factor (PSF) from the TH gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p73---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"E2F1\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"E2F1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is p73\"],\"sentence\":[\"We identify p73 as an important E2F1 apoptotic target gene in DNA damage response and we show that acetylation is required for E2F1 recruitment on the P1p73 promoter and for its transcriptional activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|p73?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"E2F1\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"E2F1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"p73?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We identify p73 as an important E2F1 apoptotic target gene in DNA damage response and we show that acetylation is required for E2F1 recruitment on the P1p73 promoter and for its transcriptional activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is p73---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"E2F1\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"E2F1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is p73\"],\"sentence\":[\"We identify p73 as an important E2F1 apoptotic target gene in DNA damage response and we show that acetylation is required for E2F1 recruitment on the P1p73 promoter and for its transcriptional activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|p73?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"E2F1\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"E2F1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"p73?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We identify p73 as an important E2F1 apoptotic target gene in DNA damage response and we show that acetylation is required for E2F1 recruitment on the P1p73 promoter and for its transcriptional activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 228 (char 227)\n",
      "{\"transcription factor (associated gene name)\":[\"MLXIPL\"],\"target gene (associated gene name)\":[\"FAS\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|target proposed as TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIPL\"],\"tg (associated gene name)\":[\"FAS\"],\"tf text\":[\"ChREBP\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"target proposed as TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"MiR-613 dramatically suppressed the expression of LXR?? and its target genes including sterol-regulatory element binding protein 1c (SREBP-1c), fatty acid synthase (FAS), carbohydrate responsive element-binding protein (ChREBP) and acetyl-CoA carboxylase (ACC).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"MLXIPL\"],\"target gene (associated gene name)\":[\"FAS\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|target proposed as TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIPL\"],\"tg (associated gene name)\":[\"FAS\"],\"tf text\":[\"ChREBP\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"target proposed as TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"MiR-613 dramatically suppressed the expression of LXR?? and its target genes including sterol-regulatory element binding protein 1c (SREBP-1c), fatty acid synthase (FAS), carbohydrate responsive element-binding protein (ChREBP) and acetyl-CoA carboxylase (ACC).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is SP1---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"SREBF1\"],\"tf text\":[\"SREBP-1a\"],\"tg text\":[\"SREBP-1a\"],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"TF is SP1\"],\"sentence\":[\"The gene expression of SREBP-1a\",\"a predominant form of SREBP-1 in most cultured cells and a target of the general transcription factor Sp1\",\"is significantly augmented by glutamine via an increased binding of Sp1 to the SREBP-1a promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF=SP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"SREBF1\"],\"tf text\":[\"SREBP-1a\"],\"tg text\":[\"SREBP-1a\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=SP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The gene expression of SREBP-1a, a predominant form of SREBP-1 in most cultured cells and a target of the general transcription factor Sp1, is significantly augmented by glutamine via an increased binding of Sp1 to the SREBP-1a promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is SP1---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"SREBF1\"],\"tf text\":[\"SREBP-1a\"],\"tg text\":[\"SREBP-1a\"],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"TF is SP1\"],\"sentence\":[\"The gene expression of SREBP-1a\",\"a predominant form of SREBP-1 in most cultured cells and a target of the general transcription factor Sp1\",\"is significantly augmented by glutamine via an increased binding of Sp1 to the SREBP-1a promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF=SP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"SREBF1\"],\"tf text\":[\"SREBP-1a\"],\"tg text\":[\"SREBP-1a\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=SP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The gene expression of SREBP-1a, a predominant form of SREBP-1 in most cultured cells and a target of the general transcription factor Sp1, is significantly augmented by glutamine via an increased binding of Sp1 to the SREBP-1a promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 412 (char 411)\n",
      "{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"SENP8\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"SENP8\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of SENP8\",\"a NEDD8-specific cysteine protease\",\"resulted in deNEDDylation of E2F1 and promoted its transactivation activity at the p73 gene but not at the E2F2 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"SENP8\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"SENP8\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of SENP8, a NEDD8-specific cysteine protease, resulted in deNEDDylation of E2F1 and promoted its transactivation activity at the p73 gene but not at the E2F2 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"SENP8\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"SENP8\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of SENP8\",\"a NEDD8-specific cysteine protease\",\"resulted in deNEDDylation of E2F1 and promoted its transactivation activity at the p73 gene but not at the E2F2 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"SENP8\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"SENP8\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of SENP8, a NEDD8-specific cysteine protease, resulted in deNEDDylation of E2F1 and promoted its transactivation activity at the p73 gene but not at the E2F2 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is CHR-mutated promoter (maybe not normalizeable)---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"SGCG\"],\"tf text\":[\"E2F1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is CHR-mutated promoter (maybe not normalizeable)\"],\"sentence\":[\"The CHR-mutated promoter is activated by the E2F family of proteins (E2F1\",\"E2F2\",\"E2F3\",\"and E2F4) but unable to be repressed by any of the Rb family members (Rb\",\"p107\",\"and p130) to the level of the wild-type promoter activity in G0\",\"indicating that the repression by the Rb family members is required for the corepressor element.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is CHR-mutated promoter (maybe not normalizeable)---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"SGCG\"],\"tf text\":[\"E2F1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is CHR-mutated promoter (maybe not normalizeable)\"],\"sentence\":[\"The CHR-mutated promoter is activated by the E2F family of proteins (E2F1\",\"E2F2\",\"E2F3\",\"and E2F4) but unable to be repressed by any of the Rb family members (Rb\",\"p107\",\"and p130) to the level of the wild-type promoter activity in G0\",\"indicating that the repression by the Rb family members is required for the corepressor element.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"E2F1\"],\"target gene (associated gene name)\":[\"SGCG\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"The CHR-mutated promoter is activated by the E2F family of proteins (E2F1\",\"E2F2\",\"E2F3\",\"and E2F4) but unable to be repressed by any of the Rb family members (Rb\",\"p107\",\"and p130) to the level of the wild-type promoter activity in G0\",\"indicating that the repression by the Rb family members is required for the corepressor element.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.697113\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"20\"],\"sentence length\":[\"328\"],\"sentence pair density\":[\"0.00979020979020979\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CHR?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"SGCG\"],\"tf text\":[\"E2F1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CHR?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The CHR-mutated promoter is activated by the E2F family of proteins (E2F1, E2F2, E2F3, and E2F4) but unable to be repressed by any of the Rb family members (Rb, p107, and p130) to the level of the wild-type promoter activity in G0, indicating that the repression by the Rb family members is required for the corepressor element.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG probably should be normalize to IFNA1---{\"tf associated gene name\":[\"IRF7\"],\"tg associated gene name\":[\"IFN1@\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG probably should be normalize to IFNA1\"],\"sentence\":[\"In transient transfection assays with the IFNACAT reporter\",\"vIRF-3 functioned as a dominant-negative mutant of both IRF-3 and IRF-7 and inhibited virus-mediated transcriptional activity of the IFNA promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TG probably should be normalize to IFNA1---{\"tf associated gene name\":[\"IRF7\"],\"tg associated gene name\":[\"IFN1@\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG probably should be normalize to IFNA1\"],\"sentence\":[\"In transient transfection assays with the IFNACAT reporter\",\"vIRF-3 functioned as a dominant-negative mutant of both IRF-3 and IRF-7 and inhibited virus-mediated transcriptional activity of the IFNA promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor\":[\"IRF7\"],\"target gene\":[\"IFN1@\"],\"sentence\":[\"In transient transfection assays with the IFNACAT reporter\",\"vIRF-3 functioned as a dominant-negative mutant of both IRF-3 and IRF-7 and inhibited virus-mediated transcriptional activity of the IFNA promoter.\"],\"valid\":[\"1\"],\"sign\":[\" +\"],\"negation\":[\"0\"],\"fusion tf\":[],\"qc'ed\":[\"1\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[false],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"IRF7\"],\"target gene (associated gene name)\":[\"IFN1@\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In transient transfection assays with the IFNACAT reporter\",\"vIRF-3 functioned as a dominant-negative mutant of both IRF-3 and IRF-7 and inhibited virus-mediated transcriptional activity of the IFNA promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF7\"],\"tg (associated gene name)\":[\"IFN1@\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"In transient transfection assays with the IFNACAT reporter, vIRF-3 functioned as a dominant-negative mutant of both IRF-3 and IRF-7 and inhibited virus-mediated transcriptional activity of the IFNA promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "mapre1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"MAPRE1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"mapre1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Integrity of both EB1 and EB2 is required for Twist1-mediated transactivation of the Alx3 promoter, even though Twist1 does not bind to EB1, indicating that binding of USF1 and USF2 to this element is required for Twist1-dependent Alx3 promoter activity.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "mapre1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"MAPRE1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"mapre1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Integrity of both EB1 and EB2 is required for Twist1-mediated transactivation of the Alx3 promoter, even though Twist1 does not bind to EB1, indicating that binding of USF1 and USF2 to this element is required for Twist1-dependent Alx3 promoter activity.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 405 (char 404)\n",
      "{\"transcription factor (associated gene name)\":[\"BCL6\"],\"target gene (associated gene name)\":[\"IL5\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The exogenous Bcl6 repressed expression of the reporter gene with the IL5BS in K562 cells and the repressor activity was lost by a point mutation of the IL5BS.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|IL5BS not in GeneCards but...---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BCL6\"],\"tg (associated gene name)\":[\"IL5\"],\"tf text\":[\"Bcl6\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"IL5BS not in GeneCards but...\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The exogenous Bcl6 repressed expression of the reporter gene with the IL5BS in K562 cells and the repressor activity was lost by a point mutation of the IL5BS.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"BCL6\"],\"tg associated gene name\":[\"IL5\"],\"tf text\":[\"Bcl6\"],\"tg text\":[],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"The exogenous Bcl6 repressed expression of the reporter gene with the IL5BS in K562 cells and the repressor activity was lost by a point mutation of the IL5BS.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"BCL6\"],\"target gene (associated gene name)\":[\"IL5\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The exogenous Bcl6 repressed expression of the reporter gene with the IL5BS in K562 cells and the repressor activity was lost by a point mutation of the IL5BS.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|IL5BS not in GeneCards but...---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BCL6\"],\"tg (associated gene name)\":[\"IL5\"],\"tf text\":[\"Bcl6\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"IL5BS not in GeneCards but...\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The exogenous Bcl6 repressed expression of the reporter gene with the IL5BS in K562 cells and the repressor activity was lost by a point mutation of the IL5BS.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 416 (char 415)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"PTGS2\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Gastric adenocarcinoma cell lines derived from Caucasian (AGS) cells and Korean (SNU-484) cells were used to study the role of NF-kappaB on COX-2 expression by H. pylori.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|nCOX2 in pylori?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"PTGS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"nCOX2 in pylori?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gastric adenocarcinoma cell lines derived from Caucasian (AGS) cells and Korean (SNU-484) cells were used to study the role of NF-kappaB on COX-2 expression by H. pylori.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"PTGS2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"not true\":[],\"comment\":[],\"sentence\":[\"Gastric adenocarcinoma cell lines derived from Caucasian (AGS) cells and Korean (SNU-484) cells were used to study the role of NF-kappaB on COX-2 expression by H. pylori.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"PTGS2\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Gastric adenocarcinoma cell lines derived from Caucasian (AGS) cells and Korean (SNU-484) cells were used to study the role of NF-kappaB on COX-2 expression by H. pylori.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|nCOX2 in pylori?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"PTGS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"nCOX2 in pylori?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gastric adenocarcinoma cell lines derived from Caucasian (AGS) cells and Korean (SNU-484) cells were used to study the role of NF-kappaB on COX-2 expression by H. pylori.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The focal adhesion dependence of IL-1-induced ERK activation and c-fos expression could be circumvented in cells plated on poly-L-lysine by simultaneous incubation with IL-1 and the calcium ionophore ionomycin.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The focal adhesion dependence of IL-1-induced ERK activation and c-fos expression could be circumvented in cells plated on poly-L-lysine by simultaneous incubation with IL-1 and the calcium ionophore ionomycin.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs are hCGalpha and hCGbeta---{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"CGB\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r-fam\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TGs are hCGalpha and hCGbeta\"],\"sentence\":[\"Oct-3/4 is therefore capable of silencing both hCGalpha and hCGbeta gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TGs are hCGalpha and hCGbeta---{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"CGB\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r-fam\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TGs are hCGalpha and hCGbeta\"],\"sentence\":[\"Oct-3/4 is therefore capable of silencing both hCGalpha and hCGbeta gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"CGB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Oct-3/4 is therefore capable of silencing both hCGalpha and hCGbeta gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "INDEPENDENT OF dna-INTERACTION---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HAND2\"],\"tg (associated gene name)\":[\"DBH\"],\"tf text\":[],\"tg text\":[\"DBH\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[\"INDEPENDENT OF dna-INTERACTION\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These results indicate that the activation of the DBH promoter by Arix is potentiated by dHAND via a mechanism independent of a direct interaction of dHAND with DNA.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "INDEPENDENT OF dna-INTERACTION---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HAND2\"],\"tg (associated gene name)\":[\"DBH\"],\"tf text\":[],\"tg text\":[\"DBH\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[\"INDEPENDENT OF dna-INTERACTION\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These results indicate that the activation of the DBH promoter by Arix is potentiated by dHAND via a mechanism independent of a direct interaction of dHAND with DNA.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 375 (char 374)\n",
      "{\"tf associated gene name\":[\"FOXO1\"],\"tg associated gene name\":[\"AKT1\"],\"tf text\":[\"FKHR\"],\"tg text\":[\"Akt\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"These results suggest that FKHR may be a direct nuclear regulatory target for Akt in both metabolic and cell survival pathways.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|role akt?|UPSTREAM---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO1\"],\"tg (associated gene name)\":[\"AKT1\"],\"tf text\":[\"FKHR\"],\"tg text\":[\"Akt\"],\"consensus\":[\"N\"],\"martin_comments\":[\"role akt?\"],\"astrid_comments\":[\"UPSTREAM\"],\"sentences with greek letters\":[\"These results suggest that FKHR may be a direct nuclear regulatory target for Akt in both metabolic and cell survival pathways.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOXO1\"],\"tg associated gene name\":[\"AKT1\"],\"tf text\":[\"FKHR\"],\"tg text\":[\"Akt\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"These results suggest that FKHR may be a direct nuclear regulatory target for Akt in both metabolic and cell survival pathways.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|role akt?|UPSTREAM---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO1\"],\"tg (associated gene name)\":[\"AKT1\"],\"tf text\":[\"FKHR\"],\"tg text\":[\"Akt\"],\"consensus\":[\"N\"],\"martin_comments\":[\"role akt?\"],\"astrid_comments\":[\"UPSTREAM\"],\"sentences with greek letters\":[\"These results suggest that FKHR may be a direct nuclear regulatory target for Akt in both metabolic and cell survival pathways.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "AR=TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"SIN3A\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AR=TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated, p/CAF-containing coactivator assembly for a p53-interacting, Groucho/TLE1-, and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "AR=TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"SIN3A\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AR=TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated, p/CAF-containing coactivator assembly for a p53-interacting, Groucho/TLE1-, and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 344 (char 343)\n",
      "{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"CDC25A\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"cdc25A\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Chromatin immunoprecipitations showed HPV16 E7 increases histone acetylation on the E2F1 and cdc25A promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"CDC25A\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"cdc25A\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Chromatin immunoprecipitations showed HPV16 E7 increases histone acetylation on the E2F1 and cdc25A promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"CDC25A\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"cdc25A\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Chromatin immunoprecipitations showed HPV16 E7 increases histone acetylation on the E2F1 and cdc25A promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"CDC25A\"],\"tf text\":[\"E2F1\"],\"tg text\":[\"cdc25A\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Chromatin immunoprecipitations showed HPV16 E7 increases histone acetylation on the E2F1 and cdc25A promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 314 (char 313)\n",
      "{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"TF\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Functional studies indicate that Sp1 controls basal TF gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|TF=TG :-)---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"TF\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"TF=TG :-)\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Functional studies indicate that Sp1 controls basal TF gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"TF\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Functional studies indicate that Sp1 controls basal TF gene expression.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"TF\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Functional studies indicate that Sp1 controls basal TF gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|TF=TG :-)---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"TF\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"TF=TG :-)\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Functional studies indicate that Sp1 controls basal TF gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 531 (char 530)\n",
      "{\"tf associated gene name\":[\"GFI1\"],\"tg associated gene name\":[\"GFI1\"],\"tf text\":[\"Gfi1\"],\"tg text\":[\"Gfi1\"],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"p\"],\"comment\":[],\"sentence\":[\"We show that upon LPS stimulation\",\"expression of Gfi1 is induced with kinetics similar to those of nuclear translocation of p65 and that Gfi1 interacts with p65 and inhibits p65-mediated transcriptional transactivation by interfering with p65 binding to target gene promoter DNA.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GFI1\"],\"tg (associated gene name)\":[\"GFI1\"],\"tf text\":[\"Gfi1\"],\"tg text\":[\"Gfi1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that upon LPS stimulation, expression of Gfi1 is induced with kinetics similar to those of nuclear translocation of p65 and that Gfi1 interacts with p65 and inhibits p65-mediated transcriptional transactivation by interfering with p65 binding to target gene promoter DNA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"GFI1\"],\"tg associated gene name\":[\"GFI1\"],\"tf text\":[\"Gfi1\"],\"tg text\":[\"Gfi1\"],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"p\"],\"comment\":[],\"sentence\":[\"We show that upon LPS stimulation\",\"expression of Gfi1 is induced with kinetics similar to those of nuclear translocation of p65 and that Gfi1 interacts with p65 and inhibits p65-mediated transcriptional transactivation by interfering with p65 binding to target gene promoter DNA.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GFI1\"],\"tg (associated gene name)\":[\"GFI1\"],\"tf text\":[\"Gfi1\"],\"tg text\":[\"Gfi1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that upon LPS stimulation, expression of Gfi1 is induced with kinetics similar to those of nuclear translocation of p65 and that Gfi1 interacts with p65 and inhibits p65-mediated transcriptional transactivation by interfering with p65 binding to target gene promoter DNA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF most likely HNF4-alpha---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"DST\"],\"tf text\":[\"IRF1\"],\"tg text\":[\"BPAG1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF most likely HNF4-alpha\"],\"sentence\":[\"Transient transfection studies with BPAG1 promoter-luciferase reporter gene plasmids and IRF1 and IRF2 expression plasmids revealed that IRF1 and IRF2 directly down-regulated BPAG1 gene transcription in cultured normal human epidermal keratinocytes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF most likely HNF4-alpha---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"DST\"],\"tf text\":[\"IRF1\"],\"tg text\":[\"BPAG1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF most likely HNF4-alpha\"],\"sentence\":[\"Transient transfection studies with BPAG1 promoter-luciferase reporter gene plasmids and IRF1 and IRF2 expression plasmids revealed that IRF1 and IRF2 directly down-regulated BPAG1 gene transcription in cultured normal human epidermal keratinocytes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF1\"],\"target gene (associated gene name)\":[\"DST\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Transient transfection studies with BPAG1 promoter-luciferase reporter gene plasmids and IRF1 and IRF2 expression plasmids revealed that IRF1 and IRF2 directly down-regulated BPAG1 gene transcription in cultured normal human epidermal keratinocytes.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"DST\"],\"tf text\":[\"IRF1\"],\"tg text\":[\"BPAG1\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transient transfection studies with BPAG1 promoter-luciferase reporter gene plasmids and IRF1 and IRF2 expression plasmids revealed that IRF1 and IRF2 directly down-regulated BPAG1 gene transcription in cultured normal human epidermal keratinocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is IFN-beta---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"IFNA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is IFN-beta\"],\"sentence\":[\"Overexpression of YY1 strongly repressed the transcriptional capacity of a stably integrated IFN-beta promoter fused to a chloramphenicol acetyltransferase reporter gene as well as the endogenous IFN activity of murine L929 cells via an HDAC activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is IFN-beta---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"IFNA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is IFN-beta\"],\"sentence\":[\"Overexpression of YY1 strongly repressed the transcriptional capacity of a stably integrated IFN-beta promoter fused to a chloramphenicol acetyltransferase reporter gene as well as the endogenous IFN activity of murine L929 cells via an HDAC activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"IFNA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of YY1 strongly repressed the transcriptional capacity of a stably integrated IFN-beta promoter fused to a chloramphenicol acetyltransferase reporter gene as well as the endogenous IFN activity of murine L929 cells via an HDAC activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is SREBP-1 (for TG ACACA); SREBF2 is TF for HMG reductase---{\"tf associated gene name\":[\"SREBF2\"],\"tg associated gene name\":[\"ACACA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is SREBP-1 (for TG ACACA); SREBF2 is TF for HMG reductase\"],\"sentence\":[\"Addition of 25HC3S to primary rat hepatocytes decreased nuclear LXR and SREBP-1 protein levels\",\"down-regulated their target genes\",\"acetyl CoA carboxylase 1 (ACC1)\",\"fatty acid synthase (FAS)\",\"and SREBP-2 target gene HMG reductase\",\"key enzymes involved in fatty acid and cholesterol biosynthesis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is SREBP-1 (for TG ACACA); SREBF2 is TF for HMG reductase---{\"tf associated gene name\":[\"SREBF2\"],\"tg associated gene name\":[\"ACACA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is SREBP-1 (for TG ACACA); SREBF2 is TF for HMG reductase\"],\"sentence\":[\"Addition of 25HC3S to primary rat hepatocytes decreased nuclear LXR and SREBP-1 protein levels\",\"down-regulated their target genes\",\"acetyl CoA carboxylase 1 (ACC1)\",\"fatty acid synthase (FAS)\",\"and SREBP-2 target gene HMG reductase\",\"key enzymes involved in fatty acid and cholesterol biosynthesis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SREBF2\"],\"target gene (associated gene name)\":[\"ACACA\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Addition of 25HC3S to primary rat hepatocytes decreased nuclear LXR and SREBP-1 protein levels\",\"down-regulated their target genes\",\"acetyl CoA carboxylase 1 (ACC1)\",\"fatty acid synthase (FAS)\",\"and SREBP-2 target gene HMG reductase\",\"key enzymes involved in fatty acid and cholesterol biosynthesis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF2\"],\"tg (associated gene name)\":[\"ACACA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Addition of 25HC3S to primary rat hepatocytes decreased nuclear LXR and SREBP-1 protein levels, down-regulated their target genes, acetyl CoA carboxylase 1 (ACC1), fatty acid synthase (FAS), and SREBP-2 target gene HMG reductase, key enzymes involved in fatty acid and cholesterol biosynthesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 497 (char 496)\n",
      "{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"WDR5\"],\"tf text\":[],\"tg text\":[\"WDR5\"],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Using ChIP-seq\",\"we find that androgen stimulation results in a 6-fold increase in the number of H3T11P-marked regions and induces WDR5 colocalization to one third of H3T11P-enriched promoters\",\"thus establishing a genome-wide relationship between H3T11P and recruitment of WDR5.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"WDR5\"],\"tf text\":[],\"tg text\":[\"WDR5\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using ChIP-seq, we find that androgen stimulation results in a 6-fold increase in the number of H3T11P-marked regions and induces WDR5 colocalization to one third of H3T11P-enriched promoters, thus establishing a genome-wide relationship between H3T11P and recruitment of WDR5.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"WDR5\"],\"tf text\":[],\"tg text\":[\"WDR5\"],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Using ChIP-seq\",\"we find that androgen stimulation results in a 6-fold increase in the number of H3T11P-marked regions and induces WDR5 colocalization to one third of H3T11P-enriched promoters\",\"thus establishing a genome-wide relationship between H3T11P and recruitment of WDR5.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"WDR5\"],\"tf text\":[],\"tg text\":[\"WDR5\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using ChIP-seq, we find that androgen stimulation results in a 6-fold increase in the number of H3T11P-marked regions and induces WDR5 colocalization to one third of H3T11P-enriched promoters, thus establishing a genome-wide relationship between H3T11P and recruitment of WDR5.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Il17---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"AHR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is Il17\"],\"sentence\":[\"Further analysis revealed that Srebp-1\",\"which is encoded by an LXR target gene\",\"mediated the suppression of Th17 differentiation by binding to the E-box element on the Il17 promoter\",\"physically interacting with aryl hydrocarbon receptor (Ahr) and inhibiting Ahr-controlled Il17 transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"AHR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Further analysis revealed that Srebp-1, which is encoded by an LXR target gene, mediated the suppression of Th17 differentiation by binding to the E-box element on the Il17 promoter, physically interacting with aryl hydrocarbon receptor (Ahr) and inhibiting Ahr-controlled Il17 transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Il17---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"AHR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is Il17\"],\"sentence\":[\"Further analysis revealed that Srebp-1\",\"which is encoded by an LXR target gene\",\"mediated the suppression of Th17 differentiation by binding to the E-box element on the Il17 promoter\",\"physically interacting with aryl hydrocarbon receptor (Ahr) and inhibiting Ahr-controlled Il17 transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"AHR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Further analysis revealed that Srebp-1, which is encoded by an LXR target gene, mediated the suppression of Th17 differentiation by binding to the E-box element on the Il17 promoter, physically interacting with aryl hydrocarbon receptor (Ahr) and inhibiting Ahr-controlled Il17 transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "avian c-myc---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[\"progesterone receptor\"],\"tg text\":[\"c-myc\"],\"1\":[\"u\"],\"0\":[\"s?\"],\"negation\":[],\"comment\":[\"avian c-myc\"],\"sentence\":[\"A nuclear matrix acceptor site for the progesterone receptor in the avian c-myc gene promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "avian c-myc---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[\"progesterone receptor\"],\"tg text\":[\"c-myc\"],\"1\":[\"u\"],\"0\":[\"s?\"],\"negation\":[],\"comment\":[\"avian c-myc\"],\"sentence\":[\"A nuclear matrix acceptor site for the progesterone receptor in the avian c-myc gene promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PGR\"],\"target gene (associated gene name)\":[\"MYC\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"A nuclear matrix acceptor site for the progesterone receptor in the avian c-myc gene promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|AVIAN---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[\"progesterone receptor\"],\"tg text\":[\"c-myc\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[\"AVIAN\"],\"sentences with greek letters\":[\"A nuclear matrix acceptor site for the progesterone receptor in the avian c-myc gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GRE|but TF/TG not identifyable/named---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"CASP8AP2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"GRE\",\"but TF/TG not identifyable/named\"],\"sentence\":[\"Supporting data include demonstration that GnRH increases activity of TOPflash\",\"a TCF/LEF-dependent luciferase reporter\",\"in LbetaT2 cells\",\"a gonadotrope-derived cell line.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF4A\"],\"tg (associated gene name)\":[\"CASP8AP2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Supporting data include demonstration that GnRH increases activity of TOPflash, a TCF/LEF-dependent luciferase reporter, in LbetaT2 cells, a gonadotrope-derived cell line.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "GRE|but TF/TG not identifyable/named---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"CASP8AP2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"GRE\",\"but TF/TG not identifyable/named\"],\"sentence\":[\"Supporting data include demonstration that GnRH increases activity of TOPflash\",\"a TCF/LEF-dependent luciferase reporter\",\"in LbetaT2 cells\",\"a gonadotrope-derived cell line.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF4A\"],\"tg (associated gene name)\":[\"CASP8AP2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Supporting data include demonstration that GnRH increases activity of TOPflash, a TCF/LEF-dependent luciferase reporter, in LbetaT2 cells, a gonadotrope-derived cell line.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is glucocorticoid (GR)---{\"tf associated gene name\":[\"FOXM1\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[],\"tg text\":[\"insulin\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is glucocorticoid (GR)\"],\"sentence\":[\"Here\",\"we used INS-1 cells to test the hypothesis that PGC-1alpha regulates human insulin gene transcription by modulating glucocorticoid (GR) binding to the insulin gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is glucocorticoid (GR)---{\"tf associated gene name\":[\"FOXM1\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[],\"tg text\":[\"insulin\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is glucocorticoid (GR)\"],\"sentence\":[\"Here\",\"we used INS-1 cells to test the hypothesis that PGC-1alpha regulates human insulin gene transcription by modulating glucocorticoid (GR) binding to the insulin gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXM1\"],\"target gene (associated gene name)\":[\"INS\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Here\",\"we used INS-1 cells to test the hypothesis that PGC-1alpha regulates human insulin gene transcription by modulating glucocorticoid (GR) binding to the insulin gene promoter.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.030725\"],\"pmid counts\":[\"18\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"179\"],\"sentence pair density\":[\"0.015267175572519\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXM1\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[],\"tg text\":[\"insulin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here, we used INS-1 cells to test the hypothesis that PGC-1alpha regulates human insulin gene transcription by modulating glucocorticoid (GR) binding to the insulin gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "JUN is TG|unsure about TF---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"JUN is TG\",\"unsure about TF\"],\"sentence\":[\"The extent of transcriptional activation of the fos\",\"jun\",\"and HO genes in M1 cells treated with TPA was reduced by a specific inhibitor of C-kinase and a scavenger of oxygen radicals.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "JUN is TG|unsure about TF---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"JUN is TG\",\"unsure about TF\"],\"sentence\":[\"The extent of transcriptional activation of the fos\",\"jun\",\"and HO genes in M1 cells treated with TPA was reduced by a specific inhibitor of C-kinase and a scavenger of oxygen radicals.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"The extent of transcriptional activation of the fos, jun, and HO genes in M1 cells treated with TPA was reduced by a specific inhibitor of C-kinase and a scavenger of oxygen radicals.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "DAC treatment|TF not named---{\"tf associated gene name\":[\"HES1\"],\"tg associated gene name\":[\"TERT\"],\"tf text\":[],\"tg text\":[\"TERT\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"DAC treatment\",\"TF not named\"],\"sentence\":[\"DAC treatment reduced TERT expression and telomerase activity in OCI/AML3 and HL-60 cells\",\"but not in U937 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HES1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HES1\"],\"tg (associated gene name)\":[\"TERT\"],\"tf text\":[],\"tg text\":[\"TERT\"],\"consensus\":[\"N\"],\"martin_comments\":[\"HES1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"DAC treatment reduced TERT expression and telomerase activity in OCI/AML3 and HL-60 cells, but not in U937 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "DAC treatment|TF not named---{\"tf associated gene name\":[\"HES1\"],\"tg associated gene name\":[\"TERT\"],\"tf text\":[],\"tg text\":[\"TERT\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"DAC treatment\",\"TF not named\"],\"sentence\":[\"DAC treatment reduced TERT expression and telomerase activity in OCI/AML3 and HL-60 cells\",\"but not in U937 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HES1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HES1\"],\"tg (associated gene name)\":[\"TERT\"],\"tf text\":[],\"tg text\":[\"TERT\"],\"consensus\":[\"N\"],\"martin_comments\":[\"HES1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"DAC treatment reduced TERT expression and telomerase activity in OCI/AML3 and HL-60 cells, but not in U937 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "doesn't look like a TRE|TG not found---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"doesn't look like a TRE\",\"TG not found\"],\"sentence\":[\"p53-null mice exhibit developmentally delayed AFP repression\",\"concomitant with acetylation of H3K9\",\"methylation of H3K4\",\"and loss of DiMetH3K9\",\"mSin3A/HDAC1\",\"and HP1 interactions.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"p53-null mice exhibit developmentally delayed AFP repression, concomitant with acetylation of H3K9, methylation of H3K4, and loss of DiMetH3K9, mSin3A/HDAC1, and HP1 interactions.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "doesn't look like a TRE|TG not found---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"doesn't look like a TRE\",\"TG not found\"],\"sentence\":[\"p53-null mice exhibit developmentally delayed AFP repression\",\"concomitant with acetylation of H3K9\",\"methylation of H3K4\",\"and loss of DiMetH3K9\",\"mSin3A/HDAC1\",\"and HP1 interactions.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"p53-null mice exhibit developmentally delayed AFP repression, concomitant with acetylation of H3K9, methylation of H3K4, and loss of DiMetH3K9, mSin3A/HDAC1, and HP1 interactions.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Runx1|TGFB1 is upstream regulator|SPI1 not mentioned---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\",\"n\"],\"negation\":[],\"comment\":[\"TG is Runx1\",\"TGFB1 is upstream regulator\",\"SPI1 not mentioned\"],\"sentence\":[\"Gene array results revealed that Runx1\",\"a member of the Runx family of transcription factors\",\"was induced by TGFbeta\",\"and this induction was confirmed at the mRNA and protein levels.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Runx1|TGFB1 is upstream regulator|SPI1 not mentioned---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\",\"n\"],\"negation\":[],\"comment\":[\"TG is Runx1\",\"TGFB1 is upstream regulator\",\"SPI1 not mentioned\"],\"sentence\":[\"Gene array results revealed that Runx1\",\"a member of the Runx family of transcription factors\",\"was induced by TGFbeta\",\"and this induction was confirmed at the mRNA and protein levels.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SPI1\"],\"target gene (associated gene name)\":[\"TGFB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Gene array results revealed that Runx1\",\"a member of the Runx family of transcription factors\",\"was induced by TGFbeta\",\"and this induction was confirmed at the mRNA and protein levels.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.865709\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"182\"],\"sentence pair density\":[\"0.0198019801980198\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gene array results revealed that Runx1, a member of the Runx family of transcription factors, was induced by TGFbeta, and this induction was confirmed at the mRNA and protein levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CAT-problem---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"CAT\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"1\":[\"r\"],\"not true\":[],\"comment\":[\"CAT-problem\"],\"sentence\":[\"The mRNA level of hsp90 alpha gene in Jurkat cells was decreased when transfected by STAT1 expression construct\",\"over-expression of STAT1 down-regulates the expression of CAT reporter gene with the present of a distal fragment from -1756 to -1463 within the 5' flanking regulatory sequences of hsp90 alpha gene.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "CAT-problem---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"CAT\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"1\":[\"r\"],\"not true\":[],\"comment\":[\"CAT-problem\"],\"sentence\":[\"The mRNA level of hsp90 alpha gene in Jurkat cells was decreased when transfected by STAT1 expression construct\",\"over-expression of STAT1 down-regulates the expression of CAT reporter gene with the present of a distal fragment from -1756 to -1463 within the 5' flanking regulatory sequences of hsp90 alpha gene.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"STAT1\"],\"target gene (associated gene name)\":[\"CAT\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The mRNA level of hsp90 alpha gene in Jurkat cells was decreased when transfected by STAT1 expression construct\",\"over-expression of STAT1 down-regulates the expression of CAT reporter gene with the present of a distal fragment from -1756 to -1463 within the 5' flanking regulatory sequences of hsp90 alpha gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TG|CAT-REPORTER---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"CAT\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[\"CAT-REPORTER\"],\"sentences with greek letters\":[\"The mRNA level of hsp90 alpha gene in Jurkat cells was decreased when transfected by STAT1 expression construct, over-expression of STAT1 down-regulates the expression of CAT reporter gene with the present of a distal fragment from -1756 to -1463 within the 5' flanking regulatory sequences of hsp90 alpha gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain TG normalization---{\"tf associated gene name\":[\"MYCN\"],\"tg associated gene name\":[\"GCLC\"],\"tf text\":[\"MYCN\"],\"tg text\":[\"GCL\"],\"1\":[\"a?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain TG normalization\"],\"sentence\":[\"Furthermore\",\"we found that MYCN directly binds to an E-box containing GCL(cat) promoter and that over-expression of MYCN in MYCN-non-amplified cells stimulated GCL(cat) expression and provided resistance to oxidative damage; whereas knock down of MYCN in MYCN-amplified cells decreased GCL(cat) expression and sensitized them to oxidative damage.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "uncertain TG normalization---{\"tf associated gene name\":[\"MYCN\"],\"tg associated gene name\":[\"GCLC\"],\"tf text\":[\"MYCN\"],\"tg text\":[\"GCL\"],\"1\":[\"a?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain TG normalization\"],\"sentence\":[\"Furthermore\",\"we found that MYCN directly binds to an E-box containing GCL(cat) promoter and that over-expression of MYCN in MYCN-non-amplified cells stimulated GCL(cat) expression and provided resistance to oxidative damage; whereas knock down of MYCN in MYCN-amplified cells decreased GCL(cat) expression and sensitized them to oxidative damage.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MYCN\"],\"target gene (associated gene name)\":[\"GCLC\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Furthermore\",\"we found that MYCN directly binds to an E-box containing GCL(cat) promoter and that over-expression of MYCN in MYCN-non-amplified cells stimulated GCL(cat) expression and provided resistance to oxidative damage; whereas knock down of MYCN in MYCN-amplified cells decreased GCL(cat) expression and sensitized them to oxidative damage.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYCN\"],\"tg (associated gene name)\":[\"GCLC\"],\"tf text\":[\"MYCN\"],\"tg text\":[\"GCL\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, we found that MYCN directly binds to an E-box containing GCL(cat) promoter and that over-expression of MYCN in MYCN-non-amplified cells stimulated GCL(cat) expression and provided resistance to oxidative damage; whereas knock down of MYCN in MYCN-amplified cells decreased GCL(cat) expression and sensitized them to oxidative damage.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no GRE stated---{\"tf associated gene name\":[\"FOSB\"],\"tg associated gene name\":[\"CCR4\"],\"tf text\":[],\"tg text\":[\"CCR4\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no GRE stated\"],\"sentence\":[\"Thus\",\"Fra-2 is a novel oncogene of ATL\",\"and CCR4 may be regarded as a useful tumor marker of ATL.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no GRE stated---{\"tf associated gene name\":[\"FOSB\"],\"tg associated gene name\":[\"CCR4\"],\"tf text\":[],\"tg text\":[\"CCR4\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no GRE stated\"],\"sentence\":[\"Thus\",\"Fra-2 is a novel oncogene of ATL\",\"and CCR4 may be regarded as a useful tumor marker of ATL.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"CCR4\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[\"Thus\",\"Fra-2 is a novel oncogene of ATL\",\"and CCR4 may be regarded as a useful tumor marker of ATL.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"CCR4\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Thus\",\"Fra-2 is a novel oncogene of ATL\",\"and CCR4 may be regarded as a useful tumor marker of ATL.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.11594\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"97\"],\"sentence pair density\":[\"0.0091743119266055\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no GR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOSB\"],\"tg (associated gene name)\":[\"CCR4\"],\"tf text\":[],\"tg text\":[\"CCR4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no GR\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, Fra-2 is a novel oncogene of ATL, and CCR4 may be regarded as a useful tumor marker of ATL.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 381 (char 380)\n",
      "{\"transcription factor (associated gene name)\":[\"ETV1\"],\"target gene (associated gene name)\":[\"DYM\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"ERM and ER81\",\"the other two members of the PEA3 subfamily of transcription factors\",\"could not transactivate the Muc4/SMC promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETV1\"],\"tg (associated gene name)\":[\"DYM\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"ERM and ER81, the other two members of the PEA3 subfamily of transcription factors, could not transactivate the Muc4/SMC promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ETV1\"],\"tg associated gene name\":[\"DYM\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[\"x\"],\"comment\":[],\"sentence\":[\"ERM and ER81\",\"the other two members of the PEA3 subfamily of transcription factors\",\"could not transactivate the Muc4/SMC promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ETV1\"],\"target gene (associated gene name)\":[\"DYM\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"ERM and ER81\",\"the other two members of the PEA3 subfamily of transcription factors\",\"could not transactivate the Muc4/SMC promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETV1\"],\"tg (associated gene name)\":[\"DYM\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"ERM and ER81, the other two members of the PEA3 subfamily of transcription factors, could not transactivate the Muc4/SMC promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 371 (char 370)\n",
      "{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"ABL1\"],\"tf text\":[\"STAT5\"],\"tg text\":[\"Bcr-Abl\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Moreover\",\"the cyclin D1 and bcl-x promoters contain STAT binding sites to which STAT5 constitutively binds in Bcr-Abl transformed cells.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"ABL1\"],\"tf text\":[\"STAT5\"],\"tg text\":[\"Bcr-Abl\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, the cyclin D1 and bcl-x promoters contain STAT binding sites to which STAT5 constitutively binds in Bcr-Abl transformed cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"ABL1\"],\"tf text\":[\"STAT5\"],\"tg text\":[\"Bcr-Abl\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Moreover\",\"the cyclin D1 and bcl-x promoters contain STAT binding sites to which STAT5 constitutively binds in Bcr-Abl transformed cells.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"ABL1\"],\"tf text\":[\"STAT5\"],\"tg text\":[\"Bcr-Abl\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, the cyclin D1 and bcl-x promoters contain STAT binding sites to which STAT5 constitutively binds in Bcr-Abl transformed cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 493 (char 492)\n",
      "{\"tf associated gene name\":[\"NKX2-2\"],\"tg associated gene name\":[\"VGLL2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Top neonatal markers include transcriptional regulators such as Vgll2\",\"SF-1\",\"Sox14\",\"Satb2\",\"Fezf1\",\"Dax1\",\"Nkx2-2\",\"and COUP-TFII\",\"but interestingly\",\"the highest expressed VMH transcript\",\"the transcriptional coregulator Vgll2\",\"is completely absent in older animals.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX2-2\"],\"tg (associated gene name)\":[\"VGLL2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Top neonatal markers include transcriptional regulators such as Vgll2, SF-1, Sox14, Satb2, Fezf1, Dax1, Nkx2-2, and COUP-TFII, but interestingly, the highest expressed VMH transcript, the transcriptional coregulator Vgll2, is completely absent in older animals.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NKX2-2\"],\"tg associated gene name\":[\"VGLL2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Top neonatal markers include transcriptional regulators such as Vgll2\",\"SF-1\",\"Sox14\",\"Satb2\",\"Fezf1\",\"Dax1\",\"Nkx2-2\",\"and COUP-TFII\",\"but interestingly\",\"the highest expressed VMH transcript\",\"the transcriptional coregulator Vgll2\",\"is completely absent in older animals.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX2-2\"],\"tg (associated gene name)\":[\"VGLL2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Top neonatal markers include transcriptional regulators such as Vgll2, SF-1, Sox14, Satb2, Fezf1, Dax1, Nkx2-2, and COUP-TFII, but interestingly, the highest expressed VMH transcript, the transcriptional coregulator Vgll2, is completely absent in older animals.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 473 (char 472)\n",
      "{\"transcription factor (associated gene name)\":[\"RARB\"],\"target gene (associated gene name)\":[\"AFP\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Acyclic retinoid and all-trans-RA both induced upregulation of retinoic acid receptor-beta (RAR beta)\",\"and both suppressed cell proliferation-related phenotypic expressions by the alpha-fetoprotein gene and the c-myc oncogene.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|TG unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RARB\"],\"tg (associated gene name)\":[\"AFP\"],\"tf text\":[\"retinoic acid receptor-beta\"],\"tg text\":[\"alpha-fetoprotein\"],\"consensus\":[\"V\"],\"martin_comments\":[\"TG unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Acyclic retinoid and all-trans-RA both induced upregulation of retinoic acid receptor-beta (RAR beta), and both suppressed cell proliferation-related phenotypic expressions by the alpha-fetoprotein gene and the c-myc oncogene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RARB\"],\"tg associated gene name\":[\"AFP\"],\"tf text\":[\"retinoic acid receptor-beta\"],\"tg text\":[\"alpha-fetoprotein\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Acyclic retinoid and all-trans-RA both induced upregulation of retinoic acid receptor-beta (RAR beta)\",\"and both suppressed cell proliferation-related phenotypic expressions by the alpha-fetoprotein gene and the c-myc oncogene.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RARB\"],\"target gene (associated gene name)\":[\"AFP\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Acyclic retinoid and all-trans-RA both induced upregulation of retinoic acid receptor-beta (RAR beta)\",\"and both suppressed cell proliferation-related phenotypic expressions by the alpha-fetoprotein gene and the c-myc oncogene.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|TG unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RARB\"],\"tg (associated gene name)\":[\"AFP\"],\"tf text\":[\"retinoic acid receptor-beta\"],\"tg text\":[\"alpha-fetoprotein\"],\"consensus\":[\"V\"],\"martin_comments\":[\"TG unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Acyclic retinoid and all-trans-RA both induced upregulation of retinoic acid receptor-beta (RAR beta), and both suppressed cell proliferation-related phenotypic expressions by the alpha-fetoprotein gene and the c-myc oncogene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 450 (char 449)\n",
      "{\"transcription factor (associated gene name)\":[\"LRRFIP1\"],\"target gene (associated gene name)\":[\"PDGFA\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"5' Deletion and transient cotransfection analysis in vascular endothelial cells revealed that GCF2 repression is mediated by a nucleotide region located in the proximal region of the PDGF-A promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|sure?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LRRFIP1\"],\"tg (associated gene name)\":[\"PDGFA\"],\"tf text\":[\"GCF2\"],\"tg text\":[\"PDGF-A\"],\"consensus\":[\"V\"],\"martin_comments\":[\"sure?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"5' Deletion and transient cotransfection analysis in vascular endothelial cells revealed that GCF2 repression is mediated by a nucleotide region located in the proximal region of the PDGF-A promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"LRRFIP1\"],\"tg associated gene name\":[\"PDGFA\"],\"tf text\":[\"GCF2\"],\"tg text\":[\"PDGF-A\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"5' Deletion and transient cotransfection analysis in vascular endothelial cells revealed that GCF2 repression is mediated by a nucleotide region located in the proximal region of the PDGF-A promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"LRRFIP1\"],\"target gene (associated gene name)\":[\"PDGFA\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"5' Deletion and transient cotransfection analysis in vascular endothelial cells revealed that GCF2 repression is mediated by a nucleotide region located in the proximal region of the PDGF-A promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|sure?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LRRFIP1\"],\"tg (associated gene name)\":[\"PDGFA\"],\"tf text\":[\"GCF2\"],\"tg text\":[\"PDGF-A\"],\"consensus\":[\"V\"],\"martin_comments\":[\"sure?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"5' Deletion and transient cotransfection analysis in vascular endothelial cells revealed that GCF2 repression is mediated by a nucleotide region located in the proximal region of the PDGF-A promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is HSC---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is HSC\"],\"sentence\":[\"We then constructed HSC-specific pri-miRNA mimic and pri-miRNA cluster mimic expression plasmids in which shRNA expression was driven by a glial fibrillary acidic protein (GFAP) promoter to achieve HSC-specific TGF-??1 gene silencing to avoid nonspecific inhibition of TGF-??1 expression in other cells and organs.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is HSC---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is HSC\"],\"sentence\":[\"We then constructed HSC-specific pri-miRNA mimic and pri-miRNA cluster mimic expression plasmids in which shRNA expression was driven by a glial fibrillary acidic protein (GFAP) promoter to achieve HSC-specific TGF-??1 gene silencing to avoid nonspecific inhibition of TGF-??1 expression in other cells and organs.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MITF\"],\"target gene (associated gene name)\":[\"TGFB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"We then constructed HSC-specific pri-miRNA mimic and pri-miRNA cluster mimic expression plasmids in which shRNA expression was driven by a glial fibrillary acidic protein (GFAP) promoter to achieve HSC-specific TGF-1 gene silencing to avoid nonspecific inhibition of TGF-1 expression in other cells and organs.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"3.852194\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"310\"],\"sentence pair density\":[\"0.00689655172413793\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We then constructed HSC-specific pri-miRNA mimic and pri-miRNA cluster mimic expression plasmids in which shRNA expression was driven by a glial fibrillary acidic protein (GFAP) promoter to achieve HSC-specific TGF-1 gene silencing to avoid nonspecific inhibition of TGF-1 expression in other cells and organs.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is P3; not normalizeable?|not likely to be GNAS---{\"tf associated gene name\":[\"WT1\"],\"tg associated gene name\":[\"GNAS\"],\"tf text\":[\"WT1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is P3; not normalizeable?\",\"not likely to be GNAS\"],\"sentence\":[\"To further delineate which factors mediate this response\",\"the expression pattern of regulators of the P3 promoter\",\"Egr-1\",\"Sp1\",\"and WT1\",\"were analyzed by reverse transcription-PCR and Northern blot analysis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|P3 = TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"WT1\"],\"tg (associated gene name)\":[\"GNAS\"],\"tf text\":[\"WT1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"P3 = TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To further delineate which factors mediate this response, the expression pattern of regulators of the P3 promoter, Egr-1, Sp1, and WT1, were analyzed by reverse transcription-PCR and Northern blot analysis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is P3; not normalizeable?|not likely to be GNAS---{\"tf associated gene name\":[\"WT1\"],\"tg associated gene name\":[\"GNAS\"],\"tf text\":[\"WT1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is P3; not normalizeable?\",\"not likely to be GNAS\"],\"sentence\":[\"To further delineate which factors mediate this response\",\"the expression pattern of regulators of the P3 promoter\",\"Egr-1\",\"Sp1\",\"and WT1\",\"were analyzed by reverse transcription-PCR and Northern blot analysis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|P3 = TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"WT1\"],\"tg (associated gene name)\":[\"GNAS\"],\"tf text\":[\"WT1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"P3 = TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To further delineate which factors mediate this response, the expression pattern of regulators of the P3 promoter, Egr-1, Sp1, and WT1, were analyzed by reverse transcription-PCR and Northern blot analysis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEAD1\"],\"tg (associated gene name)\":[\"USP7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The betaMyHC promoter required an intact M-CAT element to respond to TEF-1 and RTEF-1, whereas the SKA promoter M-CAT was required for the TEF-1 response but not for the RTEF-1 response, suggesting that SKA promoter-specific cofactors may be involved.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEAD1\"],\"tg (associated gene name)\":[\"USP7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The betaMyHC promoter required an intact M-CAT element to respond to TEF-1 and RTEF-1, whereas the SKA promoter M-CAT was required for the TEF-1 response but not for the RTEF-1 response, suggesting that SKA promoter-specific cofactors may be involved.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG|REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"EGF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In contrast, the anti-apoptotic stimulus, epidermal growth factor (EGF), fails to increase DR5 expression but effectively activates NF-kappaB and induces p65 binding to the DR5 gene.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG|REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"EGF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In contrast, the anti-apoptotic stimulus, epidermal growth factor (EGF), fails to increase DR5 expression but effectively activates NF-kappaB and induces p65 binding to the DR5 gene.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no direct involvelemnt of SP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"PCYT1A\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no direct involvelemnt of SP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In cells that overexpress Sp1, co-expression of cyclin A and CDK2 induces a high and constant level of CTalpha expression, whereas reduction in the expression of cyclin A, cyclin E, and CDK2 eliminates the induction of CTalpha expression in S phase.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no direct involvelemnt of SP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"PCYT1A\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no direct involvelemnt of SP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In cells that overexpress Sp1, co-expression of cyclin A and CDK2 induces a high and constant level of CTalpha expression, whereas reduction in the expression of cyclin A, cyclin E, and CDK2 eliminates the induction of CTalpha expression in S phase.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"ATF2\"],\"tg associated gene name\":[\"ATF2\"],\"tf text\":[\"ATF-2\"],\"tg text\":[\"ATF-2\"],\"1\":[],\"0\":[\"t\",\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"Polyamine depletion did not alter transcription via the ATF-2 gene promoter but increased the stability of ATF-2 mRNA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ATF2\"],\"tg (associated gene name)\":[\"ATF2\"],\"tf text\":[\"ATF-2\"],\"tg text\":[\"ATF-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Polyamine depletion did not alter transcription via the ATF-2 gene promoter but increased the stability of ATF-2 mRNA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"ATF2\"],\"tg associated gene name\":[\"ATF2\"],\"tf text\":[\"ATF-2\"],\"tg text\":[\"ATF-2\"],\"1\":[],\"0\":[\"t\",\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"Polyamine depletion did not alter transcription via the ATF-2 gene promoter but increased the stability of ATF-2 mRNA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ATF2\"],\"tg (associated gene name)\":[\"ATF2\"],\"tf text\":[\"ATF-2\"],\"tg text\":[\"ATF-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Polyamine depletion did not alter transcription via the ATF-2 gene promoter but increased the stability of ATF-2 mRNA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TG named (both enzymes)---{\"tf associated gene name\":[\"EGR2\"],\"tg associated gene name\":[\"BTG2\"],\"tf text\":[\"EGR2\"],\"tg text\":[\"BTG2\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TG named (both enzymes)\"],\"sentence\":[\"BHLHB2\",\"BTG2\",\"and SALL1 decreased expression of both enzymes\",\"whereas CITED2\",\"EGR2\",\"ELL2\",\"FOS\",\"FOSB\",\"HDAC5\",\"MAFF\",\"MITF\",\"NFIL3\",\"NR4A1\",\"NR4A2\",\"NR4A3\",\"PER1\",\"and VDR increased expression for both enzymes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR2\"],\"tg (associated gene name)\":[\"BTG2\"],\"tf text\":[\"EGR2\"],\"tg text\":[\"BTG2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"BHLHB2, BTG2, and SALL1 decreased expression of both enzymes, whereas CITED2, EGR2, ELL2, FOS, FOSB, HDAC5, MAFF, MITF, NFIL3, NR4A1, NR4A2, NR4A3, PER1, and VDR increased expression for both enzymes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TG named (both enzymes)---{\"tf associated gene name\":[\"EGR2\"],\"tg associated gene name\":[\"BTG2\"],\"tf text\":[\"EGR2\"],\"tg text\":[\"BTG2\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TG named (both enzymes)\"],\"sentence\":[\"BHLHB2\",\"BTG2\",\"and SALL1 decreased expression of both enzymes\",\"whereas CITED2\",\"EGR2\",\"ELL2\",\"FOS\",\"FOSB\",\"HDAC5\",\"MAFF\",\"MITF\",\"NFIL3\",\"NR4A1\",\"NR4A2\",\"NR4A3\",\"PER1\",\"and VDR increased expression for both enzymes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR2\"],\"tg (associated gene name)\":[\"BTG2\"],\"tf text\":[\"EGR2\"],\"tg text\":[\"BTG2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"BHLHB2, BTG2, and SALL1 decreased expression of both enzymes, whereas CITED2, EGR2, ELL2, FOS, FOSB, HDAC5, MAFF, MITF, NFIL3, NR4A1, NR4A2, NR4A3, PER1, and VDR increased expression for both enzymes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOR2\"],\"tg (associated gene name)\":[\"DYNAP\"],\"tf text\":[\"SMRT\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"SMRT is required for full expression of the ERalpha target genes cyclin D1, BCL-2, and progesterone receptor but not pS2, and its depletion significantly attenuated estrogen-dependent proliferation of MCF-7 cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOR2\"],\"tg (associated gene name)\":[\"DYNAP\"],\"tf text\":[\"SMRT\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"SMRT is required for full expression of the ERalpha target genes cyclin D1, BCL-2, and progesterone receptor but not pS2, and its depletion significantly attenuated estrogen-dependent proliferation of MCF-7 cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not named---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TF not named\"],\"sentence\":[\"Inside the P3 upstream promoter of human ER gene we identified a transcriptional regulatory sequence able to bind protein factors expressed in ER-negative MDA-MB-231 breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF not named---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TF not named\"],\"sentence\":[\"Inside the P3 upstream promoter of human ER gene we identified a transcriptional regulatory sequence able to bind protein factors expressed in ER-negative MDA-MB-231 breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"ESR1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Inside the P3 upstream promoter of human ER gene we identified a transcriptional regulatory sequence able to bind protein factors expressed in ER-negative MDA-MB-231 breast cancer cells.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.379731\"],\"pmid counts\":[\"185\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"186\"],\"sentence pair density\":[\"0.0119521912350597\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF unlcear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF unlcear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inside the P3 upstream promoter of human ER gene we identified a transcriptional regulatory sequence able to bind protein factors expressed in ER-negative MDA-MB-231 breast cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is VEGF---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"SP3\"],\"tf text\":[],\"tg text\":[\"Sp3\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is VEGF\"],\"sentence\":[\"Our evidence suggests that AP-2 repressed transcriptionally the VEGF promoter by competing with the transcriptional activator Sp3.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TFAP2A\"],\"tg (associated gene name)\":[\"SP3\"],\"tf text\":[],\"tg text\":[\"Sp3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our evidence suggests that AP-2 repressed transcriptionally the VEGF promoter by competing with the transcriptional activator Sp3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is VEGF---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"SP3\"],\"tf text\":[],\"tg text\":[\"Sp3\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is VEGF\"],\"sentence\":[\"Our evidence suggests that AP-2 repressed transcriptionally the VEGF promoter by competing with the transcriptional activator Sp3.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TFAP2A\"],\"tg (associated gene name)\":[\"SP3\"],\"tf text\":[],\"tg text\":[\"Sp3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our evidence suggests that AP-2 repressed transcriptionally the VEGF promoter by competing with the transcriptional activator Sp3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is TERT---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"IFNG\"],\"tf text\":[\"IRF-1\"],\"tg text\":[\"interferon-gamma\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is TERT\"],\"sentence\":[\"Interferon regulatory factor-1 (IRF-1) is a mediator for interferon-gamma induced attenuation of telomerase activity and human telomerase reverse transcriptase (hTERT) expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[\"IRF-1\"],\"tg text\":[\"interferon-gamma\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Interferon regulatory factor-1 (IRF-1) is a mediator for interferon-gamma induced attenuation of telomerase activity and human telomerase reverse transcriptase (hTERT) expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is TERT---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"IFNG\"],\"tf text\":[\"IRF-1\"],\"tg text\":[\"interferon-gamma\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is TERT\"],\"sentence\":[\"Interferon regulatory factor-1 (IRF-1) is a mediator for interferon-gamma induced attenuation of telomerase activity and human telomerase reverse transcriptase (hTERT) expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[\"IRF-1\"],\"tg text\":[\"interferon-gamma\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Interferon regulatory factor-1 (IRF-1) is a mediator for interferon-gamma induced attenuation of telomerase activity and human telomerase reverse transcriptase (hTERT) expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 359 (char 358)\n",
      "{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Wild-type p53 was found to repress expression from TATA box- but not initiator (Inr)-containing promoters activated by GAL4-VP16\",\"GAL4-ftzQ or Sp1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Wild-type p53 was found to repress expression from TATA box- but not initiator (Inr)-containing promoters activated by GAL4-VP16, GAL4-ftzQ or Sp1.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Wild-type p53 was found to repress expression from TATA box- but not initiator (Inr)-containing promoters activated by GAL4-VP16\",\"GAL4-ftzQ or Sp1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Wild-type p53 was found to repress expression from TATA box- but not initiator (Inr)-containing promoters activated by GAL4-VP16, GAL4-ftzQ or Sp1.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 821 (char 820)\n",
      "{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"MAPK14\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"The present study shows that CRP induces parallel activation of the redox-responsive transcription factors NF-kappa B (NF-kappaB) and AP-1 and increases the activity of the MAP kinases (MAPKs)\",\"extracellular signal-regulated kinase (ERK)\",\"c-Jun N-terminal kinase (JNK) and p38MAPK\",\"in VSMC.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.821228\"],\"pmid counts\":[\"6\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"290\"],\"sentence pair density\":[\"0.010752688172043\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no link between AP and TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MAPK14\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no link between AP and TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The present study shows that CRP induces parallel activation of the redox-responsive transcription factors NF-kappa B (NF-kappaB) and AP-1 and increases the activity of the MAP kinases (MAPKs), extracellular signal-regulated kinase (ERK), c-Jun N-terminal kinase (JNK) and p38MAPK, in VSMC.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"MAPK14\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The present study shows that CRP induces parallel activation of the redox-responsive transcription factors NF-kappa B (NF-kappaB) and AP-1 and increases the activity of the MAP kinases (MAPKs)\",\"extracellular signal-regulated kinase (ERK)\",\"c-Jun N-terminal kinase (JNK) and p38MAPK\",\"in VSMC.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"MAPK14\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"The present study shows that CRP induces parallel activation of the redox-responsive transcription factors NF-kappa B (NF-kappaB) and AP-1 and increases the activity of the MAP kinases (MAPKs)\",\"extracellular signal-regulated kinase (ERK)\",\"c-Jun N-terminal kinase (JNK) and p38MAPK\",\"in VSMC.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.821228\"],\"pmid counts\":[\"6\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"290\"],\"sentence pair density\":[\"0.010752688172043\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no link between AP and TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MAPK14\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no link between AP and TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The present study shows that CRP induces parallel activation of the redox-responsive transcription factors NF-kappa B (NF-kappaB) and AP-1 and increases the activity of the MAP kinases (MAPKs), extracellular signal-regulated kinase (ERK), c-Jun N-terminal kinase (JNK) and p38MAPK, in VSMC.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "AP1 link?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"CDKN1C\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AP1 link?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We used the 16-mer core sequences of the RRSs in the AAV ITRs and AAVS1 separately as query sequences and identified 18 new RRSs in or flanking the genes coding for the following: tyrosine kinase activator protein 1 (TKA-1); colony stimulating factor-1; insulin-like growth factor binding protein 2 (IGFBP-2); histone H2B.1; basement membrane heparan sulfate proteoglycan, also known as perlecan; the AF-9 gene product, which is involved in the chromosomal translocation t (9:11)(p22:q23); the betaB subunit of the hormone known as inhibin; interleukin-2 enhancer binding factor; an endoplasmic reticulum-Golgi intermediate compartment resident protein called p63; a global transcription activator (hSNF2L); the beta-actin repair domain; a retinoic acid-inducible factor, also known as midkine; a breast tumor autoantigen; a growth-arrest- and DNA-damage-inducible protein called gadd45; the cyclin-dependent kinase inhibitor called KIP2, which inhibits several G1 cyclin-cyclin-dependent kinase complexes; and the hereditary breast and ovarian cancer gene (BRCA1).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "AP1 link?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"CDKN1C\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AP1 link?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We used the 16-mer core sequences of the RRSs in the AAV ITRs and AAVS1 separately as query sequences and identified 18 new RRSs in or flanking the genes coding for the following: tyrosine kinase activator protein 1 (TKA-1); colony stimulating factor-1; insulin-like growth factor binding protein 2 (IGFBP-2); histone H2B.1; basement membrane heparan sulfate proteoglycan, also known as perlecan; the AF-9 gene product, which is involved in the chromosomal translocation t (9:11)(p22:q23); the betaB subunit of the hormone known as inhibin; interleukin-2 enhancer binding factor; an endoplasmic reticulum-Golgi intermediate compartment resident protein called p63; a global transcription activator (hSNF2L); the beta-actin repair domain; a retinoic acid-inducible factor, also known as midkine; a breast tumor autoantigen; a growth-arrest- and DNA-damage-inducible protein called gadd45; the cyclin-dependent kinase inhibitor called KIP2, which inhibits several G1 cyclin-cyclin-dependent kinase complexes; and the hereditary breast and ovarian cancer gene (BRCA1).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI1\"],\"tg (associated gene name)\":[\"MEN1\"],\"tf text\":[\"Gli1\"],\"tg text\":[\"MEN1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Inhibition of Gli1 by the small-molecule inhibitor GANT-61 led to decreased expression of Gli1 and its target genes in MEN1-depeleted cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI1\"],\"tg (associated gene name)\":[\"MEN1\"],\"tf text\":[\"Gli1\"],\"tg text\":[\"MEN1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Inhibition of Gli1 by the small-molecule inhibitor GANT-61 led to decreased expression of Gli1 and its target genes in MEN1-depeleted cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 312 (char 311)\n",
      "{\"transcription factor (associated gene name)\":[\"TXK\"],\"target gene (associated gene name)\":[\"IL2\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Txk transfection did not affect IL-2 and IL-4 promoter activities.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TXK\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[\"Txk\"],\"tg text\":[\"IL-2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Txk transfection did not affect IL-2 and IL-4 promoter activities.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[\"Txk\"],\"tg text\":[\"IL-2\"],\"1\":[\"u\"],\"0\":[],\"negation\":[\"x\"],\"comment\":[],\"sentence\":[\"Txk transfection did not affect IL-2 and IL-4 promoter activities.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TXK\"],\"target gene (associated gene name)\":[\"IL2\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Txk transfection did not affect IL-2 and IL-4 promoter activities.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|NOT, but---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TXK\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[\"Txk\"],\"tg text\":[\"IL-2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT, but\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Txk transfection did not affect IL-2 and IL-4 promoter activities.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Ig VH---{\"tf associated gene name\":[\"CEBPG\"],\"tg associated gene name\":[\"ATP11C\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is Ig VH\"],\"sentence\":[\"The C/EBP family of transcriptional activators is functionally important for Ig VH promoter activity in vivo and in vitro.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|ATP11C?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPG\"],\"tg (associated gene name)\":[\"ATP11C\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"ATP11C?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The C/EBP family of transcriptional activators is functionally important for Ig VH promoter activity in vivo and in vitro.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Ig VH---{\"tf associated gene name\":[\"CEBPG\"],\"tg associated gene name\":[\"ATP11C\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is Ig VH\"],\"sentence\":[\"The C/EBP family of transcriptional activators is functionally important for Ig VH promoter activity in vivo and in vitro.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|ATP11C?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPG\"],\"tg (associated gene name)\":[\"ATP11C\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"ATP11C?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The C/EBP family of transcriptional activators is functionally important for Ig VH promoter activity in vivo and in vitro.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "Wrog TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"Wrog TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We showed that STAT1 interacts with the GILT promoter, even in the absence of IFN-??, and that STAT1 represses GILT expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "Wrog TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"Wrog TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We showed that STAT1 interacts with the GILT promoter, even in the absence of IFN-??, and that STAT1 represses GILT expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is IL8---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"DYNAP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is IL8\"],\"sentence\":[\"In parallel\",\"apicidin treatment leads to histone hyperacetylation in the IL-8 promoter region independent of NF-kappaB signaling pathway\",\"which is not sufficient for full transcription of IL-8 gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is IL8---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"DYNAP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is IL8\"],\"sentence\":[\"In parallel\",\"apicidin treatment leads to histone hyperacetylation in the IL-8 promoter region independent of NF-kappaB signaling pathway\",\"which is not sufficient for full transcription of IL-8 gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"DYNAP\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"In parallel\",\"apicidin treatment leads to histone hyperacetylation in the IL-8 promoter region independent of NF-kappaB signaling pathway\",\"which is not sufficient for full transcription of IL-8 gene.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.193721\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"198\"],\"sentence pair density\":[\"0.0168918918918918\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"DYNAP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In parallel, apicidin treatment leads to histone hyperacetylation in the IL-8 promoter region independent of NF-kappaB signaling pathway, which is not sufficient for full transcription of IL-8 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG ---{\"tf associated gene name\":[\"NKX3-1\"],\"tg associated gene name\":[\"LBX1\"],\"tf text\":[\"NKX3.1\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TG \"],\"sentence\":[\"NKX3.1 is a prostate-specific homeobox gene related strongly to prostate development and prostate cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX3-1\"],\"tg (associated gene name)\":[\"LBX1\"],\"tf text\":[\"NKX3.1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"NKX3.1 is a prostate-specific homeobox gene related strongly to prostate development and prostate cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG ---{\"tf associated gene name\":[\"NKX3-1\"],\"tg associated gene name\":[\"LBX1\"],\"tf text\":[\"NKX3.1\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TG \"],\"sentence\":[\"NKX3.1 is a prostate-specific homeobox gene related strongly to prostate development and prostate cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX3-1\"],\"tg (associated gene name)\":[\"LBX1\"],\"tf text\":[\"NKX3.1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"NKX3.1 is a prostate-specific homeobox gene related strongly to prostate development and prostate cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 514 (char 513)\n",
      "{\"tf associated gene name\":[\"BCL6\"],\"tg associated gene name\":[\"SELL\"],\"tf text\":[\"Bcl-6\"],\"tg text\":[\"CD62L\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Expression of the transcriptional repressor Blimp-1 distinguished T zone 'lymphoid' effector helper T cells (CD62L(hi)CCR7(hi)) from CXCR5(lo) 'emigrant' effector helper T cells and CXCR5(hi) 'resident' T(FH) cells expressing the transcriptional repressor Bcl-6 (CD62L(lo)CCR7(lo)).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG role unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BCL6\"],\"tg (associated gene name)\":[\"SELL\"],\"tf text\":[\"Bcl-6\"],\"tg text\":[\"CD62L\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG role unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression of the transcriptional repressor Blimp-1 distinguished T zone 'lymphoid' effector helper T cells (CD62L(hi)CCR7(hi)) from CXCR5(lo) 'emigrant' effector helper T cells and CXCR5(hi) 'resident' T(FH) cells expressing the transcriptional repressor Bcl-6 (CD62L(lo)CCR7(lo)).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"BCL6\"],\"tg associated gene name\":[\"SELL\"],\"tf text\":[\"Bcl-6\"],\"tg text\":[\"CD62L\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Expression of the transcriptional repressor Blimp-1 distinguished T zone 'lymphoid' effector helper T cells (CD62L(hi)CCR7(hi)) from CXCR5(lo) 'emigrant' effector helper T cells and CXCR5(hi) 'resident' T(FH) cells expressing the transcriptional repressor Bcl-6 (CD62L(lo)CCR7(lo)).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG role unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BCL6\"],\"tg (associated gene name)\":[\"SELL\"],\"tf text\":[\"Bcl-6\"],\"tg text\":[\"CD62L\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG role unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression of the transcriptional repressor Blimp-1 distinguished T zone 'lymphoid' effector helper T cells (CD62L(hi)CCR7(hi)) from CXCR5(lo) 'emigrant' effector helper T cells and CXCR5(hi) 'resident' T(FH) cells expressing the transcriptional repressor Bcl-6 (CD62L(lo)CCR7(lo)).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "c-REL TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF1\"],\"tg (associated gene name)\":[\"IL12A\"],\"tf text\":[],\"tg text\":[\"il12a\"],\"consensus\":[\"N\"],\"martin_comments\":[\"c-REL TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These data indicate that an enhancement of SIRT1 activity in response to phagocytic stimuli may reduce the accessibility of c-Rel to the il12a promoter and its transcriptional activation, thus regulating the IL-12 p70/IL-23 balance and modulating the ongoing immune response.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "c-REL TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF1\"],\"tg (associated gene name)\":[\"IL12A\"],\"tf text\":[],\"tg text\":[\"il12a\"],\"consensus\":[\"N\"],\"martin_comments\":[\"c-REL TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These data indicate that an enhancement of SIRT1 activity in response to phagocytic stimuli may reduce the accessibility of c-Rel to the il12a promoter and its transcriptional activation, thus regulating the IL-12 p70/IL-23 balance and modulating the ongoing immune response.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is c-maf|TG is IL-12p35---{\"tf associated gene name\":[\"ELF1\"],\"tg associated gene name\":[\"IL12A\"],\"tf text\":[],\"tg text\":[\"p35\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is c-maf\",\"TG is IL-12p35\"],\"sentence\":[\"We report that c-Maf is a physiologic modulator of IL-12p35 gene expression and IL-12p70 production.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|c-Maf TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF1\"],\"tg (associated gene name)\":[\"IL12A\"],\"tf text\":[],\"tg text\":[\"p35\"],\"consensus\":[\"N\"],\"martin_comments\":[\"c-Maf TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We report that c-Maf is a physiologic modulator of IL-12p35 gene expression and IL-12p70 production.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is c-maf|TG is IL-12p35---{\"tf associated gene name\":[\"ELF1\"],\"tg associated gene name\":[\"IL12A\"],\"tf text\":[],\"tg text\":[\"p35\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is c-maf\",\"TG is IL-12p35\"],\"sentence\":[\"We report that c-Maf is a physiologic modulator of IL-12p35 gene expression and IL-12p70 production.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|c-Maf TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF1\"],\"tg (associated gene name)\":[\"IL12A\"],\"tf text\":[],\"tg text\":[\"p35\"],\"consensus\":[\"N\"],\"martin_comments\":[\"c-Maf TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We report that c-Maf is a physiologic modulator of IL-12p35 gene expression and IL-12p70 production.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 541 (char 540)\n",
      "{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"CDX2\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"CDX2\"],\"1\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"We report data implicating repression of the transcription factor KLF4 as important for the oncogenic activity of CDX2\",\"and demonstrate that CDX2 differentially regulates KLF4 in AML versus colon cancer cells through a mechanism that involves tissue-specific patterns of promoter binding and epigenetic modifications.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|KLF4 = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"CDX2\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"CDX2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"KLF4 = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We report data implicating repression of the transcription factor KLF4 as important for the oncogenic activity of CDX2, and demonstrate that CDX2 differentially regulates KLF4 in AML versus colon cancer cells through a mechanism that involves tissue-specific patterns of promoter binding and epigenetic modifications.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"CDX2\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"CDX2\"],\"1\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"We report data implicating repression of the transcription factor KLF4 as important for the oncogenic activity of CDX2\",\"and demonstrate that CDX2 differentially regulates KLF4 in AML versus colon cancer cells through a mechanism that involves tissue-specific patterns of promoter binding and epigenetic modifications.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|KLF4 = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"CDX2\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"CDX2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"KLF4 = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We report data implicating repression of the transcription factor KLF4 as important for the oncogenic activity of CDX2, and demonstrate that CDX2 differentially regulates KLF4 in AML versus colon cancer cells through a mechanism that involves tissue-specific patterns of promoter binding and epigenetic modifications.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 367 (char 366)\n",
      "{\"transcription factor (associated gene name)\":[\"USF1\"],\"target gene (associated gene name)\":[\"SHBG\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Upstream stimulatory factor induces Nr5a1 and Shbg gene expression during the onset of rat Sertoli cell differentiation.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|upstream stim fact enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"SHBG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"upstream stim fact enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Upstream stimulatory factor induces Nr5a1 and Shbg gene expression during the onset of rat Sertoli cell differentiation.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"USF1\"],\"target gene (associated gene name)\":[\"SHBG\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"a\"],\"negation\":[],\"sentence\":[\"Upstream stimulatory factor induces Nr5a1 and Shbg gene expression during the onset of rat Sertoli cell differentiation.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.002268\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"120\"],\"sentence pair density\":[\"0.00588235294117647\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"USF1\"],\"target gene (associated gene name)\":[\"SHBG\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Upstream stimulatory factor induces Nr5a1 and Shbg gene expression during the onset of rat Sertoli cell differentiation.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|upstream stim fact enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"SHBG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"upstream stim fact enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Upstream stimulatory factor induces Nr5a1 and Shbg gene expression during the onset of rat Sertoli cell differentiation.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GRE relations (several) not correctly dientified/normalized)---{\"tf associated gene name\":[\"EOMES\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"GRE relations (several) not correctly dientified/normalized)\"],\"sentence\":[\"In this study\",\"we report that STAT5CA induced sustained expression of genes controlling tissue homing\",\"cytolytic granule composition\",\"type 1 CD8 cytotoxic T cell-associated effector molecules granzyme B(+)\",\"IFN-??(+)\",\"TNF-??(+)\",\"and CCL3(+)\",\"but not IL-2\",\"and transcription factors T-bet and eomesodermin (Eomes).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|STAT5CA = TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EOMES\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"STAT5CA = TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this study, we report that STAT5CA induced sustained expression of genes controlling tissue homing, cytolytic granule composition, type 1 CD8 cytotoxic T cell-associated effector molecules granzyme B(+), IFN-(+), TNF-(+), and CCL3(+), but not IL-2, and transcription factors T-bet and eomesodermin (Eomes).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "GRE relations (several) not correctly dientified/normalized)---{\"tf associated gene name\":[\"EOMES\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"GRE relations (several) not correctly dientified/normalized)\"],\"sentence\":[\"In this study\",\"we report that STAT5CA induced sustained expression of genes controlling tissue homing\",\"cytolytic granule composition\",\"type 1 CD8 cytotoxic T cell-associated effector molecules granzyme B(+)\",\"IFN-??(+)\",\"TNF-??(+)\",\"and CCL3(+)\",\"but not IL-2\",\"and transcription factors T-bet and eomesodermin (Eomes).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|STAT5CA = TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EOMES\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"STAT5CA = TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this study, we report that STAT5CA induced sustained expression of genes controlling tissue homing, cytolytic granule composition, type 1 CD8 cytotoxic T cell-associated effector molecules granzyme B(+), IFN-(+), TNF-(+), and CCL3(+), but not IL-2, and transcription factors T-bet and eomesodermin (Eomes).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "SLU7 TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C2\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SLU7 TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Conversely, Stat5 negatively interferes with glucocorticoid receptor, mineralocorticoid receptor and progesterone receptor induced transcription from the MMTV LTR and the estrogen receptor induced transcription from an ERE-containing promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "SLU7 TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C2\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SLU7 TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Conversely, Stat5 negatively interferes with glucocorticoid receptor, mineralocorticoid receptor and progesterone receptor induced transcription from the MMTV LTR and the estrogen receptor induced transcription from an ERE-containing promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "c-fos TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"PTPN13\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"c-fos TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Coordinate regulation of STAT signaling and c-fos expression by the tyrosine phosphatase SHP-2.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "c-fos TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"PTPN13\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"c-fos TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Coordinate regulation of STAT signaling and c-fos expression by the tyrosine phosphatase SHP-2.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF named---{\"tf associated gene name\":[\"NRG1\"],\"tg associated gene name\":[\"PLAU\"],\"tf text\":[],\"tg text\":[\"uPA\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no TF named\"],\"sentence\":[\"The stimulation by heregulin-beta1 of noninvasive human breast cancer MCF-7 cells induced the expression of uPA mRNA\",\"protein\",\"and its plasminogenic activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|heregulin Beta!---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRG1\"],\"tg (associated gene name)\":[\"PLAU\"],\"tf text\":[],\"tg text\":[\"uPA\"],\"consensus\":[\"N\"],\"martin_comments\":[\"heregulin Beta!\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The stimulation by heregulin-beta1 of noninvasive human breast cancer MCF-7 cells induced the expression of uPA mRNA, protein, and its plasminogenic activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF named---{\"tf associated gene name\":[\"NRG1\"],\"tg associated gene name\":[\"PLAU\"],\"tf text\":[],\"tg text\":[\"uPA\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no TF named\"],\"sentence\":[\"The stimulation by heregulin-beta1 of noninvasive human breast cancer MCF-7 cells induced the expression of uPA mRNA\",\"protein\",\"and its plasminogenic activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|heregulin Beta!---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRG1\"],\"tg (associated gene name)\":[\"PLAU\"],\"tf text\":[],\"tg text\":[\"uPA\"],\"consensus\":[\"N\"],\"martin_comments\":[\"heregulin Beta!\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The stimulation by heregulin-beta1 of noninvasive human breast cancer MCF-7 cells induced the expression of uPA mRNA, protein, and its plasminogenic activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulation expression a bit on the weak side---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"LHB\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"not true\":[],\"comment\":[\"regulation expression a bit on the weak side\"],\"sentence\":[\"Granulosa cell expression of PGR is induced by the pituitary Luteinizing Hormone (LH) surge via mechanisms that are not entirely understood\",\"but which involve activation of Protein Kinase A and modification of Sp1/Sp3 transcription factors on the PGR promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "regulation expression a bit on the weak side---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"LHB\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"not true\":[],\"comment\":[\"regulation expression a bit on the weak side\"],\"sentence\":[\"Granulosa cell expression of PGR is induced by the pituitary Luteinizing Hormone (LH) surge via mechanisms that are not entirely understood\",\"but which involve activation of Protein Kinase A and modification of Sp1/Sp3 transcription factors on the PGR promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP3\"],\"tg (associated gene name)\":[\"LHB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Granulosa cell expression of PGR is induced by the pituitary Luteinizing Hormone (LH) surge via mechanisms that are not entirely understood, but which involve activation of Protein Kinase A and modification of Sp1/Sp3 transcription factors on the PGR promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 423 (char 422)\n",
      "{\"transcription factor (associated gene name)\":[\"NR1H3\"],\"target gene (associated gene name)\":[\"ABCG1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Furthermore\",\"the expression of \\\"classical\\\" liver X receptor (LXR) target genes involved in reverse cholesterol transport (Abca1 and Abcg1) were both significantly repressed.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|LXR lacks alpha---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1H3\"],\"tg (associated gene name)\":[\"ABCG1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"LXR lacks alpha\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, the expression of \\\"classical\\\" liver X receptor (LXR) target genes involved in reverse cholesterol transport (Abca1 and Abcg1) were both significantly repressed.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR1H3\"],\"tg associated gene name\":[\"ABCG1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Furthermore\",\"the expression of \\\"classical\\\" liver X receptor (LXR) target genes involved in reverse cholesterol transport (Abca1 and Abcg1) were both significantly repressed.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR1H3\"],\"target gene (associated gene name)\":[\"ABCG1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Furthermore\",\"the expression of \\\"classical\\\" liver X receptor (LXR) target genes involved in reverse cholesterol transport (Abca1 and Abcg1) were both significantly repressed.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|LXR lacks alpha---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1H3\"],\"tg (associated gene name)\":[\"ABCG1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"LXR lacks alpha\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, the expression of \\\"classical\\\" liver X receptor (LXR) target genes involved in reverse cholesterol transport (Abca1 and Abcg1) were both significantly repressed.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"FOXO1\"],\"tg associated gene name\":[\"AKT1\"],\"tf text\":[\"FKHR\"],\"tg text\":[\"Akt\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"FKHR mutants with alanine substitutions at three Akt phosphorylation consensus sites (T24\",\"S256 and S319) were inhibited by Akt\",\"but mutation of all three sites rendered FKHR resistant to suppression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO1\"],\"tg (associated gene name)\":[\"AKT1\"],\"tf text\":[\"FKHR\"],\"tg text\":[\"Akt\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"FKHR mutants with alanine substitutions at three Akt phosphorylation consensus sites (T24, S256 and S319) were inhibited by Akt, but mutation of all three sites rendered FKHR resistant to suppression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"FOXO1\"],\"tg associated gene name\":[\"AKT1\"],\"tf text\":[\"FKHR\"],\"tg text\":[\"Akt\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"FKHR mutants with alanine substitutions at three Akt phosphorylation consensus sites (T24\",\"S256 and S319) were inhibited by Akt\",\"but mutation of all three sites rendered FKHR resistant to suppression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO1\"],\"tg (associated gene name)\":[\"AKT1\"],\"tf text\":[\"FKHR\"],\"tg text\":[\"Akt\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"FKHR mutants with alanine substitutions at three Akt phosphorylation consensus sites (T24, S256 and S319) were inhibited by Akt, but mutation of all three sites rendered FKHR resistant to suppression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TEF is TG|unclear weak TRE-statement---{\"tf associated gene name\":[\"TEF\"],\"tg associated gene name\":[\"CYP2B6\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TEF is TG\",\"unclear weak TRE-statement\"],\"sentence\":[\"We found that CLOCK was absolutely concerned with the circadian transcription of one type of liver genes (such as DBP\",\"TEF\",\"and Usp2) and partially with another (such as mPer1\",\"mPer2\",\"mDec1\",\"Nocturnin\",\"P450 oxidoreductase\",\"and FKBP51) because the latter were damped but remained rhythmic in the mutant mice.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CLOCK?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEF\"],\"tg (associated gene name)\":[\"CYP2B6\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CLOCK?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that CLOCK was absolutely concerned with the circadian transcription of one type of liver genes (such as DBP, TEF, and Usp2) and partially with another (such as mPer1, mPer2, mDec1, Nocturnin, P450 oxidoreductase, and FKBP51) because the latter were damped but remained rhythmic in the mutant mice.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TEF is TG|unclear weak TRE-statement---{\"tf associated gene name\":[\"TEF\"],\"tg associated gene name\":[\"CYP2B6\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TEF is TG\",\"unclear weak TRE-statement\"],\"sentence\":[\"We found that CLOCK was absolutely concerned with the circadian transcription of one type of liver genes (such as DBP\",\"TEF\",\"and Usp2) and partially with another (such as mPer1\",\"mPer2\",\"mDec1\",\"Nocturnin\",\"P450 oxidoreductase\",\"and FKBP51) because the latter were damped but remained rhythmic in the mutant mice.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CLOCK?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEF\"],\"tg (associated gene name)\":[\"CYP2B6\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CLOCK?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that CLOCK was absolutely concerned with the circadian transcription of one type of liver genes (such as DBP, TEF, and Usp2) and partially with another (such as mPer1, mPer2, mDec1, Nocturnin, P450 oxidoreductase, and FKBP51) because the latter were damped but remained rhythmic in the mutant mice.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE/GRE---{\"tf associated gene name\":[\"ELF1\"],\"tg associated gene name\":[\"TSC2\"],\"tf text\":[],\"tg text\":[\"TSC2\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE/GRE\"],\"sentence\":[\"Here\",\"we show that TSC1-TSC2 inhibits the p70 ribosomal protein S6 kinase 1 (an activator of translation) and activates the eukaryotic initiation factor 4E binding protein 1 (4E-BP1\",\"an inhibitor of translational initiation).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE/GRE---{\"tf associated gene name\":[\"ELF1\"],\"tg associated gene name\":[\"TSC2\"],\"tf text\":[],\"tg text\":[\"TSC2\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE/GRE\"],\"sentence\":[\"Here\",\"we show that TSC1-TSC2 inhibits the p70 ribosomal protein S6 kinase 1 (an activator of translation) and activates the eukaryotic initiation factor 4E binding protein 1 (4E-BP1\",\"an inhibitor of translational initiation).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF1\"],\"tg (associated gene name)\":[\"TSC2\"],\"tf text\":[],\"tg text\":[\"TSC2\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here, we show that TSC1-TSC2 inhibits the p70 ribosomal protein S6 kinase 1 (an activator of translation) and activates the eukaryotic initiation factor 4E binding protein 1 (4E-BP1, an inhibitor of translational initiation).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "T-problem|not TRE---{\"tf associated gene name\":[\"T\"],\"tg associated gene name\":[\"MUC1\"],\"tf text\":[],\"tg text\":[\"MUC1\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"T-problem\",\"not TRE\"],\"sentence\":[\"Low doses of IFN?? and TNF?? synergistically stimulate MUC1 promoter activity\",\"enhance PRB stimulation of MUC1 promoter activity and cooperate with PRA to stimulate MUC1 promoter activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|T?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"T\"],\"tg (associated gene name)\":[\"MUC1\"],\"tf text\":[],\"tg text\":[\"MUC1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"T?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Low doses of IFN and TNF synergistically stimulate MUC1 promoter activity, enhance PRB stimulation of MUC1 promoter activity and cooperate with PRA to stimulate MUC1 promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "T-problem|not TRE---{\"tf associated gene name\":[\"T\"],\"tg associated gene name\":[\"MUC1\"],\"tf text\":[],\"tg text\":[\"MUC1\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"T-problem\",\"not TRE\"],\"sentence\":[\"Low doses of IFN?? and TNF?? synergistically stimulate MUC1 promoter activity\",\"enhance PRB stimulation of MUC1 promoter activity and cooperate with PRA to stimulate MUC1 promoter activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|T?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"T\"],\"tg (associated gene name)\":[\"MUC1\"],\"tf text\":[],\"tg text\":[\"MUC1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"T?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Low doses of IFN and TNF synergistically stimulate MUC1 promoter activity, enhance PRB stimulation of MUC1 promoter activity and cooperate with PRA to stimulate MUC1 promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"NPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"The CCAAT/enhancer-binding protein ?? (C/EBP??) plays a major role in the pathogenesis of anaplastic large cell lymphomas (ALCL) that express the nucleophosmin-anaplastic lymphoma kinase (NPM-ALK) tyrosine kinase (ALK(+)).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"NPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"The CCAAT/enhancer-binding protein ?? (C/EBP??) plays a major role in the pathogenesis of anaplastic large cell lymphomas (ALCL) that express the nucleophosmin-anaplastic lymphoma kinase (NPM-ALK) tyrosine kinase (ALK(+)).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TXK\"],\"tg (associated gene name)\":[\"NPM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"The CCAAT/enhancer-binding protein  (C/EBP) plays a major role in the pathogenesis of anaplastic large cell lymphomas (ALCL) that express the nucleophosmin-anaplastic lymphoma kinase (NPM-ALK) tyrosine kinase (ALK(+)).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"RASSF1\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[\"RASSF1a\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"DNA methylation in the P16, RASSF1a, progesterone receptor (PGR) and estrogen receptor alpha (ERalpha) promoters was determined by quantitative bisulfite-pyrosequencing technique in HCC patients.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"RASSF1\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[\"RASSF1a\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"DNA methylation in the P16, RASSF1a, progesterone receptor (PGR) and estrogen receptor alpha (ERalpha) promoters was determined by quantitative bisulfite-pyrosequencing technique in HCC patients.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"We confirmed that increased expression of miR-206 and miR-29 resulted in the translational repression of HDAC4 in the presence or absence of TGF-?? via interaction with the HDAC4 3'-untranslated region.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"We confirmed that increased expression of miR-206 and miR-29 resulted in the translational repression of HDAC4 in the presence or absence of TGF-?? via interaction with the HDAC4 3'-untranslated region.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MLXIP\"],\"target gene (associated gene name)\":[\"TGFB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"We confirmed that increased expression of miR-206 and miR-29 resulted in the translational repression of HDAC4 in the presence or absence of TGF- via interaction with the HDAC4 3'-untranslated region.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.120805\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"200\"],\"sentence pair density\":[\"0.0208333333333333\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"We confirmed that increased expression of miR-206 and miR-29 resulted in the translational repression of HDAC4 in the presence or absence of TGF- via interaction with the HDAC4 3'-untranslated region.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KAT7\"],\"tg (associated gene name)\":[\"HDAC9\"],\"tf text\":[],\"tg text\":[\"histone deacetylase\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Transcriptional activation of NF-??B was measured by luciferase reporter assay, gene expression by real-time RT-PCR, DNA binding of transcription factors by chromatin immunoprecipitation assay, protein-protein interaction by coimmunoprecipitation/immunoblotting, histone deacetylase (HDAC)/histone acetyltransferase enzyme activity by fluorometry, and nucleosome eviction by partial microccocal DNase digestion.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KAT7\"],\"tg (associated gene name)\":[\"HDAC9\"],\"tf text\":[],\"tg text\":[\"histone deacetylase\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Transcriptional activation of NF-??B was measured by luciferase reporter assay, gene expression by real-time RT-PCR, DNA binding of transcription factors by chromatin immunoprecipitation assay, protein-protein interaction by coimmunoprecipitation/immunoblotting, histone deacetylase (HDAC)/histone acetyltransferase enzyme activity by fluorometry, and nucleosome eviction by partial microccocal DNase digestion.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs are rhodopsin and IRBP---{\"tf associated gene name\":[\"KLF15\"],\"tg associated gene name\":[\"GRB10\"],\"tf text\":[\"KLF15\"],\"tg text\":[\"IRBP\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TGs are rhodopsin and IRBP\"],\"sentence\":[\"KLF15 repressed transactivation of rhodopsin and IRBP promoters alone and in combination with the transcriptional activators Crx and/or Nrl.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TGs are rhodopsin and IRBP---{\"tf associated gene name\":[\"KLF15\"],\"tg associated gene name\":[\"GRB10\"],\"tf text\":[\"KLF15\"],\"tg text\":[\"IRBP\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TGs are rhodopsin and IRBP\"],\"sentence\":[\"KLF15 repressed transactivation of rhodopsin and IRBP promoters alone and in combination with the transcriptional activators Crx and/or Nrl.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"KLF15\"],\"target gene (associated gene name)\":[\"GRB10\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"KLF15 repressed transactivation of rhodopsin and IRBP promoters alone and in combination with the transcriptional activators Crx and/or Nrl.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KLF15\"],\"tg (associated gene name)\":[\"GRB10\"],\"tf text\":[\"KLF15\"],\"tg text\":[\"IRBP\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"KLF15 repressed transactivation of rhodopsin and IRBP promoters alone and in combination with the transcriptional activators Crx and/or Nrl.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG other gene---{\"tf associated gene name\":[\"MYB\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"c-Myb\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG other gene\"],\"sentence\":[\"We hypothesized that BQ can increase c-Myb activity\",\"which in turn increases Topo IIalpha promoter activity resulting in increased DNA strand breaks.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYB\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"c-Myb\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We hypothesized that BQ can increase c-Myb activity, which in turn increases Topo IIalpha promoter activity resulting in increased DNA strand breaks.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG other gene---{\"tf associated gene name\":[\"MYB\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"c-Myb\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG other gene\"],\"sentence\":[\"We hypothesized that BQ can increase c-Myb activity\",\"which in turn increases Topo IIalpha promoter activity resulting in increased DNA strand breaks.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYB\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"c-Myb\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We hypothesized that BQ can increase c-Myb activity, which in turn increases Topo IIalpha promoter activity resulting in increased DNA strand breaks.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 378 (char 377)\n",
      "{\"tf associated gene name\":[\"SMAD4\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[\"Smad4\"],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Smad4 acts as a cofactor that binds transforming growth factor-beta (TGF-beta) receptor-activated Smad2 and Smad3 generating transcriptional complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD4\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[\"Smad4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Smad4 acts as a cofactor that binds transforming growth factor-beta (TGF-beta) receptor-activated Smad2 and Smad3 generating transcriptional complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SMAD4\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[\"Smad4\"],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Smad4 acts as a cofactor that binds transforming growth factor-beta (TGF-beta) receptor-activated Smad2 and Smad3 generating transcriptional complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD4\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[\"Smad4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Smad4 acts as a cofactor that binds transforming growth factor-beta (TGF-beta) receptor-activated Smad2 and Smad3 generating transcriptional complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "indirect?---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CDH1\"],\"tf text\":[],\"tg text\":[\"E-cadherin\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"indirect?\"],\"sentence\":[\"NF-??B activation resulted in binding of ZEB-1 to the E-cadherin promoter as demonstrated by ChIP analysis with subsequent E-cadherin suppression (fold increase in ZEB-1 binding to the E-cadherin promoter region: untreated + ZEB-1 antibody vs CTGF(200 ng/mL) + ZEB-1 antibody: 1.5 vs 6.4\",\"difference = 4.9\",\"95% CI = 4.8 to 5.0\",\"P < .001).\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|role NFKB indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CDH1\"],\"tf text\":[],\"tg text\":[\"E-cadherin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"role NFKB indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"NF-B activation resulted in binding of ZEB-1 to the E-cadherin promoter as demonstrated by ChIP analysis with subsequent E-cadherin suppression (fold increase in ZEB-1 binding to the E-cadherin promoter region: untreated + ZEB-1 antibody vs CTGF(200 ng/mL) + ZEB-1 antibody: 1.5 vs 6.4, difference = 4.9, 95% CI = 4.8 to 5.0, P < .001).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "indirect?---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CDH1\"],\"tf text\":[],\"tg text\":[\"E-cadherin\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"indirect?\"],\"sentence\":[\"NF-??B activation resulted in binding of ZEB-1 to the E-cadherin promoter as demonstrated by ChIP analysis with subsequent E-cadherin suppression (fold increase in ZEB-1 binding to the E-cadherin promoter region: untreated + ZEB-1 antibody vs CTGF(200 ng/mL) + ZEB-1 antibody: 1.5 vs 6.4\",\"difference = 4.9\",\"95% CI = 4.8 to 5.0\",\"P < .001).\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|role NFKB indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CDH1\"],\"tf text\":[],\"tg text\":[\"E-cadherin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"role NFKB indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"NF-B activation resulted in binding of ZEB-1 to the E-cadherin promoter as demonstrated by ChIP analysis with subsequent E-cadherin suppression (fold increase in ZEB-1 binding to the E-cadherin promoter region: untreated + ZEB-1 antibody vs CTGF(200 ng/mL) + ZEB-1 antibody: 1.5 vs 6.4, difference = 4.9, 95% CI = 4.8 to 5.0, P < .001).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is MARE-CEB; however insulin as TG is indirectly linked via homolous TFbinding site---{\"tf associated gene name\":[\"MAFA\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[\"MafA\"],\"tg text\":[\"insulin\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is MARE-CEB; however insulin as TG is indirectly linked via homolous TFbinding site\"],\"sentence\":[\"The MARE-CEB is repressed by MafA\",\"whereas the CEB-MARE site\",\"which is homologous to the A2C1 component of the glucose-sensitive RIPE3b region of the insulin gene promoter\",\"is activated by MafA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|homology only---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAFA\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"MafA\"],\"tg text\":[\"insulin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"homology only\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The MARE-CEB is repressed by MafA, whereas the CEB-MARE site, which is homologous to the A2C1 component of the glucose-sensitive RIPE3b region of the insulin gene promoter, is activated by MafA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is MARE-CEB; however insulin as TG is indirectly linked via homolous TFbinding site---{\"tf associated gene name\":[\"MAFA\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[\"MafA\"],\"tg text\":[\"insulin\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is MARE-CEB; however insulin as TG is indirectly linked via homolous TFbinding site\"],\"sentence\":[\"The MARE-CEB is repressed by MafA\",\"whereas the CEB-MARE site\",\"which is homologous to the A2C1 component of the glucose-sensitive RIPE3b region of the insulin gene promoter\",\"is activated by MafA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|homology only---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAFA\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"MafA\"],\"tg text\":[\"insulin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"homology only\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The MARE-CEB is repressed by MafA, whereas the CEB-MARE site, which is homologous to the A2C1 component of the glucose-sensitive RIPE3b region of the insulin gene promoter, is activated by MafA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 559 (char 558)\n",
      "{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"RELA\"],\"tf text\":[\"p65\"],\"tg text\":[\"p65\"],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"p\"],\"comment\":[],\"sentence\":[\"Dexamethasone represses interleukin-1beta-stimulated histone acetylation and granulocyte-macrophage colony-stimulating factor expression through a combination of direct inhibition of p65-associated histone acetyltransferase (HAT) activity and by recruiting histone deacetylase 2 (HDAC2) to the p65-HAT complex.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"RELA\"],\"tf text\":[\"p65\"],\"tg text\":[\"p65\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Dexamethasone represses interleukin-1beta-stimulated histone acetylation and granulocyte-macrophage colony-stimulating factor expression through a combination of direct inhibition of p65-associated histone acetyltransferase (HAT) activity and by recruiting histone deacetylase 2 (HDAC2) to the p65-HAT complex.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"RELA\"],\"tf text\":[\"p65\"],\"tg text\":[\"p65\"],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"p\"],\"comment\":[],\"sentence\":[\"Dexamethasone represses interleukin-1beta-stimulated histone acetylation and granulocyte-macrophage colony-stimulating factor expression through a combination of direct inhibition of p65-associated histone acetyltransferase (HAT) activity and by recruiting histone deacetylase 2 (HDAC2) to the p65-HAT complex.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"RELA\"],\"tf text\":[\"p65\"],\"tg text\":[\"p65\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Dexamethasone represses interleukin-1beta-stimulated histone acetylation and granulocyte-macrophage colony-stimulating factor expression through a combination of direct inhibition of p65-associated histone acetyltransferase (HAT) activity and by recruiting histone deacetylase 2 (HDAC2) to the p65-HAT complex.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 381 (char 380)\n",
      "{\"transcription factor (associated gene name)\":[\"SMAD3\"],\"target gene (associated gene name)\":[\"GLI2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Cloning of the human GLI2 Promoter: transcriptional activation by transforming growth factor-beta via SMAD3/beta-catenin cooperation.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|SMAD3 cooperation enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD3\"],\"tg (associated gene name)\":[\"GLI2\"],\"tf text\":[\"SMAD3\"],\"tg text\":[\"GLI2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"SMAD3 cooperation enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Cloning of the human GLI2 Promoter: transcriptional activation by transforming growth factor-beta via SMAD3/beta-catenin cooperation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SMAD3\"],\"tg associated gene name\":[\"GLI2\"],\"tf text\":[\"SMAD3\"],\"tg text\":[\"GLI2\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Cloning of the human GLI2 Promoter: transcriptional activation by transforming growth factor-beta via SMAD3/beta-catenin cooperation.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SMAD3\"],\"target gene (associated gene name)\":[\"GLI2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Cloning of the human GLI2 Promoter: transcriptional activation by transforming growth factor-beta via SMAD3/beta-catenin cooperation.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|SMAD3 cooperation enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD3\"],\"tg (associated gene name)\":[\"GLI2\"],\"tf text\":[\"SMAD3\"],\"tg text\":[\"GLI2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"SMAD3 cooperation enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Cloning of the human GLI2 Promoter: transcriptional activation by transforming growth factor-beta via SMAD3/beta-catenin cooperation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GNAS-problem; unclear/no TRE---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"GNAS\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"GNAS-problem; unclear/no TRE\"],\"sentence\":[\"A deletion analysis of the hb promoter using lacZ reporter constructs defines a 1.2-kb genomic DNA fragment surrounding the P1 promoter that is sufficient to reproduce the wild-type pattern of hb ovarian transcript accumulation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"GNAS\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"A deletion analysis of the hb promoter using lacZ reporter constructs defines a 1.2-kb genomic DNA fragment surrounding the P1 promoter that is sufficient to reproduce the wild-type pattern of hb ovarian transcript accumulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "GNAS-problem; unclear/no TRE---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"GNAS\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"GNAS-problem; unclear/no TRE\"],\"sentence\":[\"A deletion analysis of the hb promoter using lacZ reporter constructs defines a 1.2-kb genomic DNA fragment surrounding the P1 promoter that is sufficient to reproduce the wild-type pattern of hb ovarian transcript accumulation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"GNAS\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"A deletion analysis of the hb promoter using lacZ reporter constructs defines a 1.2-kb genomic DNA fragment surrounding the P1 promoter that is sufficient to reproduce the wild-type pattern of hb ovarian transcript accumulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "auto?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"FOXP3\"],\"tf text\":[\"Foxp3\"],\"tg text\":[\"Foxp3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"auto?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In contrast, RAPA prevented activation-dependent DNA methylation of the Foxp3 promoter favoring Foxp3 expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "auto?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"FOXP3\"],\"tf text\":[\"Foxp3\"],\"tg text\":[\"Foxp3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"auto?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In contrast, RAPA prevented activation-dependent DNA methylation of the Foxp3 promoter favoring Foxp3 expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE; AR is Aldose reductase---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"n\"],\"comment\":[\"not TRE; AR is Aldose reductase\"],\"sentence\":[\"Aldose reductase (AR) was proven to be one of the TGF-beta1 responsive genes in cultured rat mesangial cells using the SSH-PCR method and there were positive correlation between the AR and TGF-beta1 in our previous studies.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Aldose reductase (AR) was proven to be one of the TGF-beta1 responsive genes in cultured rat mesangial cells using the SSH-PCR method and there were positive correlation between the AR and TGF-beta1 in our previous studies.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE; AR is Aldose reductase---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"n\"],\"comment\":[\"not TRE; AR is Aldose reductase\"],\"sentence\":[\"Aldose reductase (AR) was proven to be one of the TGF-beta1 responsive genes in cultured rat mesangial cells using the SSH-PCR method and there were positive correlation between the AR and TGF-beta1 in our previous studies.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Aldose reductase (AR) was proven to be one of the TGF-beta1 responsive genes in cultured rat mesangial cells using the SSH-PCR method and there were positive correlation between the AR and TGF-beta1 in our previous studies.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is RBB|most likely not cofrectly normalized---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[],\"tg text\":[\"HDM2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is RBB\",\"most likely not cofrectly normalized\"],\"sentence\":[\"Genome wide transcription target analysis by ChIP sequencing revealed that RBB represses the transcription of a series of functionally important genes including HDM2.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|RBB?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[],\"tg text\":[\"HDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"RBB?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Genome wide transcription target analysis by ChIP sequencing revealed that RBB represses the transcription of a series of functionally important genes including HDM2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is RBB|most likely not cofrectly normalized---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[],\"tg text\":[\"HDM2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is RBB\",\"most likely not cofrectly normalized\"],\"sentence\":[\"Genome wide transcription target analysis by ChIP sequencing revealed that RBB represses the transcription of a series of functionally important genes including HDM2.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|RBB?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[],\"tg text\":[\"HDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"RBB?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Genome wide transcription target analysis by ChIP sequencing revealed that RBB represses the transcription of a series of functionally important genes including HDM2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is interferon (IFN)-beta---{\"tf associated gene name\":[\"IRF3\"],\"tg associated gene name\":[\"TICAM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is interferon (IFN)-beta\"],\"sentence\":[\"TICAM-1 is known to induce the activation of NF-kappaB and IRF-3\",\"which leads to activation of the interferon (IFN)-beta promoter\",\"an activity that is conserved in the N+TIR fragment (aa 1-533).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF3\"],\"tg (associated gene name)\":[\"TICAM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TICAM-1 is known to induce the activation of NF-kappaB and IRF-3, which leads to activation of the interferon (IFN)-beta promoter, an activity that is conserved in the N+TIR fragment (aa 1-533).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is interferon (IFN)-beta---{\"tf associated gene name\":[\"IRF3\"],\"tg associated gene name\":[\"TICAM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is interferon (IFN)-beta\"],\"sentence\":[\"TICAM-1 is known to induce the activation of NF-kappaB and IRF-3\",\"which leads to activation of the interferon (IFN)-beta promoter\",\"an activity that is conserved in the N+TIR fragment (aa 1-533).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF3\"],\"tg (associated gene name)\":[\"TICAM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TICAM-1 is known to induce the activation of NF-kappaB and IRF-3, which leads to activation of the interferon (IFN)-beta promoter, an activity that is conserved in the N+TIR fragment (aa 1-533).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs not named---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"RNF20\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TGs not named\"],\"sentence\":[\"In accordance with these findings\",\"RNF20 represses the transcriptional activity of SREBP1c and turns off the expression of lipogenic genes that are targets of SREBP1c.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped|REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"RNF20\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[\"In accordance with these findings, RNF20 represses the transcriptional activity of SREBP1c and turns off the expression of lipogenic genes that are targets of SREBP1c.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TGs not named---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"RNF20\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TGs not named\"],\"sentence\":[\"In accordance with these findings\",\"RNF20 represses the transcriptional activity of SREBP1c and turns off the expression of lipogenic genes that are targets of SREBP1c.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped|REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"RNF20\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[\"In accordance with these findings, RNF20 represses the transcriptional activity of SREBP1c and turns off the expression of lipogenic genes that are targets of SREBP1c.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Pax6---{\"tf associated gene name\":[\"CTCF\"],\"tg associated gene name\":[\"MPZ\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Pax6\"],\"sentence\":[\"More importantly\",\"the CTCF binding sites upstream from Pax6 p0 promoter were required for regulating p0 promoter activity in islet ??-cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Pax6---{\"tf associated gene name\":[\"CTCF\"],\"tg associated gene name\":[\"MPZ\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Pax6\"],\"sentence\":[\"More importantly\",\"the CTCF binding sites upstream from Pax6 p0 promoter were required for regulating p0 promoter activity in islet ??-cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CTCF\"],\"target gene (associated gene name)\":[\"MPZ\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"More importantly\",\"the CTCF binding sites upstream from Pax6 p0 promoter were required for regulating p0 promoter activity in islet -cells.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|p0 is MPZ alilas, but it MPZ not what is meant in sentence---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CTCF\"],\"tg (associated gene name)\":[\"MPZ\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[\"p0 is MPZ alilas, but it MPZ not what is meant in sentence\"],\"sentences with greek letters\":[\"More importantly, the CTCF binding sites upstream from Pax6 p0 promoter were required for regulating p0 promoter activity in islet -cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 736 (char 735)\n",
      "{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"EGR1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"In Med23-/- cells\",\"histone acetylation\",\"methylation\",\"and chromatin remodeling complex association at the Egr1 promoter were equivalent to that of wild-type cells\",\"yet Egr1 induction was greatly reduced.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.779411\"],\"pmid counts\":[\"92\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"202\"],\"sentence pair density\":[\"0.0648648648648648\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"EGR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In Med23-/- cells, histone acetylation, methylation, and chromatin remodeling complex association at the Egr1 promoter were equivalent to that of wild-type cells, yet Egr1 induction was greatly reduced.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"EGR1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"In Med23-/- cells\",\"histone acetylation\",\"methylation\",\"and chromatin remodeling complex association at the Egr1 promoter were equivalent to that of wild-type cells\",\"yet Egr1 induction was greatly reduced.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.779411\"],\"pmid counts\":[\"92\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"202\"],\"sentence pair density\":[\"0.0648648648648648\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"EGR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In Med23-/- cells, histone acetylation, methylation, and chromatin remodeling complex association at the Egr1 promoter were equivalent to that of wild-type cells, yet Egr1 induction was greatly reduced.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not clear TRE---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"GH1\"],\"tf text\":[\"STAT5\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f?\"],\"comment\":[\"not clear TRE\"],\"sentence\":[\"Particular attention will be given to the novel role that STAT5 plays in sexually dimorphic gene expression in the liver as determined by the secretory pattern of GH and the role of STAT5 in body growth.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|secretory pattern?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"GH1\"],\"tf text\":[\"STAT5\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"secretory pattern?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Particular attention will be given to the novel role that STAT5 plays in sexually dimorphic gene expression in the liver as determined by the secretory pattern of GH and the role of STAT5 in body growth.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not clear TRE---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"GH1\"],\"tf text\":[\"STAT5\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f?\"],\"comment\":[\"not clear TRE\"],\"sentence\":[\"Particular attention will be given to the novel role that STAT5 plays in sexually dimorphic gene expression in the liver as determined by the secretory pattern of GH and the role of STAT5 in body growth.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|secretory pattern?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"GH1\"],\"tf text\":[\"STAT5\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"secretory pattern?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Particular attention will be given to the novel role that STAT5 plays in sexually dimorphic gene expression in the liver as determined by the secretory pattern of GH and the role of STAT5 in body growth.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 394 (char 393)\n",
      "{\"transcription factor (associated gene name)\":[\"PPARA\"],\"target gene (associated gene name)\":[\"CDKN2A\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results identify an unexpected role for p16 in SMC cell-cycle control and demonstrate that PPARalpha inhibits SMC proliferation through p16.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hm, 'unexpected? Maybe P16 is wrong?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"CDKN2A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"hm, 'unexpected? Maybe P16 is wrong?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results identify an unexpected role for p16 in SMC cell-cycle control and demonstrate that PPARalpha inhibits SMC proliferation through p16.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"CDKN2A\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"These results identify an unexpected role for p16 in SMC cell-cycle control and demonstrate that PPARalpha inhibits SMC proliferation through p16.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PPARA\"],\"target gene (associated gene name)\":[\"CDKN2A\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results identify an unexpected role for p16 in SMC cell-cycle control and demonstrate that PPARalpha inhibits SMC proliferation through p16.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hm, 'unexpected? Maybe P16 is wrong?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"CDKN2A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"hm, 'unexpected? Maybe P16 is wrong?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results identify an unexpected role for p16 in SMC cell-cycle control and demonstrate that PPARalpha inhibits SMC proliferation through p16.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Hnf4?? |TGs are Oat1|Oat3|and Oct1---{\"tf associated gene name\":[\"POU2F1\"],\"tg associated gene name\":[\"SLC22A6\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is Hnf4?? \",\"TGs are Oat1\",\"Oat3\",\"and Oct1\"],\"sentence\":[\"Taken together with the ChIP-qPCR finding that Hnf4?? occupies Oat1\",\"Oat3\",\"and Oct1 proximal promoters in the in vivo differentiating rat kidney\",\"the data suggest a network of genes with Hnf4?? at its center plays a role in regulating the terminal differentiation and capacity for drug and toxin handling by the nascent proximal tubule of the kidney.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|POU2F1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU2F1\"],\"tg (associated gene name)\":[\"SLC22A6\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"POU2F1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together with the ChIP-qPCR finding that Hnf4 occupies Oat1, Oat3, and Oct1 proximal promoters in the in vivo differentiating rat kidney, the data suggest a network of genes with Hnf4 at its center plays a role in regulating the terminal differentiation and capacity for drug and toxin handling by the nascent proximal tubule of the kidney.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is Hnf4?? |TGs are Oat1|Oat3|and Oct1---{\"tf associated gene name\":[\"POU2F1\"],\"tg associated gene name\":[\"SLC22A6\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is Hnf4?? \",\"TGs are Oat1\",\"Oat3\",\"and Oct1\"],\"sentence\":[\"Taken together with the ChIP-qPCR finding that Hnf4?? occupies Oat1\",\"Oat3\",\"and Oct1 proximal promoters in the in vivo differentiating rat kidney\",\"the data suggest a network of genes with Hnf4?? at its center plays a role in regulating the terminal differentiation and capacity for drug and toxin handling by the nascent proximal tubule of the kidney.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|POU2F1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU2F1\"],\"tg (associated gene name)\":[\"SLC22A6\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"POU2F1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together with the ChIP-qPCR finding that Hnf4 occupies Oat1, Oat3, and Oct1 proximal promoters in the in vivo differentiating rat kidney, the data suggest a network of genes with Hnf4 at its center plays a role in regulating the terminal differentiation and capacity for drug and toxin handling by the nascent proximal tubule of the kidney.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 484 (char 483)\n",
      "{\"tf associated gene name\":[\"NFKBIA\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[\"IkappaBalpha\"],\"tg text\":[\"TNFalpha\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In contrast\",\"either the proteasome inhibitor carbobenzoxy-L-leucy-L-leucy-L-leucinal (MG 132) or the IkappaBalpha inhibitor BAY 11-7082 ablated TNFalpha-induced ICAM-1 gene expression and MG132 inhibited TNFalpha-induced NFkappaB complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TNFalpha induces?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKBIA\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[\"IkappaBalpha\"],\"tg text\":[\"TNFalpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TNFalpha induces?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast, either the proteasome inhibitor carbobenzoxy-L-leucy-L-leucy-L-leucinal (MG 132) or the IkappaBalpha inhibitor BAY 11-7082 ablated TNFalpha-induced ICAM-1 gene expression and MG132 inhibited TNFalpha-induced NFkappaB complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKBIA\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[\"IkappaBalpha\"],\"tg text\":[\"TNFalpha\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In contrast\",\"either the proteasome inhibitor carbobenzoxy-L-leucy-L-leucy-L-leucinal (MG 132) or the IkappaBalpha inhibitor BAY 11-7082 ablated TNFalpha-induced ICAM-1 gene expression and MG132 inhibited TNFalpha-induced NFkappaB complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TNFalpha induces?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKBIA\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[\"IkappaBalpha\"],\"tg text\":[\"TNFalpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TNFalpha induces?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast, either the proteasome inhibitor carbobenzoxy-L-leucy-L-leucy-L-leucinal (MG 132) or the IkappaBalpha inhibitor BAY 11-7082 ablated TNFalpha-induced ICAM-1 gene expression and MG132 inhibited TNFalpha-induced NFkappaB complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 487 (char 486)\n",
      "{\"transcription factor (associated gene name)\":[\"RELA\"],\"target gene (associated gene name)\":[\"CCL2\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The results suggest that the interaction of p65p65 homodimers and C/EBP transcriptional factors with IL-8 promoters and not MCP-1 promoters account for the capacity of HCECs to produce IL-8 selectively\",\"in the absence of MCP-1 production.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|bwaah, why MCP-1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"CCL2\"],\"tf text\":[],\"tg text\":[\"MCP-1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"bwaah, why MCP-1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results suggest that the interaction of p65p65 homodimers and C/EBP transcriptional factors with IL-8 promoters and not MCP-1 promoters account for the capacity of HCECs to produce IL-8 selectively, in the absence of MCP-1 production.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"CCL2\"],\"tf text\":[],\"tg text\":[\"MCP-1\"],\"1\":[\"u\"],\"0\":[],\"negation\":[\"x\"],\"comment\":[],\"sentence\":[\"The results suggest that the interaction of p65p65 homodimers and C/EBP transcriptional factors with IL-8 promoters and not MCP-1 promoters account for the capacity of HCECs to produce IL-8 selectively\",\"in the absence of MCP-1 production.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RELA\"],\"target gene (associated gene name)\":[\"CCL2\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The results suggest that the interaction of p65p65 homodimers and C/EBP transcriptional factors with IL-8 promoters and not MCP-1 promoters account for the capacity of HCECs to produce IL-8 selectively\",\"in the absence of MCP-1 production.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|bwaah, why MCP-1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"CCL2\"],\"tf text\":[],\"tg text\":[\"MCP-1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"bwaah, why MCP-1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results suggest that the interaction of p65p65 homodimers and C/EBP transcriptional factors with IL-8 promoters and not MCP-1 promoters account for the capacity of HCECs to produce IL-8 selectively, in the absence of MCP-1 production.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "YY-problem|no TF named---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"CIITA\"],\"tf text\":[],\"tg text\":[\"CIITA\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"YY-problem\",\"no TF named\"],\"sentence\":[\"Role of PKCdelta in IFN-gamma-inducible CIITA gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|delta TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"CIITA\"],\"tf text\":[],\"tg text\":[\"CIITA\"],\"consensus\":[\"N\"],\"martin_comments\":[\"delta TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Role of PKCdelta in IFN-gamma-inducible CIITA gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "YY-problem|no TF named---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"CIITA\"],\"tf text\":[],\"tg text\":[\"CIITA\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"YY-problem\",\"no TF named\"],\"sentence\":[\"Role of PKCdelta in IFN-gamma-inducible CIITA gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|delta TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YY1\"],\"tg (associated gene name)\":[\"CIITA\"],\"tf text\":[],\"tg text\":[\"CIITA\"],\"consensus\":[\"N\"],\"martin_comments\":[\"delta TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Role of PKCdelta in IFN-gamma-inducible CIITA gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ER stress---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"DDIT4\"],\"tf text\":[],\"tg text\":[\"Redd1\"],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"ER stress\"],\"sentence\":[\"In our experiment\",\"overexpression of ATF4 was associated with reduced mTOR activity via Redd1 expression\",\"whereas suppression of ATF4 levels with small interfering RNA led to the recovery of decreased mTOR activity mediated by downregulation of Redd1 during oxidative and ER stress.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|esr=atf4?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"DDIT4\"],\"tf text\":[],\"tg text\":[\"Redd1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"esr=atf4?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In our experiment, overexpression of ATF4 was associated with reduced mTOR activity via Redd1 expression, whereas suppression of ATF4 levels with small interfering RNA led to the recovery of decreased mTOR activity mediated by downregulation of Redd1 during oxidative and ER stress.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "ER stress---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"DDIT4\"],\"tf text\":[],\"tg text\":[\"Redd1\"],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"ER stress\"],\"sentence\":[\"In our experiment\",\"overexpression of ATF4 was associated with reduced mTOR activity via Redd1 expression\",\"whereas suppression of ATF4 levels with small interfering RNA led to the recovery of decreased mTOR activity mediated by downregulation of Redd1 during oxidative and ER stress.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|esr=atf4?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"DDIT4\"],\"tf text\":[],\"tg text\":[\"Redd1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"esr=atf4?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In our experiment, overexpression of ATF4 was associated with reduced mTOR activity via Redd1 expression, whereas suppression of ATF4 levels with small interfering RNA led to the recovery of decreased mTOR activity mediated by downregulation of Redd1 during oxidative and ER stress.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Consistent with a direct mechanism of target regulation by c-Jun\",\"gel shift analysis of selected AP-1-containing promoter regions revealed elevated and specific binding by proteins present in nuclear extracts of c-Jun expressing MCF7 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG not named---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Consistent with a direct mechanism of target regulation by c-Jun\",\"gel shift analysis of selected AP-1-containing promoter regions revealed elevated and specific binding by proteins present in nuclear extracts of c-Jun expressing MCF7 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consistent with a direct mechanism of target regulation by c-Jun, gel shift analysis of selected AP-1-containing promoter regions revealed elevated and specific binding by proteins present in nuclear extracts of c-Jun expressing MCF7 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "correct autoregulation---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"SPI1\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"PU.1\"],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[\"correct autoregulation\"],\"sentence\":[\"Positive autoregulation of the PU.1 promoter may play an important role in the function of PU.1 in myeloid cells.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "correct autoregulation---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"SPI1\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"PU.1\"],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[\"correct autoregulation\"],\"sentence\":[\"Positive autoregulation of the PU.1 promoter may play an important role in the function of PU.1 in myeloid cells.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SPI1\"],\"target gene (associated gene name)\":[\"SPI1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Positive autoregulation of the PU.1 promoter may play an important role in the function of PU.1 in myeloid cells.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"SPI1\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"PU.1\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Positive autoregulation of the PU.1 promoter may play an important role in the function of PU.1 in myeloid cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "indirect (i.e. most likely not TRE)---{\"tf associated gene name\":[\"SMAD2\"],\"tg associated gene name\":[\"POU5F1\"],\"tf text\":[\"Smad2\"],\"tg text\":[\"Oct4\"],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[\"indirect (i.e. most likely not TRE)\"],\"sentence\":[\"Additionally\",\"induction of Cdx2 expression\",\"as a result of decreased Smad2 expression\",\"leads to repression of Oct4 expression\",\"which\",\"together with the decreased Nanog expression\",\"accelerates the loss of pluripotency.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "indirect (i.e. most likely not TRE)---{\"tf associated gene name\":[\"SMAD2\"],\"tg associated gene name\":[\"POU5F1\"],\"tf text\":[\"Smad2\"],\"tg text\":[\"Oct4\"],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[\"indirect (i.e. most likely not TRE)\"],\"sentence\":[\"Additionally\",\"induction of Cdx2 expression\",\"as a result of decreased Smad2 expression\",\"leads to repression of Oct4 expression\",\"which\",\"together with the decreased Nanog expression\",\"accelerates the loss of pluripotency.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor\":[\"SMAD2\"],\"target gene\":[\"POU5F1\"],\"sentence\":[\"Additionally\",\"induction of Cdx2 expression\",\"as a result of decreased Smad2 expression\",\"leads to repression of Oct4 expression\",\"which\",\"together with the decreased Nanog expression\",\"accelerates the loss of pluripotency.\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"fusion tf\":[],\"qc'ed\":[\"0\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"SMAD2\"],\"target gene (associated gene name)\":[\"POU5F1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Additionally\",\"induction of Cdx2 expression\",\"as a result of decreased Smad2 expression\",\"leads to repression of Oct4 expression\",\"which\",\"together with the decreased Nanog expression\",\"accelerates the loss of pluripotency.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.399816\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"7\"],\"sentence length\":[\"217\"],\"sentence pair density\":[\"0.0109890109890109\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not direct---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD2\"],\"tg (associated gene name)\":[\"POU5F1\"],\"tf text\":[\"Smad2\"],\"tg text\":[\"Oct-04\"],\"consensus\":[\"V\"],\"martin_comments\":[\"not direct\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Additionally, induction of Cdx2 expression, as a result of decreased Smad2 expression, leads to repression of Oct4 expression, which, together with the decreased Nanog expression, accelerates the loss of pluripotency.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is TXNIP---{\"tf associated gene name\":[\"FOXO1\"],\"tg associated gene name\":[\"MLXIPL\"],\"tf text\":[\"FOXO1\"],\"tg text\":[\"ChREBP\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is TXNIP\"],\"sentence\":[\"Moreover\",\"FOXO1 blocked glucose-induced TXNIP expression and reduced glucose-induced ChREBP binding at the TXNIP promoter without affecting ChREBP expression or nuclear localization\",\"suggesting that FOXO1 may compete with ChREBP for binding to the TXNIP promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is TXNIP---{\"tf associated gene name\":[\"FOXO1\"],\"tg associated gene name\":[\"MLXIPL\"],\"tf text\":[\"FOXO1\"],\"tg text\":[\"ChREBP\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is TXNIP\"],\"sentence\":[\"Moreover\",\"FOXO1 blocked glucose-induced TXNIP expression and reduced glucose-induced ChREBP binding at the TXNIP promoter without affecting ChREBP expression or nuclear localization\",\"suggesting that FOXO1 may compete with ChREBP for binding to the TXNIP promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXO1\"],\"target gene (associated gene name)\":[\"MLXIPL\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Moreover\",\"FOXO1 blocked glucose-induced TXNIP expression and reduced glucose-induced ChREBP binding at the TXNIP promoter without affecting ChREBP expression or nuclear localization\",\"suggesting that FOXO1 may compete with ChREBP for binding to the TXNIP promoter.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.033121\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"4\"],\"sentence length\":[\"263\"],\"sentence pair density\":[\"0.0245098039215686\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO1\"],\"tg (associated gene name)\":[\"MLXIPL\"],\"tf text\":[\"FOXO1\"],\"tg text\":[\"ChREBP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, FOXO1 blocked glucose-induced TXNIP expression and reduced glucose-induced ChREBP binding at the TXNIP promoter without affecting ChREBP expression or nuclear localization, suggesting that FOXO1 may compete with ChREBP for binding to the TXNIP promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "other TFs---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"SLC1A2\"],\"tf text\":[],\"tg text\":[\"EAAT2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"other TFs\"],\"sentence\":[\"Gain- and loss-of-function studies in normal primary human fetal astrocytes and T98G glioblastoma multiforme cells revealed that AEG-1 repressed EAAT2 expression at a transcriptional level by inducing YY1 activity to inhibit CBP function as a coactivator on the EAAT2 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"SLC1A2\"],\"tf text\":[],\"tg text\":[\"EAAT2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gain- and loss-of-function studies in normal primary human fetal astrocytes and T98G glioblastoma multiforme cells revealed that AEG-1 repressed EAAT2 expression at a transcriptional level by inducing YY1 activity to inhibit CBP function as a coactivator on the EAAT2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "other TFs---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"SLC1A2\"],\"tf text\":[],\"tg text\":[\"EAAT2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"other TFs\"],\"sentence\":[\"Gain- and loss-of-function studies in normal primary human fetal astrocytes and T98G glioblastoma multiforme cells revealed that AEG-1 repressed EAAT2 expression at a transcriptional level by inducing YY1 activity to inhibit CBP function as a coactivator on the EAAT2 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"SLC1A2\"],\"tf text\":[],\"tg text\":[\"EAAT2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gain- and loss-of-function studies in normal primary human fetal astrocytes and T98G glioblastoma multiforme cells revealed that AEG-1 repressed EAAT2 expression at a transcriptional level by inducing YY1 activity to inhibit CBP function as a coactivator on the EAAT2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TFs are AP-1 and SP1---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[\"VEGF\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TFs are AP-1 and SP1\"],\"sentence\":[\"The tyrosine kinase inhibitor genistein and AP-1 inhibitor curcumin significantly blocked TGF-beta induction of VEGF expression while SP-1 and MKK1 inhibitors did not.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TXK not a TF|TXK is DbTF??---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TXK\"],\"tg (associated gene name)\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[\"VEGF\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TXK not a TF\"],\"astrid_comments\":[\"TXK is DbTF??\"],\"sentences with greek letters\":[\"The tyrosine kinase inhibitor genistein and AP-1 inhibitor curcumin significantly blocked TGF-beta induction of VEGF expression while SP-1 and MKK1 inhibitors did not.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TFs are AP-1 and SP1---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[\"VEGF\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TFs are AP-1 and SP1\"],\"sentence\":[\"The tyrosine kinase inhibitor genistein and AP-1 inhibitor curcumin significantly blocked TGF-beta induction of VEGF expression while SP-1 and MKK1 inhibitors did not.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TXK not a TF|TXK is DbTF??---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TXK\"],\"tg (associated gene name)\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[\"VEGF\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TXK not a TF\"],\"astrid_comments\":[\"TXK is DbTF??\"],\"sentences with greek letters\":[\"The tyrosine kinase inhibitor genistein and AP-1 inhibitor curcumin significantly blocked TGF-beta induction of VEGF expression while SP-1 and MKK1 inhibitors did not.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 355 (char 354)\n",
      "{\"transcription factor (associated gene name)\":[\"POU5F1\"],\"target gene (associated gene name)\":[\"SOX17\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"This switch allows the cells to turn off the pluripotency Oct4-Sox2 loop and to turn on the Sox17 promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hmm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"SOX17\"],\"tf text\":[\"Oct-04\"],\"tg text\":[\"Sox17\"],\"consensus\":[\"V\"],\"martin_comments\":[\"hmm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This switch allows the cells to turn off the pluripotency Oct4-Sox2 loop and to turn on the Sox17 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"SOX17\"],\"tf text\":[\"Oct4\"],\"tg text\":[\"Sox17\"],\"1\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"This switch allows the cells to turn off the pluripotency Oct4-Sox2 loop and to turn on the Sox17 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"POU5F1\"],\"target gene (associated gene name)\":[\"SOX17\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"This switch allows the cells to turn off the pluripotency Oct4-Sox2 loop and to turn on the Sox17 promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hmm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"SOX17\"],\"tf text\":[\"Oct-04\"],\"tg text\":[\"Sox17\"],\"consensus\":[\"V\"],\"martin_comments\":[\"hmm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This switch allows the cells to turn off the pluripotency Oct4-Sox2 loop and to turn on the Sox17 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 356 (char 355)\n",
      "{\"tf associated gene name\":[\"NR1H3\"],\"tg associated gene name\":[\"ABCA1\"],\"tf text\":[],\"tg text\":[\"ABCA1\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Environmentally relevant arsenic doses decrease the LXR/RXR ligand-induced expression of the LXR target genes (ABCA1 and SREBP-1c).\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|NR1H4?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1H3\"],\"tg (associated gene name)\":[\"ABCA1\"],\"tf text\":[],\"tg text\":[\"ABCA1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"NR1H4?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Environmentally relevant arsenic doses decrease the LXR/RXR ligand-induced expression of the LXR target genes (ABCA1 and SREBP-1c).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR1H3\"],\"tg associated gene name\":[\"ABCA1\"],\"tf text\":[],\"tg text\":[\"ABCA1\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Environmentally relevant arsenic doses decrease the LXR/RXR ligand-induced expression of the LXR target genes (ABCA1 and SREBP-1c).\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|NR1H4?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1H3\"],\"tg (associated gene name)\":[\"ABCA1\"],\"tf text\":[],\"tg text\":[\"ABCA1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"NR1H4?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Environmentally relevant arsenic doses decrease the LXR/RXR ligand-induced expression of the LXR target genes (ABCA1 and SREBP-1c).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 535 (char 534)\n",
      "{\"tf associated gene name\":[\"ZHX2\"],\"tg associated gene name\":[\"MAPK3\"],\"tf text\":[],\"tg text\":[\"ERK1\"],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"We further studied the activation and inhibition of uPA promoter by co-expression of a transactivation domain lacking c-jun: a dominant negative ERK1 and ERK2 mutant and a dominant negative c-raf in glioblastoma cell line showed repressed uPA promoter activity compared with the effect of the empty expression vector.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|c-raf = zhhx2? uPA?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZHX2\"],\"tg (associated gene name)\":[\"MAPK3\"],\"tf text\":[],\"tg text\":[\"ERK1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"c-raf = zhhx2? uPA?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We further studied the activation and inhibition of uPA promoter by co-expression of a transactivation domain lacking c-jun: a dominant negative ERK1 and ERK2 mutant and a dominant negative c-raf in glioblastoma cell line showed repressed uPA promoter activity compared with the effect of the empty expression vector.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ZHX2\"],\"tg associated gene name\":[\"MAPK3\"],\"tf text\":[],\"tg text\":[\"ERK1\"],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"We further studied the activation and inhibition of uPA promoter by co-expression of a transactivation domain lacking c-jun: a dominant negative ERK1 and ERK2 mutant and a dominant negative c-raf in glioblastoma cell line showed repressed uPA promoter activity compared with the effect of the empty expression vector.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|c-raf = zhhx2? uPA?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZHX2\"],\"tg (associated gene name)\":[\"MAPK3\"],\"tf text\":[],\"tg text\":[\"ERK1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"c-raf = zhhx2? uPA?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We further studied the activation and inhibition of uPA promoter by co-expression of a transactivation domain lacking c-jun: a dominant negative ERK1 and ERK2 mutant and a dominant negative c-raf in glioblastoma cell line showed repressed uPA promoter activity compared with the effect of the empty expression vector.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is aldolase B---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"HP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is aldolase B\"],\"sentence\":[\"Transactivation experiments in PKSV-PCT cultured cells showed that HNF1 alpha\",\"C/EBP alpha\",\"and DBP behave as transactivators of the 190-bp aldolase B gene promoter\",\"and that HNF1 beta had a low transactivating efficiency.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HP?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"HP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"HP?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transactivation experiments in PKSV-PCT cultured cells showed that HNF1 alpha, C/EBP alpha, and DBP behave as transactivators of the 190-bp aldolase B gene promoter, and that HNF1 beta had a low transactivating efficiency.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is aldolase B---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"HP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is aldolase B\"],\"sentence\":[\"Transactivation experiments in PKSV-PCT cultured cells showed that HNF1 alpha\",\"C/EBP alpha\",\"and DBP behave as transactivators of the 190-bp aldolase B gene promoter\",\"and that HNF1 beta had a low transactivating efficiency.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HP?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"HP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"HP?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transactivation experiments in PKSV-PCT cultured cells showed that HNF1 alpha, C/EBP alpha, and DBP behave as transactivators of the 190-bp aldolase B gene promoter, and that HNF1 beta had a low transactivating efficiency.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is DNMT3A---{\"tf associated gene name\":[\"DNMT1\"],\"tg associated gene name\":[\"KLF4\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u-fam\"],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is DNMT3A\"],\"sentence\":[\"In vitro\",\"DF increased DNA methylation of CpG islands within the KLF4 promoter that significantly contributed to suppression of KLF4 transcription; the effects were mitigated by DNA methyltransferase (DNMT) inhibitors and knockdown of DNMT3A.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is DNMT3A---{\"tf associated gene name\":[\"DNMT1\"],\"tg associated gene name\":[\"KLF4\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u-fam\"],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is DNMT3A\"],\"sentence\":[\"In vitro\",\"DF increased DNA methylation of CpG islands within the KLF4 promoter that significantly contributed to suppression of KLF4 transcription; the effects were mitigated by DNA methyltransferase (DNMT) inhibitors and knockdown of DNMT3A.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"DNMT1\"],\"target gene (associated gene name)\":[\"KLF4\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In vitro\",\"DF increased DNA methylation of CpG islands within the KLF4 promoter that significantly contributed to suppression of KLF4 transcription; the effects were mitigated by DNA methyltransferase (DNMT) inhibitors and knockdown of DNMT3A.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|DNMT1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"DNMT1\"],\"tg (associated gene name)\":[\"KLF4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"DNMT1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In vitro, DF increased DNA methylation of CpG islands within the KLF4 promoter that significantly contributed to suppression of KLF4 transcription; the effects were mitigated by DNA methyltransferase (DNMT) inhibitors and knockdown of DNMT3A.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG named---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"HDAC3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Smad6 suppresses GR-induced transactivation by attracting histone deacetylase 3 to DNA-bound GR and by antagonizing acetylation of histone H3 and H4 induced by p160 histone acetyltransferase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"HDAC3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Smad6 suppresses GR-induced transactivation by attracting histone deacetylase 3 to DNA-bound GR and by antagonizing acetylation of histone H3 and H4 induced by p160 histone acetyltransferase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG named---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"HDAC3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Smad6 suppresses GR-induced transactivation by attracting histone deacetylase 3 to DNA-bound GR and by antagonizing acetylation of histone H3 and H4 induced by p160 histone acetyltransferase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"HDAC3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Smad6 suppresses GR-induced transactivation by attracting histone deacetylase 3 to DNA-bound GR and by antagonizing acetylation of histone H3 and H4 induced by p160 histone acetyltransferase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulators are AT(1)R and JAK2 |none of them TFs I believe---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"KCND2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"regulators are AT(1)R and JAK2 \",\"none of them TFs I believe\"],\"sentence\":[\"Both AT(1)R and JAK2 blockade resulted in significant amelioration of the increase of protein phosphatase 1 activity and decrease in basal level of p16-phospholamban that may underlie early diastolic dysfunction\",\"as well as partial amelioration of early downregulation of Kv4.2 gene expression that may underlie increased arrhythmogenicity of 3-day post-MI heart.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MI=MITF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"KCND2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MI=MITF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Both AT(1)R and JAK2 blockade resulted in significant amelioration of the increase of protein phosphatase 1 activity and decrease in basal level of p16-phospholamban that may underlie early diastolic dysfunction, as well as partial amelioration of early downregulation of Kv4.2 gene expression that may underlie increased arrhythmogenicity of 3-day post-MI heart.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "regulators are AT(1)R and JAK2 |none of them TFs I believe---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"KCND2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"regulators are AT(1)R and JAK2 \",\"none of them TFs I believe\"],\"sentence\":[\"Both AT(1)R and JAK2 blockade resulted in significant amelioration of the increase of protein phosphatase 1 activity and decrease in basal level of p16-phospholamban that may underlie early diastolic dysfunction\",\"as well as partial amelioration of early downregulation of Kv4.2 gene expression that may underlie increased arrhythmogenicity of 3-day post-MI heart.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MI=MITF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"KCND2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MI=MITF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Both AT(1)R and JAK2 blockade resulted in significant amelioration of the increase of protein phosphatase 1 activity and decrease in basal level of p16-phospholamban that may underlie early diastolic dysfunction, as well as partial amelioration of early downregulation of Kv4.2 gene expression that may underlie increased arrhythmogenicity of 3-day post-MI heart.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF|STAT2 is TG---{\"tf associated gene name\":[\"STAT2\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"STAT2\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no TF\",\"STAT2 is TG\"],\"sentence\":[\"We show that IFN-alpha and IFN-gamma rapidly and efficiently enhanced STAT1\",\"STAT2\",\"p48\",\"and IRF-1 gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT2\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"STAT2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that IFN-alpha and IFN-gamma rapidly and efficiently enhanced STAT1, STAT2, p48, and IRF-1 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF|STAT2 is TG---{\"tf associated gene name\":[\"STAT2\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"STAT2\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no TF\",\"STAT2 is TG\"],\"sentence\":[\"We show that IFN-alpha and IFN-gamma rapidly and efficiently enhanced STAT1\",\"STAT2\",\"p48\",\"and IRF-1 gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT2\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"STAT2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that IFN-alpha and IFN-gamma rapidly and efficiently enhanced STAT1, STAT2, p48, and IRF-1 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is JUN|TG is PPARbeta/delta---{\"tf associated gene name\":[\"PPARD\"],\"tg associated gene name\":[\"YY1\"],\"tf text\":[\"PPARbeta\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is JUN\",\"TG is PPARbeta/delta\"],\"sentence\":[\"Finally\",\"c-Jun ablation inhibited PPARbeta/delta up-regulation induced by EGF\",\"and chromatin immunoprecipitation (ChIP) showed that c-Jun bound to the PPARbeta/delta promoter and the binding increased in EGF-stimulated cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is JUN|TG is PPARbeta/delta---{\"tf associated gene name\":[\"PPARD\"],\"tg associated gene name\":[\"YY1\"],\"tf text\":[\"PPARbeta\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is JUN\",\"TG is PPARbeta/delta\"],\"sentence\":[\"Finally\",\"c-Jun ablation inhibited PPARbeta/delta up-regulation induced by EGF\",\"and chromatin immunoprecipitation (ChIP) showed that c-Jun bound to the PPARbeta/delta promoter and the binding increased in EGF-stimulated cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PPARD\"],\"target gene (associated gene name)\":[\"YY1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Finally\",\"c-Jun ablation inhibited PPARbeta/delta up-regulation induced by EGF\",\"and chromatin immunoprecipitation (ChIP) showed that c-Jun bound to the PPARbeta/delta promoter and the binding increased in EGF-stimulated cells.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.237967\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"9\"],\"sentence length\":[\"225\"],\"sentence pair density\":[\"0.0275229357798165\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|PPARD = Target---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARD\"],\"tg (associated gene name)\":[\"YY1\"],\"tf text\":[\"PPARbeta\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PPARD = Target\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Finally, c-Jun ablation inhibited PPARbeta/delta up-regulation induced by EGF, and chromatin immunoprecipitation (ChIP) showed that c-Jun bound to the PPARbeta/delta promoter and the binding increased in EGF-stimulated cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is ICAM-1; also STAT1-STAT1 looks very much like TF-TG from sentence---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"STAT1\"],\"tf text\":[\"Stat1\"],\"tg text\":[\"Stat1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is ICAM-1; also STAT1-STAT1 looks very much like TF-TG from sentence\"],\"sentence\":[\"Using a Stat1-deficient cell line\",\"we demonstrated that transfection of wild-type Stat1 expression plasmid restored appropriate Stat1 expression and IFN-gamma-dependent phosphorylation as well as consequent IFN-gamma activation of cotransfected ICAM-1 promoter constructs and endogenous ICAM-1 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|autoreg through expression plasmid?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"STAT1\"],\"tf text\":[\"Stat1\"],\"tg text\":[\"Stat1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"autoreg through expression plasmid?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using a Stat1-deficient cell line, we demonstrated that transfection of wild-type Stat1 expression plasmid restored appropriate Stat1 expression and IFN-gamma-dependent phosphorylation as well as consequent IFN-gamma activation of cotransfected ICAM-1 promoter constructs and endogenous ICAM-1 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is ICAM-1; also STAT1-STAT1 looks very much like TF-TG from sentence---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"STAT1\"],\"tf text\":[\"Stat1\"],\"tg text\":[\"Stat1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is ICAM-1; also STAT1-STAT1 looks very much like TF-TG from sentence\"],\"sentence\":[\"Using a Stat1-deficient cell line\",\"we demonstrated that transfection of wild-type Stat1 expression plasmid restored appropriate Stat1 expression and IFN-gamma-dependent phosphorylation as well as consequent IFN-gamma activation of cotransfected ICAM-1 promoter constructs and endogenous ICAM-1 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|autoreg through expression plasmid?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"STAT1\"],\"tf text\":[\"Stat1\"],\"tg text\":[\"Stat1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"autoreg through expression plasmid?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using a Stat1-deficient cell line, we demonstrated that transfection of wild-type Stat1 expression plasmid restored appropriate Stat1 expression and IFN-gamma-dependent phosphorylation as well as consequent IFN-gamma activation of cotransfected ICAM-1 promoter constructs and endogenous ICAM-1 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 350 (char 349)\n",
      "{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"JUNB\"],\"tf text\":[\"egr-1\"],\"tg text\":[\"junB\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"These induced complexes in turn correlate with the activity of the c-fos\",\"egr-1\",\"and junB promoters.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"JUNB\"],\"tf text\":[\"egr-1\"],\"tg text\":[\"junB\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These induced complexes in turn correlate with the activity of the c-fos, egr-1, and junB promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"JUNB\"],\"tf text\":[\"egr-1\"],\"tg text\":[\"junB\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"These induced complexes in turn correlate with the activity of the c-fos\",\"egr-1\",\"and junB promoters.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"JUNB\"],\"tf text\":[\"egr-1\"],\"tg text\":[\"junB\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These induced complexes in turn correlate with the activity of the c-fos, egr-1, and junB promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|TGs not named---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"IFNA1\"],\"tf text\":[\"IRF-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"TGs not named\"],\"sentence\":[\"Taken together\",\"these results suggest that ICSBP is a negative regulatory factor capable of repressing transcription of target genes induced by IFN\",\"retinoic acid\",\"or IRF-1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"IFNA1\"],\"tf text\":[\"IRF-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[\"Taken together, these results suggest that ICSBP is a negative regulatory factor capable of repressing transcription of target genes induced by IFN, retinoic acid, or IRF-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE|TGs not named---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"IFNA1\"],\"tf text\":[\"IRF-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"TGs not named\"],\"sentence\":[\"Taken together\",\"these results suggest that ICSBP is a negative regulatory factor capable of repressing transcription of target genes induced by IFN\",\"retinoic acid\",\"or IRF-1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"IFNA1\"],\"tf text\":[\"IRF-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[\"Taken together, these results suggest that ICSBP is a negative regulatory factor capable of repressing transcription of target genes induced by IFN, retinoic acid, or IRF-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|TF not named---{\"tf associated gene name\":[\"ZNF91\"],\"tg associated gene name\":[\"EPCAM\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"TF not named\"],\"sentence\":[\"It was the aim of this study to achieve a targeted methylation of the EpCAM promoter and silence EpCAM gene expression using an engineered zinc finger protein that specifically binds the EpCAM promoter fused to the catalytic domain of the Dnmt3a DNA methyltransferase.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF91\"],\"tg (associated gene name)\":[\"EPCAM\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"It was the aim of this study to achieve a targeted methylation of the EpCAM promoter and silence EpCAM gene expression using an engineered zinc finger protein that specifically binds the EpCAM promoter fused to the catalytic domain of the Dnmt3a DNA methyltransferase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE|TF not named---{\"tf associated gene name\":[\"ZNF91\"],\"tg associated gene name\":[\"EPCAM\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"TF not named\"],\"sentence\":[\"It was the aim of this study to achieve a targeted methylation of the EpCAM promoter and silence EpCAM gene expression using an engineered zinc finger protein that specifically binds the EpCAM promoter fused to the catalytic domain of the Dnmt3a DNA methyltransferase.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF91\"],\"tg (associated gene name)\":[\"EPCAM\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"It was the aim of this study to achieve a targeted methylation of the EpCAM promoter and silence EpCAM gene expression using an engineered zinc finger protein that specifically binds the EpCAM promoter fused to the catalytic domain of the Dnmt3a DNA methyltransferase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG named---{\"tf associated gene name\":[\"MXD1\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Myc:Max heterodimers activate transcription while Mad:Max heterodimers repress transcription from the same promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TG named---{\"tf associated gene name\":[\"MXD1\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Myc:Max heterodimers activate transcription while Mad:Max heterodimers repress transcription from the same promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MXD1\"],\"target gene (associated gene name)\":[\"MYC\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Myc:Max heterodimers activate transcription while Mad:Max heterodimers repress transcription from the same promoter.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.129693\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"5\"],\"sentence length\":[\"116\"],\"sentence pair density\":[\"0.0134529147982062\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MXD1\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Myc:Max heterodimers activate transcription while Mad:Max heterodimers repress transcription from the same promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Pit-1---{\"tf associated gene name\":[\"VDR\"],\"tg associated gene name\":[\"XPR1\"],\"tf text\":[\"vitamin D receptor\"],\"tg text\":[\"X receptor\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is Pit-1\"],\"sentence\":[\"Gel mobility shift and chromatin immunoprecipitation assays confirmed the direct interaction between the vitamin D receptor (VDR) as homodimer (without the retinoid X receptor)\",\"and the Pit-1 promoter\",\"supporting the view that Pit-1 is a direct transcriptional target of VDR.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"VDR\"],\"tg (associated gene name)\":[\"XPR1\"],\"tf text\":[\"vitamin D receptor\"],\"tg text\":[\"X receptor\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gel mobility shift and chromatin immunoprecipitation assays confirmed the direct interaction between the vitamin D receptor (VDR) as homodimer (without the retinoid X receptor), and the Pit-1 promoter, supporting the view that Pit-1 is a direct transcriptional target of VDR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Pit-1---{\"tf associated gene name\":[\"VDR\"],\"tg associated gene name\":[\"XPR1\"],\"tf text\":[\"vitamin D receptor\"],\"tg text\":[\"X receptor\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is Pit-1\"],\"sentence\":[\"Gel mobility shift and chromatin immunoprecipitation assays confirmed the direct interaction between the vitamin D receptor (VDR) as homodimer (without the retinoid X receptor)\",\"and the Pit-1 promoter\",\"supporting the view that Pit-1 is a direct transcriptional target of VDR.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"VDR\"],\"tg (associated gene name)\":[\"XPR1\"],\"tf text\":[\"vitamin D receptor\"],\"tg text\":[\"X receptor\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gel mobility shift and chromatin immunoprecipitation assays confirmed the direct interaction between the vitamin D receptor (VDR) as homodimer (without the retinoid X receptor), and the Pit-1 promoter, supporting the view that Pit-1 is a direct transcriptional target of VDR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF ---{\"tf associated gene name\":[\"HES1\"],\"tg associated gene name\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[\"VEGF\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TF \"],\"sentence\":[\"Specific siRNAs targeting VEGF were transfected into HL-60 cells and the VEGF expression was measured with reverse transcription-polymerase chain reaction (RT-PCR) and western blot assay.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TF ---{\"tf associated gene name\":[\"HES1\"],\"tg associated gene name\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[\"VEGF\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TF \"],\"sentence\":[\"Specific siRNAs targeting VEGF were transfected into HL-60 cells and the VEGF expression was measured with reverse transcription-polymerase chain reaction (RT-PCR) and western blot assay.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HES1\"],\"tg (associated gene name)\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[\"VEGF\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Specific siRNAs targeting VEGF were transfected into HL-60 cells and the VEGF expression was measured with reverse transcription-polymerase chain reaction (RT-PCR) and western blot assay.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 411 (char 410)\n",
      "{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"DDIT3\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Deletion mutants of the human CHOP promoter identify a region comprising nucleotides -75 to -104 required for both constitutive and ER-stress-inducible expression.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|ER enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"DDIT3\"],\"tf text\":[],\"tg text\":[\"CHOP\"],\"consensus\":[\"V\"],\"martin_comments\":[\"ER enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Deletion mutants of the human CHOP promoter identify a region comprising nucleotides -75 to -104 required for both constitutive and ER-stress-inducible expression.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"DDIT3\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Deletion mutants of the human CHOP promoter identify a region comprising nucleotides -75 to -104 required for both constitutive and ER-stress-inducible expression.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|ER enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"DDIT3\"],\"tf text\":[],\"tg text\":[\"CHOP\"],\"consensus\":[\"V\"],\"martin_comments\":[\"ER enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Deletion mutants of the human CHOP promoter identify a region comprising nucleotides -75 to -104 required for both constitutive and ER-stress-inducible expression.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "AP1-ectopic expressoin driven by PRL-promoter---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"AP1-ectopic expressoin driven by PRL-promoter\"],\"sentence\":[\"H2O2 also induces an up-regulation in transactivity of the activating protein-1 (AP-1) as shown with a reporter gene driven by a prolactin gene promoter with 4 copies of AP-1 binding sites inserted in the upstream of the promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "AP1-ectopic expressoin driven by PRL-promoter---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"AP1-ectopic expressoin driven by PRL-promoter\"],\"sentence\":[\"H2O2 also induces an up-regulation in transactivity of the activating protein-1 (AP-1) as shown with a reporter gene driven by a prolactin gene promoter with 4 copies of AP-1 binding sites inserted in the upstream of the promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"PRL\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"H2O2 also induces an up-regulation in transactivity of the activating protein-1 (AP-1) as shown with a reporter gene driven by a prolactin gene promoter with 4 copies of AP-1 binding sites inserted in the upstream of the promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"H2O2 also induces an up-regulation in transactivity of the activating protein-1 (AP-1) as shown with a reporter gene driven by a prolactin gene promoter with 4 copies of AP-1 binding sites inserted in the upstream of the promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p21 (CDKN1A)---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[\"p65\"],\"tg text\":[\"CBP\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is p21 (CDKN1A)\"],\"sentence\":[\"Our data suggested that both p53 and p65 limited each other's ability to stimulate p21 induction and this mutual repression mechanism was consistent with a model in which both factors were competing for limiting pool of p300/CBP coactivator protein complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is p21 (CDKN1A)---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[\"p65\"],\"tg text\":[\"CBP\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is p21 (CDKN1A)\"],\"sentence\":[\"Our data suggested that both p53 and p65 limited each other's ability to stimulate p21 induction and this mutual repression mechanism was consistent with a model in which both factors were competing for limiting pool of p300/CBP coactivator protein complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RELA\"],\"target gene (associated gene name)\":[\"CREBBP\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Our data suggested that both p53 and p65 limited each other's ability to stimulate p21 induction and this mutual repression mechanism was consistent with a model in which both factors were competing for limiting pool of p300/CBP coactivator protein complexes.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.336505\"],\"pmid counts\":[\"12\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"259\"],\"sentence pair density\":[\"0.0327868852459016\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|P21 target---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"p65\"],\"tg text\":[\"CBP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"P21 target\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our data suggested that both p53 and p65 limited each other's ability to stimulate p21 induction and this mutual repression mechanism was consistent with a model in which both factors were competing for limiting pool of p300/CBP coactivator protein complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is H5|uncertain about normalization---{\"tf associated gene name\":[\"GATA3\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[\"GATA-3\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is H5\",\"uncertain about normalization\"],\"sentence\":[\"Although the GATA sites were required for enhancer function\",\"the concentration of GATA-1\",\"GATA-2\",\"and GATA-3 decreased during cell differentiation\",\"and overexpression of these factors had little effect on H5 transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|H5?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA3\"],\"tg (associated gene name)\":[\"FGFR1\"],\"tf text\":[\"GATA-3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"H5?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although the GATA sites were required for enhancer function, the concentration of GATA-1, GATA-2, and GATA-3 decreased during cell differentiation, and overexpression of these factors had little effect on H5 transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is H5|uncertain about normalization---{\"tf associated gene name\":[\"GATA3\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[\"GATA-3\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is H5\",\"uncertain about normalization\"],\"sentence\":[\"Although the GATA sites were required for enhancer function\",\"the concentration of GATA-1\",\"GATA-2\",\"and GATA-3 decreased during cell differentiation\",\"and overexpression of these factors had little effect on H5 transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|H5?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA3\"],\"tg (associated gene name)\":[\"FGFR1\"],\"tf text\":[\"GATA-3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"H5?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although the GATA sites were required for enhancer function, the concentration of GATA-1, GATA-2, and GATA-3 decreased during cell differentiation, and overexpression of these factors had little effect on H5 transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CXCR = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"ANXA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CXCR = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Chromatin immunoprecipitation experiments confirmed that in ANXA1 overexpressed cells, NF-B was recruited to CXCR4 promoter without external stimulation, indicating that ANXA1 is critical for the constitutive activation of NF-B in breast cancer to promote metastasis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "CXCR = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"ANXA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CXCR = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Chromatin immunoprecipitation experiments confirmed that in ANXA1 overexpressed cells, NF-B was recruited to CXCR4 promoter without external stimulation, indicating that ANXA1 is critical for the constitutive activation of NF-B in breast cancer to promote metastasis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is beta 2-integrin CD11b---{\"tf associated gene name\":[\"ZNF148\"],\"tg associated gene name\":[\"NEUROD1\"],\"tf text\":[\"ZBP-89\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is beta 2-integrin CD11b\"],\"sentence\":[\"The zinc finger transcription factor ZBP-89 is a repressor of the human beta 2-integrin CD11b gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CD11b?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF148\"],\"tg (associated gene name)\":[\"NEUROD1\"],\"tf text\":[\"ZBP-89\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CD11b?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The zinc finger transcription factor ZBP-89 is a repressor of the human beta 2-integrin CD11b gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is beta 2-integrin CD11b---{\"tf associated gene name\":[\"ZNF148\"],\"tg associated gene name\":[\"NEUROD1\"],\"tf text\":[\"ZBP-89\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is beta 2-integrin CD11b\"],\"sentence\":[\"The zinc finger transcription factor ZBP-89 is a repressor of the human beta 2-integrin CD11b gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CD11b?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF148\"],\"tg (associated gene name)\":[\"NEUROD1\"],\"tf text\":[\"ZBP-89\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CD11b?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The zinc finger transcription factor ZBP-89 is a repressor of the human beta 2-integrin CD11b gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 374 (char 373)\n",
      "{\"transcription factor (associated gene name)\":[\"NR5A1\"],\"target gene (associated gene name)\":[\"POU5F1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Interestingly\",\"SF1 and RAR form a novel complex on the RAREoct sequence that synergistically activate the Oct-3/4 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|POU5F1 = TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR5A1\"],\"tg (associated gene name)\":[\"POU5F1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"POU5F1 = TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Interestingly, SF1 and RAR form a novel complex on the RAREoct sequence that synergistically activate the Oct-3/4 promoter.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"NR5A1\"],\"target gene (associated gene name)\":[\"POU5F1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Interestingly\",\"SF1 and RAR form a novel complex on the RAREoct sequence that synergistically activate the Oct-3/4 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|POU5F1 = TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR5A1\"],\"tg (associated gene name)\":[\"POU5F1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"POU5F1 = TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Interestingly, SF1 and RAR form a novel complex on the RAREoct sequence that synergistically activate the Oct-3/4 promoter.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear whether TRE---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"CDK6\"],\"tf text\":[],\"tg text\":[\"CDK6\"],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"unclear\"],\"comment\":[\"unclear whether TRE\"],\"sentence\":[\"The T877A mutant of the AR frequently found in advanced cases of prostate cancer displays an exaggerated stimulation of transcriptional activity by CDK6.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|by?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"CDK6\"],\"tf text\":[],\"tg text\":[\"CDK6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"by?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The T877A mutant of the AR frequently found in advanced cases of prostate cancer displays an exaggerated stimulation of transcriptional activity by CDK6.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "unclear whether TRE---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"CDK6\"],\"tf text\":[],\"tg text\":[\"CDK6\"],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"unclear\"],\"comment\":[\"unclear whether TRE\"],\"sentence\":[\"The T877A mutant of the AR frequently found in advanced cases of prostate cancer displays an exaggerated stimulation of transcriptional activity by CDK6.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|by?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"CDK6\"],\"tf text\":[],\"tg text\":[\"CDK6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"by?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The T877A mutant of the AR frequently found in advanced cases of prostate cancer displays an exaggerated stimulation of transcriptional activity by CDK6.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is E-cadherin---{\"tf associated gene name\":[\"SNAI1\"],\"tg associated gene name\":[\"HDAC2\"],\"tf text\":[],\"tg text\":[\"HDAC2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is E-cadherin\"],\"sentence\":[\"Silencing of E-cadherin in highly metastatic cells is mediated by a transcriptional repressor complex containing Snail and histone deacetylase 1 (HDAC1) and HDAC2.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SNAI1\"],\"tg (associated gene name)\":[\"HDAC2\"],\"tf text\":[],\"tg text\":[\"HDAC2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Silencing of E-cadherin in highly metastatic cells is mediated by a transcriptional repressor complex containing Snail and histone deacetylase 1 (HDAC1) and HDAC2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is E-cadherin---{\"tf associated gene name\":[\"SNAI1\"],\"tg associated gene name\":[\"HDAC2\"],\"tf text\":[],\"tg text\":[\"HDAC2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is E-cadherin\"],\"sentence\":[\"Silencing of E-cadherin in highly metastatic cells is mediated by a transcriptional repressor complex containing Snail and histone deacetylase 1 (HDAC1) and HDAC2.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SNAI1\"],\"tg (associated gene name)\":[\"HDAC2\"],\"tf text\":[],\"tg text\":[\"HDAC2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Silencing of E-cadherin in highly metastatic cells is mediated by a transcriptional repressor complex containing Snail and histone deacetylase 1 (HDAC1) and HDAC2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is SHP-1|c-fos is TG---{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[\"c-fos\"],\"tg text\":[\"cyclin D1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is SHP-1\",\"c-fos is TG\"],\"sentence\":[\"Overexpression of SHP-1 in intestinal epithelial crypt cells significantly inhibited dhfr\",\"c-myc\",\"and cyclin D1 gene expression but did not interfere with c-fos gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|c-fos not TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[\"c-fos\"],\"tg text\":[\"cyclin D1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"c-fos not TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of SHP-1 in intestinal epithelial crypt cells significantly inhibited dhfr, c-myc, and cyclin D1 gene expression but did not interfere with c-fos gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is SHP-1|c-fos is TG---{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[\"c-fos\"],\"tg text\":[\"cyclin D1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is SHP-1\",\"c-fos is TG\"],\"sentence\":[\"Overexpression of SHP-1 in intestinal epithelial crypt cells significantly inhibited dhfr\",\"c-myc\",\"and cyclin D1 gene expression but did not interfere with c-fos gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|c-fos not TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[\"c-fos\"],\"tg text\":[\"cyclin D1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"c-fos not TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of SHP-1 in intestinal epithelial crypt cells significantly inhibited dhfr, c-myc, and cyclin D1 gene expression but did not interfere with c-fos gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"ZFP90\"],\"tg associated gene name\":[\"FGF2\"],\"tf text\":[\"FIK\"],\"tg text\":[\"basic fibroblast growth factor\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"Ras/MEK/ERK Up-regulation of the fibroblast KCa channel FIK is a common mechanism for basic fibroblast growth factor and transforming growth factor-beta suppression of myogenesis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|FIK = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZFP90\"],\"tg (associated gene name)\":[\"FGF2\"],\"tf text\":[\"FIK\"],\"tg text\":[\"basic fibroblast growth factor\"],\"consensus\":[\"N\"],\"martin_comments\":[\"FIK = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Ras/MEK/ERK Up-regulation of the fibroblast KCa channel FIK is a common mechanism for basic fibroblast growth factor and transforming growth factor-beta suppression of myogenesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"ZFP90\"],\"tg associated gene name\":[\"FGF2\"],\"tf text\":[\"FIK\"],\"tg text\":[\"basic fibroblast growth factor\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"Ras/MEK/ERK Up-regulation of the fibroblast KCa channel FIK is a common mechanism for basic fibroblast growth factor and transforming growth factor-beta suppression of myogenesis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|FIK = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZFP90\"],\"tg (associated gene name)\":[\"FGF2\"],\"tf text\":[\"FIK\"],\"tg text\":[\"basic fibroblast growth factor\"],\"consensus\":[\"N\"],\"martin_comments\":[\"FIK = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Ras/MEK/ERK Up-regulation of the fibroblast KCa channel FIK is a common mechanism for basic fibroblast growth factor and transforming growth factor-beta suppression of myogenesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "Random norm Gli3? TF---{\"tf associated gene name\":[\"GLI3\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[\"interleukin-2\"],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"Random norm Gli3? TF\"],\"sentence\":[\"Regulation of interleukin-2 (IL-2) gene expression by the p50 and p65 subunits of the DNA binding protein NF-kappa B was studied in nontransformed CD4+ T lymphocyte clones.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|GLI3?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI3\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[\"interleukin-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"GLI3?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Regulation of interleukin-2 (IL-2) gene expression by the p50 and p65 subunits of the DNA binding protein NF-kappa B was studied in nontransformed CD4+ T lymphocyte clones.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "Random norm Gli3? TF---{\"tf associated gene name\":[\"GLI3\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[\"interleukin-2\"],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"Random norm Gli3? TF\"],\"sentence\":[\"Regulation of interleukin-2 (IL-2) gene expression by the p50 and p65 subunits of the DNA binding protein NF-kappa B was studied in nontransformed CD4+ T lymphocyte clones.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|GLI3?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI3\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[\"interleukin-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"GLI3?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Regulation of interleukin-2 (IL-2) gene expression by the p50 and p65 subunits of the DNA binding protein NF-kappa B was studied in nontransformed CD4+ T lymphocyte clones.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ARID1A\"],\"tg (associated gene name)\":[\"SMARCA4\"],\"tf text\":[\"ARID1A\"],\"tg text\":[\"SMARCA4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Notable among the alterations newly identified, genomic deletions, mutations, and rearrangements recurrently targeted genes encoding components of the SWItch/Sucrose NonFermentable (SWI/SNF) chromatin remodeling complex, including all three putative DNA binding subunits (ARID1A, ARID1B, and PBRM1) and both enzymatic subunits (SMARCA2 and SMARCA4).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ARID1A\"],\"tg (associated gene name)\":[\"SMARCA4\"],\"tf text\":[\"ARID1A\"],\"tg text\":[\"SMARCA4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Notable among the alterations newly identified, genomic deletions, mutations, and rearrangements recurrently targeted genes encoding components of the SWItch/Sucrose NonFermentable (SWI/SNF) chromatin remodeling complex, including all three putative DNA binding subunits (ARID1A, ARID1B, and PBRM1) and both enzymatic subunits (SMARCA2 and SMARCA4).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 849 (char 848)\n",
      "{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"TREH\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP-1 TRE\",\"c-jun TRE\",\"and c-fos SRE DNA sequence motifs\",\"age-related reductions in the activation of AP-1 were accompanied by decreased levels of c-jun TRE and c-fos SRE binding complexes.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.186809\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"318\"],\"sentence pair density\":[\"0.00862068965517241\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TRE misplaced?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"TREH\"],\"tf text\":[\"c-jun\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TRE misplaced?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP-1 TRE, c-jun TRE, and c-fos SRE DNA sequence motifs, age-related reductions in the activation of AP-1 were accompanied by decreased levels of c-jun TRE and c-fos SRE binding complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"TREH\"],\"tf text\":[\"c-jun\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP-1 TRE\",\"c-jun TRE\",\"and c-fos SRE DNA sequence motifs\",\"age-related reductions in the activation of AP-1 were accompanied by decreased levels of c-jun TRE and c-fos SRE binding complexes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"TREH\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP-1 TRE\",\"c-jun TRE\",\"and c-fos SRE DNA sequence motifs\",\"age-related reductions in the activation of AP-1 were accompanied by decreased levels of c-jun TRE and c-fos SRE binding complexes.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.186809\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"318\"],\"sentence pair density\":[\"0.00862068965517241\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TRE misplaced?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"TREH\"],\"tf text\":[\"c-jun\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TRE misplaced?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP-1 TRE, c-jun TRE, and c-fos SRE DNA sequence motifs, age-related reductions in the activation of AP-1 were accompanied by decreased levels of c-jun TRE and c-fos SRE binding complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG---{\"tf associated gene name\":[\"MAX\"],\"tg associated gene name\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"Transient transfection studies with different c-Myc\",\"Max and Mad constructs in COS-7 cells showed that the balance between c-Myc/Max\",\"Max/Max and Max/Mad complexes is crucial for the regulation\",\"resulting in either transactivation or transrepression of an ODC-CAT reporter gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG---{\"tf associated gene name\":[\"MAX\"],\"tg associated gene name\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"Transient transfection studies with different c-Myc\",\"Max and Mad constructs in COS-7 cells showed that the balance between c-Myc/Max\",\"Max/Max and Max/Mad complexes is crucial for the regulation\",\"resulting in either transactivation or transrepression of an ODC-CAT reporter gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MAX\"],\"target gene (associated gene name)\":[\"CAT\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Transient transfection studies with different c-Myc\",\"Max and Mad constructs in COS-7 cells showed that the balance between c-Myc/Max\",\"Max/Max and Max/Mad complexes is crucial for the regulation\",\"resulting in either transactivation or transrepression of an ODC-CAT reporter gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|CAT misplaced|CAT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAX\"],\"tg (associated gene name)\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"CAT misplaced\"],\"astrid_comments\":[\"CAT\"],\"sentences with greek letters\":[\"Transient transfection studies with different c-Myc, Max and Mad constructs in COS-7 cells showed that the balance between c-Myc/Max, Max/Max and Max/Mad complexes is crucial for the regulation, resulting in either transactivation or transrepression of an ODC-CAT reporter gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE/GRE---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\",\"f\"],\"negation\":[],\"comment\":[\"not TRE/GRE\"],\"sentence\":[\"Thus\",\"rSV5deltaSH induced cell death by activating TNF-alpha expression\",\"possibly through activation of the NF-kappaB subunit p65 and then targeting p65 for degradation\",\"leading to apoptosis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE/GRE---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\",\"f\"],\"negation\":[],\"comment\":[\"not TRE/GRE\"],\"sentence\":[\"Thus\",\"rSV5deltaSH induced cell death by activating TNF-alpha expression\",\"possibly through activation of the NF-kappaB subunit p65 and then targeting p65 for degradation\",\"leading to apoptosis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"TNF\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Thus\",\"rSV5deltaSH induced cell death by activating TNF-alpha expression\",\"possibly through activation of the NF-kappaB subunit p65 and then targeting p65 for degradation\",\"leading to apoptosis.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, rSV5deltaSH induced cell death by activating TNF-alpha expression, possibly through activation of the NF-kappaB subunit p65 and then targeting p65 for degradation, leading to apoptosis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Tst-1---{\"tf associated gene name\":[\"POU3F1\"],\"tg associated gene name\":[\"MMP19\"],\"tf text\":[],\"tg text\":[\"MMP-19\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Tst-1\"],\"sentence\":[\"HaCaT cells stably transfected with Tst-1 showed a strong decrease of activity of MMP-19 promoter that correlated with suppression of MMP-19\",\"cytokeratin 14 and 5\",\"decreased cell proliferation\",\"and altered expression of involucrin and loricrin.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Tst-1---{\"tf associated gene name\":[\"POU3F1\"],\"tg associated gene name\":[\"MMP19\"],\"tf text\":[],\"tg text\":[\"MMP-19\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Tst-1\"],\"sentence\":[\"HaCaT cells stably transfected with Tst-1 showed a strong decrease of activity of MMP-19 promoter that correlated with suppression of MMP-19\",\"cytokeratin 14 and 5\",\"decreased cell proliferation\",\"and altered expression of involucrin and loricrin.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"POU3F1\"],\"target gene (associated gene name)\":[\"MMP19\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"HaCaT cells stably transfected with Tst-1 showed a strong decrease of activity of MMP-19 promoter that correlated with suppression of MMP-19\",\"cytokeratin 14 and 5\",\"decreased cell proliferation\",\"and altered expression of involucrin and loricrin.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|tst-1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU3F1\"],\"tg (associated gene name)\":[\"MMP19\"],\"tf text\":[],\"tg text\":[\"MMP-19\"],\"consensus\":[\"V\"],\"martin_comments\":[\"tst-1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HaCaT cells stably transfected with Tst-1 showed a strong decrease of activity of MMP-19 promoter that correlated with suppression of MMP-19, cytokeratin 14 and 5, decreased cell proliferation, and altered expression of involucrin and loricrin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear sentence|weak TRE-evidence---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CLDN1\"],\"tf text\":[\"estrogen receptor\"],\"tg text\":[\"claudin 1\"],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[\"unclear sentence\",\"weak TRE-evidence\"],\"sentence\":[\"We found that methylation of the claudin 1 promoter CpG island is relatively frequent in estrogen receptor positive (ER+) breast cancer and is associated with low claudin 1 expression.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "unclear sentence|weak TRE-evidence---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CLDN1\"],\"tf text\":[\"estrogen receptor\"],\"tg text\":[\"claudin 1\"],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[\"unclear sentence\",\"weak TRE-evidence\"],\"sentence\":[\"We found that methylation of the claudin 1 promoter CpG island is relatively frequent in estrogen receptor positive (ER+) breast cancer and is associated with low claudin 1 expression.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"CLDN1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"We found that methylation of the claudin 1 promoter CpG island is relatively frequent in estrogen receptor positive (ER+) breast cancer and is associated with low claudin 1 expression.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.058152\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"184\"],\"sentence pair density\":[\"0.00546448087431694\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|direct enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"CLDN1\"],\"tf text\":[\"estrogen receptor\"],\"tg text\":[\"claudin 1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"direct enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that methylation of the claudin 1 promoter CpG island is relatively frequent in estrogen receptor positive (ER+) breast cancer and is associated with low claudin 1 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG s are TCF4 as well as p300 genes---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"CTNNB1\"],\"tf text\":[\"Egr1\"],\"tg text\":[\"beta-catenin\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG s are TCF4 as well as p300 genes\"],\"sentence\":[\"Transfection of Egr1 led to transactivation of TCF4 as well as p300 genes\",\"through specific binding to a promoter region\",\"and thus in turn resulted in nuclear accumulation of beta-catenin mediated by the up-regulating TCF4.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG s are TCF4 as well as p300 genes---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"CTNNB1\"],\"tf text\":[\"Egr1\"],\"tg text\":[\"beta-catenin\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG s are TCF4 as well as p300 genes\"],\"sentence\":[\"Transfection of Egr1 led to transactivation of TCF4 as well as p300 genes\",\"through specific binding to a promoter region\",\"and thus in turn resulted in nuclear accumulation of beta-catenin mediated by the up-regulating TCF4.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"CTNNB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Transfection of Egr1 led to transactivation of TCF4 as well as p300 genes\",\"through specific binding to a promoter region\",\"and thus in turn resulted in nuclear accumulation of beta-catenin mediated by the up-regulating TCF4.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"4.048776\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"223\"],\"sentence pair density\":[\"0.0684931506849315\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|beta-catenin too indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"CTNNB1\"],\"tf text\":[\"Egr1\"],\"tg text\":[\"beta-catenin\"],\"consensus\":[\"V\"],\"martin_comments\":[\"beta-catenin too indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transfection of Egr1 led to transactivation of TCF4 as well as p300 genes, through specific binding to a promoter region, and thus in turn resulted in nuclear accumulation of beta-catenin mediated by the up-regulating TCF4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "IRF6-problem; not TRE---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"IRF6-problem; not TRE\"],\"sentence\":[\"MD-2 transfection in AGS cells markedly increased cell surface expression of TLR4 and augmented the activation of NF-kappaB and IL-8 promoter upon stimulation with H. pylori LPS.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "IRF6-problem; not TRE---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"IRF6-problem; not TRE\"],\"sentence\":[\"MD-2 transfection in AGS cells markedly increased cell surface expression of TLR4 and augmented the activation of NF-kappaB and IL-8 promoter upon stimulation with H. pylori LPS.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF6\"],\"target gene (associated gene name)\":[\"NFKB\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"MD-2 transfection in AGS cells markedly increased cell surface expression of TLR4 and augmented the activation of NF-kappaB and IL-8 promoter upon stimulation with H. pylori LPS.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|LPS?|LPS---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"LPS?\"],\"astrid_comments\":[\"LPS\"],\"sentences with greek letters\":[\"MD-2 transfection in AGS cells markedly increased cell surface expression of TLR4 and augmented the activation of NF-kappaB and IL-8 promoter upon stimulation with H. pylori LPS.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE|no TF named (opnly upstream events)---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"FOS\"],\"tf text\":[],\"tg text\":[\"c-fos\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"not GRE\",\"no TF named (opnly upstream events)\"],\"sentence\":[\"Activation of c-fos promoter by Gbetagamma-mediated signaling: involvement of Rho and c-Jun N-terminal kinase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"FOS\"],\"tf text\":[],\"tg text\":[\"c-fos\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Activation of c-fos promoter by Gbetagamma-mediated signaling: involvement of Rho and c-Jun N-terminal kinase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not GRE|no TF named (opnly upstream events)---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"FOS\"],\"tf text\":[],\"tg text\":[\"c-fos\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"not GRE\",\"no TF named (opnly upstream events)\"],\"sentence\":[\"Activation of c-fos promoter by Gbetagamma-mediated signaling: involvement of Rho and c-Jun N-terminal kinase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"FOS\"],\"tf text\":[],\"tg text\":[\"c-fos\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Activation of c-fos promoter by Gbetagamma-mediated signaling: involvement of Rho and c-Jun N-terminal kinase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 541 (char 540)\n",
      "{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"KDM3A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Using purified murine T cells and the Il2 target locus\",\"and a colon cancer cell line and the Cdx2 target locus\",\"we show that Oct1 recruits the NuRD chromatin-remodeling complex to promote a repressed state\",\"but in a regulated manner can switch to a different capacity and mediate Jmjd1a recruitment to block repression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CDX target?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"KDM3A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CDX target?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using purified murine T cells and the Il2 target locus, and a colon cancer cell line and the Cdx2 target locus, we show that Oct1 recruits the NuRD chromatin-remodeling complex to promote a repressed state, but in a regulated manner can switch to a different capacity and mediate Jmjd1a recruitment to block repression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"KDM3A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Using purified murine T cells and the Il2 target locus\",\"and a colon cancer cell line and the Cdx2 target locus\",\"we show that Oct1 recruits the NuRD chromatin-remodeling complex to promote a repressed state\",\"but in a regulated manner can switch to a different capacity and mediate Jmjd1a recruitment to block repression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CDX target?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"KDM3A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CDX target?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using purified murine T cells and the Il2 target locus, and a colon cancer cell line and the Cdx2 target locus, we show that Oct1 recruits the NuRD chromatin-remodeling complex to promote a repressed state, but in a regulated manner can switch to a different capacity and mediate Jmjd1a recruitment to block repression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is hepatic fibroblast growth factor 21---{\"tf associated gene name\":[\"NR4A1\"],\"tg associated gene name\":[\"FGF21\"],\"tf text\":[\"Nur77\"],\"tg text\":[\"fibroblast growth factor 21\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is hepatic fibroblast growth factor 21\"],\"sentence\":[\"Orphan nuclear receptor Nur77 mediates fasting-induced hepatic fibroblast growth factor 21 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is hepatic fibroblast growth factor 21---{\"tf associated gene name\":[\"NR4A1\"],\"tg associated gene name\":[\"FGF21\"],\"tf text\":[\"Nur77\"],\"tg text\":[\"fibroblast growth factor 21\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is hepatic fibroblast growth factor 21\"],\"sentence\":[\"Orphan nuclear receptor Nur77 mediates fasting-induced hepatic fibroblast growth factor 21 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"valid\":[\"TRUE\"],\"sign\":[\"+\"],\"negation\":[],\"qc\":[\"TRUE\"],\"sentence\":[\"Orphan nuclear receptor Nur77 mediates fasting-induced hepatic fibroblast growth factor 21 expression.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"NR4A1\"],\"target gene (associated gene name)\":[\"FGF21\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Orphan nuclear receptor Nur77 mediates fasting-induced hepatic fibroblast growth factor 21 expression.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A1\"],\"tg (associated gene name)\":[\"FGF21\"],\"tf text\":[\"Nur77\"],\"tg text\":[\"fibroblast growth factor 21\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Orphan nuclear receptor Nur77 mediates fasting-induced hepatic fibroblast growth factor 21 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is SIP1/NHERF2|not sure whether this is TF---{\"tf associated gene name\":[\"ZEB2\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[\"estrogen receptor alpha\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"regulator is SIP1/NHERF2\",\"not sure whether this is TF\"],\"sentence\":[\"SIP1/NHERF2 enhances estrogen receptor alpha transactivation in breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "regulator is SIP1/NHERF2|not sure whether this is TF---{\"tf associated gene name\":[\"ZEB2\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[\"estrogen receptor alpha\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"regulator is SIP1/NHERF2\",\"not sure whether this is TF\"],\"sentence\":[\"SIP1/NHERF2 enhances estrogen receptor alpha transactivation in breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ZEB2\"],\"target gene (associated gene name)\":[\"ESR1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"SIP1/NHERF2 enhances estrogen receptor alpha transactivation in breast cancer cells.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZEB2\"],\"tg (associated gene name)\":[\"ESR1\"],\"tf text\":[],\"tg text\":[\"estrogen receptor alpha\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"SIP1/NHERF2 enhances estrogen receptor alpha transactivation in breast cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 719 (char 718)\n",
      "{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"LHB\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"These results suggest that GnRHa differentially activates ERK and JNK\",\"and a JNK cascade is necessary to elicit the rat LHbeta promoter activity in a c-Jun-dependent mechanism in LbetaT2 cells.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.3153\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"193\"],\"sentence pair density\":[\"0.0072463768115942\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|a bit indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"LHB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These results suggest that GnRHa differentially activates ERK and JNK, and a JNK cascade is necessary to elicit the rat LHbeta promoter activity in a c-Jun-dependent mechanism in LbetaT2 cells.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"LHB\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"These results suggest that GnRHa differentially activates ERK and JNK\",\"and a JNK cascade is necessary to elicit the rat LHbeta promoter activity in a c-Jun-dependent mechanism in LbetaT2 cells.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.3153\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"193\"],\"sentence pair density\":[\"0.0072463768115942\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|a bit indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"LHB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These results suggest that GnRHa differentially activates ERK and JNK, and a JNK cascade is necessary to elicit the rat LHbeta promoter activity in a c-Jun-dependent mechanism in LbetaT2 cells.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "gamma does it---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RORA\"],\"tg (associated gene name)\":[\"SUPT7L\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"gamma does it\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Identification of a novel peroxisome proliferator-activated receptor (PPAR) gamma promoter in man and transactivation by the nuclear receptor RORalpha1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "gamma does it---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RORA\"],\"tg (associated gene name)\":[\"SUPT7L\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"gamma does it\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Identification of a novel peroxisome proliferator-activated receptor (PPAR) gamma promoter in man and transactivation by the nuclear receptor RORalpha1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear TRE|most likely not the one suggested---{\"tf associated gene name\":[\"TEAD1\"],\"tg associated gene name\":[\"CYP2E1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"unclear TRE\",\"most likely not the one suggested\"],\"sentence\":[\"Incubation with NS-398\",\"a COX-2 inhibitor\",\"blocked the effect of AA\",\"but not of ethanol\",\"on COL1A2 expression suggesting that CYP2E1 activates COX-2 expression\",\"and the oxidation of AA by COX-2 is responsible for the increase in COL1A2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEAD1\"],\"tg (associated gene name)\":[\"CYP2E1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Incubation with NS-398, a COX-2 inhibitor, blocked the effect of AA, but not of ethanol, on COL1A2 expression suggesting that CYP2E1 activates COX-2 expression, and the oxidation of AA by COX-2 is responsible for the increase in COL1A2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "unclear TRE|most likely not the one suggested---{\"tf associated gene name\":[\"TEAD1\"],\"tg associated gene name\":[\"CYP2E1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"unclear TRE\",\"most likely not the one suggested\"],\"sentence\":[\"Incubation with NS-398\",\"a COX-2 inhibitor\",\"blocked the effect of AA\",\"but not of ethanol\",\"on COL1A2 expression suggesting that CYP2E1 activates COX-2 expression\",\"and the oxidation of AA by COX-2 is responsible for the increase in COL1A2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEAD1\"],\"tg (associated gene name)\":[\"CYP2E1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Incubation with NS-398, a COX-2 inhibitor, blocked the effect of AA, but not of ethanol, on COL1A2 expression suggesting that CYP2E1 activates COX-2 expression, and the oxidation of AA by COX-2 is responsible for the increase in COL1A2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 662 (char 661)\n",
      "{\"transcription factor (associated gene name)\":[\"NR1I2\"],\"target gene (associated gene name)\":[\"CYP3A4\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Our results implicate a novel and important role for PP2Cbetal in regulating hPXR activity and CYP3A4 expression by inhibiting or desensitizing signaling pathways that negatively regulate the function of pregnane X receptor in liver cells and are consistent with the notion that both the activity of hPXR and the expression of CYP3A4 are regulated in a cell cycle-dependent and cell proliferation-dependent manner.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1I2\"],\"tg (associated gene name)\":[\"CYP3A4\"],\"tf text\":[\"pregnane X receptor\"],\"tg text\":[\"CYP3A4\"],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results implicate a novel and important role for PP2Cbetal in regulating hPXR activity and CYP3A4 expression by inhibiting or desensitizing signaling pathways that negatively regulate the function of pregnane X receptor in liver cells and are consistent with the notion that both the activity of hPXR and the expression of CYP3A4 are regulated in a cell cycle-dependent and cell proliferation-dependent manner.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR1I2\"],\"tg associated gene name\":[\"CYP3A4\"],\"tf text\":[\"pregnane X receptor\"],\"tg text\":[\"CYP3A4\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Our results implicate a novel and important role for PP2Cbetal in regulating hPXR activity and CYP3A4 expression by inhibiting or desensitizing signaling pathways that negatively regulate the function of pregnane X receptor in liver cells and are consistent with the notion that both the activity of hPXR and the expression of CYP3A4 are regulated in a cell cycle-dependent and cell proliferation-dependent manner.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR1I2\"],\"target gene (associated gene name)\":[\"CYP3A4\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Our results implicate a novel and important role for PP2Cbetal in regulating hPXR activity and CYP3A4 expression by inhibiting or desensitizing signaling pathways that negatively regulate the function of pregnane X receptor in liver cells and are consistent with the notion that both the activity of hPXR and the expression of CYP3A4 are regulated in a cell cycle-dependent and cell proliferation-dependent manner.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1I2\"],\"tg (associated gene name)\":[\"CYP3A4\"],\"tf text\":[\"pregnane X receptor\"],\"tg text\":[\"CYP3A4\"],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results implicate a novel and important role for PP2Cbetal in regulating hPXR activity and CYP3A4 expression by inhibiting or desensitizing signaling pathways that negatively regulate the function of pregnane X receptor in liver cells and are consistent with the notion that both the activity of hPXR and the expression of CYP3A4 are regulated in a cell cycle-dependent and cell proliferation-dependent manner.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TFs are IL-6DBP/NF-IL6 and C/EBP delta/NF-IL6 beta (may be correctly normalized|did not check carefully) ---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"IL6\"],\"tf text\":[\"NF-IL6\"],\"tg text\":[\"IL-6\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TFs are IL-6DBP/NF-IL6 and C/EBP delta/NF-IL6 beta (may be correctly normalized\",\"did not check carefully) \"],\"sentence\":[\"The two C/EBP isoforms\",\"IL-6DBP/NF-IL6 and C/EBP delta/NF-IL6 beta\",\"are induced by IL-6 to promote acute phase gene transcription via different mechanisms.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[\"NF-IL6\"],\"tg text\":[\"IL-6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The two C/EBP isoforms, IL-6DBP/NF-IL6 and C/EBP delta/NF-IL6 beta, are induced by IL-6 to promote acute phase gene transcription via different mechanisms.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TFs are IL-6DBP/NF-IL6 and C/EBP delta/NF-IL6 beta (may be correctly normalized|did not check carefully) ---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"IL6\"],\"tf text\":[\"NF-IL6\"],\"tg text\":[\"IL-6\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TFs are IL-6DBP/NF-IL6 and C/EBP delta/NF-IL6 beta (may be correctly normalized\",\"did not check carefully) \"],\"sentence\":[\"The two C/EBP isoforms\",\"IL-6DBP/NF-IL6 and C/EBP delta/NF-IL6 beta\",\"are induced by IL-6 to promote acute phase gene transcription via different mechanisms.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[\"NF-IL6\"],\"tg text\":[\"IL-6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The two C/EBP isoforms, IL-6DBP/NF-IL6 and C/EBP delta/NF-IL6 beta, are induced by IL-6 to promote acute phase gene transcription via different mechanisms.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 398 (char 397)\n",
      "{\"transcription factor (associated gene name)\":[\"MITF\"],\"target gene (associated gene name)\":[\"TYR\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We investigated whether MITF regulates human MC1R gene expression through the same transcriptional mechanism as tyrosinase and TRP-1 genes in melanocytes.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|gene roles unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"TYR\"],\"tf text\":[\"MITF\"],\"tg text\":[\"tyrosinase\"],\"consensus\":[\"V\"],\"martin_comments\":[\"gene roles unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We investigated whether MITF regulates human MC1R gene expression through the same transcriptional mechanism as tyrosinase and TRP-1 genes in melanocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"TYR\"],\"tf text\":[\"MITF\"],\"tg text\":[\"tyrosinase\"],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"We investigated whether MITF regulates human MC1R gene expression through the same transcriptional mechanism as tyrosinase and TRP-1 genes in melanocytes.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MITF\"],\"target gene (associated gene name)\":[\"TYR\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We investigated whether MITF regulates human MC1R gene expression through the same transcriptional mechanism as tyrosinase and TRP-1 genes in melanocytes.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|gene roles unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"TYR\"],\"tf text\":[\"MITF\"],\"tg text\":[\"tyrosinase\"],\"consensus\":[\"V\"],\"martin_comments\":[\"gene roles unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We investigated whether MITF regulates human MC1R gene expression through the same transcriptional mechanism as tyrosinase and TRP-1 genes in melanocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PLAG1\"],\"tg (associated gene name)\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"RRME treatment also decreased transcription of AR regulated genes and also reduced histone H3 and AR acetylation in the promoters of prostate-specific antigen (PSA) and beta-2-microglobulin (B2M).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PLAG1\"],\"tg (associated gene name)\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"RRME treatment also decreased transcription of AR regulated genes and also reduced histone H3 and AR acetylation in the promoters of prostate-specific antigen (PSA) and beta-2-microglobulin (B2M).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "random ADAM? TGs---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"random ADAM? TGs\"],\"sentence\":[\"Virus infection potently activates the binding and nuclear localization of NF-kappa B and\",\"in pre-B lymphocytes\",\"results in the expression of both the beta-IFN gene and the Ig kappa gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|beta distorts---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"beta distorts\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Virus infection potently activates the binding and nuclear localization of NF-kappa B and, in pre-B lymphocytes, results in the expression of both the beta-IFN gene and the Ig kappa gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "random ADAM? TGs---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"random ADAM? TGs\"],\"sentence\":[\"Virus infection potently activates the binding and nuclear localization of NF-kappa B and\",\"in pre-B lymphocytes\",\"results in the expression of both the beta-IFN gene and the Ig kappa gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|beta distorts---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"beta distorts\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Virus infection potently activates the binding and nuclear localization of NF-kappa B and, in pre-B lymphocytes, results in the expression of both the beta-IFN gene and the Ig kappa gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is BZLF1|\"lytic induction\" indicates virus---{\"tf associated gene name\":[\"KMT2A\"],\"tg associated gene name\":[\"KRT4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\",\"n\"],\"comment\":[\"TG is BZLF1\",\"\\\"lytic induction\\\" indicates virus\"],\"sentence\":[\"MLL1\",\"the host protein known to dimethylate H3K4\",\"was found to associate with the BZLF1 promoter upon lytic induction in a TAF-I-dependent manner\",\"and MLL1 depletion decreased BZLF1 expression\",\"confirming its contribution to lytic reactivation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|K4?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KMT2A\"],\"tg (associated gene name)\":[\"KRT4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"K4?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MLL1, the host protein known to dimethylate H3K4, was found to associate with the BZLF1 promoter upon lytic induction in a TAF-I-dependent manner, and MLL1 depletion decreased BZLF1 expression, confirming its contribution to lytic reactivation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is BZLF1|\"lytic induction\" indicates virus---{\"tf associated gene name\":[\"KMT2A\"],\"tg associated gene name\":[\"KRT4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\",\"n\"],\"comment\":[\"TG is BZLF1\",\"\\\"lytic induction\\\" indicates virus\"],\"sentence\":[\"MLL1\",\"the host protein known to dimethylate H3K4\",\"was found to associate with the BZLF1 promoter upon lytic induction in a TAF-I-dependent manner\",\"and MLL1 depletion decreased BZLF1 expression\",\"confirming its contribution to lytic reactivation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|K4?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KMT2A\"],\"tg (associated gene name)\":[\"KRT4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"K4?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MLL1, the host protein known to dimethylate H3K4, was found to associate with the BZLF1 promoter upon lytic induction in a TAF-I-dependent manner, and MLL1 depletion decreased BZLF1 expression, confirming its contribution to lytic reactivation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 363 (char 362)\n",
      "{\"transcription factor (associated gene name)\":[\"DNMT3B\"],\"target gene (associated gene name)\":[\"DNMT3L\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Genetic studies showed that Dnmt3a\",\"Dnmt3b and Dnmt3L are all involved in the methylation of the Dnmt3L promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|no POLII act?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"DNMT3B\"],\"tg (associated gene name)\":[\"DNMT3L\"],\"tf text\":[\"Dnmt3b\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"no POLII act?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Genetic studies showed that Dnmt3a, Dnmt3b and Dnmt3L are all involved in the methylation of the Dnmt3L promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"DNMT3B\"],\"tg associated gene name\":[\"DNMT3L\"],\"tf text\":[\"Dnmt3b\"],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Genetic studies showed that Dnmt3a\",\"Dnmt3b and Dnmt3L are all involved in the methylation of the Dnmt3L promoter.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"DNMT3B\"],\"target gene (associated gene name)\":[\"DNMT3L\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Genetic studies showed that Dnmt3a\",\"Dnmt3b and Dnmt3L are all involved in the methylation of the Dnmt3L promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|no POLII act?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"DNMT3B\"],\"tg (associated gene name)\":[\"DNMT3L\"],\"tf text\":[\"Dnmt3b\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"no POLII act?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Genetic studies showed that Dnmt3a, Dnmt3b and Dnmt3L are all involved in the methylation of the Dnmt3L promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG normalization very uncertain---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"SUPT7L\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TG normalization very uncertain\"],\"sentence\":[\"Chromatin immunoprecipitation assays show the endogenous IkappaBalpha/epsilon\",\"Gro-beta/gamma\",\"and Naf-1 promoters directly bound NF-kappaB/Rel A in TNF-stimulated cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|gamma does it---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"SUPT7L\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"gamma does it\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Chromatin immunoprecipitation assays show the endogenous IkappaBalpha/epsilon, Gro-beta/gamma, and Naf-1 promoters directly bound NF-kappaB/Rel A in TNF-stimulated cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG normalization very uncertain---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"SUPT7L\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TG normalization very uncertain\"],\"sentence\":[\"Chromatin immunoprecipitation assays show the endogenous IkappaBalpha/epsilon\",\"Gro-beta/gamma\",\"and Naf-1 promoters directly bound NF-kappaB/Rel A in TNF-stimulated cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|gamma does it---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"SUPT7L\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"gamma does it\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Chromatin immunoprecipitation assays show the endogenous IkappaBalpha/epsilon, Gro-beta/gamma, and Naf-1 promoters directly bound NF-kappaB/Rel A in TNF-stimulated cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 400 (char 399)\n",
      "{\"tf associated gene name\":[\"SMAD2\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"SMAD2\"],\"tg text\":[\"MDM2\"],\"1\":[],\"0\":[\"p/up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Here we show that MDM2 and the structurally related protein MDMX can inhibit the transcriptional activity of ectopically expressed SMAD1\",\"SMAD2\",\"SMAD3\",\"and SMAD4.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD2\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"SMAD2\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we show that MDM2 and the structurally related protein MDMX can inhibit the transcriptional activity of ectopically expressed SMAD1, SMAD2, SMAD3, and SMAD4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SMAD2\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"SMAD2\"],\"tg text\":[\"MDM2\"],\"1\":[],\"0\":[\"p/up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Here we show that MDM2 and the structurally related protein MDMX can inhibit the transcriptional activity of ectopically expressed SMAD1\",\"SMAD2\",\"SMAD3\",\"and SMAD4.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD2\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"SMAD2\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we show that MDM2 and the structurally related protein MDMX can inhibit the transcriptional activity of ectopically expressed SMAD1, SMAD2, SMAD3, and SMAD4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"CHUK\"],\"tf text\":[\"ETS-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These results suggest that the proximal 5'-flanking region of the IKKalpha gene contains a functional promoter reciprocally regulated by p53 and ETS-1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"CHUK\"],\"tf text\":[\"ETS-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These results suggest that the proximal 5'-flanking region of the IKKalpha gene contains a functional promoter reciprocally regulated by p53 and ETS-1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"THRA\"],\"tg (associated gene name)\":[\"CA2\"],\"tf text\":[],\"tg text\":[\"carbonic anhydrase II\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Apart from repression by v-ErbA via recruitment of corepressor complexes, other repressors and corepressors appear to be involved in repression of v-ErbA target genes, such as carbonic anhydrase II (CAII).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"THRA\"],\"tg (associated gene name)\":[\"CA2\"],\"tf text\":[],\"tg text\":[\"carbonic anhydrase II\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Apart from repression by v-ErbA via recruitment of corepressor complexes, other repressors and corepressors appear to be involved in repression of v-ErbA target genes, such as carbonic anhydrase II (CAII).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TRE---{\"tf associated gene name\":[\"NEUROD1\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TRE\"],\"sentence\":[\"Thus we constructed two beta-galactosidase reporter genes\",\"one (A beta 2.0kb-lac z) containing the promoter region to a part of the second exon of the class II gene\",\"and the other (A beta pro-lac z) containing the promoter region of the class II gene alone.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|classII?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NEUROD1\"],\"tg (associated gene name)\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"classII?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus we constructed two beta-galactosidase reporter genes, one (A beta 2.0kb-lac z) containing the promoter region to a part of the second exon of the class II gene, and the other (A beta pro-lac z) containing the promoter region of the class II gene alone.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TRE---{\"tf associated gene name\":[\"NEUROD1\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TRE\"],\"sentence\":[\"Thus we constructed two beta-galactosidase reporter genes\",\"one (A beta 2.0kb-lac z) containing the promoter region to a part of the second exon of the class II gene\",\"and the other (A beta pro-lac z) containing the promoter region of the class II gene alone.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|classII?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NEUROD1\"],\"tg (associated gene name)\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"classII?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus we constructed two beta-galactosidase reporter genes, one (A beta 2.0kb-lac z) containing the promoter region to a part of the second exon of the class II gene, and the other (A beta pro-lac z) containing the promoter region of the class II gene alone.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 415 (char 414)\n",
      "{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The Egr-1-binding protein Nab2 is an endogenous inhibitor of Egr-1-mediated signalling and abrogates the stimulation of fibrotic responses induced by transforming growth factor-?? (TGF??).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no transcription implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"TGFB1\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no transcription implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The Egr-1-binding protein Nab2 is an endogenous inhibitor of Egr-1-mediated signalling and abrogates the stimulation of fibrotic responses induced by transforming growth factor- (TGF).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The Egr-1-binding protein Nab2 is an endogenous inhibitor of Egr-1-mediated signalling and abrogates the stimulation of fibrotic responses induced by transforming growth factor-?? (TGF??).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no transcription implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"TGFB1\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no transcription implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The Egr-1-binding protein Nab2 is an endogenous inhibitor of Egr-1-mediated signalling and abrogates the stimulation of fibrotic responses induced by transforming growth factor- (TGF).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 616 (char 615)\n",
      "{\"transcription factor (associated gene name)\":[\"E2F6\"],\"target gene (associated gene name)\":[\"DTL\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"By measuring the luciferase activity in A549 cells\",\"the intergenic human sequence was shown to be able to drive the reporter gene expression in either direction and notably\",\"among transcription factors E2F\",\"E2F1 approximately E2F4\",\"but not E2F5 and E2F6\",\"this sequence clearly up-regulated the reporter gene expression exclusively in the direction of the CDT2 gene.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---| in the direction of the CDT2 gene. Unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F6\"],\"tg (associated gene name)\":[\"DTL\"],\"tf text\":[\"E2F6\"],\"tg text\":[\"CDT2\"],\"consensus\":[\"V\"],\"martin_comments\":[\" in the direction of the CDT2 gene. Unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By measuring the luciferase activity in A549 cells, the intergenic human sequence was shown to be able to drive the reporter gene expression in either direction and notably, among transcription factors E2F, E2F1 approximately E2F4, but not E2F5 and E2F6, this sequence clearly up-regulated the reporter gene expression exclusively in the direction of the CDT2 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"E2F6\"],\"tg associated gene name\":[\"DTL\"],\"tf text\":[\"E2F6\"],\"tg text\":[\"CDT2\"],\"1\":[\"u\"],\"negation\":[\"x\"],\"not true\":[],\"comment\":[],\"sentence\":[\"By measuring the luciferase activity in A549 cells\",\"the intergenic human sequence was shown to be able to drive the reporter gene expression in either direction and notably\",\"among transcription factors E2F\",\"E2F1 approximately E2F4\",\"but not E2F5 and E2F6\",\"this sequence clearly up-regulated the reporter gene expression exclusively in the direction of the CDT2 gene.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"E2F6\"],\"target gene (associated gene name)\":[\"DTL\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"By measuring the luciferase activity in A549 cells\",\"the intergenic human sequence was shown to be able to drive the reporter gene expression in either direction and notably\",\"among transcription factors E2F\",\"E2F1 approximately E2F4\",\"but not E2F5 and E2F6\",\"this sequence clearly up-regulated the reporter gene expression exclusively in the direction of the CDT2 gene.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---| in the direction of the CDT2 gene. Unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F6\"],\"tg (associated gene name)\":[\"DTL\"],\"tf text\":[\"E2F6\"],\"tg text\":[\"CDT2\"],\"consensus\":[\"V\"],\"martin_comments\":[\" in the direction of the CDT2 gene. Unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By measuring the luciferase activity in A549 cells, the intergenic human sequence was shown to be able to drive the reporter gene expression in either direction and notably, among transcription factors E2F, E2F1 approximately E2F4, but not E2F5 and E2F6, this sequence clearly up-regulated the reporter gene expression exclusively in the direction of the CDT2 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Stat3---{\"tf associated gene name\":[\"SNAI1\"],\"tg associated gene name\":[\"ENPP2\"],\"tf text\":[],\"tg text\":[\"ENPP2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Stat3\"],\"sentence\":[\"In addition to known Stat3 transcriptional targets (Twist\",\"Snail\",\"Tenascin-C and IL-8)\",\"we identified ENPP2 as a novel Stat3 regulated gene\",\"which encodes autotaxin (ATX)\",\"a secreted lysophospholipase which mediates mammary tumorigenesis and cancer cell migration.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|STAT is TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SNAI1\"],\"tg (associated gene name)\":[\"ENPP2\"],\"tf text\":[],\"tg text\":[\"ENPP2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"STAT is TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In addition to known Stat3 transcriptional targets (Twist, Snail, Tenascin-C and IL-8), we identified ENPP2 as a novel Stat3 regulated gene, which encodes autotaxin (ATX), a secreted lysophospholipase which mediates mammary tumorigenesis and cancer cell migration.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is Stat3---{\"tf associated gene name\":[\"SNAI1\"],\"tg associated gene name\":[\"ENPP2\"],\"tf text\":[],\"tg text\":[\"ENPP2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Stat3\"],\"sentence\":[\"In addition to known Stat3 transcriptional targets (Twist\",\"Snail\",\"Tenascin-C and IL-8)\",\"we identified ENPP2 as a novel Stat3 regulated gene\",\"which encodes autotaxin (ATX)\",\"a secreted lysophospholipase which mediates mammary tumorigenesis and cancer cell migration.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|STAT is TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SNAI1\"],\"tg (associated gene name)\":[\"ENPP2\"],\"tf text\":[],\"tg text\":[\"ENPP2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"STAT is TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In addition to known Stat3 transcriptional targets (Twist, Snail, Tenascin-C and IL-8), we identified ENPP2 as a novel Stat3 regulated gene, which encodes autotaxin (ATX), a secreted lysophospholipase which mediates mammary tumorigenesis and cancer cell migration.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  ALR---{\"tf associated gene name\":[\"FOXA2\"],\"tg associated gene name\":[\"EXOC4\"],\"tf text\":[\"Foxa2\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is  ALR\"],\"sentence\":[\"Analysis of the promoter activity by performing luciferase assays revealed that co-transfection with Foxa2 significantly induced the activity of ALR promoter in HepG2 cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|EXOC4?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXA2\"],\"tg (associated gene name)\":[\"EXOC4\"],\"tf text\":[\"Foxa2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"EXOC4?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Analysis of the promoter activity by performing luciferase assays revealed that co-transfection with Foxa2 significantly induced the activity of ALR promoter in HepG2 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is  ALR---{\"tf associated gene name\":[\"FOXA2\"],\"tg associated gene name\":[\"EXOC4\"],\"tf text\":[\"Foxa2\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is  ALR\"],\"sentence\":[\"Analysis of the promoter activity by performing luciferase assays revealed that co-transfection with Foxa2 significantly induced the activity of ALR promoter in HepG2 cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|EXOC4?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXA2\"],\"tg (associated gene name)\":[\"EXOC4\"],\"tf text\":[\"Foxa2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"EXOC4?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Analysis of the promoter activity by performing luciferase assays revealed that co-transfection with Foxa2 significantly induced the activity of ALR promoter in HepG2 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 702 (char 701)\n",
      "{\"transcription factor (associated gene name)\":[\"E2F4\"],\"target gene (associated gene name)\":[\"E2F4\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Coexpression of DP1 with E2F1 or E2F4 in the epidermis of bigenic mice modestly enhanced proliferation and apoptosis over the levels induced by E2F1 or E2F4 expression alone.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.451762\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"174\"],\"sentence pair density\":[\"0.0235294117647058\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F4\"],\"tg (associated gene name)\":[\"E2F4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Coexpression of DP1 with E2F1 or E2F4 in the epidermis of bigenic mice modestly enhanced proliferation and apoptosis over the levels induced by E2F1 or E2F4 expression alone.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"E2F4\"],\"tg associated gene name\":[\"E2F4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Coexpression of DP1 with E2F1 or E2F4 in the epidermis of bigenic mice modestly enhanced proliferation and apoptosis over the levels induced by E2F1 or E2F4 expression alone.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"E2F4\"],\"target gene (associated gene name)\":[\"E2F4\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Coexpression of DP1 with E2F1 or E2F4 in the epidermis of bigenic mice modestly enhanced proliferation and apoptosis over the levels induced by E2F1 or E2F4 expression alone.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.451762\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"174\"],\"sentence pair density\":[\"0.0235294117647058\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F4\"],\"tg (associated gene name)\":[\"E2F4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Coexpression of DP1 with E2F1 or E2F4 in the epidermis of bigenic mice modestly enhanced proliferation and apoptosis over the levels induced by E2F1 or E2F4 expression alone.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain norm p39---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"1\":[\"u\"],\"not true\":[\"n?\"],\"comment\":[\"uncertain norm p39\"],\"sentence\":[\"Electrophoretic mobility shift assays revealed that Sp1 and Sp3 bound to sequences required for p39 promoter function and chromatin immunoprecipitation assays confirmed binding of these proteins to the endogenous p39 promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "uncertain norm p39---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"1\":[\"u\"],\"not true\":[\"n?\"],\"comment\":[\"uncertain norm p39\"],\"sentence\":[\"Electrophoretic mobility shift assays revealed that Sp1 and Sp3 bound to sequences required for p39 promoter function and chromatin immunoprecipitation assays confirmed binding of these proteins to the endogenous p39 promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP3\"],\"target gene (associated gene name)\":[\"JUN\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"v\"],\"negation\":[],\"sentence\":[\"Electrophoretic mobility shift assays revealed that Sp1 and Sp3 bound to sequences required for p39 promoter function and chromatin immunoprecipitation assays confirmed binding of these proteins to the endogenous p39 promoter.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.067856\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"226\"],\"sentence pair density\":[\"0.0048076923076923\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP3\"],\"target gene (associated gene name)\":[\"JUN\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Electrophoretic mobility shift assays revealed that Sp1 and Sp3 bound to sequences required for p39 promoter function and chromatin immunoprecipitation assays confirmed binding of these proteins to the endogenous p39 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP3\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Electrophoretic mobility shift assays revealed that Sp1 and Sp3 bound to sequences required for p39 promoter function and chromatin immunoprecipitation assays confirmed binding of these proteins to the endogenous p39 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG|NOTCH---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZEB1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[\"NOTCH\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Thus, while ZEB-1/deltaEF1 binds to p300 and promotes the formation of a p300-Smad transcriptional complex, ZEB-2/SIP1 acts as a repressor by recruiting CtBP.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG|NOTCH---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZEB1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[\"NOTCH\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Thus, while ZEB-1/deltaEF1 binds to p300 and promotes the formation of a p300-Smad transcriptional complex, ZEB-2/SIP1 acts as a repressor by recruiting CtBP.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "is DNMT1 DbTF?---{\"tf associated gene name\":[\"DNMT1\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[\"DNMT1\"],\"tg text\":[],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"is DNMT1 DbTF?\"],\"sentence\":[\"Direct inhibition of ERK activity similarly decreased DNMT1 protein levels and reversed the basal hypermethylation of the promoters and silencing of the RARB2\",\"p21 and p16 tumor suppressor genes.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "is DNMT1 DbTF?---{\"tf associated gene name\":[\"DNMT1\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[\"DNMT1\"],\"tg text\":[],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"is DNMT1 DbTF?\"],\"sentence\":[\"Direct inhibition of ERK activity similarly decreased DNMT1 protein levels and reversed the basal hypermethylation of the promoters and silencing of the RARB2\",\"p21 and p16 tumor suppressor genes.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"DNMT1\"],\"target gene (associated gene name)\":[\"CDKN1A\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Direct inhibition of ERK activity similarly decreased DNMT1 protein levels and reversed the basal hypermethylation of the promoters and silencing of the RARB2\",\"p21 and p16 tumor suppressor genes.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|no POLII act, through methylation ok?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"DNMT1\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[\"DNMT1\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"no POLII act, through methylation ok?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Direct inhibition of ERK activity similarly decreased DNMT1 protein levels and reversed the basal hypermethylation of the promoters and silencing of the RARB2, p21 and p16 tumor suppressor genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "AKT not TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"AKT1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AKT not TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In EC cell lines, transfection of the NF-kappaB subunit p65 led to transactivation of Par-4 through specific binding to its promoter region, in contrast to the suppression by active Akt, suggesting that the balance between the two signals may be important to determine Par-4 expression levels.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "AKT not TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"AKT1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AKT not TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In EC cell lines, transfection of the NF-kappaB subunit p65 led to transactivation of Par-4 through specific binding to its promoter region, in contrast to the suppression by active Akt, suggesting that the balance between the two signals may be important to determine Par-4 expression levels.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "PLAG1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PLAG1\"],\"tg (associated gene name)\":[\"KLK4\"],\"tf text\":[],\"tg text\":[\"KLK4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"PLAG1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Like other KLKs, such as KLK3/PSA and KLK2, KLK4 gene expression is also regulated by steroid hormones in hormone-dependent cancers, although the transcriptional mechanisms are ill defined.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "PLAG1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PLAG1\"],\"tg (associated gene name)\":[\"KLK4\"],\"tf text\":[],\"tg text\":[\"KLK4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"PLAG1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Like other KLKs, such as KLK3/PSA and KLK2, KLK4 gene expression is also regulated by steroid hormones in hormone-dependent cancers, although the transcriptional mechanisms are ill defined.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 723 (char 722)\n",
      "{\"transcription factor (associated gene name)\":[\"FOS\"],\"target gene (associated gene name)\":[\"JUND\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"The AP-1 complex formed in UVB-irradiated HaCaT cells is specifically composed of c-fos and Jun D. c-Fos expression was induced in a manner that correlated with the UVB-induced activation of AP-1.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.124558\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"5\"],\"sentence length\":[\"196\"],\"sentence pair density\":[\"0.0161290322580645\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|No TG. Two sentences!---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"JUND\"],\"tf text\":[\"c-Fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"No TG. Two sentences!\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The AP-1 complex formed in UVB-irradiated HaCaT cells is specifically composed of c-fos and Jun D. c-Fos expression was induced in a manner that correlated with the UVB-induced activation of AP-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"JUND\"],\"tf text\":[\"c-Fos\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"The AP-1 complex formed in UVB-irradiated HaCaT cells is specifically composed of c-fos and Jun D. c-Fos expression was induced in a manner that correlated with the UVB-induced activation of AP-1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOS\"],\"target gene (associated gene name)\":[\"JUND\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"The AP-1 complex formed in UVB-irradiated HaCaT cells is specifically composed of c-fos and Jun D. c-Fos expression was induced in a manner that correlated with the UVB-induced activation of AP-1.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.124558\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"5\"],\"sentence length\":[\"196\"],\"sentence pair density\":[\"0.0161290322580645\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|No TG. Two sentences!---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"JUND\"],\"tf text\":[\"c-Fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"No TG. Two sentences!\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The AP-1 complex formed in UVB-irradiated HaCaT cells is specifically composed of c-fos and Jun D. c-Fos expression was induced in a manner that correlated with the UVB-induced activation of AP-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 512 (char 511)\n",
      "{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[\"CREB\"],\"tg text\":[\"cAMP-response element-binding protein (CREB)-binding protein\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"The interactions between cAMP-response element-binding protein (CREB)-binding protein (CBP) and gene-specific transcription factors play an important role in activation of transcription from numerous genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"CREB\"],\"tg text\":[\"cAMP-response element-binding protein (CREB)-binding protein\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The interactions between cAMP-response element-binding protein (CREB)-binding protein (CBP) and gene-specific transcription factors play an important role in activation of transcription from numerous genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[\"CREB\"],\"tg text\":[\"cAMP-response element-binding protein (CREB)-binding protein\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"The interactions between cAMP-response element-binding protein (CREB)-binding protein (CBP) and gene-specific transcription factors play an important role in activation of transcription from numerous genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"CREB\"],\"tg text\":[\"cAMP-response element-binding protein (CREB)-binding protein\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The interactions between cAMP-response element-binding protein (CREB)-binding protein (CBP) and gene-specific transcription factors play an important role in activation of transcription from numerous genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG identifyable---{\"tf associated gene name\":[\"REST\"],\"tg associated gene name\":[\"KRT9\"],\"tf text\":[\"REST\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TG identifyable\"],\"sentence\":[\"An SMCX complex isolated from HeLa cells contains additional chromatin modifiers (the histone deacetylases HDAC1 and HDAC2\",\"and the histone H3K9 methyltransferase G9a) and the transcriptional repressor REST\",\"suggesting a direct role for SMCX in chromatin dynamics and REST-mediated repression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"REST\"],\"tg (associated gene name)\":[\"KRT9\"],\"tf text\":[\"REST\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"An SMCX complex isolated from HeLa cells contains additional chromatin modifiers (the histone deacetylases HDAC1 and HDAC2, and the histone H3K9 methyltransferase G9a) and the transcriptional repressor REST, suggesting a direct role for SMCX in chromatin dynamics and REST-mediated repression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG identifyable---{\"tf associated gene name\":[\"REST\"],\"tg associated gene name\":[\"KRT9\"],\"tf text\":[\"REST\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TG identifyable\"],\"sentence\":[\"An SMCX complex isolated from HeLa cells contains additional chromatin modifiers (the histone deacetylases HDAC1 and HDAC2\",\"and the histone H3K9 methyltransferase G9a) and the transcriptional repressor REST\",\"suggesting a direct role for SMCX in chromatin dynamics and REST-mediated repression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"REST\"],\"tg (associated gene name)\":[\"KRT9\"],\"tf text\":[\"REST\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"An SMCX complex isolated from HeLa cells contains additional chromatin modifiers (the histone deacetylases HDAC1 and HDAC2, and the histone H3K9 methyltransferase G9a) and the transcriptional repressor REST, suggesting a direct role for SMCX in chromatin dynamics and REST-mediated repression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 430 (char 429)\n",
      "{\"transcription factor (associated gene name)\":[\"NFE2L2\"],\"target gene (associated gene name)\":[\"TFAM\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Here we establish that the proximal promoter of the human mtTFA gene is highly dependent upon recognition sites for the nuclear respiratory factors\",\"NRF-1 and NRF-2\",\"for activity.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|NRF-2 = NRF2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"TFAM\"],\"tf text\":[],\"tg text\":[\"mtTFA\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NRF-2 = NRF2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we establish that the proximal promoter of the human mtTFA gene is highly dependent upon recognition sites for the nuclear respiratory factors, NRF-1 and NRF-2, for activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"TFAM\"],\"tf text\":[],\"tg text\":[\"mtTFA\"],\"1\":[\"a\"],\"not true\":[],\"comment\":[],\"sentence\":[\"Here we establish that the proximal promoter of the human mtTFA gene is highly dependent upon recognition sites for the nuclear respiratory factors\",\"NRF-1 and NRF-2\",\"for activity.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFE2L2\"],\"target gene (associated gene name)\":[\"TFAM\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Here we establish that the proximal promoter of the human mtTFA gene is highly dependent upon recognition sites for the nuclear respiratory factors\",\"NRF-1 and NRF-2\",\"for activity.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|NRF-2 = NRF2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"TFAM\"],\"tf text\":[],\"tg text\":[\"mtTFA\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NRF-2 = NRF2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we establish that the proximal promoter of the human mtTFA gene is highly dependent upon recognition sites for the nuclear respiratory factors, NRF-1 and NRF-2, for activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"MMP9\"],\"tf text\":[],\"tg text\":[\"MMP-9\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"c-Raf-1 is an upstream activator of MEK1 and a kinase-deficient c-Raf-1 expression construct decreased the activity of a promoter driven by either the MMP-9 promoter or three tandem AP-1 repeats.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"MMP9\"],\"tf text\":[],\"tg text\":[\"MMP-9\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"c-Raf-1 is an upstream activator of MEK1 and a kinase-deficient c-Raf-1 expression construct decreased the activity of a promoter driven by either the MMP-9 promoter or three tandem AP-1 repeats.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"MMP9\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"c-Raf-1 is an upstream activator of MEK1 and a kinase-deficient c-Raf-1 expression construct decreased the activity of a promoter driven by either the MMP-9 promoter or three tandem AP-1 repeats.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|AP1 ok? c-RAF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MMP9\"],\"tf text\":[],\"tg text\":[\"MMP-9\"],\"consensus\":[\"V\"],\"martin_comments\":[\"AP1 ok? c-RAF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"c-Raf-1 is an upstream activator of MEK1 and a kinase-deficient c-Raf-1 expression construct decreased the activity of a promoter driven by either the MMP-9 promoter or three tandem AP-1 repeats.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not named---{\"tf associated gene name\":[\"DNMT1\"],\"tg associated gene name\":[\"PSMD10\"],\"tf text\":[],\"tg text\":[\"gankyrin\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF not named\"],\"sentence\":[\"The aim of this study was to determine the mechanisms that repress gankyrin in quiescent livers and mechanisms of activation of gankyrin in liver cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"DNMT1\"],\"tg (associated gene name)\":[\"PSMD10\"],\"tf text\":[],\"tg text\":[\"gankyrin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The aim of this study was to determine the mechanisms that repress gankyrin in quiescent livers and mechanisms of activation of gankyrin in liver cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF not named---{\"tf associated gene name\":[\"DNMT1\"],\"tg associated gene name\":[\"PSMD10\"],\"tf text\":[],\"tg text\":[\"gankyrin\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF not named\"],\"sentence\":[\"The aim of this study was to determine the mechanisms that repress gankyrin in quiescent livers and mechanisms of activation of gankyrin in liver cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"DNMT1\"],\"tg (associated gene name)\":[\"PSMD10\"],\"tf text\":[],\"tg text\":[\"gankyrin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The aim of this study was to determine the mechanisms that repress gankyrin in quiescent livers and mechanisms of activation of gankyrin in liver cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "is GRE---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"RSAD2\"],\"tf text\":[\"IRF1\"],\"tg text\":[\"Viperin\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"is GRE\"],\"sentence\":[\"In this study\",\"we show that CD40 receptor occupancy results in IFN-?? upregulation through an unconventional \\\"feed-forward\\\" mechanism\",\"which is orchestrated by canonical NF-??B and involves the sequential de novo synthesis of IFN regulatory factor (IRF)1 and Viperin (RSAD2)\",\"an IRF1 target.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "is GRE---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"RSAD2\"],\"tf text\":[\"IRF1\"],\"tg text\":[\"Viperin\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"is GRE\"],\"sentence\":[\"In this study\",\"we show that CD40 receptor occupancy results in IFN-?? upregulation through an unconventional \\\"feed-forward\\\" mechanism\",\"which is orchestrated by canonical NF-??B and involves the sequential de novo synthesis of IFN regulatory factor (IRF)1 and Viperin (RSAD2)\",\"an IRF1 target.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF1\"],\"target gene (associated gene name)\":[\"RSAD2\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"a\"],\"negation\":[],\"sentence\":[\"In this study\",\"we show that CD40 receptor occupancy results in IFN- upregulation through an unconventional \\\"feed-forward\\\" mechanism\",\"which is orchestrated by canonical NF-B and involves the sequential de novo synthesis of IFN regulatory factor (IRF)1 and Viperin (RSAD2)\",\"an IRF1 target.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.207439\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"287\"],\"sentence pair density\":[\"0.0207253886010362\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF1\"],\"target gene (associated gene name)\":[\"RSAD2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In this study\",\"we show that CD40 receptor occupancy results in IFN- upregulation through an unconventional \\\"feed-forward\\\" mechanism\",\"which is orchestrated by canonical NF-B and involves the sequential de novo synthesis of IFN regulatory factor (IRF)1 and Viperin (RSAD2)\",\"an IRF1 target.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|complex---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"RSAD2\"],\"tf text\":[\"IRF1\"],\"tg text\":[\"Viperin\"],\"consensus\":[\"V\"],\"martin_comments\":[\"complex\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this study, we show that CD40 receptor occupancy results in IFN- upregulation through an unconventional \\\"feed-forward\\\" mechanism, which is orchestrated by canonical NF-B and involves the sequential de novo synthesis of IFN regulatory factor (IRF)1 and Viperin (RSAD2), an IRF1 target.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 389 (char 388)\n",
      "{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"TP53\"],\"tf text\":[\"p53\"],\"tg text\":[\"p53\"],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"p\"],\"comment\":[],\"sentence\":[\"PAK1IP1 bound to MDM2 and inhibited its ability to ubiquitinate and to degrade p53\",\"consequently leading to the accumulation of p53 levels.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[\"p53\"],\"tg text\":[\"p53\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PAK1IP1 bound to MDM2 and inhibited its ability to ubiquitinate and to degrade p53, consequently leading to the accumulation of p53 levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"TP53\"],\"tf text\":[\"p53\"],\"tg text\":[\"p53\"],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"p\"],\"comment\":[],\"sentence\":[\"PAK1IP1 bound to MDM2 and inhibited its ability to ubiquitinate and to degrade p53\",\"consequently leading to the accumulation of p53 levels.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[\"p53\"],\"tg text\":[\"p53\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PAK1IP1 bound to MDM2 and inhibited its ability to ubiquitinate and to degrade p53, consequently leading to the accumulation of p53 levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs are NF-L and NF-M---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"NEFH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TGs are NF-L and NF-M\"],\"sentence\":[\"While NGF treatment resulted in 12- and 14-fold increases in NF-L and NF-M mRNA levels\",\"respectively\",\"over a 14-day period\",\"no increase in the level of NF-H mRNA was observed.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"NEFH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"While NGF treatment resulted in 12- and 14-fold increases in NF-L and NF-M mRNA levels, respectively, over a 14-day period, no increase in the level of NF-H mRNA was observed.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TGs are NF-L and NF-M---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"NEFH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TGs are NF-L and NF-M\"],\"sentence\":[\"While NGF treatment resulted in 12- and 14-fold increases in NF-L and NF-M mRNA levels\",\"respectively\",\"over a 14-day period\",\"no increase in the level of NF-H mRNA was observed.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"NEFH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"While NGF treatment resulted in 12- and 14-fold increases in NF-L and NF-M mRNA levels, respectively, over a 14-day period, no increase in the level of NF-H mRNA was observed.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Bax---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"YAP1\"],\"tf text\":[\"EGR-1\"],\"tg text\":[\"Yes kinase-associated protein 1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is Bax\"],\"sentence\":[\"Interaction of EGR-1 and Yes kinase-associated protein 1 (YAP-1) through the WW domain of YAP-1 enhances the transcriptional activity of EGR-1 on the Bax promoter as shown by chromatin immunoprecipitation and reporter assays.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Bax---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"YAP1\"],\"tf text\":[\"EGR-1\"],\"tg text\":[\"Yes kinase-associated protein 1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is Bax\"],\"sentence\":[\"Interaction of EGR-1 and Yes kinase-associated protein 1 (YAP-1) through the WW domain of YAP-1 enhances the transcriptional activity of EGR-1 on the Bax promoter as shown by chromatin immunoprecipitation and reporter assays.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"YAP1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Interaction of EGR-1 and Yes kinase-associated protein 1 (YAP-1) through the WW domain of YAP-1 enhances the transcriptional activity of EGR-1 on the Bax promoter as shown by chromatin immunoprecipitation and reporter assays.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.29999\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"225\"],\"sentence pair density\":[\"0.00816326530612245\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=BAX---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"YAP1\"],\"tf text\":[\"EGR-1\"],\"tg text\":[\"Yes kinase-associated protein 1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=BAX\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Interaction of EGR-1 and Yes kinase-associated protein 1 (YAP-1) through the WW domain of YAP-1 enhances the transcriptional activity of EGR-1 on the Bax promoter as shown by chromatin immunoprecipitation and reporter assays.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is endothelial tissue factor---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is endothelial tissue factor\"],\"sentence\":[\"As a consequence\",\"increased cellular oxidative stress led to the activation of the transcription factor NF-kappaB and thus promoted the upregulation of various NF-kappaB-controlled genes\",\"including endothelial tissue factor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=endothelial tissue factor---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=endothelial tissue factor\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As a consequence, increased cellular oxidative stress led to the activation of the transcription factor NF-kappaB and thus promoted the upregulation of various NF-kappaB-controlled genes, including endothelial tissue factor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is endothelial tissue factor---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is endothelial tissue factor\"],\"sentence\":[\"As a consequence\",\"increased cellular oxidative stress led to the activation of the transcription factor NF-kappaB and thus promoted the upregulation of various NF-kappaB-controlled genes\",\"including endothelial tissue factor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=endothelial tissue factor---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=endothelial tissue factor\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As a consequence, increased cellular oxidative stress led to the activation of the transcription factor NF-kappaB and thus promoted the upregulation of various NF-kappaB-controlled genes, including endothelial tissue factor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 419 (char 418)\n",
      "{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"AHR\"],\"tf text\":[\"ERalpha\"],\"tg text\":[\"AhR\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Our data suggest that ERalpha acts as a coregulator of AhR-mediated transcriptional activation and that the recruitment of ERalpha by AhR represents a novel mechanism AhR-ERalpha cross talk.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"AHR\"],\"tf text\":[\"ERalpha\"],\"tg text\":[\"AhR\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our data suggest that ERalpha acts as a coregulator of AhR-mediated transcriptional activation and that the recruitment of ERalpha by AhR represents a novel mechanism AhR-ERalpha cross talk.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"AHR\"],\"tf text\":[\"ERalpha\"],\"tg text\":[\"AhR\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Our data suggest that ERalpha acts as a coregulator of AhR-mediated transcriptional activation and that the recruitment of ERalpha by AhR represents a novel mechanism AhR-ERalpha cross talk.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"AHR\"],\"tf text\":[\"ERalpha\"],\"tg text\":[\"AhR\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our data suggest that ERalpha acts as a coregulator of AhR-mediated transcriptional activation and that the recruitment of ERalpha by AhR represents a novel mechanism AhR-ERalpha cross talk.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not identified correctly---{\"tf associated gene name\":[\"ESRRA\"],\"tg associated gene name\":[\"POMC\"],\"tf text\":[],\"tg text\":[\"POMC\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not identified correctly\"],\"sentence\":[\"We have previously shown that corticotropin-releasing hormone action is also exerted at the POMC promoter through the orphan nuclear receptor NGFI-B and its recruitment of SRC co-activators.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF not identified correctly---{\"tf associated gene name\":[\"ESRRA\"],\"tg associated gene name\":[\"POMC\"],\"tf text\":[],\"tg text\":[\"POMC\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not identified correctly\"],\"sentence\":[\"We have previously shown that corticotropin-releasing hormone action is also exerted at the POMC promoter through the orphan nuclear receptor NGFI-B and its recruitment of SRC co-activators.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESRRA\"],\"target gene (associated gene name)\":[\"POMC\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|ESRRA?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESRRA\"],\"tg (associated gene name)\":[\"POMC\"],\"tf text\":[],\"tg text\":[\"POMC\"],\"consensus\":[\"N\"],\"martin_comments\":[\"ESRRA?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We have previously shown that corticotropin-releasing hormone action is also exerted at the POMC promoter through the orphan nuclear receptor NGFI-B and its recruitment of SRC co-activators.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"FERD3L\"],\"tg associated gene name\":[\"ASCL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"We show that N-Twist inhibits MASH1-dependent transcriptional activation by sequestering E protein in a dominant negative fashion.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FERD3L\"],\"tg (associated gene name)\":[\"ASCL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that N-Twist inhibits MASH1-dependent transcriptional activation by sequestering E protein in a dominant negative fashion.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named---{\"tf associated gene name\":[\"FERD3L\"],\"tg associated gene name\":[\"ASCL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"We show that N-Twist inhibits MASH1-dependent transcriptional activation by sequestering E protein in a dominant negative fashion.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FERD3L\"],\"tg (associated gene name)\":[\"ASCL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that N-Twist inhibits MASH1-dependent transcriptional activation by sequestering E protein in a dominant negative fashion.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Na|K-ATPase beta 1---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"B3GNT9\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Na\",\"K-ATPase beta 1\"],\"sentence\":[\"Up-regulation of Na\",\"K-ATPase beta 1 transcription by hyperoxia is mediated by SP1/SP3 binding.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG = Na,K-ATPase beta 1 ---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"B3GNT9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG = Na,K-ATPase beta 1 \"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Up-regulation of Na,K-ATPase beta 1 transcription by hyperoxia is mediated by SP1/SP3 binding.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Na|K-ATPase beta 1---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"B3GNT9\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Na\",\"K-ATPase beta 1\"],\"sentence\":[\"Up-regulation of Na\",\"K-ATPase beta 1 transcription by hyperoxia is mediated by SP1/SP3 binding.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG = Na,K-ATPase beta 1 ---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"B3GNT9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG = Na,K-ATPase beta 1 \"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Up-regulation of Na,K-ATPase beta 1 transcription by hyperoxia is mediated by SP1/SP3 binding.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TF|IRF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF8\"],\"tg (associated gene name)\":[\"IRF8\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[\"IRF\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Deletion analysis of the ICSBP promoter identified regions that were IFN-gamma responsive, regions that mediate the ability of LPS and IFN-gamma to activate this promoter synergistically, as well as regions that normally repress ICSBP transcription.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TF|IRF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF8\"],\"tg (associated gene name)\":[\"IRF8\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[\"IRF\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Deletion analysis of the ICSBP promoter identified regions that were IFN-gamma responsive, regions that mediate the ability of LPS and IFN-gamma to activate this promoter synergistically, as well as regions that normally repress ICSBP transcription.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF=STAT5|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT4\"],\"tg (associated gene name)\":[\"CISH\"],\"tf text\":[\"Stat4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=STAT5\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Importantly, the expression of suppressor of cytokine signaling (SOCS)3, a potent inhibitor of IL-12-induced Stat4 activation, was decreased in Stat5a(-/-) CD4(+) T cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF=STAT5|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT4\"],\"tg (associated gene name)\":[\"CISH\"],\"tf text\":[\"Stat4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=STAT5\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Importantly, the expression of suppressor of cytokine signaling (SOCS)3, a potent inhibitor of IL-12-induced Stat4 activation, was decreased in Stat5a(-/-) CD4(+) T cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 369 (char 368)\n",
      "{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"EGF\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In contrast\",\"the toxin does not prevent the transient\",\"augmented expression of c-myc and c-fos mRNA seen in response to EGF in both cell types.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"EGF\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast, the toxin does not prevent the transient, augmented expression of c-myc and c-fos mRNA seen in response to EGF in both cell types.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"EGF\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In contrast\",\"the toxin does not prevent the transient\",\"augmented expression of c-myc and c-fos mRNA seen in response to EGF in both cell types.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"EGF\"],\"tf text\":[\"c-fos\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In contrast, the toxin does not prevent the transient, augmented expression of c-myc and c-fos mRNA seen in response to EGF in both cell types.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is PC---{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"PPARG\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is PC\"],\"sentence\":[\"Our findings identified PC as a PPARgamma-regulated gene and suggested a role for PPARgamma regulating intermediary metabolism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=PC?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"PPARG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=PC?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our findings identified PC as a PPARgamma-regulated gene and suggested a role for PPARgamma regulating intermediary metabolism.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is PC---{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"PPARG\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is PC\"],\"sentence\":[\"Our findings identified PC as a PPARgamma-regulated gene and suggested a role for PPARgamma regulating intermediary metabolism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=PC?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"PPARG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=PC?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our findings identified PC as a PPARgamma-regulated gene and suggested a role for PPARgamma regulating intermediary metabolism.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 443 (char 442)\n",
      "{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Together\",\"these results suggest that the interactions between AR\",\"Smad3\",\"and Smad4 may result in the differential regulation of the AR transactivation\",\"which further strengthens their roles in the prostate cancer progression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|hmmm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"hmmm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Together, these results suggest that the interactions between AR, Smad3, and Smad4 may result in the differential regulation of the AR transactivation, which further strengthens their roles in the prostate cancer progression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Together\",\"these results suggest that the interactions between AR\",\"Smad3\",\"and Smad4 may result in the differential regulation of the AR transactivation\",\"which further strengthens their roles in the prostate cancer progression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|hmmm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"hmmm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Together, these results suggest that the interactions between AR, Smad3, and Smad4 may result in the differential regulation of the AR transactivation, which further strengthens their roles in the prostate cancer progression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 524 (char 523)\n",
      "{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IL8\"],\"tf text\":[],\"tg text\":[\"IL-8\"],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Taken together\",\"these results suggest that PGN induces IL-8 expression in a CD14-enhanced manner in human alveolar type II epithelial cells\",\"through the formation of lipid rafts and the activation of MAP kinases\",\"which ultimately leads to activation of AP-1\",\"NF-kappaB\",\"and NF-IL6.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|maybe a bit indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL8\"],\"tf text\":[],\"tg text\":[\"IL-8\"],\"consensus\":[\"V\"],\"martin_comments\":[\"maybe a bit indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, these results suggest that PGN induces IL-8 expression in a CD14-enhanced manner in human alveolar type II epithelial cells, through the formation of lipid rafts and the activation of MAP kinases, which ultimately leads to activation of AP-1, NF-kappaB, and NF-IL6.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IL8\"],\"tf text\":[],\"tg text\":[\"IL-8\"],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Taken together\",\"these results suggest that PGN induces IL-8 expression in a CD14-enhanced manner in human alveolar type II epithelial cells\",\"through the formation of lipid rafts and the activation of MAP kinases\",\"which ultimately leads to activation of AP-1\",\"NF-kappaB\",\"and NF-IL6.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|maybe a bit indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL8\"],\"tf text\":[],\"tg text\":[\"IL-8\"],\"consensus\":[\"V\"],\"martin_comments\":[\"maybe a bit indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, these results suggest that PGN induces IL-8 expression in a CD14-enhanced manner in human alveolar type II epithelial cells, through the formation of lipid rafts and the activation of MAP kinases, which ultimately leads to activation of AP-1, NF-kappaB, and NF-IL6.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is NE---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"ELANE\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is NE\"],\"sentence\":[\"In NIH 3T3 cells\",\"the NE promoter was activated by c-Myb\",\"C/EBPalpha\",\"and PU.1\",\"via their respective binding sites.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is NE---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"ELANE\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is NE\"],\"sentence\":[\"In NIH 3T3 cells\",\"the NE promoter was activated by c-Myb\",\"C/EBPalpha\",\"and PU.1\",\"via their respective binding sites.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SPI1\"],\"target gene (associated gene name)\":[\"ELANE\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In NIH 3T3 cells\",\"the NE promoter was activated by c-Myb\",\"C/EBPalpha\",\"and PU.1\",\"via their respective binding sites.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"ELANE\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"In NIH 3T3 cells, the NE promoter was activated by c-Myb, C/EBPalpha, and PU.1, via their respective binding sites.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"PTGS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"no TF\"],\"sentence\":[\"Moreover\",\"a protein tyrosine kinase inhibitor herbimycin A suppressed both the expression of COX-2 mRNA and the luciferase activity induced by LPS.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TF---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"PTGS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"no TF\"],\"sentence\":[\"Moreover\",\"a protein tyrosine kinase inhibitor herbimycin A suppressed both the expression of COX-2 mRNA and the luciferase activity induced by LPS.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TXK\"],\"target gene (associated gene name)\":[\"PTGS2\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Moreover\",\"a protein tyrosine kinase inhibitor herbimycin A suppressed both the expression of COX-2 mRNA and the luciferase activity induced by LPS.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|TXK---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TXK\"],\"tg (associated gene name)\":[\"PTGS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[\"TXK\"],\"sentences with greek letters\":[\"Moreover, a protein tyrosine kinase inhibitor herbimycin A suppressed both the expression of COX-2 mRNA and the luciferase activity induced by LPS.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"ZFP42\"],\"tg associated gene name\":[\"BMP4\"],\"tf text\":[],\"tg text\":[\"Bmp4\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Cell proliferation assays and Northern blot analyses of a number of ES cell markers (e.g. Rex1\",\"Oct3/4\",\"Fgf4\",\"and Bmp4) suggest that the Hoxa1 protein plays a role in the inhibition of cell proliferation by RA in ES cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZFP42\"],\"tg (associated gene name)\":[\"BMP4\"],\"tf text\":[],\"tg text\":[\"Bmp4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Cell proliferation assays and Northern blot analyses of a number of ES cell markers (e.g. Rex1, Oct3/4, Fgf4, and Bmp4) suggest that the Hoxa1 protein plays a role in the inhibition of cell proliferation by RA in ES cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"ZFP42\"],\"tg associated gene name\":[\"BMP4\"],\"tf text\":[],\"tg text\":[\"Bmp4\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Cell proliferation assays and Northern blot analyses of a number of ES cell markers (e.g. Rex1\",\"Oct3/4\",\"Fgf4\",\"and Bmp4) suggest that the Hoxa1 protein plays a role in the inhibition of cell proliferation by RA in ES cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZFP42\"],\"tg (associated gene name)\":[\"BMP4\"],\"tf text\":[],\"tg text\":[\"Bmp4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Cell proliferation assays and Northern blot analyses of a number of ES cell markers (e.g. Rex1, Oct3/4, Fgf4, and Bmp4) suggest that the Hoxa1 protein plays a role in the inhibition of cell proliferation by RA in ES cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF mentioned---{\"tf associated gene name\":[\"PBX1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TF mentioned\"],\"sentence\":[\"Rather\",\"inefficient use of the PRL promoter in the variant cells may reflect an increased methylation state of the PRL gene itself (cis model).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PBX1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Rather, inefficient use of the PRL promoter in the variant cells may reflect an increased methylation state of the PRL gene itself (cis model).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF mentioned---{\"tf associated gene name\":[\"PBX1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TF mentioned\"],\"sentence\":[\"Rather\",\"inefficient use of the PRL promoter in the variant cells may reflect an increased methylation state of the PRL gene itself (cis model).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PBX1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Rather, inefficient use of the PRL promoter in the variant cells may reflect an increased methylation state of the PRL gene itself (cis model).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GATA4-regulated TG is TAL1---{\"tf associated gene name\":[\"GATA4\"],\"tg associated gene name\":[\"GATA4\"],\"tf text\":[\"GATA-4\"],\"tg text\":[\"GATA-4\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"GATA4-regulated TG is TAL1\"],\"sentence\":[\"Sirt1 was found to bind to the GATA-4 promoter to directly regulate GATA-4 expression and GATA-4 binds to the TAL1 promoter to regulate TAL1 expression positively.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no auto---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA4\"],\"tg (associated gene name)\":[\"GATA4\"],\"tf text\":[\"GATA-4\"],\"tg text\":[\"GATA-4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no auto\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sirt1 was found to bind to the GATA-4 promoter to directly regulate GATA-4 expression and GATA-4 binds to the TAL1 promoter to regulate TAL1 expression positively.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "GATA4-regulated TG is TAL1---{\"tf associated gene name\":[\"GATA4\"],\"tg associated gene name\":[\"GATA4\"],\"tf text\":[\"GATA-4\"],\"tg text\":[\"GATA-4\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"GATA4-regulated TG is TAL1\"],\"sentence\":[\"Sirt1 was found to bind to the GATA-4 promoter to directly regulate GATA-4 expression and GATA-4 binds to the TAL1 promoter to regulate TAL1 expression positively.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no auto---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA4\"],\"tg (associated gene name)\":[\"GATA4\"],\"tf text\":[\"GATA-4\"],\"tg text\":[\"GATA-4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no auto\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sirt1 was found to bind to the GATA-4 promoter to directly regulate GATA-4 expression and GATA-4 binds to the TAL1 promoter to regulate TAL1 expression positively.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "normalization of TF and TG uncertain - but within \"family\" for each of them---{\"tf associated gene name\":[\"SMARCA1\"],\"tg associated gene name\":[\"CSN1S2AP\"],\"tf text\":[\"SWI\"],\"tg text\":[],\"1\":[\"u\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"normalization of TF and TG uncertain - but within \\\"family\\\" for each of them\"],\"sentence\":[\"Using chromatin immunoprecipitation assays and mammary-specific genes as models\",\"we show here that extracellular matrix molecules and prolactin cooperate to induce histone acetylation and binding of transcription factors and the SWI/SNF complex to the beta- and gamma-casein promoters.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|not casein alpha?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMARCA1\"],\"tg (associated gene name)\":[\"CSN1S2AP\"],\"tf text\":[\"SWI\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not casein alpha?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using chromatin immunoprecipitation assays and mammary-specific genes as models, we show here that extracellular matrix molecules and prolactin cooperate to induce histone acetylation and binding of transcription factors and the SWI/SNF complex to the beta- and gamma-casein promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "normalization of TF and TG uncertain - but within \"family\" for each of them---{\"tf associated gene name\":[\"SMARCA1\"],\"tg associated gene name\":[\"CSN1S2AP\"],\"tf text\":[\"SWI\"],\"tg text\":[],\"1\":[\"u\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"normalization of TF and TG uncertain - but within \\\"family\\\" for each of them\"],\"sentence\":[\"Using chromatin immunoprecipitation assays and mammary-specific genes as models\",\"we show here that extracellular matrix molecules and prolactin cooperate to induce histone acetylation and binding of transcription factors and the SWI/SNF complex to the beta- and gamma-casein promoters.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|not casein alpha?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMARCA1\"],\"tg (associated gene name)\":[\"CSN1S2AP\"],\"tf text\":[\"SWI\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not casein alpha?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using chromatin immunoprecipitation assays and mammary-specific genes as models, we show here that extracellular matrix molecules and prolactin cooperate to induce histone acetylation and binding of transcription factors and the SWI/SNF complex to the beta- and gamma-casein promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped roles|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"RAC1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We conclude that 5-HT stimulates c-fos and cyclin D1 expression through a ROS-dependent mechanism that requires Ras, Rac1, and MAPK.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped roles|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"RAC1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We conclude that 5-HT stimulates c-fos and cyclin D1 expression through a ROS-dependent mechanism that requires Ras, Rac1, and MAPK.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is SP-A (not FASL)|several possible TFs|but not NCOA1|I think---{\"tf associated gene name\":[\"NCOA1\"],\"tg associated gene name\":[\"FASLG\"],\"tf text\":[\"SRC-1\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is SP-A (not FASL)\",\"several possible TFs\",\"but not NCOA1\",\"I think\"],\"sentence\":[\"Adenoviral E1A overexpression reduced TTF-1 +/- SRC-1 induction of SP-A promoter activity\",\"suggesting a role of endogenous CBP/p300.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SP-A = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA1\"],\"tg (associated gene name)\":[\"FASLG\"],\"tf text\":[\"SRC-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SP-A = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Adenoviral E1A overexpression reduced TTF-1 +/- SRC-1 induction of SP-A promoter activity, suggesting a role of endogenous CBP/p300.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is SP-A (not FASL)|several possible TFs|but not NCOA1|I think---{\"tf associated gene name\":[\"NCOA1\"],\"tg associated gene name\":[\"FASLG\"],\"tf text\":[\"SRC-1\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is SP-A (not FASL)\",\"several possible TFs\",\"but not NCOA1\",\"I think\"],\"sentence\":[\"Adenoviral E1A overexpression reduced TTF-1 +/- SRC-1 induction of SP-A promoter activity\",\"suggesting a role of endogenous CBP/p300.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SP-A = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA1\"],\"tg (associated gene name)\":[\"FASLG\"],\"tf text\":[\"SRC-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SP-A = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Adenoviral E1A overexpression reduced TTF-1 +/- SRC-1 induction of SP-A promoter activity, suggesting a role of endogenous CBP/p300.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 566 (char 565)\n",
      "{\"tf associated gene name\":[\"PAX6\"],\"tg associated gene name\":[\"HES1\"],\"tf text\":[\"aN2\"],\"tg text\":[\"HES1\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[],\"sentence\":[\"To clarify functional diversities among the Notch receptors\",\"we generated truncated forms of Notch1\",\"Notch2\",\"and Notch3 comprising the intracellular domain (aN1\",\"aN2\",\"and aN3) and investigated their transcriptional activities for HES1 and HES5 promoters driving the luciferase reporter gene (HES1-Luc and HES5-Luc).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong use of aN2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX6\"],\"tg (associated gene name)\":[\"HES1\"],\"tf text\":[\"aN2\"],\"tg text\":[\"HES1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong use of aN2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To clarify functional diversities among the Notch receptors, we generated truncated forms of Notch1, Notch2, and Notch3 comprising the intracellular domain (aN1, aN2, and aN3) and investigated their transcriptional activities for HES1 and HES5 promoters driving the luciferase reporter gene (HES1-Luc and HES5-Luc).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PAX6\"],\"tg associated gene name\":[\"HES1\"],\"tf text\":[\"aN2\"],\"tg text\":[\"HES1\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[],\"sentence\":[\"To clarify functional diversities among the Notch receptors\",\"we generated truncated forms of Notch1\",\"Notch2\",\"and Notch3 comprising the intracellular domain (aN1\",\"aN2\",\"and aN3) and investigated their transcriptional activities for HES1 and HES5 promoters driving the luciferase reporter gene (HES1-Luc and HES5-Luc).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong use of aN2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX6\"],\"tg (associated gene name)\":[\"HES1\"],\"tf text\":[\"aN2\"],\"tg text\":[\"HES1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong use of aN2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To clarify functional diversities among the Notch receptors, we generated truncated forms of Notch1, Notch2, and Notch3 comprising the intracellular domain (aN1, aN2, and aN3) and investigated their transcriptional activities for HES1 and HES5 promoters driving the luciferase reporter gene (HES1-Luc and HES5-Luc).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 471 (char 470)\n",
      "{\"tf associated gene name\":[\"EGR3\"],\"tg associated gene name\":[\"RASD1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Twenty upregulated genes\",\"including early stress response and transcription factor genes\",\"were identified\",\"most of which\",\"e.g.\",\"EGR1\",\"EGR3\",\"SNAI1\",\"RASD1 and GADD45B\",\"were also induced by hypertonicity\",\"indicating common regulatory mechanisms.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR3\"],\"tg (associated gene name)\":[\"RASD1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Twenty upregulated genes, including early stress response and transcription factor genes, were identified, most of which, e.g., EGR1, EGR3, SNAI1, RASD1 and GADD45B, were also induced by hypertonicity, indicating common regulatory mechanisms.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"EGR3\"],\"tg associated gene name\":[\"RASD1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Twenty upregulated genes\",\"including early stress response and transcription factor genes\",\"were identified\",\"most of which\",\"e.g.\",\"EGR1\",\"EGR3\",\"SNAI1\",\"RASD1 and GADD45B\",\"were also induced by hypertonicity\",\"indicating common regulatory mechanisms.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR3\"],\"tg (associated gene name)\":[\"RASD1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Twenty upregulated genes, including early stress response and transcription factor genes, were identified, most of which, e.g., EGR1, EGR3, SNAI1, RASD1 and GADD45B, were also induced by hypertonicity, indicating common regulatory mechanisms.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is GABP alpha / beta---{\"tf associated gene name\":[\"NRG1\"],\"tg associated gene name\":[\"UTRN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is GABP alpha / beta\"],\"sentence\":[\"Utrophin's complex transcriptional regulation is not yet fully understood\",\"however\",\"GABP alpha / beta has recently been shown to bind the N box and activate the utrophin promoter in response to heregulin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|heregulin k?|discuss?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRG1\"],\"tg (associated gene name)\":[\"UTRN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"heregulin k?\"],\"astrid_comments\":[\"discuss?\"],\"sentences with greek letters\":[\"Utrophin's complex transcriptional regulation is not yet fully understood, however, GABP alpha / beta has recently been shown to bind the N box and activate the utrophin promoter in response to heregulin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is GABP alpha / beta---{\"tf associated gene name\":[\"NRG1\"],\"tg associated gene name\":[\"UTRN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is GABP alpha / beta\"],\"sentence\":[\"Utrophin's complex transcriptional regulation is not yet fully understood\",\"however\",\"GABP alpha / beta has recently been shown to bind the N box and activate the utrophin promoter in response to heregulin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|heregulin k?|discuss?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRG1\"],\"tg (associated gene name)\":[\"UTRN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"heregulin k?\"],\"astrid_comments\":[\"discuss?\"],\"sentences with greek letters\":[\"Utrophin's complex transcriptional regulation is not yet fully understood, however, GABP alpha / beta has recently been shown to bind the N box and activate the utrophin promoter in response to heregulin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 538 (char 537)\n",
      "{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"PPARA\"],\"tf text\":[\"PPARalpha\"],\"tg text\":[\"PPARalpha\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"PPARalpha(mut) was transcriptionally inactive\",\"repressed basal activity from a PPAR response element-containing promoter\",\"inhibited the coactivator function of cotransfected PPAR-gamma coactivator 1alpha\",\"and strongly inhibited the transcriptional response to cotransfected wild-type receptor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|really autoreg?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"PPARA\"],\"tf text\":[\"PPARalpha\"],\"tg text\":[\"PPARalpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"really autoreg?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PPARalpha(mut) was transcriptionally inactive, repressed basal activity from a PPAR response element-containing promoter, inhibited the coactivator function of cotransfected PPAR-gamma coactivator 1alpha, and strongly inhibited the transcriptional response to cotransfected wild-type receptor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"PPARA\"],\"tf text\":[\"PPARalpha\"],\"tg text\":[\"PPARalpha\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"PPARalpha(mut) was transcriptionally inactive\",\"repressed basal activity from a PPAR response element-containing promoter\",\"inhibited the coactivator function of cotransfected PPAR-gamma coactivator 1alpha\",\"and strongly inhibited the transcriptional response to cotransfected wild-type receptor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|really autoreg?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"PPARA\"],\"tf text\":[\"PPARalpha\"],\"tg text\":[\"PPARalpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"really autoreg?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PPARalpha(mut) was transcriptionally inactive, repressed basal activity from a PPAR response element-containing promoter, inhibited the coactivator function of cotransfected PPAR-gamma coactivator 1alpha, and strongly inhibited the transcriptional response to cotransfected wild-type receptor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is KiSS1---{\"tf associated gene name\":[\"TTF1\"],\"tg associated gene name\":[\"DAXX\"],\"tf text\":[\"TTF1\"],\"tg text\":[\"EAP1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is KiSS1\"],\"sentence\":[\"Promoter assays performed using 293 MSR cells showed that the KiSS1 promoter is activated by TTF1 and CUX1-p200\",\"and repressed by EAP1\",\"YY1\",\"and CUX1-p110.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG not EAP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TTF1\"],\"tg (associated gene name)\":[\"DAXX\"],\"tf text\":[\"TTF1\"],\"tg text\":[\"EAP1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG not EAP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Promoter assays performed using 293 MSR cells showed that the KiSS1 promoter is activated by TTF1 and CUX1-p200, and repressed by EAP1, YY1, and CUX1-p110.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is KiSS1---{\"tf associated gene name\":[\"TTF1\"],\"tg associated gene name\":[\"DAXX\"],\"tf text\":[\"TTF1\"],\"tg text\":[\"EAP1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is KiSS1\"],\"sentence\":[\"Promoter assays performed using 293 MSR cells showed that the KiSS1 promoter is activated by TTF1 and CUX1-p200\",\"and repressed by EAP1\",\"YY1\",\"and CUX1-p110.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG not EAP1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TTF1\"],\"tg (associated gene name)\":[\"DAXX\"],\"tf text\":[\"TTF1\"],\"tg text\":[\"EAP1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG not EAP1\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Promoter assays performed using 293 MSR cells showed that the KiSS1 promoter is activated by TTF1 and CUX1-p200, and repressed by EAP1, YY1, and CUX1-p110.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 655 (char 654)\n",
      "{\"transcription factor (associated gene name)\":[\"FOS\"],\"target gene (associated gene name)\":[\"DACH1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"c-jun and c-fos were transcriptionally repressed by DACH1\",\"requiring the conserved N-terminal (dac and ski/sno [DS]) domain.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"0.988826\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"124\"],\"sentence pair density\":[\"0.0126582278481012\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles|REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"DACH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"c-jun and c-fos were transcriptionally repressed by DACH1, requiring the conserved N-terminal (dac and ski/sno [DS]) domain.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"FOS\"],\"target gene (associated gene name)\":[\"DACH1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"c-jun and c-fos were transcriptionally repressed by DACH1\",\"requiring the conserved N-terminal (dac and ski/sno [DS]) domain.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"0.988826\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"124\"],\"sentence pair density\":[\"0.0126582278481012\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles|REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"DACH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"c-jun and c-fos were transcriptionally repressed by DACH1, requiring the conserved N-terminal (dac and ski/sno [DS]) domain.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "bacteriophage---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"PGR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"s\"],\"negation\":[],\"comment\":[\"bacteriophage\"],\"sentence\":[\"RNA polymerase bound to the PR promoter of bacteriophage lambda inhibits open complex formation at the divergently transcribed PRM promoter. Implications for an indirect mechanism of transcriptional activation by lambda repressor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|lambda?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"PGR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"lambda?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"RNA polymerase bound to the PR promoter of bacteriophage lambda inhibits open complex formation at the divergently transcribed PRM promoter. Implications for an indirect mechanism of transcriptional activation by lambda repressor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "bacteriophage---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"PGR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"s\"],\"negation\":[],\"comment\":[\"bacteriophage\"],\"sentence\":[\"RNA polymerase bound to the PR promoter of bacteriophage lambda inhibits open complex formation at the divergently transcribed PRM promoter. Implications for an indirect mechanism of transcriptional activation by lambda repressor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|lambda?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"PGR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"lambda?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"RNA polymerase bound to the PR promoter of bacteriophage lambda inhibits open complex formation at the divergently transcribed PRM promoter. Implications for an indirect mechanism of transcriptional activation by lambda repressor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "EIF3K-problem; no TG named---{\"tf associated gene name\":[\"TEAD1\"],\"tg associated gene name\":[\"EIF3K\"],\"tf text\":[\"TEF-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"EIF3K-problem; no TG named\"],\"sentence\":[\"Our results suggest that by their interaction with MEF2 factors\",\"TEF-1 factors can control MEF2-dependent muscle-specific gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEAD1\"],\"tg (associated gene name)\":[\"EIF3K\"],\"tf text\":[\"TEF-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that by their interaction with MEF2 factors, TEF-1 factors can control MEF2-dependent muscle-specific gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "EIF3K-problem; no TG named---{\"tf associated gene name\":[\"TEAD1\"],\"tg associated gene name\":[\"EIF3K\"],\"tf text\":[\"TEF-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"EIF3K-problem; no TG named\"],\"sentence\":[\"Our results suggest that by their interaction with MEF2 factors\",\"TEF-1 factors can control MEF2-dependent muscle-specific gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEAD1\"],\"tg (associated gene name)\":[\"EIF3K\"],\"tf text\":[\"TEF-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that by their interaction with MEF2 factors, TEF-1 factors can control MEF2-dependent muscle-specific gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 379 (char 378)\n",
      "{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Moreover\",\"the MEKK1 dominant negative mutant as well as deletion of the AP-1 binding sites within the c-jun promoter inhibited the c-jun promoter activation by hypoxia.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, the MEKK1 dominant negative mutant as well as deletion of the AP-1 binding sites within the c-jun promoter inhibited the c-jun promoter activation by hypoxia.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Moreover\",\"the MEKK1 dominant negative mutant as well as deletion of the AP-1 binding sites within the c-jun promoter inhibited the c-jun promoter activation by hypoxia.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, the MEKK1 dominant negative mutant as well as deletion of the AP-1 binding sites within the c-jun promoter inhibited the c-jun promoter activation by hypoxia.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not obvious GRE/TRE---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"FAS\"],\"tf text\":[\"p53\"],\"tg text\":[\"CD 95\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not obvious GRE/TRE\"],\"sentence\":[\"Alterations of Fas (APO-1/CD 95) gene and its relationship with p53 in non small cell lung cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not obvious GRE/TRE---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"FAS\"],\"tf text\":[\"p53\"],\"tg text\":[\"CD 95\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not obvious GRE/TRE\"],\"sentence\":[\"Alterations of Fas (APO-1/CD 95) gene and its relationship with p53 in non small cell lung cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"FAS\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|unspecified relation---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"FAS\"],\"tf text\":[\"p53\"],\"tg text\":[\"CD 95\"],\"consensus\":[\"N\"],\"martin_comments\":[\"unspecified relation\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Alterations of Fas (APO-1/CD 95) gene and its relationship with p53 in non small cell lung cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is SRC-2|likely signalling protein---{\"tf associated gene name\":[\"NCOA2\"],\"tg associated gene name\":[\"PPARA\"],\"tf text\":[\"SRC-2\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"regulator is SRC-2\",\"likely signalling protein\"],\"sentence\":[\"Additionally\",\"we describe SRC-2 as a novel regulator of PPAR?? expression\",\"thus controlling critical steps in metabolic gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "regulator is SRC-2|likely signalling protein---{\"tf associated gene name\":[\"NCOA2\"],\"tg associated gene name\":[\"PPARA\"],\"tf text\":[\"SRC-2\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"regulator is SRC-2\",\"likely signalling protein\"],\"sentence\":[\"Additionally\",\"we describe SRC-2 as a novel regulator of PPAR?? expression\",\"thus controlling critical steps in metabolic gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NCOA2\"],\"target gene (associated gene name)\":[\"PPARA\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Additionally\",\"we describe SRC-2 as a novel regulator of PPAR expression\",\"thus controlling critical steps in metabolic gene expression.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA2\"],\"tg (associated gene name)\":[\"PPARA\"],\"tf text\":[\"SRC-2\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Additionally, we describe SRC-2 as a novel regulator of PPAR expression, thus controlling critical steps in metabolic gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HOXA10\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Cotransfection of cells with Tat/Jun and the AP-1 PL LUC or LTR AP-1 CAT reporter plasmid resulted in a marked increase in reporter gene activity which was comparable with that induced by transfection of cells with several different AP-1 expression plasmids (e.g., JunD, JunB, c-Fos), or that elicited by stimulation of the cells transfected with LTR AP-1 CAT plasmids with phorbol ester or tumor necrosis factor-alpha.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HOXA10\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Cotransfection of cells with Tat/Jun and the AP-1 PL LUC or LTR AP-1 CAT reporter plasmid resulted in a marked increase in reporter gene activity which was comparable with that induced by transfection of cells with several different AP-1 expression plasmids (e.g., JunD, JunB, c-Fos), or that elicited by stimulation of the cells transfected with LTR AP-1 CAT plasmids with phorbol ester or tumor necrosis factor-alpha.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 368 (char 367)\n",
      "{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"PCYT1A\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that Sp1 binding to the CT alpha proximal promoter is necessary to enhance transcription during the S phase.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|CT alpha close enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"PCYT1A\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"CT alpha\"],\"consensus\":[\"V\"],\"martin_comments\":[\"CT alpha close enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We conclude that Sp1 binding to the CT alpha proximal promoter is necessary to enhance transcription during the S phase.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"PCYT1A\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that Sp1 binding to the CT alpha proximal promoter is necessary to enhance transcription during the S phase.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|CT alpha close enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"PCYT1A\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"CT alpha\"],\"consensus\":[\"V\"],\"martin_comments\":[\"CT alpha close enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We conclude that Sp1 binding to the CT alpha proximal promoter is necessary to enhance transcription during the S phase.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is DNA topoisomerase IIalpha---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is DNA topoisomerase IIalpha\"],\"sentence\":[\"Sp1 mediates cell proliferation-dependent regulation of rat DNA topoisomerase IIalpha gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG name---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG name\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sp1 mediates cell proliferation-dependent regulation of rat DNA topoisomerase IIalpha gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is DNA topoisomerase IIalpha---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is DNA topoisomerase IIalpha\"],\"sentence\":[\"Sp1 mediates cell proliferation-dependent regulation of rat DNA topoisomerase IIalpha gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG name---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG name\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sp1 mediates cell proliferation-dependent regulation of rat DNA topoisomerase IIalpha gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"MYOD1\"],\"tg associated gene name\":[\"ACHE\"],\"tf text\":[],\"tg text\":[\"AChE\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"We conclude that PUM2 binds to AChE mRNA and regulates AChE expression translationally at the neuromuscular synapse.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"MYOD1\"],\"tg associated gene name\":[\"ACHE\"],\"tf text\":[],\"tg text\":[\"AChE\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"We conclude that PUM2 binds to AChE mRNA and regulates AChE expression translationally at the neuromuscular synapse.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MYOD1\"],\"target gene (associated gene name)\":[\"ACHE\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that PUM2 binds to AChE mRNA and regulates AChE expression translationally at the neuromuscular synapse.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYOD1\"],\"tg (associated gene name)\":[\"ACHE\"],\"tf text\":[],\"tg text\":[\"AChE\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"We conclude that PUM2 binds to AChE mRNA and regulates AChE expression translationally at the neuromuscular synapse.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not identifyable---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"BIRC5\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF not identifyable\"],\"sentence\":[\"When the MAR fragment was inserted into the upstream of the survivin promoter\",\"the qPCR result showed that the expression of the CD/TK genes significantly increased 7.7-fold in the transgenic SGC-7901 cells with plasmid pMS-CD/TK compared with that without MAR.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF not identifyable---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"BIRC5\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF not identifyable\"],\"sentence\":[\"When the MAR fragment was inserted into the upstream of the survivin promoter\",\"the qPCR result showed that the expression of the CD/TK genes significantly increased 7.7-fold in the transgenic SGC-7901 cells with plasmid pMS-CD/TK compared with that without MAR.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF1\"],\"target gene (associated gene name)\":[\"BIRC5\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"When the MAR fragment was inserted into the upstream of the survivin promoter\",\"the qPCR result showed that the expression of the CD/TK genes significantly increased 7.7-fold in the transgenic SGC-7901 cells with plasmid pMS-CD/TK compared with that without MAR.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hmmmm difficult---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"BIRC5\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"hmmmm difficult\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"When the MAR fragment was inserted into the upstream of the survivin promoter, the qPCR result showed that the expression of the CD/TK genes significantly increased 7.7-fold in the transgenic SGC-7901 cells with plasmid pMS-CD/TK compared with that without MAR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"RNF2\"],\"tg associated gene name\":[\"RPS28\"],\"tf text\":[\"Ring1B\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Aurora B phosphorylates histone H3S28 at active promoters in resting B cells as well as inhibiting Ring1B-mediated ubiquitination of histone H2A and enhancing binding and activity of the USP16 deubiquitinase at transcribed genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RNF2\"],\"tg (associated gene name)\":[\"RPS28\"],\"tf text\":[\"Ring1B\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Aurora B phosphorylates histone H3S28 at active promoters in resting B cells as well as inhibiting Ring1B-mediated ubiquitination of histone H2A and enhancing binding and activity of the USP16 deubiquitinase at transcribed genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named---{\"tf associated gene name\":[\"RNF2\"],\"tg associated gene name\":[\"RPS28\"],\"tf text\":[\"Ring1B\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Aurora B phosphorylates histone H3S28 at active promoters in resting B cells as well as inhibiting Ring1B-mediated ubiquitination of histone H2A and enhancing binding and activity of the USP16 deubiquitinase at transcribed genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RNF2\"],\"tg (associated gene name)\":[\"RPS28\"],\"tf text\":[\"Ring1B\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Aurora B phosphorylates histone H3S28 at active promoters in resting B cells as well as inhibiting Ring1B-mediated ubiquitination of histone H2A and enhancing binding and activity of the USP16 deubiquitinase at transcribed genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs are E2F1-2-3|regulator is Rb---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"RB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TGs are E2F1-2-3\",\"regulator is Rb\"],\"sentence\":[\"In this report\",\"we show that both pRb and E2F1-2-3 act as positive modulators of detoxification pathways in mice\",\"challenging the conventional view of E2F1-2-3 as transcriptional repressors negatively regulated by pRb.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"RB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this report, we show that both pRb and E2F1-2-3 act as positive modulators of detoxification pathways in mice, challenging the conventional view of E2F1-2-3 as transcriptional repressors negatively regulated by pRb.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TGs are E2F1-2-3|regulator is Rb---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"RB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TGs are E2F1-2-3\",\"regulator is Rb\"],\"sentence\":[\"In this report\",\"we show that both pRb and E2F1-2-3 act as positive modulators of detoxification pathways in mice\",\"challenging the conventional view of E2F1-2-3 as transcriptional repressors negatively regulated by pRb.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F1\"],\"tg (associated gene name)\":[\"RB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this report, we show that both pRb and E2F1-2-3 act as positive modulators of detoxification pathways in mice, challenging the conventional view of E2F1-2-3 as transcriptional repressors negatively regulated by pRb.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "mTOR not mentioned|I think---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"MTOR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"mTOR not mentioned\",\"I think\"],\"sentence\":[\"Transcriptional activation by PDX-1 is increased by addition of Bridge-1 in multiple contexts\",\"including synergistic activation of a Gal4 reporter by Gal4-Bridge-1 and Gal4-PDX-1 fusion proteins\",\"activation of the somatostatin promoter TAAT1 enhancer\",\"and synergistic activation of the rat insulin I promoter FarFlat enhancer by PDX-1\",\"E12\",\"and E47.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "mTOR not mentioned|I think---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"MTOR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"mTOR not mentioned\",\"I think\"],\"sentence\":[\"Transcriptional activation by PDX-1 is increased by addition of Bridge-1 in multiple contexts\",\"including synergistic activation of a Gal4 reporter by Gal4-Bridge-1 and Gal4-PDX-1 fusion proteins\",\"activation of the somatostatin promoter TAAT1 enhancer\",\"and synergistic activation of the rat insulin I promoter FarFlat enhancer by PDX-1\",\"E12\",\"and E47.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"MTOR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional activation by PDX-1 is increased by addition of Bridge-1 in multiple contexts, including synergistic activation of a Gal4 reporter by Gal4-Bridge-1 and Gal4-PDX-1 fusion proteins, activation of the somatostatin promoter TAAT1 enhancer, and synergistic activation of the rat insulin I promoter FarFlat enhancer by PDX-1, E12, and E47.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"tf associated gene name\":[\"TAF1\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[\"TAF(II)250\",\"and polymerase II\"],\"tg text\":[\"CBP\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"no TG\"],\"sentence\":[\"Unexpectedly\",\"we detected a nearly equal distribution of activator (Tax\",\"CREB\",\"ATF-1\",\"ATF-2\",\"c-Fos\",\"and c-Jun) and regulatory protein (CBP\",\"p300\",\"TAF(II)250\",\"and polymerase II) binding at both the upstream and downstream promoters.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TAF1\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"TAF(II)250\",\"and polymerase II\"],\"tg text\":[\"CBP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Unexpectedly, we detected a nearly equal distribution of activator (Tax, CREB, ATF-1, ATF-2, c-Fos, and c-Jun) and regulatory protein (CBP, p300, TAF(II)250, and polymerase II) binding at both the upstream and downstream promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"tf associated gene name\":[\"TAF1\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[\"TAF(II)250\",\"and polymerase II\"],\"tg text\":[\"CBP\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"no TG\"],\"sentence\":[\"Unexpectedly\",\"we detected a nearly equal distribution of activator (Tax\",\"CREB\",\"ATF-1\",\"ATF-2\",\"c-Fos\",\"and c-Jun) and regulatory protein (CBP\",\"p300\",\"TAF(II)250\",\"and polymerase II) binding at both the upstream and downstream promoters.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TAF1\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"TAF(II)250\",\"and polymerase II\"],\"tg text\":[\"CBP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Unexpectedly, we detected a nearly equal distribution of activator (Tax, CREB, ATF-1, ATF-2, c-Fos, and c-Jun) and regulatory protein (CBP, p300, TAF(II)250, and polymerase II) binding at both the upstream and downstream promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 401 (char 400)\n",
      "{\"tf associated gene name\":[\"AHR\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Immunoprecipitation revealed that AHR directly bound to p300\",\"thus suggesting the intriguing possibility that AHR is involved in control of the cell cycle via interaction with p300.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG, protein?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AHR\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG, protein?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Immunoprecipitation revealed that AHR directly bound to p300, thus suggesting the intriguing possibility that AHR is involved in control of the cell cycle via interaction with p300.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AHR\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Immunoprecipitation revealed that AHR directly bound to p300\",\"thus suggesting the intriguing possibility that AHR is involved in control of the cell cycle via interaction with p300.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG, protein?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AHR\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG, protein?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Immunoprecipitation revealed that AHR directly bound to p300, thus suggesting the intriguing possibility that AHR is involved in control of the cell cycle via interaction with p300.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs not named---{\"tf associated gene name\":[\"TEF\"],\"tg associated gene name\":[\"EIF3K\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TGs not named\"],\"sentence\":[\"MCAT elements and its cognate binding partners\",\"the transcription enhancer factors (TEFs) play important roles in the regulation of expression of several muscle-specific genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|muscle-specific enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEF\"],\"tg (associated gene name)\":[\"EIF3K\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"muscle-specific enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MCAT elements and its cognate binding partners, the transcription enhancer factors (TEFs) play important roles in the regulation of expression of several muscle-specific genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TGs not named---{\"tf associated gene name\":[\"TEF\"],\"tg associated gene name\":[\"EIF3K\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TGs not named\"],\"sentence\":[\"MCAT elements and its cognate binding partners\",\"the transcription enhancer factors (TEFs) play important roles in the regulation of expression of several muscle-specific genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|muscle-specific enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TEF\"],\"tg (associated gene name)\":[\"EIF3K\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"muscle-specific enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MCAT elements and its cognate binding partners, the transcription enhancer factors (TEFs) play important roles in the regulation of expression of several muscle-specific genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not clear TRE---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"EPO\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not clear TRE\"],\"sentence\":[\"However\",\"unlike Epo-deficient embryos that die from anemia\",\"the erythropoietic deficiency in RXR alpha(-/-) embryos is transient; Epo mRNA is expressed at normal levels by E12.5\",\"and erythropoiesis and liver morphology are normal by E14.5.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TCF3?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"EPO\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TCF3?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, unlike Epo-deficient embryos that die from anemia, the erythropoietic deficiency in RXR alpha(-/-) embryos is transient; Epo mRNA is expressed at normal levels by E12.5, and erythropoiesis and liver morphology are normal by E14.5.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not clear TRE---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"EPO\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not clear TRE\"],\"sentence\":[\"However\",\"unlike Epo-deficient embryos that die from anemia\",\"the erythropoietic deficiency in RXR alpha(-/-) embryos is transient; Epo mRNA is expressed at normal levels by E12.5\",\"and erythropoiesis and liver morphology are normal by E14.5.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TCF3?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"EPO\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TCF3?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, unlike Epo-deficient embryos that die from anemia, the erythropoietic deficiency in RXR alpha(-/-) embryos is transient; Epo mRNA is expressed at normal levels by E12.5, and erythropoiesis and liver morphology are normal by E14.5.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 473 (char 472)\n",
      "{\"tf associated gene name\":[\"RORC\"],\"tg associated gene name\":[\"RPS6KB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"With increasing concentration of cucurmosin\",\"the expression of EGFR\",\"p-PI3K\",\"Akt\",\"p-Akt\",\"mTOR\",\"p-mTOR\",\"P70S6K-??\",\"p-P70S6K-??\",\"4E-BP1 and p-4E-BP1 at the protein level was decreased\",\"whereas the expression of p-Bad and caspase-9 was elevated.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|???---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RORC\"],\"tg (associated gene name)\":[\"RPS6KB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"???\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"With increasing concentration of cucurmosin, the expression of EGFR, p-PI3K, Akt, p-Akt, mTOR, p-mTOR, P70S6K-, p-P70S6K-, 4E-BP1 and p-4E-BP1 at the protein level was decreased, whereas the expression of p-Bad and caspase-9 was elevated.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RORC\"],\"tg associated gene name\":[\"RPS6KB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"With increasing concentration of cucurmosin\",\"the expression of EGFR\",\"p-PI3K\",\"Akt\",\"p-Akt\",\"mTOR\",\"p-mTOR\",\"P70S6K-??\",\"p-P70S6K-??\",\"4E-BP1 and p-4E-BP1 at the protein level was decreased\",\"whereas the expression of p-Bad and caspase-9 was elevated.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|???---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RORC\"],\"tg (associated gene name)\":[\"RPS6KB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"???\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"With increasing concentration of cucurmosin, the expression of EGFR, p-PI3K, Akt, p-Akt, mTOR, p-mTOR, P70S6K-, p-P70S6K-, 4E-BP1 and p-4E-BP1 at the protein level was decreased, whereas the expression of p-Bad and caspase-9 was elevated.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 280 (char 279)\n",
      "{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"SCN5A\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"PARP-1 was enriched and H1 was depleted at these promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"SCN5A\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PARP-1 was enriched and H1 was depleted at these promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"SCN5A\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"PARP-1 was enriched and H1 was depleted at these promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"SCN5A\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PARP-1 was enriched and H1 was depleted at these promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "normalization of TG very uncertain---{\"tf associated gene name\":[\"CEBPZ\"],\"tg associated gene name\":[\"PDLIM7\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"normalization of TG very uncertain\"],\"sentence\":[\"In addition\",\"the affinities of CBF2 for binding to the LMP-1\",\"LMP-2\",\"and CD23 promoters were also measured.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "normalization of TG very uncertain---{\"tf associated gene name\":[\"CEBPZ\"],\"tg associated gene name\":[\"PDLIM7\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"normalization of TG very uncertain\"],\"sentence\":[\"In addition\",\"the affinities of CBF2 for binding to the LMP-1\",\"LMP-2\",\"and CD23 promoters were also measured.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CEBPZ\"],\"target gene (associated gene name)\":[\"PDLIM7\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In addition\",\"the affinities of CBF2 for binding to the LMP-1\",\"LMP-2\",\"and CD23 promoters were also measured.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|LMP part enouf=gh?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPZ\"],\"tg (associated gene name)\":[\"PDLIM7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"LMP part enouf=gh?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In addition, the affinities of CBF2 for binding to the LMP-1, LMP-2, and CD23 promoters were also measured.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "chick---{\"tf associated gene name\":[\"MTF2\"],\"tg associated gene name\":[\"SHH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"s\"],\"negation\":[],\"comment\":[\"chick\"],\"sentence\":[\"Repression of chick Pcl2 expression in the early embryo results in randomized heart looping direction\",\"which is accompanied by the ectopic expression of Shh in the right side of the node and Shh downstream genes in the right lateral plate mesoderm (LPM)\",\"while overexpression of chick Pcl2 represses Shh expression in the node.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "chick---{\"tf associated gene name\":[\"MTF2\"],\"tg associated gene name\":[\"SHH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"s\"],\"negation\":[],\"comment\":[\"chick\"],\"sentence\":[\"Repression of chick Pcl2 expression in the early embryo results in randomized heart looping direction\",\"which is accompanied by the ectopic expression of Shh in the right side of the node and Shh downstream genes in the right lateral plate mesoderm (LPM)\",\"while overexpression of chick Pcl2 represses Shh expression in the node.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MTF2\"],\"target gene (associated gene name)\":[\"SHH\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Repression of chick Pcl2 expression in the early embryo results in randomized heart looping direction\",\"which is accompanied by the ectopic expression of Shh in the right side of the node and Shh downstream genes in the right lateral plate mesoderm (LPM)\",\"while overexpression of chick Pcl2 represses Shh expression in the node.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MTF2\"],\"tg (associated gene name)\":[\"SHH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Repression of chick Pcl2 expression in the early embryo results in randomized heart looping direction, which is accompanied by the ectopic expression of Shh in the right side of the node and Shh downstream genes in the right lateral plate mesoderm (LPM), while overexpression of chick Pcl2 represses Shh expression in the node.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 375 (char 374)\n",
      "{\"transcription factor (associated gene name)\":[\"STAT3\"],\"target gene (associated gene name)\":[\"REN\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that in As4.1 cells\",\"IL-1beta down-regulates renin gene expression via a mechanism involving the Erk-STAT3 pathway.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|pathway sufficient?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"REN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"pathway sufficient?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We conclude that in As4.1 cells, IL-1beta down-regulates renin gene expression via a mechanism involving the Erk-STAT3 pathway.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"STAT3\"],\"target gene (associated gene name)\":[\"REN\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that in As4.1 cells\",\"IL-1beta down-regulates renin gene expression via a mechanism involving the Erk-STAT3 pathway.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|pathway sufficient?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"REN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"pathway sufficient?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We conclude that in As4.1 cells, IL-1beta down-regulates renin gene expression via a mechanism involving the Erk-STAT3 pathway.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is RCOR1---{\"tf associated gene name\":[\"RCOR2\"],\"tg associated gene name\":[\"HSPA4\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u-fam\"],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is RCOR1\"],\"sentence\":[\"Via chromatin immunoprecipitation assays we show that CoREST is bound to the hsp70 gene promoter under basal conditions and that its binding increases during heat shock response.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is RCOR1---{\"tf associated gene name\":[\"RCOR2\"],\"tg associated gene name\":[\"HSPA4\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u-fam\"],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is RCOR1\"],\"sentence\":[\"Via chromatin immunoprecipitation assays we show that CoREST is bound to the hsp70 gene promoter under basal conditions and that its binding increases during heat shock response.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RCOR2\"],\"target gene (associated gene name)\":[\"HSPA4\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Via chromatin immunoprecipitation assays we show that CoREST is bound to the hsp70 gene promoter under basal conditions and that its binding increases during heat shock response.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|coREST?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RCOR2\"],\"tg (associated gene name)\":[\"HSPA4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"coREST?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Via chromatin immunoprecipitation assays we show that CoREST is bound to the hsp70 gene promoter under basal conditions and that its binding increases during heat shock response.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is JAM-C---{\"tf associated gene name\":[\"FOXP3\"],\"tg associated gene name\":[\"CD4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is JAM-C\"],\"sentence\":[\"The highest level of JAM-C up-regulation by PHA was observed in CD3(+)forkhead box P3(+) and CD4(+)CD25(high) T cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"CD4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The highest level of JAM-C up-regulation by PHA was observed in CD3(+)forkhead box P3(+) and CD4(+)CD25(high) T cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is JAM-C---{\"tf associated gene name\":[\"FOXP3\"],\"tg associated gene name\":[\"CD4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is JAM-C\"],\"sentence\":[\"The highest level of JAM-C up-regulation by PHA was observed in CD3(+)forkhead box P3(+) and CD4(+)CD25(high) T cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"CD4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The highest level of JAM-C up-regulation by PHA was observed in CD3(+)forkhead box P3(+) and CD4(+)CD25(high) T cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GNAS-problem; TG is PTHrP ---{\"tf associated gene name\":[\"NFKB1\"],\"tg associated gene name\":[\"GNAS\"],\"tf text\":[\"p50\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"GNAS-problem; TG is PTHrP \"],\"sentence\":[\"Using transient co-transfection with NF-kappaB expression plasmids and PTHrP P2 luciferase reporter-plasmid\",\"we showed that NF-kappaB p50/p50 alone and p50/c-Rel or p50/Bcl-3 cooperatively upregulated the PTHrP P2 promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|GNAS?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB1\"],\"tg (associated gene name)\":[\"GNAS\"],\"tf text\":[\"p50\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"GNAS?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using transient co-transfection with NF-kappaB expression plasmids and PTHrP P2 luciferase reporter-plasmid, we showed that NF-kappaB p50/p50 alone and p50/c-Rel or p50/Bcl-3 cooperatively upregulated the PTHrP P2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "GNAS-problem; TG is PTHrP ---{\"tf associated gene name\":[\"NFKB1\"],\"tg associated gene name\":[\"GNAS\"],\"tf text\":[\"p50\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"GNAS-problem; TG is PTHrP \"],\"sentence\":[\"Using transient co-transfection with NF-kappaB expression plasmids and PTHrP P2 luciferase reporter-plasmid\",\"we showed that NF-kappaB p50/p50 alone and p50/c-Rel or p50/Bcl-3 cooperatively upregulated the PTHrP P2 promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|GNAS?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB1\"],\"tg (associated gene name)\":[\"GNAS\"],\"tf text\":[\"p50\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"GNAS?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using transient co-transfection with NF-kappaB expression plasmids and PTHrP P2 luciferase reporter-plasmid, we showed that NF-kappaB p50/p50 alone and p50/c-Rel or p50/Bcl-3 cooperatively upregulated the PTHrP P2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 648 (char 647)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"EDN1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"HG\",\"PKC activators\",\"and ETs (ET-1 and ET-3) that increased FN expression also caused activation of NF-kappaB and AP-1.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.312797\"],\"pmid counts\":[\"9\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"118\"],\"sentence pair density\":[\"0.0254777070063694\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"EDN1\"],\"tf text\":[],\"tg text\":[\"ET-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HG, PKC activators, and ETs (ET-1 and ET-3) that increased FN expression also caused activation of NF-kappaB and AP-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"EDN1\"],\"tf text\":[],\"tg text\":[\"ET-1\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"HG\",\"PKC activators\",\"and ETs (ET-1 and ET-3) that increased FN expression also caused activation of NF-kappaB and AP-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"EDN1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"HG\",\"PKC activators\",\"and ETs (ET-1 and ET-3) that increased FN expression also caused activation of NF-kappaB and AP-1.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.312797\"],\"pmid counts\":[\"9\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"118\"],\"sentence pair density\":[\"0.0254777070063694\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"EDN1\"],\"tf text\":[],\"tg text\":[\"ET-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HG, PKC activators, and ETs (ET-1 and ET-3) that increased FN expression also caused activation of NF-kappaB and AP-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 706 (char 705)\n",
      "{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"HIF1A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"To test the hypothesis that the activation of the hypoxia-responsive transcription factor hypoxia inducible factor-1alpha (HIF-1alpha) is involved in the pathophysiology of glaucoma\",\"we used a rat model of glaucoma to study (1) HIF-1alpha retinal protein levels by immunoblot analysis\",\"(2) cellular localization of HIF-1alpha in the retina by immunohistochemistry\",\"and (3) expression of retinal HIF-1 gene targets by quantitative real-time polymerase chain reaction.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"HIF1A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To test the hypothesis that the activation of the hypoxia-responsive transcription factor hypoxia inducible factor-1alpha (HIF-1alpha) is involved in the pathophysiology of glaucoma, we used a rat model of glaucoma to study (1) HIF-1alpha retinal protein levels by immunoblot analysis, (2) cellular localization of HIF-1alpha in the retina by immunohistochemistry, and (3) expression of retinal HIF-1 gene targets by quantitative real-time polymerase chain reaction.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"HIF1A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"To test the hypothesis that the activation of the hypoxia-responsive transcription factor hypoxia inducible factor-1alpha (HIF-1alpha) is involved in the pathophysiology of glaucoma\",\"we used a rat model of glaucoma to study (1) HIF-1alpha retinal protein levels by immunoblot analysis\",\"(2) cellular localization of HIF-1alpha in the retina by immunohistochemistry\",\"and (3) expression of retinal HIF-1 gene targets by quantitative real-time polymerase chain reaction.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"HIF1A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To test the hypothesis that the activation of the hypoxia-responsive transcription factor hypoxia inducible factor-1alpha (HIF-1alpha) is involved in the pathophysiology of glaucoma, we used a rat model of glaucoma to study (1) HIF-1alpha retinal protein levels by immunoblot analysis, (2) cellular localization of HIF-1alpha in the retina by immunohistochemistry, and (3) expression of retinal HIF-1 gene targets by quantitative real-time polymerase chain reaction.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is ALR---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"EXOC4\"],\"tf text\":[\"Nrf2\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is ALR\"],\"sentence\":[\"Here we determined the Nrf2/antioxidant response element (ARE) regulated expression of ALR and show ALR as a target gene of Nrf2 in vitro and in vivo.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"EXOC4\"],\"tf text\":[\"Nrf2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we determined the Nrf2/antioxidant response element (ARE) regulated expression of ALR and show ALR as a target gene of Nrf2 in vitro and in vivo.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is ALR---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"EXOC4\"],\"tf text\":[\"Nrf2\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is ALR\"],\"sentence\":[\"Here we determined the Nrf2/antioxidant response element (ARE) regulated expression of ALR and show ALR as a target gene of Nrf2 in vitro and in vivo.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"EXOC4\"],\"tf text\":[\"Nrf2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we determined the Nrf2/antioxidant response element (ARE) regulated expression of ALR and show ALR as a target gene of Nrf2 in vitro and in vivo.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG=MGAM---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=MGAM\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We showed that the HS-diet increased acetylation of histones, bindings of a coactivator, Creb binding protein (CREBBP), and the transcriptional factors caudal type homeobox 2 (CDX2) and HNF1 homeobox (HNF1) in the promoter/enhancer and transcriptional regions of Mgam gene.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG=MGAM---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=MGAM\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We showed that the HS-diet increased acetylation of histones, bindings of a coactivator, Creb binding protein (CREBBP), and the transcriptional factors caudal type homeobox 2 (CDX2) and HNF1 homeobox (HNF1) in the promoter/enhancer and transcriptional regions of Mgam gene.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GNAS-proble; TG is bcl-2---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"GNAS\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"GNAS-proble; TG is bcl-2\"],\"sentence\":[\"Expression of C/EBPalpha increased bcl-2 expression\",\"and two regions of the bcl-2 P2 promoter that mediated this effect were identified.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG+Bcl2---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"GNAS\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG+Bcl2\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression of C/EBPalpha increased bcl-2 expression, and two regions of the bcl-2 P2 promoter that mediated this effect were identified.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "GNAS-proble; TG is bcl-2---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"GNAS\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"GNAS-proble; TG is bcl-2\"],\"sentence\":[\"Expression of C/EBPalpha increased bcl-2 expression\",\"and two regions of the bcl-2 P2 promoter that mediated this effect were identified.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG+Bcl2---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"GNAS\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG+Bcl2\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression of C/EBPalpha increased bcl-2 expression, and two regions of the bcl-2 P2 promoter that mediated this effect were identified.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is apoptosis protease-activating factor 1 ---{\"tf associated gene name\":[\"E2F6\"],\"tg associated gene name\":[\"ERVK-6\"],\"tf text\":[\"E2F6\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is apoptosis protease-activating factor 1 \"],\"sentence\":[\"Moreover\",\"hypoxia up-regulated expression of E2F1-responsive proapoptotic gene apoptosis protease-activating factor 1 was repressed by E2F6 overexpression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|correct TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F6\"],\"tg (associated gene name)\":[\"ERVK-6\"],\"tf text\":[\"E2F6\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"correct TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, hypoxia up-regulated expression of E2F1-responsive proapoptotic gene apoptosis protease-activating factor 1 was repressed by E2F6 overexpression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is apoptosis protease-activating factor 1 ---{\"tf associated gene name\":[\"E2F6\"],\"tg associated gene name\":[\"ERVK-6\"],\"tf text\":[\"E2F6\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is apoptosis protease-activating factor 1 \"],\"sentence\":[\"Moreover\",\"hypoxia up-regulated expression of E2F1-responsive proapoptotic gene apoptosis protease-activating factor 1 was repressed by E2F6 overexpression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|correct TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F6\"],\"tg (associated gene name)\":[\"ERVK-6\"],\"tf text\":[\"E2F6\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"correct TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, hypoxia up-regulated expression of E2F1-responsive proapoptotic gene apoptosis protease-activating factor 1 was repressed by E2F6 overexpression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 488 (char 487)\n",
      "{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"POU5F1\"],\"tf text\":[\"Oct4\"],\"tg text\":[\"Oct4\"],\"1\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"Through promoter mapping and chromatin immuno-precipitation assays\",\"we found that Oct4 is a direct target of ??-catenin/TCF-mediated transcription and the binding site at -875/-881 of Oct4 promoter is critical for b-catenin/TCF-dependent expression regulation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"POU5F1\"],\"tf text\":[\"Oct-04\"],\"tg text\":[\"Oct-04\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Through promoter mapping and chromatin immuno-precipitation assays, we found that Oct4 is a direct target of -catenin/TCF-mediated transcription and the binding site at -875/-881 of Oct4 promoter is critical for b-catenin/TCF-dependent expression regulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"POU5F1\"],\"tf text\":[\"Oct4\"],\"tg text\":[\"Oct4\"],\"1\":[],\"not true\":[\"t\"],\"comment\":[],\"sentence\":[\"Through promoter mapping and chromatin immuno-precipitation assays\",\"we found that Oct4 is a direct target of ??-catenin/TCF-mediated transcription and the binding site at -875/-881 of Oct4 promoter is critical for b-catenin/TCF-dependent expression regulation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"POU5F1\"],\"tf text\":[\"Oct-04\"],\"tg text\":[\"Oct-04\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Through promoter mapping and chromatin immuno-precipitation assays, we found that Oct4 is a direct target of -catenin/TCF-mediated transcription and the binding site at -875/-881 of Oct4 promoter is critical for b-catenin/TCF-dependent expression regulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TH?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF4\"],\"tg (associated gene name)\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TH?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The increased expression of genes involved in Th17 differentiation observed in Elf4(-/-) CD4(+) T cells suggested that ELF4 controls their programming into the Th17 lineage rather than only IL-17A gene expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TH?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF4\"],\"tg (associated gene name)\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TH?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The increased expression of genes involved in Th17 differentiation observed in Elf4(-/-) CD4(+) T cells suggested that ELF4 controls their programming into the Th17 lineage rather than only IL-17A gene expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is NFKB---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF is NFKB\"],\"sentence\":[\"CHOP KD resulted in reduced cytokine-induced NF-??B activity and expression of key NF-??B target genes involved in apoptosis and inflammation\",\"including iNOS\",\"FAS\",\"IRF-7\",\"IL-15\",\"CCL5 and CXCL10.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|KD TF????---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"KD TF????\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"CHOP KD resulted in reduced cytokine-induced NF-B activity and expression of key NF-B target genes involved in apoptosis and inflammation, including iNOS, FAS, IRF-7, IL-15, CCL5 and CXCL10.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is NFKB---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF is NFKB\"],\"sentence\":[\"CHOP KD resulted in reduced cytokine-induced NF-??B activity and expression of key NF-??B target genes involved in apoptosis and inflammation\",\"including iNOS\",\"FAS\",\"IRF-7\",\"IL-15\",\"CCL5 and CXCL10.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|KD TF????---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"KD TF????\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"CHOP KD resulted in reduced cytokine-induced NF-B activity and expression of key NF-B target genes involved in apoptosis and inflammation, including iNOS, FAS, IRF-7, IL-15, CCL5 and CXCL10.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG|MAD1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD1\"],\"tg (associated gene name)\":[\"CDK2\"],\"tf text\":[],\"tg text\":[\"CDK2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[\"MAD1\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Our findings define a novel interplay between the cell cycle regulator cyclin E/CDK2 and Mad1 and its associated repressor complex and suggests an additional mechanism how cyclin E/CDK2 affects the G(1) to S phase transition.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG|MAD1---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD1\"],\"tg (associated gene name)\":[\"CDK2\"],\"tf text\":[],\"tg text\":[\"CDK2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[\"MAD1\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Our findings define a novel interplay between the cell cycle regulator cyclin E/CDK2 and Mad1 and its associated repressor complex and suggests an additional mechanism how cyclin E/CDK2 affects the G(1) to S phase transition.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped roles?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRF1\"],\"tg (associated gene name)\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Moreover, the expression of mitochondrial biogenesis genes nuclear respiratory factor 1 (NRF-1), mitochondrial transcription factor A (TFAM), mitochondrial transcription factor B1 (TFB1M) and mitochondrial transcription factor B2 (TFB2M), which are all regulated by PGC1?? activity, remained unchanged in fasted CDE-treated mice.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped roles?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NRF1\"],\"tg (associated gene name)\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Moreover, the expression of mitochondrial biogenesis genes nuclear respiratory factor 1 (NRF-1), mitochondrial transcription factor A (TFAM), mitochondrial transcription factor B1 (TFB1M) and mitochondrial transcription factor B2 (TFB2M), which are all regulated by PGC1?? activity, remained unchanged in fasted CDE-treated mice.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is CXCR4---{\"tf associated gene name\":[\"GABPA\"],\"tg associated gene name\":[\"SYVN1\"],\"tf text\":[\"GABPalpha\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is CXCR4\"],\"sentence\":[\"ILF-3 was demonstrated to activate the synoviolin promoter via association with GABPalpha in rheumatoid synovial cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is CXCR4---{\"tf associated gene name\":[\"GABPA\"],\"tg associated gene name\":[\"SYVN1\"],\"tf text\":[\"GABPalpha\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is CXCR4\"],\"sentence\":[\"ILF-3 was demonstrated to activate the synoviolin promoter via association with GABPalpha in rheumatoid synovial cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"GABPA\"],\"target gene (associated gene name)\":[\"SYVN1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"ILF-3 was demonstrated to activate the synoviolin promoter via association with GABPalpha in rheumatoid synovial cells.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GABPA\"],\"tg (associated gene name)\":[\"SYVN1\"],\"tf text\":[\"GABPalpha\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"ILF-3 was demonstrated to activate the synoviolin promoter via association with GABPalpha in rheumatoid synovial cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 471 (char 470)\n",
      "{\"tf associated gene name\":[\"NCOR2\"],\"tg associated gene name\":[\"NCOR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Taken together\",\"our data indicate that SMRT and N-CoR corepressors are involved in transcriptional regulation by both agonist- and antagonist-bound AR and regulate the magnitude of hormone response\",\"at least in part\",\"by competing with coactivators.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOR2\"],\"tg (associated gene name)\":[\"NCOR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, our data indicate that SMRT and N-CoR corepressors are involved in transcriptional regulation by both agonist- and antagonist-bound AR and regulate the magnitude of hormone response, at least in part, by competing with coactivators.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NCOR2\"],\"tg associated gene name\":[\"NCOR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Taken together\",\"our data indicate that SMRT and N-CoR corepressors are involved in transcriptional regulation by both agonist- and antagonist-bound AR and regulate the magnitude of hormone response\",\"at least in part\",\"by competing with coactivators.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOR2\"],\"tg (associated gene name)\":[\"NCOR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, our data indicate that SMRT and N-CoR corepressors are involved in transcriptional regulation by both agonist- and antagonist-bound AR and regulate the magnitude of hormone response, at least in part, by competing with coactivators.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 651 (char 650)\n",
      "{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"ERVW-4\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"To explore the global mechanisms of estrogen-regulated transcription\",\"we used chromatin immunoprecipitation coupled with DNA microarrays to determine the localization of RNA polymerase II (Pol II)\",\"estrogen receptor alpha (ERalpha)\",\"steroid receptor coactivator proteins (SRC)\",\"and acetylated histones H3/H4 (AcH) at estrogen-regulated promoters in MCF-7 cells with or without estradiol (E2) treatment.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ERVW-4\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To explore the global mechanisms of estrogen-regulated transcription, we used chromatin immunoprecipitation coupled with DNA microarrays to determine the localization of RNA polymerase II (Pol II), estrogen receptor alpha (ERalpha), steroid receptor coactivator proteins (SRC), and acetylated histones H3/H4 (AcH) at estrogen-regulated promoters in MCF-7 cells with or without estradiol (E2) treatment.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"ERVW-4\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"To explore the global mechanisms of estrogen-regulated transcription\",\"we used chromatin immunoprecipitation coupled with DNA microarrays to determine the localization of RNA polymerase II (Pol II)\",\"estrogen receptor alpha (ERalpha)\",\"steroid receptor coactivator proteins (SRC)\",\"and acetylated histones H3/H4 (AcH) at estrogen-regulated promoters in MCF-7 cells with or without estradiol (E2) treatment.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ERVW-4\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To explore the global mechanisms of estrogen-regulated transcription, we used chromatin immunoprecipitation coupled with DNA microarrays to determine the localization of RNA polymerase II (Pol II), estrogen receptor alpha (ERalpha), steroid receptor coactivator proteins (SRC), and acetylated histones H3/H4 (AcH) at estrogen-regulated promoters in MCF-7 cells with or without estradiol (E2) treatment.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "protein level?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD2\"],\"tg (associated gene name)\":[\"SMARCA4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Furthermore, we show that endogenous Smad2 interacts with the SWI/SNF ATPase, Brg1, in a TGF-beta-dependent manner, and demonstrate that Brg1 is recruited to Smad2-dependent promoters and is specifically required for TGF-beta-induced expression of endogenous Smad2 target genes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "protein level?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD2\"],\"tg (associated gene name)\":[\"SMARCA4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Furthermore, we show that endogenous Smad2 interacts with the SWI/SNF ATPase, Brg1, in a TGF-beta-dependent manner, and demonstrate that Brg1 is recruited to Smad2-dependent promoters and is specifically required for TGF-beta-induced expression of endogenous Smad2 target genes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MTF1\"],\"tg (associated gene name)\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We propose that Hsp60 may act as a molecular chaperone that folds mitochondrial Mtf1 into a functional form and that ETB inhibits this Hsp60 chaperone activity by disrupting Mtf1 binding to Hsp60 and thus inhibits mitochondrial transcription in fission yeast.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MTF1\"],\"tg (associated gene name)\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We propose that Hsp60 may act as a molecular chaperone that folds mitochondrial Mtf1 into a functional form and that ETB inhibits this Hsp60 chaperone activity by disrupting Mtf1 binding to Hsp60 and thus inhibits mitochondrial transcription in fission yeast.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 436 (char 435)\n",
      "{\"transcription factor (associated gene name)\":[\"ASCL2\"],\"target gene (associated gene name)\":[\"CYP19A1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These findings suggest that increased protein levels and DNA binding of USF1 and USF2 mediate the inhibitory effects of hypoxia and of Mash-2 on CYP19 gene expression in human placenta.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|Mash2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ASCL2\"],\"tg (associated gene name)\":[\"CYP19A1\"],\"tf text\":[\"Mash-2\"],\"tg text\":[\"CYP19\"],\"consensus\":[\"V\"],\"martin_comments\":[\"Mash2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These findings suggest that increased protein levels and DNA binding of USF1 and USF2 mediate the inhibitory effects of hypoxia and of Mash-2 on CYP19 gene expression in human placenta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ASCL2\"],\"tg associated gene name\":[\"CYP19A1\"],\"tf text\":[\"Mash-2\"],\"tg text\":[\"CYP19\"],\"1\":[\"r\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"These findings suggest that increased protein levels and DNA binding of USF1 and USF2 mediate the inhibitory effects of hypoxia and of Mash-2 on CYP19 gene expression in human placenta.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ASCL2\"],\"target gene (associated gene name)\":[\"CYP19A1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These findings suggest that increased protein levels and DNA binding of USF1 and USF2 mediate the inhibitory effects of hypoxia and of Mash-2 on CYP19 gene expression in human placenta.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|Mash2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ASCL2\"],\"tg (associated gene name)\":[\"CYP19A1\"],\"tf text\":[\"Mash-2\"],\"tg text\":[\"CYP19\"],\"consensus\":[\"V\"],\"martin_comments\":[\"Mash2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These findings suggest that increased protein levels and DNA binding of USF1 and USF2 mediate the inhibitory effects of hypoxia and of Mash-2 on CYP19 gene expression in human placenta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TFs are specificity protein 1/signal transducer and activator of transcription 3 (most likely STAT1 and -3)---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"TIMP1\"],\"tf text\":[],\"tg text\":[\"TIMP-1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TFs are specificity protein 1/signal transducer and activator of transcription 3 (most likely STAT1 and -3)\"],\"sentence\":[\"Leptin increases tissue inhibitor of metalloproteinase I (TIMP-1) gene expression by a specificity protein 1/signal transducer and activator of transcription 3 mechanism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SPI1 not found---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"TIMP1\"],\"tf text\":[],\"tg text\":[\"TIMP-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"SPI1 not found\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Leptin increases tissue inhibitor of metalloproteinase I (TIMP-1) gene expression by a specificity protein 1/signal transducer and activator of transcription 3 mechanism.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TFs are specificity protein 1/signal transducer and activator of transcription 3 (most likely STAT1 and -3)---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"TIMP1\"],\"tf text\":[],\"tg text\":[\"TIMP-1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TFs are specificity protein 1/signal transducer and activator of transcription 3 (most likely STAT1 and -3)\"],\"sentence\":[\"Leptin increases tissue inhibitor of metalloproteinase I (TIMP-1) gene expression by a specificity protein 1/signal transducer and activator of transcription 3 mechanism.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SPI1 not found---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"TIMP1\"],\"tf text\":[],\"tg text\":[\"TIMP-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"SPI1 not found\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Leptin increases tissue inhibitor of metalloproteinase I (TIMP-1) gene expression by a specificity protein 1/signal transducer and activator of transcription 3 mechanism.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 324 (char 323)\n",
      "{\"tf associated gene name\":[\"FOXO3\"],\"tg associated gene name\":[\"GDF9\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Moreover\",\"the activity of BIM promoter encompassing the FOXO3a-binding site could be regulated by GDF9.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=BIM---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO3\"],\"tg (associated gene name)\":[\"GDF9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=BIM\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, the activity of BIM promoter encompassing the FOXO3a-binding site could be regulated by GDF9.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOXO3\"],\"tg associated gene name\":[\"GDF9\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Moreover\",\"the activity of BIM promoter encompassing the FOXO3a-binding site could be regulated by GDF9.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=BIM---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO3\"],\"tg (associated gene name)\":[\"GDF9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=BIM\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, the activity of BIM promoter encompassing the FOXO3a-binding site could be regulated by GDF9.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG=MOR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TFCP2\"],\"tg (associated gene name)\":[\"HDAC2\"],\"tf text\":[],\"tg text\":[\"HDAC-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=MOR\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We also observed by chromatin immunoprecipitation (ChIP) analysis that JNK inhibition led to increased bindings of CBP and histone-3 dimethyl K4, and decreased bindings of HDAC-2, MeCP2, and histone-3 trimethyl K9 to the MOR promoter indicating a transcriptional regulation of MOR by JNK inhibition.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG=MOR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TFCP2\"],\"tg (associated gene name)\":[\"HDAC2\"],\"tf text\":[],\"tg text\":[\"HDAC-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=MOR\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We also observed by chromatin immunoprecipitation (ChIP) analysis that JNK inhibition led to increased bindings of CBP and histone-3 dimethyl K4, and decreased bindings of HDAC-2, MeCP2, and histone-3 trimethyl K9 to the MOR promoter indicating a transcriptional regulation of MOR by JNK inhibition.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 408 (char 407)\n",
      "{\"tf associated gene name\":[\"MBD2\"],\"tg associated gene name\":[\"CYP51A1\"],\"tf text\":[],\"tg text\":[\"P45014DM\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Sterol 14alpha-demethylase (P45014DM) encoded by CYP51 is a member of the cytochrome P450 (CYP) gene superfamily involved in sterol biosynthesis in fungi\",\"plants\",\"and animals.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no transcription implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MBD2\"],\"tg (associated gene name)\":[\"CYP51A1\"],\"tf text\":[],\"tg text\":[\"P45014DM\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no transcription implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sterol 14alpha-demethylase (P45014DM) encoded by CYP51 is a member of the cytochrome P450 (CYP) gene superfamily involved in sterol biosynthesis in fungi, plants, and animals.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MBD2\"],\"tg associated gene name\":[\"CYP51A1\"],\"tf text\":[],\"tg text\":[\"P45014DM\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Sterol 14alpha-demethylase (P45014DM) encoded by CYP51 is a member of the cytochrome P450 (CYP) gene superfamily involved in sterol biosynthesis in fungi\",\"plants\",\"and animals.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no transcription implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MBD2\"],\"tg (associated gene name)\":[\"CYP51A1\"],\"tf text\":[],\"tg text\":[\"P45014DM\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no transcription implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sterol 14alpha-demethylase (P45014DM) encoded by CYP51 is a member of the cytochrome P450 (CYP) gene superfamily involved in sterol biosynthesis in fungi, plants, and animals.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is RGS2 ---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"PLA2G6\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is RGS2 \"],\"sentence\":[\"Forskolin-stimulated RGS2 mRNA up-regulation is inhibited by CREB sequestration or specific disruption of the CREB-RGS2 promoter interaction\",\"and Ang II-induced CREB phosphorylation and nuclear localization are blocked by iPLA(2)?? pharmacologic inhibition or genetic ablation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|RGS2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"PLA2G6\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"RGS2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Forskolin-stimulated RGS2 mRNA up-regulation is inhibited by CREB sequestration or specific disruption of the CREB-RGS2 promoter interaction, and Ang II-induced CREB phosphorylation and nuclear localization are blocked by iPLA(2) pharmacologic inhibition or genetic ablation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is RGS2 ---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"PLA2G6\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is RGS2 \"],\"sentence\":[\"Forskolin-stimulated RGS2 mRNA up-regulation is inhibited by CREB sequestration or specific disruption of the CREB-RGS2 promoter interaction\",\"and Ang II-induced CREB phosphorylation and nuclear localization are blocked by iPLA(2)?? pharmacologic inhibition or genetic ablation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|RGS2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"PLA2G6\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"RGS2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Forskolin-stimulated RGS2 mRNA up-regulation is inhibited by CREB sequestration or specific disruption of the CREB-RGS2 promoter interaction, and Ang II-induced CREB phosphorylation and nuclear localization are blocked by iPLA(2) pharmacologic inhibition or genetic ablation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "contains upstream---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"PTGS2\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"contains upstream\"],\"sentence\":[\"The inductive effects of HER-2/neu were mediated\",\"in part\",\"by enhanced binding of AP-1 (c-Jun\",\"c-Fos\",\"and ATF-2) to the cyclic AMP-response element (-59/-53) of the COX-2 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "contains upstream---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"PTGS2\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"contains upstream\"],\"sentence\":[\"The inductive effects of HER-2/neu were mediated\",\"in part\",\"by enhanced binding of AP-1 (c-Jun\",\"c-Fos\",\"and ATF-2) to the cyclic AMP-response element (-59/-53) of the COX-2 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"PTGS2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The inductive effects of HER-2/neu were mediated\",\"in part\",\"by enhanced binding of AP-1 (c-Jun\",\"c-Fos\",\"and ATF-2) to the cyclic AMP-response element (-59/-53) of the COX-2 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"PTGS2\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"The inductive effects of HER-2/neu were mediated, in part, by enhanced binding of AP-1 (c-Jun, c-Fos, and ATF-2) to the cyclic AMP-response element (-59/-53) of the COX-2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "xenopus---{\"tf associated gene name\":[\"FOXH1\"],\"tg associated gene name\":[\"INHBA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"up\",\"s\"],\"comment\":[\"xenopus\"],\"sentence\":[\"The role of FAST-1 and Smads in transcriptional regulation by activin during early Xenopus embryogenesis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXH1\"],\"tg (associated gene name)\":[\"INHBA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The role of FAST-1 and Smads in transcriptional regulation by activin during early Xenopus embryogenesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "xenopus---{\"tf associated gene name\":[\"FOXH1\"],\"tg associated gene name\":[\"INHBA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"up\",\"s\"],\"comment\":[\"xenopus\"],\"sentence\":[\"The role of FAST-1 and Smads in transcriptional regulation by activin during early Xenopus embryogenesis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXH1\"],\"tg (associated gene name)\":[\"INHBA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The role of FAST-1 and Smads in transcriptional regulation by activin during early Xenopus embryogenesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "coexpressed---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"NES\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"coexpressed\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Oct4, a Class V POU factor, was found to be coexpressed with nestin in EC cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "coexpressed---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"NES\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"coexpressed\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Oct4, a Class V POU factor, was found to be coexpressed with nestin in EC cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"IL10\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Decreased TGIF expression resulted in reduced proliferation and differentiation and lower expression of CEBPbeta, CEBPepsilon, PU.1 and RUNX1, key myeloid transcription factors.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"IL10\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Decreased TGIF expression resulted in reduced proliferation and differentiation and lower expression of CEBPbeta, CEBPepsilon, PU.1 and RUNX1, key myeloid transcription factors.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is prostate specific antigen ---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[\"Stat3\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is prostate specific antigen \"],\"sentence\":[\"Activation of Stat3 in androgen-sensitive LNCaP prostate cancer cells results in enhancement of tumor growth in both intact and castrated male nude mice and enhances androgen receptor-mediated prostate specific antigen expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is prostate specific antigen ---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[\"Stat3\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is prostate specific antigen \"],\"sentence\":[\"Activation of Stat3 in androgen-sensitive LNCaP prostate cancer cells results in enhancement of tumor growth in both intact and castrated male nude mice and enhances androgen receptor-mediated prostate specific antigen expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"STAT3\"],\"target gene (associated gene name)\":[\"AR\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Activation of Stat3 in androgen-sensitive LNCaP prostate cancer cells results in enhancement of tumor growth in both intact and castrated male nude mice and enhances androgen receptor-mediated prostate specific antigen expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[\"Stat3\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Activation of Stat3 in androgen-sensitive LNCaP prostate cancer cells results in enhancement of tumor growth in both intact and castrated male nude mice and enhances androgen receptor-mediated prostate specific antigen expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 556 (char 555)\n",
      "{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"1\":[],\"negation\":[],\"not true\":[\"o\"],\"comment\":[],\"sentence\":[\"Conversely\",\"the inhibition of p53 by three different p53 inhibitors\",\"Pifithrin-1alpha (PFT-1alpha)\",\"an inhibitor of p53-dependent transactivation; MDM2\",\"a regulator of p53 expression; and a dominant-negative N-terminally truncated p53 gene also reduced transactivation of androgen-dependent reporter genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Conversely, the inhibition of p53 by three different p53 inhibitors, Pifithrin-1alpha (PFT-1alpha), an inhibitor of p53-dependent transactivation; MDM2, a regulator of p53 expression; and a dominant-negative N-terminally truncated p53 gene also reduced transactivation of androgen-dependent reporter genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"1\":[],\"negation\":[],\"not true\":[\"o\"],\"comment\":[],\"sentence\":[\"Conversely\",\"the inhibition of p53 by three different p53 inhibitors\",\"Pifithrin-1alpha (PFT-1alpha)\",\"an inhibitor of p53-dependent transactivation; MDM2\",\"a regulator of p53 expression; and a dominant-negative N-terminally truncated p53 gene also reduced transactivation of androgen-dependent reporter genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Conversely, the inhibition of p53 by three different p53 inhibitors, Pifithrin-1alpha (PFT-1alpha), an inhibitor of p53-dependent transactivation; MDM2, a regulator of p53 expression; and a dominant-negative N-terminally truncated p53 gene also reduced transactivation of androgen-dependent reporter genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 525 (char 524)\n",
      "{\"tf associated gene name\":[\"SOX2\"],\"tg associated gene name\":[\"SAR1B\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The increase in cell proliferation induced by hypoxia was greatly attenuated by blocking Notch signaling with the inhibitor DAPT\",\"resulting in the reduced expression of positive cell-cycle regulators (cyclins A and D1) and neural and retinal progenitor markers (Notch1\",\"Hes1\",\"Sox2\",\"Pax6\",\"and NeuroD1).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG? TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SOX2\"],\"tg (associated gene name)\":[\"SAR1B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG? TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The increase in cell proliferation induced by hypoxia was greatly attenuated by blocking Notch signaling with the inhibitor DAPT, resulting in the reduced expression of positive cell-cycle regulators (cyclins A and D1) and neural and retinal progenitor markers (Notch1, Hes1, Sox2, Pax6, and NeuroD1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SOX2\"],\"tg associated gene name\":[\"SAR1B\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"The increase in cell proliferation induced by hypoxia was greatly attenuated by blocking Notch signaling with the inhibitor DAPT\",\"resulting in the reduced expression of positive cell-cycle regulators (cyclins A and D1) and neural and retinal progenitor markers (Notch1\",\"Hes1\",\"Sox2\",\"Pax6\",\"and NeuroD1).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG? TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SOX2\"],\"tg (associated gene name)\":[\"SAR1B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG? TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The increase in cell proliferation induced by hypoxia was greatly attenuated by blocking Notch signaling with the inhibitor DAPT, resulting in the reduced expression of positive cell-cycle regulators (cyclins A and D1) and neural and retinal progenitor markers (Notch1, Hes1, Sox2, Pax6, and NeuroD1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 465 (char 464)\n",
      "{\"tf associated gene name\":[\"NCOA1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[\"SRC-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"Thus\",\"HNF4 is involved in the regulation of glucose homeostasis at several levels and along with the SRC-1\",\"GRIP1\",\"and p300 may play an important role in the pathophysiology of non-insulin-dependent diabetes mellitus.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[\"SRC-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, HNF4 is involved in the regulation of glucose homeostasis at several levels and along with the SRC-1, GRIP1, and p300 may play an important role in the pathophysiology of non-insulin-dependent diabetes mellitus.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NCOA1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[\"SRC-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"Thus\",\"HNF4 is involved in the regulation of glucose homeostasis at several levels and along with the SRC-1\",\"GRIP1\",\"and p300 may play an important role in the pathophysiology of non-insulin-dependent diabetes mellitus.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[\"SRC-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, HNF4 is involved in the regulation of glucose homeostasis at several levels and along with the SRC-1, GRIP1, and p300 may play an important role in the pathophysiology of non-insulin-dependent diabetes mellitus.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is H4-genes (histone 4 genes)---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is H4-genes (histone 4 genes)\"],\"sentence\":[\"Based on gel shift assays\",\"nucleotide variations in the promoters of these H4 genes preclude or reduce binding of several histone gene transcription factors\",\"including IRF2\",\"HiNF-D\",\"SP-1 and/or YY1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|H4 genes?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"H4 genes?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Based on gel shift assays, nucleotide variations in the promoters of these H4 genes preclude or reduce binding of several histone gene transcription factors, including IRF2, HiNF-D, SP-1 and/or YY1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is H4-genes (histone 4 genes)---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is H4-genes (histone 4 genes)\"],\"sentence\":[\"Based on gel shift assays\",\"nucleotide variations in the promoters of these H4 genes preclude or reduce binding of several histone gene transcription factors\",\"including IRF2\",\"HiNF-D\",\"SP-1 and/or YY1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|H4 genes?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"H4 genes?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Based on gel shift assays, nucleotide variations in the promoters of these H4 genes preclude or reduce binding of several histone gene transcription factors, including IRF2, HiNF-D, SP-1 and/or YY1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "indirect at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBPL1\"],\"tg (associated gene name)\":[\"GRSF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"indirect at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These included mRNA encoding Pam, a protein associated with myc; GRSF1, an mRNA-binding protein involved in translation activation, and TRF2, a TATA box-binding protein-like protein involved in transcriptional regulation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "indirect at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBPL1\"],\"tg (associated gene name)\":[\"GRSF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"indirect at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These included mRNA encoding Pam, a protein associated with myc; GRSF1, an mRNA-binding protein involved in translation activation, and TRF2, a TATA box-binding protein-like protein involved in transcriptional regulation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "viral? indicated b: \"lytic reactivation of HHV-8\"---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"MRGPRF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"viral? indicated b: \\\"lytic reactivation of HHV-8\\\"\"],\"sentence\":[\"These studies suggest that Rta expression during lytic reactivation of HHV-8 would lead to expression of some cellular genes\",\"including IL-6\",\"whereas activation of NF-kappa B could inhibit some responses to Rta.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "viral? indicated b: \"lytic reactivation of HHV-8\"---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"MRGPRF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"viral? indicated b: \\\"lytic reactivation of HHV-8\\\"\"],\"sentence\":[\"These studies suggest that Rta expression during lytic reactivation of HHV-8 would lead to expression of some cellular genes\",\"including IL-6\",\"whereas activation of NF-kappa B could inhibit some responses to Rta.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"MRGPRF\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"These studies suggest that Rta expression during lytic reactivation of HHV-8 would lead to expression of some cellular genes\",\"including IL-6\",\"whereas activation of NF-kappa B could inhibit some responses to Rta.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.130504\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"211\"],\"sentence pair density\":[\"0.00757575757575757\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|some cellular genes---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"MRGPRF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"some cellular genes\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These studies suggest that Rta expression during lytic reactivation of HHV-8 would lead to expression of some cellular genes, including IL-6, whereas activation of NF-kappa B could inhibit some responses to Rta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"HHEX\"],\"tg associated gene name\":[\"LBX1\"],\"tf text\":[\"Hex\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"cDNA cloning and expression of rat homeobox gene\",\"Hex\",\"and functional characterization of the protein.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"HHEX\"],\"tg associated gene name\":[\"LBX1\"],\"tf text\":[\"Hex\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"cDNA cloning and expression of rat homeobox gene\",\"Hex\",\"and functional characterization of the protein.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HHEX\"],\"tg (associated gene name)\":[\"LBX1\"],\"tf text\":[\"Hex\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"cDNA cloning and expression of rat homeobox gene, Hex, and functional characterization of the protein.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 628 (char 627)\n",
      "{\"transcription factor (associated gene name)\":[\"FOS\"],\"target gene (associated gene name)\":[\"EDN1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Consistently\",\"the increase of c-fos mRNA levels and promoter activities by Et-1 were also inhibited.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.517629\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"100\"],\"sentence pair density\":[\"0.0054054054054054\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"EDN1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consistently, the increase of c-fos mRNA levels and promoter activities by Et-1 were also inhibited.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"EDN1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Consistently\",\"the increase of c-fos mRNA levels and promoter activities by Et-1 were also inhibited.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOS\"],\"target gene (associated gene name)\":[\"EDN1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Consistently\",\"the increase of c-fos mRNA levels and promoter activities by Et-1 were also inhibited.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.517629\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"100\"],\"sentence pair density\":[\"0.0054054054054054\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"EDN1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consistently, the increase of c-fos mRNA levels and promoter activities by Et-1 were also inhibited.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 670 (char 669)\n",
      "{\"transcription factor (associated gene name)\":[\"THRA\"],\"target gene (associated gene name)\":[\"KDR\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Using chromatin immuno-precipitation and transient transfection/RNA interference assays we show that the ERalpha/Sp protein-promoter interactions are accompanied by recruitment of the co-repressors SMRT (silencing mediator of retinoid and thyroid hormone receptor) and NCoR (nuclear receptor corepressor) to the promoter and that SMRT and NCoR knockdown reverse E2-mediated down-regulation of VEGFR2 expression in MCF-7 cells.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|a bit iffy---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"THRA\"],\"tg (associated gene name)\":[\"KDR\"],\"tf text\":[],\"tg text\":[\"VEGFR2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit iffy\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using chromatin immuno-precipitation and transient transfection/RNA interference assays we show that the ERalpha/Sp protein-promoter interactions are accompanied by recruitment of the co-repressors SMRT (silencing mediator of retinoid and thyroid hormone receptor) and NCoR (nuclear receptor corepressor) to the promoter and that SMRT and NCoR knockdown reverse E2-mediated down-regulation of VEGFR2 expression in MCF-7 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"THRA\"],\"tg associated gene name\":[\"KDR\"],\"tf text\":[],\"tg text\":[\"VEGFR2\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Using chromatin immuno-precipitation and transient transfection/RNA interference assays we show that the ERalpha/Sp protein-promoter interactions are accompanied by recruitment of the co-repressors SMRT (silencing mediator of retinoid and thyroid hormone receptor) and NCoR (nuclear receptor corepressor) to the promoter and that SMRT and NCoR knockdown reverse E2-mediated down-regulation of VEGFR2 expression in MCF-7 cells.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"THRA\"],\"target gene (associated gene name)\":[\"KDR\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Using chromatin immuno-precipitation and transient transfection/RNA interference assays we show that the ERalpha/Sp protein-promoter interactions are accompanied by recruitment of the co-repressors SMRT (silencing mediator of retinoid and thyroid hormone receptor) and NCoR (nuclear receptor corepressor) to the promoter and that SMRT and NCoR knockdown reverse E2-mediated down-regulation of VEGFR2 expression in MCF-7 cells.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|a bit iffy---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"THRA\"],\"tg (associated gene name)\":[\"KDR\"],\"tf text\":[],\"tg text\":[\"VEGFR2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit iffy\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using chromatin immuno-precipitation and transient transfection/RNA interference assays we show that the ERalpha/Sp protein-promoter interactions are accompanied by recruitment of the co-repressors SMRT (silencing mediator of retinoid and thyroid hormone receptor) and NCoR (nuclear receptor corepressor) to the promoter and that SMRT and NCoR knockdown reverse E2-mediated down-regulation of VEGFR2 expression in MCF-7 cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear sentence; unclear that LBP is TG (or other relation)---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"LBP\"],\"tf text\":[],\"tg text\":[\"LBP\"],\"1\":[\"u?\"],\"negation\":[],\"not true\":[],\"comment\":[\"unclear sentence; unclear that LBP is TG (or other relation)\"],\"sentence\":[\"The transcriptional activation pattern of lipopolysaccharide binding protein (LBP) involving transcription factors AP-1 and C/EBP beta.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "unclear sentence; unclear that LBP is TG (or other relation)---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"LBP\"],\"tf text\":[],\"tg text\":[\"LBP\"],\"1\":[\"u?\"],\"negation\":[],\"not true\":[],\"comment\":[\"unclear sentence; unclear that LBP is TG (or other relation)\"],\"sentence\":[\"The transcriptional activation pattern of lipopolysaccharide binding protein (LBP) involving transcription factors AP-1 and C/EBP beta.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CEBPB\"],\"target gene (associated gene name)\":[\"LBP\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The transcriptional activation pattern of lipopolysaccharide binding protein (LBP) involving transcription factors AP-1 and C/EBP beta.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"LBP\"],\"tf text\":[],\"tg text\":[\"LBP\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcriptional activation pattern of lipopolysaccharide binding protein (LBP) involving transcription factors AP-1 and C/EBP beta.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is P4|not likely the suggested normalization---{\"tf associated gene name\":[\"SMAD4\"],\"tg associated gene name\":[\"SLC10A4\"],\"tf text\":[\"SMAD4\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is P4\",\"not likely the suggested normalization\"],\"sentence\":[\"The overexpression of wild-type SMAD4 in deficient cell lines (AsPC-1\",\"Capan-1) stimulated the activity of the P4 promoter\",\"whereas interference of endogenous SMAD4 function with a dominant-negative mutant decreased the viral promoter activity in wild-type SMAD4-expressing cells (Panc-1\",\"MiaPaCa-2) reducing progeny virus production.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is P4|not likely the suggested normalization---{\"tf associated gene name\":[\"SMAD4\"],\"tg associated gene name\":[\"SLC10A4\"],\"tf text\":[\"SMAD4\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is P4\",\"not likely the suggested normalization\"],\"sentence\":[\"The overexpression of wild-type SMAD4 in deficient cell lines (AsPC-1\",\"Capan-1) stimulated the activity of the P4 promoter\",\"whereas interference of endogenous SMAD4 function with a dominant-negative mutant decreased the viral promoter activity in wild-type SMAD4-expressing cells (Panc-1\",\"MiaPaCa-2) reducing progeny virus production.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SMAD4\"],\"target gene (associated gene name)\":[\"SLC10A4\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"a\"],\"negation\":[],\"sentence\":[\"The overexpression of wild-type SMAD4 in deficient cell lines (AsPC-1\",\"Capan-1) stimulated the activity of the P4 promoter\",\"whereas interference of endogenous SMAD4 function with a dominant-negative mutant decreased the viral promoter activity in wild-type SMAD4-expressing cells (Panc-1\",\"MiaPaCa-2) reducing progeny virus production.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.861964\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"334\"],\"sentence pair density\":[\"0.083743842364532\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SMAD4\"],\"target gene (associated gene name)\":[\"SLC10A4\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The overexpression of wild-type SMAD4 in deficient cell lines (AsPC-1\",\"Capan-1) stimulated the activity of the P4 promoter\",\"whereas interference of endogenous SMAD4 function with a dominant-negative mutant decreased the viral promoter activity in wild-type SMAD4-expressing cells (Panc-1\",\"MiaPaCa-2) reducing progeny virus production.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD4\"],\"tg (associated gene name)\":[\"SLC10A4\"],\"tf text\":[\"SMAD4\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"The overexpression of wild-type SMAD4 in deficient cell lines (AsPC-1, Capan-1) stimulated the activity of the P4 promoter, whereas interference of endogenous SMAD4 function with a dominant-negative mutant decreased the viral promoter activity in wild-type SMAD4-expressing cells (Panc-1, MiaPaCa-2) reducing progeny virus production.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GAST correct here?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"GAST\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"GAST correct here?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Activation of MVP expression by IFN-gamma involved transcriptional upregulation through the JAK/STAT pathway based on an interaction of STAT1 with an interferon-gamma-activated site (GAS) within the proximal MVP promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "GAST correct here?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"GAST\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"GAST correct here?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Activation of MVP expression by IFN-gamma involved transcriptional upregulation through the JAK/STAT pathway based on an interaction of STAT1 with an interferon-gamma-activated site (GAS) within the proximal MVP promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 355 (char 354)\n",
      "{\"tf associated gene name\":[\"GATA1\"],\"tg associated gene name\":[\"NFE2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We propose that only when GATA-1 attracts pol II to the locus control region can pol II access the promoter in a NF-E2-dependent manner.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA1\"],\"tg (associated gene name)\":[\"NFE2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We propose that only when GATA-1 attracts pol II to the locus control region can pol II access the promoter in a NF-E2-dependent manner.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"GATA1\"],\"tg associated gene name\":[\"NFE2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We propose that only when GATA-1 attracts pol II to the locus control region can pol II access the promoter in a NF-E2-dependent manner.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA1\"],\"tg (associated gene name)\":[\"NFE2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We propose that only when GATA-1 attracts pol II to the locus control region can pol II access the promoter in a NF-E2-dependent manner.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no specific TF mentioned---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"IRF7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"no specific TF mentioned\"],\"sentence\":[\"The IRF7 gene includes two IFN responsive elements\",\"an IRF-binding element (IRFE) in the promoter region and an ISRE in the first intron\",\"and is induced by the IFN-triggered Jak-STAT pathway by binding of the IFN-stimulated gene factor 3 (ISGF3) complex to the ISRE.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no specific TF mentioned---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"IRF7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"no specific TF mentioned\"],\"sentence\":[\"The IRF7 gene includes two IFN responsive elements\",\"an IRF-binding element (IRFE) in the promoter region and an ISRE in the first intron\",\"and is induced by the IFN-triggered Jak-STAT pathway by binding of the IFN-stimulated gene factor 3 (ISGF3) complex to the ISRE.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"STAT1\"],\"target gene (associated gene name)\":[\"IRF7\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The IRF7 gene includes two IFN responsive elements\",\"an IRF-binding element (IRFE) in the promoter region and an ISRE in the first intron\",\"and is induced by the IFN-triggered Jak-STAT pathway by binding of the IFN-stimulated gene factor 3 (ISGF3) complex to the ISRE.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|pathway?|discuss---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"IRF7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"pathway?\"],\"astrid_comments\":[\"discuss\"],\"sentences with greek letters\":[\"The IRF7 gene includes two IFN responsive elements, an IRF-binding element (IRFE) in the promoter region and an ISRE in the first intron, and is induced by the IFN-triggered Jak-STAT pathway by binding of the IFN-stimulated gene factor 3 (ISGF3) complex to the ISRE.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 412 (char 411)\n",
      "{\"tf associated gene name\":[\"HEY2\"],\"tg associated gene name\":[\"SRF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[],\"sentence\":[\"The present study characterizes the expression of HERP1 in normal and diseased vessels\",\"and tests the hypothesis that HERP1 inhibits SRF/myocardin-dependent SMC gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HEY2\"],\"tg (associated gene name)\":[\"SRF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The present study characterizes the expression of HERP1 in normal and diseased vessels, and tests the hypothesis that HERP1 inhibits SRF/myocardin-dependent SMC gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"HEY2\"],\"tg associated gene name\":[\"SRF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[],\"sentence\":[\"The present study characterizes the expression of HERP1 in normal and diseased vessels\",\"and tests the hypothesis that HERP1 inhibits SRF/myocardin-dependent SMC gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HEY2\"],\"tg (associated gene name)\":[\"SRF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The present study characterizes the expression of HERP1 in normal and diseased vessels, and tests the hypothesis that HERP1 inhibits SRF/myocardin-dependent SMC gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is transforming growth factor (TGF)-beta1---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"B3GNT9\"],\"tf text\":[\"EGR-1\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is transforming growth factor (TGF)-beta1\"],\"sentence\":[\"55\",\"5054-5062) owing in part to up-regulation of the transforming growth factor (TGF)-beta1 promoter by EGR-1 which suppresses growth by an autocrine mechanism (Liu\",\"C.\",\"Adamson\",\"E.\",\"and Mercola\",\"D. (1996) Proc. Natl. Acad. Sci. U. S. A. 93\",\"11831-11836).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"B3GNT9\"],\"tf text\":[\"EGR-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"55, 5054-5062) owing in part to up-regulation of the transforming growth factor (TGF)-beta1 promoter by EGR-1 which suppresses growth by an autocrine mechanism (Liu, C., Adamson, E., and Mercola, D. (1996) Proc. Natl. Acad. Sci. U. S. A. 93, 11831-11836).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is transforming growth factor (TGF)-beta1---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"B3GNT9\"],\"tf text\":[\"EGR-1\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is transforming growth factor (TGF)-beta1\"],\"sentence\":[\"55\",\"5054-5062) owing in part to up-regulation of the transforming growth factor (TGF)-beta1 promoter by EGR-1 which suppresses growth by an autocrine mechanism (Liu\",\"C.\",\"Adamson\",\"E.\",\"and Mercola\",\"D. (1996) Proc. Natl. Acad. Sci. U. S. A. 93\",\"11831-11836).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"B3GNT9\"],\"tf text\":[\"EGR-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"55, 5054-5062) owing in part to up-regulation of the transforming growth factor (TGF)-beta1 promoter by EGR-1 which suppresses growth by an autocrine mechanism (Liu, C., Adamson, E., and Mercola, D. (1996) Proc. Natl. Acad. Sci. U. S. A. 93, 11831-11836).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "IRF(6)-problem; TG is MIP-2---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IRF6\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"IRF(6)-problem; TG is MIP-2\"],\"sentence\":[\"By deletion analysis of the MIP-2 promoter region\",\"we show that NF-kappaB and c-Jun binding sites are essential for LPS-induced MIP-2 gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|IRF6???---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IRF6\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"IRF6???\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By deletion analysis of the MIP-2 promoter region, we show that NF-kappaB and c-Jun binding sites are essential for LPS-induced MIP-2 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "IRF(6)-problem; TG is MIP-2---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IRF6\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"IRF(6)-problem; TG is MIP-2\"],\"sentence\":[\"By deletion analysis of the MIP-2 promoter region\",\"we show that NF-kappaB and c-Jun binding sites are essential for LPS-induced MIP-2 gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|IRF6???---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IRF6\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"IRF6???\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By deletion analysis of the MIP-2 promoter region, we show that NF-kappaB and c-Jun binding sites are essential for LPS-induced MIP-2 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "VHL not target---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"VHL\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"pVHL\"],\"consensus\":[\"N\"],\"martin_comments\":[\"VHL not target\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In conclusion, we show the first evidence of a novel mechanism of degradation of HIF-1alpha at normoxia that involves pVHL but is not mediated by PHDs 1-3 or by degradation boxes surrounding Pro(402) and Pro(563).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "VHL not target---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"VHL\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"pVHL\"],\"consensus\":[\"N\"],\"martin_comments\":[\"VHL not target\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In conclusion, we show the first evidence of a novel mechanism of degradation of HIF-1alpha at normoxia that involves pVHL but is not mediated by PHDs 1-3 or by degradation boxes surrounding Pro(402) and Pro(563).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 332 (char 331)\n",
      "{\"tf associated gene name\":[\"EZH2\"],\"tg associated gene name\":[\"ME3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"EZH2 suppression resulted in a generalized loss of H3K27me3 as well as increase in H3 acetylation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EZH2\"],\"tg (associated gene name)\":[\"ME3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"EZH2 suppression resulted in a generalized loss of H3K27me3 as well as increase in H3 acetylation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"EZH2\"],\"tg associated gene name\":[\"ME3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"EZH2 suppression resulted in a generalized loss of H3K27me3 as well as increase in H3 acetylation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EZH2\"],\"tg (associated gene name)\":[\"ME3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"EZH2 suppression resulted in a generalized loss of H3K27me3 as well as increase in H3 acetylation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "may be correct---{\"tf associated gene name\":[\"NFE2\"],\"tg associated gene name\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n?\"],\"comment\":[\"may be correct\"],\"sentence\":[\"An analysis of the immediate promoter of the human erythroid ALAS gene revealed several putative erythroid-specific cis-acting elements including both a GATA-1 and an NF-E2 binding site.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|ALAS gene?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"ALAS gene?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"An analysis of the immediate promoter of the human erythroid ALAS gene revealed several putative erythroid-specific cis-acting elements including both a GATA-1 and an NF-E2 binding site.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "may be correct---{\"tf associated gene name\":[\"NFE2\"],\"tg associated gene name\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n?\"],\"comment\":[\"may be correct\"],\"sentence\":[\"An analysis of the immediate promoter of the human erythroid ALAS gene revealed several putative erythroid-specific cis-acting elements including both a GATA-1 and an NF-E2 binding site.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|ALAS gene?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"ALAS gene?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"An analysis of the immediate promoter of the human erythroid ALAS gene revealed several putative erythroid-specific cis-acting elements including both a GATA-1 and an NF-E2 binding site.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is proglucagon---{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"SLC2A1\"],\"tf text\":[\"cdx-2/3\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is proglucagon\"],\"sentence\":[\"We show here that cdx-2/3 activates the proglucagon promoter in both islet (InR1-G9) and enteroendocrine (STC-1 and GLUTag) cell lines.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"SLC2A1\"],\"tf text\":[\"cdx-2/3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show here that cdx-2/3 activates the proglucagon promoter in both islet (InR1-G9) and enteroendocrine (STC-1 and GLUTag) cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is proglucagon---{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"SLC2A1\"],\"tf text\":[\"cdx-2/3\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is proglucagon\"],\"sentence\":[\"We show here that cdx-2/3 activates the proglucagon promoter in both islet (InR1-G9) and enteroendocrine (STC-1 and GLUTag) cell lines.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CDX2\"],\"tg (associated gene name)\":[\"SLC2A1\"],\"tf text\":[\"cdx-2/3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show here that cdx-2/3 activates the proglucagon promoter in both islet (InR1-G9) and enteroendocrine (STC-1 and GLUTag) cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF no PAX---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX8\"],\"tg (associated gene name)\":[\"PAX8\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF no PAX\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These results indicate that TGF-beta1 decreases Pax-8 mRNA levels as well as Pax-8 DNA-binding activity, which, at least in part, seems to be involved in the TGF-beta1-induced suppression of TG gene expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF no PAX---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX8\"],\"tg (associated gene name)\":[\"PAX8\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF no PAX\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These results indicate that TGF-beta1 decreases Pax-8 mRNA levels as well as Pax-8 DNA-binding activity, which, at least in part, seems to be involved in the TGF-beta1-induced suppression of TG gene expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 475 (char 474)\n",
      "{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"SOCS3\"],\"tf text\":[\"PPARgamma\"],\"tg text\":[\"SOCS-3\"],\"1\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"However\",\"acute CT-1 treatment caused an increase in SOCS-3 mRNA in adipocytes and a transient decrease in peroxisome proliferator-activated receptor gamma (PPARgamma) mRNA that was regulated by the binding of STAT1 to the PPARgamma2 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"SOCS3\"],\"tf text\":[\"PPARgamma\"],\"tg text\":[\"SOCS-3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, acute CT-1 treatment caused an increase in SOCS-3 mRNA in adipocytes and a transient decrease in peroxisome proliferator-activated receptor gamma (PPARgamma) mRNA that was regulated by the binding of STAT1 to the PPARgamma2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"SOCS3\"],\"tf text\":[\"PPARgamma\"],\"tg text\":[\"SOCS-3\"],\"1\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"However\",\"acute CT-1 treatment caused an increase in SOCS-3 mRNA in adipocytes and a transient decrease in peroxisome proliferator-activated receptor gamma (PPARgamma) mRNA that was regulated by the binding of STAT1 to the PPARgamma2 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"SOCS3\"],\"tf text\":[\"PPARgamma\"],\"tg text\":[\"SOCS-3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, acute CT-1 treatment caused an increase in SOCS-3 mRNA in adipocytes and a transient decrease in peroxisome proliferator-activated receptor gamma (PPARgamma) mRNA that was regulated by the binding of STAT1 to the PPARgamma2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "DLST?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"DLST\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"DLST?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Transient transfections of CDK4, Rad51, E2F-DP1, p21 and Stat3 increased mRNA expression from the endogenous Sp1 gene in HeLa cells whereas overexpression of NF kappaB, and p53 decreased Sp1 mRNA levels.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "DLST?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"DLST\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"DLST?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Transient transfections of CDK4, Rad51, E2F-DP1, p21 and Stat3 increased mRNA expression from the endogenous Sp1 gene in HeLa cells whereas overexpression of NF kappaB, and p53 decreased Sp1 mRNA levels.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not named---{\"tf associated gene name\":[\"NKX3-1\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF not named\"],\"sentence\":[\"The aim of this study was to determine the effect of chronic ethanol feeding on acetylation of histone H3 at lysine 9 (H3-Lys9) at promoter and coding regions of genes for class I alcohol dehydrogenase (ADH I)\",\"inducible nitric oxide synthase (iNOS)\",\"Bax\",\"p21\",\"c-met and hepatocyte growth factor in the rat liver.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX3-1\"],\"tg (associated gene name)\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The aim of this study was to determine the effect of chronic ethanol feeding on acetylation of histone H3 at lysine 9 (H3-Lys9) at promoter and coding regions of genes for class I alcohol dehydrogenase (ADH I), inducible nitric oxide synthase (iNOS), Bax, p21, c-met and hepatocyte growth factor in the rat liver.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF not named---{\"tf associated gene name\":[\"NKX3-1\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF not named\"],\"sentence\":[\"The aim of this study was to determine the effect of chronic ethanol feeding on acetylation of histone H3 at lysine 9 (H3-Lys9) at promoter and coding regions of genes for class I alcohol dehydrogenase (ADH I)\",\"inducible nitric oxide synthase (iNOS)\",\"Bax\",\"p21\",\"c-met and hepatocyte growth factor in the rat liver.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX3-1\"],\"tg (associated gene name)\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The aim of this study was to determine the effect of chronic ethanol feeding on acetylation of histone H3 at lysine 9 (H3-Lys9) at promoter and coding regions of genes for class I alcohol dehydrogenase (ADH I), inducible nitric oxide synthase (iNOS), Bax, p21, c-met and hepatocyte growth factor in the rat liver.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is maspin---{\"tf associated gene name\":[\"TP63\"],\"tg associated gene name\":[\"TSC1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is maspin\"],\"sentence\":[\"In summary\",\"we show that TAp63gamma can substitute for p53 in transcriptional activation of the maspin tumor suppressor gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|maspin tumour suppressor---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP63\"],\"tg (associated gene name)\":[\"TSC1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"maspin tumour suppressor\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In summary, we show that TAp63gamma can substitute for p53 in transcriptional activation of the maspin tumor suppressor gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is maspin---{\"tf associated gene name\":[\"TP63\"],\"tg associated gene name\":[\"TSC1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is maspin\"],\"sentence\":[\"In summary\",\"we show that TAp63gamma can substitute for p53 in transcriptional activation of the maspin tumor suppressor gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|maspin tumour suppressor---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP63\"],\"tg (associated gene name)\":[\"TSC1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"maspin tumour suppressor\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In summary, we show that TAp63gamma can substitute for p53 in transcriptional activation of the maspin tumor suppressor gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped roles?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RARA\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[\"CREB-binding protein\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Our previous study indicated that the transcriptional activity of RA receptor (RAR) is enhanced by association of MED25 with CREB-binding protein (CBP) through the PTOV domain, which is also present in prostate tumor over-expressed protein 1 (PTOV1).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped roles?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RARA\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[\"CREB-binding protein\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Our previous study indicated that the transcriptional activity of RA receptor (RAR) is enhanced by association of MED25 with CREB-binding protein (CBP) through the PTOV domain, which is also present in prostate tumor over-expressed protein 1 (PTOV1).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "most likely activation (inferred from \"counter-represses)---{\"tf associated gene name\":[\"YBX1\"],\"tg associated gene name\":[\"COL1A2\"],\"tf text\":[\"YB-1\"],\"tg text\":[\"COL1A2\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"most likely activation (inferred from \\\"counter-represses)\"],\"sentence\":[\"Here we report that YB-1 counter-represses TGF-beta-stimulated COL1A2 transcription by interfering with Smad3 bound to the upstream sequence around -265 and subsequently by interrupting the Smad3-p300 interaction.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "most likely activation (inferred from \"counter-represses)---{\"tf associated gene name\":[\"YBX1\"],\"tg associated gene name\":[\"COL1A2\"],\"tf text\":[\"YB-1\"],\"tg text\":[\"COL1A2\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"most likely activation (inferred from \\\"counter-represses)\"],\"sentence\":[\"Here we report that YB-1 counter-represses TGF-beta-stimulated COL1A2 transcription by interfering with Smad3 bound to the upstream sequence around -265 and subsequently by interrupting the Smad3-p300 interaction.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"YBX1\"],\"target gene (associated gene name)\":[\"COL1A2\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Here we report that YB-1 counter-represses TGF-beta-stimulated COL1A2 transcription by interfering with Smad3 bound to the upstream sequence around -265 and subsequently by interrupting the Smad3-p300 interaction.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"YBX1\"],\"tg (associated gene name)\":[\"COL1A2\"],\"tf text\":[\"YB-1\"],\"tg text\":[\"COL1A2\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we report that YB-1 counter-represses TGF-beta-stimulated COL1A2 transcription by interfering with Smad3 bound to the upstream sequence around -265 and subsequently by interrupting the Smad3-p300 interaction.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  IL-15Ralpha---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is  IL-15Ralpha\"],\"sentence\":[\"Furthermore\",\"using promoter deletion constructs and gel shift analysis\",\"we defined a functional NF-kappaB-binding motif in the human IL-15Ralpha promoter\",\"suggesting that Tax activation of IL-15Ralpha is due\",\"in part\",\"to the induction of NF-kappaB.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, using promoter deletion constructs and gel shift analysis, we defined a functional NF-kappaB-binding motif in the human IL-15Ralpha promoter, suggesting that Tax activation of IL-15Ralpha is due, in part, to the induction of NF-kappaB.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is  IL-15Ralpha---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is  IL-15Ralpha\"],\"sentence\":[\"Furthermore\",\"using promoter deletion constructs and gel shift analysis\",\"we defined a functional NF-kappaB-binding motif in the human IL-15Ralpha promoter\",\"suggesting that Tax activation of IL-15Ralpha is due\",\"in part\",\"to the induction of NF-kappaB.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, using promoter deletion constructs and gel shift analysis, we defined a functional NF-kappaB-binding motif in the human IL-15Ralpha promoter, suggesting that Tax activation of IL-15Ralpha is due, in part, to the induction of NF-kappaB.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 384 (char 383)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"CCND1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Results from deletion and mutant analysis indicate that Pak1 regulates cyclin D1 transcription by means of an NF-kappaB-dependent pathway.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|bit indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"bit indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Results from deletion and mutant analysis indicate that Pak1 regulates cyclin D1 transcription by means of an NF-kappaB-dependent pathway.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Results from deletion and mutant analysis indicate that Pak1 regulates cyclin D1 transcription by means of an NF-kappaB-dependent pathway.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"CCND1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Results from deletion and mutant analysis indicate that Pak1 regulates cyclin D1 transcription by means of an NF-kappaB-dependent pathway.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|bit indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"bit indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Results from deletion and mutant analysis indicate that Pak1 regulates cyclin D1 transcription by means of an NF-kappaB-dependent pathway.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is bcl-2---{\"tf associated gene name\":[\"IRF8\"],\"tg associated gene name\":[\"BCR\"],\"tf text\":[\"ICSBP\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is bcl-2\"],\"sentence\":[\"Collectively\",\"our findings suggest that ICSBP antagonizes BCR/ABL by down-regulation of bcl-2 and implicates a central role for ICSBP in the pathogenesis of CML\",\"as well as a therapeutic target to overcome drug resistance in bcl-2-dependent tumors.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|BCR tg?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF8\"],\"tg (associated gene name)\":[\"BCR\"],\"tf text\":[\"ICSBP\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"BCR tg?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Collectively, our findings suggest that ICSBP antagonizes BCR/ABL by down-regulation of bcl-2 and implicates a central role for ICSBP in the pathogenesis of CML, as well as a therapeutic target to overcome drug resistance in bcl-2-dependent tumors.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is bcl-2---{\"tf associated gene name\":[\"IRF8\"],\"tg associated gene name\":[\"BCR\"],\"tf text\":[\"ICSBP\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is bcl-2\"],\"sentence\":[\"Collectively\",\"our findings suggest that ICSBP antagonizes BCR/ABL by down-regulation of bcl-2 and implicates a central role for ICSBP in the pathogenesis of CML\",\"as well as a therapeutic target to overcome drug resistance in bcl-2-dependent tumors.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|BCR tg?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF8\"],\"tg (associated gene name)\":[\"BCR\"],\"tf text\":[\"ICSBP\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"BCR tg?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Collectively, our findings suggest that ICSBP antagonizes BCR/ABL by down-regulation of bcl-2 and implicates a central role for ICSBP in the pathogenesis of CML, as well as a therapeutic target to overcome drug resistance in bcl-2-dependent tumors.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 394 (char 393)\n",
      "{\"transcription factor (associated gene name)\":[\"SMAD1\"],\"target gene (associated gene name)\":[\"MMP2\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Bone morphogenetic proteins induce pancreatic cancer cell invasiveness through a Smad1-dependent mechanism that involves matrix metalloproteinase-2.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|involves too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD1\"],\"tg (associated gene name)\":[\"MMP2\"],\"tf text\":[\"Smad1\"],\"tg text\":[\"matrix metalloproteinase-2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"involves too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Bone morphogenetic proteins induce pancreatic cancer cell invasiveness through a Smad1-dependent mechanism that involves matrix metalloproteinase-2.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"SMAD1\"],\"target gene (associated gene name)\":[\"MMP2\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Bone morphogenetic proteins induce pancreatic cancer cell invasiveness through a Smad1-dependent mechanism that involves matrix metalloproteinase-2.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|involves too indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMAD1\"],\"tg (associated gene name)\":[\"MMP2\"],\"tf text\":[\"Smad1\"],\"tg text\":[\"matrix metalloproteinase-2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"involves too indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Bone morphogenetic proteins induce pancreatic cancer cell invasiveness through a Smad1-dependent mechanism that involves matrix metalloproteinase-2.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|no TG mentioned---{\"tf associated gene name\":[\"IRF2\"],\"tg associated gene name\":[\"GHRHR\"],\"tf text\":[\"IRF-2\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TG mentioned\"],\"sentence\":[\"Interferon regulatory factor-2 (IRF-2)\",\"an antagonistic transcriptional repressor of IRF-1\",\"evidences oncogenic potential\",\"but little is currently known regarding the mechanism underlying the oncogenic activities of IRF-2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF2\"],\"tg (associated gene name)\":[\"GHRHR\"],\"tf text\":[\"IRF-2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Interferon regulatory factor-2 (IRF-2), an antagonistic transcriptional repressor of IRF-1, evidences oncogenic potential, but little is currently known regarding the mechanism underlying the oncogenic activities of IRF-2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE|no TG mentioned---{\"tf associated gene name\":[\"IRF2\"],\"tg associated gene name\":[\"GHRHR\"],\"tf text\":[\"IRF-2\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TG mentioned\"],\"sentence\":[\"Interferon regulatory factor-2 (IRF-2)\",\"an antagonistic transcriptional repressor of IRF-1\",\"evidences oncogenic potential\",\"but little is currently known regarding the mechanism underlying the oncogenic activities of IRF-2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF2\"],\"tg (associated gene name)\":[\"GHRHR\"],\"tf text\":[\"IRF-2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Interferon regulatory factor-2 (IRF-2), an antagonistic transcriptional repressor of IRF-1, evidences oncogenic potential, but little is currently known regarding the mechanism underlying the oncogenic activities of IRF-2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is DREF---{\"tf associated gene name\":[\"ZNF699\"],\"tg associated gene name\":[\"PCNA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\",\"s?\"],\"negation\":[],\"comment\":[\"TF is DREF\"],\"sentence\":[\"These results suggest that the B. mori DREF homolog binds to DRE to regulate transcription of the PCNA gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not dref homolog---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF699\"],\"tg (associated gene name)\":[\"PCNA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not dref homolog\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results suggest that the B. mori DREF homolog binds to DRE to regulate transcription of the PCNA gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is DREF---{\"tf associated gene name\":[\"ZNF699\"],\"tg associated gene name\":[\"PCNA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\",\"s?\"],\"negation\":[],\"comment\":[\"TF is DREF\"],\"sentence\":[\"These results suggest that the B. mori DREF homolog binds to DRE to regulate transcription of the PCNA gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not dref homolog---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF699\"],\"tg (associated gene name)\":[\"PCNA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not dref homolog\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results suggest that the B. mori DREF homolog binds to DRE to regulate transcription of the PCNA gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "EIF3K-proble; this IS AUTOREGULATION---{\"tf associated gene name\":[\"MYOG\"],\"tg associated gene name\":[\"EIF3K\"],\"tf text\":[\"myogenin\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"EIF3K-proble; this IS AUTOREGULATION\"],\"sentence\":[\"The muscle-specific helix-loop-helix proteins MyoD and myogenin have been shown to positively autoregulate their own expression and to cross-activate one another's expression following transfection into a variety of nonmyogenic cell lines.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not 'muscle specific' but MYOG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYOG\"],\"tg (associated gene name)\":[\"EIF3K\"],\"tf text\":[\"myogenin\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not 'muscle specific' but MYOG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The muscle-specific helix-loop-helix proteins MyoD and myogenin have been shown to positively autoregulate their own expression and to cross-activate one another's expression following transfection into a variety of nonmyogenic cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "EIF3K-proble; this IS AUTOREGULATION---{\"tf associated gene name\":[\"MYOG\"],\"tg associated gene name\":[\"EIF3K\"],\"tf text\":[\"myogenin\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"EIF3K-proble; this IS AUTOREGULATION\"],\"sentence\":[\"The muscle-specific helix-loop-helix proteins MyoD and myogenin have been shown to positively autoregulate their own expression and to cross-activate one another's expression following transfection into a variety of nonmyogenic cell lines.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not 'muscle specific' but MYOG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYOG\"],\"tg (associated gene name)\":[\"EIF3K\"],\"tf text\":[\"myogenin\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not 'muscle specific' but MYOG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The muscle-specific helix-loop-helix proteins MyoD and myogenin have been shown to positively autoregulate their own expression and to cross-activate one another's expression following transfection into a variety of nonmyogenic cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "here CAT is catalase---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[\"here CAT is catalase\"],\"sentence\":[\"The results suggest a novel regulating influence of p53 on catalase in the retina--more specifically in the RGC--and an influence of p53 on the susceptibility of the cell to oxidative-signaling-induced apoptosis\",\"which could implicate p53 as a potential neuroprotectant for the RGC.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "here CAT is catalase---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[\"here CAT is catalase\"],\"sentence\":[\"The results suggest a novel regulating influence of p53 on catalase in the retina--more specifically in the RGC--and an influence of p53 on the susceptibility of the cell to oxidative-signaling-induced apoptosis\",\"which could implicate p53 as a potential neuroprotectant for the RGC.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"CAT\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The results suggest a novel regulating influence of p53 on catalase in the retina--more specifically in the RGC--and an influence of p53 on the susceptibility of the cell to oxidative-signaling-induced apoptosis\",\"which could implicate p53 as a potential neuroprotectant for the RGC.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results suggest a novel regulating influence of p53 on catalase in the retina--more specifically in the RGC--and an influence of p53 on the susceptibility of the cell to oxidative-signaling-induced apoptosis, which could implicate p53 as a potential neuroprotectant for the RGC.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 371 (char 370)\n",
      "{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"CTCFL\"],\"tf text\":[\"OCT4\"],\"tg text\":[\"CTCFL\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"DNA methylation appears the crucial mechanism in the control of CTCFL transcription\",\"but less decisive in that of OCT4.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|independent statements---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"CTCFL\"],\"tf text\":[\"Oct-04\"],\"tg text\":[\"CTCFL\"],\"consensus\":[\"N\"],\"martin_comments\":[\"independent statements\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"DNA methylation appears the crucial mechanism in the control of CTCFL transcription, but less decisive in that of OCT4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"CTCFL\"],\"tf text\":[\"OCT4\"],\"tg text\":[\"CTCFL\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"DNA methylation appears the crucial mechanism in the control of CTCFL transcription\",\"but less decisive in that of OCT4.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|independent statements---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"CTCFL\"],\"tf text\":[\"Oct-04\"],\"tg text\":[\"CTCFL\"],\"consensus\":[\"N\"],\"martin_comments\":[\"independent statements\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"DNA methylation appears the crucial mechanism in the control of CTCFL transcription, but less decisive in that of OCT4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 441 (char 440)\n",
      "{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"STAT5B\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"not true\":[],\"comment\":[],\"sentence\":[\"The PPAR response elements regulatory element of STAT5A exists at a promoter region ranging from -346 to -101\",\"and the CCAAT/enhancer-binding protein (C/EBP) regulatory element is located at -196 to -118 of the STAT5B promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"STAT5B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The PPAR response elements regulatory element of STAT5A exists at a promoter region ranging from -346 to -101, and the CCAAT/enhancer-binding protein (C/EBP) regulatory element is located at -196 to -118 of the STAT5B promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"STAT5B\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"not true\":[],\"comment\":[],\"sentence\":[\"The PPAR response elements regulatory element of STAT5A exists at a promoter region ranging from -346 to -101\",\"and the CCAAT/enhancer-binding protein (C/EBP) regulatory element is located at -196 to -118 of the STAT5B promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"STAT5B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The PPAR response elements regulatory element of STAT5A exists at a promoter region ranging from -346 to -101, and the CCAAT/enhancer-binding protein (C/EBP) regulatory element is located at -196 to -118 of the STAT5B promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped|REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"RPL11\"],\"tf text\":[\"c-Myc\"],\"tg text\":[\"Ribosomal protein L11\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Ribosomal protein L11 recruits miR-24/miRISC to repress c-Myc expression in response to ribosomal stress.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped|REVERSE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"RPL11\"],\"tf text\":[\"c-Myc\"],\"tg text\":[\"Ribosomal protein L11\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[\"REVERSE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Ribosomal protein L11 recruits miR-24/miRISC to repress c-Myc expression in response to ribosomal stress.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "other TF---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"TERT\"],\"tf text\":[],\"tg text\":[\"telomerase reverse transcriptase\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"other TF\"],\"sentence\":[\"While exogenous EGF failed to activate telomerase in normal ovarian surface epithelial cells\",\"in cancer cells we herein report that: exogenous EGF activates telomerase activity and human telomerase reverse transcriptase gene (hTERT) transcription; EGF-induced telomerase activity is ERK 1/2-dependent; EGF targets Sp1 and c-Myc binding sites within the core region of the hTERT promoter; and proline-rich tyrosine kinase 2 (Pyk2) is a key mediator of EGF-mediated telomerase activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TXK\"],\"tg (associated gene name)\":[\"TERT\"],\"tf text\":[],\"tg text\":[\"telomerase reverse transcriptase\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"While exogenous EGF failed to activate telomerase in normal ovarian surface epithelial cells, in cancer cells we herein report that: exogenous EGF activates telomerase activity and human telomerase reverse transcriptase gene (hTERT) transcription; EGF-induced telomerase activity is ERK 1/2-dependent; EGF targets Sp1 and c-Myc binding sites within the core region of the hTERT promoter; and proline-rich tyrosine kinase 2 (Pyk2) is a key mediator of EGF-mediated telomerase activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "other TF---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"TERT\"],\"tf text\":[],\"tg text\":[\"telomerase reverse transcriptase\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"other TF\"],\"sentence\":[\"While exogenous EGF failed to activate telomerase in normal ovarian surface epithelial cells\",\"in cancer cells we herein report that: exogenous EGF activates telomerase activity and human telomerase reverse transcriptase gene (hTERT) transcription; EGF-induced telomerase activity is ERK 1/2-dependent; EGF targets Sp1 and c-Myc binding sites within the core region of the hTERT promoter; and proline-rich tyrosine kinase 2 (Pyk2) is a key mediator of EGF-mediated telomerase activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TXK\"],\"tg (associated gene name)\":[\"TERT\"],\"tf text\":[],\"tg text\":[\"telomerase reverse transcriptase\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"While exogenous EGF failed to activate telomerase in normal ovarian surface epithelial cells, in cancer cells we herein report that: exogenous EGF activates telomerase activity and human telomerase reverse transcriptase gene (hTERT) transcription; EGF-induced telomerase activity is ERK 1/2-dependent; EGF targets Sp1 and c-Myc binding sites within the core region of the hTERT promoter; and proline-rich tyrosine kinase 2 (Pyk2) is a key mediator of EGF-mediated telomerase activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "NOTCH-problem|TG is leptin---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"NOTCH-problem\",\"TG is leptin\"],\"sentence\":[\"Cotransfection of p300 with p42C/EBPalpha results in a synergistic increase in leptin promoter activity\",\"indicating that p300 acts as a transcriptional coactivator of C/EBPalpha.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|leptin TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"leptin TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Cotransfection of p300 with p42C/EBPalpha results in a synergistic increase in leptin promoter activity, indicating that p300 acts as a transcriptional coactivator of C/EBPalpha.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "NOTCH-problem|TG is leptin---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"NOTCH-problem\",\"TG is leptin\"],\"sentence\":[\"Cotransfection of p300 with p42C/EBPalpha results in a synergistic increase in leptin promoter activity\",\"indicating that p300 acts as a transcriptional coactivator of C/EBPalpha.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|leptin TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[\"C/EBPalpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"leptin TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Cotransfection of p300 with p42C/EBPalpha results in a synergistic increase in leptin promoter activity, indicating that p300 acts as a transcriptional coactivator of C/EBPalpha.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 686 (char 685)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKBID\"],\"target gene (associated gene name)\":[\"PTGS2\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Moreover\",\"IFN-gamma induction of the COX-2 promoter was also strongly reduced by transfection of plasmid encoding the NF-kappaB inhibitor\",\"IkappaBalpha.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.922652\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"152\"],\"sentence pair density\":[\"0.0108695652173913\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKBID\"],\"tg (associated gene name)\":[\"PTGS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Moreover, IFN-gamma induction of the COX-2 promoter was also strongly reduced by transfection of plasmid encoding the NF-kappaB inhibitor, IkappaBalpha.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"NFKBID\"],\"target gene (associated gene name)\":[\"PTGS2\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Moreover\",\"IFN-gamma induction of the COX-2 promoter was also strongly reduced by transfection of plasmid encoding the NF-kappaB inhibitor\",\"IkappaBalpha.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.922652\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"152\"],\"sentence pair density\":[\"0.0108695652173913\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKBID\"],\"tg (associated gene name)\":[\"PTGS2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Moreover, IFN-gamma induction of the COX-2 promoter was also strongly reduced by transfection of plasmid encoding the NF-kappaB inhibitor, IkappaBalpha.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is zebrafish IFN3---{\"tf associated gene name\":[\"IRF7\"],\"tg associated gene name\":[\"IFNA1\"],\"tf text\":[\"IRF7\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\",\"n\"],\"comment\":[\"TG is zebrafish IFN3\"],\"sentence\":[\"Whereas fish IFN genes cannot be classified into IFN-?? or IFN-??\",\"zebrafish IFN1 is primarily regulated by IRF3\",\"thereby resembling that of IFN-??\",\"and zebrafish IFN3 is regulated by IRF7\",\"thereby resembling of those of IFN-??s.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|IFN3 not IFNA1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF7\"],\"tg (associated gene name)\":[\"IFNA1\"],\"tf text\":[\"IRF7\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"IFN3 not IFNA1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Whereas fish IFN genes cannot be classified into IFN- or IFN-, zebrafish IFN1 is primarily regulated by IRF3, thereby resembling that of IFN-, and zebrafish IFN3 is regulated by IRF7, thereby resembling of those of IFN-s.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is zebrafish IFN3---{\"tf associated gene name\":[\"IRF7\"],\"tg associated gene name\":[\"IFNA1\"],\"tf text\":[\"IRF7\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\",\"n\"],\"comment\":[\"TG is zebrafish IFN3\"],\"sentence\":[\"Whereas fish IFN genes cannot be classified into IFN-?? or IFN-??\",\"zebrafish IFN1 is primarily regulated by IRF3\",\"thereby resembling that of IFN-??\",\"and zebrafish IFN3 is regulated by IRF7\",\"thereby resembling of those of IFN-??s.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|IFN3 not IFNA1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF7\"],\"tg (associated gene name)\":[\"IFNA1\"],\"tf text\":[\"IRF7\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"IFN3 not IFNA1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Whereas fish IFN genes cannot be classified into IFN- or IFN-, zebrafish IFN1 is primarily regulated by IRF3, thereby resembling that of IFN-, and zebrafish IFN3 is regulated by IRF7, thereby resembling of those of IFN-s.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Mucin---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"SLC13A2\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Mucin\"],\"sentence\":[\"Mucin gene transcription is also regulated by ATF-1\",\"CREB and RAR-alpha transcription factors.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Mucin---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"SLC13A2\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Mucin\"],\"sentence\":[\"Mucin gene transcription is also regulated by ATF-1\",\"CREB and RAR-alpha transcription factors.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CREB1\"],\"target gene (associated gene name)\":[\"SLC13A2\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Mucin gene transcription is also regulated by ATF-1\",\"CREB and RAR-alpha transcription factors.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.579652\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[],\"sentence pair density\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=mucin gene---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"SLC13A2\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=mucin gene\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Mucin gene transcription is also regulated by ATF-1, CREB and RAR-alpha transcription factors.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 461 (char 460)\n",
      "{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Taken together with the finding that IL-2 but not PMA protects MLA-E7T from dexamethasone-induced apoptosis\",\"our results indicate that IL-2 does not induce such a protection by repressing the transcriptional activity of the glucocorticoid receptor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together with the finding that IL-2 but not PMA protects MLA-E7T from dexamethasone-induced apoptosis, our results indicate that IL-2 does not induce such a protection by repressing the transcriptional activity of the glucocorticoid receptor.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Taken together with the finding that IL-2 but not PMA protects MLA-E7T from dexamethasone-induced apoptosis\",\"our results indicate that IL-2 does not induce such a protection by repressing the transcriptional activity of the glucocorticoid receptor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together with the finding that IL-2 but not PMA protects MLA-E7T from dexamethasone-induced apoptosis, our results indicate that IL-2 does not induce such a protection by repressing the transcriptional activity of the glucocorticoid receptor.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"MT-ND1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Using promoter-reporter deletion constructs, we identified a distal region with the ability to negatively regulate basal transcription and a proximal region containing an Sp1 (stimulating protein 1) site that confers approx. 75% of basal transcriptional activity.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"MT-ND1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Using promoter-reporter deletion constructs, we identified a distal region with the ability to negatively regulate basal transcription and a proximal region containing an Sp1 (stimulating protein 1) site that confers approx. 75% of basal transcriptional activity.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF (TF normalization very uncertain)---{\"tf associated gene name\":[\"PBX1\"],\"tg associated gene name\":[\"INS-IGF2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"no TF (TF normalization very uncertain)\"],\"sentence\":[\"A PRL-responsive region of the rat insulin-1 promoter was localized between nucleotides -165 and -109.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG just ins---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PBX1\"],\"tg (associated gene name)\":[\"INS-IGF2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG just ins\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"A PRL-responsive region of the rat insulin-1 promoter was localized between nucleotides -165 and -109.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF (TF normalization very uncertain)---{\"tf associated gene name\":[\"PBX1\"],\"tg associated gene name\":[\"INS-IGF2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"no TF (TF normalization very uncertain)\"],\"sentence\":[\"A PRL-responsive region of the rat insulin-1 promoter was localized between nucleotides -165 and -109.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG just ins---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PBX1\"],\"tg (associated gene name)\":[\"INS-IGF2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG just ins\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"A PRL-responsive region of the rat insulin-1 promoter was localized between nucleotides -165 and -109.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE|no TG---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"TNFRSF11A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE\",\"no TG\"],\"sentence\":[\"Moreover\",\"ablation of E2F1 levels caused abrogation of the protective effects of nicotine against cisplatin-induced apoptosis in A549 cells whereas ablation of signal transducer and activator of transcription 3 levels had no effect.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF? TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"TNFRSF11A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF? TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, ablation of E2F1 levels caused abrogation of the protective effects of nicotine against cisplatin-induced apoptosis in A549 cells whereas ablation of signal transducer and activator of transcription 3 levels had no effect.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not GRE|no TG---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"TNFRSF11A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE\",\"no TG\"],\"sentence\":[\"Moreover\",\"ablation of E2F1 levels caused abrogation of the protective effects of nicotine against cisplatin-induced apoptosis in A549 cells whereas ablation of signal transducer and activator of transcription 3 levels had no effect.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF? TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"TNFRSF11A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF? TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, ablation of E2F1 levels caused abrogation of the protective effects of nicotine against cisplatin-induced apoptosis in A549 cells whereas ablation of signal transducer and activator of transcription 3 levels had no effect.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is not TF (chemical?)---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"MIR150\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"regulator is not TF (chemical?)\"],\"sentence\":[\"Our results showed that miR-150 expression was inhibited and the growth inhibition rate of A549 cells was higher in the PR-ASO-150-treated group compared with the control\",\"which indicated that PR-ASO-150 could inhibit A549 cell proliferation by regulating miR-150 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MLXIP from miR?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"MIR150\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MLXIP from miR?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results showed that miR-150 expression was inhibited and the growth inhibition rate of A549 cells was higher in the PR-ASO-150-treated group compared with the control, which indicated that PR-ASO-150 could inhibit A549 cell proliferation by regulating miR-150 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "regulator is not TF (chemical?)---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"MIR150\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"regulator is not TF (chemical?)\"],\"sentence\":[\"Our results showed that miR-150 expression was inhibited and the growth inhibition rate of A549 cells was higher in the PR-ASO-150-treated group compared with the control\",\"which indicated that PR-ASO-150 could inhibit A549 cell proliferation by regulating miR-150 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MLXIP from miR?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MLXIP\"],\"tg (associated gene name)\":[\"MIR150\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MLXIP from miR?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results showed that miR-150 expression was inhibited and the growth inhibition rate of A549 cells was higher in the PR-ASO-150-treated group compared with the control, which indicated that PR-ASO-150 could inhibit A549 cell proliferation by regulating miR-150 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is cyclin D1---{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"HDAC7\"],\"tf text\":[\"HIF-1\"],\"tg text\":[\"HDAC7\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is cyclin D1\"],\"sentence\":[\"Inhibition of histone deacetylase (HDAC) by pretreatment of cells with trichostatin A or specific knockdown of HDAC7 by its shRNA antagonized the suppression of cyclin D1 by HIF-1\",\"suggesting that HDAC7 is required for HIF-1-mediated cyclin D1 downregulation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HDAC7 not TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"HDAC7\"],\"tf text\":[\"HIF-1\"],\"tg text\":[\"HDAC7\"],\"consensus\":[\"N\"],\"martin_comments\":[\"HDAC7 not TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of histone deacetylase (HDAC) by pretreatment of cells with trichostatin A or specific knockdown of HDAC7 by its shRNA antagonized the suppression of cyclin D1 by HIF-1, suggesting that HDAC7 is required for HIF-1-mediated cyclin D1 downregulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is cyclin D1---{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"HDAC7\"],\"tf text\":[\"HIF-1\"],\"tg text\":[\"HDAC7\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is cyclin D1\"],\"sentence\":[\"Inhibition of histone deacetylase (HDAC) by pretreatment of cells with trichostatin A or specific knockdown of HDAC7 by its shRNA antagonized the suppression of cyclin D1 by HIF-1\",\"suggesting that HDAC7 is required for HIF-1-mediated cyclin D1 downregulation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HDAC7 not TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"HDAC7\"],\"tf text\":[\"HIF-1\"],\"tg text\":[\"HDAC7\"],\"consensus\":[\"N\"],\"martin_comments\":[\"HDAC7 not TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of histone deacetylase (HDAC) by pretreatment of cells with trichostatin A or specific knockdown of HDAC7 by its shRNA antagonized the suppression of cyclin D1 by HIF-1, suggesting that HDAC7 is required for HIF-1-mediated cyclin D1 downregulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is SMGA---{\"tf associated gene name\":[\"SRF\"],\"tg associated gene name\":[\"ACTG2\"],\"tf text\":[\"SRF\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is SMGA\"],\"sentence\":[\"SMGA gene transcriptional activity was androgen responsive in these cells and required a segment of the human SMGA promoter containing NKE and SRF (serum response factor) binding elements.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SMGA makes lot of sense as alias---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SRF\"],\"tg (associated gene name)\":[\"ACTG2\"],\"tf text\":[\"SRF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SMGA makes lot of sense as alias\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"SMGA gene transcriptional activity was androgen responsive in these cells and required a segment of the human SMGA promoter containing NKE and SRF (serum response factor) binding elements.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is SMGA---{\"tf associated gene name\":[\"SRF\"],\"tg associated gene name\":[\"ACTG2\"],\"tf text\":[\"SRF\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is SMGA\"],\"sentence\":[\"SMGA gene transcriptional activity was androgen responsive in these cells and required a segment of the human SMGA promoter containing NKE and SRF (serum response factor) binding elements.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SMGA makes lot of sense as alias---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SRF\"],\"tg (associated gene name)\":[\"ACTG2\"],\"tf text\":[\"SRF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SMGA makes lot of sense as alias\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"SMGA gene transcriptional activity was androgen responsive in these cells and required a segment of the human SMGA promoter containing NKE and SRF (serum response factor) binding elements.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 228 (char 227)\n",
      "{\"transcription factor (associated gene name)\":[\"ID1\"],\"target gene (associated gene name)\":[\"CDKN2A\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|ID1 targets the TF AA residues|TOO INDIRECT?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID1\"],\"tg (associated gene name)\":[\"CDKN2A\"],\"tf text\":[],\"tg text\":[\"p16\"],\"consensus\":[\"N\"],\"martin_comments\":[\"ID1 targets the TF AA residues\"],\"astrid_comments\":[\"TOO INDIRECT?\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"From this interaction analysis, the residues of ETS-1 involved in interaction with the p16(INK4a) promoter were found to be targeted by ID1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"ID1\"],\"target gene (associated gene name)\":[\"CDKN2A\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|ID1 targets the TF AA residues|TOO INDIRECT?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID1\"],\"tg (associated gene name)\":[\"CDKN2A\"],\"tf text\":[],\"tg text\":[\"p16\"],\"consensus\":[\"N\"],\"martin_comments\":[\"ID1 targets the TF AA residues\"],\"astrid_comments\":[\"TOO INDIRECT?\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"From this interaction analysis, the residues of ETS-1 involved in interaction with the p16(INK4a) promoter were found to be targeted by ID1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 362 (char 361)\n",
      "{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"HLA-DRB1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"We also observed that c-Jun overexpression in proliferating QM7 cells restored T3R transcriptional activity through a DR4 TRE.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|TG T3R---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"HLA-DRB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG T3R\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We also observed that c-Jun overexpression in proliferating QM7 cells restored T3R transcriptional activity through a DR4 TRE.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"HLA-DRB1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"We also observed that c-Jun overexpression in proliferating QM7 cells restored T3R transcriptional activity through a DR4 TRE.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|TG T3R---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"HLA-DRB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG T3R\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We also observed that c-Jun overexpression in proliferating QM7 cells restored T3R transcriptional activity through a DR4 TRE.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 455 (char 454)\n",
      "{\"transcription factor (associated gene name)\":[\"NR2F2\"],\"target gene (associated gene name)\":[\"KDR\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Taken together\",\"our results implicate COUP-TFII as a critical factor in tumor angiogenesis through regulation of VEGF/VEGFR-2 signaling\",\"suggesting COUP-TFII as a candidate target for antiangiogenic therapy.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|maybe veg too indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR2F2\"],\"tg (associated gene name)\":[\"KDR\"],\"tf text\":[\"COUP-TFII\"],\"tg text\":[\"VEGFR-2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"maybe veg too indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Taken together, our results implicate COUP-TFII as a critical factor in tumor angiogenesis through regulation of VEGF/VEGFR-2 signaling, suggesting COUP-TFII as a candidate target for antiangiogenic therapy.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"NR2F2\"],\"target gene (associated gene name)\":[\"KDR\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Taken together\",\"our results implicate COUP-TFII as a critical factor in tumor angiogenesis through regulation of VEGF/VEGFR-2 signaling\",\"suggesting COUP-TFII as a candidate target for antiangiogenic therapy.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|maybe veg too indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR2F2\"],\"tg (associated gene name)\":[\"KDR\"],\"tf text\":[\"COUP-TFII\"],\"tg text\":[\"VEGFR-2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"maybe veg too indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Taken together, our results implicate COUP-TFII as a critical factor in tumor angiogenesis through regulation of VEGF/VEGFR-2 signaling, suggesting COUP-TFII as a candidate target for antiangiogenic therapy.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "example that GRE can be recognized in sentece stating upstream regulating mechanisms---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"TYR\"],\"tf text\":[\"MITF\"],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"example that GRE can be recognized in sentece stating upstream regulating mechanisms\"],\"sentence\":[\"Taking into account that GSK3beta phosphorylates MITF and increases the ability of MITF to bind its target sequence\",\"our results indicate that activation of GSK3beta by cAMP facilitates MITF binding to the tyrosinase promoter\",\"thereby leading to stimulation of melanogenesis.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "example that GRE can be recognized in sentece stating upstream regulating mechanisms---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"TYR\"],\"tf text\":[\"MITF\"],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"example that GRE can be recognized in sentece stating upstream regulating mechanisms\"],\"sentence\":[\"Taking into account that GSK3beta phosphorylates MITF and increases the ability of MITF to bind its target sequence\",\"our results indicate that activation of GSK3beta by cAMP facilitates MITF binding to the tyrosinase promoter\",\"thereby leading to stimulation of melanogenesis.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MITF\"],\"target gene (associated gene name)\":[\"TYR\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Taking into account that GSK3beta phosphorylates MITF and increases the ability of MITF to bind its target sequence\",\"our results indicate that activation of GSK3beta by cAMP facilitates MITF binding to the tyrosinase promoter\",\"thereby leading to stimulation of melanogenesis.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"TYR\"],\"tf text\":[\"MITF\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taking into account that GSK3beta phosphorylates MITF and increases the ability of MITF to bind its target sequence, our results indicate that activation of GSK3beta by cAMP facilitates MITF binding to the tyrosinase promoter, thereby leading to stimulation of melanogenesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is TSP50|most likely not correct to normalize to NFkB---{\"tf associated gene name\":[\"NKX3-1\"],\"tg associated gene name\":[\"CASP3\"],\"tf text\":[],\"tg text\":[\"caspase-3\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"regulator is TSP50\",\"most likely not correct to normalize to NFkB\"],\"sentence\":[\"Moreover\",\"we found alantolactone could increase the ratio of Bax/Bcl-2\",\"and activate caspase-9 and caspase-3 in the cancer cells with high expression of TSP50\",\"surprisingly\",\"the same effects can also be observed in the same cells just by knockdown of TSP50 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX3-1\"],\"tg (associated gene name)\":[\"CASP3\"],\"tf text\":[],\"tg text\":[\"caspase-3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, we found alantolactone could increase the ratio of Bax/Bcl-2, and activate caspase-9 and caspase-3 in the cancer cells with high expression of TSP50, surprisingly, the same effects can also be observed in the same cells just by knockdown of TSP50 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "regulator is TSP50|most likely not correct to normalize to NFkB---{\"tf associated gene name\":[\"NKX3-1\"],\"tg associated gene name\":[\"CASP3\"],\"tf text\":[],\"tg text\":[\"caspase-3\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"regulator is TSP50\",\"most likely not correct to normalize to NFkB\"],\"sentence\":[\"Moreover\",\"we found alantolactone could increase the ratio of Bax/Bcl-2\",\"and activate caspase-9 and caspase-3 in the cancer cells with high expression of TSP50\",\"surprisingly\",\"the same effects can also be observed in the same cells just by knockdown of TSP50 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX3-1\"],\"tg (associated gene name)\":[\"CASP3\"],\"tf text\":[],\"tg text\":[\"caspase-3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, we found alantolactone could increase the ratio of Bax/Bcl-2, and activate caspase-9 and caspase-3 in the cancer cells with high expression of TSP50, surprisingly, the same effects can also be observed in the same cells just by knockdown of TSP50 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA1\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Transcription factor GATA-1 is required for the terminal differentiation of both the primitive and definitive erythroid cell lineages, and yet the regulatory mechanisms of GATA-1 itself are not well understood.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA1\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Transcription factor GATA-1 is required for the terminal differentiation of both the primitive and definitive erythroid cell lineages, and yet the regulatory mechanisms of GATA-1 itself are not well understood.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG most likely not correctly normalized---{\"tf associated gene name\":[\"NFKB2\"],\"tg associated gene name\":[\"ATP8A1\"],\"tf text\":[\"H2TF1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG most likely not correctly normalized\"],\"sentence\":[\"In this paper we show that both NF-kappa B and KBF1 (p50 dimer) binding activity to the H2TF1 element in the Class I promoter is reduced in Ad12-13S-E1A-transformed cells compared to Ad5E1- or Ad12-12S-E1A-transformed cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB2\"],\"tg (associated gene name)\":[\"ATP8A1\"],\"tf text\":[\"H2TF1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this paper we show that both NF-kappa B and KBF1 (p50 dimer) binding activity to the H2TF1 element in the Class I promoter is reduced in Ad12-13S-E1A-transformed cells compared to Ad5E1- or Ad12-12S-E1A-transformed cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG most likely not correctly normalized---{\"tf associated gene name\":[\"NFKB2\"],\"tg associated gene name\":[\"ATP8A1\"],\"tf text\":[\"H2TF1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG most likely not correctly normalized\"],\"sentence\":[\"In this paper we show that both NF-kappa B and KBF1 (p50 dimer) binding activity to the H2TF1 element in the Class I promoter is reduced in Ad12-13S-E1A-transformed cells compared to Ad5E1- or Ad12-12S-E1A-transformed cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB2\"],\"tg (associated gene name)\":[\"ATP8A1\"],\"tf text\":[\"H2TF1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In this paper we show that both NF-kappa B and KBF1 (p50 dimer) binding activity to the H2TF1 element in the Class I promoter is reduced in Ad12-13S-E1A-transformed cells compared to Ad5E1- or Ad12-12S-E1A-transformed cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TRE is: GATA-6 selectively inhibits telokin expression---{\"tf associated gene name\":[\"SRF\"],\"tg associated gene name\":[\"MYOCD\"],\"tf text\":[],\"tg text\":[\"myocardin\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TRE is: GATA-6 selectively inhibits telokin expression\"],\"sentence\":[\"These data suggest that GATA-6 selectively inhibits telokin expression by triggering the displacement of myocardin from the serum-response factor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TRE is: GATA-6 selectively inhibits telokin expression---{\"tf associated gene name\":[\"SRF\"],\"tg associated gene name\":[\"MYOCD\"],\"tf text\":[],\"tg text\":[\"myocardin\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TRE is: GATA-6 selectively inhibits telokin expression\"],\"sentence\":[\"These data suggest that GATA-6 selectively inhibits telokin expression by triggering the displacement of myocardin from the serum-response factor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SRF\"],\"tg (associated gene name)\":[\"MYOCD\"],\"tf text\":[],\"tg text\":[\"myocardin\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"These data suggest that GATA-6 selectively inhibits telokin expression by triggering the displacement of myocardin from the serum-response factor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is PSA---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[\"tumour necrosis factor alpha\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is PSA\"],\"sentence\":[\"We observed that activation of NF-kappaB by TNFalpha (tumour necrosis factor alpha) inhibited both basal and androgen-stimulated PSA expression\",\"and that this down-regulation occurred at the promoter level\",\"as confirmed by the super-repressor IkappaBalpha (S32A/S36A)\",\"a dominant negative inhibitor of NF-kappaB.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|if anything: flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[\"tumour necrosis factor alpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"if anything: flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We observed that activation of NF-kappaB by TNFalpha (tumour necrosis factor alpha) inhibited both basal and androgen-stimulated PSA expression, and that this down-regulation occurred at the promoter level, as confirmed by the super-repressor IkappaBalpha (S32A/S36A), a dominant negative inhibitor of NF-kappaB.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is PSA---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[\"tumour necrosis factor alpha\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is PSA\"],\"sentence\":[\"We observed that activation of NF-kappaB by TNFalpha (tumour necrosis factor alpha) inhibited both basal and androgen-stimulated PSA expression\",\"and that this down-regulation occurred at the promoter level\",\"as confirmed by the super-repressor IkappaBalpha (S32A/S36A)\",\"a dominant negative inhibitor of NF-kappaB.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|if anything: flipped roles---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[\"tumour necrosis factor alpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"if anything: flipped roles\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We observed that activation of NF-kappaB by TNFalpha (tumour necrosis factor alpha) inhibited both basal and androgen-stimulated PSA expression, and that this down-regulation occurred at the promoter level, as confirmed by the super-repressor IkappaBalpha (S32A/S36A), a dominant negative inhibitor of NF-kappaB.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is STAT5---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"CSN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF is STAT5\"],\"sentence\":[\"Activation of NF-kappaB p50/p65 is regulated in the developing mammary gland and inhibits STAT5-mediated beta-casein gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is STAT5---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"CSN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF is STAT5\"],\"sentence\":[\"Activation of NF-kappaB p50/p65 is regulated in the developing mammary gland and inhibits STAT5-mediated beta-casein gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RELA\"],\"target gene (associated gene name)\":[\"CSN2\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Activation of NF-kappaB p50/p65 is regulated in the developing mammary gland and inhibits STAT5-mediated beta-casein gene expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"CSN2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Activation of NF-kappaB p50/p65 is regulated in the developing mammary gland and inhibits STAT5-mediated beta-casein gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 425 (char 424)\n",
      "{\"transcription factor (associated gene name)\":[\"RARA\"],\"target gene (associated gene name)\":[\"CYP4F2\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results indicate that retinoic acid can regulate CYP4F2 gene activity with RXRalpha heterodimers stimulating while RARalpha functioning to repress CYP4F2 gene expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|english?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RARA\"],\"tg (associated gene name)\":[\"CYP4F2\"],\"tf text\":[\"RARalpha\"],\"tg text\":[\"CYP4F2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"english?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that retinoic acid can regulate CYP4F2 gene activity with RXRalpha heterodimers stimulating while RARalpha functioning to repress CYP4F2 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RARA\"],\"tg associated gene name\":[\"CYP4F2\"],\"tf text\":[\"RARalpha\"],\"tg text\":[\"CYP4F2\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"These results indicate that retinoic acid can regulate CYP4F2 gene activity with RXRalpha heterodimers stimulating while RARalpha functioning to repress CYP4F2 gene expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RARA\"],\"target gene (associated gene name)\":[\"CYP4F2\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results indicate that retinoic acid can regulate CYP4F2 gene activity with RXRalpha heterodimers stimulating while RARalpha functioning to repress CYP4F2 gene expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|english?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RARA\"],\"tg (associated gene name)\":[\"CYP4F2\"],\"tf text\":[\"RARalpha\"],\"tg text\":[\"CYP4F2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"english?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that retinoic acid can regulate CYP4F2 gene activity with RXRalpha heterodimers stimulating while RARalpha functioning to repress CYP4F2 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is betaB2---{\"tf associated gene name\":[\"MAF\"],\"tg associated gene name\":[\"BDKRB2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is betaB2\"],\"sentence\":[\"Although betaB2 promoter activity is enhanced by c-Maf in both in vitro differentiating rat lens fiber cells and CHO cells\",\"deletion of the betaB2 MARE\",\"which was mapped to -143/-123\",\"does not decrease betaB2 promoter activity in lens fiber cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAF\"],\"tg (associated gene name)\":[\"BDKRB2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although betaB2 promoter activity is enhanced by c-Maf in both in vitro differentiating rat lens fiber cells and CHO cells, deletion of the betaB2 MARE, which was mapped to -143/-123, does not decrease betaB2 promoter activity in lens fiber cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is betaB2---{\"tf associated gene name\":[\"MAF\"],\"tg associated gene name\":[\"BDKRB2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is betaB2\"],\"sentence\":[\"Although betaB2 promoter activity is enhanced by c-Maf in both in vitro differentiating rat lens fiber cells and CHO cells\",\"deletion of the betaB2 MARE\",\"which was mapped to -143/-123\",\"does not decrease betaB2 promoter activity in lens fiber cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAF\"],\"tg (associated gene name)\":[\"BDKRB2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Although betaB2 promoter activity is enhanced by c-Maf in both in vitro differentiating rat lens fiber cells and CHO cells, deletion of the betaB2 MARE, which was mapped to -143/-123, does not decrease betaB2 promoter activity in lens fiber cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "rDNA|POL---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"UBTF\"],\"tg (associated gene name)\":[\"POLI\"],\"tf text\":[\"UBF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"rDNA\"],\"astrid_comments\":[\"POL\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"UBF is an HMG box-containing factor that binds to the rDNA promoter and activates Pol I transcription through its acidic carboxy-terminal tail.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "rDNA|POL---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"UBTF\"],\"tg (associated gene name)\":[\"POLI\"],\"tf text\":[\"UBF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"rDNA\"],\"astrid_comments\":[\"POL\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"UBF is an HMG box-containing factor that binds to the rDNA promoter and activates Pol I transcription through its acidic carboxy-terminal tail.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 457 (char 456)\n",
      "{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Furthermore\",\"FSH +/- TGFbeta1 increased ERalpha association with histone acetylases (CBP and SRC-1) and coactivator of peroxisome proliferator-activated receptor gamma (PGC-1alpha)\",\"and MPP/ICI dramatically reduced these interactions.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|proteinlevel interactions---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"proteinlevel interactions\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, FSH +/- TGFbeta1 increased ERalpha association with histone acetylases (CBP and SRC-1) and coactivator of peroxisome proliferator-activated receptor gamma (PGC-1alpha), and MPP/ICI dramatically reduced these interactions.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Furthermore\",\"FSH +/- TGFbeta1 increased ERalpha association with histone acetylases (CBP and SRC-1) and coactivator of peroxisome proliferator-activated receptor gamma (PGC-1alpha)\",\"and MPP/ICI dramatically reduced these interactions.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|proteinlevel interactions---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"proteinlevel interactions\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, FSH +/- TGFbeta1 increased ERalpha association with histone acetylases (CBP and SRC-1) and coactivator of peroxisome proliferator-activated receptor gamma (PGC-1alpha), and MPP/ICI dramatically reduced these interactions.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "Saccharomyces cerevisiae---{\"tf associated gene name\":[\"MAFB\"],\"tg associated gene name\":[\"MT-TL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"Saccharomyces cerevisiae\"],\"sentence\":[\"Upon transfer of Saccharomyces cerevisiae from repressive to favorable growth conditions\",\"CK2 activity is required for the release of Maf1 from Pol III bound to a tRNA gene and for subsequent activation of tRNA transcription.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|PolIII---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAFB\"],\"tg (associated gene name)\":[\"MT-TL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PolIII\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Upon transfer of Saccharomyces cerevisiae from repressive to favorable growth conditions, CK2 activity is required for the release of Maf1 from Pol III bound to a tRNA gene and for subsequent activation of tRNA transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "Saccharomyces cerevisiae---{\"tf associated gene name\":[\"MAFB\"],\"tg associated gene name\":[\"MT-TL1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"Saccharomyces cerevisiae\"],\"sentence\":[\"Upon transfer of Saccharomyces cerevisiae from repressive to favorable growth conditions\",\"CK2 activity is required for the release of Maf1 from Pol III bound to a tRNA gene and for subsequent activation of tRNA transcription.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|PolIII---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAFB\"],\"tg (associated gene name)\":[\"MT-TL1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"PolIII\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Upon transfer of Saccharomyces cerevisiae from repressive to favorable growth conditions, CK2 activity is required for the release of Maf1 from Pol III bound to a tRNA gene and for subsequent activation of tRNA transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 405 (char 404)\n",
      "{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"p\"],\"comment\":[],\"sentence\":[\"Our results suggest dual roles for p300/CBP in Myc regulation: as a Myc coactivator that stabilizes Myc and as an inducer of Myc instability via direct Myc acetylation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest dual roles for p300/CBP in Myc regulation: as a Myc coactivator that stabilizes Myc and as an inducer of Myc instability via direct Myc acetylation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\",\"p\"],\"comment\":[],\"sentence\":[\"Our results suggest dual roles for p300/CBP in Myc regulation: as a Myc coactivator that stabilizes Myc and as an inducer of Myc instability via direct Myc acetylation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest dual roles for p300/CBP in Myc regulation: as a Myc coactivator that stabilizes Myc and as an inducer of Myc instability via direct Myc acetylation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 442 (char 441)\n",
      "{\"transcription factor (associated gene name)\":[\"MITF\"],\"target gene (associated gene name)\":[\"TYR\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The results suggest that Sox10 directly\",\"and Mitf\",\"most likely indirectly\",\"activate the tyrosinase enhancer\",\"underlining the contribution of Sox10 to tyrosinase gene regulation in melanocytes.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|indirectly---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"TYR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"indirectly\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results suggest that Sox10 directly, and Mitf, most likely indirectly, activate the tyrosinase enhancer, underlining the contribution of Sox10 to tyrosinase gene regulation in melanocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"TYR\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"The results suggest that Sox10 directly\",\"and Mitf\",\"most likely indirectly\",\"activate the tyrosinase enhancer\",\"underlining the contribution of Sox10 to tyrosinase gene regulation in melanocytes.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MITF\"],\"target gene (associated gene name)\":[\"TYR\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The results suggest that Sox10 directly\",\"and Mitf\",\"most likely indirectly\",\"activate the tyrosinase enhancer\",\"underlining the contribution of Sox10 to tyrosinase gene regulation in melanocytes.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|indirectly---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"TYR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"indirectly\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results suggest that Sox10 directly, and Mitf, most likely indirectly, activate the tyrosinase enhancer, underlining the contribution of Sox10 to tyrosinase gene regulation in melanocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"PDGFRA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We have identified platelet-derived growth factor receptor alpha (PDGFR??) as an early marker of the PrE lineage and its derivatives in both mouse embryos and ex vivo paradigms of extra-embryonic endoderm (ExEn).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"PDGFRA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We have identified platelet-derived growth factor receptor alpha (PDGFR??) as an early marker of the PrE lineage and its derivatives in both mouse embryos and ex vivo paradigms of extra-embryonic endoderm (ExEn).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 423 (char 422)\n",
      "{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"GADD45A\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Canine p53 protein produced in vitro was shown to recognize and bind to p53-specific DNA targets derived from the p21 and GADD45 promoters and to a consensus p53 binding site.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|CANINE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"GADD45A\"],\"tf text\":[\"p53\"],\"tg text\":[\"GADD45\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[\"CANINE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Canine p53 protein produced in vitro was shown to recognize and bind to p53-specific DNA targets derived from the p21 and GADD45 promoters and to a consensus p53 binding site.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"GADD45A\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Canine p53 protein produced in vitro was shown to recognize and bind to p53-specific DNA targets derived from the p21 and GADD45 promoters and to a consensus p53 binding site.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|CANINE---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"GADD45A\"],\"tf text\":[\"p53\"],\"tg text\":[\"GADD45\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[\"CANINE\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Canine p53 protein produced in vitro was shown to recognize and bind to p53-specific DNA targets derived from the p21 and GADD45 promoters and to a consensus p53 binding site.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 742 (char 741)\n",
      "{\"transcription factor (associated gene name)\":[\"NR5A1\"],\"target gene (associated gene name)\":[\"CTNNB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Thus\",\"we propose that TCF-4N inhibits coactivation by beta-catenin of TCF/LEF transcription factors and potentiates the coactivation by beta-catenin of other transcription factors\",\"such as SF-1 and C/EBPalpha.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.056032\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"209\"],\"sentence pair density\":[\"0.0165289256198347\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR5A1\"],\"tg (associated gene name)\":[\"CTNNB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, we propose that TCF-4N inhibits coactivation by beta-catenin of TCF/LEF transcription factors and potentiates the coactivation by beta-catenin of other transcription factors, such as SF-1 and C/EBPalpha.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR5A1\"],\"tg associated gene name\":[\"CTNNB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Thus\",\"we propose that TCF-4N inhibits coactivation by beta-catenin of TCF/LEF transcription factors and potentiates the coactivation by beta-catenin of other transcription factors\",\"such as SF-1 and C/EBPalpha.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR5A1\"],\"target gene (associated gene name)\":[\"CTNNB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Thus\",\"we propose that TCF-4N inhibits coactivation by beta-catenin of TCF/LEF transcription factors and potentiates the coactivation by beta-catenin of other transcription factors\",\"such as SF-1 and C/EBPalpha.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.056032\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"209\"],\"sentence pair density\":[\"0.0165289256198347\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR5A1\"],\"tg (associated gene name)\":[\"CTNNB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, we propose that TCF-4N inhibits coactivation by beta-catenin of TCF/LEF transcription factors and potentiates the coactivation by beta-catenin of other transcription factors, such as SF-1 and C/EBPalpha.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 411 (char 410)\n",
      "{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"SREBF1\"],\"tf text\":[\"SREBP-1c\"],\"tg text\":[\"SREBP-1c\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In the present study\",\"we aimed to decipher the mechanisms involved in the transcriptional effect of insulin on the SREBP-1c specific promoter of the human srebf-1 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|insulin is actor, indorectly---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"SREBF1\"],\"tf text\":[\"SREBP-1c\"],\"tg text\":[\"SREBP-1c\"],\"consensus\":[\"N\"],\"martin_comments\":[\"insulin is actor, indorectly\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In the present study, we aimed to decipher the mechanisms involved in the transcriptional effect of insulin on the SREBP-1c specific promoter of the human srebf-1 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"SREBF1\"],\"tf text\":[\"SREBP-1c\"],\"tg text\":[\"SREBP-1c\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"In the present study\",\"we aimed to decipher the mechanisms involved in the transcriptional effect of insulin on the SREBP-1c specific promoter of the human srebf-1 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|insulin is actor, indorectly---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"SREBF1\"],\"tf text\":[\"SREBP-1c\"],\"tg text\":[\"SREBP-1c\"],\"consensus\":[\"N\"],\"martin_comments\":[\"insulin is actor, indorectly\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In the present study, we aimed to decipher the mechanisms involved in the transcriptional effect of insulin on the SREBP-1c specific promoter of the human srebf-1 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[\"c-Jun N-terminal kinase\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We determined that rapamycin efficiently activates AP-1-driven transcription by rapidly inducing c-jun/AP-1 phosphorylation with activation of the c-Jun N-terminal kinase (JNK) cascade, resulting in enhanced binding of AP-1.DNA complex formation and AP-1-dependent gene transactivation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[\"c-Jun N-terminal kinase\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We determined that rapamycin efficiently activates AP-1-driven transcription by rapidly inducing c-jun/AP-1 phosphorylation with activation of the c-Jun N-terminal kinase (JNK) cascade, resulting in enhanced binding of AP-1.DNA complex formation and AP-1-dependent gene transactivation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is MMCP-6---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"TPSB2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is MMCP-6\"],\"sentence\":[\"Overexpression of +-MITF but not mi-MITF normalized the expression of the MMCP-6 gene\",\"indicating the involvement of +-MITF in the MMCP-6 gene transactivation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MMCP-6 does not exist?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"TPSB2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MMCP-6 does not exist?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of +-MITF but not mi-MITF normalized the expression of the MMCP-6 gene, indicating the involvement of +-MITF in the MMCP-6 gene transactivation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is MMCP-6---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"TPSB2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is MMCP-6\"],\"sentence\":[\"Overexpression of +-MITF but not mi-MITF normalized the expression of the MMCP-6 gene\",\"indicating the involvement of +-MITF in the MMCP-6 gene transactivation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MMCP-6 does not exist?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"TPSB2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MMCP-6 does not exist?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of +-MITF but not mi-MITF normalized the expression of the MMCP-6 gene, indicating the involvement of +-MITF in the MMCP-6 gene transactivation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is gamma-crystallin---{\"tf associated gene name\":[\"HSF4\"],\"tg associated gene name\":[\"MTPN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is gamma-crystallin\"],\"sentence\":[\"These findings reveal the lens-specific role of HSF4\",\"which activates gamma-crystallin genes\",\"and also indicate that HSF1 and HSF4 are involved in regulating expression of growth factor genes\",\"which are essential for cell growth and differentiation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MTPN from where?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HSF4\"],\"tg (associated gene name)\":[\"MTPN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MTPN from where?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These findings reveal the lens-specific role of HSF4, which activates gamma-crystallin genes, and also indicate that HSF1 and HSF4 are involved in regulating expression of growth factor genes, which are essential for cell growth and differentiation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is gamma-crystallin---{\"tf associated gene name\":[\"HSF4\"],\"tg associated gene name\":[\"MTPN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is gamma-crystallin\"],\"sentence\":[\"These findings reveal the lens-specific role of HSF4\",\"which activates gamma-crystallin genes\",\"and also indicate that HSF1 and HSF4 are involved in regulating expression of growth factor genes\",\"which are essential for cell growth and differentiation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MTPN from where?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HSF4\"],\"tg (associated gene name)\":[\"MTPN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MTPN from where?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These findings reveal the lens-specific role of HSF4, which activates gamma-crystallin genes, and also indicate that HSF1 and HSF4 are involved in regulating expression of growth factor genes, which are essential for cell growth and differentiation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain about TF (MUC1/22TR) normalization---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about TF (MUC1/22TR) normalization\"],\"sentence\":[\"ChIP and real-time PCR experiments revealed that MUC1/22TR up-regulated IL-6 and TNF-?? expression by binding to their promoter regions in a NF-??B p65-dependent manner in both MUC1-transfected and human breast cancer cells that express endogenous MUC1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF=MUC---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=MUC\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"ChIP and real-time PCR experiments revealed that MUC1/22TR up-regulated IL-6 and TNF- expression by binding to their promoter regions in a NF-B p65-dependent manner in both MUC1-transfected and human breast cancer cells that express endogenous MUC1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "uncertain about TF (MUC1/22TR) normalization---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about TF (MUC1/22TR) normalization\"],\"sentence\":[\"ChIP and real-time PCR experiments revealed that MUC1/22TR up-regulated IL-6 and TNF-?? expression by binding to their promoter regions in a NF-??B p65-dependent manner in both MUC1-transfected and human breast cancer cells that express endogenous MUC1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF=MUC---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF=MUC\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"ChIP and real-time PCR experiments revealed that MUC1/22TR up-regulated IL-6 and TNF- expression by binding to their promoter regions in a NF-B p65-dependent manner in both MUC1-transfected and human breast cancer cells that express endogenous MUC1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 381 (char 380)\n",
      "{\"transcription factor (associated gene name)\":[\"ELF1\"],\"target gene (associated gene name)\":[\"PYGO2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Human papilloma virus (HPV) E7-mediated attenuation of retinoblastoma (Rb) induces hPygopus2 expression via Elf-1 in cervical cancer.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|but indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF1\"],\"tg (associated gene name)\":[\"PYGO2\"],\"tf text\":[],\"tg text\":[\"hPygopus2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"but indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Human papilloma virus (HPV) E7-mediated attenuation of retinoblastoma (Rb) induces hPygopus2 expression via Elf-1 in cervical cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ELF1\"],\"tg associated gene name\":[\"PYGO2\"],\"tf text\":[],\"tg text\":[\"hPygopus2\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Human papilloma virus (HPV) E7-mediated attenuation of retinoblastoma (Rb) induces hPygopus2 expression via Elf-1 in cervical cancer.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ELF1\"],\"target gene (associated gene name)\":[\"PYGO2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Human papilloma virus (HPV) E7-mediated attenuation of retinoblastoma (Rb) induces hPygopus2 expression via Elf-1 in cervical cancer.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|but indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF1\"],\"tg (associated gene name)\":[\"PYGO2\"],\"tf text\":[],\"tg text\":[\"hPygopus2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"but indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Human papilloma virus (HPV) E7-mediated attenuation of retinoblastoma (Rb) induces hPygopus2 expression via Elf-1 in cervical cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG named---{\"tf associated gene name\":[\"ID2\"],\"tg associated gene name\":[\"MTPN\"],\"tf text\":[\"Id2\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Mad upregulation and Id2 repression accompany transforming growth factor (TGF)-beta-mediated epithelial cell growth suppression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no causality---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID2\"],\"tg (associated gene name)\":[\"MTPN\"],\"tf text\":[\"Id2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no causality\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Mad upregulation and Id2 repression accompany transforming growth factor (TGF)-beta-mediated epithelial cell growth suppression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG named---{\"tf associated gene name\":[\"ID2\"],\"tg associated gene name\":[\"MTPN\"],\"tf text\":[\"Id2\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Mad upregulation and Id2 repression accompany transforming growth factor (TGF)-beta-mediated epithelial cell growth suppression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no causality---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID2\"],\"tg (associated gene name)\":[\"MTPN\"],\"tf text\":[\"Id2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no causality\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Mad upregulation and Id2 repression accompany transforming growth factor (TGF)-beta-mediated epithelial cell growth suppression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p300---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is p300\"],\"sentence\":[\"We showed that p300 was indeed induced by Tat expression and HIV-1 infection and that the induction occurred at the transcriptional level through the cis-acting elements of early growth response 1 (egr-1) within its promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We showed that p300 was indeed induced by Tat expression and HIV-1 infection and that the induction occurred at the transcriptional level through the cis-acting elements of early growth response 1 (egr-1) within its promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is p300---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is p300\"],\"sentence\":[\"We showed that p300 was indeed induced by Tat expression and HIV-1 infection and that the induction occurred at the transcriptional level through the cis-acting elements of early growth response 1 (egr-1) within its promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We showed that p300 was indeed induced by Tat expression and HIV-1 infection and that the induction occurred at the transcriptional level through the cis-acting elements of early growth response 1 (egr-1) within its promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is COL2A1---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"SP1\"],\"tf text\":[\"Sp3\"],\"tg text\":[\"Sp1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is COL2A1\"],\"sentence\":[\"Nevertheless\",\"IL-1 beta decreased the binding activity of both Sp1 and Sp3 to the 63-bp short COL2A1 promoter\",\"suggesting that the cytokine exerts a post-transcriptional regulatory mechanism on Sp1 and Sp3 gene expressions.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP3\"],\"tg (associated gene name)\":[\"SP1\"],\"tf text\":[\"Sp3\"],\"tg text\":[\"Sp1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Nevertheless, IL-1 beta decreased the binding activity of both Sp1 and Sp3 to the 63-bp short COL2A1 promoter, suggesting that the cytokine exerts a post-transcriptional regulatory mechanism on Sp1 and Sp3 gene expressions.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is COL2A1---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"SP1\"],\"tf text\":[\"Sp3\"],\"tg text\":[\"Sp1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is COL2A1\"],\"sentence\":[\"Nevertheless\",\"IL-1 beta decreased the binding activity of both Sp1 and Sp3 to the 63-bp short COL2A1 promoter\",\"suggesting that the cytokine exerts a post-transcriptional regulatory mechanism on Sp1 and Sp3 gene expressions.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP3\"],\"tg (associated gene name)\":[\"SP1\"],\"tf text\":[\"Sp3\"],\"tg text\":[\"Sp1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Nevertheless, IL-1 beta decreased the binding activity of both Sp1 and Sp3 to the 63-bp short COL2A1 promoter, suggesting that the cytokine exerts a post-transcriptional regulatory mechanism on Sp1 and Sp3 gene expressions.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is CD154---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"CD4\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is CD154\"],\"sentence\":[\"Similarly\",\"upon activation\",\"CD154 message is notably decreased in splenic CD4 T cells from Egr-1-deficient mice compared with wild-type controls.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is CD154---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"CD4\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is CD154\"],\"sentence\":[\"Similarly\",\"upon activation\",\"CD154 message is notably decreased in splenic CD4 T cells from Egr-1-deficient mice compared with wild-type controls.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"CD4\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Similarly\",\"upon activation\",\"CD154 message is notably decreased in splenic CD4 T cells from Egr-1-deficient mice compared with wild-type controls.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.943767\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"145\"],\"sentence pair density\":[\"0.009009009009009\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CD154 TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"CD4\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CD154 TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Similarly, upon activation, CD154 message is notably decreased in splenic CD4 T cells from Egr-1-deficient mice compared with wild-type controls.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 413 (char 412)\n",
      "{\"tf associated gene name\":[\"GATA2\"],\"tg associated gene name\":[\"IL3\"],\"tf text\":[\"GATA-2\"],\"tg text\":[\"IL-3\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Overexpression of GATA-1 alone transactivates the E4bp4 reporter\",\"whereas transactivation of the E4bp4 reporter by GATA-2 is dependent on the stimulation of IL-3.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA2\"],\"tg (associated gene name)\":[\"IL3\"],\"tf text\":[\"GATA-2\"],\"tg text\":[\"IL-3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of GATA-1 alone transactivates the E4bp4 reporter, whereas transactivation of the E4bp4 reporter by GATA-2 is dependent on the stimulation of IL-3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"GATA2\"],\"tg associated gene name\":[\"IL3\"],\"tf text\":[\"GATA-2\"],\"tg text\":[\"IL-3\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Overexpression of GATA-1 alone transactivates the E4bp4 reporter\",\"whereas transactivation of the E4bp4 reporter by GATA-2 is dependent on the stimulation of IL-3.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA2\"],\"tg (associated gene name)\":[\"IL3\"],\"tf text\":[\"GATA-2\"],\"tg text\":[\"IL-3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of GATA-1 alone transactivates the E4bp4 reporter, whereas transactivation of the E4bp4 reporter by GATA-2 is dependent on the stimulation of IL-3.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not named---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IL1RN\"],\"tf text\":[],\"tg text\":[\"IL-1ra\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF not named\"],\"sentence\":[\"These results indicate that the net activation of the human IL-1ra promoter in response to LPS involves the functional interaction of at least four cis-acting DNA elements within the proximal 294 bp.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF not named---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IL1RN\"],\"tf text\":[],\"tg text\":[\"IL-1ra\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF not named\"],\"sentence\":[\"These results indicate that the net activation of the human IL-1ra promoter in response to LPS involves the functional interaction of at least four cis-acting DNA elements within the proximal 294 bp.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF6\"],\"target gene (associated gene name)\":[\"IL1RN\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results indicate that the net activation of the human IL-1ra promoter in response to LPS involves the functional interaction of at least four cis-acting DNA elements within the proximal 294 bp.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|LPS?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IL1RN\"],\"tf text\":[],\"tg text\":[\"IL-1ra\"],\"consensus\":[\"V\"],\"martin_comments\":[\"LPS?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that the net activation of the human IL-1ra promoter in response to LPS involves the functional interaction of at least four cis-acting DNA elements within the proximal 294 bp.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "impled at best?|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"impled at best?\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Overexpression of a dominant-negative (DN) mutant of PKC-alpha inhibited the expression of IFN-gamma-induced MHC-II but had no effect on IFN-gamma-induced STAT1 nuclear translocation and DNA binding activity, as well as on the expression of inducible NO synthase, IFN consensus sequence binding protein, MHC class I, IFN regulatory factor (IRF)-1, and IFN-gamma-inducible protein-10.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "impled at best?|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"impled at best?\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Overexpression of a dominant-negative (DN) mutant of PKC-alpha inhibited the expression of IFN-gamma-induced MHC-II but had no effect on IFN-gamma-induced STAT1 nuclear translocation and DNA binding activity, as well as on the expression of inducible NO synthase, IFN consensus sequence binding protein, MHC class I, IFN regulatory factor (IRF)-1, and IFN-gamma-inducible protein-10.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no ADAM2?---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[\"no ADAM2?\"],\"sentence\":[\"Nuclear translocation of NF-kappaB is regulated from the cytoplasm by IkappaB kinase-beta (IKKbeta)\",\"which earmarks inhibitors of NF-kappaB for polyubiquination and proteasome-mediated degradation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein leve;---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein leve;\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Nuclear translocation of NF-kappaB is regulated from the cytoplasm by IkappaB kinase-beta (IKKbeta), which earmarks inhibitors of NF-kappaB for polyubiquination and proteasome-mediated degradation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no ADAM2?---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[\"no ADAM2?\"],\"sentence\":[\"Nuclear translocation of NF-kappaB is regulated from the cytoplasm by IkappaB kinase-beta (IKKbeta)\",\"which earmarks inhibitors of NF-kappaB for polyubiquination and proteasome-mediated degradation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein leve;---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein leve;\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Nuclear translocation of NF-kappaB is regulated from the cytoplasm by IkappaB kinase-beta (IKKbeta), which earmarks inhibitors of NF-kappaB for polyubiquination and proteasome-mediated degradation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 463 (char 462)\n",
      "{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[\"Mammary gland factor\"],\"tg text\":[\"prolactin\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Mammary gland factor activated by prolactin on mammary epithelial cells and acute-phase response factor activated by interleukin-6 in liver cells share DNA binding and transactivation potential.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[\"Mammary gland factor\"],\"tg text\":[\"prolactin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Mammary gland factor activated by prolactin on mammary epithelial cells and acute-phase response factor activated by interleukin-6 in liver cells share DNA binding and transactivation potential.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[\"Mammary gland factor\"],\"tg text\":[\"prolactin\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Mammary gland factor activated by prolactin on mammary epithelial cells and acute-phase response factor activated by interleukin-6 in liver cells share DNA binding and transactivation potential.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[\"Mammary gland factor\"],\"tg text\":[\"prolactin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Mammary gland factor activated by prolactin on mammary epithelial cells and acute-phase response factor activated by interleukin-6 in liver cells share DNA binding and transactivation potential.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is SHP-1---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"TSC1\"],\"tf text\":[\"STAT3\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is SHP-1\"],\"sentence\":[\"STAT3- and DNA methyltransferase 1-mediated epigenetic silencing of SHP-1 tyrosine phosphatase tumor suppressor gene in malignant T lymphocytes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|technically yes, but probably wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"TSC1\"],\"tf text\":[\"STAT3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"technically yes, but probably wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"STAT3- and DNA methyltransferase 1-mediated epigenetic silencing of SHP-1 tyrosine phosphatase tumor suppressor gene in malignant T lymphocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is SHP-1---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"TSC1\"],\"tf text\":[\"STAT3\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is SHP-1\"],\"sentence\":[\"STAT3- and DNA methyltransferase 1-mediated epigenetic silencing of SHP-1 tyrosine phosphatase tumor suppressor gene in malignant T lymphocytes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|technically yes, but probably wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"TSC1\"],\"tf text\":[\"STAT3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"technically yes, but probably wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"STAT3- and DNA methyltransferase 1-mediated epigenetic silencing of SHP-1 tyrosine phosphatase tumor suppressor gene in malignant T lymphocytes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF should be normalized to NR1D2---{\"tf associated gene name\":[\"NR1D1\"],\"tg associated gene name\":[\"APOC3\"],\"tf text\":[],\"tg text\":[\"apoCIII\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF should be normalized to NR1D2\"],\"sentence\":[\"Finally\",\"ZNHIT-1 was recruited by Rev-erbbeta to the apoCIII promoter and removed the Rev-erbbeta-induced inhibition of apoCIII transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|norm error RevErbbeta---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1D1\"],\"tg (associated gene name)\":[\"APOC3\"],\"tf text\":[],\"tg text\":[\"apoCIII\"],\"consensus\":[\"N\"],\"martin_comments\":[\"norm error RevErbbeta\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Finally, ZNHIT-1 was recruited by Rev-erbbeta to the apoCIII promoter and removed the Rev-erbbeta-induced inhibition of apoCIII transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF should be normalized to NR1D2---{\"tf associated gene name\":[\"NR1D1\"],\"tg associated gene name\":[\"APOC3\"],\"tf text\":[],\"tg text\":[\"apoCIII\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF should be normalized to NR1D2\"],\"sentence\":[\"Finally\",\"ZNHIT-1 was recruited by Rev-erbbeta to the apoCIII promoter and removed the Rev-erbbeta-induced inhibition of apoCIII transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|norm error RevErbbeta---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1D1\"],\"tg (associated gene name)\":[\"APOC3\"],\"tf text\":[],\"tg text\":[\"apoCIII\"],\"consensus\":[\"N\"],\"martin_comments\":[\"norm error RevErbbeta\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Finally, ZNHIT-1 was recruited by Rev-erbbeta to the apoCIII promoter and removed the Rev-erbbeta-induced inhibition of apoCIII transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"PSEN2\"],\"tf text\":[\"progesterone receptor\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Experiments in which the p21WAF1/CIP1 protein was knocked down by RNA interference showed that the induction of the expression of the gene encoding the progesterone receptor required p21WAF1/CIP1, in contrast with that of the cyclin D1 and pS2 genes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"PSEN2\"],\"tf text\":[\"progesterone receptor\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Experiments in which the p21WAF1/CIP1 protein was knocked down by RNA interference showed that the induction of the expression of the gene encoding the progesterone receptor required p21WAF1/CIP1, in contrast with that of the cyclin D1 and pS2 genes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 536 (char 535)\n",
      "{\"transcription factor (associated gene name)\":[\"RFXANK\"],\"target gene (associated gene name)\":[\"HLA-DOB\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Mutational analysis of the DOB promoter demonstrated that the X1\",\"Y and OCT boxes are necessary for maximum promoter activity.Furthermore\",\"our results demonstrate that CREB-1\",\"RFXANK and Oct-2 occupy the DOB promoter in vivo\",\"However\",\"CIITA and Bob-1 were only minimally recruited.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|two sentences---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RFXANK\"],\"tg (associated gene name)\":[\"HLA-DOB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"two sentences\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Mutational analysis of the DOB promoter demonstrated that the X1, Y and OCT boxes are necessary for maximum promoter activity.Furthermore, our results demonstrate that CREB-1, RFXANK and Oct-2 occupy the DOB promoter in vivo, However, CIITA and Bob-1 were only minimally recruited.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RFXANK\"],\"tg associated gene name\":[\"HLA-DOB\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Mutational analysis of the DOB promoter demonstrated that the X1\",\"Y and OCT boxes are necessary for maximum promoter activity.Furthermore\",\"our results demonstrate that CREB-1\",\"RFXANK and Oct-2 occupy the DOB promoter in vivo\",\"However\",\"CIITA and Bob-1 were only minimally recruited.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RFXANK\"],\"target gene (associated gene name)\":[\"HLA-DOB\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Mutational analysis of the DOB promoter demonstrated that the X1\",\"Y and OCT boxes are necessary for maximum promoter activity.Furthermore\",\"our results demonstrate that CREB-1\",\"RFXANK and Oct-2 occupy the DOB promoter in vivo\",\"However\",\"CIITA and Bob-1 were only minimally recruited.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|two sentences---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RFXANK\"],\"tg (associated gene name)\":[\"HLA-DOB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"two sentences\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Mutational analysis of the DOB promoter demonstrated that the X1, Y and OCT boxes are necessary for maximum promoter activity.Furthermore, our results demonstrate that CREB-1, RFXANK and Oct-2 occupy the DOB promoter in vivo, However, CIITA and Bob-1 were only minimally recruited.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF named---{\"tf associated gene name\":[\"ZNF135\"],\"tg associated gene name\":[\"CDH2\"],\"tf text\":[\"pT3\"],\"tg text\":[\"N-cadherin\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TF named\"],\"sentence\":[\"N-cadherin expression was absent in normal urothelium\",\"appeared in stage pT1\",\"and increased in pT2-pT3 tumors.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|technically ues, but tumor state indication---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF135\"],\"tg (associated gene name)\":[\"CDH2\"],\"tf text\":[\"pT3\"],\"tg text\":[\"N-cadherin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"technically ues, but tumor state indication\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"N-cadherin expression was absent in normal urothelium, appeared in stage pT1, and increased in pT2-pT3 tumors.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF named---{\"tf associated gene name\":[\"ZNF135\"],\"tg associated gene name\":[\"CDH2\"],\"tf text\":[\"pT3\"],\"tg text\":[\"N-cadherin\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TF named\"],\"sentence\":[\"N-cadherin expression was absent in normal urothelium\",\"appeared in stage pT1\",\"and increased in pT2-pT3 tumors.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|technically ues, but tumor state indication---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF135\"],\"tg (associated gene name)\":[\"CDH2\"],\"tf text\":[\"pT3\"],\"tg text\":[\"N-cadherin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"technically ues, but tumor state indication\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"N-cadherin expression was absent in normal urothelium, appeared in stage pT1, and increased in pT2-pT3 tumors.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TTF1\"],\"tg (associated gene name)\":[\"TTF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"DETA-NO inhibited SP-B promoter induction by exogenous TTF-1 without altering TTF-1 levels.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TTF1\"],\"tg (associated gene name)\":[\"TTF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"DETA-NO inhibited SP-B promoter induction by exogenous TTF-1 without altering TTF-1 levels.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 675 (char 674)\n",
      "{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"FOS\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Fra-1\",\"Fra-2\",\"FosB\",\"JunB\",\"JunD\",\"c-Jun\",\"and c-Fos levels are increased by EGCG treatment\",\"as is AP1 factor binding to hINV promoter AP1 site.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.191881\"],\"pmid counts\":[\"6\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"4\"],\"sentence length\":[\"140\"],\"sentence pair density\":[\"0.00699300699300699\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOSB\"],\"tg (associated gene name)\":[\"FOS\"],\"tf text\":[\"FosB\"],\"tg text\":[\"c-Fos\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Fra-1, Fra-2, FosB, JunB, JunD, c-Jun, and c-Fos levels are increased by EGCG treatment, as is AP1 factor binding to hINV promoter AP1 site.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"FOSB\"],\"tg associated gene name\":[\"FOS\"],\"tf text\":[\"FosB\"],\"tg text\":[\"c-Fos\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"Fra-1\",\"Fra-2\",\"FosB\",\"JunB\",\"JunD\",\"c-Jun\",\"and c-Fos levels are increased by EGCG treatment\",\"as is AP1 factor binding to hINV promoter AP1 site.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"FOS\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[\"Fra-1\",\"Fra-2\",\"FosB\",\"JunB\",\"JunD\",\"c-Jun\",\"and c-Fos levels are increased by EGCG treatment\",\"as is AP1 factor binding to hINV promoter AP1 site.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"FOS\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Fra-1\",\"Fra-2\",\"FosB\",\"JunB\",\"JunD\",\"c-Jun\",\"and c-Fos levels are increased by EGCG treatment\",\"as is AP1 factor binding to hINV promoter AP1 site.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.191881\"],\"pmid counts\":[\"6\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"4\"],\"sentence length\":[\"140\"],\"sentence pair density\":[\"0.00699300699300699\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOSB\"],\"tg (associated gene name)\":[\"FOS\"],\"tf text\":[\"FosB\"],\"tg text\":[\"c-Fos\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Fra-1, Fra-2, FosB, JunB, JunD, c-Jun, and c-Fos levels are increased by EGCG treatment, as is AP1 factor binding to hINV promoter AP1 site.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named (extracellular matrix genes)---{\"tf associated gene name\":[\"BHLHE41\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG not named (extracellular matrix genes)\"],\"sentence\":[\"In vitro\",\"Sharp-1 associates with Smad3\",\"and its overexpression inhibits TGF-??- and Smad3-mediated expression of extracellular matrix genes in myofibroblasts.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG not named (extracellular matrix genes)---{\"tf associated gene name\":[\"BHLHE41\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG not named (extracellular matrix genes)\"],\"sentence\":[\"In vitro\",\"Sharp-1 associates with Smad3\",\"and its overexpression inhibits TGF-??- and Smad3-mediated expression of extracellular matrix genes in myofibroblasts.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"BHLHE41\"],\"target gene (associated gene name)\":[\"TGFB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"In vitro\",\"Sharp-1 associates with Smad3\",\"and its overexpression inhibits TGF-- and Smad3-mediated expression of extracellular matrix genes in myofibroblasts.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"0.789096\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"157\"],\"sentence pair density\":[\"0.0170454545454545\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BHLHE41\"],\"tg (associated gene name)\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In vitro, Sharp-1 associates with Smad3, and its overexpression inhibits TGF-- and Smad3-mediated expression of extracellular matrix genes in myofibroblasts.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR2F1\"],\"tg (associated gene name)\":[\"TERT\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We localized a region -750 to -700 in the CaMKIV gene 5'-flanking region that conferred T(3) responsiveness and bound thyroid hormone receptor (TR), retinoic acid receptor (RAR), and chicken ovalbumin upstream promoter-transcription factor 1 (COUP-TF1).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR2F1\"],\"tg (associated gene name)\":[\"TERT\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We localized a region -750 to -700 in the CaMKIV gene 5'-flanking region that conferred T(3) responsiveness and bound thyroid hormone receptor (TR), retinoic acid receptor (RAR), and chicken ovalbumin upstream promoter-transcription factor 1 (COUP-TF1).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 364 (char 363)\n",
      "{\"tf associated gene name\":[\"MAFG\"],\"tg associated gene name\":[\"IGF1\"],\"tf text\":[\"MafG\"],\"tg text\":[\"IGF-1\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"IGF-1 increased the expression of c-Jun\",\"FosB\",\"MafG\",\"p65\",\"c-Myb\",\"E2F-1 and YY1 at the pre-translational level.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAFG\"],\"tg (associated gene name)\":[\"IGF1\"],\"tf text\":[\"MafG\"],\"tg text\":[\"IGF-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"IGF-1 increased the expression of c-Jun, FosB, MafG, p65, c-Myb, E2F-1 and YY1 at the pre-translational level.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MAFG\"],\"tg associated gene name\":[\"IGF1\"],\"tf text\":[\"MafG\"],\"tg text\":[\"IGF-1\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"IGF-1 increased the expression of c-Jun\",\"FosB\",\"MafG\",\"p65\",\"c-Myb\",\"E2F-1 and YY1 at the pre-translational level.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MAFG\"],\"tg (associated gene name)\":[\"IGF1\"],\"tf text\":[\"MafG\"],\"tg text\":[\"IGF-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"IGF-1 increased the expression of c-Jun, FosB, MafG, p65, c-Myb, E2F-1 and YY1 at the pre-translational level.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is PAF AH; various TFs other than IRF6---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"LIPH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is PAF AH; various TFs other than IRF6\"],\"sentence\":[\"The ability of LPS to induce transcriptional activation of PAF AH is not due to enhanced Sp1/Sp3 binding to the promoter but involves enhanced transactivation function of Sp1 via p38 MAPK activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is PAF AH; various TFs other than IRF6---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"LIPH\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is PAF AH; various TFs other than IRF6\"],\"sentence\":[\"The ability of LPS to induce transcriptional activation of PAF AH is not due to enhanced Sp1/Sp3 binding to the promoter but involves enhanced transactivation function of Sp1 via p38 MAPK activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF6\"],\"target gene (associated gene name)\":[\"LIPH\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The ability of LPS to induce transcriptional activation of PAF AH is not due to enhanced Sp1/Sp3 binding to the promoter but involves enhanced transactivation function of Sp1 via p38 MAPK activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|LPS?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"LIPH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"LPS?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The ability of LPS to induce transcriptional activation of PAF AH is not due to enhanced Sp1/Sp3 binding to the promoter but involves enhanced transactivation function of Sp1 via p38 MAPK activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG named---{\"tf associated gene name\":[\"MECOM\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[\"Evi-1\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Here we study the mechanisms that underlie oncogenesis induced by Evi-1 by investigating whether Evi-1 perturbs signalling through transforming growth factor-beta (TGF-beta)\",\"one of the most studied growth-regulatory factors\",\"which inhibits proliferation of a wide range of cell types.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|norm derailed by 'beta'?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MECOM\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[\"Evi-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"norm derailed by 'beta'?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we study the mechanisms that underlie oncogenesis induced by Evi-1 by investigating whether Evi-1 perturbs signalling through transforming growth factor-beta (TGF-beta), one of the most studied growth-regulatory factors, which inhibits proliferation of a wide range of cell types.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG named---{\"tf associated gene name\":[\"MECOM\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[\"Evi-1\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Here we study the mechanisms that underlie oncogenesis induced by Evi-1 by investigating whether Evi-1 perturbs signalling through transforming growth factor-beta (TGF-beta)\",\"one of the most studied growth-regulatory factors\",\"which inhibits proliferation of a wide range of cell types.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|norm derailed by 'beta'?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MECOM\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[\"Evi-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"norm derailed by 'beta'?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we study the mechanisms that underlie oncogenesis induced by Evi-1 by investigating whether Evi-1 perturbs signalling through transforming growth factor-beta (TGF-beta), one of the most studied growth-regulatory factors, which inhibits proliferation of a wide range of cell types.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GRE|but TG not named---{\"tf associated gene name\":[\"REL\"],\"tg associated gene name\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"GRE\",\"but TG not named\"],\"sentence\":[\"The increase in Sp1-site transactivation does not precisely correlate with transformation by v-Rel since one non-transforming v-Rel mutant still induced expression from the Sp1 site-containing promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"REL\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The increase in Sp1-site transactivation does not precisely correlate with transformation by v-Rel since one non-transforming v-Rel mutant still induced expression from the Sp1 site-containing promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "GRE|but TG not named---{\"tf associated gene name\":[\"REL\"],\"tg associated gene name\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"GRE\",\"but TG not named\"],\"sentence\":[\"The increase in Sp1-site transactivation does not precisely correlate with transformation by v-Rel since one non-transforming v-Rel mutant still induced expression from the Sp1 site-containing promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"REL\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The increase in Sp1-site transactivation does not precisely correlate with transformation by v-Rel since one non-transforming v-Rel mutant still induced expression from the Sp1 site-containing promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is ARA54---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"regulator is ARA54\"],\"sentence\":[\"Furthermore\",\"the amount of ARA54 mRNA appeared to positively correlate with cyclin D1 mRNA levels in specimens of clinical colon carcinomas\",\"indicating that ARA54 is not only involved in the regulation of cyclin D1 expression in cultured cell lines but also in clinical cancer specimens.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "regulator is ARA54---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"regulator is ARA54\"],\"sentence\":[\"Furthermore\",\"the amount of ARA54 mRNA appeared to positively correlate with cyclin D1 mRNA levels in specimens of clinical colon carcinomas\",\"indicating that ARA54 is not only involved in the regulation of cyclin D1 expression in cultured cell lines but also in clinical cancer specimens.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXC1\"],\"target gene (associated gene name)\":[\"CCND1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Furthermore\",\"the amount of ARA54 mRNA appeared to positively correlate with cyclin D1 mRNA levels in specimens of clinical colon carcinomas\",\"indicating that ARA54 is not only involved in the regulation of cyclin D1 expression in cultured cell lines but also in clinical cancer specimens.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.746781\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"3\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"287\"],\"sentence pair density\":[\"0.0103626943005181\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|ARA54 not ARA---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"ARA54 not ARA\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, the amount of ARA54 mRNA appeared to positively correlate with cyclin D1 mRNA levels in specimens of clinical colon carcinomas, indicating that ARA54 is not only involved in the regulation of cyclin D1 expression in cultured cell lines but also in clinical cancer specimens.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is APOA5---{\"tf associated gene name\":[\"USF2\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[\"USF2\"],\"tg text\":[\"insulin\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is APOA5\"],\"sentence\":[\"Moreover\",\"in transfection studies\",\"USF1 stimulates APOA5 promoter activity\",\"and the treatment with insulin reduced the binding of USF1/USF2 to the APOA5 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF2\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"USF2\"],\"tg text\":[\"insulin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, in transfection studies, USF1 stimulates APOA5 promoter activity, and the treatment with insulin reduced the binding of USF1/USF2 to the APOA5 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is APOA5---{\"tf associated gene name\":[\"USF2\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[\"USF2\"],\"tg text\":[\"insulin\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is APOA5\"],\"sentence\":[\"Moreover\",\"in transfection studies\",\"USF1 stimulates APOA5 promoter activity\",\"and the treatment with insulin reduced the binding of USF1/USF2 to the APOA5 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF2\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"USF2\"],\"tg text\":[\"insulin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, in transfection studies, USF1 stimulates APOA5 promoter activity, and the treatment with insulin reduced the binding of USF1/USF2 to the APOA5 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F7\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[\"E2F7\"],\"tg text\":[\"p53\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Taken together, these results show that p53-dependent transcriptional up-regulation of its target, E2F7, leads to repression of relevant gene expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F7\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[\"E2F7\"],\"tg text\":[\"p53\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Taken together, these results show that p53-dependent transcriptional up-regulation of its target, E2F7, leads to repression of relevant gene expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 488 (char 487)\n",
      "{\"tf associated gene name\":[\"E2F2\"],\"tg associated gene name\":[\"CDK1\"],\"tf text\":[\"E2F2\"],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"A large set of E2F target genes involved in DNA replication and cell cycle progression (such as Mcm's\",\"cyclins and Cdc2a) that are silent in G(0) and typically transcribed late in G(1) phase are already actively expressed in quiescent T cells and MEFs lacking E2F2.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|CDK1 is lucky guess---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F2\"],\"tg (associated gene name)\":[\"CDK1\"],\"tf text\":[\"E2F2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CDK1 is lucky guess\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"A large set of E2F target genes involved in DNA replication and cell cycle progression (such as Mcm's, cyclins and Cdc2a) that are silent in G(0) and typically transcribed late in G(1) phase are already actively expressed in quiescent T cells and MEFs lacking E2F2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"E2F2\"],\"tg associated gene name\":[\"CDK1\"],\"tf text\":[\"E2F2\"],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"A large set of E2F target genes involved in DNA replication and cell cycle progression (such as Mcm's\",\"cyclins and Cdc2a) that are silent in G(0) and typically transcribed late in G(1) phase are already actively expressed in quiescent T cells and MEFs lacking E2F2.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|CDK1 is lucky guess---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"E2F2\"],\"tg (associated gene name)\":[\"CDK1\"],\"tf text\":[\"E2F2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CDK1 is lucky guess\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"A large set of E2F target genes involved in DNA replication and cell cycle progression (such as Mcm's, cyclins and Cdc2a) that are silent in G(0) and typically transcribed late in G(1) phase are already actively expressed in quiescent T cells and MEFs lacking E2F2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG---{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"t\",\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"By binding to the c-myc promoter(s)\",\"C/EBPbeta represses c-Myc expression and\",\"therefore\",\"arrests T cells in the G1 phase of the cell cycle.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By binding to the c-myc promoter(s), C/EBPbeta represses c-Myc expression and, therefore, arrests T cells in the G1 phase of the cell cycle.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG---{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"t\",\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"By binding to the c-myc promoter(s)\",\"C/EBPbeta represses c-Myc expression and\",\"therefore\",\"arrests T cells in the G1 phase of the cell cycle.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"By binding to the c-myc promoter(s), C/EBPbeta represses c-Myc expression and, therefore, arrests T cells in the G1 phase of the cell cycle.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CARD8\"],\"tf text\":[],\"tg text\":[\"Caspase activating and recruitment domain 8\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Caspase activating and recruitment domain 8 (CARD8) potently inhibits NF-kappaB signaling\",\"which plays a key role in inflammation\",\"and may contribute to avoid a pathologic activation of NF-kappaB; however\",\"the transcriptional mechanisms regulating CARD8 expression and the relevance of this protein in inflammatory diseases are poorly understood.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|No TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CARD8\"],\"tf text\":[],\"tg text\":[\"Caspase activating and recruitment domain 8\"],\"consensus\":[\"N\"],\"martin_comments\":[\"No TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Caspase activating and recruitment domain 8 (CARD8) potently inhibits NF-kappaB signaling, which plays a key role in inflammation, and may contribute to avoid a pathologic activation of NF-kappaB; however, the transcriptional mechanisms regulating CARD8 expression and the relevance of this protein in inflammatory diseases are poorly understood.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CARD8\"],\"tf text\":[],\"tg text\":[\"Caspase activating and recruitment domain 8\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Caspase activating and recruitment domain 8 (CARD8) potently inhibits NF-kappaB signaling\",\"which plays a key role in inflammation\",\"and may contribute to avoid a pathologic activation of NF-kappaB; however\",\"the transcriptional mechanisms regulating CARD8 expression and the relevance of this protein in inflammatory diseases are poorly understood.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|No TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CARD8\"],\"tf text\":[],\"tg text\":[\"Caspase activating and recruitment domain 8\"],\"consensus\":[\"N\"],\"martin_comments\":[\"No TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Caspase activating and recruitment domain 8 (CARD8) potently inhibits NF-kappaB signaling, which plays a key role in inflammation, and may contribute to avoid a pathologic activation of NF-kappaB; however, the transcriptional mechanisms regulating CARD8 expression and the relevance of this protein in inflammatory diseases are poorly understood.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 356 (char 355)\n",
      "{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"MDM2 was present in complexes with mutant p53 in tumor cells\",\"and stabilization of MDM2 required direct binding to mutant p53.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MDM2 was present in complexes with mutant p53 in tumor cells, and stabilization of MDM2 required direct binding to mutant p53.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"MDM2 was present in complexes with mutant p53 in tumor cells\",\"and stabilization of MDM2 required direct binding to mutant p53.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"p53\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MDM2 was present in complexes with mutant p53 in tumor cells, and stabilization of MDM2 required direct binding to mutant p53.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "association at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PDX1\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"association at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Dicer1-deficient ??-cells lost insulin expression while maintaining the expression of ??-cell transcription factors such as Pdx1 and Nkx6.1 early in the postnatal period.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "association at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PDX1\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"association at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Dicer1-deficient ??-cells lost insulin expression while maintaining the expression of ??-cell transcription factors such as Pdx1 and Nkx6.1 early in the postnatal period.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is E-cadherin (CDH1)---{\"tf associated gene name\":[\"ZEB1\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E-cadherin (CDH1)\"],\"sentence\":[\"We conclude that Smad3 regulates\",\"at the transcriptional level\",\"miR-200 family members\",\"which themselves regulate ZEB1 and ZEB2\",\"known transcriptional repressors of E-cadherin\",\"at the posttranscriptional level in a TGF-??-independent manner.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong cadherin---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZEB1\"],\"tg (associated gene name)\":[\"CDH17\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong cadherin\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We conclude that Smad3 regulates, at the transcriptional level, miR-200 family members, which themselves regulate ZEB1 and ZEB2, known transcriptional repressors of E-cadherin, at the posttranscriptional level in a TGF--independent manner.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is E-cadherin (CDH1)---{\"tf associated gene name\":[\"ZEB1\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E-cadherin (CDH1)\"],\"sentence\":[\"We conclude that Smad3 regulates\",\"at the transcriptional level\",\"miR-200 family members\",\"which themselves regulate ZEB1 and ZEB2\",\"known transcriptional repressors of E-cadherin\",\"at the posttranscriptional level in a TGF-??-independent manner.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong cadherin---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZEB1\"],\"tg (associated gene name)\":[\"CDH17\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong cadherin\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We conclude that Smad3 regulates, at the transcriptional level, miR-200 family members, which themselves regulate ZEB1 and ZEB2, known transcriptional repressors of E-cadherin, at the posttranscriptional level in a TGF--independent manner.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF normal. very uncertain---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"CYP1A2\"],\"tf text\":[],\"tg text\":[\"CYP1A2\"],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TF normal. very uncertain\"],\"sentence\":[\"Adenovirus-mediated transfection of PGC1alpha in human hepatoma cells caused a significant dose-dependent increase in CYP2C9\",\"CYP1A1\",\"and CYP1A2 and in the positive control CYP7A1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"CYP1A2\"],\"tf text\":[],\"tg text\":[\"CYP1A2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Adenovirus-mediated transfection of PGC1alpha in human hepatoma cells caused a significant dose-dependent increase in CYP2C9, CYP1A1, and CYP1A2 and in the positive control CYP7A1.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF normal. very uncertain---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"CYP1A2\"],\"tf text\":[],\"tg text\":[\"CYP1A2\"],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TF normal. very uncertain\"],\"sentence\":[\"Adenovirus-mediated transfection of PGC1alpha in human hepatoma cells caused a significant dose-dependent increase in CYP2C9\",\"CYP1A1\",\"and CYP1A2 and in the positive control CYP7A1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"CYP1A2\"],\"tf text\":[],\"tg text\":[\"CYP1A2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Adenovirus-mediated transfection of PGC1alpha in human hepatoma cells caused a significant dose-dependent increase in CYP2C9, CYP1A1, and CYP1A2 and in the positive control CYP7A1.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Fshr---{\"tf associated gene name\":[\"ID2\"],\"tg associated gene name\":[\"FSHB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Fshr\"],\"sentence\":[\"Together\",\"these studies suggest that stimulation of Sertoli cells with FSH transiently decreases expression of the USF1 activator and induces accumulation of the ID2 repressor\",\"to block USF binding to the Fshr promoter and delay activation of Fshr transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong FSH---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID2\"],\"tg (associated gene name)\":[\"FSHB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong FSH\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Together, these studies suggest that stimulation of Sertoli cells with FSH transiently decreases expression of the USF1 activator and induces accumulation of the ID2 repressor, to block USF binding to the Fshr promoter and delay activation of Fshr transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Fshr---{\"tf associated gene name\":[\"ID2\"],\"tg associated gene name\":[\"FSHB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Fshr\"],\"sentence\":[\"Together\",\"these studies suggest that stimulation of Sertoli cells with FSH transiently decreases expression of the USF1 activator and induces accumulation of the ID2 repressor\",\"to block USF binding to the Fshr promoter and delay activation of Fshr transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong FSH---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID2\"],\"tg (associated gene name)\":[\"FSHB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong FSH\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Together, these studies suggest that stimulation of Sertoli cells with FSH transiently decreases expression of the USF1 activator and induces accumulation of the ID2 repressor, to block USF binding to the Fshr promoter and delay activation of Fshr transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 460 (char 459)\n",
      "{\"transcription factor (associated gene name)\":[\"ISL1\"],\"target gene (associated gene name)\":[\"SLC2A10\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Using real-time PCR we have confirmed upregulation of Caps2\",\"Sec14l4\",\"Slc2a10\",\"P2rx7\",\"Afamin\",\"and Neurogenin 3 that may in part mediate the observed improved insulin secretion in Isl-1 overexpressing mice.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|a bit indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ISL1\"],\"tg (associated gene name)\":[\"SLC2A10\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using real-time PCR we have confirmed upregulation of Caps2, Sec14l4, Slc2a10, P2rx7, Afamin, and Neurogenin 3 that may in part mediate the observed improved insulin secretion in Isl-1 overexpressing mice.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ISL1\"],\"tg associated gene name\":[\"SLC2A10\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Using real-time PCR we have confirmed upregulation of Caps2\",\"Sec14l4\",\"Slc2a10\",\"P2rx7\",\"Afamin\",\"and Neurogenin 3 that may in part mediate the observed improved insulin secretion in Isl-1 overexpressing mice.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ISL1\"],\"target gene (associated gene name)\":[\"SLC2A10\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Using real-time PCR we have confirmed upregulation of Caps2\",\"Sec14l4\",\"Slc2a10\",\"P2rx7\",\"Afamin\",\"and Neurogenin 3 that may in part mediate the observed improved insulin secretion in Isl-1 overexpressing mice.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|a bit indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ISL1\"],\"tg (associated gene name)\":[\"SLC2A10\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using real-time PCR we have confirmed upregulation of Caps2, Sec14l4, Slc2a10, P2rx7, Afamin, and Neurogenin 3 that may in part mediate the observed improved insulin secretion in Isl-1 overexpressing mice.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CD2-driven BCL3 transgene expression---{\"tf associated gene name\":[\"BCL3\"],\"tg associated gene name\":[\"CD2\"],\"tf text\":[\"Bcl-3\"],\"tg text\":[\"CD2\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"CD2-driven BCL3 transgene expression\"],\"sentence\":[\"Bcl-3 was expressed in mice via a transgene driven by the human CD2 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no biolink---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BCL3\"],\"tg (associated gene name)\":[\"CD2\"],\"tf text\":[\"Bcl-3\"],\"tg text\":[\"CD2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no biolink\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Bcl-3 was expressed in mice via a transgene driven by the human CD2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "CD2-driven BCL3 transgene expression---{\"tf associated gene name\":[\"BCL3\"],\"tg associated gene name\":[\"CD2\"],\"tf text\":[\"Bcl-3\"],\"tg text\":[\"CD2\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"CD2-driven BCL3 transgene expression\"],\"sentence\":[\"Bcl-3 was expressed in mice via a transgene driven by the human CD2 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no biolink---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BCL3\"],\"tg (associated gene name)\":[\"CD2\"],\"tf text\":[\"Bcl-3\"],\"tg text\":[\"CD2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no biolink\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Bcl-3 was expressed in mice via a transgene driven by the human CD2 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 428 (char 427)\n",
      "{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"MMP1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Surprisingly\",\"although deletion of the MMP1 Ets cognate at nucleotides -88 to -83 abrogates FGF2 induction\",\"it does not prevent suppression of the AP1-dependent MMP1 promoter by PE1.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hmm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MMP1\"],\"tf text\":[],\"tg text\":[\"MMP1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"hmm\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Surprisingly, although deletion of the MMP1 Ets cognate at nucleotides -88 to -83 abrogates FGF2 induction, it does not prevent suppression of the AP1-dependent MMP1 promoter by PE1.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"MMP1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Surprisingly\",\"although deletion of the MMP1 Ets cognate at nucleotides -88 to -83 abrogates FGF2 induction\",\"it does not prevent suppression of the AP1-dependent MMP1 promoter by PE1.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hmm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MMP1\"],\"tf text\":[],\"tg text\":[\"MMP1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"hmm\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Surprisingly, although deletion of the MMP1 Ets cognate at nucleotides -88 to -83 abrogates FGF2 induction, it does not prevent suppression of the AP1-dependent MMP1 promoter by PE1.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPZ\"],\"tg (associated gene name)\":[\"HSPA4\"],\"tf text\":[\"CBF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"It is demonstrated here that p53 both represses transcription from the human hsp70 promoter and also interacts with CBF.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPZ\"],\"tg (associated gene name)\":[\"HSPA4\"],\"tf text\":[\"CBF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"It is demonstrated here that p53 both represses transcription from the human hsp70 promoter and also interacts with CBF.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is CD13/APN ---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"ADIPOQ\"],\"tf text\":[\"Ets-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is CD13/APN \"],\"sentence\":[\"CD13/APN is transactivated by a cooperative interaction between c-Myb bound to its cognate site and Ets-1 tethered to one of three GGA core-containing sites located 30-50 base pairs downstream.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"ADIPOQ\"],\"tf text\":[\"Ets-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"CD13/APN is transactivated by a cooperative interaction between c-Myb bound to its cognate site and Ets-1 tethered to one of three GGA core-containing sites located 30-50 base pairs downstream.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is CD13/APN ---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"ADIPOQ\"],\"tf text\":[\"Ets-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is CD13/APN \"],\"sentence\":[\"CD13/APN is transactivated by a cooperative interaction between c-Myb bound to its cognate site and Ets-1 tethered to one of three GGA core-containing sites located 30-50 base pairs downstream.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"ADIPOQ\"],\"tf text\":[\"Ets-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"CD13/APN is transactivated by a cooperative interaction between c-Myb bound to its cognate site and Ets-1 tethered to one of three GGA core-containing sites located 30-50 base pairs downstream.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  IL-2 receptor alpha---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"FCGRT\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is  IL-2 receptor alpha\"],\"sentence\":[\"An IL-2 response element in the human IL-2 receptor alpha chain promoter is a composite element that binds Stat5\",\"Elf-1\",\"HMG-I(Y) and a GATA family protein.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|alpha chain enough---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"FCGRT\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"alpha chain enough\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"An IL-2 response element in the human IL-2 receptor alpha chain promoter is a composite element that binds Stat5, Elf-1, HMG-I(Y) and a GATA family protein.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is  IL-2 receptor alpha---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"FCGRT\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is  IL-2 receptor alpha\"],\"sentence\":[\"An IL-2 response element in the human IL-2 receptor alpha chain promoter is a composite element that binds Stat5\",\"Elf-1\",\"HMG-I(Y) and a GATA family protein.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|alpha chain enough---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"FCGRT\"],\"tf text\":[\"Stat5\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"alpha chain enough\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"An IL-2 response element in the human IL-2 receptor alpha chain promoter is a composite element that binds Stat5, Elf-1, HMG-I(Y) and a GATA family protein.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is ADE2---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"BAX\"],\"tf text\":[\"p53\"],\"tg text\":[\"Bax\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is ADE2\"],\"sentence\":[\"We have determined the transactivation ability of 77 p53 mutants using reporter yeast strains containing a p53-regulated ADE2 gene whose promoter is regulated by p53 responsive elements derived from the regulatory region of the p21\",\"Bax and PIG3 genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is ADE2---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"BAX\"],\"tf text\":[\"p53\"],\"tg text\":[\"Bax\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is ADE2\"],\"sentence\":[\"We have determined the transactivation ability of 77 p53 mutants using reporter yeast strains containing a p53-regulated ADE2 gene whose promoter is regulated by p53 responsive elements derived from the regulatory region of the p21\",\"Bax and PIG3 genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"BAX\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We have determined the transactivation ability of 77 p53 mutants using reporter yeast strains containing a p53-regulated ADE2 gene whose promoter is regulated by p53 responsive elements derived from the regulatory region of the p21\",\"Bax and PIG3 genes.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"BAX\"],\"tf text\":[\"p53\"],\"tg text\":[\"Bax\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"We have determined the transactivation ability of 77 p53 mutants using reporter yeast strains containing a p53-regulated ADE2 gene whose promoter is regulated by p53 responsive elements derived from the regulatory region of the p21, Bax and PIG3 genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 497 (char 496)\n",
      "{\"transcription factor (associated gene name)\":[\"HNF1A\"],\"target gene (associated gene name)\":[\"INS\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Thus\",\"in addition to the previously described indirect action of HNF4 alpha on insulin gene expression mediated through elevated HNF1 alpha levels\",\"HNF4 alpha also activates the insulin gene directly\",\"through a previously unrecognized cis element.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|HNF1A declared indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF1A\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"HNF1 alpha\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"HNF1A declared indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Thus, in addition to the previously described indirect action of HNF4 alpha on insulin gene expression mediated through elevated HNF1 alpha levels, HNF4 alpha also activates the insulin gene directly, through a previously unrecognized cis element.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"HNF1A\"],\"target gene (associated gene name)\":[\"INS\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Thus\",\"in addition to the previously described indirect action of HNF4 alpha on insulin gene expression mediated through elevated HNF1 alpha levels\",\"HNF4 alpha also activates the insulin gene directly\",\"through a previously unrecognized cis element.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|HNF1A declared indirect?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF1A\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"HNF1 alpha\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"HNF1A declared indirect?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Thus, in addition to the previously described indirect action of HNF4 alpha on insulin gene expression mediated through elevated HNF1 alpha levels, HNF4 alpha also activates the insulin gene directly, through a previously unrecognized cis element.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is CIITA; TG is IL4---{\"tf associated gene name\":[\"HAND1\"],\"tg associated gene name\":[\"CIITA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is CIITA; TG is IL4\"],\"sentence\":[\"These results imply that CIITA is not a direct Th1-specific repressor of the IL-4 gene and that tight control over the expression of CIITA and MHC-II is required to maintain the normal balance between Th1 and Th2 responses.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is CIITA; TG is IL4---{\"tf associated gene name\":[\"HAND1\"],\"tg associated gene name\":[\"CIITA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is CIITA; TG is IL4\"],\"sentence\":[\"These results imply that CIITA is not a direct Th1-specific repressor of the IL-4 gene and that tight control over the expression of CIITA and MHC-II is required to maintain the normal balance between Th1 and Th2 responses.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HAND1\"],\"tg (associated gene name)\":[\"CIITA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results imply that CIITA is not a direct Th1-specific repressor of the IL-4 gene and that tight control over the expression of CIITA and MHC-II is required to maintain the normal balance between Th1 and Th2 responses.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG norm problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG norm problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Among the Sp1 target genes we looked at the possible alteration of binding to copper-zinc superoxide dismutase gene (sod1) promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG norm problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG norm problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Among the Sp1 target genes we looked at the possible alteration of binding to copper-zinc superoxide dismutase gene (sod1) promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named|not TRE---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"RAE1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"TG not named\",\"not TRE\"],\"sentence\":[\"DNase I footprinting studies using nuclear extract from NB41A3 suggested Sp1 binding to its consensus sequence at nucleotide -48 but inhibition of Sp1 binding at nucleotide -86 by the extract.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"RAE1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"DNase I footprinting studies using nuclear extract from NB41A3 suggested Sp1 binding to its consensus sequence at nucleotide -48 but inhibition of Sp1 binding at nucleotide -86 by the extract.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named|not TRE---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"RAE1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"TG not named\",\"not TRE\"],\"sentence\":[\"DNase I footprinting studies using nuclear extract from NB41A3 suggested Sp1 binding to its consensus sequence at nucleotide -48 but inhibition of Sp1 binding at nucleotide -86 by the extract.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"RAE1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"DNase I footprinting studies using nuclear extract from NB41A3 suggested Sp1 binding to its consensus sequence at nucleotide -48 but inhibition of Sp1 binding at nucleotide -86 by the extract.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"TLX1\"],\"tg associated gene name\":[\"SPTA1\"],\"tf text\":[\"HOX11\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"We hypothesized that HOX11 may be repressed in normal T-cells and erythroid cells by the action of negative elements which may be deleted or mutated in leukaemia.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TLX1\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[\"HOX11\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We hypothesized that HOX11 may be repressed in normal T-cells and erythroid cells by the action of negative elements which may be deleted or mutated in leukaemia.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"TLX1\"],\"tg associated gene name\":[\"SPTA1\"],\"tf text\":[\"HOX11\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"We hypothesized that HOX11 may be repressed in normal T-cells and erythroid cells by the action of negative elements which may be deleted or mutated in leukaemia.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TLX1\"],\"tg (associated gene name)\":[\"SPTA1\"],\"tf text\":[\"HOX11\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We hypothesized that HOX11 may be repressed in normal T-cells and erythroid cells by the action of negative elements which may be deleted or mutated in leukaemia.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"TTF1\"],\"tg associated gene name\":[\"TTF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Treatment of cells with hormones plus small inhibitory RNA directed toward TTF-1 reduced TTF-1 content by approximately 50% and inhibited hormone induction of the 23 genes induced by rTTF-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TTF1\"],\"tg (associated gene name)\":[\"TTF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Treatment of cells with hormones plus small inhibitory RNA directed toward TTF-1 reduced TTF-1 content by approximately 50% and inhibited hormone induction of the 23 genes induced by rTTF-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named---{\"tf associated gene name\":[\"TTF1\"],\"tg associated gene name\":[\"TTF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Treatment of cells with hormones plus small inhibitory RNA directed toward TTF-1 reduced TTF-1 content by approximately 50% and inhibited hormone induction of the 23 genes induced by rTTF-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TTF1\"],\"tg (associated gene name)\":[\"TTF1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Treatment of cells with hormones plus small inhibitory RNA directed toward TTF-1 reduced TTF-1 content by approximately 50% and inhibited hormone induction of the 23 genes induced by rTTF-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is ERalpha---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is ERalpha\"],\"sentence\":[\"PRL-induced ERalpha gene expression is mediated by Janus kinase 2 (Jak2) while signal transducer and activator of transcription 5b (Stat5b) phosphorylation involves Jak2 and a second tyrosine kinase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is ERalpha---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is ERalpha\"],\"sentence\":[\"PRL-induced ERalpha gene expression is mediated by Janus kinase 2 (Jak2) while signal transducer and activator of transcription 5b (Stat5b) phosphorylation involves Jak2 and a second tyrosine kinase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"PRL\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"PRL-induced ERalpha gene expression is mediated by Janus kinase 2 (Jak2) while signal transducer and activator of transcription 5b (Stat5b) phosphorylation involves Jak2 and a second tyrosine kinase.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.077965\"],\"pmid counts\":[\"11\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"10\"],\"sentence length\":[\"199\"],\"sentence pair density\":[\"0.0134228187919463\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PRL-induced ERalpha gene expression is mediated by Janus kinase 2 (Jak2) while signal transducer and activator of transcription 5b (Stat5b) phosphorylation involves Jak2 and a second tyrosine kinase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Current therapies include the use of antiandrogens aimed at inhibiting the transcriptional activation of AR-regulated genes by AR.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Current therapies include the use of antiandrogens aimed at inhibiting the transcriptional activation of AR-regulated genes by AR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Current therapies include the use of antiandrogens aimed at inhibiting the transcriptional activation of AR-regulated genes by AR.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Current therapies include the use of antiandrogens aimed at inhibiting the transcriptional activation of AR-regulated genes by AR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 384 (char 383)\n",
      "{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[\"c-Jun-NH(2) kinase\"],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Our previous study suggested that REIC/Dkk-3-induced apoptosis mainly resulted from phosphorylation of c-Jun-NH(2) kinase (JNK) in prostate cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no transcription implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[\"c-Jun-NH(2) kinase\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no transcription implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our previous study suggested that REIC/Dkk-3-induced apoptosis mainly resulted from phosphorylation of c-Jun-NH(2) kinase (JNK) in prostate cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[\"c-Jun-NH(2) kinase\"],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Our previous study suggested that REIC/Dkk-3-induced apoptosis mainly resulted from phosphorylation of c-Jun-NH(2) kinase (JNK) in prostate cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no transcription implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[\"c-Jun-NH(2) kinase\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no transcription implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our previous study suggested that REIC/Dkk-3-induced apoptosis mainly resulted from phosphorylation of c-Jun-NH(2) kinase (JNK) in prostate cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 392 (char 391)\n",
      "{\"transcription factor (associated gene name)\":[\"RBPJ\"],\"target gene (associated gene name)\":[\"MYC\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Furthermore\",\"N1IC\",\"YY1\",\"and alpha-enolase or MBP-1 but not CBF1 bound to the c-myc promoter through associating with the YY1 response element.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|NOT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RBPJ\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[\"CBF1\"],\"tg text\":[\"c-myc\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, N1IC, YY1, and alpha-enolase or MBP-1 but not CBF1 bound to the c-myc promoter through associating with the YY1 response element.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RBPJ\"],\"tg associated gene name\":[\"MYC\"],\"tf text\":[\"CBF1\"],\"tg text\":[\"c-myc\"],\"1\":[\"u\"],\"negation\":[\"x\"],\"not true\":[],\"comment\":[],\"sentence\":[\"Furthermore\",\"N1IC\",\"YY1\",\"and alpha-enolase or MBP-1 but not CBF1 bound to the c-myc promoter through associating with the YY1 response element.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RBPJ\"],\"target gene (associated gene name)\":[\"MYC\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Furthermore\",\"N1IC\",\"YY1\",\"and alpha-enolase or MBP-1 but not CBF1 bound to the c-myc promoter through associating with the YY1 response element.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[true],\"QC\":[true]}---|NOT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RBPJ\"],\"tg (associated gene name)\":[\"MYC\"],\"tf text\":[\"CBF1\"],\"tg text\":[\"c-myc\"],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, N1IC, YY1, and alpha-enolase or MBP-1 but not CBF1 bound to the c-myc promoter through associating with the YY1 response element.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "other GRE/TREs present---{\"tf associated gene name\":[\"IRF3\"],\"tg associated gene name\":[\"IKBKE\"],\"tf text\":[\"IRF-3\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"other GRE/TREs present\"],\"sentence\":[\"Paradoxically\",\"FOXA3 inhibited rhinovirus-induced IFN production\",\"IRF-3 phosphorylation\",\"and IKK?? expression and inhibited viral clearance and expression of genes required for antiviral defenses\",\"including MDA5\",\"RIG-I\",\"TLR3\",\"IRF7/9\",\"and nuclear factor-??B.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|FOAXA3 TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF3\"],\"tg (associated gene name)\":[\"IKBKE\"],\"tf text\":[\"IRF-3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"FOAXA3 TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Paradoxically, FOXA3 inhibited rhinovirus-induced IFN production, IRF-3 phosphorylation, and IKK expression and inhibited viral clearance and expression of genes required for antiviral defenses, including MDA5, RIG-I, TLR3, IRF7/9, and nuclear factor-B.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "other GRE/TREs present---{\"tf associated gene name\":[\"IRF3\"],\"tg associated gene name\":[\"IKBKE\"],\"tf text\":[\"IRF-3\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"other GRE/TREs present\"],\"sentence\":[\"Paradoxically\",\"FOXA3 inhibited rhinovirus-induced IFN production\",\"IRF-3 phosphorylation\",\"and IKK?? expression and inhibited viral clearance and expression of genes required for antiviral defenses\",\"including MDA5\",\"RIG-I\",\"TLR3\",\"IRF7/9\",\"and nuclear factor-??B.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|FOAXA3 TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF3\"],\"tg (associated gene name)\":[\"IKBKE\"],\"tf text\":[\"IRF-3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"FOAXA3 TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Paradoxically, FOXA3 inhibited rhinovirus-induced IFN production, IRF-3 phosphorylation, and IKK expression and inhibited viral clearance and expression of genes required for antiviral defenses, including MDA5, RIG-I, TLR3, IRF7/9, and nuclear factor-B.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 364 (char 363)\n",
      "{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CCR7\"],\"tf text\":[],\"tg text\":[\"CCR7\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"Immunohistochemical staining of a SCCHN tumor cohort (n = 47) strongly linked NF-??B staining and CCR7 expression in SCCHN.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|but association at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CCR7\"],\"tf text\":[],\"tg text\":[\"CCR7\"],\"consensus\":[\"N\"],\"martin_comments\":[\"but association at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Immunohistochemical staining of a SCCHN tumor cohort (n = 47) strongly linked NF-B staining and CCR7 expression in SCCHN.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CCR7\"],\"tf text\":[],\"tg text\":[\"CCR7\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"Immunohistochemical staining of a SCCHN tumor cohort (n = 47) strongly linked NF-??B staining and CCR7 expression in SCCHN.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|but association at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CCR7\"],\"tf text\":[],\"tg text\":[\"CCR7\"],\"consensus\":[\"N\"],\"martin_comments\":[\"but association at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Immunohistochemical staining of a SCCHN tumor cohort (n = 47) strongly linked NF-B staining and CCR7 expression in SCCHN.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 339 (char 338)\n",
      "{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"CTNNB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"PLK1 then potentiated the expression of Stat3; ??-catenin was involved in PLK1-dependent transcriptional activation of Stat3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|reverse---{\"_\":[\"_\"],\"tf\":[\"STAT3\"],\"tg\":[\"CTNNB1\"],\"validation\":[\"N\"],\"comment 1\":[\"reverse\"],\"comment 2\":[],\"\":[],\"sentence\":[\"PLK1 then potentiated the expression of Stat3; -catenin was involved in PLK1-dependent transcriptional activation of Stat3.\"],\"interaction score\":[\"2.633672\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"CTNNB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"PLK1 then potentiated the expression of Stat3; ??-catenin was involved in PLK1-dependent transcriptional activation of Stat3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|reverse---{\"_\":[\"_\"],\"tf\":[\"STAT3\"],\"tg\":[\"CTNNB1\"],\"validation\":[\"N\"],\"comment 1\":[\"reverse\"],\"comment 2\":[],\"\":[],\"sentence\":[\"PLK1 then potentiated the expression of Stat3; -catenin was involved in PLK1-dependent transcriptional activation of Stat3.\"],\"interaction score\":[\"2.633672\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"STAT3\"],\"target gene (associated gene name)\":[\"CTNNB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"PLK1 then potentiated the expression of Stat3; -catenin was involved in PLK1-dependent transcriptional activation of Stat3.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.633672\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"123\"],\"sentence pair density\":[\"0.0224719101123595\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|STAT3 is target---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"CTNNB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"STAT3 is target\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PLK1 then potentiated the expression of Stat3; -catenin was involved in PLK1-dependent transcriptional activation of Stat3.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 433 (char 432)\n",
      "{\"transcription factor (associated gene name)\":[\"GATA6\"],\"target gene (associated gene name)\":[\"VWF\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We propose that NFY recruits HDACs to the VWF promoter\",\"which may result in deacetylation of GATA6 as well as of histones in non-endothelial cells\",\"thus leading to promoter inactivation.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|GATA6 only implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA6\"],\"tg (associated gene name)\":[\"VWF\"],\"tf text\":[\"GATA6\"],\"tg text\":[\"VWF\"],\"consensus\":[\"V\"],\"martin_comments\":[\"GATA6 only implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We propose that NFY recruits HDACs to the VWF promoter, which may result in deacetylation of GATA6 as well as of histones in non-endothelial cells, thus leading to promoter inactivation.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"GATA6\"],\"tg associated gene name\":[\"VWF\"],\"tf text\":[\"GATA6\"],\"tg text\":[\"VWF\"],\"1\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"We propose that NFY recruits HDACs to the VWF promoter\",\"which may result in deacetylation of GATA6 as well as of histones in non-endothelial cells\",\"thus leading to promoter inactivation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"GATA6\"],\"target gene (associated gene name)\":[\"VWF\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We propose that NFY recruits HDACs to the VWF promoter\",\"which may result in deacetylation of GATA6 as well as of histones in non-endothelial cells\",\"thus leading to promoter inactivation.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|GATA6 only implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA6\"],\"tg (associated gene name)\":[\"VWF\"],\"tf text\":[\"GATA6\"],\"tg text\":[\"VWF\"],\"consensus\":[\"V\"],\"martin_comments\":[\"GATA6 only implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We propose that NFY recruits HDACs to the VWF promoter, which may result in deacetylation of GATA6 as well as of histones in non-endothelial cells, thus leading to promoter inactivation.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p21(Cip1) (CDKN1A) ---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"CDKN3\"],\"tf text\":[\"Mitf\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is p21(Cip1) (CDKN1A) \"],\"sentence\":[\"Here we show that Mitf can act as a novel anti-proliferative transcription factor able to induce a G1 cell-cycle arrest that is dependent on Mitf-mediated activation of the p21(Cip1) (CDKN1A) cyclin-dependent kinase inhibitor gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not cip2---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"CDKN3\"],\"tf text\":[\"Mitf\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not cip2\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we show that Mitf can act as a novel anti-proliferative transcription factor able to induce a G1 cell-cycle arrest that is dependent on Mitf-mediated activation of the p21(Cip1) (CDKN1A) cyclin-dependent kinase inhibitor gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is p21(Cip1) (CDKN1A) ---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"CDKN3\"],\"tf text\":[\"Mitf\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is p21(Cip1) (CDKN1A) \"],\"sentence\":[\"Here we show that Mitf can act as a novel anti-proliferative transcription factor able to induce a G1 cell-cycle arrest that is dependent on Mitf-mediated activation of the p21(Cip1) (CDKN1A) cyclin-dependent kinase inhibitor gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not cip2---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"CDKN3\"],\"tf text\":[\"Mitf\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not cip2\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we show that Mitf can act as a novel anti-proliferative transcription factor able to induce a G1 cell-cycle arrest that is dependent on Mitf-mediated activation of the p21(Cip1) (CDKN1A) cyclin-dependent kinase inhibitor gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ets family?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"ets family?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Members of the Ets family of transcription factors are key regulators controlling prolactin (PRL) gene expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "ets family?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"ets family?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Members of the Ets family of transcription factors are key regulators controlling prolactin (PRL) gene expression.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is crystallin genes---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"SMARCA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is crystallin genes\"],\"sentence\":[\"The transcriptional control of crystallin genes is characterized by the reiterative use of transcription factors required for the establishment of lens precursors in combination with more ubiquitously expressed factors (e.g. AP-1\",\"AP-2alpha\",\"CREB and USF) and recruitment of histone acetyltransferases (HATs) CBP and p300\",\"and chromatin remodeling complexes SWI/SNF and ISWI.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TFAP2A\"],\"tg (associated gene name)\":[\"SMARCA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcriptional control of crystallin genes is characterized by the reiterative use of transcription factors required for the establishment of lens precursors in combination with more ubiquitously expressed factors (e.g. AP-1, AP-2alpha, CREB and USF) and recruitment of histone acetyltransferases (HATs) CBP and p300, and chromatin remodeling complexes SWI/SNF and ISWI.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is crystallin genes---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"SMARCA1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is crystallin genes\"],\"sentence\":[\"The transcriptional control of crystallin genes is characterized by the reiterative use of transcription factors required for the establishment of lens precursors in combination with more ubiquitously expressed factors (e.g. AP-1\",\"AP-2alpha\",\"CREB and USF) and recruitment of histone acetyltransferases (HATs) CBP and p300\",\"and chromatin remodeling complexes SWI/SNF and ISWI.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TFAP2A\"],\"tg (associated gene name)\":[\"SMARCA1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcriptional control of crystallin genes is characterized by the reiterative use of transcription factors required for the establishment of lens precursors in combination with more ubiquitously expressed factors (e.g. AP-1, AP-2alpha, CREB and USF) and recruitment of histone acetyltransferases (HATs) CBP and p300, and chromatin remodeling complexes SWI/SNF and ISWI.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p21---{\"tf associated gene name\":[\"JUND\"],\"tg associated gene name\":[\"JUND\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"TG is p21\"],\"sentence\":[\"Polyamines negatively regulate junD gene expression posttranscriptionally\",\"and increased JunD/AP-1 inhibits intestinal epithelial cell proliferation at least partially through the activation of p21 promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|nautoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUND\"],\"tg (associated gene name)\":[\"JUND\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"nautoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Polyamines negatively regulate junD gene expression posttranscriptionally, and increased JunD/AP-1 inhibits intestinal epithelial cell proliferation at least partially through the activation of p21 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is p21---{\"tf associated gene name\":[\"JUND\"],\"tg associated gene name\":[\"JUND\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"TG is p21\"],\"sentence\":[\"Polyamines negatively regulate junD gene expression posttranscriptionally\",\"and increased JunD/AP-1 inhibits intestinal epithelial cell proliferation at least partially through the activation of p21 promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|nautoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUND\"],\"tg (associated gene name)\":[\"JUND\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"nautoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Polyamines negatively regulate junD gene expression posttranscriptionally, and increased JunD/AP-1 inhibits intestinal epithelial cell proliferation at least partially through the activation of p21 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|TF not named---{\"tf associated gene name\":[\"ZHX2\"],\"tg associated gene name\":[\"CDKN3\"],\"tf text\":[\"Raf\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"TF not named\"],\"sentence\":[\"p21Cip1/Waf1 cyclin-dependent kinase inhibitor (p21) is inducible by Raf and mitogen-activated protein kinase kinase (MAPKK)\",\"but the level of regulation is unknown.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG norm problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZHX2\"],\"tg (associated gene name)\":[\"CDKN3\"],\"tf text\":[\"Raf\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG norm problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"p21Cip1/Waf1 cyclin-dependent kinase inhibitor (p21) is inducible by Raf and mitogen-activated protein kinase kinase (MAPKK), but the level of regulation is unknown.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE|TF not named---{\"tf associated gene name\":[\"ZHX2\"],\"tg associated gene name\":[\"CDKN3\"],\"tf text\":[\"Raf\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"TF not named\"],\"sentence\":[\"p21Cip1/Waf1 cyclin-dependent kinase inhibitor (p21) is inducible by Raf and mitogen-activated protein kinase kinase (MAPKK)\",\"but the level of regulation is unknown.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG norm problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZHX2\"],\"tg (associated gene name)\":[\"CDKN3\"],\"tf text\":[\"Raf\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG norm problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"p21Cip1/Waf1 cyclin-dependent kinase inhibitor (p21) is inducible by Raf and mitogen-activated protein kinase kinase (MAPKK), but the level of regulation is unknown.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 346 (char 345)\n",
      "{\"transcription factor (associated gene name)\":[\"MYBL2\"],\"target gene (associated gene name)\":[\"COL1A1\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of B-Myb in SSc fibroblasts was correlated with decreased COL1A1 mRNA expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|association|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYBL2\"],\"tg (associated gene name)\":[\"COL1A1\"],\"tf text\":[\"B-Myb\"],\"tg text\":[\"COL1A1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"association\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Overexpression of B-Myb in SSc fibroblasts was correlated with decreased COL1A1 mRNA expression.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"MYBL2\"],\"target gene (associated gene name)\":[\"COL1A1\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of B-Myb in SSc fibroblasts was correlated with decreased COL1A1 mRNA expression.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|association|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYBL2\"],\"tg (associated gene name)\":[\"COL1A1\"],\"tf text\":[\"B-Myb\"],\"tg text\":[\"COL1A1\"],\"consensus\":[\"V\"],\"martin_comments\":[\"association\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Overexpression of B-Myb in SSc fibroblasts was correlated with decreased COL1A1 mRNA expression.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG named|not TRE---{\"tf associated gene name\":[\"NFATC4\"],\"tg associated gene name\":[\"NUP214\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no TG named\",\"not TRE\"],\"sentence\":[\"Maintaining immature CGNs in a depolarized state blocked NFI temporal occupancy of late-expressed genes and the NFI switch program via activation of the CaN/nuclear factor of activated T-cells\",\"cytoplasmic (NFATc) pathway and promotion of late-gene occupancy by NFATc4\",\"and these mechanisms inhibited dendritogenesis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFATC4\"],\"tg (associated gene name)\":[\"NUP214\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Maintaining immature CGNs in a depolarized state blocked NFI temporal occupancy of late-expressed genes and the NFI switch program via activation of the CaN/nuclear factor of activated T-cells, cytoplasmic (NFATc) pathway and promotion of late-gene occupancy by NFATc4, and these mechanisms inhibited dendritogenesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG named|not TRE---{\"tf associated gene name\":[\"NFATC4\"],\"tg associated gene name\":[\"NUP214\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no TG named\",\"not TRE\"],\"sentence\":[\"Maintaining immature CGNs in a depolarized state blocked NFI temporal occupancy of late-expressed genes and the NFI switch program via activation of the CaN/nuclear factor of activated T-cells\",\"cytoplasmic (NFATc) pathway and promotion of late-gene occupancy by NFATc4\",\"and these mechanisms inhibited dendritogenesis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFATC4\"],\"tg (associated gene name)\":[\"NUP214\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Maintaining immature CGNs in a depolarized state blocked NFI temporal occupancy of late-expressed genes and the NFI switch program via activation of the CaN/nuclear factor of activated T-cells, cytoplasmic (NFATc) pathway and promotion of late-gene occupancy by NFATc4, and these mechanisms inhibited dendritogenesis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is EGFR---{\"tf associated gene name\":[\"LRRFIP1\"],\"tg associated gene name\":[\"GC\"],\"tf text\":[\"GCF2\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is EGFR\"],\"sentence\":[\"GC-binding factor 2 (GCF2)\",\"a transcriptional repressor that decreases the activity of several genes is capable of binding directly to the GC-rich sequence of the EGFR promoter and repressing the transcriptional activity of EGFR.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG norm on GC rich :-)---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LRRFIP1\"],\"tg (associated gene name)\":[\"GC\"],\"tf text\":[\"GCF2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG norm on GC rich :-)\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"GC-binding factor 2 (GCF2), a transcriptional repressor that decreases the activity of several genes is capable of binding directly to the GC-rich sequence of the EGFR promoter and repressing the transcriptional activity of EGFR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is EGFR---{\"tf associated gene name\":[\"LRRFIP1\"],\"tg associated gene name\":[\"GC\"],\"tf text\":[\"GCF2\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is EGFR\"],\"sentence\":[\"GC-binding factor 2 (GCF2)\",\"a transcriptional repressor that decreases the activity of several genes is capable of binding directly to the GC-rich sequence of the EGFR promoter and repressing the transcriptional activity of EGFR.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG norm on GC rich :-)---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LRRFIP1\"],\"tg (associated gene name)\":[\"GC\"],\"tf text\":[\"GCF2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG norm on GC rich :-)\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"GC-binding factor 2 (GCF2), a transcriptional repressor that decreases the activity of several genes is capable of binding directly to the GC-rich sequence of the EGFR promoter and repressing the transcriptional activity of EGFR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is ETS-1---{\"tf associated gene name\":[\"TP63\"],\"tg associated gene name\":[\"MMP3\"],\"tf text\":[],\"tg text\":[\"stromelysin-1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is ETS-1\"],\"sentence\":[\"By using surface plasmon resonance\",\"electrophoretic mobility shift assay\",\"and photo-cross-linking\",\"we showed that full-length human ETS-1 (p51) binds cooperatively to the ETS-binding site palindrome of the human stromelysin-1 promoter\",\"with facilitated binding of the second ETS-1 molecule to form an ETS-1.DNA.ETS-1 ternary complex.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is ETS-1---{\"tf associated gene name\":[\"TP63\"],\"tg associated gene name\":[\"MMP3\"],\"tf text\":[],\"tg text\":[\"stromelysin-1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is ETS-1\"],\"sentence\":[\"By using surface plasmon resonance\",\"electrophoretic mobility shift assay\",\"and photo-cross-linking\",\"we showed that full-length human ETS-1 (p51) binds cooperatively to the ETS-binding site palindrome of the human stromelysin-1 promoter\",\"with facilitated binding of the second ETS-1 molecule to form an ETS-1.DNA.ETS-1 ternary complex.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TP63\"],\"target gene (associated gene name)\":[\"MMP3\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"By using surface plasmon resonance\",\"electrophoretic mobility shift assay\",\"and photo-cross-linking\",\"we showed that full-length human ETS-1 (p51) binds cooperatively to the ETS-binding site palindrome of the human stromelysin-1 promoter\",\"with facilitated binding of the second ETS-1 molecule to form an ETS-1.DNA.ETS-1 ternary complex.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP63\"],\"tg (associated gene name)\":[\"MMP3\"],\"tf text\":[],\"tg text\":[\"stromelysin-1\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"By using surface plasmon resonance, electrophoretic mobility shift assay, and photo-cross-linking, we showed that full-length human ETS-1 (p51) binds cooperatively to the ETS-binding site palindrome of the human stromelysin-1 promoter, with facilitated binding of the second ETS-1 molecule to form an ETS-1.DNA.ETS-1 ternary complex.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"APRF\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"Remarkably\",\"APRF was also activated by IFN alpha\",\"IFN gamma\",\"epidermal growth factor\",\"platelet-derived growth factor\",\"colony stimulating factor-1\",\"and the cytokines leukemia inhibitory factor and oncostatin M.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|APRF is TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"APRF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"APRF is TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Remarkably, APRF was also activated by IFN alpha, IFN gamma, epidermal growth factor, platelet-derived growth factor, colony stimulating factor-1, and the cytokines leukemia inhibitory factor and oncostatin M.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"APRF\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"Remarkably\",\"APRF was also activated by IFN alpha\",\"IFN gamma\",\"epidermal growth factor\",\"platelet-derived growth factor\",\"colony stimulating factor-1\",\"and the cytokines leukemia inhibitory factor and oncostatin M.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|APRF is TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT3\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"APRF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"APRF is TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Remarkably, APRF was also activated by IFN alpha, IFN gamma, epidermal growth factor, platelet-derived growth factor, colony stimulating factor-1, and the cytokines leukemia inhibitory factor and oncostatin M.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs are urokinase-type plasminogen activator and collagenase 1---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"MMP13\"],\"tf text\":[\"ETS1\"],\"tg text\":[],\"1\":[\"u-fam\"],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TGs are urokinase-type plasminogen activator and collagenase 1\"],\"sentence\":[\"Furthermore\",\"we showed that SF/HGF was able to induce a transcriptional response involving ETS1 by using artificial as well as cellular promoters\",\"such as the urokinase-type plasminogen activator and collagenase 1 promoters\",\"containing RAS-responsive elements with essential ETS-binding sites.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"MMP13\"],\"tf text\":[\"ETS1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, we showed that SF/HGF was able to induce a transcriptional response involving ETS1 by using artificial as well as cellular promoters, such as the urokinase-type plasminogen activator and collagenase 1 promoters, containing RAS-responsive elements with essential ETS-binding sites.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TGs are urokinase-type plasminogen activator and collagenase 1---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"MMP13\"],\"tf text\":[\"ETS1\"],\"tg text\":[],\"1\":[\"u-fam\"],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TGs are urokinase-type plasminogen activator and collagenase 1\"],\"sentence\":[\"Furthermore\",\"we showed that SF/HGF was able to induce a transcriptional response involving ETS1 by using artificial as well as cellular promoters\",\"such as the urokinase-type plasminogen activator and collagenase 1 promoters\",\"containing RAS-responsive elements with essential ETS-binding sites.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"MMP13\"],\"tf text\":[\"ETS1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, we showed that SF/HGF was able to induce a transcriptional response involving ETS1 by using artificial as well as cellular promoters, such as the urokinase-type plasminogen activator and collagenase 1 promoters, containing RAS-responsive elements with essential ETS-binding sites.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"NOS2\"],\"tf text\":[],\"tg text\":[\"NOS2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Over-expression of SUMO-1, Ubc9, or SENP1 reduced the activation of a NOS2 promoter, whereas activation of a 4 x NFkappaB binding-element reporter was only reduced by SUMO-1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"NOS2\"],\"tf text\":[],\"tg text\":[\"NOS2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Over-expression of SUMO-1, Ubc9, or SENP1 reduced the activation of a NOS2 promoter, whereas activation of a 4 x NFkappaB binding-element reporter was only reduced by SUMO-1.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CAT-problem (reportergene)|TF family member not clear---{\"tf associated gene name\":[\"CEBPG\"],\"tg associated gene name\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"CAT-problem (reportergene)\",\"TF family member not clear\"],\"sentence\":[\"The factor limiting transcription by nuclear extract from preadipocytes appears to be C/EBP\",\"since recombinant C/EBP (rC/EBP) markedly activates transcription of the 422(aP2) promoter-CAT gene with preadipocyte extract but not with adipocyte extract.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CAT problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPG\"],\"tg (associated gene name)\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CAT problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The factor limiting transcription by nuclear extract from preadipocytes appears to be C/EBP, since recombinant C/EBP (rC/EBP) markedly activates transcription of the 422(aP2) promoter-CAT gene with preadipocyte extract but not with adipocyte extract.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "CAT-problem (reportergene)|TF family member not clear---{\"tf associated gene name\":[\"CEBPG\"],\"tg associated gene name\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"CAT-problem (reportergene)\",\"TF family member not clear\"],\"sentence\":[\"The factor limiting transcription by nuclear extract from preadipocytes appears to be C/EBP\",\"since recombinant C/EBP (rC/EBP) markedly activates transcription of the 422(aP2) promoter-CAT gene with preadipocyte extract but not with adipocyte extract.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|CAT problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPG\"],\"tg (associated gene name)\":[\"CAT\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"CAT problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The factor limiting transcription by nuclear extract from preadipocytes appears to be C/EBP, since recombinant C/EBP (rC/EBP) markedly activates transcription of the 422(aP2) promoter-CAT gene with preadipocyte extract but not with adipocyte extract.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"SALL4\"],\"tg associated gene name\":[\"HDAC9\"],\"tf text\":[\"SALL4\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Components of a transcription repressor Mi-2/Nucleosome Remodeling and Deacetylase (NuRD) complex were found in the SALL4-immunocomplexes with histone deacetylase (HDAC) activity in ESCs with endogenous SALL4 expression and 293T cells overexpressing SALL4.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SALL4\"],\"tg (associated gene name)\":[\"HDAC9\"],\"tf text\":[\"SALL4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Components of a transcription repressor Mi-2/Nucleosome Remodeling and Deacetylase (NuRD) complex were found in the SALL4-immunocomplexes with histone deacetylase (HDAC) activity in ESCs with endogenous SALL4 expression and 293T cells overexpressing SALL4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"SALL4\"],\"tg associated gene name\":[\"HDAC9\"],\"tf text\":[\"SALL4\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Components of a transcription repressor Mi-2/Nucleosome Remodeling and Deacetylase (NuRD) complex were found in the SALL4-immunocomplexes with histone deacetylase (HDAC) activity in ESCs with endogenous SALL4 expression and 293T cells overexpressing SALL4.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SALL4\"],\"tg (associated gene name)\":[\"HDAC9\"],\"tf text\":[\"SALL4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Components of a transcription repressor Mi-2/Nucleosome Remodeling and Deacetylase (NuRD) complex were found in the SALL4-immunocomplexes with histone deacetylase (HDAC) activity in ESCs with endogenous SALL4 expression and 293T cells overexpressing SALL4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no IL2RG mentione|TG is  E4 and E2A adenovirus promoter|---{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"IL2RG\"],\"tf text\":[\"c-myc\"],\"tg text\":[\"p64\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\",\"s\"],\"comment\":[\"no IL2RG mentione\",\"TG is  E4 and E2A adenovirus promoter\",\"\"],\"sentence\":[\"We have also observed that only a c-myc plasmid coding for both p67 and 64 proteins\",\"in contrast to one coding for p64 only\",\"is able to trans-activate the E4 and E2A adenovirus promoter\",\"suggesting that the p67 protein plays an essential part in activation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"IL2RG\"],\"tf text\":[\"c-myc\"],\"tg text\":[\"p64\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We have also observed that only a c-myc plasmid coding for both p67 and 64 proteins, in contrast to one coding for p64 only, is able to trans-activate the E4 and E2A adenovirus promoter, suggesting that the p67 protein plays an essential part in activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no IL2RG mentione|TG is  E4 and E2A adenovirus promoter|---{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"IL2RG\"],\"tf text\":[\"c-myc\"],\"tg text\":[\"p64\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\",\"s\"],\"comment\":[\"no IL2RG mentione\",\"TG is  E4 and E2A adenovirus promoter\",\"\"],\"sentence\":[\"We have also observed that only a c-myc plasmid coding for both p67 and 64 proteins\",\"in contrast to one coding for p64 only\",\"is able to trans-activate the E4 and E2A adenovirus promoter\",\"suggesting that the p67 protein plays an essential part in activation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"IL2RG\"],\"tf text\":[\"c-myc\"],\"tg text\":[\"p64\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We have also observed that only a c-myc plasmid coding for both p67 and 64 proteins, in contrast to one coding for p64 only, is able to trans-activate the E4 and E2A adenovirus promoter, suggesting that the p67 protein plays an essential part in activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is EphA2---{\"tf associated gene name\":[\"HIC1\"],\"tg associated gene name\":[\"NTRK1\"],\"tf text\":[\"HIC1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is EphA2\"],\"sentence\":[\"To conclude\",\"our results involve the tumor suppressor HIC1 in the transcriptional regulation of the tyrosine kinase receptor EphA2\",\"whose ligand ephrin-A1 is also a HIC1 target gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG norm problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIC1\"],\"tg (associated gene name)\":[\"NTRK1\"],\"tf text\":[\"HIC1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG norm problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To conclude, our results involve the tumor suppressor HIC1 in the transcriptional regulation of the tyrosine kinase receptor EphA2, whose ligand ephrin-A1 is also a HIC1 target gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is EphA2---{\"tf associated gene name\":[\"HIC1\"],\"tg associated gene name\":[\"NTRK1\"],\"tf text\":[\"HIC1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is EphA2\"],\"sentence\":[\"To conclude\",\"our results involve the tumor suppressor HIC1 in the transcriptional regulation of the tyrosine kinase receptor EphA2\",\"whose ligand ephrin-A1 is also a HIC1 target gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG norm problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIC1\"],\"tg (associated gene name)\":[\"NTRK1\"],\"tf text\":[\"HIC1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG norm problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To conclude, our results involve the tumor suppressor HIC1 in the transcriptional regulation of the tyrosine kinase receptor EphA2, whose ligand ephrin-A1 is also a HIC1 target gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 447 (char 446)\n",
      "{\"tf associated gene name\":[\"ETV6\"],\"tg associated gene name\":[\"UBE2I\"],\"tf text\":[\"TEL\"],\"tg text\":[\"UBC9\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We also show that TEL expressed as fusion to the DNA-binding domain of Gal4 completely represses a Gal4-responsive promoter\",\"but that the coexpression of UBC9 in the same system restores the activity of the promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETV6\"],\"tg (associated gene name)\":[\"UBE2I\"],\"tf text\":[\"TEL\"],\"tg text\":[\"UBC9\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We also show that TEL expressed as fusion to the DNA-binding domain of Gal4 completely represses a Gal4-responsive promoter, but that the coexpression of UBC9 in the same system restores the activity of the promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ETV6\"],\"tg associated gene name\":[\"UBE2I\"],\"tf text\":[\"TEL\"],\"tg text\":[\"UBC9\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We also show that TEL expressed as fusion to the DNA-binding domain of Gal4 completely represses a Gal4-responsive promoter\",\"but that the coexpression of UBC9 in the same system restores the activity of the promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETV6\"],\"tg (associated gene name)\":[\"UBE2I\"],\"tf text\":[\"TEL\"],\"tg text\":[\"UBC9\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We also show that TEL expressed as fusion to the DNA-binding domain of Gal4 completely represses a Gal4-responsive promoter, but that the coexpression of UBC9 in the same system restores the activity of the promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "causality between proposed TF and TG?|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IFNB1\"],\"tf text\":[],\"tg text\":[\"IFN-beta\"],\"consensus\":[\"N\"],\"martin_comments\":[\"causality between proposed TF and TG?\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"By reporter gene assay using HEK293 cells transfected with a human TLR3 expression vector, TLR3 recognized dsRNA to activate NF-kappaB and the IFN-beta promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "causality between proposed TF and TG?|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IFNB1\"],\"tf text\":[],\"tg text\":[\"IFN-beta\"],\"consensus\":[\"N\"],\"martin_comments\":[\"causality between proposed TF and TG?\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"By reporter gene assay using HEK293 cells transfected with a human TLR3 expression vector, TLR3 recognized dsRNA to activate NF-kappaB and the IFN-beta promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"MET\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In EC cell lines, transfection of the NF-kappaB subunit p65 led to transactivation of Par-4 through specific binding to its promoter region, in contrast to the suppression by active Akt, suggesting that the balance between the two signals may be important to determine Par-4 expression levels.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"MET\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In EC cell lines, transfection of the NF-kappaB subunit p65 led to transactivation of Par-4 through specific binding to its promoter region, in contrast to the suppression by active Akt, suggesting that the balance between the two signals may be important to determine Par-4 expression levels.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 585 (char 584)\n",
      "{\"tf associated gene name\":[\"SOX2\"],\"tg associated gene name\":[\"CCNA2\"],\"tf text\":[\"SOX2\"],\"tg text\":[\"CCNA2\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"Affymetrix and real-time PCR studies of siRNA-treated human PS cells showed that reduced RBBP4 or RBBP9 expression concomitantly decreased expression of POU5F1\",\"NANOG\",\"SOX2\",\"and/or FOXD3 plus certain cell cycle genes (e.g.\",\"CCNA2\",\"CCNB1)\",\"while increasing expression of genes involved in organogenesis (particularly neurogenesis).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|correlation only---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SOX2\"],\"tg (associated gene name)\":[\"CCNA2\"],\"tf text\":[\"SOX2\"],\"tg text\":[\"CCNA2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"correlation only\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Affymetrix and real-time PCR studies of siRNA-treated human PS cells showed that reduced RBBP4 or RBBP9 expression concomitantly decreased expression of POU5F1, NANOG, SOX2, and/or FOXD3 plus certain cell cycle genes (e.g., CCNA2, CCNB1), while increasing expression of genes involved in organogenesis (particularly neurogenesis).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SOX2\"],\"tg associated gene name\":[\"CCNA2\"],\"tf text\":[\"SOX2\"],\"tg text\":[\"CCNA2\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"Affymetrix and real-time PCR studies of siRNA-treated human PS cells showed that reduced RBBP4 or RBBP9 expression concomitantly decreased expression of POU5F1\",\"NANOG\",\"SOX2\",\"and/or FOXD3 plus certain cell cycle genes (e.g.\",\"CCNA2\",\"CCNB1)\",\"while increasing expression of genes involved in organogenesis (particularly neurogenesis).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|correlation only---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SOX2\"],\"tg (associated gene name)\":[\"CCNA2\"],\"tf text\":[\"SOX2\"],\"tg text\":[\"CCNA2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"correlation only\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Affymetrix and real-time PCR studies of siRNA-treated human PS cells showed that reduced RBBP4 or RBBP9 expression concomitantly decreased expression of POU5F1, NANOG, SOX2, and/or FOXD3 plus certain cell cycle genes (e.g., CCNA2, CCNB1), while increasing expression of genes involved in organogenesis (particularly neurogenesis).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ER stress is not ESR1; TF is ATF-6---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"HSPA5\"],\"tf text\":[],\"tg text\":[\"grp-78\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"\\\"ER stress\\\" is not ESR1; TF is ATF-6\"],\"sentence\":[\"IL-10 inhibits inflammation-induced ER stress response mechanisms by modulating ATF-6 nuclear recruitment to the grp-78 gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "ER stress is not ESR1; TF is ATF-6---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"HSPA5\"],\"tf text\":[],\"tg text\":[\"grp-78\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"\\\"ER stress\\\" is not ESR1; TF is ATF-6\"],\"sentence\":[\"IL-10 inhibits inflammation-induced ER stress response mechanisms by modulating ATF-6 nuclear recruitment to the grp-78 gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"HSPA5\"],\"tf text\":[],\"tg text\":[\"grp-78\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"IL-10 inhibits inflammation-induced ER stress response mechanisms by modulating ATF-6 nuclear recruitment to the grp-78 gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 445 (char 444)\n",
      "{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"MMP1\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Androgen receptor negatively regulates matrix metalloproteinase-1 expression not through AP-1 but through a family of Ets-related transcription factors that are also required for positive regulation.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|NOT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MMP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Androgen receptor negatively regulates matrix metalloproteinase-1 expression not through AP-1 but through a family of Ets-related transcription factors that are also required for positive regulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"MMP1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[\"x\"],\"not true\":[],\"comment\":[],\"sentence\":[\"Androgen receptor negatively regulates matrix metalloproteinase-1 expression not through AP-1 but through a family of Ets-related transcription factors that are also required for positive regulation.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[true],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"MMP1\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Androgen receptor negatively regulates matrix metalloproteinase-1 expression not through AP-1 but through a family of Ets-related transcription factors that are also required for positive regulation.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|NOT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MMP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Androgen receptor negatively regulates matrix metalloproteinase-1 expression not through AP-1 but through a family of Ets-related transcription factors that are also required for positive regulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Finally, the selective NF-kappa B inhibitor MG-132 dose dependently reduced NF-kappa B binding and beta(2)-AR promoter activity but increased AP-1 binding.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Finally, the selective NF-kappa B inhibitor MG-132 dose dependently reduced NF-kappa B binding and beta(2)-AR promoter activity but increased AP-1 binding.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 802 (char 801)\n",
      "{\"transcription factor (associated gene name)\":[\"TWIST1\"],\"target gene (associated gene name)\":[\"TNFRSF11A\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"The results of this study indicate that Twist1 induces HCC cell plasticity in VM cells more through the suppression of E-cadherin expression and the induction of VE-cadherin up-regulation than through the VM pattern in vivo and in a three-dimensional in vitro system.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.340733\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"267\"],\"sentence pair density\":[\"0.0324324324324324\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TWIST1\"],\"tg (associated gene name)\":[\"TNFRSF11A\"],\"tf text\":[\"Twist1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The results of this study indicate that Twist1 induces HCC cell plasticity in VM cells more through the suppression of E-cadherin expression and the induction of VE-cadherin up-regulation than through the VM pattern in vivo and in a three-dimensional in vitro system.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"TWIST1\"],\"target gene (associated gene name)\":[\"TNFRSF11A\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"The results of this study indicate that Twist1 induces HCC cell plasticity in VM cells more through the suppression of E-cadherin expression and the induction of VE-cadherin up-regulation than through the VM pattern in vivo and in a three-dimensional in vitro system.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.340733\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"267\"],\"sentence pair density\":[\"0.0324324324324324\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TWIST1\"],\"tg (associated gene name)\":[\"TNFRSF11A\"],\"tf text\":[\"Twist1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The results of this study indicate that Twist1 induces HCC cell plasticity in VM cells more through the suppression of E-cadherin expression and the induction of VE-cadherin up-regulation than through the VM pattern in vivo and in a three-dimensional in vitro system.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is fam member UGT2B1---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"UGT1A1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a-fam\"],\"negation\":[],\"not true\":[],\"comment\":[\"TG is fam member UGT2B1\"],\"sentence\":[\"These data strongly suggest that the liver-enriched factor C/EBPalpha binds to\",\"and activates\",\"the UGT2B1 gene promoter.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong UGT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"UGT1A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong UGT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These data strongly suggest that the liver-enriched factor C/EBPalpha binds to, and activates, the UGT2B1 gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is fam member UGT2B1---{\"tf associated gene name\":[\"CEBPA\"],\"tg associated gene name\":[\"UGT1A1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a-fam\"],\"negation\":[],\"not true\":[],\"comment\":[\"TG is fam member UGT2B1\"],\"sentence\":[\"These data strongly suggest that the liver-enriched factor C/EBPalpha binds to\",\"and activates\",\"the UGT2B1 gene promoter.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong UGT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPA\"],\"tg (associated gene name)\":[\"UGT1A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong UGT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These data strongly suggest that the liver-enriched factor C/EBPalpha binds to, and activates, the UGT2B1 gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 509 (char 508)\n",
      "{\"transcription factor (associated gene name)\":[\"NCOR2\"],\"target gene (associated gene name)\":[\"SLC2A1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"HNF-4alpha was also shown to target the histone deacetylase complex associated with silencing mediator of retinoic acid and thyroid hormone receptor\",\"both at the PED promoter\",\"and at GRB14 and USP21 regulatory regions\",\"leading to a reduction of mRNA levels.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOR2\"],\"tg (associated gene name)\":[\"SLC2A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HNF-4alpha was also shown to target the histone deacetylase complex associated with silencing mediator of retinoic acid and thyroid hormone receptor, both at the PED promoter, and at GRB14 and USP21 regulatory regions, leading to a reduction of mRNA levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"NCOR2\"],\"target gene (associated gene name)\":[\"SLC2A1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"r\"],\"negation\":[],\"sentence\":[\"HNF-4alpha was also shown to target the histone deacetylase complex associated with silencing mediator of retinoic acid and thyroid hormone receptor\",\"both at the PED promoter\",\"and at GRB14 and USP21 regulatory regions\",\"leading to a reduction of mRNA levels.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"3.682674\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"257\"],\"sentence pair density\":[\"0.0609756097560975\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NCOR2\"],\"target gene (associated gene name)\":[\"SLC2A1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"HNF-4alpha was also shown to target the histone deacetylase complex associated with silencing mediator of retinoic acid and thyroid hormone receptor\",\"both at the PED promoter\",\"and at GRB14 and USP21 regulatory regions\",\"leading to a reduction of mRNA levels.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOR2\"],\"tg (associated gene name)\":[\"SLC2A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"wrong TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HNF-4alpha was also shown to target the histone deacetylase complex associated with silencing mediator of retinoic acid and thyroid hormone receptor, both at the PED promoter, and at GRB14 and USP21 regulatory regions, leading to a reduction of mRNA levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 339 (char 338)\n",
      "{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"FASLG\"],\"tf text\":[],\"tg text\":[\"FasL\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Gel shift studies also revealed that FasL mobilized the p50/p65 heterodimeric form of NF-kappaB.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"FASLG\"],\"tf text\":[],\"tg text\":[\"FasL\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gel shift studies also revealed that FasL mobilized the p50/p65 heterodimeric form of NF-kappaB.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"FASLG\"],\"tf text\":[],\"tg text\":[\"FasL\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Gel shift studies also revealed that FasL mobilized the p50/p65 heterodimeric form of NF-kappaB.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"FASLG\"],\"tf text\":[],\"tg text\":[\"FasL\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gel shift studies also revealed that FasL mobilized the p50/p65 heterodimeric form of NF-kappaB.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 347 (char 346)\n",
      "{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"TP53\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We identified Sp1 as one of the p53 modulators\",\"which confer specificity to p53-mediated transcriptional response upon RITA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|suggestion for protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"suggestion for protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We identified Sp1 as one of the p53 modulators, which confer specificity to p53-mediated transcriptional response upon RITA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"TP53\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"We identified Sp1 as one of the p53 modulators\",\"which confer specificity to p53-mediated transcriptional response upon RITA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|suggestion for protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"suggestion for protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We identified Sp1 as one of the p53 modulators, which confer specificity to p53-mediated transcriptional response upon RITA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "other TFs---{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"MIR27A\"],\"tf text\":[],\"tg text\":[\"miR-27a\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"other TFs\"],\"sentence\":[\"Here we demonstrate that PEITC-induced ROS decreased expression of microRNA 27a (miR-27a)/miR-20a:miR-17-5p and induced miR-regulated ZBTB10/ZBTB4 and ZBTB34 transcriptional repressors\",\"which\",\"in turn\",\"downregulate specificity protein (Sp) transcription factors (TFs) Sp1\",\"Sp3\",\"and Sp4 in pancreatic cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX3\"],\"tg (associated gene name)\":[\"MIR27A\"],\"tf text\":[],\"tg text\":[\"miR-27a\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we demonstrate that PEITC-induced ROS decreased expression of microRNA 27a (miR-27a)/miR-20a:miR-17-5p and induced miR-regulated ZBTB10/ZBTB4 and ZBTB34 transcriptional repressors, which, in turn, downregulate specificity protein (Sp) transcription factors (TFs) Sp1, Sp3, and Sp4 in pancreatic cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "other TFs---{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"MIR27A\"],\"tf text\":[],\"tg text\":[\"miR-27a\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"other TFs\"],\"sentence\":[\"Here we demonstrate that PEITC-induced ROS decreased expression of microRNA 27a (miR-27a)/miR-20a:miR-17-5p and induced miR-regulated ZBTB10/ZBTB4 and ZBTB34 transcriptional repressors\",\"which\",\"in turn\",\"downregulate specificity protein (Sp) transcription factors (TFs) Sp1\",\"Sp3\",\"and Sp4 in pancreatic cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX3\"],\"tg (associated gene name)\":[\"MIR27A\"],\"tf text\":[],\"tg text\":[\"miR-27a\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we demonstrate that PEITC-induced ROS decreased expression of microRNA 27a (miR-27a)/miR-20a:miR-17-5p and induced miR-regulated ZBTB10/ZBTB4 and ZBTB34 transcriptional repressors, which, in turn, downregulate specificity protein (Sp) transcription factors (TFs) Sp1, Sp3, and Sp4 in pancreatic cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 448 (char 447)\n",
      "{\"tf associated gene name\":[\"ATOH1\"],\"tg associated gene name\":[\"ATOH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"However\",\"besides activating Atoh1\",\"Sox2 also promotes the expression of Atoh1 negative regulators and the temporal profile of Atoh1 activation by Sox2 is transient suggesting that Sox2 triggers an incoherent feed-forward loop.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ATOH1\"],\"tg (associated gene name)\":[\"ATOH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, besides activating Atoh1, Sox2 also promotes the expression of Atoh1 negative regulators and the temporal profile of Atoh1 activation by Sox2 is transient suggesting that Sox2 triggers an incoherent feed-forward loop.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ATOH1\"],\"tg associated gene name\":[\"ATOH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"However\",\"besides activating Atoh1\",\"Sox2 also promotes the expression of Atoh1 negative regulators and the temporal profile of Atoh1 activation by Sox2 is transient suggesting that Sox2 triggers an incoherent feed-forward loop.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ATOH1\"],\"tg (associated gene name)\":[\"ATOH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, besides activating Atoh1, Sox2 also promotes the expression of Atoh1 negative regulators and the temporal profile of Atoh1 activation by Sox2 is transient suggesting that Sox2 triggers an incoherent feed-forward loop.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 505 (char 504)\n",
      "{\"transcription factor (associated gene name)\":[\"NR1H3\"],\"target gene (associated gene name)\":[\"LPCAT3\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Through an in silico search for LXRE (LXR response element) consensus sequences in the promoter of genes whose expression was shown to be sensitive to TO901317\",\"we identified a LXRE in the promoter of the LPCAT3 (lysophosphatidylcholine acyltransferase 3).\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|LXRE not A---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1H3\"],\"tg (associated gene name)\":[\"LPCAT3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"LXRE not A\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Through an in silico search for LXRE (LXR response element) consensus sequences in the promoter of genes whose expression was shown to be sensitive to TO901317, we identified a LXRE in the promoter of the LPCAT3 (lysophosphatidylcholine acyltransferase 3).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR1H3\"],\"tg associated gene name\":[\"LPCAT3\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Through an in silico search for LXRE (LXR response element) consensus sequences in the promoter of genes whose expression was shown to be sensitive to TO901317\",\"we identified a LXRE in the promoter of the LPCAT3 (lysophosphatidylcholine acyltransferase 3).\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR1H3\"],\"target gene (associated gene name)\":[\"LPCAT3\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Through an in silico search for LXRE (LXR response element) consensus sequences in the promoter of genes whose expression was shown to be sensitive to TO901317\",\"we identified a LXRE in the promoter of the LPCAT3 (lysophosphatidylcholine acyltransferase 3).\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|LXRE not A---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1H3\"],\"tg (associated gene name)\":[\"LPCAT3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"LXRE not A\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Through an in silico search for LXRE (LXR response element) consensus sequences in the promoter of genes whose expression was shown to be sensitive to TO901317, we identified a LXRE in the promoter of the LPCAT3 (lysophosphatidylcholine acyltransferase 3).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 478 (char 477)\n",
      "{\"tf associated gene name\":[\"KAT7\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"In particular\",\"we found that in developing mouse retinas-an organ that critically requires cyclin D1 function-cyclin D1 binds the upstream regulatory region of the Notch1 gene\",\"where it serves to recruit CREB binding protein (CBP) histone acetyltransferase.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|wrong TF norm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KAT7\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF norm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In particular, we found that in developing mouse retinas-an organ that critically requires cyclin D1 function-cyclin D1 binds the upstream regulatory region of the Notch1 gene, where it serves to recruit CREB binding protein (CBP) histone acetyltransferase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"KAT7\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"In particular\",\"we found that in developing mouse retinas-an organ that critically requires cyclin D1 function-cyclin D1 binds the upstream regulatory region of the Notch1 gene\",\"where it serves to recruit CREB binding protein (CBP) histone acetyltransferase.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|wrong TF norm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KAT7\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF norm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In particular, we found that in developing mouse retinas-an organ that critically requires cyclin D1 function-cyclin D1 binds the upstream regulatory region of the Notch1 gene, where it serves to recruit CREB binding protein (CBP) histone acetyltransferase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[\"p53\"],\"tg text\":[\"p53\"],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We first demonstrated that S-nitrosylation of parkin decreased its activity as a repressor of p53 gene expression, leading to upregulation of p53.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"TP53\"],\"tf text\":[\"p53\"],\"tg text\":[\"p53\"],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We first demonstrated that S-nitrosylation of parkin decreased its activity as a repressor of p53 gene expression, leading to upregulation of p53.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is estrogen receptor alpha---{\"tf associated gene name\":[\"ESR2\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"Estrogen receptor beta\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is estrogen receptor alpha\"],\"sentence\":[\"Estrogen receptor beta binds Sp1 and recruits a corepressor complex to the estrogen receptor alpha gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR2\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"Estrogen receptor beta\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Estrogen receptor beta binds Sp1 and recruits a corepressor complex to the estrogen receptor alpha gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is estrogen receptor alpha---{\"tf associated gene name\":[\"ESR2\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"Estrogen receptor beta\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is estrogen receptor alpha\"],\"sentence\":[\"Estrogen receptor beta binds Sp1 and recruits a corepressor complex to the estrogen receptor alpha gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR2\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"Estrogen receptor beta\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Estrogen receptor beta binds Sp1 and recruits a corepressor complex to the estrogen receptor alpha gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"PIP\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"PIP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Recent evidence suggests that PIP (LSIRF) sequesters the Ets protein PU.1 at a composite DNA element lends support to this adaptor hypothesis.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"PIP\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"PIP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Recent evidence suggests that PIP (LSIRF) sequesters the Ets protein PU.1 at a composite DNA element lends support to this adaptor hypothesis.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG human cytomegalovirus DNA polymerase promoter.---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"ERVK-11\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"TG human cytomegalovirus DNA polymerase promoter.\"],\"sentence\":[\"Binding of SP1 to the immediate-early protein-responsive element of the human cytomegalovirus DNA polymerase promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG human cytomegalovirus DNA polymerase promoter.---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"ERVK-11\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"TG human cytomegalovirus DNA polymerase promoter.\"],\"sentence\":[\"Binding of SP1 to the immediate-early protein-responsive element of the human cytomegalovirus DNA polymerase promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"ERVK-11\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Binding of SP1 to the immediate-early protein-responsive element of the human cytomegalovirus DNA polymerase promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Bim---{\"tf associated gene name\":[\"FOXO3\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is Bim\"],\"sentence\":[\"Our results support a mechanism in which TGFbeta stimulates Bim transcription by up-regulating Runx1 expression\",\"which binds FOXO3\",\"and the two cooperate in the transcriptional induction of Bim.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO3\"],\"tg (associated gene name)\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results support a mechanism in which TGFbeta stimulates Bim transcription by up-regulating Runx1 expression, which binds FOXO3, and the two cooperate in the transcriptional induction of Bim.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Bim---{\"tf associated gene name\":[\"FOXO3\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is Bim\"],\"sentence\":[\"Our results support a mechanism in which TGFbeta stimulates Bim transcription by up-regulating Runx1 expression\",\"which binds FOXO3\",\"and the two cooperate in the transcriptional induction of Bim.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO3\"],\"tg (associated gene name)\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results support a mechanism in which TGFbeta stimulates Bim transcription by up-regulating Runx1 expression, which binds FOXO3, and the two cooperate in the transcriptional induction of Bim.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 226 (char 225)\n",
      "{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"FAS\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|only co-mention---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"FAS\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"only co-mention\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Fas and its ligand expression were evaluated by RT-PCR; the promoter and the entire coding region of Fas, and the coding exons 4-9 of p53 were investigated by polymerase chain reaction, single strand conformation polymorphism, and DNA sequencing.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"FAS\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|only co-mention---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"FAS\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"only co-mention\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Fas and its ligand expression were evaluated by RT-PCR; the promoter and the entire coding region of Fas, and the coding exons 4-9 of p53 were investigated by polymerase chain reaction, single strand conformation polymorphism, and DNA sequencing.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is nurr1|TG is ocn---{\"tf associated gene name\":[\"ESRRA\"],\"tg associated gene name\":[\"SRGN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is nurr1\",\"TG is ocn\"],\"sentence\":[\"The NR4A/NGFI-B orphan nuclear receptor (NR) Nurr1 is a PTH-induced\",\"cAMP-responsive primary response gene (PRG) that transactivates osteocalcin (Ocn) expression through a putative NGFI-B response element (NBRE) in the proximal promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is nurr1|TG is ocn---{\"tf associated gene name\":[\"ESRRA\"],\"tg associated gene name\":[\"SRGN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is nurr1\",\"TG is ocn\"],\"sentence\":[\"The NR4A/NGFI-B orphan nuclear receptor (NR) Nurr1 is a PTH-induced\",\"cAMP-responsive primary response gene (PRG) that transactivates osteocalcin (Ocn) expression through a putative NGFI-B response element (NBRE) in the proximal promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESRRA\"],\"target gene (associated gene name)\":[\"SRGN\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESRRA\"],\"tg (associated gene name)\":[\"SRGN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"The NR4A/NGFI-B orphan nuclear receptor (NR) Nurr1 is a PTH-induced, cAMP-responsive primary response gene (PRG) that transactivates osteocalcin (Ocn) expression through a putative NGFI-B response element (NBRE) in the proximal promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF FOS---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ODC1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF FOS\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These data show that FosER, and by extrapolation c-Fos, can directly activate transcription of the endogenous ODC gene in PC12 cells by binding to cis-regulatory sequences.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF FOS---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ODC1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF FOS\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"These data show that FosER, and by extrapolation c-Fos, can directly activate transcription of the endogenous ODC gene in PC12 cells by binding to cis-regulatory sequences.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"SLC26A3\"],\"tf text\":[],\"tg text\":[\"DRA\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"The similarities of the hXBP-1 promoter and of the DRA promoter (the gene that the hXBP-1 protein regulates) are further emphasized by the fact that a Y box element is located 3' of both hX2 and X2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SSRP1\"],\"tg (associated gene name)\":[\"SLC26A3\"],\"tf text\":[],\"tg text\":[\"DRA\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The similarities of the hXBP-1 promoter and of the DRA promoter (the gene that the hXBP-1 protein regulates) are further emphasized by the fact that a Y box element is located 3' of both hX2 and X2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"SLC26A3\"],\"tf text\":[],\"tg text\":[\"DRA\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"The similarities of the hXBP-1 promoter and of the DRA promoter (the gene that the hXBP-1 protein regulates) are further emphasized by the fact that a Y box element is located 3' of both hX2 and X2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SSRP1\"],\"tg (associated gene name)\":[\"SLC26A3\"],\"tf text\":[],\"tg text\":[\"DRA\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The similarities of the hXBP-1 promoter and of the DRA promoter (the gene that the hXBP-1 protein regulates) are further emphasized by the fact that a Y box element is located 3' of both hX2 and X2.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain normalization of TG---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"PKLR\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain normalization of TG\"],\"sentence\":[\"The PPARalpha-null mouse was used to evaluate the role of the PPARalpha in hepatic transcriptional control of L-PK.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|L-PK enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"PKLR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"L-PK enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The PPARalpha-null mouse was used to evaluate the role of the PPARalpha in hepatic transcriptional control of L-PK.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "uncertain normalization of TG---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"PKLR\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain normalization of TG\"],\"sentence\":[\"The PPARalpha-null mouse was used to evaluate the role of the PPARalpha in hepatic transcriptional control of L-PK.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|L-PK enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"PKLR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"L-PK enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The PPARalpha-null mouse was used to evaluate the role of the PPARalpha in hepatic transcriptional control of L-PK.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE; MMTV promoter indicates mammary tumour virus---{\"tf associated gene name\":[\"ELK3\"],\"tg associated gene name\":[\"HIST1H4A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\",\"s\"],\"comment\":[\"not TRE; MMTV promoter indicates mammary tumour virus\"],\"sentence\":[\"In fact\",\"histone acetylation at the MMTV promoter is highly dynamic\",\"with an initial increase in acetylation followed by an eventual net deacetylation of histone H4.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no transcription implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELK3\"],\"tg (associated gene name)\":[\"HIST1H4A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no transcription implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In fact, histone acetylation at the MMTV promoter is highly dynamic, with an initial increase in acetylation followed by an eventual net deacetylation of histone H4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE; MMTV promoter indicates mammary tumour virus---{\"tf associated gene name\":[\"ELK3\"],\"tg associated gene name\":[\"HIST1H4A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\",\"s\"],\"comment\":[\"not TRE; MMTV promoter indicates mammary tumour virus\"],\"sentence\":[\"In fact\",\"histone acetylation at the MMTV promoter is highly dynamic\",\"with an initial increase in acetylation followed by an eventual net deacetylation of histone H4.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no transcription implied---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELK3\"],\"tg (associated gene name)\":[\"HIST1H4A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no transcription implied\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In fact, histone acetylation at the MMTV promoter is highly dynamic, with an initial increase in acetylation followed by an eventual net deacetylation of histone H4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not clear TRE---{\"tf associated gene name\":[\"TAL1\"],\"tg associated gene name\":[\"MYB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"not clear TRE\"],\"sentence\":[\"The increase in expression was temporally associated with an increase in the expression of Eklf\",\"Scl\",\"and Nfe2\",\"three genes required for erythroid differentiation\",\"and preceded by 24 h the repression of Gata2 and Myb expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TAL1\"],\"tg (associated gene name)\":[\"MYB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The increase in expression was temporally associated with an increase in the expression of Eklf, Scl, and Nfe2, three genes required for erythroid differentiation, and preceded by 24 h the repression of Gata2 and Myb expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not clear TRE---{\"tf associated gene name\":[\"TAL1\"],\"tg associated gene name\":[\"MYB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[\"not clear TRE\"],\"sentence\":[\"The increase in expression was temporally associated with an increase in the expression of Eklf\",\"Scl\",\"and Nfe2\",\"three genes required for erythroid differentiation\",\"and preceded by 24 h the repression of Gata2 and Myb expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TAL1\"],\"tg (associated gene name)\":[\"MYB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The increase in expression was temporally associated with an increase in the expression of Eklf, Scl, and Nfe2, three genes required for erythroid differentiation, and preceded by 24 h the repression of Gata2 and Myb expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear sentence|TRE-relation---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[],\"comment\":[\"unclear sentence\",\"TRE-relation\"],\"sentence\":[\"We demonstrate that the increased co-stimulatory molecule expression (MHC II and CD86) and antigen-specific T cell activation capacity observed in Nrf2(-/-) iDCs was reversed by inhibition of p38 MAPK but not JNK.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MHCII---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MHCII\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrate that the increased co-stimulatory molecule expression (MHC II and CD86) and antigen-specific T cell activation capacity observed in Nrf2(-/-) iDCs was reversed by inhibition of p38 MAPK but not JNK.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "unclear sentence|TRE-relation---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[],\"comment\":[\"unclear sentence\",\"TRE-relation\"],\"sentence\":[\"We demonstrate that the increased co-stimulatory molecule expression (MHC II and CD86) and antigen-specific T cell activation capacity observed in Nrf2(-/-) iDCs was reversed by inhibition of p38 MAPK but not JNK.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MHCII---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MHCII\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrate that the increased co-stimulatory molecule expression (MHC II and CD86) and antigen-specific T cell activation capacity observed in Nrf2(-/-) iDCs was reversed by inhibition of p38 MAPK but not JNK.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Foxa---{\"tf associated gene name\":[\"FOXM1\"],\"tg associated gene name\":[\"VTN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Foxa\"],\"sentence\":[\"Foxa (HNF3) up-regulates vitronectin expression during retinoic acid-induced differentiation in mouse neuroblastoma Neuro2a cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Foxa---{\"tf associated gene name\":[\"FOXM1\"],\"tg associated gene name\":[\"VTN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Foxa\"],\"sentence\":[\"Foxa (HNF3) up-regulates vitronectin expression during retinoic acid-induced differentiation in mouse neuroblastoma Neuro2a cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXM1\"],\"target gene (associated gene name)\":[\"VTN\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Foxa (HNF3) up-regulates vitronectin expression during retinoic acid-induced differentiation in mouse neuroblastoma Neuro2a cells.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXM1\"],\"tg (associated gene name)\":[\"VTN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Foxa (HNF3) up-regulates vitronectin expression during retinoic acid-induced differentiation in mouse neuroblastoma Neuro2a cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain whether this is GRE---{\"tf associated gene name\":[\"ERG\"],\"tg associated gene name\":[\"CDH5\"],\"tf text\":[\"Erg\"],\"tg text\":[\"VE-cadherin\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"uncertain whether this is GRE\"],\"sentence\":[\"Apoptosis induced by inhibition of Erg was partly rescued by overexpression of VE-cadherin-GFP\",\"suggesting that VE-cadherin is involved in the Erg-dependent survival signals.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|very indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ERG\"],\"tg (associated gene name)\":[\"CDH5\"],\"tf text\":[\"Erg\"],\"tg text\":[\"VE-cadherin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"very indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Apoptosis induced by inhibition of Erg was partly rescued by overexpression of VE-cadherin-GFP, suggesting that VE-cadherin is involved in the Erg-dependent survival signals.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "uncertain whether this is GRE---{\"tf associated gene name\":[\"ERG\"],\"tg associated gene name\":[\"CDH5\"],\"tf text\":[\"Erg\"],\"tg text\":[\"VE-cadherin\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"uncertain whether this is GRE\"],\"sentence\":[\"Apoptosis induced by inhibition of Erg was partly rescued by overexpression of VE-cadherin-GFP\",\"suggesting that VE-cadherin is involved in the Erg-dependent survival signals.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|very indirect---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ERG\"],\"tg (associated gene name)\":[\"CDH5\"],\"tf text\":[\"Erg\"],\"tg text\":[\"VE-cadherin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"very indirect\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Apoptosis induced by inhibition of Erg was partly rescued by overexpression of VE-cadherin-GFP, suggesting that VE-cadherin is involved in the Erg-dependent survival signals.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"DEK\"],\"tf text\":[],\"tg text\":[\"DEK\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Together, our data suggest that DEK promotes the pathogenesis of ER+ breast cancer and that the targeted inhibition of DEK may enhance the efficacy of conventional hormone therapies.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"DEK\"],\"tf text\":[],\"tg text\":[\"DEK\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Together, our data suggest that DEK promotes the pathogenesis of ER+ breast cancer and that the targeted inhibition of DEK may enhance the efficacy of conventional hormone therapies.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PITX2\"],\"tg (associated gene name)\":[\"PITX2\"],\"tf text\":[\"PITX2\"],\"tg text\":[\"PITX2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We hypothesized that PITX2 hypermethylation leads to PITX2 silencing and that decreased PITX2 expression is likewise associated with poor prognosis in prostate cancers.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PITX2\"],\"tg (associated gene name)\":[\"PITX2\"],\"tf text\":[\"PITX2\"],\"tg text\":[\"PITX2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We hypothesized that PITX2 hypermethylation leads to PITX2 silencing and that decreased PITX2 expression is likewise associated with poor prognosis in prostate cancers.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 369 (char 368)\n",
      "{\"tf associated gene name\":[\"PLAG1\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[\"PSA\"],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Ang-II induced cell proliferation with enhancement of AR\",\"prostate specific antigen (PSA)\",\"NF-??B\",\"and c-myc\",\"and the activity of AR and PSA promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|causality?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PLAG1\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[\"PSA\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"causality?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Ang-II induced cell proliferation with enhancement of AR, prostate specific antigen (PSA), NF-B, and c-myc, and the activity of AR and PSA promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PLAG1\"],\"tg associated gene name\":[\"AR\"],\"tf text\":[\"PSA\"],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"Ang-II induced cell proliferation with enhancement of AR\",\"prostate specific antigen (PSA)\",\"NF-??B\",\"and c-myc\",\"and the activity of AR and PSA promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|causality?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PLAG1\"],\"tg (associated gene name)\":[\"AR\"],\"tf text\":[\"PSA\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"causality?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Ang-II induced cell proliferation with enhancement of AR, prostate specific antigen (PSA), NF-B, and c-myc, and the activity of AR and PSA promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 355 (char 354)\n",
      "{\"tf associated gene name\":[\"TP73\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"p73\"],\"tg text\":[\"MDM2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These differences between the regulation of p53 and p73 by MDM2/MDMX may highlight a physiological difference in their action.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP73\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"p73\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These differences between the regulation of p53 and p73 by MDM2/MDMX may highlight a physiological difference in their action.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TP73\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[\"p73\"],\"tg text\":[\"MDM2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These differences between the regulation of p53 and p73 by MDM2/MDMX may highlight a physiological difference in their action.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP73\"],\"tg (associated gene name)\":[\"MDM2\"],\"tf text\":[\"p73\"],\"tg text\":[\"MDM2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These differences between the regulation of p53 and p73 by MDM2/MDMX may highlight a physiological difference in their action.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETV2\"],\"tg (associated gene name)\":[\"ETV2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"This inhibition of etv2 expression in the cardiac progenitor population is partly mediated by Scl and Nkx2.5, likely through direct binding to the etv2 promoter, and cis-regulatory elements located in the first and second introns.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETV2\"],\"tg (associated gene name)\":[\"ETV2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"This inhibition of etv2 expression in the cardiac progenitor population is partly mediated by Scl and Nkx2.5, likely through direct binding to the etv2 promoter, and cis-regulatory elements located in the first and second introns.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 332 (char 331)\n",
      "{\"transcription factor (associated gene name)\":[\"TAF1\"],\"target gene (associated gene name)\":[\"CCND1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"TAF1 histone acetyltransferase activity in Sp1 activation of the cyclin D1 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|TAF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TAF1\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"TAF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TAF1 histone acetyltransferase activity in Sp1 activation of the cyclin D1 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TAF1\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"TAF1 histone acetyltransferase activity in Sp1 activation of the cyclin D1 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TAF1\"],\"target gene (associated gene name)\":[\"CCND1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"TAF1 histone acetyltransferase activity in Sp1 activation of the cyclin D1 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|TAF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TAF1\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"TAF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TAF1 histone acetyltransferase activity in Sp1 activation of the cyclin D1 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 354 (char 353)\n",
      "{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"CDKN1A\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Cdc2 is inhibited simultaneously by three transcriptional targets of p53\",\"Gadd45\",\"p21\",\"and 14-3-3 sigma.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|not direct---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"not direct\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Cdc2 is inhibited simultaneously by three transcriptional targets of p53, Gadd45, p21, and 14-3-3 sigma.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"CDKN1A\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Cdc2 is inhibited simultaneously by three transcriptional targets of p53\",\"Gadd45\",\"p21\",\"and 14-3-3 sigma.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|not direct---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"not direct\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Cdc2 is inhibited simultaneously by three transcriptional targets of p53, Gadd45, p21, and 14-3-3 sigma.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX3-1\"],\"tg (associated gene name)\":[\"ADH1C\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The aim of this study was to determine the effect of chronic ethanol feeding on acetylation of histone H3 at lysine 9 (H3-Lys9) at promoter and coding regions of genes for class I alcohol dehydrogenase (ADH I), inducible nitric oxide synthase (iNOS), Bax, p21, c-met and hepatocyte growth factor in the rat liver.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NKX3-1\"],\"tg (associated gene name)\":[\"ADH1C\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The aim of this study was to determine the effect of chronic ethanol feeding on acetylation of histone H3 at lysine 9 (H3-Lys9) at promoter and coding regions of genes for class I alcohol dehydrogenase (ADH I), inducible nitric oxide synthase (iNOS), Bax, p21, c-met and hepatocyte growth factor in the rat liver.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 371 (char 370)\n",
      "{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"This phosphorylation is required for the dissociation of HDAC3 from the CCND1 and MYC promoter regions and subsequent acetylation of histone H3 at K9.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This phosphorylation is required for the dissociation of HDAC3 from the CCND1 and MYC promoter regions and subsequent acetylation of histone H3 at K9.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"This phosphorylation is required for the dissociation of HDAC3 from the CCND1 and MYC promoter regions and subsequent acetylation of histone H3 at K9.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This phosphorylation is required for the dissociation of HDAC3 from the CCND1 and MYC promoter regions and subsequent acetylation of histone H3 at K9.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"DNMT1\"],\"tg (associated gene name)\":[\"HBG2\"],\"tf text\":[],\"tg text\":[\"HBG2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The aim of this study was to analyze in 57 beta-thalassemia intermedia patients with very various genotypes the effects on fetal hemoglobin levels of SNPs lying in three genes or chromosome regions which include the XmnI (G)gamma polymorphism at position -158 of the HBG2 promoter (rs7482144), two SNPs located in the BCL11A region (rs4671393 and rs11886868) and three SNPs located in the HBS1L-MYB region (rs28384513, rs9399137 and rs4895441).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"DNMT1\"],\"tg (associated gene name)\":[\"HBG2\"],\"tf text\":[],\"tg text\":[\"HBG2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"The aim of this study was to analyze in 57 beta-thalassemia intermedia patients with very various genotypes the effects on fetal hemoglobin levels of SNPs lying in three genes or chromosome regions which include the XmnI (G)gamma polymorphism at position -158 of the HBG2 promoter (rs7482144), two SNPs located in the BCL11A region (rs4671393 and rs11886868) and three SNPs located in the HBS1L-MYB region (rs28384513, rs9399137 and rs4895441).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  IL-4---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"HAND2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is  IL-4\"],\"sentence\":[\"Previous studies found that P1\",\"an element of the interleukin 4 (IL-4) promoter that binds AP-1\",\"is important for Th2-restricted IL-4 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"HAND2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Previous studies found that P1, an element of the interleukin 4 (IL-4) promoter that binds AP-1, is important for Th2-restricted IL-4 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is  IL-4---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"HAND2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is  IL-4\"],\"sentence\":[\"Previous studies found that P1\",\"an element of the interleukin 4 (IL-4) promoter that binds AP-1\",\"is important for Th2-restricted IL-4 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"HAND2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Previous studies found that P1, an element of the interleukin 4 (IL-4) promoter that binds AP-1, is important for Th2-restricted IL-4 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"MITF\"],\"tf text\":[\"MITF\"],\"tg text\":[\"MITF\"],\"1\":[],\"not true\":[\"t\"],\"comment\":[\"TF\"],\"sentence\":[\"Promoter deletion and mutational analyses indicate that SOX10 can activate MITF expression through binding to a region that is evolutionarily conserved between the mouse and human MITF promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"MITF\"],\"tf text\":[\"MITF\"],\"tg text\":[\"MITF\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Promoter deletion and mutational analyses indicate that SOX10 can activate MITF expression through binding to a region that is evolutionarily conserved between the mouse and human MITF promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF---{\"tf associated gene name\":[\"MITF\"],\"tg associated gene name\":[\"MITF\"],\"tf text\":[\"MITF\"],\"tg text\":[\"MITF\"],\"1\":[],\"not true\":[\"t\"],\"comment\":[\"TF\"],\"sentence\":[\"Promoter deletion and mutational analyses indicate that SOX10 can activate MITF expression through binding to a region that is evolutionarily conserved between the mouse and human MITF promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MITF\"],\"tg (associated gene name)\":[\"MITF\"],\"tf text\":[\"MITF\"],\"tg text\":[\"MITF\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Promoter deletion and mutational analyses indicate that SOX10 can activate MITF expression through binding to a region that is evolutionarily conserved between the mouse and human MITF promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is COX-2---{\"tf associated gene name\":[\"TCF7L2\"],\"tg associated gene name\":[\"HGF\"],\"tf text\":[],\"tg text\":[\"HGF\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is COX-2\"],\"sentence\":[\"COX-2 promoter analysis in luciferase reporter constructs showed that HGF regulation required the beta-catenin-responsive T cell factor-4 binding element (TBE).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is COX-2---{\"tf associated gene name\":[\"TCF7L2\"],\"tg associated gene name\":[\"HGF\"],\"tf text\":[],\"tg text\":[\"HGF\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is COX-2\"],\"sentence\":[\"COX-2 promoter analysis in luciferase reporter constructs showed that HGF regulation required the beta-catenin-responsive T cell factor-4 binding element (TBE).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TCF7L2\"],\"target gene (associated gene name)\":[\"HGF\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF7L2\"],\"tg (associated gene name)\":[\"HGF\"],\"tf text\":[],\"tg text\":[\"HGF\"],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"COX-2 promoter analysis in luciferase reporter constructs showed that HGF regulation required the beta-catenin-responsive T cell factor-4 binding element (TBE).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  X11??|most likely APBA1 is not correct normalization of TG---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"APBA1\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is  X11??\",\"most likely APBA1 is not correct normalization of TG\"],\"sentence\":[\"Moreover\",\"inhibition of Sp3 reduces endogenous X11?? expression and promotes the generation of A??.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is  X11??|most likely APBA1 is not correct normalization of TG---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"APBA1\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is  X11??\",\"most likely APBA1 is not correct normalization of TG\"],\"sentence\":[\"Moreover\",\"inhibition of Sp3 reduces endogenous X11?? expression and promotes the generation of A??.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP3\"],\"target gene (associated gene name)\":[\"APBA1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Moreover\",\"inhibition of Sp3 reduces endogenous X11 expression and promotes the generation of A.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP3\"],\"tg (associated gene name)\":[\"APBA1\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, inhibition of Sp3 reduces endogenous X11 expression and promotes the generation of A.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG (CYP2H1) may be incorrectly normalized---{\"tf associated gene name\":[\"USF1\"],\"tg associated gene name\":[\"CYP2A\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG (CYP2H1) may be incorrectly normalized\"],\"sentence\":[\"The results demonstrate that liver-enriched transcription factors and USF direct strong expression of the CYP2H1 promoter in transiently transfected cells.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TG (CYP2H1) may be incorrectly normalized---{\"tf associated gene name\":[\"USF1\"],\"tg associated gene name\":[\"CYP2A\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG (CYP2H1) may be incorrectly normalized\"],\"sentence\":[\"The results demonstrate that liver-enriched transcription factors and USF direct strong expression of the CYP2H1 promoter in transiently transfected cells.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"CYP2A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results demonstrate that liver-enriched transcription factors and USF direct strong expression of the CYP2H1 promoter in transiently transfected cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "via SOX2---{\"tf associated gene name\":[\"ID4\"],\"tg associated gene name\":[\"ABCC6\"],\"tf text\":[\"ID4\"],\"tg text\":[\"ABCC6\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"via SOX2\"],\"sentence\":[\"Consequently\",\"ID4-mediated SOX2 induction enhanced ABCC3 and ABCC6 expression through direct transcriptional regulation\",\"indicating that ID4 regulates the chemoresistance of iGSCs by promoting SOX2-mediated induction of ABC transporters.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "via SOX2---{\"tf associated gene name\":[\"ID4\"],\"tg associated gene name\":[\"ABCC6\"],\"tf text\":[\"ID4\"],\"tg text\":[\"ABCC6\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"via SOX2\"],\"sentence\":[\"Consequently\",\"ID4-mediated SOX2 induction enhanced ABCC3 and ABCC6 expression through direct transcriptional regulation\",\"indicating that ID4 regulates the chemoresistance of iGSCs by promoting SOX2-mediated induction of ABC transporters.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ID4\"],\"target gene (associated gene name)\":[\"ABCC6\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Consequently\",\"ID4-mediated SOX2 induction enhanced ABCC3 and ABCC6 expression through direct transcriptional regulation\",\"indicating that ID4 regulates the chemoresistance of iGSCs by promoting SOX2-mediated induction of ABC transporters.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|as agreed on indirectness---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID4\"],\"tg (associated gene name)\":[\"ABCC6\"],\"tf text\":[\"ID4\"],\"tg text\":[\"ABCC6\"],\"consensus\":[\"V\"],\"martin_comments\":[\"as agreed on indirectness\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consequently, ID4-mediated SOX2 induction enhanced ABCC3 and ABCC6 expression through direct transcriptional regulation, indicating that ID4 regulates the chemoresistance of iGSCs by promoting SOX2-mediated induction of ABC transporters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 509 (char 508)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"MMP1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The results showed that Smad3 and Smad4\",\"but not Smad1 or Smad2\",\"mimicked the inhibitory effect of TGF-beta and abrogated interleukin-1beta (IL-1beta)-induced stimulation of MMP-1 promoter activity and NFkappaB-specific gene transcription in dermal fibroblasts.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|a bit vague---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"MMP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit vague\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results showed that Smad3 and Smad4, but not Smad1 or Smad2, mimicked the inhibitory effect of TGF-beta and abrogated interleukin-1beta (IL-1beta)-induced stimulation of MMP-1 promoter activity and NFkappaB-specific gene transcription in dermal fibroblasts.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"MMP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"The results showed that Smad3 and Smad4\",\"but not Smad1 or Smad2\",\"mimicked the inhibitory effect of TGF-beta and abrogated interleukin-1beta (IL-1beta)-induced stimulation of MMP-1 promoter activity and NFkappaB-specific gene transcription in dermal fibroblasts.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"MMP1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The results showed that Smad3 and Smad4\",\"but not Smad1 or Smad2\",\"mimicked the inhibitory effect of TGF-beta and abrogated interleukin-1beta (IL-1beta)-induced stimulation of MMP-1 promoter activity and NFkappaB-specific gene transcription in dermal fibroblasts.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|a bit vague---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"MMP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"a bit vague\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The results showed that Smad3 and Smad4, but not Smad1 or Smad2, mimicked the inhibitory effect of TGF-beta and abrogated interleukin-1beta (IL-1beta)-induced stimulation of MMP-1 promoter activity and NFkappaB-specific gene transcription in dermal fibroblasts.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 352 (char 351)\n",
      "{\"tf associated gene name\":[\"NHLH1\"],\"tg associated gene name\":[\"ME1\"],\"tf text\":[\"NSCL-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"Gel mobility shift assays show that NSCL-1 only binds DNA as a heterodimeric complex with the ME1a E-protein.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NHLH1\"],\"tg (associated gene name)\":[\"ME1\"],\"tf text\":[\"NSCL-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gel mobility shift assays show that NSCL-1 only binds DNA as a heterodimeric complex with the ME1a E-protein.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NHLH1\"],\"tg associated gene name\":[\"ME1\"],\"tf text\":[\"NSCL-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"Gel mobility shift assays show that NSCL-1 only binds DNA as a heterodimeric complex with the ME1a E-protein.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NHLH1\"],\"tg (associated gene name)\":[\"ME1\"],\"tf text\":[\"NSCL-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gel mobility shift assays show that NSCL-1 only binds DNA as a heterodimeric complex with the ME1a E-protein.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is YY1---{\"tf associated gene name\":[\"GLI3\"],\"tg associated gene name\":[\"FOS\"],\"tf text\":[],\"tg text\":[\"c-fos\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is YY1\"],\"sentence\":[\"We present evidence that YY1\",\"a ubiquitously expressed DNA-binding protein\",\"regulates the activity of the c-fos promoter primarily through an effect on DNA structure.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI3\"],\"tg (associated gene name)\":[\"FOS\"],\"tf text\":[],\"tg text\":[\"c-fos\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We present evidence that YY1, a ubiquitously expressed DNA-binding protein, regulates the activity of the c-fos promoter primarily through an effect on DNA structure.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is YY1---{\"tf associated gene name\":[\"GLI3\"],\"tg associated gene name\":[\"FOS\"],\"tf text\":[],\"tg text\":[\"c-fos\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is YY1\"],\"sentence\":[\"We present evidence that YY1\",\"a ubiquitously expressed DNA-binding protein\",\"regulates the activity of the c-fos promoter primarily through an effect on DNA structure.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GLI3\"],\"tg (associated gene name)\":[\"FOS\"],\"tf text\":[],\"tg text\":[\"c-fos\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We present evidence that YY1, a ubiquitously expressed DNA-binding protein, regulates the activity of the c-fos promoter primarily through an effect on DNA structure.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 227 (char 226)\n",
      "{\"transcription factor (associated gene name)\":[\"TCF4\"],\"target gene (associated gene name)\":[\"RAC3\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|RAC1b?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF4\"],\"tg (associated gene name)\":[\"RAC3\"],\"tf text\":[\"TCF4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"RAC1b?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Both splicing factors were found to be regulated by upstream signaling pathways: the inhibition of the phosphatidylinositol 3-kinase pathway increased protein levels of ASF/SF2 and promoted Rac1b, whereas activation of beta-catenin/TCF4 increased expression of SRp20 and inhibited that of Rac1b.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TCF4\"],\"tg associated gene name\":[\"RAC3\"],\"tf text\":[\"TCF4\"],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Both splicing factors were found to be regulated by upstream signaling pathways: the inhibition of the phosphatidylinositol 3-kinase pathway increased protein levels of ASF/SF2 and promoted Rac1b\",\"whereas activation of beta-catenin/TCF4 increased expression of SRp20 and inhibited that of Rac1b.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TCF4\"],\"target gene (associated gene name)\":[\"RAC3\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|RAC1b?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF4\"],\"tg (associated gene name)\":[\"RAC3\"],\"tf text\":[\"TCF4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"RAC1b?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Both splicing factors were found to be regulated by upstream signaling pathways: the inhibition of the phosphatidylinositol 3-kinase pathway increased protein levels of ASF/SF2 and promoted Rac1b, whereas activation of beta-catenin/TCF4 increased expression of SRp20 and inhibited that of Rac1b.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 384 (char 383)\n",
      "{\"transcription factor (associated gene name)\":[\"SRF\"],\"target gene (associated gene name)\":[\"SERPINE1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Net represses PAI-1 promoter activity and binds to a specific region of the promoter containing Ets binding sites in the absence of SRF.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hmm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SRF\"],\"tg (associated gene name)\":[\"SERPINE1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"hmm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Net represses PAI-1 promoter activity and binds to a specific region of the promoter containing Ets binding sites in the absence of SRF.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SRF\"],\"tg associated gene name\":[\"SERPINE1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"not true\":[],\"comment\":[],\"sentence\":[\"Net represses PAI-1 promoter activity and binds to a specific region of the promoter containing Ets binding sites in the absence of SRF.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SRF\"],\"target gene (associated gene name)\":[\"SERPINE1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Net represses PAI-1 promoter activity and binds to a specific region of the promoter containing Ets binding sites in the absence of SRF.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|hmm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SRF\"],\"tg (associated gene name)\":[\"SERPINE1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"hmm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Net represses PAI-1 promoter activity and binds to a specific region of the promoter containing Ets binding sites in the absence of SRF.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOSL1\"],\"tg (associated gene name)\":[\"NGF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Using DNA-protein gel shift assays we determined that nerve growth factor stimulates AP-1 binding in both PC12 and B5P cells, and identified c-Fos, FosB, Fra-1, Fra-2, c-Jun, JunB and JunD in AP-1 complexes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOSL1\"],\"tg (associated gene name)\":[\"NGF\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Using DNA-protein gel shift assays we determined that nerve growth factor stimulates AP-1 binding in both PC12 and B5P cells, and identified c-Fos, FosB, Fra-1, Fra-2, c-Jun, JunB and JunD in AP-1 complexes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is EGR1---{\"tf associated gene name\":[\"ZNF699\"],\"tg associated gene name\":[\"PTEN\"],\"tf text\":[],\"tg text\":[\"PTEN\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is EGR1\"],\"sentence\":[\"Moreover\",\"we find that under these conditions phosphatase and tensin homolog deleted in chromosome 10 (PTEN) is upregulated and this occurs through direct binding of EGR1 to an element upstream of the PTEN promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF699\"],\"tg (associated gene name)\":[\"PTEN\"],\"tf text\":[],\"tg text\":[\"PTEN\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, we find that under these conditions phosphatase and tensin homolog deleted in chromosome 10 (PTEN) is upregulated and this occurs through direct binding of EGR1 to an element upstream of the PTEN promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is EGR1---{\"tf associated gene name\":[\"ZNF699\"],\"tg associated gene name\":[\"PTEN\"],\"tf text\":[],\"tg text\":[\"PTEN\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is EGR1\"],\"sentence\":[\"Moreover\",\"we find that under these conditions phosphatase and tensin homolog deleted in chromosome 10 (PTEN) is upregulated and this occurs through direct binding of EGR1 to an element upstream of the PTEN promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF699\"],\"tg (associated gene name)\":[\"PTEN\"],\"tf text\":[],\"tg text\":[\"PTEN\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, we find that under these conditions phosphatase and tensin homolog deleted in chromosome 10 (PTEN) is upregulated and this occurs through direct binding of EGR1 to an element upstream of the PTEN promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 540 (char 539)\n",
      "{\"transcription factor (associated gene name)\":[\"POU2F2\"],\"target gene (associated gene name)\":[\"HLA-DRA\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Here we show that the lymphoid-specific transcription factor Oct-2A plays a critical role in HLA-DRA gene expression in class II-positive B cell lines\",\"and that the high mobility group protein (HMG) I/Y binds to multiple sites within the DRA promoter\",\"including the Oct-2A binding site.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|Oct2a?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU2F2\"],\"tg (associated gene name)\":[\"HLA-DRA\"],\"tf text\":[],\"tg text\":[\"HLA-DRA\"],\"consensus\":[\"V\"],\"martin_comments\":[\"Oct2a?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Here we show that the lymphoid-specific transcription factor Oct-2A plays a critical role in HLA-DRA gene expression in class II-positive B cell lines, and that the high mobility group protein (HMG) I/Y binds to multiple sites within the DRA promoter, including the Oct-2A binding site.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"POU2F2\"],\"target gene (associated gene name)\":[\"HLA-DRA\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Here we show that the lymphoid-specific transcription factor Oct-2A plays a critical role in HLA-DRA gene expression in class II-positive B cell lines\",\"and that the high mobility group protein (HMG) I/Y binds to multiple sites within the DRA promoter\",\"including the Oct-2A binding site.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|Oct2a?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU2F2\"],\"tg (associated gene name)\":[\"HLA-DRA\"],\"tf text\":[],\"tg text\":[\"HLA-DRA\"],\"consensus\":[\"V\"],\"martin_comments\":[\"Oct2a?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Here we show that the lymphoid-specific transcription factor Oct-2A plays a critical role in HLA-DRA gene expression in class II-positive B cell lines, and that the high mobility group protein (HMG) I/Y binds to multiple sites within the DRA promoter, including the Oct-2A binding site.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|ZNF384 not mentioned---{\"tf associated gene name\":[\"ZNF384\"],\"tg associated gene name\":[\"ARC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\",\"ZNF384 not mentioned\"],\"sentence\":[\"Acute ethanol exposure decreased amygdaloid HDAC activity and HDAC2 protein levels\",\"increased global and gene (Bdnf and Arc)-specific histone acetylation\",\"and attenuated anxiety-like behaviors in P rats but had no effects in NP rats.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF384\"],\"tg (associated gene name)\":[\"ARC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Acute ethanol exposure decreased amygdaloid HDAC activity and HDAC2 protein levels, increased global and gene (Bdnf and Arc)-specific histone acetylation, and attenuated anxiety-like behaviors in P rats but had no effects in NP rats.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE|ZNF384 not mentioned---{\"tf associated gene name\":[\"ZNF384\"],\"tg associated gene name\":[\"ARC\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\",\"ZNF384 not mentioned\"],\"sentence\":[\"Acute ethanol exposure decreased amygdaloid HDAC activity and HDAC2 protein levels\",\"increased global and gene (Bdnf and Arc)-specific histone acetylation\",\"and attenuated anxiety-like behaviors in P rats but had no effects in NP rats.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF384\"],\"tg (associated gene name)\":[\"ARC\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Acute ethanol exposure decreased amygdaloid HDAC activity and HDAC2 protein levels, increased global and gene (Bdnf and Arc)-specific histone acetylation, and attenuated anxiety-like behaviors in P rats but had no effects in NP rats.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is miR-211---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"KRT4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is miR-211\"],\"sentence\":[\"We found that Tcf3 down-regulation in the context of constitutively active Wnt signaling does not result from promoter DNA methylation but is likely to be caused by a plethora of mechanisms at both the RNA and protein level as shown by the observed decrease in activating histone marks (H3K4me3 and H3-acetylation) and the upregulation of miR-211\",\"a novel Wnt-regulated microRNA that targets Tcf3 and attenuates early neural differentiation in mouse ESCs.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TCF3 not TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"KRT4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TCF3 not TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that Tcf3 down-regulation in the context of constitutively active Wnt signaling does not result from promoter DNA methylation but is likely to be caused by a plethora of mechanisms at both the RNA and protein level as shown by the observed decrease in activating histone marks (H3K4me3 and H3-acetylation) and the upregulation of miR-211, a novel Wnt-regulated microRNA that targets Tcf3 and attenuates early neural differentiation in mouse ESCs.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is miR-211---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"KRT4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is miR-211\"],\"sentence\":[\"We found that Tcf3 down-regulation in the context of constitutively active Wnt signaling does not result from promoter DNA methylation but is likely to be caused by a plethora of mechanisms at both the RNA and protein level as shown by the observed decrease in activating histone marks (H3K4me3 and H3-acetylation) and the upregulation of miR-211\",\"a novel Wnt-regulated microRNA that targets Tcf3 and attenuates early neural differentiation in mouse ESCs.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TCF3 not TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"KRT4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TCF3 not TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that Tcf3 down-regulation in the context of constitutively active Wnt signaling does not result from promoter DNA methylation but is likely to be caused by a plethora of mechanisms at both the RNA and protein level as shown by the observed decrease in activating histone marks (H3K4me3 and H3-acetylation) and the upregulation of miR-211, a novel Wnt-regulated microRNA that targets Tcf3 and attenuates early neural differentiation in mouse ESCs.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"GFI1\"],\"tg associated gene name\":[\"CSF3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"The transcription factor growth factor independence 1 (Gfi1) and the growth factor granulocyte colony-stimulating factor (G-CSF) are individually essential for neutrophil differentiation from myeloid progenitors.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GFI1\"],\"tg (associated gene name)\":[\"CSF3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcription factor growth factor independence 1 (Gfi1) and the growth factor granulocyte colony-stimulating factor (G-CSF) are individually essential for neutrophil differentiation from myeloid progenitors.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"GFI1\"],\"tg associated gene name\":[\"CSF3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"The transcription factor growth factor independence 1 (Gfi1) and the growth factor granulocyte colony-stimulating factor (G-CSF) are individually essential for neutrophil differentiation from myeloid progenitors.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GFI1\"],\"tg (associated gene name)\":[\"CSF3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcription factor growth factor independence 1 (Gfi1) and the growth factor granulocyte colony-stimulating factor (G-CSF) are individually essential for neutrophil differentiation from myeloid progenitors.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 469 (char 468)\n",
      "{\"tf associated gene name\":[\"SRF\"],\"tg associated gene name\":[\"KRAS\"],\"tf text\":[\"SRF\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These results show that SRF\",\"thought to be involved in stimulation of cell growth through activation of growth factor-inducible genes\",\"can actually have the opposite effect and suggest a novel mechanism for suppression of transformation by Ras.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG problem and no transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SRF\"],\"tg (associated gene name)\":[\"KRAS\"],\"tf text\":[\"SRF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG problem and no transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results show that SRF, thought to be involved in stimulation of cell growth through activation of growth factor-inducible genes, can actually have the opposite effect and suggest a novel mechanism for suppression of transformation by Ras.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SRF\"],\"tg associated gene name\":[\"KRAS\"],\"tf text\":[\"SRF\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These results show that SRF\",\"thought to be involved in stimulation of cell growth through activation of growth factor-inducible genes\",\"can actually have the opposite effect and suggest a novel mechanism for suppression of transformation by Ras.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG problem and no transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SRF\"],\"tg (associated gene name)\":[\"KRAS\"],\"tf text\":[\"SRF\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG problem and no transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results show that SRF, thought to be involved in stimulation of cell growth through activation of growth factor-inducible genes, can actually have the opposite effect and suggest a novel mechanism for suppression of transformation by Ras.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 386 (char 385)\n",
      "{\"tf associated gene name\":[\"KDM2A\"],\"tg associated gene name\":[\"NANOS1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"HDAC3 knockdown significantly recovered the proliferation and invasiveness of KDM2A-depleted NSCLC cells as well as the levels of CDK6 and NANOS1 expression in these cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no causality---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KDM2A\"],\"tg (associated gene name)\":[\"NANOS1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no causality\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HDAC3 knockdown significantly recovered the proliferation and invasiveness of KDM2A-depleted NSCLC cells as well as the levels of CDK6 and NANOS1 expression in these cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"KDM2A\"],\"tg associated gene name\":[\"NANOS1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"HDAC3 knockdown significantly recovered the proliferation and invasiveness of KDM2A-depleted NSCLC cells as well as the levels of CDK6 and NANOS1 expression in these cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no causality---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KDM2A\"],\"tg (associated gene name)\":[\"NANOS1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no causality\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HDAC3 knockdown significantly recovered the proliferation and invasiveness of KDM2A-depleted NSCLC cells as well as the levels of CDK6 and NANOS1 expression in these cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 398 (char 397)\n",
      "{\"tf associated gene name\":[\"ARNT\"],\"tg associated gene name\":[\"PML\"],\"tf text\":[\"ARNT\"],\"tg text\":[\"PML\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Our data suggest that the sumoylation of ARNT modulates its transcriptional role through affecting the ability of ARNT to interact with cooperative molecules such as PML.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ARNT\"],\"tg (associated gene name)\":[\"PML\"],\"tf text\":[\"ARNT\"],\"tg text\":[\"PML\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our data suggest that the sumoylation of ARNT modulates its transcriptional role through affecting the ability of ARNT to interact with cooperative molecules such as PML.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ARNT\"],\"tg associated gene name\":[\"PML\"],\"tf text\":[\"ARNT\"],\"tg text\":[\"PML\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Our data suggest that the sumoylation of ARNT modulates its transcriptional role through affecting the ability of ARNT to interact with cooperative molecules such as PML.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ARNT\"],\"tg (associated gene name)\":[\"PML\"],\"tf text\":[\"ARNT\"],\"tg text\":[\"PML\"],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our data suggest that the sumoylation of ARNT modulates its transcriptional role through affecting the ability of ARNT to interact with cooperative molecules such as PML.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is PGDH---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"HPGD\"],\"tf text\":[],\"tg text\":[\"PGDH\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is PGDH\"],\"sentence\":[\"Taken together\",\"these findings demonstrate control of the PGDH promoter by multiple pathways and provide evidence for cross-talk among Ets\",\"AP-1\",\"cAMP\",\"and PR-mediated signaling\",\"suggesting complex regulatory mechanisms for the expression of PGDH.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|causality yes, but through pathway?|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"HPGD\"],\"tf text\":[],\"tg text\":[\"PGDH\"],\"consensus\":[\"N\"],\"martin_comments\":[\"causality yes, but through pathway?\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[\"Taken together, these findings demonstrate control of the PGDH promoter by multiple pathways and provide evidence for cross-talk among Ets, AP-1, cAMP, and PR-mediated signaling, suggesting complex regulatory mechanisms for the expression of PGDH.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is PGDH---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"HPGD\"],\"tf text\":[],\"tg text\":[\"PGDH\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is PGDH\"],\"sentence\":[\"Taken together\",\"these findings demonstrate control of the PGDH promoter by multiple pathways and provide evidence for cross-talk among Ets\",\"AP-1\",\"cAMP\",\"and PR-mediated signaling\",\"suggesting complex regulatory mechanisms for the expression of PGDH.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|causality yes, but through pathway?|UPSTR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"HPGD\"],\"tf text\":[],\"tg text\":[\"PGDH\"],\"consensus\":[\"N\"],\"martin_comments\":[\"causality yes, but through pathway?\"],\"astrid_comments\":[\"UPSTR\"],\"sentences with greek letters\":[\"Taken together, these findings demonstrate control of the PGDH promoter by multiple pathways and provide evidence for cross-talk among Ets, AP-1, cAMP, and PR-mediated signaling, suggesting complex regulatory mechanisms for the expression of PGDH.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no protein looks like IRF6---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IFNB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"no protein looks like IRF6\"],\"sentence\":[\"These results indicate that both LPS-induced NF-kappaB activation and endogenous production of IFN-beta that subsequently induces STAT-1alpha activation play critical roles in the transcriptional activation of the CD40 gene by LPS.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF and TG not linked---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IFNB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF and TG not linked\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that both LPS-induced NF-kappaB activation and endogenous production of IFN-beta that subsequently induces STAT-1alpha activation play critical roles in the transcriptional activation of the CD40 gene by LPS.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no protein looks like IRF6---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IFNB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"no protein looks like IRF6\"],\"sentence\":[\"These results indicate that both LPS-induced NF-kappaB activation and endogenous production of IFN-beta that subsequently induces STAT-1alpha activation play critical roles in the transcriptional activation of the CD40 gene by LPS.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF and TG not linked---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IFNB1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF and TG not linked\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results indicate that both LPS-induced NF-kappaB activation and endogenous production of IFN-beta that subsequently induces STAT-1alpha activation play critical roles in the transcriptional activation of the CD40 gene by LPS.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Salmonella SPI-1---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"TF is Salmonella SPI-1\"],\"sentence\":[\"We present evidence that two such loci (stdE and stdF) are involved in Dam-dependent control of Salmonella SPI-1: in a Dam(-) background\",\"deletion of stdE or stdF suppresses SPI-1 repression; in a Dam(+) background\",\"constitutive expression of StdE and/or StdF represses SPI-1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SPI1 = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SPI1 = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We present evidence that two such loci (stdE and stdF) are involved in Dam-dependent control of Salmonella SPI-1: in a Dam(-) background, deletion of stdE or stdF suppresses SPI-1 repression; in a Dam(+) background, constitutive expression of StdE and/or StdF represses SPI-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is Salmonella SPI-1---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"s\"],\"comment\":[\"TF is Salmonella SPI-1\"],\"sentence\":[\"We present evidence that two such loci (stdE and stdF) are involved in Dam-dependent control of Salmonella SPI-1: in a Dam(-) background\",\"deletion of stdE or stdF suppresses SPI-1 repression; in a Dam(+) background\",\"constitutive expression of StdE and/or StdF represses SPI-1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|SPI1 = TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"SPI1 = TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We present evidence that two such loci (stdE and stdF) are involved in Dam-dependent control of Salmonella SPI-1: in a Dam(-) background, deletion of stdE or stdF suppresses SPI-1 repression; in a Dam(+) background, constitutive expression of StdE and/or StdF represses SPI-1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 400 (char 399)\n",
      "{\"tf associated gene name\":[\"ELF4\"],\"tg associated gene name\":[\"MAP2K2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Using mouse embryonic fibroblasts (MEF) from MK2 knockout (MK2(-/-)) mice\",\"we demonstrate that disrupting expression of MK2 expression reduces filamentous actin and stress fibers.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MEF definitely not a gene her---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF4\"],\"tg (associated gene name)\":[\"MAP2K2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MEF definitely not a gene her\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using mouse embryonic fibroblasts (MEF) from MK2 knockout (MK2(-/-)) mice, we demonstrate that disrupting expression of MK2 expression reduces filamentous actin and stress fibers.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ELF4\"],\"tg associated gene name\":[\"MAP2K2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Using mouse embryonic fibroblasts (MEF) from MK2 knockout (MK2(-/-)) mice\",\"we demonstrate that disrupting expression of MK2 expression reduces filamentous actin and stress fibers.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|MEF definitely not a gene her---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ELF4\"],\"tg (associated gene name)\":[\"MAP2K2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"MEF definitely not a gene her\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using mouse embryonic fibroblasts (MEF) from MK2 knockout (MK2(-/-)) mice, we demonstrate that disrupting expression of MK2 expression reduces filamentous actin and stress fibers.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs are HES-1 and c-Myb.---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"RAG1\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"Rag-1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TGs are HES-1 and c-Myb.\"],\"sentence\":[\"Both transcription factors can interfere with expression of the Rag-1 and Rag-2 recombinases\",\"while GATA-3 notably blocks PU.1 and IL-7Ralpha expression\",\"and PU.1 reduces expression of HES-1 and c-Myb.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|comention at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"RAG1\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"Rag-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"comention at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Both transcription factors can interfere with expression of the Rag-1 and Rag-2 recombinases, while GATA-3 notably blocks PU.1 and IL-7Ralpha expression, and PU.1 reduces expression of HES-1 and c-Myb.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TGs are HES-1 and c-Myb.---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"RAG1\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"Rag-1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TGs are HES-1 and c-Myb.\"],\"sentence\":[\"Both transcription factors can interfere with expression of the Rag-1 and Rag-2 recombinases\",\"while GATA-3 notably blocks PU.1 and IL-7Ralpha expression\",\"and PU.1 reduces expression of HES-1 and c-Myb.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|comention at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"RAG1\"],\"tf text\":[\"PU.1\"],\"tg text\":[\"Rag-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"comention at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Both transcription factors can interfere with expression of the Rag-1 and Rag-2 recombinases, while GATA-3 notably blocks PU.1 and IL-7Ralpha expression, and PU.1 reduces expression of HES-1 and c-Myb.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "Neurog is TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NEUROG3\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"neurogenin 3\"],\"tg text\":[\"insulin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"Neurog is TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"When we overexpressed three ITFs, INSM1/Pdx-1/NeuroD1, in the Panc-1 differentiation model, higher insulin expression was observed in parallel with the activation of an additional ITF, neurogenin 3, signifying endocrine cell activation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "Neurog is TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NEUROG3\"],\"tg (associated gene name)\":[\"INS\"],\"tf text\":[\"neurogenin 3\"],\"tg text\":[\"insulin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"Neurog is TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"When we overexpressed three ITFs, INSM1/Pdx-1/NeuroD1, in the Panc-1 differentiation model, higher insulin expression was observed in parallel with the activation of an additional ITF, neurogenin 3, signifying endocrine cell activation.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is ERRalpha---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is ERRalpha\"],\"sentence\":[\"ERRalpha fulfills this role by acting directly at genes important for mitochondrial function\",\"parallel to other factors controlling mitochondrial gene expression\",\"such as NRF1 and NRF2/GABPA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"ERRalpha fulfills this role by acting directly at genes important for mitochondrial function, parallel to other factors controlling mitochondrial gene expression, such as NRF1 and NRF2/GABPA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is ERRalpha---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is ERRalpha\"],\"sentence\":[\"ERRalpha fulfills this role by acting directly at genes important for mitochondrial function\",\"parallel to other factors controlling mitochondrial gene expression\",\"such as NRF1 and NRF2/GABPA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"GFM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"ERRalpha fulfills this role by acting directly at genes important for mitochondrial function, parallel to other factors controlling mitochondrial gene expression, such as NRF1 and NRF2/GABPA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is PIG3---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"MRPL41\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is PIG3\"],\"sentence\":[\"The p53-homolog p73beta also activated the PIG3 promoter\",\"but in contrast to p53\",\"the proline-rich domain of p73beta (residues 81-113) was dispensable to induce the PIG3 promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"MRPL41\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The p53-homolog p73beta also activated the PIG3 promoter, but in contrast to p53, the proline-rich domain of p73beta (residues 81-113) was dispensable to induce the PIG3 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is PIG3---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"MRPL41\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is PIG3\"],\"sentence\":[\"The p53-homolog p73beta also activated the PIG3 promoter\",\"but in contrast to p53\",\"the proline-rich domain of p73beta (residues 81-113) was dispensable to induce the PIG3 promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"MRPL41\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The p53-homolog p73beta also activated the PIG3 promoter, but in contrast to p53, the proline-rich domain of p73beta (residues 81-113) was dispensable to induce the PIG3 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TFs are OCT3/4|NANOG|SOX2 and Polycomb Group proteins---{\"tf associated gene name\":[\"AHR\"],\"tg associated gene name\":[\"AHR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TFs are OCT3/4\",\"NANOG\",\"SOX2 and Polycomb Group proteins\"],\"sentence\":[\"In ES cells\",\"interactions between OCT3/4\",\"NANOG\",\"SOX2 and Polycomb Group proteins at the Ahr promoter repress AHR expression\",\"which can also be repressed by ectopic expression of reprogramming factors in hepatoma cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AHR\"],\"tg (associated gene name)\":[\"AHR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In ES cells, interactions between OCT3/4, NANOG, SOX2 and Polycomb Group proteins at the Ahr promoter repress AHR expression, which can also be repressed by ectopic expression of reprogramming factors in hepatoma cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TFs are OCT3/4|NANOG|SOX2 and Polycomb Group proteins---{\"tf associated gene name\":[\"AHR\"],\"tg associated gene name\":[\"AHR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TFs are OCT3/4\",\"NANOG\",\"SOX2 and Polycomb Group proteins\"],\"sentence\":[\"In ES cells\",\"interactions between OCT3/4\",\"NANOG\",\"SOX2 and Polycomb Group proteins at the Ahr promoter repress AHR expression\",\"which can also be repressed by ectopic expression of reprogramming factors in hepatoma cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AHR\"],\"tg (associated gene name)\":[\"AHR\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In ES cells, interactions between OCT3/4, NANOG, SOX2 and Polycomb Group proteins at the Ahr promoter repress AHR expression, which can also be repressed by ectopic expression of reprogramming factors in hepatoma cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"CTNNBL1\"],\"tg associated gene name\":[\"TIMP3\"],\"tf text\":[],\"tg text\":[\"TIMP3\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"We have now analyzed a series of promoter hypermethylation changes in 12 genes (p16(INK4a)\",\"p15(INK4b)\",\"p14(ARF)\",\"p73\",\"APC\",\"(5) BRCA1\",\"hMLH1\",\"GSTP1\",\"MGMT\",\"CDH1\",\"TIMP3\",\"and DAPK)\",\"each rigorously characterized for association with abnormal gene silencing in cancer\",\"in DNA from over 600 primary tumor samples representing 15 major tumor types.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CTNNBL1\"],\"tg (associated gene name)\":[\"TIMP3\"],\"tf text\":[],\"tg text\":[\"TIMP3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We have now analyzed a series of promoter hypermethylation changes in 12 genes (p16(INK4a), p15(INK4b), p14(ARF), p73, APC,(5) BRCA1, hMLH1, GSTP1, MGMT, CDH1, TIMP3, and DAPK), each rigorously characterized for association with abnormal gene silencing in cancer, in DNA from over 600 primary tumor samples representing 15 major tumor types.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"CTNNBL1\"],\"tg associated gene name\":[\"TIMP3\"],\"tf text\":[],\"tg text\":[\"TIMP3\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"We have now analyzed a series of promoter hypermethylation changes in 12 genes (p16(INK4a)\",\"p15(INK4b)\",\"p14(ARF)\",\"p73\",\"APC\",\"(5) BRCA1\",\"hMLH1\",\"GSTP1\",\"MGMT\",\"CDH1\",\"TIMP3\",\"and DAPK)\",\"each rigorously characterized for association with abnormal gene silencing in cancer\",\"in DNA from over 600 primary tumor samples representing 15 major tumor types.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CTNNBL1\"],\"tg (associated gene name)\":[\"TIMP3\"],\"tf text\":[],\"tg text\":[\"TIMP3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We have now analyzed a series of promoter hypermethylation changes in 12 genes (p16(INK4a), p15(INK4b), p14(ARF), p73, APC,(5) BRCA1, hMLH1, GSTP1, MGMT, CDH1, TIMP3, and DAPK), each rigorously characterized for association with abnormal gene silencing in cancer, in DNA from over 600 primary tumor samples representing 15 major tumor types.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is CYP11B2---{\"tf associated gene name\":[\"NR4A2\"],\"tg associated gene name\":[\"AGT\"],\"tf text\":[\"NURR1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is CYP11B2\"],\"sentence\":[\"Our results suggest that NGFIB and NURR1 are key regulators of hCYP11B2 expression and may partially mediate the regulation of hCYP11B2 by Ang II.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A2\"],\"tg (associated gene name)\":[\"AGT\"],\"tf text\":[\"NURR1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that NGFIB and NURR1 are key regulators of hCYP11B2 expression and may partially mediate the regulation of hCYP11B2 by Ang II.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is CYP11B2---{\"tf associated gene name\":[\"NR4A2\"],\"tg associated gene name\":[\"AGT\"],\"tf text\":[\"NURR1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is CYP11B2\"],\"sentence\":[\"Our results suggest that NGFIB and NURR1 are key regulators of hCYP11B2 expression and may partially mediate the regulation of hCYP11B2 by Ang II.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A2\"],\"tg (associated gene name)\":[\"AGT\"],\"tf text\":[\"NURR1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our results suggest that NGFIB and NURR1 are key regulators of hCYP11B2 expression and may partially mediate the regulation of hCYP11B2 by Ang II.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EZH2\"],\"tg (associated gene name)\":[\"HDAC9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"EZH2-mediated gene silencing was shown to be dependent on histone deacetylase (HDAC) activity.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EZH2\"],\"tg (associated gene name)\":[\"HDAC9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"EZH2-mediated gene silencing was shown to be dependent on histone deacetylase (HDAC) activity.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is MMP-9---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[\"tumor necrosis factor alpha\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is MMP-9\"],\"sentence\":[\"NFkappaB plays a predominant role in MMP-9 gene induction in RKO cells\",\"as evidenced by the failure of tumor necrosis factor alpha (TNFalpha) to induce MMP-9 in either of the NFkappaB signaling mutants.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[\"tumor necrosis factor alpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"NFkappaB plays a predominant role in MMP-9 gene induction in RKO cells, as evidenced by the failure of tumor necrosis factor alpha (TNFalpha) to induce MMP-9 in either of the NFkappaB signaling mutants.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is MMP-9---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[\"tumor necrosis factor alpha\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is MMP-9\"],\"sentence\":[\"NFkappaB plays a predominant role in MMP-9 gene induction in RKO cells\",\"as evidenced by the failure of tumor necrosis factor alpha (TNFalpha) to induce MMP-9 in either of the NFkappaB signaling mutants.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TNF\"],\"tf text\":[],\"tg text\":[\"tumor necrosis factor alpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"NFkappaB plays a predominant role in MMP-9 gene induction in RKO cells, as evidenced by the failure of tumor necrosis factor alpha (TNFalpha) to induce MMP-9 in either of the NFkappaB signaling mutants.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"TBP\"],\"tg associated gene name\":[\"ERVW-4\"],\"tf text\":[\"TBP\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Here\",\"we employ single-molecule F??rster resonance energy transfer experiments and Nano-Positioning System analysis to determine the three-dimensional architecture of a minimal OC consisting of promoter DNA\",\"including a TATA box and an 11-nucleotide mismatched region around the transcription start site\",\"TATA box-binding protein (TBP)\",\"RNA polymerase (Pol) II\",\"and general transcription factor (TF)IIB and TFIIF.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|notheing here---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBP\"],\"tg (associated gene name)\":[\"ERVW-4\"],\"tf text\":[\"TBP\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"notheing here\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here, we employ single-molecule Frster resonance energy transfer experiments and Nano-Positioning System analysis to determine the three-dimensional architecture of a minimal OC consisting of promoter DNA, including a TATA box and an 11-nucleotide mismatched region around the transcription start site, TATA box-binding protein (TBP), RNA polymerase (Pol) II, and general transcription factor (TF)IIB and TFIIF.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named---{\"tf associated gene name\":[\"TBP\"],\"tg associated gene name\":[\"ERVW-4\"],\"tf text\":[\"TBP\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Here\",\"we employ single-molecule F??rster resonance energy transfer experiments and Nano-Positioning System analysis to determine the three-dimensional architecture of a minimal OC consisting of promoter DNA\",\"including a TATA box and an 11-nucleotide mismatched region around the transcription start site\",\"TATA box-binding protein (TBP)\",\"RNA polymerase (Pol) II\",\"and general transcription factor (TF)IIB and TFIIF.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|notheing here---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBP\"],\"tg (associated gene name)\":[\"ERVW-4\"],\"tf text\":[\"TBP\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"notheing here\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here, we employ single-molecule Frster resonance energy transfer experiments and Nano-Positioning System analysis to determine the three-dimensional architecture of a minimal OC consisting of promoter DNA, including a TATA box and an 11-nucleotide mismatched region around the transcription start site, TATA box-binding protein (TBP), RNA polymerase (Pol) II, and general transcription factor (TF)IIB and TFIIF.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 407 (char 406)\n",
      "{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"In addition\",\"one line had rearranged and transcribed endogenous TCR alpha and beta genes\",\"in spite of the fact that transgenic alpha and beta genes were also expressed.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SSRP1\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In addition, one line had rearranged and transcribed endogenous TCR alpha and beta genes, in spite of the fact that transgenic alpha and beta genes were also expressed.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[],\"sentence\":[\"In addition\",\"one line had rearranged and transcribed endogenous TCR alpha and beta genes\",\"in spite of the fact that transgenic alpha and beta genes were also expressed.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SSRP1\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In addition, one line had rearranged and transcribed endogenous TCR alpha and beta genes, in spite of the fact that transgenic alpha and beta genes were also expressed.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "sentence states that regulator is ligand of nuclear ligand activated TF PPARA; TF-TG correctly found ---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"ALDH2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"sentence states that regulator is ligand of nuclear ligand activated TF PPARA; TF-TG correctly found \"],\"sentence\":[\"Treatment of rats with the PPAR ligand clofibrate repressed expression of ALDH2 in rats fed either stock rodent chow or a low-protein diet.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "sentence states that regulator is ligand of nuclear ligand activated TF PPARA; TF-TG correctly found ---{\"tf associated gene name\":[\"PPARA\"],\"tg associated gene name\":[\"ALDH2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"sentence states that regulator is ligand of nuclear ligand activated TF PPARA; TF-TG correctly found \"],\"sentence\":[\"Treatment of rats with the PPAR ligand clofibrate repressed expression of ALDH2 in rats fed either stock rodent chow or a low-protein diet.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PPARA\"],\"target gene (associated gene name)\":[\"ALDH2\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Treatment of rats with the PPAR ligand clofibrate repressed expression of ALDH2 in rats fed either stock rodent chow or a low-protein diet.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARA\"],\"tg (associated gene name)\":[\"ALDH2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Treatment of rats with the PPAR ligand clofibrate repressed expression of ALDH2 in rats fed either stock rodent chow or a low-protein diet.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Fgf21---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"ASIP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is Fgf21\"],\"sentence\":[\"We demonstrate that the G9a-specific inhibitor BIX01294 abolishes suppression of the Fgf21 promoter activity by E4BP4\",\"whereas overexpression of E4bp4 leads to increased levels of dimethylation of histone 3 lysine 9 (H3K9me2) around the Fgf21 promoter region.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"ASIP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrate that the G9a-specific inhibitor BIX01294 abolishes suppression of the Fgf21 promoter activity by E4BP4, whereas overexpression of E4bp4 leads to increased levels of dimethylation of histone 3 lysine 9 (H3K9me2) around the Fgf21 promoter region.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Fgf21---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"ASIP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is Fgf21\"],\"sentence\":[\"We demonstrate that the G9a-specific inhibitor BIX01294 abolishes suppression of the Fgf21 promoter activity by E4BP4\",\"whereas overexpression of E4bp4 leads to increased levels of dimethylation of histone 3 lysine 9 (H3K9me2) around the Fgf21 promoter region.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"ASIP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrate that the G9a-specific inhibitor BIX01294 abolishes suppression of the Fgf21 promoter activity by E4BP4, whereas overexpression of E4bp4 leads to increased levels of dimethylation of histone 3 lysine 9 (H3K9me2) around the Fgf21 promoter region.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG---{\"tf associated gene name\":[\"NR5A1\"],\"tg associated gene name\":[\"NRIP1\"],\"tf text\":[],\"tg text\":[\"RIP 140\"],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"We conclude that RIP 140 binds to the orphan nuclear receptor SF-1 and DAX-1 and modulates their actions on the human steroidogenic acute regulatory protein promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR5A1\"],\"tg (associated gene name)\":[\"NRIP1\"],\"tf text\":[],\"tg text\":[\"RIP 140\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We conclude that RIP 140 binds to the orphan nuclear receptor SF-1 and DAX-1 and modulates their actions on the human steroidogenic acute regulatory protein promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG---{\"tf associated gene name\":[\"NR5A1\"],\"tg associated gene name\":[\"NRIP1\"],\"tf text\":[],\"tg text\":[\"RIP 140\"],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"We conclude that RIP 140 binds to the orphan nuclear receptor SF-1 and DAX-1 and modulates their actions on the human steroidogenic acute regulatory protein promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR5A1\"],\"tg (associated gene name)\":[\"NRIP1\"],\"tf text\":[],\"tg text\":[\"RIP 140\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We conclude that RIP 140 binds to the orphan nuclear receptor SF-1 and DAX-1 and modulates their actions on the human steroidogenic acute regulatory protein promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 339 (char 338)\n",
      "{\"tf associated gene name\":[\"NR4A1\"],\"tg associated gene name\":[\"INSL3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Using RT-PCR\",\"we found that Nr4a1 is coexpressed with Insl3 in purified Leydig cells and in several Leydig cell lines.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|coexpressed is not causal?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A1\"],\"tg (associated gene name)\":[\"INSL3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"coexpressed is not causal?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using RT-PCR, we found that Nr4a1 is coexpressed with Insl3 in purified Leydig cells and in several Leydig cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR4A1\"],\"tg associated gene name\":[\"INSL3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Using RT-PCR\",\"we found that Nr4a1 is coexpressed with Insl3 in purified Leydig cells and in several Leydig cell lines.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|coexpressed is not causal?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR4A1\"],\"tg (associated gene name)\":[\"INSL3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"coexpressed is not causal?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using RT-PCR, we found that Nr4a1 is coexpressed with Insl3 in purified Leydig cells and in several Leydig cell lines.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is cyclin D1---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"MAPK1\"],\"tf text\":[\"ERalpha\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is cyclin D1\"],\"sentence\":[\"We found that palmitoylation of ERalpha enacts ERalpha association with the plasma membrane\",\"interaction with the membrane protein caveolin-1\",\"and nongenomic activities\",\"including activation of signaling pathways and cell proliferation (i.e.\",\"ERK and AKT activation\",\"cyclin D1 promoter activity\",\"DNA synthesis).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|a bit iffy---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"MAPK1\"],\"tf text\":[\"ERalpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"a bit iffy\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that palmitoylation of ERalpha enacts ERalpha association with the plasma membrane, interaction with the membrane protein caveolin-1, and nongenomic activities, including activation of signaling pathways and cell proliferation (i.e., ERK and AKT activation, cyclin D1 promoter activity, DNA synthesis).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is cyclin D1---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"MAPK1\"],\"tf text\":[\"ERalpha\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is cyclin D1\"],\"sentence\":[\"We found that palmitoylation of ERalpha enacts ERalpha association with the plasma membrane\",\"interaction with the membrane protein caveolin-1\",\"and nongenomic activities\",\"including activation of signaling pathways and cell proliferation (i.e.\",\"ERK and AKT activation\",\"cyclin D1 promoter activity\",\"DNA synthesis).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|a bit iffy---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"MAPK1\"],\"tf text\":[\"ERalpha\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"a bit iffy\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that palmitoylation of ERalpha enacts ERalpha association with the plasma membrane, interaction with the membrane protein caveolin-1, and nongenomic activities, including activation of signaling pathways and cell proliferation (i.e., ERK and AKT activation, cyclin D1 promoter activity, DNA synthesis).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 320 (char 319)\n",
      "{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[\"interferon regulatory factor-1\"],\"tg text\":[\"prolactin\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Stimulation of interferon regulatory factor-1 by prolactin.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[\"interferon regulatory factor-1\"],\"tg text\":[\"prolactin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Stimulation of interferon regulatory factor-1 by prolactin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[\"interferon regulatory factor-1\"],\"tg text\":[\"prolactin\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Stimulation of interferon regulatory factor-1 by prolactin.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[\"interferon regulatory factor-1\"],\"tg text\":[\"prolactin\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Stimulation of interferon regulatory factor-1 by prolactin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYOD1\"],\"tg (associated gene name)\":[\"MYOD1\"],\"tf text\":[\"MyoD\"],\"tg text\":[\"MyoD\"],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We now show that the homeobox gene Pitx2 is required for initial activation of the MyoD gene in limb muscle precursors through direct binding of Pitx2 to the MyoD core enhancer.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYOD1\"],\"tg (associated gene name)\":[\"MYOD1\"],\"tf text\":[\"MyoD\"],\"tg text\":[\"MyoD\"],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"We now show that the homeobox gene Pitx2 is required for initial activation of the MyoD gene in limb muscle precursors through direct binding of Pitx2 to the MyoD core enhancer.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 402 (char 401)\n",
      "{\"tf associated gene name\":[\"MXI1\"],\"tg associated gene name\":[\"MAX\"],\"tf text\":[\"Mxi1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"These results suggest that alterations in the levels of Max-associated proteins such as Mxi1 can modulate critical levels of functional Myc/Max protein complexes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level|NEUROG3 IS TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MXI1\"],\"tg (associated gene name)\":[\"MAX\"],\"tf text\":[\"Mxi1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[\"NEUROG3 IS TG\"],\"sentences with greek letters\":[\"These results suggest that alterations in the levels of Max-associated proteins such as Mxi1 can modulate critical levels of functional Myc/Max protein complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MXI1\"],\"tg associated gene name\":[\"MAX\"],\"tf text\":[\"Mxi1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[],\"sentence\":[\"These results suggest that alterations in the levels of Max-associated proteins such as Mxi1 can modulate critical levels of functional Myc/Max protein complexes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level|NEUROG3 IS TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MXI1\"],\"tg (associated gene name)\":[\"MAX\"],\"tf text\":[\"Mxi1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[\"NEUROG3 IS TG\"],\"sentences with greek letters\":[\"These results suggest that alterations in the levels of Max-associated proteins such as Mxi1 can modulate critical levels of functional Myc/Max protein complexes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF normalization unsure|but is chromatin modeller---{\"tf associated gene name\":[\"SMARCA1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[],\"tg text\":[\"Prl\"],\"1\":[\"u\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TF normalization unsure\",\"but is chromatin modeller\"],\"sentence\":[\"Circadian Prl transcription depended on an interaction between the pituitary-specific transcription factor\",\"PIT-1\",\"and the helicase-like transcription factor (HLTF)\",\"a SWI/SNF chromatin remodeler\",\"shown here to bind the Prl promoter on an E-box that differs from the specific E-box preferentially bound by clock proteins.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TF normalization unsure|but is chromatin modeller---{\"tf associated gene name\":[\"SMARCA1\"],\"tg associated gene name\":[\"PRL\"],\"tf text\":[],\"tg text\":[\"Prl\"],\"1\":[\"u\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TF normalization unsure\",\"but is chromatin modeller\"],\"sentence\":[\"Circadian Prl transcription depended on an interaction between the pituitary-specific transcription factor\",\"PIT-1\",\"and the helicase-like transcription factor (HLTF)\",\"a SWI/SNF chromatin remodeler\",\"shown here to bind the Prl promoter on an E-box that differs from the specific E-box preferentially bound by clock proteins.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SMARCA1\"],\"target gene (associated gene name)\":[\"PRL\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Circadian Prl transcription depended on an interaction between the pituitary-specific transcription factor\",\"PIT-1\",\"and the helicase-like transcription factor (HLTF)\",\"a SWI/SNF chromatin remodeler\",\"shown here to bind the Prl promoter on an E-box that differs from the specific E-box preferentially bound by clock proteins.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMARCA1\"],\"tg (associated gene name)\":[\"PRL\"],\"tf text\":[],\"tg text\":[\"Prl\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Circadian Prl transcription depended on an interaction between the pituitary-specific transcription factor, PIT-1, and the helicase-like transcription factor (HLTF), a SWI/SNF chromatin remodeler, shown here to bind the Prl promoter on an E-box that differs from the specific E-box preferentially bound by clock proteins.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 348 (char 347)\n",
      "{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"SOCS3\"],\"tf text\":[\"STAT5\"],\"tg text\":[\"SOCS3\"],\"1\":[],\"not true\":[\"o\"],\"comment\":[],\"sentence\":[\"Gfi1 directly regulates expression levels of IL7Ralpha and indirectly controls STAT5 signalling via expression of SOCS3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"SOCS3\"],\"tf text\":[\"STAT5\"],\"tg text\":[\"SOCS3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gfi1 directly regulates expression levels of IL7Ralpha and indirectly controls STAT5 signalling via expression of SOCS3.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"SOCS3\"],\"tf text\":[\"STAT5\"],\"tg text\":[\"SOCS3\"],\"1\":[],\"not true\":[\"o\"],\"comment\":[],\"sentence\":[\"Gfi1 directly regulates expression levels of IL7Ralpha and indirectly controls STAT5 signalling via expression of SOCS3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"SOCS3\"],\"tf text\":[\"STAT5\"],\"tg text\":[\"SOCS3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Gfi1 directly regulates expression levels of IL7Ralpha and indirectly controls STAT5 signalling via expression of SOCS3.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "greek letter needed---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"greek letter needed\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Therefore, although IFN-?? alone are not able to induce cell death in tumor cells in vitro, Noxa induction by IFN-??, which is regulated by the balance between its activators (IRF1, 3, and 7) and its repressor (IRF4), is crucial to increasing the susceptibility of tumor cells to immune cell-mediated cytotoxicity.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "greek letter needed---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"greek letter needed\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Therefore, although IFN-?? alone are not able to induce cell death in tumor cells in vitro, Noxa induction by IFN-??, which is regulated by the balance between its activators (IRF1, 3, and 7) and its repressor (IRF4), is crucial to increasing the susceptibility of tumor cells to immune cell-mediated cytotoxicity.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 331 (char 330)\n",
      "{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Taken together\",\"our data suggest that FRA2 is a novel STAT5 target gene\",\"regulated by IL-2 in activated CD4 T cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|IL2 not target---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"IL2 not target\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, our data suggest that FRA2 is a novel STAT5 target gene, regulated by IL-2 in activated CD4 T cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Taken together\",\"our data suggest that FRA2 is a novel STAT5 target gene\",\"regulated by IL-2 in activated CD4 T cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|IL2 not target---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"IL2 not target\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, our data suggest that FRA2 is a novel STAT5 target gene, regulated by IL-2 in activated CD4 T cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"T\"],\"tg associated gene name\":[\"CSF2\"],\"tf text\":[],\"tg text\":[\"granulocyte-macrophage colony-stimulating factor\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Low concentrations of dexamethasone (10(-10) M) repress IL-1beta-stimulated granulocyte-macrophage colony-stimulating factor (GM-CSF) expression and fail to stimulate secretory leukocyte proteinase inhibitor expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|T again---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"T\"],\"tg (associated gene name)\":[\"CSF2\"],\"tf text\":[],\"tg text\":[\"granulocyte-macrophage colony-stimulating factor\"],\"consensus\":[\"N\"],\"martin_comments\":[\"T again\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Low concentrations of dexamethasone (10(-10) M) repress IL-1beta-stimulated granulocyte-macrophage colony-stimulating factor (GM-CSF) expression and fail to stimulate secretory leukocyte proteinase inhibitor expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"T\"],\"tg associated gene name\":[\"CSF2\"],\"tf text\":[],\"tg text\":[\"granulocyte-macrophage colony-stimulating factor\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Low concentrations of dexamethasone (10(-10) M) repress IL-1beta-stimulated granulocyte-macrophage colony-stimulating factor (GM-CSF) expression and fail to stimulate secretory leukocyte proteinase inhibitor expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|T again---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"T\"],\"tg (associated gene name)\":[\"CSF2\"],\"tf text\":[],\"tg text\":[\"granulocyte-macrophage colony-stimulating factor\"],\"consensus\":[\"N\"],\"martin_comments\":[\"T again\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Low concentrations of dexamethasone (10(-10) M) repress IL-1beta-stimulated granulocyte-macrophage colony-stimulating factor (GM-CSF) expression and fail to stimulate secretory leukocyte proteinase inhibitor expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"SPDEF\"],\"tg associated gene name\":[\"RNU6-1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Collectively\",\"the results demonstrate that the PSE binding protein stably associates with a high affinity to its cognate promoter sequence\",\"and this process represents one of the primary events in the formation of the preinitiation complex on the U6 gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"SPDEF\"],\"tg associated gene name\":[\"RNU6-1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Collectively\",\"the results demonstrate that the PSE binding protein stably associates with a high affinity to its cognate promoter sequence\",\"and this process represents one of the primary events in the formation of the preinitiation complex on the U6 gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SPDEF\"],\"target gene (associated gene name)\":[\"RNU6-1\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Collectively\",\"the results demonstrate that the PSE binding protein stably associates with a high affinity to its cognate promoter sequence\",\"and this process represents one of the primary events in the formation of the preinitiation complex on the U6 gene.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|PSE binding?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPDEF\"],\"tg (associated gene name)\":[\"RNU6-1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"PSE binding?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Collectively, the results demonstrate that the PSE binding protein stably associates with a high affinity to its cognate promoter sequence, and this process represents one of the primary events in the formation of the preinitiation complex on the U6 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is MMP-13---{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is MMP-13\"],\"sentence\":[\"The binding of c-Fos and c-Jun to the AP-1 element on the MMP-13 promoter and the enhancement of AP-1 luciferase activity was enhanced by US stimulation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The binding of c-Fos and c-Jun to the AP-1 element on the MMP-13 promoter and the enhancement of AP-1 luciferase activity was enhanced by US stimulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is MMP-13---{\"tf associated gene name\":[\"FOS\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is MMP-13\"],\"sentence\":[\"The binding of c-Fos and c-Jun to the AP-1 element on the MMP-13 promoter and the enhancement of AP-1 luciferase activity was enhanced by US stimulation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOS\"],\"tg (associated gene name)\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The binding of c-Fos and c-Jun to the AP-1 element on the MMP-13 promoter and the enhancement of AP-1 luciferase activity was enhanced by US stimulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is HDL receptor SR-BI---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"HSD11B1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is HDL receptor SR-BI\"],\"sentence\":[\"Consistent with the possibility that SREBP-1a may be involved in the regulation of the HDL receptor SR-BI gene\",\"we found that mature SREBP-1a levels increased up to 11-fold in the ovary after treatment with 50 U hCG.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is HDL receptor SR-BI---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"HSD11B1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is HDL receptor SR-BI\"],\"sentence\":[\"Consistent with the possibility that SREBP-1a may be involved in the regulation of the HDL receptor SR-BI gene\",\"we found that mature SREBP-1a levels increased up to 11-fold in the ovary after treatment with 50 U hCG.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SREBF1\"],\"target gene (associated gene name)\":[\"HSD11B1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Consistent with the possibility that SREBP-1a may be involved in the regulation of the HDL receptor SR-BI gene\",\"we found that mature SREBP-1a levels increased up to 11-fold in the ovary after treatment with 50 U hCG.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"0.517445\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"4\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"216\"],\"sentence pair density\":[\"0.02803738317757\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"HSD11B1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consistent with the possibility that SREBP-1a may be involved in the regulation of the HDL receptor SR-BI gene, we found that mature SREBP-1a levels increased up to 11-fold in the ovary after treatment with 50 U hCG.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Differences in DNA-binding efficiency of Sp1 to aldolase and pyruvate kinase promoter correlate with altered redox states in resting and proliferating rat thymocytes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"TRIB3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Differences in DNA-binding efficiency of Sp1 to aldolase and pyruvate kinase promoter correlate with altered redox states in resting and proliferating rat thymocytes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 412 (char 411)\n",
      "{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"GZMB\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Down-regulation of human granzyme B expression by glucocorticoids. Dexamethasone inhibits binding to the Ikaros and AP-1 regulatory elements of the granzyme B promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|2 sentences---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"GZMB\"],\"tf text\":[],\"tg text\":[\"granzyme B\"],\"consensus\":[\"V\"],\"martin_comments\":[\"2 sentences\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Down-regulation of human granzyme B expression by glucocorticoids. Dexamethasone inhibits binding to the Ikaros and AP-1 regulatory elements of the granzyme B promoter.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"GZMB\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"v\"],\"negation\":[],\"sentence\":[\"Down-regulation of human granzyme B expression by glucocorticoids. Dexamethasone inhibits binding to the Ikaros and AP-1 regulatory elements of the granzyme B promoter.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.730738\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"168\"],\"sentence pair density\":[\"0.0121212121212121\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"GZMB\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Down-regulation of human granzyme B expression by glucocorticoids. Dexamethasone inhibits binding to the Ikaros and AP-1 regulatory elements of the granzyme B promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|2 sentences---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"GZMB\"],\"tf text\":[],\"tg text\":[\"granzyme B\"],\"consensus\":[\"V\"],\"martin_comments\":[\"2 sentences\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Down-regulation of human granzyme B expression by glucocorticoids. Dexamethasone inhibits binding to the Ikaros and AP-1 regulatory elements of the granzyme B promoter.\"],\"set\":[\"Extended\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 482 (char 481)\n",
      "{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"HSD11B2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of p300 enhanced 11-HSD2 expression\",\"which was attenuated by Sp1 knockdown\",\"whereas p300 knockdown and C646 reduced both basal and cAMP-stimulated acetylation of H3K9 and H3K27 associated with 11-HSD2 expression.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|chain?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"HSD11B2\"],\"tf text\":[],\"tg text\":[\"HSD2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"chain?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of p300 enhanced 11-HSD2 expression, which was attenuated by Sp1 knockdown, whereas p300 knockdown and C646 reduced both basal and cAMP-stimulated acetylation of H3K9 and H3K27 associated with 11-HSD2 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"HSD11B2\"],\"tf text\":[],\"tg text\":[\"HSD2\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Overexpression of p300 enhanced 11??-HSD2 expression\",\"which was attenuated by Sp1 knockdown\",\"whereas p300 knockdown and C646 reduced both basal and cAMP-stimulated acetylation of H3K9 and H3K27 associated with 11??-HSD2 expression.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"HSD11B2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of p300 enhanced 11-HSD2 expression\",\"which was attenuated by Sp1 knockdown\",\"whereas p300 knockdown and C646 reduced both basal and cAMP-stimulated acetylation of H3K9 and H3K27 associated with 11-HSD2 expression.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|chain?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"HSD11B2\"],\"tf text\":[],\"tg text\":[\"HSD2\"],\"consensus\":[\"V\"],\"martin_comments\":[\"chain?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of p300 enhanced 11-HSD2 expression, which was attenuated by Sp1 knockdown, whereas p300 knockdown and C646 reduced both basal and cAMP-stimulated acetylation of H3K9 and H3K27 associated with 11-HSD2 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 794 (char 793)\n",
      "{\"tf associated gene name\":[\"ID2\"],\"tg associated gene name\":[\"BMP7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Q-HSC activation in vitro (culture) and in vivo (CCl(4)-induced cirrhosis) resulted in decreased expression of Hh-interacting protein (Hhip\",\"an Hh antagonist)\",\"the EMT inhibitors bone morphogenic protein (BMP-7) and inhibitor of differentiation (Id2)\",\"the adherens junction component E-cadherin\",\"and epithelial keratins 7 and 19 and increased expression of Gli2 (an Hh target gene) and mesenchymal markers\",\"including the mesenchyme-associated transcription factors Lhx2 and Msx2\",\"the myofibroblast marker alpha-smooth muscle actin\",\"and matrix molecules such as collagen.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no causality---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID2\"],\"tg (associated gene name)\":[\"BMP7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no causality\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Q-HSC activation in vitro (culture) and in vivo (CCl(4)-induced cirrhosis) resulted in decreased expression of Hh-interacting protein (Hhip, an Hh antagonist), the EMT inhibitors bone morphogenic protein (BMP-7) and inhibitor of differentiation (Id2), the adherens junction component E-cadherin, and epithelial keratins 7 and 19 and increased expression of Gli2 (an Hh target gene) and mesenchymal markers, including the mesenchyme-associated transcription factors Lhx2 and Msx2, the myofibroblast marker alpha-smooth muscle actin, and matrix molecules such as collagen.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ID2\"],\"tg associated gene name\":[\"BMP7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Q-HSC activation in vitro (culture) and in vivo (CCl(4)-induced cirrhosis) resulted in decreased expression of Hh-interacting protein (Hhip\",\"an Hh antagonist)\",\"the EMT inhibitors bone morphogenic protein (BMP-7) and inhibitor of differentiation (Id2)\",\"the adherens junction component E-cadherin\",\"and epithelial keratins 7 and 19 and increased expression of Gli2 (an Hh target gene) and mesenchymal markers\",\"including the mesenchyme-associated transcription factors Lhx2 and Msx2\",\"the myofibroblast marker alpha-smooth muscle actin\",\"and matrix molecules such as collagen.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no causality---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ID2\"],\"tg (associated gene name)\":[\"BMP7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no causality\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Q-HSC activation in vitro (culture) and in vivo (CCl(4)-induced cirrhosis) resulted in decreased expression of Hh-interacting protein (Hhip, an Hh antagonist), the EMT inhibitors bone morphogenic protein (BMP-7) and inhibitor of differentiation (Id2), the adherens junction component E-cadherin, and epithelial keratins 7 and 19 and increased expression of Gli2 (an Hh target gene) and mesenchymal markers, including the mesenchyme-associated transcription factors Lhx2 and Msx2, the myofibroblast marker alpha-smooth muscle actin, and matrix molecules such as collagen.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 392 (char 391)\n",
      "{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"SOD2\"],\"tf text\":[],\"tg text\":[\"MnSOD\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"When transiently expressing\",\"MnSOD inhibited AP-1 but increased NF-kappaB transactivation\",\"which can be abolished by sodium pyruvate\",\"a hydrogen peroxide scavenger.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"SOD2\"],\"tf text\":[],\"tg text\":[\"MnSOD\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"When transiently expressing, MnSOD inhibited AP-1 but increased NF-kappaB transactivation, which can be abolished by sodium pyruvate, a hydrogen peroxide scavenger.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"SOD2\"],\"tf text\":[],\"tg text\":[\"MnSOD\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"When transiently expressing\",\"MnSOD inhibited AP-1 but increased NF-kappaB transactivation\",\"which can be abolished by sodium pyruvate\",\"a hydrogen peroxide scavenger.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"SOD2\"],\"tf text\":[],\"tg text\":[\"MnSOD\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"When transiently expressing, MnSOD inhibited AP-1 but increased NF-kappaB transactivation, which can be abolished by sodium pyruvate, a hydrogen peroxide scavenger.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 414 (char 413)\n",
      "{\"transcription factor (associated gene name)\":[\"PGR\"],\"target gene (associated gene name)\":[\"TH\"],\"valid\":[\"1\"],\"sign\":[\"+_-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"PR-A was able to inhibit PR-B transactivation in a dose-dependent manner\",\"although the degree of PR-A inhibition was dependent on the TH promoter deletion construct.\"],\"Valid\":[true],\"Sign\":[\"+_-\"],\"Negation\":[null],\"QC\":[true]}---|unclear, PR-A other gene?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"unclear, PR-A other gene?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PR-A was able to inhibit PR-B transactivation in a dose-dependent manner, although the degree of PR-A inhibition was dependent on the TH promoter deletion construct.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"PR-A was able to inhibit PR-B transactivation in a dose-dependent manner\",\"although the degree of PR-A inhibition was dependent on the TH promoter deletion construct.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor\":[\"PGR\"],\"target gene\":[\"TH\"],\"sentence\":[\"PR-A was able to inhibit PR-B transactivation in a dose-dependent manner\",\"although the degree of PR-A inhibition was dependent on the TH promoter deletion construct.\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"fusion tf\":[],\"qc'ed\":[\"0\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"PGR\"],\"target gene (associated gene name)\":[\"TH\"],\"valid\":[\"1\"],\"sign\":[\"+_-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"PR-A was able to inhibit PR-B transactivation in a dose-dependent manner\",\"although the degree of PR-A inhibition was dependent on the TH promoter deletion construct.\"],\"Valid\":[true],\"Sign\":[\"+_-\"],\"Negation\":[null],\"QC\":[true]}---|unclear, PR-A other gene?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PGR\"],\"tg (associated gene name)\":[\"TH\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"unclear, PR-A other gene?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PR-A was able to inhibit PR-B transactivation in a dose-dependent manner, although the degree of PR-A inhibition was dependent on the TH promoter deletion construct.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG, gamma problem?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"SUPT7L\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, gamma problem?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Granulocyte colony-stimulating factor induces the binding of STAT1 and STAT3 to the IFNgamma response region within the promoter of the Fc(gamma)RI/CD64 gene in human neutrophils.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG, gamma problem?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"SUPT7L\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, gamma problem?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Granulocyte colony-stimulating factor induces the binding of STAT1 and STAT3 to the IFNgamma response region within the promoter of the Fc(gamma)RI/CD64 gene in human neutrophils.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "Myc not TF, flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"AMY2A\"],\"tf text\":[\"c-myc\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"Myc not TF, flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Suppression of c-myc by PA is followed by increased mRNA levels of p21, an effect that is mediated by PA activation of the p21 promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "Myc not TF, flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"AMY2A\"],\"tf text\":[\"c-myc\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"Myc not TF, flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Suppression of c-myc by PA is followed by increased mRNA levels of p21, an effect that is mediated by PA activation of the p21 promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFATC3\"],\"tg (associated gene name)\":[\"CXCL13\"],\"tf text\":[\"NFATc3\"],\"tg text\":[\"CXCL13\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"siRNA suppression of c-Myc expression markedly decreased CXCL13-induced RANKL and NFATc3 expression in preosteoblast cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFATC3\"],\"tg (associated gene name)\":[\"CXCL13\"],\"tf text\":[\"NFATc3\"],\"tg text\":[\"CXCL13\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"siRNA suppression of c-Myc expression markedly decreased CXCL13-induced RANKL and NFATc3 expression in preosteoblast cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 374 (char 373)\n",
      "{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[\"Estrogen receptor\"],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Estrogen receptor inhibits c-Jun-dependent stress-induced cell death by binding and modifying c-Jun activity in human breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[\"Estrogen receptor\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Estrogen receptor inhibits c-Jun-dependent stress-induced cell death by binding and modifying c-Jun activity in human breast cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[\"Estrogen receptor\"],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Estrogen receptor inhibits c-Jun-dependent stress-induced cell death by binding and modifying c-Jun activity in human breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein level?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[\"Estrogen receptor\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Estrogen receptor inhibits c-Jun-dependent stress-induced cell death by binding and modifying c-Jun activity in human breast cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is AgRP---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"APP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is AgRP\"],\"sentence\":[\"Here\",\"we show that in response to fasting-elevated Gc levels\",\"Gc receptor (GR) and Bsx synergize to direct activation of AgRP transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG norm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"APP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG norm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here, we show that in response to fasting-elevated Gc levels, Gc receptor (GR) and Bsx synergize to direct activation of AgRP transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is AgRP---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"APP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is AgRP\"],\"sentence\":[\"Here\",\"we show that in response to fasting-elevated Gc levels\",\"Gc receptor (GR) and Bsx synergize to direct activation of AgRP transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG norm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"APP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG norm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here, we show that in response to fasting-elevated Gc levels, Gc receptor (GR) and Bsx synergize to direct activation of AgRP transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF and TG within family|but likely not exact---{\"tf associated gene name\":[\"RFX1\"],\"tg associated gene name\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u-fam\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TF and TG within family\",\"but likely not exact\"],\"sentence\":[\"Its interaction with RFX5 and RFXAP is essential for binding of the RFX complex to MHC-II promoters.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|RFX1 at best from backref---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RFX1\"],\"tg (associated gene name)\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"RFX1 at best from backref\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Its interaction with RFX5 and RFXAP is essential for binding of the RFX complex to MHC-II promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF and TG within family|but likely not exact---{\"tf associated gene name\":[\"RFX1\"],\"tg associated gene name\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u-fam\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TF and TG within family\",\"but likely not exact\"],\"sentence\":[\"Its interaction with RFX5 and RFXAP is essential for binding of the RFX complex to MHC-II promoters.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|RFX1 at best from backref---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RFX1\"],\"tg (associated gene name)\":[\"HLA-E\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"RFX1 at best from backref\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Its interaction with RFX5 and RFXAP is essential for binding of the RFX complex to MHC-II promoters.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is C/EBPbeta---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"TRIM35\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF is C/EBPbeta\"],\"sentence\":[\"BTG3 associated nuclear protein (SMAR1\",\"Banp) and tripartite motif-containing 35 (Hls5\",\"trim35) were two target genes among the 110 candidate genes which are involved in cell cycle regulation; the binding of C/EBPbeta to the promoter of banp and trim35 was verified by ChIP-PCR.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"TRIM35\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"BTG3 associated nuclear protein (SMAR1, Banp) and tripartite motif-containing 35 (Hls5, trim35) were two target genes among the 110 candidate genes which are involved in cell cycle regulation; the binding of C/EBPbeta to the promoter of banp and trim35 was verified by ChIP-PCR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is C/EBPbeta---{\"tf associated gene name\":[\"FOXC1\"],\"tg associated gene name\":[\"TRIM35\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF is C/EBPbeta\"],\"sentence\":[\"BTG3 associated nuclear protein (SMAR1\",\"Banp) and tripartite motif-containing 35 (Hls5\",\"trim35) were two target genes among the 110 candidate genes which are involved in cell cycle regulation; the binding of C/EBPbeta to the promoter of banp and trim35 was verified by ChIP-PCR.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXC1\"],\"tg (associated gene name)\":[\"TRIM35\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"BTG3 associated nuclear protein (SMAR1, Banp) and tripartite motif-containing 35 (Hls5, trim35) were two target genes among the 110 candidate genes which are involved in cell cycle regulation; the binding of C/EBPbeta to the promoter of banp and trim35 was verified by ChIP-PCR.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 644 (char 643)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"IKBKG\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"IKKalpha\",\"IKKbeta\",\"and NEMO/IKKgamma are each required for the NF-kappa B-mediated inflammatory response program.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.368046\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"113\"],\"sentence pair density\":[\"0.0387096774193548\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IKBKG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"IKKalpha, IKKbeta, and NEMO/IKKgamma are each required for the NF-kappa B-mediated inflammatory response program.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"IKBKG\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"IKBKG\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"IKKalpha\",\"IKKbeta\",\"and NEMO/IKKgamma are each required for the NF-kappa B-mediated inflammatory response program.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.368046\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"113\"],\"sentence pair density\":[\"0.0387096774193548\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IKBKG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"IKKalpha, IKKbeta, and NEMO/IKKgamma are each required for the NF-kappa B-mediated inflammatory response program.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is pS2---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"IRS1\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[\"IRS-1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is pS2\"],\"sentence\":[\"Here we report that in estrogen receptor alpha (ERalpha)-positive MCF-7 cells\",\"(1) a fraction of IRS-1 was translocated to the nucleus upon 17-beta-estradiol (E2) treatment; (2) E2-dependent nuclear translocation of IRS-1 was blocked with the antiestrogen ICI 182\",\"780; (3) nuclear IRS-1 colocalized and co-precipitated with ERalpha; (4) the IRS-1:ERalpha complex was recruited to the E2-sensitive pS2 gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|IRS- = TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"IRS1\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[\"IRS-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"IRS- = TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we report that in estrogen receptor alpha (ERalpha)-positive MCF-7 cells, (1) a fraction of IRS-1 was translocated to the nucleus upon 17-beta-estradiol (E2) treatment; (2) E2-dependent nuclear translocation of IRS-1 was blocked with the antiestrogen ICI 182,780; (3) nuclear IRS-1 colocalized and co-precipitated with ERalpha; (4) the IRS-1:ERalpha complex was recruited to the E2-sensitive pS2 gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is pS2---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"IRS1\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[\"IRS-1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is pS2\"],\"sentence\":[\"Here we report that in estrogen receptor alpha (ERalpha)-positive MCF-7 cells\",\"(1) a fraction of IRS-1 was translocated to the nucleus upon 17-beta-estradiol (E2) treatment; (2) E2-dependent nuclear translocation of IRS-1 was blocked with the antiestrogen ICI 182\",\"780; (3) nuclear IRS-1 colocalized and co-precipitated with ERalpha; (4) the IRS-1:ERalpha complex was recruited to the E2-sensitive pS2 gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|IRS- = TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"IRS1\"],\"tf text\":[\"estrogen receptor alpha\"],\"tg text\":[\"IRS-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"IRS- = TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we report that in estrogen receptor alpha (ERalpha)-positive MCF-7 cells, (1) a fraction of IRS-1 was translocated to the nucleus upon 17-beta-estradiol (E2) treatment; (2) E2-dependent nuclear translocation of IRS-1 was blocked with the antiestrogen ICI 182,780; (3) nuclear IRS-1 colocalized and co-precipitated with ERalpha; (4) the IRS-1:ERalpha complex was recruited to the E2-sensitive pS2 gene promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is GATA-6---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[\"SULT2A1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is GATA-6\"],\"sentence\":[\"Transient transfection assays with SULT2A1 deletion and mutation promoter constructs allowed the determination of specific the GATA-6 binding cis-regulatory elements necessary for transactivation of SULT2A1 transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF norm ---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[\"SULT2A1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF norm \"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transient transfection assays with SULT2A1 deletion and mutation promoter constructs allowed the determination of specific the GATA-6 binding cis-regulatory elements necessary for transactivation of SULT2A1 transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is GATA-6---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[\"SULT2A1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is GATA-6\"],\"sentence\":[\"Transient transfection assays with SULT2A1 deletion and mutation promoter constructs allowed the determination of specific the GATA-6 binding cis-regulatory elements necessary for transactivation of SULT2A1 transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF norm ---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"SULT2A1\"],\"tf text\":[],\"tg text\":[\"SULT2A1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF norm \"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transient transfection assays with SULT2A1 deletion and mutation promoter constructs allowed the determination of specific the GATA-6 binding cis-regulatory elements necessary for transactivation of SULT2A1 transcription.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is HO-1---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"SMARCA2\"],\"tf text\":[\"Nrf2\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is HO-1\"],\"sentence\":[\"Furthermore\",\"the deletion of Neh5 markedly repressed CBP [CREB (cAMP-response-element-binding protein)-binding protein] and BRG1 (Brahma-related gene 1) from associating with Nrf2\",\"diminishing their co-operative enhancement of HO-1 promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"SMARCA2\"],\"tf text\":[\"Nrf2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, the deletion of Neh5 markedly repressed CBP [CREB (cAMP-response-element-binding protein)-binding protein] and BRG1 (Brahma-related gene 1) from associating with Nrf2, diminishing their co-operative enhancement of HO-1 promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is HO-1---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"SMARCA2\"],\"tf text\":[\"Nrf2\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is HO-1\"],\"sentence\":[\"Furthermore\",\"the deletion of Neh5 markedly repressed CBP [CREB (cAMP-response-element-binding protein)-binding protein] and BRG1 (Brahma-related gene 1) from associating with Nrf2\",\"diminishing their co-operative enhancement of HO-1 promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFE2L2\"],\"tg (associated gene name)\":[\"SMARCA2\"],\"tf text\":[\"Nrf2\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Furthermore, the deletion of Neh5 markedly repressed CBP [CREB (cAMP-response-element-binding protein)-binding protein] and BRG1 (Brahma-related gene 1) from associating with Nrf2, diminishing their co-operative enhancement of HO-1 promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 433 (char 432)\n",
      "{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"MAPKAPK2\"],\"tf text\":[\"E47\"],\"tg text\":[\"MAPK-activated protein kinase 2\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Serine/Threonine kinases 3pK and MAPK-activated protein kinase 2 interact with the basic helix-loop-helix transcription factor E47 and repress its transcriptional activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"MAPKAPK2\"],\"tf text\":[\"E47\"],\"tg text\":[\"MAPK-activated protein kinase 2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Serine/Threonine kinases 3pK and MAPK-activated protein kinase 2 interact with the basic helix-loop-helix transcription factor E47 and repress its transcriptional activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"MAPKAPK2\"],\"tf text\":[\"E47\"],\"tg text\":[\"MAPK-activated protein kinase 2\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Serine/Threonine kinases 3pK and MAPK-activated protein kinase 2 interact with the basic helix-loop-helix transcription factor E47 and repress its transcriptional activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"MAPKAPK2\"],\"tf text\":[\"E47\"],\"tg text\":[\"MAPK-activated protein kinase 2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Serine/Threonine kinases 3pK and MAPK-activated protein kinase 2 interact with the basic helix-loop-helix transcription factor E47 and repress its transcriptional activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is alpha-MHC ---{\"tf associated gene name\":[\"USF1\"],\"tg associated gene name\":[\"MMP12\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is alpha-MHC \"],\"sentence\":[\"An E-box hemodynamic response element (HME) located at position -47 within the alpha-MHC promoter is both necessary and sufficient to confer contractile responsiveness to the gene and has been shown to bind upstream stimulatory factor-1 (USF1).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"MMP12\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"An E-box hemodynamic response element (HME) located at position -47 within the alpha-MHC promoter is both necessary and sufficient to confer contractile responsiveness to the gene and has been shown to bind upstream stimulatory factor-1 (USF1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is alpha-MHC ---{\"tf associated gene name\":[\"USF1\"],\"tg associated gene name\":[\"MMP12\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is alpha-MHC \"],\"sentence\":[\"An E-box hemodynamic response element (HME) located at position -47 within the alpha-MHC promoter is both necessary and sufficient to confer contractile responsiveness to the gene and has been shown to bind upstream stimulatory factor-1 (USF1).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"USF1\"],\"tg (associated gene name)\":[\"MMP12\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"An E-box hemodynamic response element (HME) located at position -47 within the alpha-MHC promoter is both necessary and sufficient to confer contractile responsiveness to the gene and has been shown to bind upstream stimulatory factor-1 (USF1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Gfi1---{\"tf associated gene name\":[\"GATA2\"],\"tg associated gene name\":[\"MYSM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is Gfi1\"],\"sentence\":[\"We found that Mysm1 directly associates with the Gfi1 enhancer element and promotes its transcription through Gata2 and Runx1 transactivation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA2\"],\"tg (associated gene name)\":[\"MYSM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that Mysm1 directly associates with the Gfi1 enhancer element and promotes its transcription through Gata2 and Runx1 transactivation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Gfi1---{\"tf associated gene name\":[\"GATA2\"],\"tg associated gene name\":[\"MYSM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is Gfi1\"],\"sentence\":[\"We found that Mysm1 directly associates with the Gfi1 enhancer element and promotes its transcription through Gata2 and Runx1 transactivation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA2\"],\"tg (associated gene name)\":[\"MYSM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We found that Mysm1 directly associates with the Gfi1 enhancer element and promotes its transcription through Gata2 and Runx1 transactivation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 553 (char 552)\n",
      "{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"PML\"],\"tf text\":[\"Oct4\"],\"tg text\":[\"Pml\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"This study demonstrates a functional role for Pml in maintaining a specific open chromatin conformation of the Oct4 promoter region for its constant expression in stem cells; and illustrates the mechanism underlying RA-induced chromatin remodeling of Oct4 gene in differentiating cells\",\"in which Pml plays a critical role.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|OCT TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"PML\"],\"tf text\":[\"Oct-04\"],\"tg text\":[\"Pml\"],\"consensus\":[\"N\"],\"martin_comments\":[\"OCT TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This study demonstrates a functional role for Pml in maintaining a specific open chromatin conformation of the Oct4 promoter region for its constant expression in stem cells; and illustrates the mechanism underlying RA-induced chromatin remodeling of Oct4 gene in differentiating cells, in which Pml plays a critical role.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"POU5F1\"],\"tg associated gene name\":[\"PML\"],\"tf text\":[\"Oct4\"],\"tg text\":[\"Pml\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"This study demonstrates a functional role for Pml in maintaining a specific open chromatin conformation of the Oct4 promoter region for its constant expression in stem cells; and illustrates the mechanism underlying RA-induced chromatin remodeling of Oct4 gene in differentiating cells\",\"in which Pml plays a critical role.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|OCT TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU5F1\"],\"tg (associated gene name)\":[\"PML\"],\"tf text\":[\"Oct-04\"],\"tg text\":[\"Pml\"],\"consensus\":[\"N\"],\"martin_comments\":[\"OCT TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This study demonstrates a functional role for Pml in maintaining a specific open chromatin conformation of the Oct4 promoter region for its constant expression in stem cells; and illustrates the mechanism underlying RA-induced chromatin remodeling of Oct4 gene in differentiating cells, in which Pml plays a critical role.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 452 (char 451)\n",
      "{\"tf associated gene name\":[\"POU1F1\"],\"tg associated gene name\":[\"AHSG\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"These studies\",\"therefore\",\"show that SMF increases BSP transcription through a tyrosine kinase-dependent pathway and that the SMF effects are mediated through juxtaposed FRE and Pit-1elements in the proximal promoter of the BSP gene.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|AHSG comes up with BSP, but BP not ion that Genecards page!---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU1F1\"],\"tg (associated gene name)\":[\"AHSG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AHSG comes up with BSP, but BP not ion that Genecards page!\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These studies, therefore, show that SMF increases BSP transcription through a tyrosine kinase-dependent pathway and that the SMF effects are mediated through juxtaposed FRE and Pit-1elements in the proximal promoter of the BSP gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"POU1F1\"],\"tg associated gene name\":[\"AHSG\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"These studies\",\"therefore\",\"show that SMF increases BSP transcription through a tyrosine kinase-dependent pathway and that the SMF effects are mediated through juxtaposed FRE and Pit-1elements in the proximal promoter of the BSP gene.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|AHSG comes up with BSP, but BP not ion that Genecards page!---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU1F1\"],\"tg (associated gene name)\":[\"AHSG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"AHSG comes up with BSP, but BP not ion that Genecards page!\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These studies, therefore, show that SMF increases BSP transcription through a tyrosine kinase-dependent pathway and that the SMF effects are mediated through juxtaposed FRE and Pit-1elements in the proximal promoter of the BSP gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain TG normal---{\"tf associated gene name\":[\"RBPJ\"],\"tg associated gene name\":[\"FCER1G\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain TG normal\"],\"sentence\":[\"Binding of CBF1 to a 192-bp mapped EBNA-2-responsive region located at position -85 bp to -277 bp upstream of the CD23 promoter was detected in electrophoretic mobility shift assays.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|TG=FCER2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RBPJ\"],\"tg (associated gene name)\":[\"FCER1G\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=FCER2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Binding of CBF1 to a 192-bp mapped EBNA-2-responsive region located at position -85 bp to -277 bp upstream of the CD23 promoter was detected in electrophoretic mobility shift assays.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "uncertain TG normal---{\"tf associated gene name\":[\"RBPJ\"],\"tg associated gene name\":[\"FCER1G\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain TG normal\"],\"sentence\":[\"Binding of CBF1 to a 192-bp mapped EBNA-2-responsive region located at position -85 bp to -277 bp upstream of the CD23 promoter was detected in electrophoretic mobility shift assays.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|TG=FCER2?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RBPJ\"],\"tg (associated gene name)\":[\"FCER1G\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=FCER2?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Binding of CBF1 to a 192-bp mapped EBNA-2-responsive region located at position -85 bp to -277 bp upstream of the CD23 promoter was detected in electrophoretic mobility shift assays.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ATF2\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"ATF-2\"],\"tg text\":[\"CBP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Unexpectedly, we detected a nearly equal distribution of activator (Tax, CREB, ATF-1, ATF-2, c-Fos, and c-Jun) and regulatory protein (CBP, p300, TAF(II)250, and polymerase II) binding at both the upstream and downstream promoters.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ATF2\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"ATF-2\"],\"tg text\":[\"CBP\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Unexpectedly, we detected a nearly equal distribution of activator (Tax, CREB, ATF-1, ATF-2, c-Fos, and c-Jun) and regulatory protein (CBP, p300, TAF(II)250, and polymerase II) binding at both the upstream and downstream promoters.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NANOG\"],\"tg (associated gene name)\":[\"RBBP4\"],\"tf text\":[\"NANOG\"],\"tg text\":[\"RBBP4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Affymetrix and real-time PCR studies of siRNA-treated human PS cells showed that reduced RBBP4 or RBBP9 expression concomitantly decreased expression of POU5F1, NANOG, SOX2, and/or FOXD3 plus certain cell cycle genes (e.g., CCNA2, CCNB1), while increasing expression of genes involved in organogenesis (particularly neurogenesis).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NANOG\"],\"tg (associated gene name)\":[\"RBBP4\"],\"tf text\":[\"NANOG\"],\"tg text\":[\"RBBP4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Affymetrix and real-time PCR studies of siRNA-treated human PS cells showed that reduced RBBP4 or RBBP9 expression concomitantly decreased expression of POU5F1, NANOG, SOX2, and/or FOXD3 plus certain cell cycle genes (e.g., CCNA2, CCNB1), while increasing expression of genes involved in organogenesis (particularly neurogenesis).\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 359 (char 358)\n",
      "{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"MMP2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Tetraspanin CD9 induces MMP-2 expression by activating p38 MAPK\",\"JNK and c-Jun pathways in human melanoma cells.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|causality, pathways enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"MMP2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"causality, pathways enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Tetraspanin CD9 induces MMP-2 expression by activating p38 MAPK, JNK and c-Jun pathways in human melanoma cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"MMP2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"Tetraspanin CD9 induces MMP-2 expression by activating p38 MAPK\",\"JNK and c-Jun pathways in human melanoma cells.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"MMP2\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Tetraspanin CD9 induces MMP-2 expression by activating p38 MAPK\",\"JNK and c-Jun pathways in human melanoma cells.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---|causality, pathways enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"MMP2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"causality, pathways enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Tetraspanin CD9 induces MMP-2 expression by activating p38 MAPK, JNK and c-Jun pathways in human melanoma cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF217\"],\"tg (associated gene name)\":[\"ITGAM\"],\"tf text\":[\"ZNF217\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Significantly, Ad5E1A, CtBP1, and ZNF217 form a stable complex which requires CR3 and the PLDLS motif.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZNF217\"],\"tg (associated gene name)\":[\"ITGAM\"],\"tf text\":[\"ZNF217\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Significantly, Ad5E1A, CtBP1, and ZNF217 form a stable complex which requires CR3 and the PLDLS motif.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 467 (char 466)\n",
      "{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Correlating with the highest levels of C/EBPzeta expression at 48 h after LPS treatment\",\"there is an increased association of C/EBPzeta with C/EBPbeta\",\"and both the abundance of C/EBP DNA-binding species and IL-6 expression are downregulated.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|LPS problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"LPS problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Correlating with the highest levels of C/EBPzeta expression at 48 h after LPS treatment, there is an increased association of C/EBPzeta with C/EBPbeta, and both the abundance of C/EBP DNA-binding species and IL-6 expression are downregulated.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Correlating with the highest levels of C/EBPzeta expression at 48 h after LPS treatment\",\"there is an increased association of C/EBPzeta with C/EBPbeta\",\"and both the abundance of C/EBP DNA-binding species and IL-6 expression are downregulated.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|LPS problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IL6\"],\"tf text\":[],\"tg text\":[\"IL-6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"LPS problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Correlating with the highest levels of C/EBPzeta expression at 48 h after LPS treatment, there is an increased association of C/EBPzeta with C/EBPbeta, and both the abundance of C/EBP DNA-binding species and IL-6 expression are downregulated.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 416 (char 415)\n",
      "{\"tf associated gene name\":[\"RARB\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Direct inhibition of ERK activity similarly decreased DNMT1 protein levels and reversed the basal hypermethylation of the promoters and silencing of the RARB2\",\"p21 and p16 tumor suppressor genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RARB\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Direct inhibition of ERK activity similarly decreased DNMT1 protein levels and reversed the basal hypermethylation of the promoters and silencing of the RARB2, p21 and p16 tumor suppressor genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RARB\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Direct inhibition of ERK activity similarly decreased DNMT1 protein levels and reversed the basal hypermethylation of the promoters and silencing of the RARB2\",\"p21 and p16 tumor suppressor genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RARB\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Direct inhibition of ERK activity similarly decreased DNMT1 protein levels and reversed the basal hypermethylation of the promoters and silencing of the RARB2, p21 and p16 tumor suppressor genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"BCL11A\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Transcriptional regulation of c-jun gene expression by arabinofuranosylcytosine in human myeloid leukemia cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BCL11A\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional regulation of c-jun gene expression by arabinofuranosylcytosine in human myeloid leukemia cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"BCL11A\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Transcriptional regulation of c-jun gene expression by arabinofuranosylcytosine in human myeloid leukemia cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"BCL11A\"],\"tg (associated gene name)\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional regulation of c-jun gene expression by arabinofuranosylcytosine in human myeloid leukemia cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not correctly identified---{\"tf associated gene name\":[\"NR0B1\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[\"interleukin-2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not correctly identified\"],\"sentence\":[\"Transcriptional repression of the interleukin-2 gene by vitamin D3: direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR0B1\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[\"interleukin-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional repression of the interleukin-2 gene by vitamin D3: direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF not correctly identified---{\"tf associated gene name\":[\"NR0B1\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[\"interleukin-2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not correctly identified\"],\"sentence\":[\"Transcriptional repression of the interleukin-2 gene by vitamin D3: direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR0B1\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[\"interleukin-2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transcriptional repression of the interleukin-2 gene by vitamin D3: direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is p53---{\"tf associated gene name\":[\"PAX1\"],\"tg associated gene name\":[\"FAS\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is p53\"],\"sentence\":[\"wt p53 bound to and transactivated the CD95 gene\",\"whereas mt p53 failed to induce apoptosis via activation of the CD95 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is p53---{\"tf associated gene name\":[\"PAX1\"],\"tg associated gene name\":[\"FAS\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is p53\"],\"sentence\":[\"wt p53 bound to and transactivated the CD95 gene\",\"whereas mt p53 failed to induce apoptosis via activation of the CD95 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PAX1\"],\"tg (associated gene name)\":[\"FAS\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"wt p53 bound to and transactivated the CD95 gene, whereas mt p53 failed to induce apoptosis via activation of the CD95 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 786 (char 785)\n",
      "{\"tf associated gene name\":[\"CEBPZ\"],\"tg associated gene name\":[\"TRIM63\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"The models predicted TFs whose differential expression during RPE EMT was successfully verified by reverse transcriptase polymerase chain reaction (RT-PCR) analysis\",\"including Oct-1\",\"hepatocyte nuclear factor 1 (HNF-1)\",\"similar to mothers against decapentaplegic 3 (SMAD3)\",\"transcription factor E (TFE)\",\"core binding factor\",\"erythroid transcription factor-1 (GATA-1)\",\"interferon regulatory factor-1 (IRF)\",\"natural killer homeobox 3A (NKX3A)\",\"Sterol regulatory element binding protein-1 (SREBP-1)\",\"and lymphocyte enhancer factor-1 (LEF-1).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPZ\"],\"tg (associated gene name)\":[\"TRIM63\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The models predicted TFs whose differential expression during RPE EMT was successfully verified by reverse transcriptase polymerase chain reaction (RT-PCR) analysis, including Oct-1, hepatocyte nuclear factor 1 (HNF-1), similar to mothers against decapentaplegic 3 (SMAD3), transcription factor E (TFE), core binding factor, erythroid transcription factor-1 (GATA-1), interferon regulatory factor-1 (IRF), natural killer homeobox 3A (NKX3A), Sterol regulatory element binding protein-1 (SREBP-1), and lymphocyte enhancer factor-1 (LEF-1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"CEBPZ\"],\"tg associated gene name\":[\"TRIM63\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"The models predicted TFs whose differential expression during RPE EMT was successfully verified by reverse transcriptase polymerase chain reaction (RT-PCR) analysis\",\"including Oct-1\",\"hepatocyte nuclear factor 1 (HNF-1)\",\"similar to mothers against decapentaplegic 3 (SMAD3)\",\"transcription factor E (TFE)\",\"core binding factor\",\"erythroid transcription factor-1 (GATA-1)\",\"interferon regulatory factor-1 (IRF)\",\"natural killer homeobox 3A (NKX3A)\",\"Sterol regulatory element binding protein-1 (SREBP-1)\",\"and lymphocyte enhancer factor-1 (LEF-1).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPZ\"],\"tg (associated gene name)\":[\"TRIM63\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The models predicted TFs whose differential expression during RPE EMT was successfully verified by reverse transcriptase polymerase chain reaction (RT-PCR) analysis, including Oct-1, hepatocyte nuclear factor 1 (HNF-1), similar to mothers against decapentaplegic 3 (SMAD3), transcription factor E (TFE), core binding factor, erythroid transcription factor-1 (GATA-1), interferon regulatory factor-1 (IRF), natural killer homeobox 3A (NKX3A), Sterol regulatory element binding protein-1 (SREBP-1), and lymphocyte enhancer factor-1 (LEF-1).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"RUNX3\"],\"tg associated gene name\":[\"RUNX3\"],\"tf text\":[\"RUNX3\"],\"tg text\":[\"RUNX3\"],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Between 45% and 60% of human gastric cancer cells do not significantly express RUNX3 due to hemizygous deletion and hypermethylation of the RUNX3 promoter region.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RUNX3\"],\"tg (associated gene name)\":[\"RUNX3\"],\"tf text\":[\"RUNX3\"],\"tg text\":[\"RUNX3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Between 45% and 60% of human gastric cancer cells do not significantly express RUNX3 due to hemizygous deletion and hypermethylation of the RUNX3 promoter region.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"RUNX3\"],\"tg associated gene name\":[\"RUNX3\"],\"tf text\":[\"RUNX3\"],\"tg text\":[\"RUNX3\"],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Between 45% and 60% of human gastric cancer cells do not significantly express RUNX3 due to hemizygous deletion and hypermethylation of the RUNX3 promoter region.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RUNX3\"],\"tg (associated gene name)\":[\"RUNX3\"],\"tf text\":[\"RUNX3\"],\"tg text\":[\"RUNX3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Between 45% and 60% of human gastric cancer cells do not significantly express RUNX3 due to hemizygous deletion and hypermethylation of the RUNX3 promoter region.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not clear that SNAI1 is transcriptionally regulated by ESRP1---{\"tf associated gene name\":[\"SNAI1\"],\"tg associated gene name\":[\"ESRP1\"],\"tf text\":[\"Snail\"],\"tg text\":[\"ESRP1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"not clear that SNAI1 is transcriptionally regulated by ESRP1\"],\"sentence\":[\"We further show that ectopically expressing ESRP1 inhibits Snail-induced EMT\",\"suggesting that down-regulation of ESRP1 is required for function by Snail in EMT.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SNAI1\"],\"tg (associated gene name)\":[\"ESRP1\"],\"tf text\":[\"Snail\"],\"tg text\":[\"ESRP1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We further show that ectopically expressing ESRP1 inhibits Snail-induced EMT, suggesting that down-regulation of ESRP1 is required for function by Snail in EMT.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not clear that SNAI1 is transcriptionally regulated by ESRP1---{\"tf associated gene name\":[\"SNAI1\"],\"tg associated gene name\":[\"ESRP1\"],\"tf text\":[\"Snail\"],\"tg text\":[\"ESRP1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"not clear that SNAI1 is transcriptionally regulated by ESRP1\"],\"sentence\":[\"We further show that ectopically expressing ESRP1 inhibits Snail-induced EMT\",\"suggesting that down-regulation of ESRP1 is required for function by Snail in EMT.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SNAI1\"],\"tg (associated gene name)\":[\"ESRP1\"],\"tf text\":[\"Snail\"],\"tg text\":[\"ESRP1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We further show that ectopically expressing ESRP1 inhibits Snail-induced EMT, suggesting that down-regulation of ESRP1 is required for function by Snail in EMT.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is socs1---{\"tf associated gene name\":[\"SP2\"],\"tg associated gene name\":[\"IFNG\"],\"tf text\":[\"Sp2\"],\"tg text\":[\"interferon-gamma\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is socs1\"],\"sentence\":[\"Sp2 regulates interferon-gamma-mediated socs1 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong causality---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP2\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[\"Sp2\"],\"tg text\":[\"interferon-gamma\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong causality\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sp2 regulates interferon-gamma-mediated socs1 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is socs1---{\"tf associated gene name\":[\"SP2\"],\"tg associated gene name\":[\"IFNG\"],\"tf text\":[\"Sp2\"],\"tg text\":[\"interferon-gamma\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is socs1\"],\"sentence\":[\"Sp2 regulates interferon-gamma-mediated socs1 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong causality---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP2\"],\"tg (associated gene name)\":[\"IFNG\"],\"tf text\":[\"Sp2\"],\"tg text\":[\"interferon-gamma\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong causality\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Sp2 regulates interferon-gamma-mediated socs1 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU2F2\"],\"tg (associated gene name)\":[\"SLC22A2\"],\"tf text\":[],\"tg text\":[\"Oct-02\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In the OCT2 promoter region, one deletion polymorphism (-578_-576delAAG) was identified; -578_-576delAAG significantly reduced OCT2 promoter activity (p < 0.05), and carriers of -578_-576delAAG tend to have lower OCT2 mRNA levels, but the difference is not significant.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"POU2F2\"],\"tg (associated gene name)\":[\"SLC22A2\"],\"tf text\":[],\"tg text\":[\"Oct-02\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"In the OCT2 promoter region, one deletion polymorphism (-578_-576delAAG) was identified; -578_-576delAAG significantly reduced OCT2 promoter activity (p < 0.05), and carriers of -578_-576delAAG tend to have lower OCT2 mRNA levels, but the difference is not significant.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is c-Maf|TG is IL-10---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IL4\"],\"tf text\":[],\"tg text\":[\"IL-4\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TF is c-Maf\",\"TG is IL-10\"],\"sentence\":[\"IL-4 can up-regulate c-Maf expression\",\"its binding to IL-10 promoter\",\"and dose dependently enhance IL-10 production induced by LPS; moreover\",\"IL-4 failed to enhance LPS-induced IL-10 production in c-Maf-null macrophages.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|LPS problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IL4\"],\"tf text\":[],\"tg text\":[\"IL-4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"LPS problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"IL-4 can up-regulate c-Maf expression, its binding to IL-10 promoter, and dose dependently enhance IL-10 production induced by LPS; moreover, IL-4 failed to enhance LPS-induced IL-10 production in c-Maf-null macrophages.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is c-Maf|TG is IL-10---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"IL4\"],\"tf text\":[],\"tg text\":[\"IL-4\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TF is c-Maf\",\"TG is IL-10\"],\"sentence\":[\"IL-4 can up-regulate c-Maf expression\",\"its binding to IL-10 promoter\",\"and dose dependently enhance IL-10 production induced by LPS; moreover\",\"IL-4 failed to enhance LPS-induced IL-10 production in c-Maf-null macrophages.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|LPS problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF6\"],\"tg (associated gene name)\":[\"IL4\"],\"tf text\":[],\"tg text\":[\"IL-4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"LPS problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"IL-4 can up-regulate c-Maf expression, its binding to IL-10 promoter, and dose dependently enhance IL-10 production induced by LPS; moreover, IL-4 failed to enhance LPS-induced IL-10 production in c-Maf-null macrophages.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ER-trouble---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[\"WAF1\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"ER-trouble\"],\"sentence\":[\"We concluded that activation of c-fosER mediated transcriptional inhibition of p21(Cip1/WAF1) through a previously uncharacterized AP-1 site\",\"revealing an important role for c-fos in negative control of cell cycle regulatory genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|thrown off by ER---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[\"WAF1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"thrown off by ER\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We concluded that activation of c-fosER mediated transcriptional inhibition of p21(Cip1/WAF1) through a previously uncharacterized AP-1 site, revealing an important role for c-fos in negative control of cell cycle regulatory genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "ER-trouble---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[\"WAF1\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"ER-trouble\"],\"sentence\":[\"We concluded that activation of c-fosER mediated transcriptional inhibition of p21(Cip1/WAF1) through a previously uncharacterized AP-1 site\",\"revealing an important role for c-fos in negative control of cell cycle regulatory genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|thrown off by ER---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[\"WAF1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"thrown off by ER\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We concluded that activation of c-fosER mediated transcriptional inhibition of p21(Cip1/WAF1) through a previously uncharacterized AP-1 site, revealing an important role for c-fos in negative control of cell cycle regulatory genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 395 (char 394)\n",
      "{\"transcription factor (associated gene name)\":[\"RELA\"],\"target gene (associated gene name)\":[\"PPARG\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Conversely\",\"overexpression of p65 inhibited induction of a PPAR-gamma-responsive reporter gene by activated PPAR-gamma in a dose-dependent manner.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"PPARG\"],\"tf text\":[\"p65\"],\"tg text\":[\"PPAR-gamma\"],\"consensus\":[\"V\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Conversely, overexpression of p65 inhibited induction of a PPAR-gamma-responsive reporter gene by activated PPAR-gamma in a dose-dependent manner.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"PPARG\"],\"tf text\":[\"p65\"],\"tg text\":[\"PPAR-gamma\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Conversely\",\"overexpression of p65 inhibited induction of a PPAR-gamma-responsive reporter gene by activated PPAR-gamma in a dose-dependent manner.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RELA\"],\"target gene (associated gene name)\":[\"PPARG\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"r\"],\"negation\":[],\"sentence\":[\"Conversely\",\"overexpression of p65 inhibited induction of a PPAR-gamma-responsive reporter gene by activated PPAR-gamma in a dose-dependent manner.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.44778\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"146\"],\"sentence pair density\":[\"0.00826446280991735\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RELA\"],\"target gene (associated gene name)\":[\"PPARG\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Conversely\",\"overexpression of p65 inhibited induction of a PPAR-gamma-responsive reporter gene by activated PPAR-gamma in a dose-dependent manner.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|protein level---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"PPARG\"],\"tf text\":[\"p65\"],\"tg text\":[\"PPAR-gamma\"],\"consensus\":[\"V\"],\"martin_comments\":[\"protein level\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Conversely, overexpression of p65 inhibited induction of a PPAR-gamma-responsive reporter gene by activated PPAR-gamma in a dose-dependent manner.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is MYCN---{\"tf associated gene name\":[\"LMO2\"],\"tg associated gene name\":[\"MPO\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF is MYCN\"],\"sentence\":[\"MYCN enhanced primitive hematopoiesis by upregulating scl and lmo2 expression and promoted myelopoiesis by inhibiting gata1 expression and inducing pu.1\",\"mpo expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LMO2\"],\"tg (associated gene name)\":[\"MPO\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MYCN enhanced primitive hematopoiesis by upregulating scl and lmo2 expression and promoted myelopoiesis by inhibiting gata1 expression and inducing pu.1, mpo expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is MYCN---{\"tf associated gene name\":[\"LMO2\"],\"tg associated gene name\":[\"MPO\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF is MYCN\"],\"sentence\":[\"MYCN enhanced primitive hematopoiesis by upregulating scl and lmo2 expression and promoted myelopoiesis by inhibiting gata1 expression and inducing pu.1\",\"mpo expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LMO2\"],\"tg (associated gene name)\":[\"MPO\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"MYCN enhanced primitive hematopoiesis by upregulating scl and lmo2 expression and promoted myelopoiesis by inhibiting gata1 expression and inducing pu.1, mpo expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"MYB\"],\"tg associated gene name\":[\"CD4\"],\"tf text\":[\"c-myb\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Moreover\",\"c-myb silencing significantly decreased both methylation of histone H3K4 and acetylation of histone H3K9 at the IL-13 locus in CD4(+) effector/memory cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYB\"],\"tg (associated gene name)\":[\"CD4\"],\"tf text\":[\"c-myb\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, c-myb silencing significantly decreased both methylation of histone H3K4 and acetylation of histone H3K9 at the IL-13 locus in CD4(+) effector/memory cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"MYB\"],\"tg associated gene name\":[\"CD4\"],\"tf text\":[\"c-myb\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Moreover\",\"c-myb silencing significantly decreased both methylation of histone H3K4 and acetylation of histone H3K9 at the IL-13 locus in CD4(+) effector/memory cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYB\"],\"tg (associated gene name)\":[\"CD4\"],\"tf text\":[\"c-myb\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Moreover, c-myb silencing significantly decreased both methylation of histone H3K4 and acetylation of histone H3K9 at the IL-13 locus in CD4(+) effector/memory cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 492 (char 491)\n",
      "{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"HIF1A\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"HIF-1alpha\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"To explore direct HIF-1 targets\",\"here we used differential gel electrophoresis (DIGE) to compare the HIF-1-regulated proteins between leukemic U937T-cell line with and without conditional induction of HIF-1alpha protein by tetracycline-off system.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"HIF1A\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"HIF-1alpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To explore direct HIF-1 targets, here we used differential gel electrophoresis (DIGE) to compare the HIF-1-regulated proteins between leukemic U937T-cell line with and without conditional induction of HIF-1alpha protein by tetracycline-off system.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"HIF1A\"],\"tg associated gene name\":[\"HIF1A\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"HIF-1alpha\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[],\"sentence\":[\"To explore direct HIF-1 targets\",\"here we used differential gel electrophoresis (DIGE) to compare the HIF-1-regulated proteins between leukemic U937T-cell line with and without conditional induction of HIF-1alpha protein by tetracycline-off system.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HIF1A\"],\"tg (associated gene name)\":[\"HIF1A\"],\"tf text\":[\"HIF-1alpha\"],\"tg text\":[\"HIF-1alpha\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To explore direct HIF-1 targets, here we used differential gel electrophoresis (DIGE) to compare the HIF-1-regulated proteins between leukemic U937T-cell line with and without conditional induction of HIF-1alpha protein by tetracycline-off system.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 914 (char 913)\n",
      "{\"transcription factor (associated gene name)\":[\"STAT1\"],\"target gene (associated gene name)\":[\"CCND1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Importantly\",\"JUN may regulate activating transcription factor 3 expression to involve cell proliferation process; STAT1 and STAT3 can inhibit tissue inhibitor of metalloproteinases-3 expression to modulate the cell adhesion molecule pathway; NF-B and E2F1 can downregulate cyclin D1\",\"but upregulate proliferating cell nuclear antigen expression to promote the cell cycle pathway.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.010686\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"18\"],\"sentence length\":[\"379\"],\"sentence pair density\":[\"0.00578034682080924\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|disjoint TF and TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[\"STAT1\"],\"tg text\":[\"cyclin D1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"disjoint TF and TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Importantly, JUN may regulate activating transcription factor 3 expression to involve cell proliferation process; STAT1 and STAT3 can inhibit tissue inhibitor of metalloproteinases-3 expression to modulate the cell adhesion molecule pathway; NF-??B and E2F1 can downregulate cyclin D1, but upregulate proliferating cell nuclear antigen expression to promote the cell cycle pathway.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"STAT1\"],\"target gene (associated gene name)\":[\"CCND1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Importantly\",\"JUN may regulate activating transcription factor 3 expression to involve cell proliferation process; STAT1 and STAT3 can inhibit tissue inhibitor of metalloproteinases-3 expression to modulate the cell adhesion molecule pathway; NF-B and E2F1 can downregulate cyclin D1\",\"but upregulate proliferating cell nuclear antigen expression to promote the cell cycle pathway.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.010686\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"18\"],\"sentence length\":[\"379\"],\"sentence pair density\":[\"0.00578034682080924\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|disjoint TF and TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"CCND1\"],\"tf text\":[\"STAT1\"],\"tg text\":[\"cyclin D1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"disjoint TF and TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Importantly, JUN may regulate activating transcription factor 3 expression to involve cell proliferation process; STAT1 and STAT3 can inhibit tissue inhibitor of metalloproteinases-3 expression to modulate the cell adhesion molecule pathway; NF-??B and E2F1 can downregulate cyclin D1, but upregulate proliferating cell nuclear antigen expression to promote the cell cycle pathway.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 708 (char 707)\n",
      "{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"CASP4\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"However\",\"knockdown of PARP-1 did not suppress the expression of caspase-11 induced by interferon- that activates signal transducer and activator of transcription 1 but not NF-B.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.066202\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"177\"],\"sentence pair density\":[\"0.0150375939849624\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CASP4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, knockdown of PARP-1 did not suppress the expression of caspase-11 induced by interferon- that activates signal transducer and activator of transcription 1 but not NF-B.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"CASP4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"However\",\"knockdown of PARP-1 did not suppress the expression of caspase-11 induced by interferon-?? that activates signal transducer and activator of transcription 1 but not NF-??B.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"CASP4\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"However\",\"knockdown of PARP-1 did not suppress the expression of caspase-11 induced by interferon- that activates signal transducer and activator of transcription 1 but not NF-B.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.066202\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"177\"],\"sentence pair density\":[\"0.0150375939849624\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"CASP4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"However, knockdown of PARP-1 did not suppress the expression of caspase-11 induced by interferon- that activates signal transducer and activator of transcription 1 but not NF-B.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is hhex|TFs are FoxH1/Smad2 binding sites---{\"tf associated gene name\":[\"HHEX\"],\"tg associated gene name\":[\"AXIN1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is hhex\",\"TFs are FoxH1/Smad2 binding sites\"],\"sentence\":[\"We show that Nodal signaling\",\"present throughout the endoderm\",\"directly activates hhex transcription via FoxH1/Smad2 binding sites in the proximal -0.44 Kb promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HHEX\"],\"tg (associated gene name)\":[\"AXIN1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that Nodal signaling, present throughout the endoderm, directly activates hhex transcription via FoxH1/Smad2 binding sites in the proximal -0.44 Kb promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is hhex|TFs are FoxH1/Smad2 binding sites---{\"tf associated gene name\":[\"HHEX\"],\"tg associated gene name\":[\"AXIN1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is hhex\",\"TFs are FoxH1/Smad2 binding sites\"],\"sentence\":[\"We show that Nodal signaling\",\"present throughout the endoderm\",\"directly activates hhex transcription via FoxH1/Smad2 binding sites in the proximal -0.44 Kb promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HHEX\"],\"tg (associated gene name)\":[\"AXIN1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that Nodal signaling, present throughout the endoderm, directly activates hhex transcription via FoxH1/Smad2 binding sites in the proximal -0.44 Kb promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unsure about TG identification/normalization---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"unsure about TG identification/normalization\"],\"sentence\":[\"GC-II binds Sp1\",\"Sp3\",\"and IKLF transcription factors\",\"collaborates with the AP1/CREB binding element\",\"and confers the EGF response.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"GC-II binds Sp1, Sp3, and IKLF transcription factors, collaborates with the AP1/CREB binding element, and confers the EGF response.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "unsure about TG identification/normalization---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"unsure about TG identification/normalization\"],\"sentence\":[\"GC-II binds Sp1\",\"Sp3\",\"and IKLF transcription factors\",\"collaborates with the AP1/CREB binding element\",\"and confers the EGF response.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"GC-II binds Sp1, Sp3, and IKLF transcription factors, collaborates with the AP1/CREB binding element, and confers the EGF response.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is LRH-1---{\"tf associated gene name\":[\"ESRRA\"],\"tg associated gene name\":[\"POU5F1\"],\"tf text\":[],\"tg text\":[\"Oct4\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF is LRH-1\"],\"sentence\":[\"Here we report that the orphan nuclear receptor LRH-1\",\"which is expressed in undifferentiated ES cells\",\"can bind to SF-1 response elements in the proximal promoter and proximal enhancer of the Oct4 gene and activate Oct4 reporter gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is LRH-1---{\"tf associated gene name\":[\"ESRRA\"],\"tg associated gene name\":[\"POU5F1\"],\"tf text\":[],\"tg text\":[\"Oct4\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF is LRH-1\"],\"sentence\":[\"Here we report that the orphan nuclear receptor LRH-1\",\"which is expressed in undifferentiated ES cells\",\"can bind to SF-1 response elements in the proximal promoter and proximal enhancer of the Oct4 gene and activate Oct4 reporter gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESRRA\"],\"target gene (associated gene name)\":[\"POU5F1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"ESRRA\"],\"target gene (associated gene name)\":[\"POU5F1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Here we report that the orphan nuclear receptor LRH-1\",\"which is expressed in undifferentiated ES cells\",\"can bind to SF-1 response elements in the proximal promoter and proximal enhancer of the Oct4 gene and activate Oct4 reporter gene expression.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.862097\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"5\"],\"sentence length\":[\"246\"],\"sentence pair density\":[\"0.009009009009009\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF norm problemthrown off by nuclear receptor---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESRRA\"],\"tg (associated gene name)\":[\"POU5F1\"],\"tf text\":[],\"tg text\":[\"Oct-04\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TF norm problemthrown off by nuclear receptor\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we report that the orphan nuclear receptor LRH-1, which is expressed in undifferentiated ES cells, can bind to SF-1 response elements in the proximal promoter and proximal enhancer of the Oct4 gene and activate Oct4 reporter gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG named---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[\"HNF4\"],\"tg text\":[\"CREB binding protein\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Overexpression of the CREB binding protein (CBP) increased the HNF4-dependent transactivation but this effect remained sensitive to cAMP inhibition.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF4A\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"HNF4\"],\"tg text\":[\"CREB binding protein\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of the CREB binding protein (CBP) increased the HNF4-dependent transactivation but this effect remained sensitive to cAMP inhibition.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG named---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[\"HNF4\"],\"tg text\":[\"CREB binding protein\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"Overexpression of the CREB binding protein (CBP) increased the HNF4-dependent transactivation but this effect remained sensitive to cAMP inhibition.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HNF4A\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[\"HNF4\"],\"tg text\":[\"CREB binding protein\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Overexpression of the CREB binding protein (CBP) increased the HNF4-dependent transactivation but this effect remained sensitive to cAMP inhibition.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBP\"],\"tg (associated gene name)\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[\"JNK1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Decreasing JNK1 expression reduces the occupancy of TBP at the Bdp1 promoter, while decreasing JNK2 expression enhances recruitment of TBP to the Bdp1 promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBP\"],\"tg (associated gene name)\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[\"JNK1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Decreasing JNK1 expression reduces the occupancy of TBP at the Bdp1 promoter, while decreasing JNK2 expression enhances recruitment of TBP to the Bdp1 promoter.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Cx32---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Cx32\"],\"sentence\":[\"The transcription factors HNF-1 and Sp1 have important functional roles in the transcriptional regulation of basal and cell-specific Cx32 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|norm problem cel---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"norm problem cel\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcription factors HNF-1 and Sp1 have important functional roles in the transcriptional regulation of basal and cell-specific Cx32 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Cx32---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Cx32\"],\"sentence\":[\"The transcription factors HNF-1 and Sp1 have important functional roles in the transcriptional regulation of basal and cell-specific Cx32 expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|norm problem cel---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SP1\"],\"tg (associated gene name)\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"norm problem cel\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The transcription factors HNF-1 and Sp1 have important functional roles in the transcriptional regulation of basal and cell-specific Cx32 expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"ZEB1\"],\"tg associated gene name\":[\"ARC\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Slug may be one of the key transcription factors involved in the HIF1alpha-induced EMT of LNCaP cells\",\"while ZEB1 and Slug may play an important role in repressing the epithelial phenotype of the ARCaP model.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|transcr. Unsp[ecified---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZEB1\"],\"tg (associated gene name)\":[\"ARC\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"transcr. Unsp[ecified\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Slug may be one of the key transcription factors involved in the HIF1alpha-induced EMT of LNCaP cells, while ZEB1 and Slug may play an important role in repressing the epithelial phenotype of the ARCaP model.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"ZEB1\"],\"tg associated gene name\":[\"ARC\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Slug may be one of the key transcription factors involved in the HIF1alpha-induced EMT of LNCaP cells\",\"while ZEB1 and Slug may play an important role in repressing the epithelial phenotype of the ARCaP model.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|transcr. Unsp[ecified---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZEB1\"],\"tg (associated gene name)\":[\"ARC\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"transcr. Unsp[ecified\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Slug may be one of the key transcription factors involved in the HIF1alpha-induced EMT of LNCaP cells, while ZEB1 and Slug may play an important role in repressing the epithelial phenotype of the ARCaP model.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear whether S/MAR 1 can/should be normalized to TF (IRF1 not correct normalization)---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"PARP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"unclear whether S/MAR 1 can/should be normalized to TF (IRF1 not correct normalization)\"],\"sentence\":[\"We conclude that S/MAR 1 represents an upstream control element that acts in conjunction with the muPARP-1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "unclear whether S/MAR 1 can/should be normalized to TF (IRF1 not correct normalization)---{\"tf associated gene name\":[\"IRF1\"],\"tg associated gene name\":[\"PARP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"unclear whether S/MAR 1 can/should be normalized to TF (IRF1 not correct normalization)\"],\"sentence\":[\"We conclude that S/MAR 1 represents an upstream control element that acts in conjunction with the muPARP-1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF1\"],\"target gene (associated gene name)\":[\"PARP1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that S/MAR 1 represents an upstream control element that acts in conjunction with the muPARP-1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|MAR enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"IRF1\"],\"tg (associated gene name)\":[\"PARP1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"MAR enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We conclude that S/MAR 1 represents an upstream control element that acts in conjunction with the muPARP-1 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 391 (char 390)\n",
      "{\"transcription factor (associated gene name)\":[\"SREBF1\"],\"target gene (associated gene name)\":[\"FDFT1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"This is due in part to a selective ability of low levels of nSREBP-2\",\"but not nSREBP-1a\",\"to activate the promoter for squalene synthase.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[true]}---|NOT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"FDFT1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This is due in part to a selective ability of low levels of nSREBP-2, but not nSREBP-1a, to activate the promoter for squalene synthase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"SREBF1\"],\"tg associated gene name\":[\"FDFT1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"This is due in part to a selective ability of low levels of nSREBP-2\",\"but not nSREBP-1a\",\"to activate the promoter for squalene synthase.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SREBF1\"],\"target gene (associated gene name)\":[\"FDFT1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[\"1\"],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"This is due in part to a selective ability of low levels of nSREBP-2\",\"but not nSREBP-1a\",\"to activate the promoter for squalene synthase.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[true],\"QC\":[true]}---|NOT---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SREBF1\"],\"tg (associated gene name)\":[\"FDFT1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"NOT\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This is due in part to a selective ability of low levels of nSREBP-2, but not nSREBP-1a, to activate the promoter for squalene synthase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is EGR1---{\"tf associated gene name\":[\"HAND1\"],\"tg associated gene name\":[\"IL4\"],\"tf text\":[],\"tg text\":[\"IL-4\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is EGR1\"],\"sentence\":[\"Our study provides the first evidence that Egr-1 protein is differentially expressed in Th1 and Th2 cells and is involved in the acute phase of the IL-4 transcription in response to TCR stimulation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HAND1 often a problem?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HAND1\"],\"tg (associated gene name)\":[\"IL4\"],\"tf text\":[],\"tg text\":[\"IL-4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"HAND1 often a problem?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our study provides the first evidence that Egr-1 protein is differentially expressed in Th1 and Th2 cells and is involved in the acute phase of the IL-4 transcription in response to TCR stimulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TF is EGR1---{\"tf associated gene name\":[\"HAND1\"],\"tg associated gene name\":[\"IL4\"],\"tf text\":[],\"tg text\":[\"IL-4\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is EGR1\"],\"sentence\":[\"Our study provides the first evidence that Egr-1 protein is differentially expressed in Th1 and Th2 cells and is involved in the acute phase of the IL-4 transcription in response to TCR stimulation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|HAND1 often a problem?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HAND1\"],\"tg (associated gene name)\":[\"IL4\"],\"tf text\":[],\"tg text\":[\"IL-4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"HAND1 often a problem?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Our study provides the first evidence that Egr-1 protein is differentially expressed in Th1 and Th2 cells and is involved in the acute phase of the IL-4 transcription in response to TCR stimulation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is elastin|ELN---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is elastin\",\"ELN\"],\"sentence\":[\"C/EBPbeta isoforms bound a GCAAT-containing sequence in the proximal elastin promoter as determined by electrophoretic gel shift studies and confirmed by using specific anti-C/EBPbeta antibodies and by competition studies with oligonucleotides.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"C/EBPbeta isoforms bound a GCAAT-containing sequence in the proximal elastin promoter as determined by electrophoretic gel shift studies and confirmed by using specific anti-C/EBPbeta antibodies and by competition studies with oligonucleotides.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is elastin|ELN---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is elastin\",\"ELN\"],\"sentence\":[\"C/EBPbeta isoforms bound a GCAAT-containing sequence in the proximal elastin promoter as determined by electrophoretic gel shift studies and confirmed by using specific anti-C/EBPbeta antibodies and by competition studies with oligonucleotides.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CEBPB\"],\"tg (associated gene name)\":[\"SLU7\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"C/EBPbeta isoforms bound a GCAAT-containing sequence in the proximal elastin promoter as determined by electrophoretic gel shift studies and confirmed by using specific anti-C/EBPbeta antibodies and by competition studies with oligonucleotides.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "Regulator is miR-23a---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"CDH1\"],\"tf text\":[],\"tg text\":[\"E-cadherin\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"Regulator is miR-23a\"],\"sentence\":[\"miR-23a inhibits E-cadherin expression and is regulated by AP-1 and NFAT4 complex during Fas-induced EMT in gastrointestinal cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "Regulator is miR-23a---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"CDH1\"],\"tf text\":[],\"tg text\":[\"E-cadherin\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"Regulator is miR-23a\"],\"sentence\":[\"miR-23a inhibits E-cadherin expression and is regulated by AP-1 and NFAT4 complex during Fas-induced EMT in gastrointestinal cancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"CDH1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"miR-23a inhibits E-cadherin expression and is regulated by AP-1 and NFAT4 complex during Fas-induced EMT in gastrointestinal cancer.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"3.21095\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"132\"],\"sentence pair density\":[\"0.00602409638554216\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|causal 3 step---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"CDH1\"],\"tf text\":[],\"tg text\":[\"E-cadherin\"],\"consensus\":[\"V\"],\"martin_comments\":[\"causal 3 step\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"miR-23a inhibits E-cadherin expression and is regulated by AP-1 and NFAT4 complex during Fas-induced EMT in gastrointestinal cancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 392 (char 391)\n",
      "{\"tf associated gene name\":[\"NR0B2\"],\"tg associated gene name\":[\"SLC10A2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"In DCA-fed rabbits\",\"ASBT expression was repressed\",\"associated with activated FXR\",\"and evidenced by increased ileal short heterodimer partner (SHP) mRNA.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|role SHP as TF unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR0B2\"],\"tg (associated gene name)\":[\"SLC10A2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"role SHP as TF unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In DCA-fed rabbits, ASBT expression was repressed, associated with activated FXR, and evidenced by increased ileal short heterodimer partner (SHP) mRNA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR0B2\"],\"tg associated gene name\":[\"SLC10A2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"In DCA-fed rabbits\",\"ASBT expression was repressed\",\"associated with activated FXR\",\"and evidenced by increased ileal short heterodimer partner (SHP) mRNA.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|role SHP as TF unclear---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR0B2\"],\"tg (associated gene name)\":[\"SLC10A2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"role SHP as TF unclear\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In DCA-fed rabbits, ASBT expression was repressed, associated with activated FXR, and evidenced by increased ileal short heterodimer partner (SHP) mRNA.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is c-jun|TF not named---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"EVPL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is c-jun\",\"TF not named\"],\"sentence\":[\"Regulation of the c-jun gene in p210 BCR-ABL transformed cells corresponds with activity of JNK\",\"the c-jun N-terminal kinase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=JUN|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"EVPL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=JUN\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[\"Regulation of the c-jun gene in p210 BCR-ABL transformed cells corresponds with activity of JNK, the c-jun N-terminal kinase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is c-jun|TF not named---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"EVPL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is c-jun\",\"TF not named\"],\"sentence\":[\"Regulation of the c-jun gene in p210 BCR-ABL transformed cells corresponds with activity of JNK\",\"the c-jun N-terminal kinase.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=JUN|CO-EXPR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"JUN\"],\"tg (associated gene name)\":[\"EVPL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=JUN\"],\"astrid_comments\":[\"CO-EXPR\"],\"sentences with greek letters\":[\"Regulation of the c-jun gene in p210 BCR-ABL transformed cells corresponds with activity of JNK, the c-jun N-terminal kinase.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is BCL-2---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"CREB1\"],\"tf text\":[\"CREB\"],\"tg text\":[\"CREB\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is BCL-2\"],\"sentence\":[\"TOX3 also interacts with native CREB and induces the CREB-responsive BCL-2 promoter\",\"which can be inhibited by coexpression of CITED1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"CREB1\"],\"tf text\":[\"CREB\"],\"tg text\":[\"CREB\"],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TOX3 also interacts with native CREB and induces the CREB-responsive BCL-2 promoter, which can be inhibited by coexpression of CITED1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is BCL-2---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"CREB1\"],\"tf text\":[\"CREB\"],\"tg text\":[\"CREB\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is BCL-2\"],\"sentence\":[\"TOX3 also interacts with native CREB and induces the CREB-responsive BCL-2 promoter\",\"which can be inhibited by coexpression of CITED1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"CREB1\"],\"tg (associated gene name)\":[\"CREB1\"],\"tf text\":[\"CREB\"],\"tg text\":[\"CREB\"],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"TOX3 also interacts with native CREB and induces the CREB-responsive BCL-2 promoter, which can be inhibited by coexpression of CITED1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "upstream IGF-mediated regulation of ESR1 activity---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\",\"up\"],\"negation\":[],\"comment\":[\"upstream IGF-mediated regulation of ESR1 activity\"],\"sentence\":[\"This study confirmed that IGF-I can cause transcriptional activation of endogenous ER in human breast cancel cells\",\"and inhibition of ER action by IGFBP-1 suggests that IGF-1 signaling may be necessary for maximal ER activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=ESR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=ESR\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This study confirmed that IGF-I can cause transcriptional activation of endogenous ER in human breast cancel cells, and inhibition of ER action by IGFBP-1 suggests that IGF-1 signaling may be necessary for maximal ER activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "upstream IGF-mediated regulation of ESR1 activity---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\",\"up\"],\"negation\":[],\"comment\":[\"upstream IGF-mediated regulation of ESR1 activity\"],\"sentence\":[\"This study confirmed that IGF-I can cause transcriptional activation of endogenous ER in human breast cancel cells\",\"and inhibition of ER action by IGFBP-1 suggests that IGF-1 signaling may be necessary for maximal ER activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG=ESR---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG=ESR\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"This study confirmed that IGF-I can cause transcriptional activation of endogenous ER in human breast cancel cells, and inhibition of ER action by IGFBP-1 suggests that IGF-1 signaling may be necessary for maximal ER activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"With sequential modification assays\",\"we found that sumoylation of p53 at K386 blocks subsequent acetylation by p300\",\"whereas p300-acetylated p53 remains permissive for ensuing sumoylation at K386 and alleviates sumoylation-inhibited DNA binding.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"With sequential modification assays, we found that sumoylation of p53 at K386 blocks subsequent acetylation by p300, whereas p300-acetylated p53 remains permissive for ensuing sumoylation at K386 and alleviates sumoylation-inhibited DNA binding.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"With sequential modification assays\",\"we found that sumoylation of p53 at K386 blocks subsequent acetylation by p300\",\"whereas p300-acetylated p53 remains permissive for ensuing sumoylation at K386 and alleviates sumoylation-inhibited DNA binding.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TP53\"],\"tg (associated gene name)\":[\"NOTCH1\"],\"tf text\":[\"p53\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"With sequential modification assays, we found that sumoylation of p53 at K386 blocks subsequent acetylation by p300, whereas p300-acetylated p53 remains permissive for ensuing sumoylation at K386 and alleviates sumoylation-inhibited DNA binding.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is IL-2---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is IL-2\"],\"sentence\":[\"These results identify a unique role for B7 costimulatory molecules and CD28 in the activation of JNK during Ag stimulation in Th1 cells\",\"and suggest that JNK regulates Elk-1 transactivation at the c-fos gene to promote the formation of AP-1 complexes important to IL-2 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results identify a unique role for B7 costimulatory molecules and CD28 in the activation of JNK during Ag stimulation in Th1 cells, and suggest that JNK regulates Elk-1 transactivation at the c-fos gene to promote the formation of AP-1 complexes important to IL-2 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is IL-2---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is IL-2\"],\"sentence\":[\"These results identify a unique role for B7 costimulatory molecules and CD28 in the activation of JNK during Ag stimulation in Th1 cells\",\"and suggest that JNK regulates Elk-1 transactivation at the c-fos gene to promote the formation of AP-1 complexes important to IL-2 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results identify a unique role for B7 costimulatory molecules and CD28 in the activation of JNK during Ag stimulation in Th1 cells, and suggest that JNK regulates Elk-1 transactivation at the c-fos gene to promote the formation of AP-1 complexes important to IL-2 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not crystal clear GRE|since regulation stated at protein level---{\"tf associated gene name\":[\"FOXO3\"],\"tg associated gene name\":[\"CCND2\"],\"tf text\":[\"FoxO3a\"],\"tg text\":[\"cyclin D2\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"not crystal clear GRE\",\"since regulation stated at protein level\"],\"sentence\":[\"We find that conditional activation of FoxO3a leads to accumulation of BCL6 and down-regulation of cyclin D2 at protein and mRNA levels.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "not crystal clear GRE|since regulation stated at protein level---{\"tf associated gene name\":[\"FOXO3\"],\"tg associated gene name\":[\"CCND2\"],\"tf text\":[\"FoxO3a\"],\"tg text\":[\"cyclin D2\"],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"not crystal clear GRE\",\"since regulation stated at protein level\"],\"sentence\":[\"We find that conditional activation of FoxO3a leads to accumulation of BCL6 and down-regulation of cyclin D2 at protein and mRNA levels.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXO3\"],\"target gene (associated gene name)\":[\"CCND2\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We find that conditional activation of FoxO3a leads to accumulation of BCL6 and down-regulation of cyclin D2 at protein and mRNA levels.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXO3\"],\"tg (associated gene name)\":[\"CCND2\"],\"tf text\":[\"FoxO3a\"],\"tg text\":[\"cyclin D2\"],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"We find that conditional activation of FoxO3a leads to accumulation of BCL6 and down-regulation of cyclin D2 at protein and mRNA levels.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 380 (char 379)\n",
      "{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"CD36\"],\"tf text\":[\"PPARgamma\"],\"tg text\":[\"CD36\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Expression of PKB stimulated the CD36 promoter as well as a PPARgamma element-driven reporter gene\",\"whereas an inactive PKB mutant had no effect.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"CD36\"],\"tf text\":[\"PPARgamma\"],\"tg text\":[\"CD36\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression of PKB stimulated the CD36 promoter as well as a PPARgamma element-driven reporter gene, whereas an inactive PKB mutant had no effect.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"CD36\"],\"tf text\":[\"PPARgamma\"],\"tg text\":[\"CD36\"],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"Expression of PKB stimulated the CD36 promoter as well as a PPARgamma element-driven reporter gene\",\"whereas an inactive PKB mutant had no effect.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PPARG\"],\"tg (associated gene name)\":[\"CD36\"],\"tf text\":[\"PPARgamma\"],\"tg text\":[\"CD36\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Expression of PKB stimulated the CD36 promoter as well as a PPARgamma element-driven reporter gene, whereas an inactive PKB mutant had no effect.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 527 (char 526)\n",
      "{\"tf associated gene name\":[\"GATA4\"],\"tg associated gene name\":[\"AMH\"],\"tf text\":[\"GATA-4\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"Specifically involved in Sertoli cell differentiation and subsequent anti-M??llerian hormone (AMH) secretion in eutherian mammals\",\"they include steroidogenic factor-1 (SF-1)\",\"SOX9 (SRY HMG box related gene 9)\",\"WT1 (Wilms' tumor 1)\",\"and GATA-4 (a zinc finger transcription factor).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|long causality chain, but transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA4\"],\"tg (associated gene name)\":[\"AMH\"],\"tf text\":[\"GATA-4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"long causality chain, but transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Specifically involved in Sertoli cell differentiation and subsequent anti-Mllerian hormone (AMH) secretion in eutherian mammals, they include steroidogenic factor-1 (SF-1), SOX9 (SRY HMG box related gene 9), WT1 (Wilms' tumor 1), and GATA-4 (a zinc finger transcription factor).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"GATA4\"],\"tg associated gene name\":[\"AMH\"],\"tf text\":[\"GATA-4\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"Specifically involved in Sertoli cell differentiation and subsequent anti-M??llerian hormone (AMH) secretion in eutherian mammals\",\"they include steroidogenic factor-1 (SF-1)\",\"SOX9 (SRY HMG box related gene 9)\",\"WT1 (Wilms' tumor 1)\",\"and GATA-4 (a zinc finger transcription factor).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|long causality chain, but transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA4\"],\"tg (associated gene name)\":[\"AMH\"],\"tf text\":[\"GATA-4\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"long causality chain, but transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Specifically involved in Sertoli cell differentiation and subsequent anti-Mllerian hormone (AMH) secretion in eutherian mammals, they include steroidogenic factor-1 (SF-1), SOX9 (SRY HMG box related gene 9), WT1 (Wilms' tumor 1), and GATA-4 (a zinc finger transcription factor).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 368 (char 367)\n",
      "{\"tf associated gene name\":[\"NCOA3\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These results suggest that p/CIP positively regulates STAT6 transcriptional activation through formation of a STAT6\",\"p300/CBP\",\"and p/CIP complex.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA3\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results suggest that p/CIP positively regulates STAT6 transcriptional activation through formation of a STAT6, p300/CBP, and p/CIP complex.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NCOA3\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These results suggest that p/CIP positively regulates STAT6 transcriptional activation through formation of a STAT6\",\"p300/CBP\",\"and p/CIP complex.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA3\"],\"tg (associated gene name)\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results suggest that p/CIP positively regulates STAT6 transcriptional activation through formation of a STAT6, p300/CBP, and p/CIP complex.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is insulin---{\"tf associated gene name\":[\"NEUROD1\"],\"tg associated gene name\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is insulin\"],\"sentence\":[\"beta-cell-enriched transcription factors\",\"such as Pdx1 and Beta2\",\"have dual roles in the activation of the insulin gene promoter establishing beta-cell-specific insulin expression\",\"and in the regulation of beta-cell differentiation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG, CEL problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NEUROD1\"],\"tg (associated gene name)\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, CEL problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"beta-cell-enriched transcription factors, such as Pdx1 and Beta2, have dual roles in the activation of the insulin gene promoter establishing beta-cell-specific insulin expression, and in the regulation of beta-cell differentiation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is insulin---{\"tf associated gene name\":[\"NEUROD1\"],\"tg associated gene name\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is insulin\"],\"sentence\":[\"beta-cell-enriched transcription factors\",\"such as Pdx1 and Beta2\",\"have dual roles in the activation of the insulin gene promoter establishing beta-cell-specific insulin expression\",\"and in the regulation of beta-cell differentiation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG, CEL problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NEUROD1\"],\"tg (associated gene name)\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, CEL problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"beta-cell-enriched transcription factors, such as Pdx1 and Beta2, have dual roles in the activation of the insulin gene promoter establishing beta-cell-specific insulin expression, and in the regulation of beta-cell differentiation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ADAM2-problem; IS autoregulation!---{\"tf associated gene name\":[\"GATA3\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[\"GATA3\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"ADAM2-problem; IS autoregulation!\"],\"sentence\":[\"Thus\",\"IFN-??/?? disrupts the GATA3-autoactivation loop and promotes epigenetic silencing of a Th2-specific regulatory region within the GATA3 gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|auroreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA3\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[\"GATA3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"auroreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, IFN-/ disrupts the GATA3-autoactivation loop and promotes epigenetic silencing of a Th2-specific regulatory region within the GATA3 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "ADAM2-problem; IS autoregulation!---{\"tf associated gene name\":[\"GATA3\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[\"GATA3\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"ADAM2-problem; IS autoregulation!\"],\"sentence\":[\"Thus\",\"IFN-??/?? disrupts the GATA3-autoactivation loop and promotes epigenetic silencing of a Th2-specific regulatory region within the GATA3 gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|auroreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"GATA3\"],\"tg (associated gene name)\":[\"ADAM2\"],\"tf text\":[\"GATA3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"auroreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Thus, IFN-/ disrupts the GATA3-autoactivation loop and promotes epigenetic silencing of a Th2-specific regulatory region within the GATA3 gene.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 412 (char 411)\n",
      "{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"IGFBP1\"],\"tf text\":[],\"tg text\":[\"IGFBP-1\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"As expected\",\"human recombinant IGFBP-1 (80 nM) completely inhibited IGF-I-mediated activation of ER\",\"however\",\"IGFBP-1 also caused a significant decrease in E2-mediated activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"IGFBP1\"],\"tf text\":[],\"tg text\":[\"IGFBP-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As expected, human recombinant IGFBP-1 (80 nM) completely inhibited IGF-I-mediated activation of ER, however, IGFBP-1 also caused a significant decrease in E2-mediated activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"IGFBP1\"],\"tf text\":[],\"tg text\":[\"IGFBP-1\"],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[],\"sentence\":[\"As expected\",\"human recombinant IGFBP-1 (80 nM) completely inhibited IGF-I-mediated activation of ER\",\"however\",\"IGFBP-1 also caused a significant decrease in E2-mediated activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"IGFBP1\"],\"tf text\":[],\"tg text\":[\"IGFBP-1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"As expected, human recombinant IGFBP-1 (80 nM) completely inhibited IGF-I-mediated activation of ER, however, IGFBP-1 also caused a significant decrease in E2-mediated activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is OPN---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"CTNNB1\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is OPN\"],\"sentence\":[\"Here we report that beta-catenin\",\"Lef-1\",\"Ets transcription factors\",\"and the AP-1 protein c-Jun each weakly enhanced luciferase expression from a OPN promoter-luciferase reporter construct\",\"transiently transfected into a rat mammary cell line.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"CTNNB1\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we report that beta-catenin, Lef-1, Ets transcription factors, and the AP-1 protein c-Jun each weakly enhanced luciferase expression from a OPN promoter-luciferase reporter construct, transiently transfected into a rat mammary cell line.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is OPN---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"CTNNB1\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is OPN\"],\"sentence\":[\"Here we report that beta-catenin\",\"Lef-1\",\"Ets transcription factors\",\"and the AP-1 protein c-Jun each weakly enhanced luciferase expression from a OPN promoter-luciferase reporter construct\",\"transiently transfected into a rat mammary cell line.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AP1\"],\"tg (associated gene name)\":[\"CTNNB1\"],\"tf text\":[\"c-Jun\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we report that beta-catenin, Lef-1, Ets transcription factors, and the AP-1 protein c-Jun each weakly enhanced luciferase expression from a OPN promoter-luciferase reporter construct, transiently transfected into a rat mammary cell line.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"LHX3\"],\"tg associated gene name\":[\"LHX3\"],\"tf text\":[\"LHX3\"],\"tg text\":[\"LHX3\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"not TRE\"],\"sentence\":[\"These results demonstrate that the downstream LHX3 enhancer is important in specific endocrine and neural cell types but also indicate that additional regulatory elements are likely involved in LHX3 gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|is this really autoreg?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LHX3\"],\"tg (associated gene name)\":[\"LHX3\"],\"tf text\":[\"LHX3\"],\"tg text\":[\"LHX3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"is this really autoreg?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results demonstrate that the downstream LHX3 enhancer is important in specific endocrine and neural cell types but also indicate that additional regulatory elements are likely involved in LHX3 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"LHX3\"],\"tg associated gene name\":[\"LHX3\"],\"tf text\":[\"LHX3\"],\"tg text\":[\"LHX3\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"not TRE\"],\"sentence\":[\"These results demonstrate that the downstream LHX3 enhancer is important in specific endocrine and neural cell types but also indicate that additional regulatory elements are likely involved in LHX3 gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|is this really autoreg?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LHX3\"],\"tg (associated gene name)\":[\"LHX3\"],\"tf text\":[\"LHX3\"],\"tg text\":[\"LHX3\"],\"consensus\":[\"N\"],\"martin_comments\":[\"is this really autoreg?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results demonstrate that the downstream LHX3 enhancer is important in specific endocrine and neural cell types but also indicate that additional regulatory elements are likely involved in LHX3 gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is Errb2---{\"tf associated gene name\":[\"ESR2\"],\"tg associated gene name\":[\"BIRC5\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[\"regulator is Errb2\"],\"sentence\":[\"Inhibition of ErbB2 by Herceptin reduces survivin expression via the ErbB2-beta-catenin/TCF4-survivin pathway in ErbB2-overexpressed breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF wrong norm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR2\"],\"tg (associated gene name)\":[\"BIRC5\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF wrong norm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of ErbB2 by Herceptin reduces survivin expression via the ErbB2-beta-catenin/TCF4-survivin pathway in ErbB2-overexpressed breast cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "regulator is Errb2---{\"tf associated gene name\":[\"ESR2\"],\"tg associated gene name\":[\"BIRC5\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[\"regulator is Errb2\"],\"sentence\":[\"Inhibition of ErbB2 by Herceptin reduces survivin expression via the ErbB2-beta-catenin/TCF4-survivin pathway in ErbB2-overexpressed breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TF wrong norm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR2\"],\"tg (associated gene name)\":[\"BIRC5\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TF wrong norm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Inhibition of ErbB2 by Herceptin reduces survivin expression via the ErbB2-beta-catenin/TCF4-survivin pathway in ErbB2-overexpressed breast cancer cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 324 (char 323)\n",
      "{\"tf associated gene name\":[\"NFKB1\"],\"tg associated gene name\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These results suggest that Tax activates NF-kappa B essentially through the p105/p100-retention pathway.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB1\"],\"tg (associated gene name)\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results suggest that Tax activates NF-kappa B essentially through the p105/p100-retention pathway.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB1\"],\"tg associated gene name\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"These results suggest that Tax activates NF-kappa B essentially through the p105/p100-retention pathway.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB1\"],\"tg (associated gene name)\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"These results suggest that Tax activates NF-kappa B essentially through the p105/p100-retention pathway.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is AR---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"RENBP\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is AR\"],\"sentence\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated\",\"p/CAF-containing coactivator assembly for a p53-interacting\",\"Groucho/TLE1-\",\"and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"RENBP\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated, p/CAF-containing coactivator assembly for a p53-interacting, Groucho/TLE1-, and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is AR---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"RENBP\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is AR\"],\"sentence\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated\",\"p/CAF-containing coactivator assembly for a p53-interacting\",\"Groucho/TLE1-\",\"and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"RENBP\"],\"tf text\":[\"PARP-1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Herein we show that transcriptional down-regulation of androgen receptor (AR) in the aging rat liver and in oxidatively stressed hepatoma cells involves exchange of a PARP-1-associated, p/CAF-containing coactivator assembly for a p53-interacting, Groucho/TLE1-, and mSin3A-included corepressor complex at an age- and oxidant-responsive DNA element (age-dependent factor (ADF) element) in the AR promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"TG not named\"],\"sentence\":[\"Here we show that interferon-alpha (IFN-alpha)-induced activation of the transcriptional activator STAT1 and inflammatory STAT1 target genes was enhanced in IFN-gamma-primed macrophages.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we show that interferon-alpha (IFN-alpha)-induced activation of the transcriptional activator STAT1 and inflammatory STAT1 target genes was enhanced in IFN-gamma-primed macrophages.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG not named---{\"tf associated gene name\":[\"STAT1\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"TG not named\"],\"sentence\":[\"Here we show that interferon-alpha (IFN-alpha)-induced activation of the transcriptional activator STAT1 and inflammatory STAT1 target genes was enhanced in IFN-gamma-primed macrophages.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[\"STAT1\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we show that interferon-alpha (IFN-alpha)-induced activation of the transcriptional activator STAT1 and inflammatory STAT1 target genes was enhanced in IFN-gamma-primed macrophages.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "wrong TG, norm on nuclear receptor?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1I3\"],\"tg (associated gene name)\":[\"NCOA6\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, norm on nuclear receptor?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Taken together, these results strongly suggest that ASC-2 is a bona fide coactivator of the xenobiotic nuclear receptor CAR and mediate the specific xenobiotic response by CAR in vivo.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "wrong TG, norm on nuclear receptor?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR1I3\"],\"tg (associated gene name)\":[\"NCOA6\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, norm on nuclear receptor?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Taken together, these results strongly suggest that ASC-2 is a bona fide coactivator of the xenobiotic nuclear receptor CAR and mediate the specific xenobiotic response by CAR in vivo.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "causal, but transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"CDK4\"],\"tf text\":[\"STAT1\"],\"tg text\":[\"Cdk4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"causal, but transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"STAT1 interacts directly with cyclin D1/Cdk4 and mediates cell cycle arrest.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "causal, but transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT1\"],\"tg (associated gene name)\":[\"CDK4\"],\"tf text\":[\"STAT1\"],\"tg text\":[\"Cdk4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"causal, but transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"STAT1 interacts directly with cyclin D1/Cdk4 and mediates cell cycle arrest.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 354 (char 353)\n",
      "{\"transcription factor (associated gene name)\":[\"MYC\"],\"target gene (associated gene name)\":[\"CEBPA\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We show that the endogenous C/EBP alpha protein level is repressed in stable cell lines overexpressing Myc.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|but transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"CEBPA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"but transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that the endogenous C/EBP alpha protein level is repressed in stable cell lines overexpressing Myc.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"MYC\"],\"tg associated gene name\":[\"CEBPA\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r\"],\"negation\":[],\"not true\":[],\"comment\":[],\"sentence\":[\"We show that the endogenous C/EBP alpha protein level is repressed in stable cell lines overexpressing Myc.\"],\"_\":[\"_\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MYC\"],\"target gene (associated gene name)\":[\"CEBPA\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We show that the endogenous C/EBP alpha protein level is repressed in stable cell lines overexpressing Myc.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|but transcription?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"MYC\"],\"tg (associated gene name)\":[\"CEBPA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"but transcription?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We show that the endogenous C/EBP alpha protein level is repressed in stable cell lines overexpressing Myc.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 346 (char 345)\n",
      "{\"transcription factor (associated gene name)\":[\"PROP1\"],\"target gene (associated gene name)\":[\"CGA\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"qc\":[\"1\"],\"sentence\":[\"To our knowledge\",\"this is the first demonstration of the role of Prop-1 in the regulation of alphaGSU gene expression.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TG, alpha norm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PROP1\"],\"tg (associated gene name)\":[\"CGA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, alpha norm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To our knowledge, this is the first demonstration of the role of Prop-1 in the regulation of alphaGSU gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"PROP1\"],\"tg associated gene name\":[\"CGA\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[],\"sentence\":[\"To our knowledge\",\"this is the first demonstration of the role of Prop-1 in the regulation of alphaGSU gene expression.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PROP1\"],\"target gene (associated gene name)\":[\"CGA\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"qc\":[\"1\"],\"sentence\":[\"To our knowledge\",\"this is the first demonstration of the role of Prop-1 in the regulation of alphaGSU gene expression.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|wrong TG, alpha norm---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PROP1\"],\"tg (associated gene name)\":[\"CGA\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, alpha norm\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"To our knowledge, this is the first demonstration of the role of Prop-1 in the regulation of alphaGSU gene expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Nrl---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"RHO\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Nrl\"],\"sentence\":[\"In vitro DNA binding assays and transient transfection analysis with monkey kidney cells have implicated Nrl\",\"a member of the Maf-Nrl subfamily of bZIP transcription factors\",\"and the Nrl response element (NRE) in the regulation of rhodopsin expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Nrl---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"RHO\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Nrl\"],\"sentence\":[\"In vitro DNA binding assays and transient transfection analysis with monkey kidney cells have implicated Nrl\",\"a member of the Maf-Nrl subfamily of bZIP transcription factors\",\"and the Nrl response element (NRE) in the regulation of rhodopsin expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SPI1\"],\"tg (associated gene name)\":[\"RHO\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"In vitro DNA binding assays and transient transfection analysis with monkey kidney cells have implicated Nrl, a member of the Maf-Nrl subfamily of bZIP transcription factors, and the Nrl response element (NRE) in the regulation of rhodopsin expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is StAR|no TF named---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is StAR\",\"no TF named\"],\"sentence\":[\"We demonstrated that BMP-4 suppresses the acetylation of histone H3 associated with the StAR promoter region at 48 and 72 h of culture in bovine granulosa cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|junmbled---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"junmbled\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrated that BMP-4 suppresses the acetylation of histone H3 associated with the StAR promoter region at 48 and 72 h of culture in bovine granulosa cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is StAR|no TF named---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is StAR\",\"no TF named\"],\"sentence\":[\"We demonstrated that BMP-4 suppresses the acetylation of histone H3 associated with the StAR promoter region at 48 and 72 h of culture in bovine granulosa cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|junmbled---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"junmbled\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We demonstrated that BMP-4 suppresses the acetylation of histone H3 associated with the StAR promoter region at 48 and 72 h of culture in bovine granulosa cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"FOXP3\"],\"tg associated gene name\":[\"CD28\"],\"tf text\":[\"FOXP3\"],\"tg text\":[\"CD28\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Using RT-PCR we show here that alloantigen specific CD8(+)CD28(-) T suppressor cells generated in vitro are FOXP3 positive and interact with human endothelial cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"CD28\"],\"tf text\":[\"FOXP3\"],\"tg text\":[\"CD28\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using RT-PCR we show here that alloantigen specific CD8(+)CD28(-) T suppressor cells generated in vitro are FOXP3 positive and interact with human endothelial cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"FOXP3\"],\"tg associated gene name\":[\"CD28\"],\"tf text\":[\"FOXP3\"],\"tg text\":[\"CD28\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Using RT-PCR we show here that alloantigen specific CD8(+)CD28(-) T suppressor cells generated in vitro are FOXP3 positive and interact with human endothelial cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"CD28\"],\"tf text\":[\"FOXP3\"],\"tg text\":[\"CD28\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Using RT-PCR we show here that alloantigen specific CD8(+)CD28(-) T suppressor cells generated in vitro are FOXP3 positive and interact with human endothelial cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"The loss of ER expression portends a poor prognosis and\",\"in a significant fraction of breast cancers\",\"this repression is a result of the hypermethylation of CpG islands within the ER-alpha promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The loss of ER expression portends a poor prognosis and, in a significant fraction of breast cancers, this repression is a result of the hypermethylation of CpG islands within the ER-alpha promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"The loss of ER expression portends a poor prognosis and\",\"in a significant fraction of breast cancers\",\"this repression is a result of the hypermethylation of CpG islands within the ER-alpha promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ESR1\"],\"tg (associated gene name)\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The loss of ER expression portends a poor prognosis and, in a significant fraction of breast cancers, this repression is a result of the hypermethylation of CpG islands within the ER-alpha promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HES1\"],\"tg (associated gene name)\":[\"RBP4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Finally, Hes1 was identified as an RBPjkappa-dependent Notch target gene important for MPC maintenance and the suppression of in vitro chondrogenesis.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HES1\"],\"tg (associated gene name)\":[\"RBP4\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Finally, Hes1 was identified as an RBPjkappa-dependent Notch target gene important for MPC maintenance and the suppression of in vitro chondrogenesis.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is \"mu\"  - likely immunoglobulin gene---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"GSTM1\"],\"tf text\":[\"Ets-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is \\\"mu\\\"  - likely immunoglobulin gene\"],\"sentence\":[\"HMGA1 associates with the ETS factor PU.1\",\"resulting in changes in PU.1 structure\",\"and enhanced transcriptional synergy with Ets-1 on the mu enhancer in nonlymphoid cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is \"mu\"  - likely immunoglobulin gene---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"GSTM1\"],\"tf text\":[\"Ets-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is \\\"mu\\\"  - likely immunoglobulin gene\"],\"sentence\":[\"HMGA1 associates with the ETS factor PU.1\",\"resulting in changes in PU.1 structure\",\"and enhanced transcriptional synergy with Ets-1 on the mu enhancer in nonlymphoid cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ETS1\"],\"target gene (associated gene name)\":[\"GSTM1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"HMGA1 associates with the ETS factor PU.1\",\"resulting in changes in PU.1 structure\",\"and enhanced transcriptional synergy with Ets-1 on the mu enhancer in nonlymphoid cells.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.723522\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"171\"],\"sentence pair density\":[\"0.00961538461538461\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|mu enough for TG?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ETS1\"],\"tg (associated gene name)\":[\"GSTM1\"],\"tf text\":[\"Ets-1\"],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[\"mu enough for TG?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"HMGA1 associates with the ETS factor PU.1, resulting in changes in PU.1 structure, and enhanced transcriptional synergy with Ets-1 on the mu enhancer in nonlymphoid cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|regulator is signaling protein Raf-1---{\"tf associated gene name\":[\"ZHX2\"],\"tg associated gene name\":[\"OCLN\"],\"tf text\":[],\"tg text\":[\"occludin\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\",\"regulator is signaling protein Raf-1\"],\"sentence\":[\"Here we report the mechanism by which Raf 1 regulates occludin expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE|regulator is signaling protein Raf-1---{\"tf associated gene name\":[\"ZHX2\"],\"tg associated gene name\":[\"OCLN\"],\"tf text\":[],\"tg text\":[\"occludin\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\",\"regulator is signaling protein Raf-1\"],\"sentence\":[\"Here we report the mechanism by which Raf 1 regulates occludin expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ZHX2\"],\"target gene (associated gene name)\":[\"OCLN\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Here we report the mechanism by which Raf 1 regulates occludin expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|raf1?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ZHX2\"],\"tg (associated gene name)\":[\"OCLN\"],\"tf text\":[],\"tg text\":[\"occludin\"],\"consensus\":[\"V\"],\"martin_comments\":[\"raf1?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Here we report the mechanism by which Raf 1 regulates occludin expression.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is MGSA/GRO alpha---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is MGSA/GRO alpha\"],\"sentence\":[\"The constitutive and cytokine IL-1-induced promoter activity of the chemokine MGSA/GRO alpha in normal retinal pigment epithelial and the Hs294T melanoma cells is partially regulated through the NF-kappa B element\",\"which binds both NF-kappa B p50 and RelA (NF-kappa B p65) homodimers and heterodimers.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrng TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrng TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The constitutive and cytokine IL-1-induced promoter activity of the chemokine MGSA/GRO alpha in normal retinal pigment epithelial and the Hs294T melanoma cells is partially regulated through the NF-kappa B element, which binds both NF-kappa B p50 and RelA (NF-kappa B p65) homodimers and heterodimers.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is MGSA/GRO alpha---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is MGSA/GRO alpha\"],\"sentence\":[\"The constitutive and cytokine IL-1-induced promoter activity of the chemokine MGSA/GRO alpha in normal retinal pigment epithelial and the Hs294T melanoma cells is partially regulated through the NF-kappa B element\",\"which binds both NF-kappa B p50 and RelA (NF-kappa B p65) homodimers and heterodimers.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrng TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrng TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The constitutive and cytokine IL-1-induced promoter activity of the chemokine MGSA/GRO alpha in normal retinal pigment epithelial and the Hs294T melanoma cells is partially regulated through the NF-kappa B element, which binds both NF-kappa B p50 and RelA (NF-kappa B p65) homodimers and heterodimers.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"CYCS\"],\"tf text\":[],\"tg text\":[\"cytochrome c\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"Transduction of MBP-1 in carcinoma cells releases cytochrome c from mitochondria into the cytosol leading to activation of procaspase-9\",\"procaspase-3 and PARP cleavage.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"CYCS\"],\"tf text\":[],\"tg text\":[\"cytochrome c\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transduction of MBP-1 in carcinoma cells releases cytochrome c from mitochondria into the cytosol leading to activation of procaspase-9, procaspase-3 and PARP cleavage.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"CYCS\"],\"tf text\":[],\"tg text\":[\"cytochrome c\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"Transduction of MBP-1 in carcinoma cells releases cytochrome c from mitochondria into the cytosol leading to activation of procaspase-9\",\"procaspase-3 and PARP cleavage.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"PARP1\"],\"tg (associated gene name)\":[\"CYCS\"],\"tf text\":[],\"tg text\":[\"cytochrome c\"],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transduction of MBP-1 in carcinoma cells releases cytochrome c from mitochondria into the cytosol leading to activation of procaspase-9, procaspase-3 and PARP cleavage.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is S100A6---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[\"p65\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is S100A6\"],\"sentence\":[\"Transient expression of NF-kappaB (p65) increased S100A6 promoter activity and expression of inhibitor of NF-kappaB (IkappaBalpha) decreased TNFalpha-induced S100A6 promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"NFKB\"],\"tf text\":[\"p65\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transient expression of NF-kappaB (p65) increased S100A6 promoter activity and expression of inhibitor of NF-kappaB (IkappaBalpha) decreased TNFalpha-induced S100A6 promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is S100A6---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[\"p65\"],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is S100A6\"],\"sentence\":[\"Transient expression of NF-kappaB (p65) increased S100A6 promoter activity and expression of inhibitor of NF-kappaB (IkappaBalpha) decreased TNFalpha-induced S100A6 promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"NFKB\"],\"tf text\":[\"p65\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transient expression of NF-kappaB (p65) increased S100A6 promoter activity and expression of inhibitor of NF-kappaB (IkappaBalpha) decreased TNFalpha-induced S100A6 promoter activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 471 (char 470)\n",
      "{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"AHSG\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[\"Promoter activity experiments in combination with DNA mobility shift assays demonstrated that BSP promoter activity is under the control of the CRE element\",\"through CREB-1\",\"JunD and Fra-2 binding\",\"in MDA-MB-231\",\"MCF-7 and in Saos-2 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOSB\"],\"tg (associated gene name)\":[\"AHSG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Promoter activity experiments in combination with DNA mobility shift assays demonstrated that BSP promoter activity is under the control of the CRE element, through CREB-1, JunD and Fra-2 binding, in MDA-MB-231, MCF-7 and in Saos-2 cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"transcription factor (associated gene name)\":[\"FOSB\"],\"target gene (associated gene name)\":[\"AHSG\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[\"Promoter activity experiments in combination with DNA mobility shift assays demonstrated that BSP promoter activity is under the control of the CRE element\",\"through CREB-1\",\"JunD and Fra-2 binding\",\"in MDA-MB-231\",\"MCF-7 and in Saos-2 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOSB\"],\"tg (associated gene name)\":[\"AHSG\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"Promoter activity experiments in combination with DNA mobility shift assays demonstrated that BSP promoter activity is under the control of the CRE element, through CREB-1, JunD and Fra-2 binding, in MDA-MB-231, MCF-7 and in Saos-2 cells.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "HES3 is TG---{\"tf associated gene name\":[\"HES3\"],\"tg associated gene name\":[\"MAMLD1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"HES3 is TG\"],\"sentence\":[\"Transactivation analysis for wild-type CXorf6 protein by luciferase assays showed that CXorf6 significantly transactivated the promoter of a noncanonical Notch target gene hairy/enhancer of split 3 (Hes3) without demonstrable DNA-binding capacity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HES3\"],\"tg (associated gene name)\":[\"MAMLD1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transactivation analysis for wild-type CXorf6 protein by luciferase assays showed that CXorf6 significantly transactivated the promoter of a noncanonical Notch target gene hairy/enhancer of split 3 (Hes3) without demonstrable DNA-binding capacity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "HES3 is TG---{\"tf associated gene name\":[\"HES3\"],\"tg associated gene name\":[\"MAMLD1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"HES3 is TG\"],\"sentence\":[\"Transactivation analysis for wild-type CXorf6 protein by luciferase assays showed that CXorf6 significantly transactivated the promoter of a noncanonical Notch target gene hairy/enhancer of split 3 (Hes3) without demonstrable DNA-binding capacity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"HES3\"],\"tg (associated gene name)\":[\"MAMLD1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Transactivation analysis for wild-type CXorf6 protein by luciferase assays showed that CXorf6 significantly transactivated the promoter of a noncanonical Notch target gene hairy/enhancer of split 3 (Hes3) without demonstrable DNA-binding capacity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"HDAC9\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"The histone deacetylase (HDAC) inhibitor Tricostatin A partially blocked the TGFbeta1 inhibition of CYP7A1 mRNA expression\",\"whereas TGFbeta1 decreased histone 3 acetylation in the CYP7A1 chromatin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"HDAC9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The histone deacetylase (HDAC) inhibitor Tricostatin A partially blocked the TGFbeta1 inhibition of CYP7A1 mRNA expression, whereas TGFbeta1 decreased histone 3 acetylation in the CYP7A1 chromatin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"HDAC9\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"The histone deacetylase (HDAC) inhibitor Tricostatin A partially blocked the TGFbeta1 inhibition of CYP7A1 mRNA expression\",\"whereas TGFbeta1 decreased histone 3 acetylation in the CYP7A1 chromatin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TCF3\"],\"tg (associated gene name)\":[\"HDAC9\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The histone deacetylase (HDAC) inhibitor Tricostatin A partially blocked the TGFbeta1 inhibition of CYP7A1 mRNA expression, whereas TGFbeta1 decreased histone 3 acetylation in the CYP7A1 chromatin.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXN1\"],\"tg (associated gene name)\":[\"CHD4\"],\"tf text\":[],\"tg text\":[\"CHD4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"MBD2 and multiple domains of CHD4 are required for transcriptional repression by Mi-2/NuRD complexes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXN1\"],\"tg (associated gene name)\":[\"CHD4\"],\"tf text\":[],\"tg text\":[\"CHD4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[],\"sentences (previous letter version)\":[\"MBD2 and multiple domains of CHD4 are required for transcriptional repression by Mi-2/NuRD complexes.\"],\"set\":[\"Extended\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG most likely  HXT2---{\"tf associated gene name\":[\"TBP\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG most likely  HXT2\"],\"sentence\":[\"For one potential activation target\",\"HXT2\",\"we analyzed promoter recruitment of Mot1p\",\"TBP\",\"Taf1p (Taf130p) and pol II by chromatin immunoprecipitation assays.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBP\"],\"tg (associated gene name)\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"For one potential activation target, HXT2, we analyzed promoter recruitment of Mot1p, TBP, Taf1p (Taf130p) and pol II by chromatin immunoprecipitation assays.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG most likely  HXT2---{\"tf associated gene name\":[\"TBP\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG most likely  HXT2\"],\"sentence\":[\"For one potential activation target\",\"HXT2\",\"we analyzed promoter recruitment of Mot1p\",\"TBP\",\"Taf1p (Taf130p) and pol II by chromatin immunoprecipitation assays.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TBP\"],\"tg (associated gene name)\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"For one potential activation target, HXT2, we analyzed promoter recruitment of Mot1p, TBP, Taf1p (Taf130p) and pol II by chromatin immunoprecipitation assays.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TFs are Msx1 and Pax9---{\"tf associated gene name\":[\"LBX1\"],\"tg associated gene name\":[\"BMP4\"],\"tf text\":[],\"tg text\":[\"bone morphogenetic protein 4\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TFs are Msx1 and Pax9\"],\"sentence\":[\"For many years the molecular mechanisms governing bone morphogenetic protein 4 (Bmp4) expression in tooth bud mesenchyme could be explained by an uncomplicated model involving the interaction of the homeobox gene Msx1 and the paired domain gene Pax9 and a limited proximal promoter segment of Bmp4.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LBX1\"],\"tg (associated gene name)\":[\"BMP4\"],\"tf text\":[],\"tg text\":[\"bone morphogenetic protein 4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"For many years the molecular mechanisms governing bone morphogenetic protein 4 (Bmp4) expression in tooth bud mesenchyme could be explained by an uncomplicated model involving the interaction of the homeobox gene Msx1 and the paired domain gene Pax9 and a limited proximal promoter segment of Bmp4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TFs are Msx1 and Pax9---{\"tf associated gene name\":[\"LBX1\"],\"tg associated gene name\":[\"BMP4\"],\"tf text\":[],\"tg text\":[\"bone morphogenetic protein 4\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TFs are Msx1 and Pax9\"],\"sentence\":[\"For many years the molecular mechanisms governing bone morphogenetic protein 4 (Bmp4) expression in tooth bud mesenchyme could be explained by an uncomplicated model involving the interaction of the homeobox gene Msx1 and the paired domain gene Pax9 and a limited proximal promoter segment of Bmp4.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"LBX1\"],\"tg (associated gene name)\":[\"BMP4\"],\"tf text\":[],\"tg text\":[\"bone morphogenetic protein 4\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"For many years the molecular mechanisms governing bone morphogenetic protein 4 (Bmp4) expression in tooth bud mesenchyme could be explained by an uncomplicated model involving the interaction of the homeobox gene Msx1 and the paired domain gene Pax9 and a limited proximal promoter segment of Bmp4.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain that this is TRE---{\"tf associated gene name\":[\"FOXP3\"],\"tg associated gene name\":[\"IL2RA\"],\"tf text\":[\"hFOXP3\"],\"tg text\":[\"CD25\"],\"1\":[],\"negation\":[],\"not true\":[\"uncertain\"],\"comment\":[\"uncertain that this is TRE\"],\"sentence\":[\"PTD-hFOXP3-transduced Jurkat cells (human T lymphoma cell line) and CD4(+)CD25(-) T cells failed to proliferate and produce IL-2 and IFN-??\",\"but produced large amounts of the cytokines IL-4\",\"IL-10\",\"and TGF-??\",\"in response to TCR stimulation in vitro.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG IL2 enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"IL2RA\"],\"tf text\":[\"hFOXP3\"],\"tg text\":[\"CD25\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG IL2 enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PTD-hFOXP3-transduced Jurkat cells (human T lymphoma cell line) and CD4(+)CD25(-) T cells failed to proliferate and produce IL-2 and IFN-, but produced large amounts of the cytokines IL-4, IL-10, and TGF-, in response to TCR stimulation in vitro.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "uncertain that this is TRE---{\"tf associated gene name\":[\"FOXP3\"],\"tg associated gene name\":[\"IL2RA\"],\"tf text\":[\"hFOXP3\"],\"tg text\":[\"CD25\"],\"1\":[],\"negation\":[],\"not true\":[\"uncertain\"],\"comment\":[\"uncertain that this is TRE\"],\"sentence\":[\"PTD-hFOXP3-transduced Jurkat cells (human T lymphoma cell line) and CD4(+)CD25(-) T cells failed to proliferate and produce IL-2 and IFN-??\",\"but produced large amounts of the cytokines IL-4\",\"IL-10\",\"and TGF-??\",\"in response to TCR stimulation in vitro.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG IL2 enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"FOXP3\"],\"tg (associated gene name)\":[\"IL2RA\"],\"tf text\":[\"hFOXP3\"],\"tg text\":[\"CD25\"],\"consensus\":[\"N\"],\"martin_comments\":[\"TG IL2 enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"PTD-hFOXP3-transduced Jurkat cells (human T lymphoma cell line) and CD4(+)CD25(-) T cells failed to proliferate and produce IL-2 and IFN-, but produced large amounts of the cytokines IL-4, IL-10, and TGF-, in response to TCR stimulation in vitro.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 716 (char 715)\n",
      "{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"TNFRSF1A\"],\"tf text\":[\"glucocorticoid receptor\"],\"tg text\":[\"tumor necrosis factor alpha receptor- and Fas-associated FLASH\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"We previously reported that tumor necrosis factor alpha receptor- and Fas-associated FLASH interacts with one of the p160 nuclear receptor coactivators\",\"glucocorticoid receptor-interacting protein (GRIP) 1\",\"at its nuclear receptor-binding (NRB) domain\",\"and that inhibits the transcriptional activity of the glucocorticoid receptor (GR) by interfering with association of GR and GRIP1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|junmbled?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"TNFRSF1A\"],\"tf text\":[\"glucocorticoid receptor\"],\"tg text\":[\"tumor necrosis factor alpha receptor- and Fas-associated FLASH\"],\"consensus\":[\"N\"],\"martin_comments\":[\"junmbled?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We previously reported that tumor necrosis factor alpha receptor- and Fas-associated FLASH interacts with one of the p160 nuclear receptor coactivators, glucocorticoid receptor-interacting protein (GRIP) 1, at its nuclear receptor-binding (NRB) domain, and that inhibits the transcriptional activity of the glucocorticoid receptor (GR) by interfering with association of GR and GRIP1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"TNFRSF1A\"],\"tf text\":[\"glucocorticoid receptor\"],\"tg text\":[\"tumor necrosis factor alpha receptor- and Fas-associated FLASH\"],\"1\":[],\"negation\":[],\"not true\":[\"c\"],\"comment\":[],\"sentence\":[\"We previously reported that tumor necrosis factor alpha receptor- and Fas-associated FLASH interacts with one of the p160 nuclear receptor coactivators\",\"glucocorticoid receptor-interacting protein (GRIP) 1\",\"at its nuclear receptor-binding (NRB) domain\",\"and that inhibits the transcriptional activity of the glucocorticoid receptor (GR) by interfering with association of GR and GRIP1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|junmbled?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NR3C1\"],\"tg (associated gene name)\":[\"TNFRSF1A\"],\"tf text\":[\"glucocorticoid receptor\"],\"tg text\":[\"tumor necrosis factor alpha receptor- and Fas-associated FLASH\"],\"consensus\":[\"N\"],\"martin_comments\":[\"junmbled?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"We previously reported that tumor necrosis factor alpha receptor- and Fas-associated FLASH interacts with one of the p160 nuclear receptor coactivators, glucocorticoid receptor-interacting protein (GRIP) 1, at its nuclear receptor-binding (NRB) domain, and that inhibits the transcriptional activity of the glucocorticoid receptor (GR) by interfering with association of GR and GRIP1.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"NEUROD1\"],\"tg associated gene name\":[\"PRKG1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"One of the key mediators of the cell's response to lowered oxygen environments is hypoxia-inducible-factor-1 (HIF-1)\",\"a basic helix-loop-helix transcription factor\",\"which enables cells to adapt to hypoxia by regulating the expression of genes involved in increasing oxygen availability (VEGF\",\"erythropoietin) and enhancing glucose uptake and metabolism (Glut-1\",\"PGK).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NEUROD1\"],\"tg (associated gene name)\":[\"PRKG1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"One of the key mediators of the cell's response to lowered oxygen environments is hypoxia-inducible-factor-1 (HIF-1), a basic helix-loop-helix transcription factor, which enables cells to adapt to hypoxia by regulating the expression of genes involved in increasing oxygen availability (VEGF, erythropoietin) and enhancing glucose uptake and metabolism (Glut-1, PGK).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"NEUROD1\"],\"tg associated gene name\":[\"PRKG1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"One of the key mediators of the cell's response to lowered oxygen environments is hypoxia-inducible-factor-1 (HIF-1)\",\"a basic helix-loop-helix transcription factor\",\"which enables cells to adapt to hypoxia by regulating the expression of genes involved in increasing oxygen availability (VEGF\",\"erythropoietin) and enhancing glucose uptake and metabolism (Glut-1\",\"PGK).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NEUROD1\"],\"tg (associated gene name)\":[\"PRKG1\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"One of the key mediators of the cell's response to lowered oxygen environments is hypoxia-inducible-factor-1 (HIF-1), a basic helix-loop-helix transcription factor, which enables cells to adapt to hypoxia by regulating the expression of genes involved in increasing oxygen availability (VEGF, erythropoietin) and enhancing glucose uptake and metabolism (Glut-1, PGK).\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is proteasome gene (i.e. non-normalizeable)---{\"tf associated gene name\":[\"NFYA\"],\"tg associated gene name\":[\"NFY\"],\"tf text\":[\"NF-YA\"],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is proteasome gene (i.e. non-normalizeable)\"],\"sentence\":[\"Knockdown of NF-YA\",\"an essential subunit of NF-Y\",\"reduced proteasome gene expression and compromised the cellular proteasome activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein lebel---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFYA\"],\"tg (associated gene name)\":[\"NFY\"],\"tf text\":[\"NF-YA\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein lebel\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Knockdown of NF-YA, an essential subunit of NF-Y, reduced proteasome gene expression and compromised the cellular proteasome activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is proteasome gene (i.e. non-normalizeable)---{\"tf associated gene name\":[\"NFYA\"],\"tg associated gene name\":[\"NFY\"],\"tf text\":[\"NF-YA\"],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is proteasome gene (i.e. non-normalizeable)\"],\"sentence\":[\"Knockdown of NF-YA\",\"an essential subunit of NF-Y\",\"reduced proteasome gene expression and compromised the cellular proteasome activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|protein lebel---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFYA\"],\"tg (associated gene name)\":[\"NFY\"],\"tf text\":[\"NF-YA\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"protein lebel\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Knockdown of NF-YA, an essential subunit of NF-Y, reduced proteasome gene expression and compromised the cellular proteasome activity.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is PSA---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"KLK3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is PSA\"],\"sentence\":[\"Hypoxia and androgen additively increased the recruitment of JMJD1A and p300 on the enhancer region of PSA through interaction with the hypoxia-inducible factor-1?? and AR\",\"both of which bind the PSA enhancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is PSA---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"KLK3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is PSA\"],\"sentence\":[\"Hypoxia and androgen additively increased the recruitment of JMJD1A and p300 on the enhancer region of PSA through interaction with the hypoxia-inducible factor-1?? and AR\",\"both of which bind the PSA enhancer.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AR\"],\"target gene (associated gene name)\":[\"KLK3\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Hypoxia and androgen additively increased the recruitment of JMJD1A and p300 on the enhancer region of PSA through interaction with the hypoxia-inducible factor-1 and AR\",\"both of which bind the PSA enhancer.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"AR\"],\"tg (associated gene name)\":[\"KLK3\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"V\"],\"martin_comments\":[],\"astrid_comments\":[],\"sentences with greek letters\":[\"Hypoxia and androgen additively increased the recruitment of JMJD1A and p300 on the enhancer region of PSA through interaction with the hypoxia-inducible factor-1 and AR, both of which bind the PSA enhancer.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 595 (char 594)\n",
      "{\"tf associated gene name\":[\"ARID1B\"],\"tg associated gene name\":[\"SMARCA2\"],\"tf text\":[\"ARID1B\"],\"tg text\":[\"SMARCA2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Notable among the alterations newly identified\",\"genomic deletions\",\"mutations\",\"and rearrangements recurrently targeted genes encoding components of the SWItch/Sucrose NonFermentable (SWI/SNF) chromatin remodeling complex\",\"including all three putative DNA binding subunits (ARID1A\",\"ARID1B\",\"and PBRM1) and both enzymatic subunits (SMARCA2 and SMARCA4).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ARID1B\"],\"tg (associated gene name)\":[\"SMARCA2\"],\"tf text\":[\"ARID1B\"],\"tg text\":[\"SMARCA2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Notable among the alterations newly identified, genomic deletions, mutations, and rearrangements recurrently targeted genes encoding components of the SWItch/Sucrose NonFermentable (SWI/SNF) chromatin remodeling complex, including all three putative DNA binding subunits (ARID1A, ARID1B, and PBRM1) and both enzymatic subunits (SMARCA2 and SMARCA4).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"ARID1B\"],\"tg associated gene name\":[\"SMARCA2\"],\"tf text\":[\"ARID1B\"],\"tg text\":[\"SMARCA2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Notable among the alterations newly identified\",\"genomic deletions\",\"mutations\",\"and rearrangements recurrently targeted genes encoding components of the SWItch/Sucrose NonFermentable (SWI/SNF) chromatin remodeling complex\",\"including all three putative DNA binding subunits (ARID1A\",\"ARID1B\",\"and PBRM1) and both enzymatic subunits (SMARCA2 and SMARCA4).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"ARID1B\"],\"tg (associated gene name)\":[\"SMARCA2\"],\"tf text\":[\"ARID1B\"],\"tg text\":[\"SMARCA2\"],\"consensus\":[\"N\"],\"martin_comments\":[\"no TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Notable among the alterations newly identified, genomic deletions, mutations, and rearrangements recurrently targeted genes encoding components of the SWItch/Sucrose NonFermentable (SWI/SNF) chromatin remodeling complex, including all three putative DNA binding subunits (ARID1A, ARID1B, and PBRM1) and both enzymatic subunits (SMARCA2 and SMARCA4).\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  syncytin-1 (ERVWE1)---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"MS\"],\"tf text\":[\"p65\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is  syncytin-1 (ERVWE1)\"],\"sentence\":[\"Taken together\",\"these results illustrate a role for p65 in regulating the ERVWE1 promoter and in TNFalpha-mediated induction of syncytin-1 in multiple sclerosis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG, MS problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"MS\"],\"tf text\":[\"p65\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, MS problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, these results illustrate a role for p65 in regulating the ERVWE1 promoter and in TNFalpha-mediated induction of syncytin-1 in multiple sclerosis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is  syncytin-1 (ERVWE1)---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"MS\"],\"tf text\":[\"p65\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is  syncytin-1 (ERVWE1)\"],\"sentence\":[\"Taken together\",\"these results illustrate a role for p65 in regulating the ERVWE1 promoter and in TNFalpha-mediated induction of syncytin-1 in multiple sclerosis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG, MS problem---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"RELA\"],\"tg (associated gene name)\":[\"MS\"],\"tf text\":[\"p65\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG, MS problem\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Taken together, these results illustrate a role for p65 in regulating the ERVWE1 promoter and in TNFalpha-mediated induction of syncytin-1 in multiple sclerosis.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 384 (char 383)\n",
      "{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Regulation of lipocalin-type prostaglandin D synthase gene expression by Hes-1 through E-box and interleukin-1 beta via two NF-kappa B elements in rat leptomeningeal cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Regulation of lipocalin-type prostaglandin D synthase gene expression by Hes-1 through E-box and interleukin-1 beta via two NF-kappa B elements in rat leptomeningeal cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"up\"],\"comment\":[],\"sentence\":[\"Regulation of lipocalin-type prostaglandin D synthase gene expression by Hes-1 through E-box and interleukin-1 beta via two NF-kappa B elements in rat leptomeningeal cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"IL1B\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Regulation of lipocalin-type prostaglandin D synthase gene expression by Hes-1 through E-box and interleukin-1 beta via two NF-kappa B elements in rat leptomeningeal cells.\"],\"sentences (previous letter version)\":[],\"set\":[\"PMID\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unsure about TG normalization (SM22 and myosin heavy chain)---{\"tf associated gene name\":[\"NCOA3\"],\"tg associated gene name\":[\"SLC3A2\"],\"tf text\":[\"SRC3\"],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"unsure about TG normalization (SM22 and myosin heavy chain)\"],\"sentence\":[\"The N terminus of SRC3\",\"which contains a basic helix-loop-helix/Per-ARNT-Sim protein-protein interaction domain\",\"binds the C-terminal activation domain of myocardin and enhances myocardin-mediated transcriptional activation of VSMC-specific\",\"CArG-containing promoters\",\"including the VSMC-specific genes SM22 and myosin heavy chain.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|TG heavy chain enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA3\"],\"tg (associated gene name)\":[\"SLC3A2\"],\"tf text\":[\"SRC3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG heavy chain enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The N terminus of SRC3, which contains a basic helix-loop-helix/Per-ARNT-Sim protein-protein interaction domain, binds the C-terminal activation domain of myocardin and enhances myocardin-mediated transcriptional activation of VSMC-specific, CArG-containing promoters, including the VSMC-specific genes SM22 and myosin heavy chain.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "unsure about TG normalization (SM22 and myosin heavy chain)---{\"tf associated gene name\":[\"NCOA3\"],\"tg associated gene name\":[\"SLC3A2\"],\"tf text\":[\"SRC3\"],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"unsure about TG normalization (SM22 and myosin heavy chain)\"],\"sentence\":[\"The N terminus of SRC3\",\"which contains a basic helix-loop-helix/Per-ARNT-Sim protein-protein interaction domain\",\"binds the C-terminal activation domain of myocardin and enhances myocardin-mediated transcriptional activation of VSMC-specific\",\"CArG-containing promoters\",\"including the VSMC-specific genes SM22 and myosin heavy chain.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|TG heavy chain enough?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NCOA3\"],\"tg (associated gene name)\":[\"SLC3A2\"],\"tf text\":[\"SRC3\"],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"TG heavy chain enough?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"The N terminus of SRC3, which contains a basic helix-loop-helix/Per-ARNT-Sim protein-protein interaction domain, binds the C-terminal activation domain of myocardin and enhances myocardin-mediated transcriptional activation of VSMC-specific, CArG-containing promoters, including the VSMC-specific genes SM22 and myosin heavy chain.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Remarkably\",\"the increased ??3 fatty acid levels primarily inhibit NF-??B-dependent inflammatory responses by uncoupling NF-??B binding and enhancer/promoter histone acetylation from subsequent steps required for proinflammatory gene activation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Remarkably\",\"the increased ??3 fatty acid levels primarily inhibit NF-??B-dependent inflammatory responses by uncoupling NF-??B binding and enhancer/promoter histone acetylation from subsequent steps required for proinflammatory gene activation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"NFKB\"],\"validation\":[\"N\"],\"comment 1\":[],\"comment 2\":[],\"\":[],\"sentence\":[\"Remarkably, the increased 3 fatty acid levels primarily inhibit NF-B-dependent inflammatory responses by uncoupling NF-B binding and enhancer/promoter histone acetylation from subsequent steps required for proinflammatory gene activation.\"],\"interaction score\":[\"2.200507\"],\"pmid counts\":[\"85\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|no TG---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"NFKB\"],\"tg (associated gene name)\":[\"NFKB\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"no TG\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Remarkably, the increased 3 fatty acid levels primarily inhibit NF-B-dependent inflammatory responses by uncoupling NF-B binding and enhancer/promoter histone acetylation from subsequent steps required for proinflammatory gene activation.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not likely TRE|regulator is PRMT5---{\"tf associated gene name\":[\"SMARCA1\"],\"tg associated gene name\":[\"ST7\"],\"tf text\":[\"SWI/SNF\"],\"tg text\":[\"ST7\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not likely TRE\",\"regulator is PRMT5\"],\"sentence\":[\"Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not likely TRE|regulator is PRMT5---{\"tf associated gene name\":[\"SMARCA1\"],\"tg associated gene name\":[\"ST7\"],\"tf text\":[\"SWI/SNF\"],\"tg text\":[\"ST7\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not likely TRE\",\"regulator is PRMT5\"],\"sentence\":[\"Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SMARCA1\"],\"target gene (associated gene name)\":[\"ST7\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---|methlation---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"SMARCA1\"],\"tg (associated gene name)\":[\"ST7\"],\"tf text\":[\"SWI/SNF\"],\"tg text\":[\"ST7\"],\"consensus\":[\"V\"],\"martin_comments\":[\"methlation\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"TWIST1\"],\"tg associated gene name\":[\"ITK\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Consistent with these morphological changes\",\"both E6 and E7 induce expression of the EMT-activating transcriptional factors Slug\",\"Twist\",\"ZEB1 and ZEB2\",\"especially ZEBs\",\"accompanied with switch from epithelial to mesenchymal markers.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TWIST1\"],\"tg (associated gene name)\":[\"ITK\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consistent with these morphological changes, both E6 and E7 induce expression of the EMT-activating transcriptional factors Slug, Twist, ZEB1 and ZEB2, especially ZEBs, accompanied with switch from epithelial to mesenchymal markers.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "not TRE---{\"tf associated gene name\":[\"TWIST1\"],\"tg associated gene name\":[\"ITK\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Consistent with these morphological changes\",\"both E6 and E7 induce expression of the EMT-activating transcriptional factors Slug\",\"Twist\",\"ZEB1 and ZEB2\",\"especially ZEBs\",\"accompanied with switch from epithelial to mesenchymal markers.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|flipped at best---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"TWIST1\"],\"tg (associated gene name)\":[\"ITK\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"flipped at best\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Consistent with these morphological changes, both E6 and E7 induce expression of the EMT-activating transcriptional factors Slug, Twist, ZEB1 and ZEB2, especially ZEBs, accompanied with switch from epithelial to mesenchymal markers.\"],\"sentences (previous letter version)\":[],\"set\":[\"Lowscore\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is SS18-SSX fusion protein---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"EGR1\"],\"tf text\":[\"EGR1\"],\"tg text\":[\"EGR1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TF is SS18-SSX fusion protein\"],\"sentence\":[\"In synovial sarcoma\",\"the SS18-SSX fusion protein represses EGR1 expression through a direct association with the EGR1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is SS18-SSX fusion protein---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"EGR1\"],\"tf text\":[\"EGR1\"],\"tg text\":[\"EGR1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TF is SS18-SSX fusion protein\"],\"sentence\":[\"In synovial sarcoma\",\"the SS18-SSX fusion protein represses EGR1 expression through a direct association with the EGR1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"EGR1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"In synovial sarcoma\",\"the SS18-SSX fusion protein represses EGR1 expression through a direct association with the EGR1 promoter.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.17411\"],\"pmid counts\":[\"92\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"127\"],\"sentence pair density\":[\"0.00485436893203883\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|wrong TF---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"EGR1\"],\"tg (associated gene name)\":[\"EGR1\"],\"tf text\":[\"EGR1\"],\"tg text\":[\"EGR1\"],\"consensus\":[\"N\"],\"martin_comments\":[\"wrong TF\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"In synovial sarcoma, the SS18-SSX fusion protein represses EGR1 expression through a direct association with the EGR1 promoter.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Extra data: line 1 column 483 (char 482)\n",
      "{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Peripheral CD3(+) PKCalpha-deficient T cells show unimpaired activation-induced IL-2 cytokine secretion\",\"surface expression of CD25\",\"CD44\",\"and CD69\",\"as well as transactivation of the critical transcription factors NF-AT\",\"NF-kappaB\",\"AP-1\",\"and STAT5 in vitro.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|enough causality?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"enough causality?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Peripheral CD3(+) PKCalpha-deficient T cells show unimpaired activation-induced IL-2 cytokine secretion, surface expression of CD25, CD44, and CD69, as well as transactivation of the critical transcription factors NF-AT, NF-kappaB, AP-1, and STAT5 in vitro.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "---{\"tf associated gene name\":[\"STAT5A\"],\"tg associated gene name\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[],\"sentence\":[\"Peripheral CD3(+) PKCalpha-deficient T cells show unimpaired activation-induced IL-2 cytokine secretion\",\"surface expression of CD25\",\"CD44\",\"and CD69\",\"as well as transactivation of the critical transcription factors NF-AT\",\"NF-kappaB\",\"AP-1\",\"and STAT5 in vitro.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|enough causality?---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"STAT5A\"],\"tg (associated gene name)\":[\"IL2\"],\"tf text\":[],\"tg text\":[],\"consensus\":[\"N\"],\"martin_comments\":[\"enough causality?\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Peripheral CD3(+) PKCalpha-deficient T cells show unimpaired activation-induced IL-2 cytokine secretion, surface expression of CD25, CD44, and CD69, as well as transactivation of the critical transcription factors NF-AT, NF-kappaB, AP-1, and STAT5 in vitro.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Gck---{\"tf associated gene name\":[\"KLF6\"],\"tg associated gene name\":[\"KLF6\"],\"tf text\":[\"Klf6\"],\"tg text\":[\"Klf6\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is Gck\"],\"sentence\":[\"Accordingly\",\"in DeltaKlf6 hepatocytes Gck expression was reduced and stable transfection of Klf6 led to up-regulation of Gck.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KLF6\"],\"tg (associated gene name)\":[\"KLF6\"],\"tf text\":[\"Klf6\"],\"tg text\":[\"Klf6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Accordingly, in DeltaKlf6 hepatocytes Gck expression was reduced and stable transfection of Klf6 led to up-regulation of Gck.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n",
      "TG is Gck---{\"tf associated gene name\":[\"KLF6\"],\"tg associated gene name\":[\"KLF6\"],\"tf text\":[\"Klf6\"],\"tg text\":[\"Klf6\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TG is Gck\"],\"sentence\":[\"Accordingly\",\"in DeltaKlf6 hepatocytes Gck expression was reduced and stable transfection of Klf6 led to up-regulation of Gck.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|not autoreg---{\"_\":[\"_\"],\"tf (associated gene name)\":[\"KLF6\"],\"tg (associated gene name)\":[\"KLF6\"],\"tf text\":[\"Klf6\"],\"tg text\":[\"Klf6\"],\"consensus\":[\"N\"],\"martin_comments\":[\"not autoreg\"],\"astrid_comments\":[],\"sentences with greek letters\":[\"Accordingly, in DeltaKlf6 hepatocytes Gck expression was reduced and stable transfection of Klf6 led to up-regulation of Gck.\"],\"sentences (previous letter version)\":[],\"set\":[\"Score\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "IRF6 - LPS---{\"_\":[\"_\"],\"tf\":[\"IRF6\"],\"tg\":[\"NOS2\"],\"validation\":[\"V\"],\"comment 1\":[\"IRF6 - LPS\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Overexpression of a C/EBP-specific dominant-negative mutant limited LPS + IFN-gamma activation of the NOS2 promoter.\"],\"interaction score\":[\"3.296122\"],\"pmid counts\":[\"63\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "IRF6 - LPS---{\"_\":[\"_\"],\"tf\":[\"IRF6\"],\"tg\":[\"NOS2\"],\"validation\":[\"V\"],\"comment 1\":[\"IRF6 - LPS\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Overexpression of a C/EBP-specific dominant-negative mutant limited LPS + IFN-gamma activation of the NOS2 promoter.\"],\"interaction score\":[\"3.296122\"],\"pmid counts\":[\"63\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF6\"],\"target gene (associated gene name)\":[\"NOS2\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of a C/EBP-specific dominant-negative mutant limited LPS + IFN-gamma activation of the NOS2 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CAT---{\"_\":[\"_\"],\"tf\":[\"IRF1\"],\"tg\":[\"CAT\"],\"validation\":[\"V\"],\"comment 1\":[\"CAT\"],\"comment 2\":[],\"\":[],\"sentence\":[\"IRF-1 also induced expression of its homologous repressor IRF-2 as indicated by EMSA analysis using an IRF-E probe from the IRF-2 promoter; and by cotransfection of IRF-1 together with an IRF-2 promoter CAT construct.\"],\"interaction score\":[\"2.095628\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "CAT---{\"_\":[\"_\"],\"tf\":[\"IRF1\"],\"tg\":[\"CAT\"],\"validation\":[\"V\"],\"comment 1\":[\"CAT\"],\"comment 2\":[],\"\":[],\"sentence\":[\"IRF-1 also induced expression of its homologous repressor IRF-2 as indicated by EMSA analysis using an IRF-E probe from the IRF-2 promoter; and by cotransfection of IRF-1 together with an IRF-2 promoter CAT construct.\"],\"interaction score\":[\"2.095628\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF1\"],\"target gene (associated gene name)\":[\"CAT\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"IRF-1 also induced expression of its homologous repressor IRF-2 as indicated by EMSA analysis using an IRF-E probe from the IRF-2 promoter; and by cotransfection of IRF-1 together with an IRF-2 promoter CAT construct.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "DLST normalized on bases of \"E2\" mention for TG---{\"_\":[\"_\"],\"tf\":[\"ESR1\"],\"tg\":[\"DLST\"],\"validation\":[\"V\"],\"comment 1\":[\"DLST normalized on bases of \\\"E2\\\" mention for TG\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Thus, hormone regulation of bcl-2 gene expression in breast cancer cells involves multiple enhancer elements and E2-mediated transactivation does not require direct binding of the estrogen receptor with promoter DNA.\"],\"interaction score\":[\"2.172883\"],\"pmid counts\":[\"27\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "DLST normalized on bases of \"E2\" mention for TG---{\"_\":[\"_\"],\"tf\":[\"ESR1\"],\"tg\":[\"DLST\"],\"validation\":[\"V\"],\"comment 1\":[\"DLST normalized on bases of \\\"E2\\\" mention for TG\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Thus, hormone regulation of bcl-2 gene expression in breast cancer cells involves multiple enhancer elements and E2-mediated transactivation does not require direct binding of the estrogen receptor with promoter DNA.\"],\"interaction score\":[\"2.172883\"],\"pmid counts\":[\"27\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"DLST\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Thus\",\"hormone regulation of bcl-2 gene expression in breast cancer cells involves multiple enhancer elements and E2-mediated transactivation does not require direct binding of the estrogen receptor with promoter DNA.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is activin A|most likely correctly normalized---{\"tf associated gene name\":[\"MYCN\"],\"tg associated gene name\":[\"INHBA\"],\"tf text\":[\"N-Myc\"],\"tg text\":[],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"TG is activin A\",\"most likely correctly normalized\"],\"sentence\":[\"Down-regulation appears to involve interaction of the N-Myc protein with the activin A promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is activin A|most likely correctly normalized---{\"tf associated gene name\":[\"MYCN\"],\"tg associated gene name\":[\"INHBA\"],\"tf text\":[\"N-Myc\"],\"tg text\":[],\"1\":[\"r\"],\"0\":[],\"negation\":[],\"comment\":[\"TG is activin A\",\"most likely correctly normalized\"],\"sentence\":[\"Down-regulation appears to involve interaction of the N-Myc protein with the activin A promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MYCN\"],\"target gene (associated gene name)\":[\"INHBA\"],\"valid\":[\"1\"],\"sign\":[\"-\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Down-regulation appears to involve interaction of the N-Myc protein with the activin A promoter.\"],\"Valid\":[true],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is FNtau---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[\"TG is FNtau\"],\"sentence\":[\"Cotransfection of CBP with c-jun and/or Ets-2\",\"but not with c-fos\",\"further increased oIFNtau gene transactivation although amounts of c-jun and c-fos expression\",\"resulting from expression vectors\",\"were similar.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is FNtau---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"CREBBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[\"TG is FNtau\"],\"sentence\":[\"Cotransfection of CBP with c-jun and/or Ets-2\",\"but not with c-fos\",\"further increased oIFNtau gene transactivation although amounts of c-jun and c-fos expression\",\"resulting from expression vectors\",\"were similar.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"CREBBP\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"v\"],\"negation\":[],\"sentence\":[\"Cotransfection of CBP with c-jun and/or Ets-2\",\"but not with c-fos\",\"further increased oIFNtau gene transactivation although amounts of c-jun and c-fos expression\",\"resulting from expression vectors\",\"were similar.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"0.30304\"],\"pmid counts\":[\"11\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"210\"],\"sentence pair density\":[\"0.00671140939597315\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"CREBBP\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Cotransfection of CBP with c-jun and/or Ets-2\",\"but not with c-fos\",\"further increased oIFNtau gene transactivation although amounts of c-jun and c-fos expression\",\"resulting from expression vectors\",\"were similar.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "example with signalling prot---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"example with signalling prot\"],\"sentence\":[\"Electrophoretic mobility shift assay and chromatin immunoprecipitation assay confirmed binding of Sp-1 to the VEGF promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "example with signalling prot---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"VEGFA\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"example with signalling prot\"],\"sentence\":[\"Electrophoretic mobility shift assay and chromatin immunoprecipitation assay confirmed binding of Sp-1 to the VEGF promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"VEGFA\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Electrophoretic mobility shift assay and chromatin immunoprecipitation assay confirmed binding of Sp-1 to the VEGF promoter.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is lactase---{\"tf associated gene name\":[\"POU2F1\"],\"tg associated gene name\":[\"GLB1\"],\"tf text\":[\"Oct-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is lactase\"],\"sentence\":[\"The data suggest that the binding of Oct-1 to the T-13\",\"910 variant directs increased lactase promoter activity and this might provide an explanation for the lactase persistence phenotype in the human population.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is lactase---{\"tf associated gene name\":[\"POU2F1\"],\"tg associated gene name\":[\"GLB1\"],\"tf text\":[\"Oct-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is lactase\"],\"sentence\":[\"The data suggest that the binding of Oct-1 to the T-13\",\"910 variant directs increased lactase promoter activity and this might provide an explanation for the lactase persistence phenotype in the human population.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"POU2F1\"],\"target gene (associated gene name)\":[\"GLB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"a\"],\"negation\":[],\"sentence\":[\"The data suggest that the binding of Oct-1 to the T-13\",\"910 variant directs increased lactase promoter activity and this might provide an explanation for the lactase persistence phenotype in the human population.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.813721\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"211\"],\"sentence pair density\":[\"0.00684931506849315\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"POU2F1\"],\"target gene (associated gene name)\":[\"GLB1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The data suggest that the binding of Oct-1 to the T-13\",\"910 variant directs increased lactase promoter activity and this might provide an explanation for the lactase persistence phenotype in the human population.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "indirect---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"MAOA\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"monoamine oxidase A\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"indirect\"],\"sentence\":[\"In summary\",\"this study shows that glucocorticoid enhances monoamine oxidase A gene expression by 1) regulation of R1 translocation; 2) direct interaction of the glucocorticoid receptor with the third glucocorticoid/androgen response element; and 3) indirect interaction of glucocorticoid receptor with the Sp1 or R1 transcription factor on Sp1-binding sites of the MAO A promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "indirect---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"MAOA\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"monoamine oxidase A\"],\"1\":[\"u\"],\"0\":[],\"negation\":[],\"comment\":[\"indirect\"],\"sentence\":[\"In summary\",\"this study shows that glucocorticoid enhances monoamine oxidase A gene expression by 1) regulation of R1 translocation; 2) direct interaction of the glucocorticoid receptor with the third glucocorticoid/androgen response element; and 3) indirect interaction of glucocorticoid receptor with the Sp1 or R1 transcription factor on Sp1-binding sites of the MAO A promoter.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"MAOA\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In summary\",\"this study shows that glucocorticoid enhances monoamine oxidase A gene expression by 1) regulation of R1 translocation; 2) direct interaction of the glucocorticoid receptor with the third glucocorticoid/androgen response element; and 3) indirect interaction of glucocorticoid receptor with the Sp1 or R1 transcription factor on Sp1-binding sites of the MAO A promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "AR-problem; ARE1/2 most likely is not AR, although AR-boinding elements could have ben meant---{\"_\":[\"_\"],\"tf\":[\"AR\"],\"tg\":[\"MME\"],\"validation\":[\"V\"],\"comment 1\":[\"AR-problem; ARE1/2 most likely is not AR, although AR-boinding elements could have ben meant\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We next showed that both ARE1 and ARE2 are involved in the transcriptional regulation of the NEP gene, demonstrating in vitro and in vivo binding with AR as determined by electrophoretic mobility gel shift and chromatin immunoprecipitation (ChIP) assays, Furthermore, ARE1 and ARE2 mediate coordinated androgen-inducibility in both an SV40 promoter and the native NEP type 1 promoter.\"],\"interaction score\":[\"6.827336\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "AR-problem; ARE1/2 most likely is not AR, although AR-boinding elements could have ben meant---{\"_\":[\"_\"],\"tf\":[\"AR\"],\"tg\":[\"MME\"],\"validation\":[\"V\"],\"comment 1\":[\"AR-problem; ARE1/2 most likely is not AR, although AR-boinding elements could have ben meant\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We next showed that both ARE1 and ARE2 are involved in the transcriptional regulation of the NEP gene, demonstrating in vitro and in vivo binding with AR as determined by electrophoretic mobility gel shift and chromatin immunoprecipitation (ChIP) assays, Furthermore, ARE1 and ARE2 mediate coordinated androgen-inducibility in both an SV40 promoter and the native NEP type 1 promoter.\"],\"interaction score\":[\"6.827336\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AR\"],\"target gene (associated gene name)\":[\"MME\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We next showed that both ARE1 and ARE2 are involved in the transcriptional regulation of the NEP gene\",\"demonstrating in vitro and in vivo binding with AR as determined by electrophoretic mobility gel shift and chromatin immunoprecipitation (ChIP) assays\",\"Furthermore\",\"ARE1 and ARE2 mediate coordinated androgen-inducibility in both an SV40 promoter and the native NEP type 1 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ER stress---{\"_\":[\"_\"],\"tf\":[\"ESR1\"],\"tg\":[\"INS\"],\"validation\":[\"V\"],\"comment 1\":[\"ER stress\"],\"comment 2\":[],\"\":[],\"sentence\":[\"In contrast, chronic exposure of beta cells to high glucose causes ER stress and hyperactivation of IRE1, leading to the suppression of insulin gene expression.\"],\"interaction score\":[\"2.760233\"],\"pmid counts\":[\"6\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "ER stress---{\"_\":[\"_\"],\"tf\":[\"ESR1\"],\"tg\":[\"INS\"],\"validation\":[\"V\"],\"comment 1\":[\"ER stress\"],\"comment 2\":[],\"\":[],\"sentence\":[\"In contrast, chronic exposure of beta cells to high glucose causes ER stress and hyperactivation of IRE1, leading to the suppression of insulin gene expression.\"],\"interaction score\":[\"2.760233\"],\"pmid counts\":[\"6\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"INS\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In contrast\",\"chronic exposure of beta cells to high glucose causes ER stress and hyperactivation of IRE1\",\"leading to the suppression of insulin gene expression.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "valid CAT---{\"_\":[\"_\"],\"tf\":[\"PPARG\"],\"tg\":[\"CAT\"],\"validation\":[\"V\"],\"comment 1\":[\"valid CAT\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Moreover, PPARgamma regulates ROS through its coordinated transcriptional control of proteins and enzymes involved in ROS scavenging, including uncoupling protein 2, catalase, and copper zinc superoxide dismutase.\"],\"interaction score\":[\"2.452095\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "valid CAT---{\"_\":[\"_\"],\"tf\":[\"PPARG\"],\"tg\":[\"CAT\"],\"validation\":[\"V\"],\"comment 1\":[\"valid CAT\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Moreover, PPARgamma regulates ROS through its coordinated transcriptional control of proteins and enzymes involved in ROS scavenging, including uncoupling protein 2, catalase, and copper zinc superoxide dismutase.\"],\"interaction score\":[\"2.452095\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PPARG\"],\"target gene (associated gene name)\":[\"CAT\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Moreover\",\"PPARgamma regulates ROS through its coordinated transcriptional control of proteins and enzymes involved in ROS scavenging\",\"including uncoupling protein 2\",\"catalase\",\"and copper zinc superoxide dismutase.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CD8+ cells frequently cause misnormalizations? ---{\"_\":[\"_\"],\"tf\":[\"ID2\"],\"tg\":[\"CD8A\"],\"validation\":[\"V\"],\"comment 1\":[],\"comment 2\":[\"CD8+ cells frequently cause misnormalizations? \"],\"\":[],\"sentence\":[\"Our data emphasize the importance of Id2 in regulating gene expression by CD8(+) T cells and the magnitude of effector responses, suggesting a mechanism involving Id protein- and E protein-mediated survival and differentiation of mature T cells.\"],\"interaction score\":[\"4.602781\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "CD8+ cells frequently cause misnormalizations? ---{\"_\":[\"_\"],\"tf\":[\"ID2\"],\"tg\":[\"CD8A\"],\"validation\":[\"V\"],\"comment 1\":[],\"comment 2\":[\"CD8+ cells frequently cause misnormalizations? \"],\"\":[],\"sentence\":[\"Our data emphasize the importance of Id2 in regulating gene expression by CD8(+) T cells and the magnitude of effector responses, suggesting a mechanism involving Id protein- and E protein-mediated survival and differentiation of mature T cells.\"],\"interaction score\":[\"4.602781\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ID2\"],\"target gene (associated gene name)\":[\"CD8A\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Our data emphasize the importance of Id2 in regulating gene expression by CD8(+) T cells and the magnitude of effector responses\",\"suggesting a mechanism involving Id protein- and E protein-mediated survival and differentiation of mature T cells.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "IRF6 - LPS---{\"_\":[\"_\"],\"tf\":[\"IRF6\"],\"tg\":[\"TREM1\"],\"validation\":[\"V\"],\"comment 1\":[\"IRF6 - LPS\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Treatment of cells with NF-kappaB inhibitors abolished the expression of message of TREM-1 induced by LPS and P. aeruginosa.\"],\"interaction score\":[\"2.035370\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "IRF6 - LPS---{\"_\":[\"_\"],\"tf\":[\"IRF6\"],\"tg\":[\"TREM1\"],\"validation\":[\"V\"],\"comment 1\":[\"IRF6 - LPS\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Treatment of cells with NF-kappaB inhibitors abolished the expression of message of TREM-1 induced by LPS and P. aeruginosa.\"],\"interaction score\":[\"2.035370\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF6\"],\"target gene (associated gene name)\":[\"TREM1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Treatment of cells with NF-kappaB inhibitors abolished the expression of message of TREM-1 induced by LPS and P. aeruginosa.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "example that mentiooning of \"element\" may not be decisively weakening---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"PLAT\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[\"example that mentiooning of \\\"element\\\" may not be decisively weakening\"],\"sentence\":[\"Mitogen-activated protein kinase regulates FSH-induced expression of tissue-type plasminogen activator through an activator protein 1 response element.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "example that mentiooning of \"element\" may not be decisively weakening---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"PLAT\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[\"example that mentiooning of \\\"element\\\" may not be decisively weakening\"],\"sentence\":[\"Mitogen-activated protein kinase regulates FSH-induced expression of tissue-type plasminogen activator through an activator protein 1 response element.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"PLAT\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Mitogen-activated protein kinase regulates FSH-induced expression of tissue-type plasminogen activator through an activator protein 1 response element.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "example of sentence where upstream signalling regulator was not implied in TF-TG---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"SREBF1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"example of sentence where upstream signalling regulator was not implied in TF-TG\"],\"sentence\":[\"Promoter analysis indicated that PKCbeta activates SREBP-1c promoter through replacement of Sp3 by Sp1 for binding to the GC box in the sterol regulatory element (SRE) complex\",\"a key cis-element of SREBP-1c promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "example of sentence where upstream signalling regulator was not implied in TF-TG---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"SREBF1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[],\"negation\":[],\"comment\":[\"example of sentence where upstream signalling regulator was not implied in TF-TG\"],\"sentence\":[\"Promoter analysis indicated that PKCbeta activates SREBP-1c promoter through replacement of Sp3 by Sp1 for binding to the GC box in the sterol regulatory element (SRE) complex\",\"a key cis-element of SREBP-1c promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"SREBF1\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Promoter analysis indicated that PKCbeta activates SREBP-1c promoter through replacement of Sp3 by Sp1 for binding to the GC box in the sterol regulatory element (SRE) complex\",\"a key cis-element of SREBP-1c promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "IRF6 - LPS---{\"_\":[\"_\"],\"tf\":[\"IRF6\"],\"tg\":[\"IL6\"],\"validation\":[\"V\"],\"comment 1\":[\"IRF6 - LPS\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Chromatin immunoprecipitation (ChIP) using an anti-Notch1 antibody demonstrated that Notch1 is associated with the IL-6 promoter in RAW264.7 cells activated by LPS/IFN- but not in unstimulated cells.\"],\"interaction score\":[\"3.222414\"],\"pmid counts\":[\"29\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "IRF6 - LPS---{\"_\":[\"_\"],\"tf\":[\"IRF6\"],\"tg\":[\"IL6\"],\"validation\":[\"V\"],\"comment 1\":[\"IRF6 - LPS\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Chromatin immunoprecipitation (ChIP) using an anti-Notch1 antibody demonstrated that Notch1 is associated with the IL-6 promoter in RAW264.7 cells activated by LPS/IFN- but not in unstimulated cells.\"],\"interaction score\":[\"3.222414\"],\"pmid counts\":[\"29\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF6\"],\"target gene (associated gene name)\":[\"IL6\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Chromatin immunoprecipitation (ChIP) using an anti-Notch1 antibody demonstrated that Notch1 is associated with the IL-6 promoter in RAW264.7 cells activated by LPS/IFN- but not in unstimulated cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "indirect---{\"_\":[\"_\"],\"tf\":[\"IRF3\"],\"tg\":[\"IFNA1\"],\"validation\":[\"V\"],\"comment 1\":[\"indirect\"],\"comment 2\":[],\"\":[],\"sentence\":[\"IFN-beta is transcriptionally upregulated by interferon regulatory factor 3 (IRF-3), NF-kB and activator protein-1 (AP-1), and the binding of IFN-beta to IFN-alpha/beta receptor (IFNAR) stimulates the transcription of IFN-alpha genes or other interferon stimulated genes (ISGs), which induces host antiviral activities, whereas host transcription suppression including IFN-beta gene by NSs prevents the gene upregulations of those ISGs in response to viral replication although IRF-3, NF-kB and activator protein-1 (AP-1) can be activated by RVFV7.\"],\"interaction score\":[\"5.128520\"],\"pmid counts\":[\"22\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "indirect---{\"_\":[\"_\"],\"tf\":[\"IRF3\"],\"tg\":[\"IFNA1\"],\"validation\":[\"V\"],\"comment 1\":[\"indirect\"],\"comment 2\":[],\"\":[],\"sentence\":[\"IFN-beta is transcriptionally upregulated by interferon regulatory factor 3 (IRF-3), NF-kB and activator protein-1 (AP-1), and the binding of IFN-beta to IFN-alpha/beta receptor (IFNAR) stimulates the transcription of IFN-alpha genes or other interferon stimulated genes (ISGs), which induces host antiviral activities, whereas host transcription suppression including IFN-beta gene by NSs prevents the gene upregulations of those ISGs in response to viral replication although IRF-3, NF-kB and activator protein-1 (AP-1) can be activated by RVFV7.\"],\"interaction score\":[\"5.128520\"],\"pmid counts\":[\"22\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"IRF3\"],\"target gene (associated gene name)\":[\"IFNA1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"IFN-beta is transcriptionally upregulated by interferon regulatory factor 3 (IRF-3)\",\"NF-kB and activator protein-1 (AP-1)\",\"and the binding of IFN-beta to IFN-alpha/beta receptor (IFNAR) stimulates the transcription of IFN-alpha genes or other interferon stimulated genes (ISGs)\",\"which induces host antiviral activities\",\"whereas host transcription suppression including IFN-beta gene by NSs prevents the gene upregulations of those ISGs in response to viral replication although IRF-3\",\"NF-kB and activator protein-1 (AP-1) can be activated by RVFV7.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "EIF3K-problem---{\"_\":[\"_\"],\"tf\":[\"YY1\"],\"tg\":[\"EIF3K\"],\"validation\":[\"V\"],\"comment 1\":[\"EIF3K-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"For example, YY1 represses the expression of muscle-specific genes and the degradation of YY1 is required for myocyte differentiation.\"],\"interaction score\":[\"4.510794\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "EIF3K-problem---{\"_\":[\"_\"],\"tf\":[\"YY1\"],\"tg\":[\"EIF3K\"],\"validation\":[\"V\"],\"comment 1\":[\"EIF3K-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"For example, YY1 represses the expression of muscle-specific genes and the degradation of YY1 is required for myocyte differentiation.\"],\"interaction score\":[\"4.510794\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"YY1\"],\"target gene (associated gene name)\":[\"EIF3K\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"For example\",\"YY1 represses the expression of muscle-specific genes and the degradation of YY1 is required for myocyte differentiation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG alpha---{\"_\":[\"_\"],\"tf\":[\"THRA\"],\"tg\":[\"TPM1\"],\"validation\":[\"V\"],\"comment 1\":[\"TG alpha\"],\"comment 2\":[],\"\":[],\"sentence\":[\"These studies suggest a mechanism for negative regulation in which the thyroid hormone receptor interacts with the alpha gene promoter immediately downstream of the TATA box to inhibit transcription.\"],\"interaction score\":[\"2.775672\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG alpha---{\"_\":[\"_\"],\"tf\":[\"THRA\"],\"tg\":[\"TPM1\"],\"validation\":[\"V\"],\"comment 1\":[\"TG alpha\"],\"comment 2\":[],\"\":[],\"sentence\":[\"These studies suggest a mechanism for negative regulation in which the thyroid hormone receptor interacts with the alpha gene promoter immediately downstream of the TATA box to inhibit transcription.\"],\"interaction score\":[\"2.775672\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"THRA\"],\"target gene (associated gene name)\":[\"TPM1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These studies suggest a mechanism for negative regulation in which the thyroid hormone receptor interacts with the alpha gene promoter immediately downstream of the TATA box to inhibit transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain about normalization of TG hINV---{\"tf associated gene name\":[\"JUND\"],\"tg associated gene name\":[\"INVS\"],\"tf text\":[\"junD\"],\"tg text\":[\"hINV\"],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about normalization of TG hINV\"],\"sentence\":[\"We conclude that (i) two AP1 sites in the hINV promoter are important elements required for keratinocyte-specific expression\",\"(ii) these AP1-1 sites mediate the phorbol ester-dependent increase in promoter activity\",\"and (iii) Fra-1\",\"junB\",\"and junD may be important regulators of hINV expression in epidermis.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "uncertain about normalization of TG hINV---{\"tf associated gene name\":[\"JUND\"],\"tg associated gene name\":[\"INVS\"],\"tf text\":[\"junD\"],\"tg text\":[\"hINV\"],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about normalization of TG hINV\"],\"sentence\":[\"We conclude that (i) two AP1 sites in the hINV promoter are important elements required for keratinocyte-specific expression\",\"(ii) these AP1-1 sites mediate the phorbol ester-dependent increase in promoter activity\",\"and (iii) Fra-1\",\"junB\",\"and junD may be important regulators of hINV expression in epidermis.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUND\"],\"target gene (associated gene name)\":[\"INVS\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"v\"],\"negation\":[],\"sentence\":[\"We conclude that (i) two AP1 sites in the hINV promoter are important elements required for keratinocyte-specific expression\",\"(ii) these AP1-1 sites mediate the phorbol ester-dependent increase in promoter activity\",\"and (iii) Fra-1\",\"junB\",\"and junD may be important regulators of hINV expression in epidermis.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"5.329124\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"4\"],\"sentence length\":[\"308\"],\"sentence pair density\":[\"0.029126213592233\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUND\"],\"target gene (associated gene name)\":[\"INVS\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"corrected\"],\"sentence\":[\"We conclude that (i) two AP1 sites in the hINV promoter are important elements required for keratinocyte-specific expression\",\"(ii) these AP1-1 sites mediate the phorbol ester-dependent increase in promoter activity\",\"and (iii) Fra-1\",\"junB\",\"and junD may be important regulators of hINV expression in epidermis.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[\"corrected\"]}---|---{\"transcription factor (associated gene name)\":[\"JUND\"],\"target gene (associated gene name)\":[\"INVS\"],\"valid\":[\"1\"],\"sign\":[\"+\"],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We conclude that (i) two AP1 sites in the hINV promoter are important elements required for keratinocyte-specific expression\",\"(ii) these AP1-1 sites mediate the phorbol ester-dependent increase in promoter activity\",\"and (iii) Fra-1\",\"junB\",\"and junD may be important regulators of hINV expression in epidermis.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "maybe correct TF-TG identified|but relation unclear from sentence---{\"tf associated gene name\":[\"MXD1\"],\"tg associated gene name\":[\"HSPA4\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[\"maybe correct TF-TG identified\",\"but relation unclear from sentence\"],\"sentence\":[\"Replacement of the normal DPP promoter with a heterologous (hsp70) promoter fails to restore DPP-dependent responses in Mad mutant midguts.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "maybe correct TF-TG identified|but relation unclear from sentence---{\"tf associated gene name\":[\"MXD1\"],\"tg associated gene name\":[\"HSPA4\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[],\"negation\":[],\"comment\":[\"maybe correct TF-TG identified\",\"but relation unclear from sentence\"],\"sentence\":[\"Replacement of the normal DPP promoter with a heterologous (hsp70) promoter fails to restore DPP-dependent responses in Mad mutant midguts.\"],\"Valid\":[true],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MXD1\"],\"target gene (associated gene name)\":[\"HSPA4\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Replacement of the normal DPP promoter with a heterologous (hsp70) promoter fails to restore DPP-dependent responses in Mad mutant midguts.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain normalization of TG (MT-I )---{\"tf associated gene name\":[\"USF1\"],\"tg associated gene name\":[\"MT1A\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain normalization of TG (MT-I )\"],\"sentence\":[\"The data further suggest that USF interacts with ARE-binding proteins to influence MT-I gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "uncertain normalization of TG (MT-I )---{\"tf associated gene name\":[\"USF1\"],\"tg associated gene name\":[\"MT1A\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain normalization of TG (MT-I )\"],\"sentence\":[\"The data further suggest that USF interacts with ARE-binding proteins to influence MT-I gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"USF1\"],\"target gene (associated gene name)\":[\"MT1A\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"v\"],\"negation\":[],\"sentence\":[\"The data further suggest that USF interacts with ARE-binding proteins to influence MT-I gene expression.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.071356\"],\"pmid counts\":[\"9\"],\"sentence counts\":[\"4\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"104\"],\"sentence pair density\":[\"0.0180722891566265\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"USF1\"],\"target gene (associated gene name)\":[\"MT1A\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The data further suggest that USF interacts with ARE-binding proteins to influence MT-I gene expression.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG normalization due to P450, CYP286 is not aromatase; i.g. get a lot of false positives with P450; i.e. NER chooses a \"smaller unit\" for recognition, when \"forced\" - compare correct normalization in sentence: \"9973254:0:NR5A1:CYP19A1\"---{\"_\":[\"_\"],\"tf\":[\"NR5A1\"],\"tg\":[\"CYP2B6\"],\"validation\":[\"V\"],\"comment 1\":[\"TG normalization due to P450, CYP286 is not aromatase; i.g. get a lot of false positives with P450; i.e. NER chooses a \\\"smaller unit\\\" for recognition, when \\\"forced\\\" - compare correct normalization in sentence: \\\"9973254:0:NR5A1:CYP19A1\\\"\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Stimulation of aromatase P450 promoter (II) activity in endometriosis and its inhibition in endometrium are regulated by competitive binding of steroidogenic factor-1 and chicken ovalbumin upstream promoter transcription factor to the same cis-acting element.\"],\"interaction score\":[\"3.441786\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG normalization due to P450, CYP286 is not aromatase; i.g. get a lot of false positives with P450; i.e. NER chooses a \"smaller unit\" for recognition, when \"forced\" - compare correct normalization in sentence: \"9973254:0:NR5A1:CYP19A1\"---{\"_\":[\"_\"],\"tf\":[\"NR5A1\"],\"tg\":[\"CYP2B6\"],\"validation\":[\"V\"],\"comment 1\":[\"TG normalization due to P450, CYP286 is not aromatase; i.g. get a lot of false positives with P450; i.e. NER chooses a \\\"smaller unit\\\" for recognition, when \\\"forced\\\" - compare correct normalization in sentence: \\\"9973254:0:NR5A1:CYP19A1\\\"\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Stimulation of aromatase P450 promoter (II) activity in endometriosis and its inhibition in endometrium are regulated by competitive binding of steroidogenic factor-1 and chicken ovalbumin upstream promoter transcription factor to the same cis-acting element.\"],\"interaction score\":[\"3.441786\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR5A1\"],\"target gene (associated gene name)\":[\"CYP2B6\"],\"valid\":[\"1\"],\"sign\":[],\"negation\":[],\"tf_is_fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Stimulation of aromatase P450 promoter (II) activity in endometriosis and its inhibition in endometrium are regulated by competitive binding of steroidogenic factor-1 and chicken ovalbumin upstream promoter transcription factor to the same cis-acting element.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GRE|but probably not TRE---{\"tf associated gene name\":[\"VSX2\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"GRE\",\"but probably not TRE\"],\"sentence\":[\"Ral activation of these responses is likely through an as yet uncharacterized effector pathway\",\"as we find activation of NF-kappaB and the cyclin D1 promoter by Ral is independent of association of Ral with active phospholipase D1 or Ral-binding protein 1\",\"two proteins proposed to mediate Ral function in cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "GRE|but probably not TRE---{\"tf associated gene name\":[\"VSX2\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"GRE\",\"but probably not TRE\"],\"sentence\":[\"Ral activation of these responses is likely through an as yet uncharacterized effector pathway\",\"as we find activation of NF-kappaB and the cyclin D1 promoter by Ral is independent of association of Ral with active phospholipase D1 or Ral-binding protein 1\",\"two proteins proposed to mediate Ral function in cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"VSX2\"],\"target gene (associated gene name)\":[\"CCND1\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"Ral activation of these responses is likely through an as yet uncharacterized effector pathway\",\"as we find activation of NF-kappaB and the cyclin D1 promoter by Ral is independent of association of Ral with active phospholipase D1 or Ral-binding protein 1\",\"two proteins proposed to mediate Ral function in cells\",\"\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"auto-regulation\":[],\"interaction score\":[\"0.897505\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"312\"],\"sentence pair density\":[\"0.00641025641025641\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"In epithelial cells\",\"when RB family factors were inactivated by SV40 T antigen (LT)\",\"p21 expression was strongly repressed\",\"whereas its expression was not affected when the cells were transformed by a mutated LT leaving RB active but inactivating p53.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"CDKN1A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"In epithelial cells\",\"when RB family factors were inactivated by SV40 T antigen (LT)\",\"p21 expression was strongly repressed\",\"whereas its expression was not affected when the cells were transformed by a mutated LT leaving RB active but inactivating p53.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"TP53\"],\"target gene (associated gene name)\":[\"CDKN1A\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"In epithelial cells\",\"when RB family factors were inactivated by SV40 T antigen (LT)\",\"p21 expression was strongly repressed\",\"whereas its expression was not affected when the cells were transformed by a mutated LT leaving RB active but inactivating p53\",\"\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"auto-regulation\":[],\"interaction score\":[\"1.232532\"],\"pmid counts\":[\"478\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"251\"],\"sentence pair density\":[\"0.00398406374501992\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is BCL6|TG is ATF3 (and others)---{\"tf associated gene name\":[\"ATF3\"],\"tg associated gene name\":[\"BCL6\"],\"tf text\":[\"activating transcription factor 3\"],\"tg text\":[\"BCL6\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is BCL6\",\"TG is ATF3 (and others)\"],\"sentence\":[\"The levels of expression of activating transcription factor 3\",\"prefoldin 4\",\"and retinoblastoma-binding protein 7 significantly increased in accordance with that of BCL6\",\"suggesting that Bcl-6 may act as a transcriptional activator.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is BCL6|TG is ATF3 (and others)---{\"tf associated gene name\":[\"ATF3\"],\"tg associated gene name\":[\"BCL6\"],\"tf text\":[\"activating transcription factor 3\"],\"tg text\":[\"BCL6\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is BCL6\",\"TG is ATF3 (and others)\"],\"sentence\":[\"The levels of expression of activating transcription factor 3\",\"prefoldin 4\",\"and retinoblastoma-binding protein 7 significantly increased in accordance with that of BCL6\",\"suggesting that Bcl-6 may act as a transcriptional activator.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ATF3\"],\"target gene (associated gene name)\":[\"BCL6\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The levels of expression of activating transcription factor 3\",\"prefoldin 4\",\"and retinoblastoma-binding protein 7 significantly increased in accordance with that of BCL6\",\"suggesting that Bcl-6 may act as a transcriptional activator\",\"\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"ATF3\"],\"target gene (associated gene name)\":[\"BCL6\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"The levels of expression of activating transcription factor 3\",\"prefoldin 4\",\"and retinoblastoma-binding protein 7 significantly increased in accordance with that of BCL6\",\"suggesting that Bcl-6 may act as a transcriptional activator\",\"\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"auto-regulation\":[],\"interaction score\":[\"0.058203\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"5\"],\"sentence length\":[\"231\"],\"sentence pair density\":[\"0.0216450216450216\"],\"ressa cluster\":[\"5\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Retinoblastoma tumor suppressor p105 protein ---{\"tf associated gene name\":[\"NFKB1\"],\"tg associated gene name\":[\"CCNE1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Retinoblastoma tumor suppressor p105 protein \"],\"sentence\":[\"Conversely\",\"the cyclin E promoter was repressed by wild-type Retinoblastoma tumor suppressor p105 protein (pRB) and by a dominant negative p300 mutant (DN p300) that lacks histone acetyltransferase activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Retinoblastoma tumor suppressor p105 protein ---{\"tf associated gene name\":[\"NFKB1\"],\"tg associated gene name\":[\"CCNE1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Retinoblastoma tumor suppressor p105 protein \"],\"sentence\":[\"Conversely\",\"the cyclin E promoter was repressed by wild-type Retinoblastoma tumor suppressor p105 protein (pRB) and by a dominant negative p300 mutant (DN p300) that lacks histone acetyltransferase activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB1\"],\"target gene (associated gene name)\":[\"CCNE1\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Conversely\",\"the cyclin E promoter was repressed by wild-type Retinoblastoma tumor suppressor p105 protein (pRB) and by a dominant negative p300 mutant (DN p300) that lacks histone acetyltransferase activity\",\"\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---|---{\"transcription factor (associated gene name)\":[\"NFKB1\"],\"target gene (associated gene name)\":[\"CCNE1\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"Conversely\",\"the cyclin E promoter was repressed by wild-type Retinoblastoma tumor suppressor p105 protein (pRB) and by a dominant negative p300 mutant (DN p300) that lacks histone acetyltransferase activity\",\"\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"auto-regulation\":[],\"interaction score\":[\"2.258572\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"207\"],\"sentence pair density\":[\"0.00966183574879227\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Ku---{\"tf associated gene name\":[\"ZIC2\"],\"tg associated gene name\":[\"SLC3A2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is Ku\"],\"sentence\":[\"The Ku protein complex interacts with YY1\",\"is up-regulated in human heart failure\",\"and represses alpha myosin heavy-chain gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Ku---{\"tf associated gene name\":[\"ZIC2\"],\"tg associated gene name\":[\"SLC3A2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is Ku\"],\"sentence\":[\"The Ku protein complex interacts with YY1\",\"is up-regulated in human heart failure\",\"and represses alpha myosin heavy-chain gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ZIC2\"],\"target gene (associated gene name)\":[\"SLC3A2\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"The Ku protein complex interacts with YY1\",\"is up-regulated in human heart failure\",\"and represses alpha myosin heavy-chain gene expression\",\"\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"auto-regulation\":[],\"interaction score\":[\"1.393724\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"4\"],\"sentence length\":[\"138\"],\"sentence pair density\":[\"0.0289855072463768\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is ATF3---{\"tf associated gene name\":[\"SMAD3\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"TG is ATF3\"],\"sentence\":[\"We demonstrate that TGF-beta increased ATF3 protein levels\",\"as did transient overexpression of Smad3E.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is ATF3---{\"tf associated gene name\":[\"SMAD3\"],\"tg associated gene name\":[\"TGFB1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"TG is ATF3\"],\"sentence\":[\"We demonstrate that TGF-beta increased ATF3 protein levels\",\"as did transient overexpression of Smad3E.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SMAD3\"],\"target gene (associated gene name)\":[\"TGFB1\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"We demonstrate that TGF-beta increased ATF3 protein levels\",\"as did transient overexpression of Smad3E\",\"\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"auto-regulation\":[],\"interaction score\":[\"0.068583\"],\"pmid counts\":[\"98\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"102\"],\"sentence pair density\":[\"0.00980392156862745\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no PAX3---{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"GIPR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[\"no PAX3\"],\"sentence\":[\"The Sp transcription factor family is involved in the cellular expression of the human GIP-R gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no PAX3---{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"GIPR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[\"no PAX3\"],\"sentence\":[\"The Sp transcription factor family is involved in the cellular expression of the human GIP-R gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PAX3\"],\"target gene (associated gene name)\":[\"GIPR\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"The Sp transcription factor family is involved in the cellular expression of the human GIP-R gene promoter\",\"\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"auto-regulation\":[],\"interaction score\":[\"0.157433\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"107\"],\"sentence pair density\":[\"0.00934579439252336\"],\"ressa cluster\":[\"7\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"VDR\"],\"tg associated gene name\":[\"CD4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"VDR(-/-) NOD mice are rachitic and have lower numbers of putative regulator cells [TCR-alpha/beta(+)CD4(-)CD8(-) (natural killer T-cells) and CD4(+)CD25(+) T-cells [in central and peripheral immune organs compared with VDR(+/+) NOD littermates.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"VDR\"],\"tg associated gene name\":[\"CD4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"VDR(-/-) NOD mice are rachitic and have lower numbers of putative regulator cells [TCR-alpha/beta(+)CD4(-)CD8(-) (natural killer T-cells) and CD4(+)CD25(+) T-cells [in central and peripheral immune organs compared with VDR(+/+) NOD littermates.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"VDR\"],\"target gene (associated gene name)\":[\"CD4\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"VDR(-/-) NOD mice are rachitic and have lower numbers of putative regulator cells [TCR-alpha/beta(+)CD4(-)CD8(-) (natural killer T-cells) and CD4(+)CD25(+) T-cells [in central and peripheral immune organs compared with VDR(+/+) NOD littermates\",\"\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"auto-regulation\":[],\"interaction score\":[\"0.808399\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"244\"],\"sentence pair density\":[\"0.0122950819672131\"],\"ressa cluster\":[\"5\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is sod2---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"KRT20\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is sod2\"],\"sentence\":[\"Trimethyl histone H4 lysine 20 (H4K20me3)\",\"acetyl histone H3 lysine 9 (H3K9)\",\"and nuclear transcriptional factor NF-??B p65 and p50 at the retinal sod2 promoter and enhancer were examined by chromatin immunoprecipitation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is sod2---{\"tf associated gene name\":[\"RELA\"],\"tg associated gene name\":[\"KRT20\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is sod2\"],\"sentence\":[\"Trimethyl histone H4 lysine 20 (H4K20me3)\",\"acetyl histone H3 lysine 9 (H3K9)\",\"and nuclear transcriptional factor NF-??B p65 and p50 at the retinal sod2 promoter and enhancer were examined by chromatin immunoprecipitation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RELA\"],\"target gene (associated gene name)\":[\"KRT20\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"Trimethyl histone H4 lysine 20 (H4K20me3)\",\"acetyl histone H3 lysine 9 (H3K9)\",\"and nuclear transcriptional factor NF-B p65 and p50 at the retinal sod2 promoter and enhancer were examined by chromatin immunoprecipitation\",\"\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"auto-regulation\":[],\"interaction score\":[\"0.376468\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"9\"],\"sentence length\":[\"219\"],\"sentence pair density\":[\"0.0410958904109589\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF most likely not named ---{\"tf associated gene name\":[\"PBX1\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF most likely not named \"],\"sentence\":[\"A PAK1 Y3F mutant lacking functional nuclear localization signals decreases PRL-induced cyclin D1 activity by 68%\",\"suggesting that there is another PAK1-dependent mechanism to activate the cyclin D1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF most likely not named ---{\"tf associated gene name\":[\"PBX1\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF most likely not named \"],\"sentence\":[\"A PAK1 Y3F mutant lacking functional nuclear localization signals decreases PRL-induced cyclin D1 activity by 68%\",\"suggesting that there is another PAK1-dependent mechanism to activate the cyclin D1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PBX1\"],\"target gene (associated gene name)\":[\"CCND1\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"A PAK1 Y3F mutant lacking functional nuclear localization signals decreases PRL-induced cyclin D1 activity by 68%\",\"suggesting that there is another PAK1-dependent mechanism to activate the cyclin D1 promoter\",\"\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"auto-regulation\":[],\"interaction score\":[\"2.776678\"],\"pmid counts\":[\"9\"],\"sentence counts\":[\"4\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"208\"],\"sentence pair density\":[\"0.0048076923076923\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "several other TF-TG pairs---{\"tf associated gene name\":[\"MEIS1\"],\"tg associated gene name\":[\"MEIS1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"several other TF-TG pairs\"],\"sentence\":[\"Chromatin immunoprecipitation (ChIP) and luciferase reporter analysis showed that Oct4 enhanced Meis1a expression via direct binding to the Meis1 promoter accompanying histone H3 acetylation and appearance of 5-hydoxymethylcytosine (5hmC)\",\"while Meis1a suppressed Oct4 expression via direct association with the Oct4 promoter together with histone deacetylase 1 (HDAC1).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "several other TF-TG pairs---{\"tf associated gene name\":[\"MEIS1\"],\"tg associated gene name\":[\"MEIS1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"several other TF-TG pairs\"],\"sentence\":[\"Chromatin immunoprecipitation (ChIP) and luciferase reporter analysis showed that Oct4 enhanced Meis1a expression via direct binding to the Meis1 promoter accompanying histone H3 acetylation and appearance of 5-hydoxymethylcytosine (5hmC)\",\"while Meis1a suppressed Oct4 expression via direct association with the Oct4 promoter together with histone deacetylase 1 (HDAC1).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MEIS1\"],\"target gene (associated gene name)\":[\"MEIS1\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"Chromatin immunoprecipitation (ChIP) and luciferase reporter analysis showed that Oct4 enhanced Meis1a expression via direct binding to the Meis1 promoter accompanying histone H3 acetylation and appearance of 5-hydoxymethylcytosine (5hmC)\",\"while Meis1a suppressed Oct4 expression via direct association with the Oct4 promoter together with histone deacetylase 1 (HDAC1)\",\"\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"auto-regulation\":[\"Auto-regulation\"],\"interaction score\":[\"2.102009\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"5\"],\"sentence length\":[\"370\"],\"sentence pair density\":[\"0.0135135135135135\"],\"ressa cluster\":[\"5\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is R1 gene---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CCNE1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is R1 gene\"],\"sentence\":[\"One complex most likely contained Sp1\",\"and another complex showed S-phase-specific binding\",\"suggesting a direct role in the cell-cycle-dependent R1 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is R1 gene---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CCNE1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is R1 gene\"],\"sentence\":[\"One complex most likely contained Sp1\",\"and another complex showed S-phase-specific binding\",\"suggesting a direct role in the cell-cycle-dependent R1 gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SP1\"],\"target gene (associated gene name)\":[\"CCNE1\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"One complex most likely contained Sp1\",\"and another complex showed S-phase-specific binding\",\"suggesting a direct role in the cell-cycle-dependent R1 gene expression\",\"\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"auto-regulation\":[],\"interaction score\":[\"1.763853\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"164\"],\"sentence pair density\":[\"0.0121951219512195\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is chemical (okadaic acid); TG EGR1 ---{\"tf associated gene name\":[\"BCL11A\"],\"tg associated gene name\":[\"EGR1\"],\"tf text\":[],\"tg text\":[\"early growth response 1\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"regulator is chemical (okadaic acid); TG EGR1 \"],\"sentence\":[\"Transcriptional regulation of the early growth response 1 gene in human myeloid leukemia cells by okadaic acid.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "regulator is chemical (okadaic acid); TG EGR1 ---{\"tf associated gene name\":[\"BCL11A\"],\"tg associated gene name\":[\"EGR1\"],\"tf text\":[],\"tg text\":[\"early growth response 1\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"regulator is chemical (okadaic acid); TG EGR1 \"],\"sentence\":[\"Transcriptional regulation of the early growth response 1 gene in human myeloid leukemia cells by okadaic acid.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"BCL11A\"],\"target gene (associated gene name)\":[\"EGR1\"],\"ev\":[\"n\"],\"negation\":[],\"is fusion\":[],\"sentence\":[\"Transcriptional regulation of the early growth response 1 gene in human myeloid leukemia cells by okadaic acid\",\"\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1,6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"auto-regulation\":[],\"interaction score\":[\"0.031315\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"111\"],\"sentence pair density\":[\"0.009009009009009\"],\"ressa cluster\":[\"2\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is SP-A---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"FASLG\"],\"tf text\":[\"glucocorticoid receptor\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is SP-A\"],\"sentence\":[\"Glucocorticoid/glucocorticoid receptor inhibition of surfactant protein-A (SP-A) gene expression in lung type II cells is mediated by repressive changes in histone modification at the SP-A promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is SP-A---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"FASLG\"],\"tf text\":[\"glucocorticoid receptor\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is SP-A\"],\"sentence\":[\"Glucocorticoid/glucocorticoid receptor inhibition of surfactant protein-A (SP-A) gene expression in lung type II cells is mediated by repressive changes in histone modification at the SP-A promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR3C1\"],\"target gene (associated gene name)\":[\"FASLG\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf-fusion\":[],\"qc\":[\"0\"],\"sentence\":[\"Glucocorticoid/glucocorticoid receptor inhibition of surfactant protein-A (SP-A) gene expression in lung type II cells is mediated by repressive changes in histone modification at the SP-A promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF NR3C1 regulates TG FOXO3; TF FOXO3 regulates STK11; TF FOXO3 regulates TG FOXO3---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"STK11\"],\"tf text\":[\"glucocorticoid receptor\"],\"tg text\":[\"LKB1\"],\"1\":[\"u\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF NR3C1 regulates TG FOXO3; TF FOXO3 regulates STK11; TF FOXO3 regulates TG FOXO3\"],\"sentence\":[\"FOXO3 is a glucocorticoid receptor target and regulates LKB1 and its own expression based on cellular AMP levels via a positive autoregulatory loop.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TF NR3C1 regulates TG FOXO3; TF FOXO3 regulates STK11; TF FOXO3 regulates TG FOXO3---{\"tf associated gene name\":[\"NR3C1\"],\"tg associated gene name\":[\"STK11\"],\"tf text\":[\"glucocorticoid receptor\"],\"tg text\":[\"LKB1\"],\"1\":[\"u\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF NR3C1 regulates TG FOXO3; TF FOXO3 regulates STK11; TF FOXO3 regulates TG FOXO3\"],\"sentence\":[\"FOXO3 is a glucocorticoid receptor target and regulates LKB1 and its own expression based on cellular AMP levels via a positive autoregulatory loop.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NR3C1\"],\"target gene (associated gene name)\":[\"STK11\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf-fusion\":[],\"qc\":[\"0\"],\"sentence\":[\"FOXO3 is a glucocorticoid receptor target and regulates LKB1 and its own expression based on cellular AMP levels via a positive autoregulatory loop.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "other TFs---{\"tf associated gene name\":[\"PRRX1\"],\"tg associated gene name\":[\"SPP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"other TFs\"],\"sentence\":[\"Smad3 binds directly to the OPN promoter as a sequence-specific activator\",\"and Smad4 displaces the transcription repressor\",\"Hoxa-9\",\"by formation of Smad4/Hox complex as part of the transcription mechanism in response to TGF-beta stimulation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "other TFs---{\"tf associated gene name\":[\"PRRX1\"],\"tg associated gene name\":[\"SPP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"other TFs\"],\"sentence\":[\"Smad3 binds directly to the OPN promoter as a sequence-specific activator\",\"and Smad4 displaces the transcription repressor\",\"Hoxa-9\",\"by formation of Smad4/Hox complex as part of the transcription mechanism in response to TGF-beta stimulation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PRRX1\"],\"target gene (associated gene name)\":[\"SPP1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf-fusion\":[],\"qc\":[\"0\"],\"sentence\":[\"Smad3 binds directly to the OPN promoter as a sequence-specific activator\",\"and Smad4 displaces the transcription repressor\",\"Hoxa-9\",\"by formation of Smad4/Hox complex as part of the transcription mechanism in response to TGF-beta stimulation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CD4+ problem---{\"_\":[\"_\"],\"tf\":[\"RUNX3\"],\"tg\":[\"CD4\"],\"validation\":[\"N\"],\"comment 1\":[\"CD4+ problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Our results suggest that the balance of Runx3 and GATA3 is one factor that influences the manifestation of CD4(+) cells as the Th1 or Th2 phenotypes.\"],\"interaction score\":[\"2.232960\"],\"pmid counts\":[\"19\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "CD4+ problem---{\"_\":[\"_\"],\"tf\":[\"RUNX3\"],\"tg\":[\"CD4\"],\"validation\":[\"N\"],\"comment 1\":[\"CD4+ problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Our results suggest that the balance of Runx3 and GATA3 is one factor that influences the manifestation of CD4(+) cells as the Th1 or Th2 phenotypes.\"],\"interaction score\":[\"2.232960\"],\"pmid counts\":[\"19\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RUNX3\"],\"target gene (associated gene name)\":[\"CD4\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf-fusion\":[],\"qc\":[\"0\"],\"sentence\":[\"Our results suggest that the balance of Runx3 and GATA3 is one factor that influences the manifestation of CD4(+) cells as the Th1 or Th2 phenotypes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no GRE-relation between these two---{\"tf associated gene name\":[\"RUNX3\"],\"tg associated gene name\":[\"IHH\"],\"tf text\":[\"Runx3\"],\"tg text\":[\"Ihh\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no GRE-relation between these two\"],\"sentence\":[\"A dramatic down-regulation of Runx2 and Runx3 could account for the lack of chondrocyte hypertrophy\",\"while a down-regulation of Ihh expression may be responsible for a significant reduction in chondrocyte proliferation in the mutant stylopod.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no GRE-relation between these two---{\"tf associated gene name\":[\"RUNX3\"],\"tg associated gene name\":[\"IHH\"],\"tf text\":[\"Runx3\"],\"tg text\":[\"Ihh\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no GRE-relation between these two\"],\"sentence\":[\"A dramatic down-regulation of Runx2 and Runx3 could account for the lack of chondrocyte hypertrophy\",\"while a down-regulation of Ihh expression may be responsible for a significant reduction in chondrocyte proliferation in the mutant stylopod.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"RUNX3\"],\"target gene (associated gene name)\":[\"IHH\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf-fusion\":[],\"qc\":[\"0\"],\"sentence\":[\"A dramatic down-regulation of Runx2 and Runx3 could account for the lack of chondrocyte hypertrophy\",\"while a down-regulation of Ihh expression may be responsible for a significant reduction in chondrocyte proliferation in the mutant stylopod.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"SNAI2\"],\"tg\":[\"CDH1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Transfection with c-Myb siRNAs blocked the increase of Slug (SNAI2) and Bcl-2 expression and reversed the decrease in E-cadherin expression induced by TGF- treatment.\"],\"interaction score\":[\"4.048058\"],\"pmid counts\":[\"37\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"SNAI2\"],\"tg\":[\"CDH1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Transfection with c-Myb siRNAs blocked the increase of Slug (SNAI2) and Bcl-2 expression and reversed the decrease in E-cadherin expression induced by TGF- treatment.\"],\"interaction score\":[\"4.048058\"],\"pmid counts\":[\"37\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SNAI2\"],\"target gene (associated gene name)\":[\"CDH1\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"tf-fusion\":[],\"qc\":[\"0\"],\"sentence\":[\"Transfection with c-Myb siRNAs blocked the increase of Slug (SNAI2) and Bcl-2 expression and reversed the decrease in E-cadherin expression induced by TGF- treatment.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is p50 and p52 homodimers; NB! check whether BCL3 is in TFClass---{\"tf associated gene name\":[\"BCL3\"],\"tg associated gene name\":[\"BCL2\"],\"tf text\":[\"Bcl-3\"],\"tg text\":[\"bcl-2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is p50 and p52 homodimers; NB! check whether BCL3 is in TFClass\"],\"sentence\":[\"As p50 and p52 homodimers are devoid of any transactivating domains\",\"we showed that they can transactivate the bcl-2 promoter through association with Bcl-3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is p50 and p52 homodimers; NB! check whether BCL3 is in TFClass---{\"tf associated gene name\":[\"BCL3\"],\"tg associated gene name\":[\"BCL2\"],\"tf text\":[\"Bcl-3\"],\"tg text\":[\"bcl-2\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is p50 and p52 homodimers; NB! check whether BCL3 is in TFClass\"],\"sentence\":[\"As p50 and p52 homodimers are devoid of any transactivating domains\",\"we showed that they can transactivate the bcl-2 promoter through association with Bcl-3.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"valid\":[\"TRUE\"],\"sign\":[\"+\"],\"negation\":[],\"tffusion\":[],\"sentence\":[\"As p50 and p52 homodimers are devoid of any transactivating domains, we showed that they can transactivate the bcl-2 promoter through association with Bcl-3,\"],\"qc\":[\"TRUE\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "T-problem|not TRE---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"T\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"T-problem\",\"not TRE\"],\"sentence\":[\"Expression of NeuT\",\"but not normal Neu\",\"caused transcriptional activation of Ets\",\"AP-1\",\"or NF-kappaB-dependent reporter genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "T-problem|not TRE---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"T\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"T-problem\",\"not TRE\"],\"sentence\":[\"Expression of NeuT\",\"but not normal Neu\",\"caused transcriptional activation of Ets\",\"AP-1\",\"or NF-kappaB-dependent reporter genes.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"T\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Expression of NeuT\",\"but not normal Neu\",\"caused transcriptional activation of Ets\",\"AP-1\",\"or NF-kappaB-dependent reporter genes.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.797087\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"126\"],\"sentence pair density\":[\"0.011049723756906\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"ATF3\"],\"tg\":[\"CCND1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Microarray analysis and luciferase reporter assays indicate that adipokines specifically induce several transcriptional programs involved in promoting tumorigenesis, including increased cell proliferation (IGF2, FOS, JUN, cyclin D1), invasive potential (MMP1, ATF3), survival (A20, NFkappaB), and angiogenesis.\"],\"interaction score\":[\"2.425356\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"ATF3\"],\"tg\":[\"CCND1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Microarray analysis and luciferase reporter assays indicate that adipokines specifically induce several transcriptional programs involved in promoting tumorigenesis, including increased cell proliferation (IGF2, FOS, JUN, cyclin D1), invasive potential (MMP1, ATF3), survival (A20, NFkappaB), and angiogenesis.\"],\"interaction score\":[\"2.425356\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ATF3\"],\"target gene (associated gene name)\":[\"CCND1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Microarray analysis and luciferase reporter assays indicate that adipokines specifically induce several transcriptional programs involved in promoting tumorigenesis\",\"including increased cell proliferation (IGF2\",\"FOS\",\"JUN\",\"cyclin D1)\",\"invasive potential (MMP1\",\"ATF3)\",\"survival (A20\",\"NFkappaB)\",\"and angiogenesis.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.425356\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"9\"],\"sentence length\":[\"310\"],\"sentence pair density\":[\"0.00819672131147541\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is IRF-3---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"MRGPRF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is IRF-3\"],\"sentence\":[\"These results suggested that Rta represses IRF-3 expression by increasing E2F1 binding to the IRF-3 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is IRF-3---{\"tf associated gene name\":[\"E2F1\"],\"tg associated gene name\":[\"MRGPRF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is IRF-3\"],\"sentence\":[\"These results suggested that Rta represses IRF-3 expression by increasing E2F1 binding to the IRF-3 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"E2F1\"],\"target gene (associated gene name)\":[\"MRGPRF\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"These results suggested that Rta represses IRF-3 expression by increasing E2F1 binding to the IRF-3 promoter.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"3.551231\"],\"pmid counts\":[\"2\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"3\"],\"sentence length\":[\"109\"],\"sentence pair density\":[\"0.0072463768115942\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Early growth response-1 (Egr-1) is an immediate-early gene induced by E2 in the rodent uterus and breast cancer cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"EGR1\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[\"Egr-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Early growth response-1 (Egr-1) is an immediate-early gene induced by E2 in the rodent uterus and breast cancer cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"EGR1\"],\"target gene (associated gene name)\":[\"JUN\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Early growth response-1 (Egr-1) is an immediate-early gene induced by E2 in the rodent uterus and breast cancer cells.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.022477\"],\"pmid counts\":[\"14\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"118\"],\"sentence pair density\":[\"0.0256410256410256\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "could not find TF correctly normalized---{\"_\":[\"_\"],\"tf\":[\"ERF\"],\"tg\":[\"SERPINE1\"],\"validation\":[\"N\"],\"comment 1\":[\"could not find TF correctly normalized\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We showed that TFE3 binds to an E-box sequence in PE2, a 56-bp promoter fragment of the PAI-1 promoter, and that mutation of this sequence abolishes both TFE3 binding as well as TGF-beta-dependent activation.\"],\"interaction score\":[\"2.908537\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "could not find TF correctly normalized---{\"_\":[\"_\"],\"tf\":[\"ERF\"],\"tg\":[\"SERPINE1\"],\"validation\":[\"N\"],\"comment 1\":[\"could not find TF correctly normalized\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We showed that TFE3 binds to an E-box sequence in PE2, a 56-bp promoter fragment of the PAI-1 promoter, and that mutation of this sequence abolishes both TFE3 binding as well as TGF-beta-dependent activation.\"],\"interaction score\":[\"2.908537\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ERF\"],\"target gene (associated gene name)\":[\"SERPINE1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"We showed that TFE3 binds to an E-box sequence in PE2\",\"a 56-bp promoter fragment of the PAI-1 promoter\",\"and that mutation of this sequence abolishes both TFE3 binding as well as TGF-beta-dependent activation.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.908537\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"208\"],\"sentence pair density\":[\"0.00947867298578199\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "PPI---{\"_\":[\"_\"],\"tf\":[\"ESR1\"],\"tg\":[\"NR5A2\"],\"validation\":[\"N\"],\"comment 1\":[\"PPI\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Taken together, our findings establish a key role for LRH-1 in the regulation of ER target genes in breast cancer cells and identify a mechanism in which co-operative binding of LRH-1 and ER at oestrogen response elements controls the expression of oestrogen-responsive genes.\"],\"interaction score\":[\"3.622678\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "PPI---{\"_\":[\"_\"],\"tf\":[\"ESR1\"],\"tg\":[\"NR5A2\"],\"validation\":[\"N\"],\"comment 1\":[\"PPI\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Taken together, our findings establish a key role for LRH-1 in the regulation of ER target genes in breast cancer cells and identify a mechanism in which co-operative binding of LRH-1 and ER at oestrogen response elements controls the expression of oestrogen-responsive genes.\"],\"interaction score\":[\"3.622678\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ESR1\"],\"target gene (associated gene name)\":[\"NR5A2\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Taken together\",\"our findings establish a key role for LRH-1 in the regulation of ER target genes in breast cancer cells and identify a mechanism in which co-operative binding of LRH-1 and ER at oestrogen response elements controls the expression of oestrogen-responsive genes.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"3.622678\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"6\"],\"sentence length\":[\"276\"],\"sentence pair density\":[\"0.00806451612903225\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  delta-opioid receptor (wrong normalization because TFs are favoured|also for TG?)---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"YY1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is  delta-opioid receptor (wrong normalization because TFs are favoured\",\"also for TG?)\"],\"sentence\":[\"Transcriptional regulation of mouse delta-opioid receptor gene: role of Ets-1 in the transcriptional activation of mouse delta-opioid receptor gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is  delta-opioid receptor (wrong normalization because TFs are favoured|also for TG?)---{\"tf associated gene name\":[\"ETS1\"],\"tg associated gene name\":[\"YY1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is  delta-opioid receptor (wrong normalization because TFs are favoured\",\"also for TG?)\"],\"sentence\":[\"Transcriptional regulation of mouse delta-opioid receptor gene: role of Ets-1 in the transcriptional activation of mouse delta-opioid receptor gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ETS1\"],\"target gene (associated gene name)\":[\"YY1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Transcriptional regulation of mouse delta-opioid receptor gene: role of Ets-1 in the transcriptional activation of mouse delta-opioid receptor gene.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.821396\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"148\"],\"sentence pair density\":[\"0.0215827338129496\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is LEP---{\"tf associated gene name\":[\"FOSL2\"],\"tg associated gene name\":[\"FOSL2\"],\"tf text\":[\"FOSL2\"],\"tg text\":[\"FOSL2\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is LEP\"],\"sentence\":[\"Moreover\",\"the elevated LEP expression observed in obesity correlated well with increased FOSL2 levels in mice and humans\",\"and adipocyte-specific genetic deletion of Fosl2 in mice reduced Lep expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is LEP---{\"tf associated gene name\":[\"FOSL2\"],\"tg associated gene name\":[\"FOSL2\"],\"tf text\":[\"FOSL2\"],\"tg text\":[\"FOSL2\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is LEP\"],\"sentence\":[\"Moreover\",\"the elevated LEP expression observed in obesity correlated well with increased FOSL2 levels in mice and humans\",\"and adipocyte-specific genetic deletion of Fosl2 in mice reduced Lep expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOSL2\"],\"target gene (associated gene name)\":[\"FOSL2\"],\"valid\":[\"0\"],\"sign\":[],\"negation\":[],\"qc\":[\"0\"],\"sentence\":[\"Moreover\",\"the elevated LEP expression observed in obesity correlated well with increased FOSL2 levels in mice and humans\",\"and adipocyte-specific genetic deletion of Fosl2 in mice reduced Lep expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[false]}---|---{\"transcription factor (associated gene name)\":[\"FOSL2\"],\"target gene (associated gene name)\":[\"FOSL2\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"Moreover\",\"the elevated LEP expression observed in obesity correlated well with increased FOSL2 levels in mice and humans\",\"and adipocyte-specific genetic deletion of Fosl2 in mice reduced Lep expression.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"0.842562\"],\"pmid counts\":[\"1\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"202\"],\"sentence pair density\":[\"0.0043103448275862\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|no TF---{\"tf associated gene name\":[\"FOXM1\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TF\"],\"sentence\":[\"INS-1 pancreatic beta-cells were transfected with promoter constructs containing fragments of the insulin gene promoter placed upstream of the firefly luciferase reporter gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE|no TF---{\"tf associated gene name\":[\"FOXM1\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TF\"],\"sentence\":[\"INS-1 pancreatic beta-cells were transfected with promoter constructs containing fragments of the insulin gene promoter placed upstream of the firefly luciferase reporter gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXM1\"],\"target gene (associated gene name)\":[\"INS\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"INS-1 pancreatic beta-cells were transfected with promoter constructs containing fragments of the insulin gene promoter placed upstream of the firefly luciferase reporter gene.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"Low\"],\"interaction score\":[\"1.595091\"],\"pmid counts\":[\"18\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"176\"],\"sentence pair density\":[\"0.0064516129032258\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "too indirect?---{\"_\":[\"_\"],\"tf\":[\"FOXM1\"],\"tg\":[\"CDKN1A\"],\"validation\":[\"V\"],\"comment 1\":[\"too indirect?\"],\"comment 2\":[],\"\":[],\"sentence\":[\"FoxM1-deficient cells also exhibit stimulation of p53 transcriptional activity, as evidenced by increased expression of the p21(cip1) gene.\"],\"interaction score\":[\"2.094626\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "too indirect?---{\"_\":[\"_\"],\"tf\":[\"FOXM1\"],\"tg\":[\"CDKN1A\"],\"validation\":[\"V\"],\"comment 1\":[\"too indirect?\"],\"comment 2\":[],\"\":[],\"sentence\":[\"FoxM1-deficient cells also exhibit stimulation of p53 transcriptional activity, as evidenced by increased expression of the p21(cip1) gene.\"],\"interaction score\":[\"2.094626\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOXM1\"],\"target gene (associated gene name)\":[\"CDKN1A\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"FoxM1-deficient cells also exhibit stimulation of p53 transcriptional activity\",\"as evidenced by increased expression of the p21(cip1) gene.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"2.094626\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"2\"],\"sentence length\":[\"139\"],\"sentence pair density\":[\"0.0140350877192982\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not named|not TRE---{\"tf associated gene name\":[\"NRG1\"],\"tg associated gene name\":[\"MMP7\"],\"tf text\":[],\"tg text\":[\"MMP-7\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"TF not named\",\"not TRE\"],\"sentence\":[\"In this study\",\"we show that heregulin-beta (HRG-beta) stimulation remarkably induced MMP-7 promoter activity and significantly enhanced the expression and activity of MMP-7 in MCF-7 cells overexpressing HER2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF not named|not TRE---{\"tf associated gene name\":[\"NRG1\"],\"tg associated gene name\":[\"MMP7\"],\"tf text\":[],\"tg text\":[\"MMP-7\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"TF not named\",\"not TRE\"],\"sentence\":[\"In this study\",\"we show that heregulin-beta (HRG-beta) stimulation remarkably induced MMP-7 promoter activity and significantly enhanced the expression and activity of MMP-7 in MCF-7 cells overexpressing HER2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NRG1\"],\"target gene (associated gene name)\":[\"MMP7\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"In this study\",\"we show that heregulin-beta (HRG-beta) stimulation remarkably induced MMP-7 promoter activity and significantly enhanced the expression and activity of MMP-7 in MCF-7 cells overexpressing HER2.\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.521984\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"2\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"208\"],\"sentence pair density\":[\"0.0112359550561797\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "reverse---{\"_\":[\"_\"],\"tf\":[\"SMAD4\"],\"tg\":[\"TGFB1\"],\"validation\":[\"N\"],\"comment 1\":[\"reverse\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We further determined putative target genes of Smad4 whose expression was regulated by TGF-beta.\"],\"interaction score\":[\"1.900373\"],\"pmid counts\":[\"55\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "reverse---{\"_\":[\"_\"],\"tf\":[\"SMAD4\"],\"tg\":[\"TGFB1\"],\"validation\":[\"N\"],\"comment 1\":[\"reverse\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We further determined putative target genes of Smad4 whose expression was regulated by TGF-beta.\"],\"interaction score\":[\"1.900373\"],\"pmid counts\":[\"55\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"SMAD4\"],\"target gene (associated gene name)\":[\"TGFB1\"],\"is tf-tf-fusion\":[],\"evaluation\":[\"n\"],\"negation\":[],\"sentence\":[\"We further determined putative target genes of Smad4 whose expression was regulated by TGF-beta.\"],\"prediction confidence\":[\"Low\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"interaction score\":[\"1.900373\"],\"pmid counts\":[\"55\"],\"sentence counts\":[\"1\"],\"sentence pairs\":[\"1\"],\"sentence length\":[\"96\"],\"sentence pair density\":[\"0.009009009009009\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "two different TFs regulating the same gene in same abstract ---{\"_\":[\"_\"],\"tf\":[\"E2F3\"],\"tg\":[\"CDKN1A\"],\"validation\":[\"V\"],\"comment 1\":[\"two different TFs regulating the same gene in same abstract \"],\"comment 2\":[],\"\":[],\"sentence\":[\"We have found that E2F1 and E2F3, transcription factors that activate genes required for cell cycle progression, are strong activators of the p21 promoter.\"],\"interaction score\":[\"2.785500\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "two different TFs regulating the same gene in same abstract ---{\"_\":[\"_\"],\"tf\":[\"E2F3\"],\"tg\":[\"CDKN1A\"],\"validation\":[\"V\"],\"comment 1\":[\"two different TFs regulating the same gene in same abstract \"],\"comment 2\":[],\"\":[],\"sentence\":[\"We have found that E2F1 and E2F3, transcription factors that activate genes required for cell cycle progression, are strong activators of the p21 promoter.\"],\"interaction score\":[\"2.785500\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"E2F3\"],\"target gene (associated gene name)\":[\"CDKN1A\"],\"valid\":[\"a\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We have found that E2F1 and E2F3\",\"transcription factors that activate genes required for cell cycle progression\",\"are strong activators of the p21 promoter.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"AP1\"],\"tg\":[\"MAPK8\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We, therefore, assessed both AP-1 DNA binding and the known AP-1 activating enzyme, c-Jun NH2-terminal kinase (JNK), in a panel of 30 ER-positive primary human breast tumors with acquired TAM resistance, as compared to a matched panel of 27 untreated control ER-positive breast tumors and a separate control set of 14 primary tumors, which included 7 ER-positive tumors that were growth-arrested by 3 months of preoperative TAM.\"],\"interaction score\":[\"3.502703\"],\"pmid counts\":[\"25\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"AP1\"],\"tg\":[\"MAPK8\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We, therefore, assessed both AP-1 DNA binding and the known AP-1 activating enzyme, c-Jun NH2-terminal kinase (JNK), in a panel of 30 ER-positive primary human breast tumors with acquired TAM resistance, as compared to a matched panel of 27 untreated control ER-positive breast tumors and a separate control set of 14 primary tumors, which included 7 ER-positive tumors that were growth-arrested by 3 months of preoperative TAM.\"],\"interaction score\":[\"3.502703\"],\"pmid counts\":[\"25\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"MAPK8\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We\",\"therefore\",\"assessed both AP-1 DNA binding and the known AP-1 activating enzyme\",\"c-Jun NH2-terminal kinase (JNK)\",\"in a panel of 30 ER-positive primary human breast tumors with acquired TAM resistance\",\"as compared to a matched panel of 27 untreated control ER-positive breast tumors and a separate control set of 14 primary tumors\",\"which included 7 ER-positive tumors that were growth-arrested by 3 months of preoperative TAM.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is beta-spectrin---{\"tf associated gene name\":[\"GATA1\"],\"tg associated gene name\":[\"HLA-DQB1\"],\"tf text\":[\"GATA-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is beta-spectrin\"],\"sentence\":[\"Using transfection of promoter/reporter plasmids in human tissue culture cell lines\",\"in vitro DNase I footprinting analyses\",\"and gel mobility shift assays\",\"a beta-spectrin gene erythroid promoter with two binding sites for GATA-1 and one site for CACCC-related proteins was identified.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is beta-spectrin---{\"tf associated gene name\":[\"GATA1\"],\"tg associated gene name\":[\"HLA-DQB1\"],\"tf text\":[\"GATA-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is beta-spectrin\"],\"sentence\":[\"Using transfection of promoter/reporter plasmids in human tissue culture cell lines\",\"in vitro DNase I footprinting analyses\",\"and gel mobility shift assays\",\"a beta-spectrin gene erythroid promoter with two binding sites for GATA-1 and one site for CACCC-related proteins was identified.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"GATA1\"],\"target gene (associated gene name)\":[\"HLA-DQB1\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Using transfection of promoter/reporter plasmids in human tissue culture cell lines\",\"in vitro DNase I footprinting analyses\",\"and gel mobility shift assays\",\"a beta-spectrin gene erythroid promoter with two binding sites for GATA-1 and one site for CACCC-related proteins was identified.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "associated with kind of co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"VCAM1\"],\"validation\":[\"N\"],\"comment 1\":[\"associated with kind of co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"The augmented TNF-alpha-induced NF-kappaB activation in HG was associated with increased TNF-alpha-mediated transcriptional activation of the vascular cell adhesion molecule-1 promoter.\"],\"interaction score\":[\"2.117070\"],\"pmid counts\":[\"50\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "associated with kind of co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"VCAM1\"],\"validation\":[\"N\"],\"comment 1\":[\"associated with kind of co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"The augmented TNF-alpha-induced NF-kappaB activation in HG was associated with increased TNF-alpha-mediated transcriptional activation of the vascular cell adhesion molecule-1 promoter.\"],\"interaction score\":[\"2.117070\"],\"pmid counts\":[\"50\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"VCAM1\"],\"valid\":[\"a\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The augmented TNF-alpha-induced NF-kappaB activation in HG was associated with increased TNF-alpha-mediated transcriptional activation of the vascular cell adhesion molecule-1 promoter\",\"\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ALDH1A1, TG problematic?---{\"_\":[\"_\"],\"tf\":[\"CUX1\"],\"tg\":[\"ALDH1A1\"],\"validation\":[\"N\"],\"comment 1\":[\"ALDH1A1, TG problematic?\"],\"comment 2\":[],\"\":[],\"sentence\":[\"CCAAT displacement protein binds to and negatively regulates human papillomavirus type 6 E6, E7, and E1 promoters.\"],\"interaction score\":[\"3.822304\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "ALDH1A1, TG problematic?---{\"_\":[\"_\"],\"tf\":[\"CUX1\"],\"tg\":[\"ALDH1A1\"],\"validation\":[\"N\"],\"comment 1\":[\"ALDH1A1, TG problematic?\"],\"comment 2\":[],\"\":[],\"sentence\":[\"CCAAT displacement protein binds to and negatively regulates human papillomavirus type 6 E6, E7, and E1 promoters.\"],\"interaction score\":[\"3.822304\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"CUX1\"],\"target gene (associated gene name)\":[\"ALDH1A1\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"CCAAT displacement protein binds to and negatively regulates human papillomavirus type 6 E6\",\"E7\",\"and E1 promoters.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "indirect---{\"_\":[\"_\"],\"tf\":[\"AP1\"],\"tg\":[\"GCLC\"],\"validation\":[\"V\"],\"comment 1\":[\"indirect\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Collectively, OxLDL induces gamma-GCS expression mediated by AP-1 resulting in an increase of GSH levels.\"],\"interaction score\":[\"3.685466\"],\"pmid counts\":[\"15\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "indirect---{\"_\":[\"_\"],\"tf\":[\"AP1\"],\"tg\":[\"GCLC\"],\"validation\":[\"V\"],\"comment 1\":[\"indirect\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Collectively, OxLDL induces gamma-GCS expression mediated by AP-1 resulting in an increase of GSH levels.\"],\"interaction score\":[\"3.685466\"],\"pmid counts\":[\"15\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"AP1\"],\"target gene (associated gene name)\":[\"GCLC\"],\"valid\":[\"a\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Collectively\",\"OxLDL induces gamma-GCS expression mediated by AP-1 resulting in an increase of GSH levels.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "pathway---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"IFNG\"],\"validation\":[\"N\"],\"comment 1\":[\"pathway\"],\"comment 2\":[],\"\":[],\"sentence\":[\"These results are the first to show the actual significance of the NF-kappa B pathway in the regulation of IFN-gamma gene expression by IL-18.\"],\"interaction score\":[\"2.014767\"],\"pmid counts\":[\"38\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "pathway---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"IFNG\"],\"validation\":[\"N\"],\"comment 1\":[\"pathway\"],\"comment 2\":[],\"\":[],\"sentence\":[\"These results are the first to show the actual significance of the NF-kappa B pathway in the regulation of IFN-gamma gene expression by IL-18.\"],\"interaction score\":[\"2.014767\"],\"pmid counts\":[\"38\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"IFNG\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results are the first to show the actual significance of the NF-kappa B pathway in the regulation of IFN-gamma gene expression by IL-18\",\"\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"NOS2\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Treatment with wortmannin promoted a prolonged activation of NF-kappaB in LPS-treated cells as well as an increase in the promoter activity of the iNOS gene as deduced from transfection experiments using a 1.7-kb fragment of the 5' flanking region of the iNOS gene.\"],\"interaction score\":[\"6.138319\"],\"pmid counts\":[\"118\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"NOS2\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Treatment with wortmannin promoted a prolonged activation of NF-kappaB in LPS-treated cells as well as an increase in the promoter activity of the iNOS gene as deduced from transfection experiments using a 1.7-kb fragment of the 5' flanking region of the iNOS gene.\"],\"interaction score\":[\"6.138319\"],\"pmid counts\":[\"118\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"NOS2\"],\"valid\":[\"a\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Treatment with wortmannin promoted a prolonged activation of NF-kappaB in LPS-treated cells as well as an increase in the promoter activity of the iNOS gene as deduced from transfection experiments using a 1\",\"7-kb fragment of the 5' flanking region of the iNOS gene\",\"\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "indirect---{\"_\":[\"_\"],\"tf\":[\"FOS\"],\"tg\":[\"TF\"],\"validation\":[\"V\"],\"comment 1\":[\"indirect\"],\"comment 2\":[],\"\":[],\"sentence\":[\"The induction of Sertoli cell differentiation appears to involve the serum response element (SRE) of the c-fos promoter to activate c-fos and intermediate bHLH factor(s) that regulate down-stream Sertoli cell-differentiated genes (e.g. transferrin expression).\"],\"interaction score\":[\"3.085128\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "indirect---{\"_\":[\"_\"],\"tf\":[\"FOS\"],\"tg\":[\"TF\"],\"validation\":[\"V\"],\"comment 1\":[\"indirect\"],\"comment 2\":[],\"\":[],\"sentence\":[\"The induction of Sertoli cell differentiation appears to involve the serum response element (SRE) of the c-fos promoter to activate c-fos and intermediate bHLH factor(s) that regulate down-stream Sertoli cell-differentiated genes (e.g. transferrin expression).\"],\"interaction score\":[\"3.085128\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"FOS\"],\"target gene (associated gene name)\":[\"TF\"],\"valid\":[\"v\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The induction of Sertoli cell differentiation appears to involve the serum response element (SRE) of the c-fos promoter to activate c-fos and intermediate bHLH factor(s) that regulate down-stream Sertoli cell-differentiated genes (e.g. transferrin expression).\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is E2F-1---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"ERVW-4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E2F-1\"],\"sentence\":[\"These results suggest that PARP plays a role in the induction of E2F-1 promoter activity\",\"which then positively regulates both E2F-1 and pol alpha expression\",\"when quiescent cells reenter the cell cycle upon recovery from aphidicolin exposure or removal of serum.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is E2F-1---{\"tf associated gene name\":[\"PARP1\"],\"tg associated gene name\":[\"ERVW-4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E2F-1\"],\"sentence\":[\"These results suggest that PARP plays a role in the induction of E2F-1 promoter activity\",\"which then positively regulates both E2F-1 and pol alpha expression\",\"when quiescent cells reenter the cell cycle upon recovery from aphidicolin exposure or removal of serum.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"PARP1\"],\"target gene (associated gene name)\":[\"ERVW-4\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"These results suggest that PARP plays a role in the induction of E2F-1 promoter activity\",\"which then positively regulates both E2F-1 and pol alpha expression\",\"when quiescent cells reenter the cell cycle upon recovery from aphidicolin exposure or removal of serum.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "other TG---{\"_\":[\"_\"],\"tf\":[\"MYC\"],\"tg\":[\"CDKN1B\"],\"validation\":[\"N\"],\"comment 1\":[\"other TG\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We now show that p27 is rapidly and transiently sequestered by cyclin D2-Cdk4 complexes upon activation of Myc and that cyclin D2 is a direct target gene of Myc.\"],\"interaction score\":[\"2.420168\"],\"pmid counts\":[\"29\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "other TG---{\"_\":[\"_\"],\"tf\":[\"MYC\"],\"tg\":[\"CDKN1B\"],\"validation\":[\"N\"],\"comment 1\":[\"other TG\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We now show that p27 is rapidly and transiently sequestered by cyclin D2-Cdk4 complexes upon activation of Myc and that cyclin D2 is a direct target gene of Myc.\"],\"interaction score\":[\"2.420168\"],\"pmid counts\":[\"29\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MYC\"],\"target gene (associated gene name)\":[\"CDKN1B\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"We now show that p27 is rapidly and transiently sequestered by cyclin D2-Cdk4 complexes upon activation of Myc and that cyclin D2 is a direct target gene of Myc.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"JUN\"],\"tg\":[\"CCND1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"The increase in Vav transforming activity correlated with the activation of the c-Jun, Elk-1, and NF-kappaB transcription factors, as well as increased transcription from the cyclin D1 promoter.\"],\"interaction score\":[\"3.707599\"],\"pmid counts\":[\"34\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"JUN\"],\"tg\":[\"CCND1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"The increase in Vav transforming activity correlated with the activation of the c-Jun, Elk-1, and NF-kappaB transcription factors, as well as increased transcription from the cyclin D1 promoter.\"],\"interaction score\":[\"3.707599\"],\"pmid counts\":[\"34\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"CCND1\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The increase in Vav transforming activity correlated with the activation of the c-Jun\",\"Elk-1\",\"and NF-kappaB transcription factors\",\"as well as increased transcription from the cyclin D1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "correlated---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"CCND1\"],\"validation\":[\"N\"],\"comment 1\":[\"correlated\"],\"comment 2\":[],\"\":[],\"sentence\":[\"The increase in Vav transforming activity correlated with the activation of the c-Jun, Elk-1, and NF-kappaB transcription factors, as well as increased transcription from the cyclin D1 promoter.\"],\"interaction score\":[\"3.302659\"],\"pmid counts\":[\"52\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "correlated---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"CCND1\"],\"validation\":[\"N\"],\"comment 1\":[\"correlated\"],\"comment 2\":[],\"\":[],\"sentence\":[\"The increase in Vav transforming activity correlated with the activation of the c-Jun, Elk-1, and NF-kappaB transcription factors, as well as increased transcription from the cyclin D1 promoter.\"],\"interaction score\":[\"3.302659\"],\"pmid counts\":[\"52\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"CCND1\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"The increase in Vav transforming activity correlated with the activation of the c-Jun\",\"Elk-1\",\"and NF-kappaB transcription factors\",\"as well as increased transcription from the cyclin D1 promoter\",\"\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"JUN\"],\"tg\":[\"CCND1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Both Dbs and Dbl activate multiple signaling pathways that include activation of the Elk-1, Jun, and NF-kappaB transcription factors and stimulation of transcription from the cyclin D1 promoter.\"],\"interaction score\":[\"3.642510\"],\"pmid counts\":[\"34\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"JUN\"],\"tg\":[\"CCND1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Both Dbs and Dbl activate multiple signaling pathways that include activation of the Elk-1, Jun, and NF-kappaB transcription factors and stimulation of transcription from the cyclin D1 promoter.\"],\"interaction score\":[\"3.642510\"],\"pmid counts\":[\"34\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"JUN\"],\"target gene (associated gene name)\":[\"CCND1\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Both Dbs and Dbl activate multiple signaling pathways that include activation of the Elk-1\",\"Jun\",\"and NF-kappaB transcription factors and stimulation of transcription from the cyclin D1 promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "linked to is too weak---{\"_\":[\"_\"],\"tf\":[\"MYC\"],\"tg\":[\"BRCA1\"],\"validation\":[\"N\"],\"comment 1\":[\"linked to is too weak\"],\"comment 2\":[],\"\":[],\"sentence\":[\"In addition, BRCA1 is linked to a number of genes involved in transcriptional regulation, including CtIP, c-Myc, the RNA holoenzyme complex, and the histone deacetylase complex.\"],\"interaction score\":[\"2.000662\"],\"pmid counts\":[\"11\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "linked to is too weak---{\"_\":[\"_\"],\"tf\":[\"MYC\"],\"tg\":[\"BRCA1\"],\"validation\":[\"N\"],\"comment 1\":[\"linked to is too weak\"],\"comment 2\":[],\"\":[],\"sentence\":[\"In addition, BRCA1 is linked to a number of genes involved in transcriptional regulation, including CtIP, c-Myc, the RNA holoenzyme complex, and the histone deacetylase complex.\"],\"interaction score\":[\"2.000662\"],\"pmid counts\":[\"11\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"MYC\"],\"target gene (associated gene name)\":[\"BRCA1\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"In addition\",\"BRCA1 is linked to a number of genes involved in transcriptional regulation\",\"including CtIP\",\"c-Myc\",\"the RNA holoenzyme complex\",\"and the histone deacetylase complex.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is not a protein coding (POLII-responsive) gene|but a tRNA (POLII-responsive) gene---{\"tf associated gene name\":[\"ZNF143\"],\"tg associated gene name\":[\"TRNAU1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is not a protein coding (POLII-responsive) gene\",\"but a tRNA (POLII-responsive) gene\"],\"sentence\":[\"mStaf is a zinc-finger protein that activates the transcription of the mouse selenocysteine tRNA gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is not a protein coding (POLII-responsive) gene|but a tRNA (POLII-responsive) gene---{\"tf associated gene name\":[\"ZNF143\"],\"tg associated gene name\":[\"TRNAU1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is not a protein coding (POLII-responsive) gene\",\"but a tRNA (POLII-responsive) gene\"],\"sentence\":[\"mStaf is a zinc-finger protein that activates the transcription of the mouse selenocysteine tRNA gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"ZNF143\"],\"target gene (associated gene name)\":[\"TRNAU1\"],\"valid\":[\"a\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"mStaf is a zinc-finger protein that activates the transcription of the mouse selenocysteine tRNA gene.\"],\"Valid\":[true],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"F3\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Overexpression of CD40, in a transient transfection system, activates the transcription factor NF-kappaB and upregulates IkappaBalpha, E-selectin, and tissue factor (TF) reporter activity.\"],\"interaction score\":[\"2.130056\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"F3\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Overexpression of CD40, in a transient transfection system, activates the transcription factor NF-kappaB and upregulates IkappaBalpha, E-selectin, and tissue factor (TF) reporter activity.\"],\"interaction score\":[\"2.130056\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"F3\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of CD40\",\"in a transient transfection system\",\"activates the transcription factor NF-kappaB and upregulates IkappaBalpha\",\"E-selectin\",\"and tissue factor (TF) reporter activity\",\"\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"SELE\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Overexpression of CD40, in a transient transfection system, activates the transcription factor NF-kappaB and upregulates IkappaBalpha, E-selectin, and tissue factor (TF) reporter activity.\"],\"interaction score\":[\"2.098910\"],\"pmid counts\":[\"26\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"SELE\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Overexpression of CD40, in a transient transfection system, activates the transcription factor NF-kappaB and upregulates IkappaBalpha, E-selectin, and tissue factor (TF) reporter activity.\"],\"interaction score\":[\"2.098910\"],\"pmid counts\":[\"26\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"SELE\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of CD40\",\"in a transient transfection system\",\"activates the transcription factor NF-kappaB and upregulates IkappaBalpha\",\"E-selectin\",\"and tissue factor (TF) reporter activity\",\"\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"TF\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Overexpression of CD40, in a transient transfection system, activates the transcription factor NF-kappaB and upregulates IkappaBalpha, E-selectin, and tissue factor (TF) reporter activity.\"],\"interaction score\":[\"2.032344\"],\"pmid counts\":[\"21\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"TF\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Overexpression of CD40, in a transient transfection system, activates the transcription factor NF-kappaB and upregulates IkappaBalpha, E-selectin, and tissue factor (TF) reporter activity.\"],\"interaction score\":[\"2.032344\"],\"pmid counts\":[\"21\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|---{\"transcription factor (associated gene name)\":[\"NFKB\"],\"target gene (associated gene name)\":[\"TF\"],\"valid\":[\"n\"],\"negation\":[],\"fusion\":[],\"qc\":[\"1\"],\"sentence\":[\"Overexpression of CD40\",\"in a transient transfection system\",\"activates the transcription factor NF-kappaB and upregulates IkappaBalpha\",\"E-selectin\",\"and tissue factor (TF) reporter activity\",\"\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GNAS-problem---{\"_\":[\"_\"],\"tf\":[\"MYC\"],\"tg\":[\"GNAS\"],\"validation\":[\"N\"],\"comment 1\":[\"GNAS-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Consistent with c-Myc regulation of the P1 promoter, E2 stimulated endogenous c-Myc in both cell lines.\"],\"interaction score\":[\"2.477057\"],\"pmid counts\":[\"16\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "GNAS-problem---{\"_\":[\"_\"],\"tf\":[\"MYC\"],\"tg\":[\"GNAS\"],\"validation\":[\"N\"],\"comment 1\":[\"GNAS-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Consistent with c-Myc regulation of the P1 promoter, E2 stimulated endogenous c-Myc in both cell lines.\"],\"interaction score\":[\"2.477057\"],\"pmid counts\":[\"16\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "SUPT7L-problem due to gamma---{\"_\":[\"_\"],\"tf\":[\"AP1\"],\"tg\":[\"SUPT7L\"],\"validation\":[\"N\"],\"comment 1\":[\"SUPT7L-problem due to gamma\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Collectively, OxLDL induces gamma-GCS expression mediated by AP-1 resulting in an increase of GSH levels.\"],\"interaction score\":[\"3.685466\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "SUPT7L-problem due to gamma---{\"_\":[\"_\"],\"tf\":[\"AP1\"],\"tg\":[\"SUPT7L\"],\"validation\":[\"N\"],\"comment 1\":[\"SUPT7L-problem due to gamma\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Collectively, OxLDL induces gamma-GCS expression mediated by AP-1 resulting in an increase of GSH levels.\"],\"interaction score\":[\"3.685466\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "adam2-problem - due to beta---{\"_\":[\"_\"],\"tf\":[\"EGR1\"],\"tg\":[\"ADAM2\"],\"validation\":[\"N\"],\"comment 1\":[\"adam2-problem - due to beta\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Coexpression of Egr1 and SF-1 led to a synergistic activation of the equine (e)LH beta promoter.\"],\"interaction score\":[\"2.986843\"],\"pmid counts\":[\"56\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "adam2-problem - due to beta---{\"_\":[\"_\"],\"tf\":[\"EGR1\"],\"tg\":[\"ADAM2\"],\"validation\":[\"N\"],\"comment 1\":[\"adam2-problem - due to beta\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Coexpression of Egr1 and SF-1 led to a synergistic activation of the equine (e)LH beta promoter.\"],\"interaction score\":[\"2.986843\"],\"pmid counts\":[\"56\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG-alpha-problem---{\"_\":[\"_\"],\"tf\":[\"AP1\"],\"tg\":[\"TPM1\"],\"validation\":[\"N\"],\"comment 1\":[\"TG-alpha-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Deletion mutants demonstrated that alpha 5 beta 1 cross-link initiated, IL-6 promoter transactivation required intact NF-kappa B and AP-1 response elements.\"],\"interaction score\":[\"2.845592\"],\"pmid counts\":[\"11\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG-alpha-problem---{\"_\":[\"_\"],\"tf\":[\"AP1\"],\"tg\":[\"TPM1\"],\"validation\":[\"N\"],\"comment 1\":[\"TG-alpha-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Deletion mutants demonstrated that alpha 5 beta 1 cross-link initiated, IL-6 promoter transactivation required intact NF-kappa B and AP-1 response elements.\"],\"interaction score\":[\"2.845592\"],\"pmid counts\":[\"11\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression|filter out \"microarray\"?---{\"_\":[\"_\"],\"tf\":[\"FOS\"],\"tg\":[\"MMP1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[\"filter out \\\"microarray\\\"?\"],\"\":[],\"sentence\":[\"Microarray analysis and luciferase reporter assays indicate that adipokines specifically induce several transcriptional programs involved in promoting tumorigenesis, including increased cell proliferation (IGF2, FOS, JUN, cyclin D1), invasive potential (MMP1, ATF3), survival (A20, NFkappaB), and angiogenesis.\"],\"interaction score\":[\"2.456501\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression|filter out \"microarray\"?---{\"_\":[\"_\"],\"tf\":[\"FOS\"],\"tg\":[\"MMP1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[\"filter out \\\"microarray\\\"?\"],\"\":[],\"sentence\":[\"Microarray analysis and luciferase reporter assays indicate that adipokines specifically induce several transcriptional programs involved in promoting tumorigenesis, including increased cell proliferation (IGF2, FOS, JUN, cyclin D1), invasive potential (MMP1, ATF3), survival (A20, NFkappaB), and angiogenesis.\"],\"interaction score\":[\"2.456501\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "microarray-measured co-expression/regulation in this paper causes a lot of false positives---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"MMP1\"],\"validation\":[\"N\"],\"comment 1\":[\"microarray-measured co-expression/regulation in this paper causes a lot of false positives\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Microarray analysis and luciferase reporter assays indicate that adipokines specifically induce several transcriptional programs involved in promoting tumorigenesis, including increased cell proliferation (IGF2, FOS, JUN, cyclin D1), invasive potential (MMP1, ATF3), survival (A20, NFkappaB), and angiogenesis.\"],\"interaction score\":[\"2.456501\"],\"pmid counts\":[\"12\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "microarray-measured co-expression/regulation in this paper causes a lot of false positives---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"MMP1\"],\"validation\":[\"N\"],\"comment 1\":[\"microarray-measured co-expression/regulation in this paper causes a lot of false positives\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Microarray analysis and luciferase reporter assays indicate that adipokines specifically induce several transcriptional programs involved in promoting tumorigenesis, including increased cell proliferation (IGF2, FOS, JUN, cyclin D1), invasive potential (MMP1, ATF3), survival (A20, NFkappaB), and angiogenesis.\"],\"interaction score\":[\"2.456501\"],\"pmid counts\":[\"12\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"E2F1\"],\"tg\":[\"MDM2\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Our data show a significant transcriptional activation of E2F-1 in primary rat hepatocytes incubated with TGF-beta1, as well as a 5-fold increase in p53 and a 2-fold decrease in its inhibitor, Mdm-2 (p < 0.05).\"],\"interaction score\":[\"3.425748\"],\"pmid counts\":[\"12\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"E2F1\"],\"tg\":[\"MDM2\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Our data show a significant transcriptional activation of E2F-1 in primary rat hepatocytes incubated with TGF-beta1, as well as a 5-fold increase in p53 and a 2-fold decrease in its inhibitor, Mdm-2 (p < 0.05).\"],\"interaction score\":[\"3.425748\"],\"pmid counts\":[\"12\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGFB1 is upstream---{\"_\":[\"_\"],\"tf\":[\"E2F1\"],\"tg\":[\"TGFB1\"],\"validation\":[\"N\"],\"comment 1\":[\"TGFB1 is upstream\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Our data show a significant transcriptional activation of E2F-1 in primary rat hepatocytes incubated with TGF-beta1, as well as a 5-fold increase in p53 and a 2-fold decrease in its inhibitor, Mdm-2 (p < 0.05).\"],\"interaction score\":[\"3.785237\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TGFB1 is upstream---{\"_\":[\"_\"],\"tf\":[\"E2F1\"],\"tg\":[\"TGFB1\"],\"validation\":[\"N\"],\"comment 1\":[\"TGFB1 is upstream\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Our data show a significant transcriptional activation of E2F-1 in primary rat hepatocytes incubated with TGF-beta1, as well as a 5-fold increase in p53 and a 2-fold decrease in its inhibitor, Mdm-2 (p < 0.05).\"],\"interaction score\":[\"3.785237\"],\"pmid counts\":[\"8\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression/regulation---{\"_\":[\"_\"],\"tf\":[\"JUN\"],\"tg\":[\"TNF\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression/regulation\"],\"comment 2\":[],\"\":[],\"sentence\":[\"MEF2C regulates c-Jun but not TNF-alpha gene expression in stimulated mast cells.\"],\"interaction score\":[\"2.671482\"],\"pmid counts\":[\"44\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression/regulation---{\"_\":[\"_\"],\"tf\":[\"JUN\"],\"tg\":[\"TNF\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression/regulation\"],\"comment 2\":[],\"\":[],\"sentence\":[\"MEF2C regulates c-Jun but not TNF-alpha gene expression in stimulated mast cells.\"],\"interaction score\":[\"2.671482\"],\"pmid counts\":[\"44\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GNAS-problem---{\"_\":[\"_\"],\"tf\":[\"ETS1\"],\"tg\":[\"GNAS\"],\"validation\":[\"N\"],\"comment 1\":[\"GNAS-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"TGF-beta and its effector Smad3 activate the P3 promoter through an AGAC box and an Ets binding site involving Ets1 and to some extent Ets2 proteins.\"],\"interaction score\":[\"2.098003\"],\"pmid counts\":[\"14\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "GNAS-problem---{\"_\":[\"_\"],\"tf\":[\"ETS1\"],\"tg\":[\"GNAS\"],\"validation\":[\"N\"],\"comment 1\":[\"GNAS-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"TGF-beta and its effector Smad3 activate the P3 promoter through an AGAC box and an Ets binding site involving Ets1 and to some extent Ets2 proteins.\"],\"interaction score\":[\"2.098003\"],\"pmid counts\":[\"14\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "IRF6-LPS problem, but also not stated that LPS regulates TNF gene expression---{\"_\":[\"_\"],\"tf\":[\"IRF6\"],\"tg\":[\"TNF\"],\"validation\":[\"N\"],\"comment 1\":[\"IRF6-LPS problem, but also not stated that LPS regulates TNF gene expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We conclude that LITAF is an important mediator of the LPS-induced inflammatory response that can be distinguished from NF-kappaB pathway and that p38alpha is the specific kinase involved in the pathway linking LPS/MyD88/LITAF to TNF.\"],\"interaction score\":[\"2.454501\"],\"pmid counts\":[\"101\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "IRF6-LPS problem, but also not stated that LPS regulates TNF gene expression---{\"_\":[\"_\"],\"tf\":[\"IRF6\"],\"tg\":[\"TNF\"],\"validation\":[\"N\"],\"comment 1\":[\"IRF6-LPS problem, but also not stated that LPS regulates TNF gene expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We conclude that LITAF is an important mediator of the LPS-induced inflammatory response that can be distinguished from NF-kappaB pathway and that p38alpha is the specific kinase involved in the pathway linking LPS/MyD88/LITAF to TNF.\"],\"interaction score\":[\"2.454501\"],\"pmid counts\":[\"101\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "pathway---{\"_\":[\"_\"],\"tf\":[\"LITAF\"],\"tg\":[\"TNF\"],\"validation\":[\"N\"],\"comment 1\":[\"pathway\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We conclude that LITAF is an important mediator of the LPS-induced inflammatory response that can be distinguished from NF-kappaB pathway and that p38alpha is the specific kinase involved in the pathway linking LPS/MyD88/LITAF to TNF.\"],\"interaction score\":[\"3.987397\"],\"pmid counts\":[\"12\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "pathway---{\"_\":[\"_\"],\"tf\":[\"LITAF\"],\"tg\":[\"TNF\"],\"validation\":[\"N\"],\"comment 1\":[\"pathway\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We conclude that LITAF is an important mediator of the LPS-induced inflammatory response that can be distinguished from NF-kappaB pathway and that p38alpha is the specific kinase involved in the pathway linking LPS/MyD88/LITAF to TNF.\"],\"interaction score\":[\"3.987397\"],\"pmid counts\":[\"12\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expr/correlates---{\"_\":[\"_\"],\"tf\":[\"ZGPAT\"],\"tg\":[\"EGFR\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expr/correlates\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We found that ZIP is downregulated in breast carcinomas and that its level of expression is negatively correlated with that of EGFR.\"],\"interaction score\":[\"3.185965\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expr/correlates---{\"_\":[\"_\"],\"tf\":[\"ZGPAT\"],\"tg\":[\"EGFR\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expr/correlates\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We found that ZIP is downregulated in breast carcinomas and that its level of expression is negatively correlated with that of EGFR.\"],\"interaction score\":[\"3.185965\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expr---{\"_\":[\"_\"],\"tf\":[\"TP53\"],\"tg\":[\"MDM2\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expr\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Small interfering RNA-induced repression of IRF8 in a GC-derived B cell line resulted in decreased expression of MDM2 at the protein level but increased expression of p53 and p21.\"],\"interaction score\":[\"2.305413\"],\"pmid counts\":[\"262\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expr---{\"_\":[\"_\"],\"tf\":[\"TP53\"],\"tg\":[\"MDM2\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expr\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Small interfering RNA-induced repression of IRF8 in a GC-derived B cell line resulted in decreased expression of MDM2 at the protein level but increased expression of p53 and p21.\"],\"interaction score\":[\"2.305413\"],\"pmid counts\":[\"262\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GNAS-problem---{\"_\":[\"_\"],\"tf\":[\"RUNX2\"],\"tg\":[\"GNAS\"],\"validation\":[\"N\"],\"comment 1\":[\"GNAS-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We determined that the Lef1DeltaN p2 promoter is active in osteoblasts and Runx2 regulates its activity.\"],\"interaction score\":[\"2.464973\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "GNAS-problem---{\"_\":[\"_\"],\"tf\":[\"RUNX2\"],\"tg\":[\"GNAS\"],\"validation\":[\"N\"],\"comment 1\":[\"GNAS-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We determined that the Lef1DeltaN p2 promoter is active in osteoblasts and Runx2 regulates its activity.\"],\"interaction score\":[\"2.464973\"],\"pmid counts\":[\"7\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"BAD\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"GAT-1 deficiency promotes T cell division and survival by down-regulating cyclin dependent kinase inhibitor p27(kip1), differentially regulating the pro- and anti-apoptotic proteins Bcl-2, Bcl-xl, and Bad and activating transcription factor NF-kappaB through induction of translocation and phosphorylation of protein kinase C (PKC) theta.\"],\"interaction score\":[\"3.401807\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NFKB\"],\"tg\":[\"BAD\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"GAT-1 deficiency promotes T cell division and survival by down-regulating cyclin dependent kinase inhibitor p27(kip1), differentially regulating the pro- and anti-apoptotic proteins Bcl-2, Bcl-xl, and Bad and activating transcription factor NF-kappaB through induction of translocation and phosphorylation of protein kinase C (PKC) theta.\"],\"interaction score\":[\"3.401807\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "pure PPIs---{\"_\":[\"_\"],\"tf\":[\"NFE2L2\"],\"tg\":[\"KEAP1\"],\"validation\":[\"N\"],\"comment 1\":[\"pure PPIs\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Upon oxidative stress, Nrf2 is induced at protein levels through redox-sensitive modifications on cysteine residues of Keap1, a component of the E3 ubiquitin ligase that targets Nrf2 for ubiquitin-dependent degradation.\"],\"interaction score\":[\"1.928410\"],\"pmid counts\":[\"50\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "pure PPIs---{\"_\":[\"_\"],\"tf\":[\"NFE2L2\"],\"tg\":[\"KEAP1\"],\"validation\":[\"N\"],\"comment 1\":[\"pure PPIs\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Upon oxidative stress, Nrf2 is induced at protein levels through redox-sensitive modifications on cysteine residues of Keap1, a component of the E3 ubiquitin ligase that targets Nrf2 for ubiquitin-dependent degradation.\"],\"interaction score\":[\"1.928410\"],\"pmid counts\":[\"50\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CD4+-problem---{\"_\":[\"_\"],\"tf\":[\"GATA3\"],\"tg\":[\"CD4\"],\"validation\":[\"N\"],\"comment 1\":[\"CD4+-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Our results suggest that the balance of Runx3 and GATA3 is one factor that influences the manifestation of CD4(+) cells as the Th1 or Th2 phenotypes.\"],\"interaction score\":[\"1.975805\"],\"pmid counts\":[\"10\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "CD4+-problem---{\"_\":[\"_\"],\"tf\":[\"GATA3\"],\"tg\":[\"CD4\"],\"validation\":[\"N\"],\"comment 1\":[\"CD4+-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Our results suggest that the balance of Runx3 and GATA3 is one factor that influences the manifestation of CD4(+) cells as the Th1 or Th2 phenotypes.\"],\"interaction score\":[\"1.975805\"],\"pmid counts\":[\"10\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"EGR1\"],\"tg\":[\"NAB2\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Although weak correlation was found between EGR-1 and NAB-2 expressions on the whole, NAB-2 expression decreased as tumors dedifferentiated, and inhibition of DNA methyltransferase/histone deacetylase increased NAB-2 expression in lung cancer cells despite no epigenetic alteration in the NAB-2 promoter.\"],\"interaction score\":[\"5.772911\"],\"pmid counts\":[\"13\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"EGR1\"],\"tg\":[\"NAB2\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Although weak correlation was found between EGR-1 and NAB-2 expressions on the whole, NAB-2 expression decreased as tumors dedifferentiated, and inhibition of DNA methyltransferase/histone deacetylase increased NAB-2 expression in lung cancer cells despite no epigenetic alteration in the NAB-2 promoter.\"],\"interaction score\":[\"5.772911\"],\"pmid counts\":[\"13\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expre---{\"_\":[\"_\"],\"tf\":[\"VDR\"],\"tg\":[\"CYP24A1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expre\"],\"comment 2\":[],\"\":[],\"sentence\":[\"In conclusion, VPA enhances transcriptional activity of VDR and increases expression of CYP24 mRNA in the presence of VD3 in physiological concentrations.\"],\"interaction score\":[\"3.219143\"],\"pmid counts\":[\"17\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expre---{\"_\":[\"_\"],\"tf\":[\"VDR\"],\"tg\":[\"CYP24A1\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expre\"],\"comment 2\":[],\"\":[],\"sentence\":[\"In conclusion, VPA enhances transcriptional activity of VDR and increases expression of CYP24 mRNA in the presence of VD3 in physiological concentrations.\"],\"interaction score\":[\"3.219143\"],\"pmid counts\":[\"17\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GNAS-problem, and SPI1 not found in sentence---{\"_\":[\"_\"],\"tf\":[\"SPI1\"],\"tg\":[\"GNAS\"],\"validation\":[\"N\"],\"comment 1\":[\"GNAS-problem, and SPI1 not found in sentence\"],\"comment 2\":[],\"\":[],\"sentence\":[\"A luciferase promoter assay and a chromosome immunoprecipitation assay showed that Runx1 expression in mouse epiphyseal cartilage was repressed by Trps1 binding to the GATA domain of the P2 promoter; the proximal segment of two promoters of the Runx1 gene.\"],\"interaction score\":[\"2.041278\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "GNAS-problem, and SPI1 not found in sentence---{\"_\":[\"_\"],\"tf\":[\"SPI1\"],\"tg\":[\"GNAS\"],\"validation\":[\"N\"],\"comment 1\":[\"GNAS-problem, and SPI1 not found in sentence\"],\"comment 2\":[],\"\":[],\"sentence\":[\"A luciferase promoter assay and a chromosome immunoprecipitation assay showed that Runx1 expression in mouse epiphyseal cartilage was repressed by Trps1 binding to the GATA domain of the P2 promoter; the proximal segment of two promoters of the Runx1 gene.\"],\"interaction score\":[\"2.041278\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NKX6-1\"],\"tg\":[\"GCG\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"siRNA knockdown of SOX9 significantly decreased islet-epithelial cell proliferation, NGN3, NKX6.1, PAX6 and INS mRNA levels and the number of NGN3(+) and insulin(+) cells (p<0.05) while increasing GCG mRNA and glucagon(+) cells (p<0.05).\"],\"interaction score\":[\"4.026050\"],\"pmid counts\":[\"10\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"NKX6-1\"],\"tg\":[\"GCG\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"siRNA knockdown of SOX9 significantly decreased islet-epithelial cell proliferation, NGN3, NKX6.1, PAX6 and INS mRNA levels and the number of NGN3(+) and insulin(+) cells (p<0.05) while increasing GCG mRNA and glucagon(+) cells (p<0.05).\"],\"interaction score\":[\"4.026050\"],\"pmid counts\":[\"10\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"PAX6\"],\"tg\":[\"GCG\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"siRNA knockdown of SOX9 significantly decreased islet-epithelial cell proliferation, NGN3, NKX6.1, PAX6 and INS mRNA levels and the number of NGN3(+) and insulin(+) cells (p<0.05) while increasing GCG mRNA and glucagon(+) cells (p<0.05).\"],\"interaction score\":[\"3.816295\"],\"pmid counts\":[\"48\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression---{\"_\":[\"_\"],\"tf\":[\"PAX6\"],\"tg\":[\"GCG\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[],\"\":[],\"sentence\":[\"siRNA knockdown of SOX9 significantly decreased islet-epithelial cell proliferation, NGN3, NKX6.1, PAX6 and INS mRNA levels and the number of NGN3(+) and insulin(+) cells (p<0.05) while increasing GCG mRNA and glucagon(+) cells (p<0.05).\"],\"interaction score\":[\"3.816295\"],\"pmid counts\":[\"48\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "PPI---{\"_\":[\"_\"],\"tf\":[\"TP53\"],\"tg\":[\"NOTCH1\"],\"validation\":[\"N\"],\"comment 1\":[\"PPI\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Here, we show that chromatin remodelling protein SMAR1 inhibits p53 acetylation and p53 dependent apoptosis by repressing p300 expression in response to DNA damage.\"],\"interaction score\":[\"3.018413\"],\"pmid counts\":[\"44\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "PPI---{\"_\":[\"_\"],\"tf\":[\"TP53\"],\"tg\":[\"NOTCH1\"],\"validation\":[\"N\"],\"comment 1\":[\"PPI\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Here, we show that chromatin remodelling protein SMAR1 inhibits p53 acetylation and p53 dependent apoptosis by repressing p300 expression in response to DNA damage.\"],\"interaction score\":[\"3.018413\"],\"pmid counts\":[\"44\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "co-expression|CD4+ cells frequently cause misnormalizations? ---{\"_\":[\"_\"],\"tf\":[\"FOXP3\"],\"tg\":[\"CD4\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[\"CD4+ cells frequently cause misnormalizations? \"],\"\":[],\"sentence\":[\"Approximately one-third of these TGFRII(+) CD4(+) T cells express the transcription factor Foxp3.\"],\"interaction score\":[\"3.542912\"],\"pmid counts\":[\"38\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "co-expression|CD4+ cells frequently cause misnormalizations? ---{\"_\":[\"_\"],\"tf\":[\"FOXP3\"],\"tg\":[\"CD4\"],\"validation\":[\"N\"],\"comment 1\":[\"co-expression\"],\"comment 2\":[\"CD4+ cells frequently cause misnormalizations? \"],\"\":[],\"sentence\":[\"Approximately one-third of these TGFRII(+) CD4(+) T cells express the transcription factor Foxp3.\"],\"interaction score\":[\"3.542912\"],\"pmid counts\":[\"38\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ER-postitive breast cancer - filter out?---{\"_\":[\"_\"],\"tf\":[\"ESR1\"],\"tg\":[\"E2F1\"],\"validation\":[\"N\"],\"comment 1\":[\"ER-postitive breast cancer - filter out?\"],\"comment 2\":[],\"\":[],\"sentence\":[\"In addition, PRMT2 silencing could enhance 17-estradiol-induced proliferation by regulating E2F1 expression and E2F1-responsive genes in ER-positive breast cancer cells.\"],\"interaction score\":[\"2.301195\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "ER-postitive breast cancer - filter out?---{\"_\":[\"_\"],\"tf\":[\"ESR1\"],\"tg\":[\"E2F1\"],\"validation\":[\"N\"],\"comment 1\":[\"ER-postitive breast cancer - filter out?\"],\"comment 2\":[],\"\":[],\"sentence\":[\"In addition, PRMT2 silencing could enhance 17-estradiol-induced proliferation by regulating E2F1 expression and E2F1-responsive genes in ER-positive breast cancer cells.\"],\"interaction score\":[\"2.301195\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "FOXC1-problem---{\"_\":[\"_\"],\"tf\":[\"FOXC1\"],\"tg\":[\"PTEN\"],\"validation\":[\"N\"],\"comment 1\":[\"FOXC1-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Transient transfections, ChIP and EMSA assays evidenced the 5'-flanking region of PTEN gene promoter E2-responsive.\"],\"interaction score\":[\"4.695966\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "FOXC1-problem---{\"_\":[\"_\"],\"tf\":[\"FOXC1\"],\"tg\":[\"PTEN\"],\"validation\":[\"N\"],\"comment 1\":[\"FOXC1-problem\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Transient transfections, ChIP and EMSA assays evidenced the 5'-flanking region of PTEN gene promoter E2-responsive.\"],\"interaction score\":[\"4.695966\"],\"pmid counts\":[\"3\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "tg not named---{\"_\":[\"_\"],\"tf\":[\"KLF1\"],\"tg\":[\"SPTA1\"],\"validation\":[\"N\"],\"comment 1\":[\"tg not named\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We define a cis-regulatory module bound by KLF1, GATA1, TAL1, and EP300 that coordinates a core set of erythroid genes.\"],\"interaction score\":[\"2.462666\"],\"pmid counts\":[\"10\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "tg not named---{\"_\":[\"_\"],\"tf\":[\"KLF1\"],\"tg\":[\"SPTA1\"],\"validation\":[\"N\"],\"comment 1\":[\"tg not named\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We define a cis-regulatory module bound by KLF1, GATA1, TAL1, and EP300 that coordinates a core set of erythroid genes.\"],\"interaction score\":[\"2.462666\"],\"pmid counts\":[\"10\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF---{\"_\":[\"_\"],\"tf\":[\"TCF3\"],\"tg\":[\"CYP19A1\"],\"validation\":[\"N\"],\"comment 1\":[\"TF\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We previously showed that human FOXL2 functions as a transcriptional repressor of several key genes involved in granulosa cell proliferation and differentiation, including steroidogenic acute regulatory protein (STAR), P450aromatase (CYP19A1 (CYP19)), P450scc (CYP11A1 (CYP11A)), and cyclin D2 (CCND2).\"],\"interaction score\":[\"2.217352\"],\"pmid counts\":[\"6\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF---{\"_\":[\"_\"],\"tf\":[\"TCF3\"],\"tg\":[\"CYP19A1\"],\"validation\":[\"N\"],\"comment 1\":[\"TF\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We previously showed that human FOXL2 functions as a transcriptional repressor of several key genes involved in granulosa cell proliferation and differentiation, including steroidogenic acute regulatory protein (STAR), P450aromatase (CYP19A1 (CYP19)), P450scc (CYP11A1 (CYP11A)), and cyclin D2 (CCND2).\"],\"interaction score\":[\"2.217352\"],\"pmid counts\":[\"6\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF---{\"_\":[\"_\"],\"tf\":[\"TCF3\"],\"tg\":[\"STAR\"],\"validation\":[\"N\"],\"comment 1\":[\"TF\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We previously showed that human FOXL2 functions as a transcriptional repressor of several key genes involved in granulosa cell proliferation and differentiation, including steroidogenic acute regulatory protein (STAR), P450aromatase (CYP19A1 (CYP19)), P450scc (CYP11A1 (CYP11A)), and cyclin D2 (CCND2).\"],\"interaction score\":[\"2.217352\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF---{\"_\":[\"_\"],\"tf\":[\"TCF3\"],\"tg\":[\"STAR\"],\"validation\":[\"N\"],\"comment 1\":[\"TF\"],\"comment 2\":[],\"\":[],\"sentence\":[\"We previously showed that human FOXL2 functions as a transcriptional repressor of several key genes involved in granulosa cell proliferation and differentiation, including steroidogenic acute regulatory protein (STAR), P450aromatase (CYP19A1 (CYP19)), P450scc (CYP11A1 (CYP11A)), and cyclin D2 (CCND2).\"],\"interaction score\":[\"2.217352\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not gene regulation? found not correct normalization---{\"_\":[\"_\"],\"tf\":[\"EPAS1\"],\"tg\":[\"KDR\"],\"validation\":[\"N\"],\"comment 1\":[\"not gene regulation? found not correct normalization\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Activation of Nrf2 blocked VEGF induction of VEGFR2-PI3K/Akt and downregulated HIF-2 in ECs, which may serve as important mechanisms for Nrf2 inhibition of Dll4 and Notch signaling.\"],\"interaction score\":[\"2.248182\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not gene regulation? found not correct normalization---{\"_\":[\"_\"],\"tf\":[\"EPAS1\"],\"tg\":[\"KDR\"],\"validation\":[\"N\"],\"comment 1\":[\"not gene regulation? found not correct normalization\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Activation of Nrf2 blocked VEGF induction of VEGFR2-PI3K/Akt and downregulated HIF-2 in ECs, which may serve as important mechanisms for Nrf2 inhibition of Dll4 and Notch signaling.\"],\"interaction score\":[\"2.248182\"],\"pmid counts\":[\"4\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "pathway ---{\"_\":[\"_\"],\"tf\":[\"SP1\"],\"tg\":[\"MMP14\"],\"validation\":[\"N\"],\"comment 1\":[\"pathway \"],\"comment 2\":[],\"\":[],\"sentence\":[\"Taken together, these results strongly suggest that CD81 stimulates melanoma cell motility by inducing MT1-MMP expression through the Akt-dependent Sp1 activation signaling pathway, leading to increased melanoma invasion and metastasis.\"],\"interaction score\":[\"2.143540\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "pathway ---{\"_\":[\"_\"],\"tf\":[\"SP1\"],\"tg\":[\"MMP14\"],\"validation\":[\"N\"],\"comment 1\":[\"pathway \"],\"comment 2\":[],\"\":[],\"sentence\":[\"Taken together, these results strongly suggest that CD81 stimulates melanoma cell motility by inducing MT1-MMP expression through the Akt-dependent Sp1 activation signaling pathway, leading to increased melanoma invasion and metastasis.\"],\"interaction score\":[\"2.143540\"],\"pmid counts\":[\"5\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "IRF3 similarity to IRF7 which is the TF here---{\"_\":[\"_\"],\"tf\":[\"IRF3\"],\"tg\":[\"IFN1@\"],\"validation\":[\"N\"],\"comment 1\":[\"IRF3 similarity to IRF7 which is the TF here\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Thus, IRF-7 exhibits functional similarity to IRF-3; however, the preferential expression of IRF-7 in lymphoid cells (the cell type that expresses IFNA) suggests that IRF-7 may play a critical role in regulating the IFNA gene expression.\"],\"interaction score\":[\"2.955011\"],\"pmid counts\":[\"16\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "IRF3 similarity to IRF7 which is the TF here---{\"_\":[\"_\"],\"tf\":[\"IRF3\"],\"tg\":[\"IFN1@\"],\"validation\":[\"N\"],\"comment 1\":[\"IRF3 similarity to IRF7 which is the TF here\"],\"comment 2\":[],\"\":[],\"sentence\":[\"Thus, IRF-7 exhibits functional similarity to IRF-3; however, the preferential expression of IRF-7 in lymphoid cells (the cell type that expresses IFNA) suggests that IRF-7 may play a critical role in regulating the IFNA gene expression.\"],\"interaction score\":[\"2.955011\"],\"pmid counts\":[\"16\"],\"sentence counts\":[\"1\"],\"prediction confidence\":[\"High\"],\"threshold confidence (score: 1.6; min # pmid: 1000; min # sentences in pmid: 2)\":[\"High\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is GATA-4---{\"tf associated gene name\":[\"CEBPD\"],\"tg associated gene name\":[\"KNG1\"],\"tf text\":[\"C/EBPdelta\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is GATA-4\"],\"sentence\":[\"In GATA-4 expressing intestinal epithelial cells\",\"GATA-4 led to decreased C/EBPbeta and C/EBPdelta protein levels\",\"and decreased thiostatin expression in response to IL-1beta and dexamethasone.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is GATA-4---{\"tf associated gene name\":[\"CEBPD\"],\"tg associated gene name\":[\"KNG1\"],\"tf text\":[\"C/EBPdelta\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is GATA-4\"],\"sentence\":[\"In GATA-4 expressing intestinal epithelial cells\",\"GATA-4 led to decreased C/EBPbeta and C/EBPdelta protein levels\",\"and decreased thiostatin expression in response to IL-1beta and dexamethasone.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is AP-2alpha---{\"tf associated gene name\":[\"HR\"],\"tg associated gene name\":[\"PPARD\"],\"tf text\":[],\"tg text\":[\"PPARbeta\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF is AP-2alpha\"],\"sentence\":[\"Of note\",\"nicotine induced complex formation between alpha7 nAChR and PPARbeta/delta protein and increased PPARbeta/delta gene promoter activity through inhibition of AP-2alpha as shown by reduced AP-2alpha binding using electrophoretic gel mobility shift and chromatin immunoprecipitation assays.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is AP-2alpha---{\"tf associated gene name\":[\"HR\"],\"tg associated gene name\":[\"PPARD\"],\"tf text\":[],\"tg text\":[\"PPARbeta\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF is AP-2alpha\"],\"sentence\":[\"Of note\",\"nicotine induced complex formation between alpha7 nAChR and PPARbeta/delta protein and increased PPARbeta/delta gene promoter activity through inhibition of AP-2alpha as shown by reduced AP-2alpha binding using electrophoretic gel mobility shift and chromatin immunoprecipitation assays.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain normalization TF---{\"tf associated gene name\":[\"NFE2\"],\"tg associated gene name\":[\"HBB\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain normalization TF\"],\"sentence\":[\"However\",\"mice nullizygous for p45 express nearly normal levels of beta-globin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "uncertain normalization TF---{\"tf associated gene name\":[\"NFE2\"],\"tg associated gene name\":[\"HBB\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"uncertain normalization TF\"],\"sentence\":[\"However\",\"mice nullizygous for p45 express nearly normal levels of beta-globin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is E-cadherin---{\"tf associated gene name\":[\"ZEB1\"],\"tg associated gene name\":[\"ASS1\"],\"tf text\":[\"ZEB-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is E-cadherin\"],\"sentence\":[\"NF-??B activation resulted in binding of ZEB-1 to the E-cadherin promoter as demonstrated by ChIP analysis with subsequent E-cadherin suppression (fold increase in ZEB-1 binding to the E-cadherin promoter region: untreated + ZEB-1 antibody vs CTGF(200 ng/mL) + ZEB-1 antibody: 1.5 vs 6.4\",\"difference = 4.9\",\"95% CI = 4.8 to 5.0\",\"P < .001).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is E-cadherin---{\"tf associated gene name\":[\"ZEB1\"],\"tg associated gene name\":[\"ASS1\"],\"tf text\":[\"ZEB-1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is E-cadherin\"],\"sentence\":[\"NF-??B activation resulted in binding of ZEB-1 to the E-cadherin promoter as demonstrated by ChIP analysis with subsequent E-cadherin suppression (fold increase in ZEB-1 binding to the E-cadherin promoter region: untreated + ZEB-1 antibody vs CTGF(200 ng/mL) + ZEB-1 antibody: 1.5 vs 6.4\",\"difference = 4.9\",\"95% CI = 4.8 to 5.0\",\"P < .001).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear|uncertain---{\"tf associated gene name\":[\"NR2F1\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"unclear\",\"uncertain\"],\"sentence\":[\"As in mammalian cells\",\"here we observed a positive regulation of the rtER gene promoter by the chicken ovalbumin upstream promoter-transcription factor I (COUP-TFI) through enhancing autoregulation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "unclear|uncertain---{\"tf associated gene name\":[\"NR2F1\"],\"tg associated gene name\":[\"ESR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"unclear\",\"uncertain\"],\"sentence\":[\"As in mammalian cells\",\"here we observed a positive regulation of the rtER gene promoter by the chicken ovalbumin upstream promoter-transcription factor I (COUP-TFI) through enhancing autoregulation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is LRH-1 ---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"CYP11A1\"],\"tf text\":[],\"tg text\":[\"CYP11A1\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is LRH-1 \"],\"sentence\":[\"Cotransfection of human granulosa cell tumor cells with CYP11A1 promoter and LRH-1 expression vector resulted in a significant increase in CYP11A1 expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is LRH-1 ---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"CYP11A1\"],\"tf text\":[],\"tg text\":[\"CYP11A1\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is LRH-1 \"],\"sentence\":[\"Cotransfection of human granulosa cell tumor cells with CYP11A1 promoter and LRH-1 expression vector resulted in a significant increase in CYP11A1 expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE/GRE---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"IGF2R\"],\"tf text\":[],\"tg text\":[\"M6P/IGF2r\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE/GRE\"],\"sentence\":[\"We show that this increase is mediated\",\"at least in part\",\"by a stabilization of M6P/IGF2r transcripts by radiation in both ER positive (MCF7 and T47D) and ER negative (MDA-MB-231) breast cancer cell lines.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE/GRE---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"IGF2R\"],\"tf text\":[],\"tg text\":[\"M6P/IGF2r\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE/GRE\"],\"sentence\":[\"We show that this increase is mediated\",\"at least in part\",\"by a stabilization of M6P/IGF2r transcripts by radiation in both ER positive (MCF7 and T47D) and ER negative (MDA-MB-231) breast cancer cell lines.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "PAX-problem? no TF here---{\"tf associated gene name\":[\"PAX1\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[],\"tg text\":[\"MDM2\"],\"1\":[],\"negation\":[],\"not true\":[\"c\",\"n\"],\"comment\":[\"PAX-problem? no TF here\"],\"sentence\":[\"To investigate a potential role for MDM2 in the inactivation of wt-p53 in pediatric acute lymphoblastic leukemia (ALL)\",\"we examined the expression of MDM2 and p53\",\"as well as the occurrence of p53 mutations and possible amplification of the MDM2 gene\",\"in 19 pediatric ALL cell lines and one pediatric acute myelogenous leukemia (AML) line.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "PAX-problem? no TF here---{\"tf associated gene name\":[\"PAX1\"],\"tg associated gene name\":[\"MDM2\"],\"tf text\":[],\"tg text\":[\"MDM2\"],\"1\":[],\"negation\":[],\"not true\":[\"c\",\"n\"],\"comment\":[\"PAX-problem? no TF here\"],\"sentence\":[\"To investigate a potential role for MDM2 in the inactivation of wt-p53 in pediatric acute lymphoblastic leukemia (ALL)\",\"we examined the expression of MDM2 and p53\",\"as well as the occurrence of p53 mutations and possible amplification of the MDM2 gene\",\"in 19 pediatric ALL cell lines and one pediatric acute myelogenous leukemia (AML) line.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"DPYSL2\"],\"tf text\":[],\"tg text\":[\"CRMP-2\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"no TF\"],\"sentence\":[\"Here we show that glial cell line-derived neurotrophic factor (GDNF) enhances CRMP-2 expression in TGW human neuroblastoma cells via activation of RET receptor tyrosine kinase.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TF---{\"tf associated gene name\":[\"TXK\"],\"tg associated gene name\":[\"DPYSL2\"],\"tf text\":[],\"tg text\":[\"CRMP-2\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"no TF\"],\"sentence\":[\"Here we show that glial cell line-derived neurotrophic factor (GDNF) enhances CRMP-2 expression in TGW human neuroblastoma cells via activation of RET receptor tyrosine kinase.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"RUNX1\"],\"tg associated gene name\":[\"CDKN2B\"],\"tf text\":[\"RUNX1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"A Novel Translocation Involving RUNX1 and HOXA Gene Clusters in a Case of Acute Myeloid Leukemia with t(7;21)(p15;q22).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"RUNX1\"],\"tg associated gene name\":[\"CDKN2B\"],\"tf text\":[\"RUNX1\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"A Novel Translocation Involving RUNX1 and HOXA Gene Clusters in a Case of Acute Myeloid Leukemia with t(7;21)(p15;q22).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Prg1; ---{\"tf associated gene name\":[\"NEUROD6\"],\"tg associated gene name\":[\"SRGN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is Prg1; \"],\"sentence\":[\"Prg1 is regulated by the basic helix-loop-helix transcription factor Math2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Prg1; ---{\"tf associated gene name\":[\"NEUROD6\"],\"tg associated gene name\":[\"SRGN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is Prg1; \"],\"sentence\":[\"Prg1 is regulated by the basic helix-loop-helix transcription factor Math2.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no clear TF-TG---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"KEAP1\"],\"tf text\":[],\"tg text\":[\"INrf2\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no clear TF-TG\"],\"sentence\":[\"The antioxidant treatment of cells led to nuclear localization and stabilization of Nrf2 and induction of INrf2 gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no clear TF-TG---{\"tf associated gene name\":[\"NFE2L2\"],\"tg associated gene name\":[\"KEAP1\"],\"tf text\":[],\"tg text\":[\"INrf2\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"no clear TF-TG\"],\"sentence\":[\"The antioxidant treatment of cells led to nuclear localization and stabilization of Nrf2 and induction of INrf2 gene expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  apoAV---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"DR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is  apoAV\"],\"sentence\":[\"Deletion\",\"mutagenesis\",\"and binding assays revealed that HNF-4alpha directly regulates human apoAV promoter through DR1 [a direct repeat separated by one nucleotide (nt)]\",\"and via a novel element for HNF-4alpha consisting of an inverted repeat separated by 8 nt (IR8).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is  apoAV---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"DR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is  apoAV\"],\"sentence\":[\"Deletion\",\"mutagenesis\",\"and binding assays revealed that HNF-4alpha directly regulates human apoAV promoter through DR1 [a direct repeat separated by one nucleotide (nt)]\",\"and via a novel element for HNF-4alpha consisting of an inverted repeat separated by 8 nt (IR8).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is alpha-spectrin---{\"tf associated gene name\":[\"NFE2\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is alpha-spectrin\"],\"sentence\":[\"Using transfection of promoter/reporter plasmids in human tissue culture cell lines\",\"in vitro DNase I footprinting analyses\",\"and gel mobility shift assays\",\"an alpha-spectrin gene erythroid promoter with binding sites for GATA-1- and NF-E2-related proteins was identified.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is alpha-spectrin---{\"tf associated gene name\":[\"NFE2\"],\"tg associated gene name\":[\"TPM1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is alpha-spectrin\"],\"sentence\":[\"Using transfection of promoter/reporter plasmids in human tissue culture cell lines\",\"in vitro DNase I footprinting analyses\",\"and gel mobility shift assays\",\"an alpha-spectrin gene erythroid promoter with binding sites for GATA-1- and NF-E2-related proteins was identified.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Ku70---{\"tf associated gene name\":[\"ZIC2\"],\"tg associated gene name\":[\"HSPA7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is Ku70\"],\"sentence\":[\"These data indicate that the repression of the HSP70B promoter by NF-IL6\",\"Ku70\",\"and MAPK occurs independently of each other and involves three complementary mechanisms.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Ku70---{\"tf associated gene name\":[\"ZIC2\"],\"tg associated gene name\":[\"HSPA7\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TF is Ku70\"],\"sentence\":[\"These data indicate that the repression of the HSP70B promoter by NF-IL6\",\"Ku70\",\"and MAPK occurs independently of each other and involves three complementary mechanisms.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GLI-problem; other TFs named here---{\"tf associated gene name\":[\"GLI2\"],\"tg associated gene name\":[\"S100A9\"],\"tf text\":[],\"tg text\":[\"MRP14\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"GLI-problem; other TFs named here\"],\"sentence\":[\"Among several known transcription factor binding motifs\",\"nuclear protein(s) of VD3-treated HL-60 cells and THP-1 cells bound to the CCAAT/enhancer binding protein (C/EBP)-binding motif that was located in the upstream region of the MRP14 gene (-81)\",\"as evidenced by the competitive gel mobility-shift assay.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "GLI-problem; other TFs named here---{\"tf associated gene name\":[\"GLI2\"],\"tg associated gene name\":[\"S100A9\"],\"tf text\":[],\"tg text\":[\"MRP14\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"GLI-problem; other TFs named here\"],\"sentence\":[\"Among several known transcription factor binding motifs\",\"nuclear protein(s) of VD3-treated HL-60 cells and THP-1 cells bound to the CCAAT/enhancer binding protein (C/EBP)-binding motif that was located in the upstream region of the MRP14 gene (-81)\",\"as evidenced by the competitive gel mobility-shift assay.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p21CIP (most likely CDKN1A)---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"MLIP\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is p21CIP (most likely CDKN1A)\"],\"sentence\":[\"Transcription factor Sp3 represses expression of p21CIP?? via inhibition of productive elongation by RNA polymerase II.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is p21CIP (most likely CDKN1A)---{\"tf associated gene name\":[\"SP3\"],\"tg associated gene name\":[\"MLIP\"],\"tf text\":[\"Sp3\"],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is p21CIP (most likely CDKN1A)\"],\"sentence\":[\"Transcription factor Sp3 represses expression of p21CIP?? via inhibition of productive elongation by RNA polymerase II.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is COMP---{\"tf associated gene name\":[\"ZBTB7A\"],\"tg associated gene name\":[\"HDAC1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is COMP\"],\"sentence\":[\"Subsequently we isolated LRF transcription repressor as an NRE-binding protein and established that LRF inhibits COMP gene expression via recruiting histone deacetylase 1 (HDAC1) to the COMP promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is COMP---{\"tf associated gene name\":[\"ZBTB7A\"],\"tg associated gene name\":[\"HDAC1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is COMP\"],\"sentence\":[\"Subsequently we isolated LRF transcription repressor as an NRE-binding protein and established that LRF inhibits COMP gene expression via recruiting histone deacetylase 1 (HDAC1) to the COMP promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is IGF-IR---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"INSR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is IGF-IR\"],\"sentence\":[\"ChIP analysis show enhanced AR binding to the IGF-IR promoter in AR-overexpressing cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is IGF-IR---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"INSR\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is IGF-IR\"],\"sentence\":[\"ChIP analysis show enhanced AR binding to the IGF-IR promoter in AR-overexpressing cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG normalization very uncertain---{\"tf associated gene name\":[\"THRA\"],\"tg associated gene name\":[\"THRSP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG normalization very uncertain\"],\"sentence\":[\"These data demonstrate that a cis-active thyroid hormone-responsive element resides in the 5'-flanking region of the rat alpha-subunit gene and that the mechanism involved in the suppression of expression of this gene by T3 could involve specific binding of the thyroid hormone receptor to this region of the gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG normalization very uncertain---{\"tf associated gene name\":[\"THRA\"],\"tg associated gene name\":[\"THRSP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG normalization very uncertain\"],\"sentence\":[\"These data demonstrate that a cis-active thyroid hormone-responsive element resides in the 5'-flanking region of the rat alpha-subunit gene and that the mechanism involved in the suppression of expression of this gene by T3 could involve specific binding of the thyroid hormone receptor to this region of the gene.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG normalization uncertain---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"TBXA2R\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TG normalization uncertain\"],\"sentence\":[\"Supershift analyses suggest that the Sp1 and Sp3 (to a lesser degree) transcription factors have a role in the regulation of TP expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG normalization uncertain---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"TBXA2R\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TG normalization uncertain\"],\"sentence\":[\"Supershift analyses suggest that the Sp1 and Sp3 (to a lesser degree) transcription factors have a role in the regulation of TP expression.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is TERT---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is TERT\"],\"sentence\":[\"We report that in endothelial cells\",\"the angiogenic effect of 17beta-estradiol (E2) is inhibited by the estrogen receptor (ER) antagonist ICI or the NO synthase (NOS) inhibitor 7-nitroindazole via downregulation of hTERT\",\"the telomerase catalytic subunit\",\"suggesting that E2 and NO are involved in controlling hTERT transcription.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is TERT---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is TERT\"],\"sentence\":[\"We report that in endothelial cells\",\"the angiogenic effect of 17beta-estradiol (E2) is inhibited by the estrogen receptor (ER) antagonist ICI or the NO synthase (NOS) inhibitor 7-nitroindazole via downregulation of hTERT\",\"the telomerase catalytic subunit\",\"suggesting that E2 and NO are involved in controlling hTERT transcription.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "NOTCH-problem; TF is not CREB but CREB-binding protein---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"NOTCH-problem; TF is not CREB but CREB-binding protein\"],\"sentence\":[\"Multiple coactivators\",\"including p300\",\"steroid receptor coactivator-1\",\"and p300/cAMP-response element-binding protein (CREB)-binding protein (CBP)-associated factor localize differentially to the E-selectin promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "NOTCH-problem; TF is not CREB but CREB-binding protein---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"NOTCH1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"NOTCH-problem; TF is not CREB but CREB-binding protein\"],\"sentence\":[\"Multiple coactivators\",\"including p300\",\"steroid receptor coactivator-1\",\"and p300/cAMP-response element-binding protein (CREB)-binding protein (CBP)-associated factor localize differentially to the E-selectin promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear whether TRE---{\"tf associated gene name\":[\"AHR\"],\"tg associated gene name\":[\"LTBP1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"f?\"],\"comment\":[\"unclear whether TRE\"],\"sentence\":[\"In agreement\",\"siRNA for HDAC2 increased Ltbp-1 expression and K8H4 acetylation in AhR+/+ but not in AhR-/- MEF cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "unclear whether TRE---{\"tf associated gene name\":[\"AHR\"],\"tg associated gene name\":[\"LTBP1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"f?\"],\"comment\":[\"unclear whether TRE\"],\"sentence\":[\"In agreement\",\"siRNA for HDAC2 increased Ltbp-1 expression and K8H4 acetylation in AhR+/+ but not in AhR-/- MEF cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Ascl1---{\"tf associated gene name\":[\"HES5\"],\"tg associated gene name\":[\"HES5\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"TF is Ascl1\"],\"sentence\":[\"Furthermore\",\"we demonstrated that Ascl1 directly binds to an E-box region within the Hes5 promoter and regulates Hes5 expression at the transcriptional level.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Ascl1---{\"tf associated gene name\":[\"HES5\"],\"tg associated gene name\":[\"HES5\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"t\"],\"comment\":[\"TF is Ascl1\"],\"sentence\":[\"Furthermore\",\"we demonstrated that Ascl1 directly binds to an E-box region within the Hes5 promoter and regulates Hes5 expression at the transcriptional level.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"ERVW-4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"TG not named\"],\"sentence\":[\"Collectively\",\"our results suggest that DeltaNp63alpha can function as a repressor of select p53 target genes involved in growth arrest\",\"DNA repair and apoptosis\",\"and that the location of the p53 consensus binding site(s) in a target gene may dictate whether pol II is constitutively bound in proliferating cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG not named---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"ERVW-4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"TG not named\"],\"sentence\":[\"Collectively\",\"our results suggest that DeltaNp63alpha can function as a repressor of select p53 target genes involved in growth arrest\",\"DNA repair and apoptosis\",\"and that the location of the p53 consensus binding site(s) in a target gene may dictate whether pol II is constitutively bound in proliferating cells.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is superoxide dismutase gene (sod1)---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is superoxide dismutase gene (sod1)\"],\"sentence\":[\"Using enhancer deletion studies we observed strong paraquat (PQ)-induced rtMT-A activation via NF-IL6 while the AP1 cis-elements showed a weak but significant activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG is rtMT-A---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is rtMT-A\"],\"sentence\":[\"Using enhancer deletion studies we observed strong paraquat (PQ)-induced rtMT-A activation via NF-IL6 while the AP1 cis-elements showed a weak but significant activation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is superoxide dismutase gene (sod1)---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is superoxide dismutase gene (sod1)\"],\"sentence\":[\"Using enhancer deletion studies we observed strong paraquat (PQ)-induced rtMT-A activation via NF-IL6 while the AP1 cis-elements showed a weak but significant activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---|TG is rtMT-A---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is rtMT-A\"],\"sentence\":[\"Using enhancer deletion studies we observed strong paraquat (PQ)-induced rtMT-A activation via NF-IL6 while the AP1 cis-elements showed a weak but significant activation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"SUPT7L\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Chromatin modifications are carried out\",\"in part\",\"through interaction with coregulator complexes\",\"including steroid coactivators (SRCs)\",\"corepressors (NcoR/SMRT)\",\"hepatocyte nuclear factor 4 alpha\",\"proliferator activated receptor ?? coactivator 1 alpha and protein arginine methyltransferase 1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"SUPT7L\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"Chromatin modifications are carried out\",\"in part\",\"through interaction with coregulator complexes\",\"including steroid coactivators (SRCs)\",\"corepressors (NcoR/SMRT)\",\"hepatocyte nuclear factor 4 alpha\",\"proliferator activated receptor ?? coactivator 1 alpha and protein arginine methyltransferase 1.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is TRB3---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[\"C/EBPbeta\"],\"tg text\":[\"insulin\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[\"TG is TRB3\"],\"sentence\":[\"We further showed that ectopic expression of C/EBPbeta augmented\",\"whereas knockdown of C/EBPbeta reduced\",\"TRB3 expression induced by insulin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is TRB3---{\"tf associated gene name\":[\"CEBPB\"],\"tg associated gene name\":[\"INS\"],\"tf text\":[\"C/EBPbeta\"],\"tg text\":[\"insulin\"],\"1\":[],\"negation\":[],\"not true\":[\"up\"],\"comment\":[\"TG is TRB3\"],\"sentence\":[\"We further showed that ectopic expression of C/EBPbeta augmented\",\"whereas knockdown of C/EBPbeta reduced\",\"TRB3 expression induced by insulin.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is DDX3X---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"TIMP3\"],\"tf text\":[],\"tg text\":[\"TIMP3\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"regulator is DDX3X\"],\"sentence\":[\"In Ishikawa cells\",\"gain of function of miR-98 repressed PGRMC1 and CYP19A1\",\"and miR-181a repressed PGR\",\"DDX3X\",\"and TIMP3 at mRNA and protein levels through direct interactions with their respective 3'-untranslated regions and CCNE1 through miR-181a-induced DDX3X repression\",\"with miR-98 reducing the rate of cell proliferation as compared with controls.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "regulator is DDX3X---{\"tf associated gene name\":[\"MLXIP\"],\"tg associated gene name\":[\"TIMP3\"],\"tf text\":[],\"tg text\":[\"TIMP3\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"regulator is DDX3X\"],\"sentence\":[\"In Ishikawa cells\",\"gain of function of miR-98 repressed PGRMC1 and CYP19A1\",\"and miR-181a repressed PGR\",\"DDX3X\",\"and TIMP3 at mRNA and protein levels through direct interactions with their respective 3'-untranslated regions and CCNE1 through miR-181a-induced DDX3X repression\",\"with miR-98 reducing the rate of cell proliferation as compared with controls.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|regulator not CREB---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"REN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"regulator not CREB\"],\"sentence\":[\"These studies underline the crucial importance of the CREB/CBP/p300 complex for the ability of renin cells to retain their cellular memory and regain renin expression\",\"a fundamental survival mechanism\",\"in response to a threat to homeostasis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE|regulator not CREB---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"REN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"regulator not CREB\"],\"sentence\":[\"These studies underline the crucial importance of the CREB/CBP/p300 complex for the ability of renin cells to retain their cellular memory and regain renin expression\",\"a fundamental survival mechanism\",\"in response to a threat to homeostasis.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear TRE-relations---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"unclear\"],\"comment\":[\"unclear TRE-relations\"],\"sentence\":[\"As c-jun is a direct target of the TCF4/beta-catenin complex\",\"the control of tcf4 expression by JNK/c-Jun leads to a positive feedback loop that connects JNK and Wnt signalling.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "unclear TRE-relations---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"MAPK8\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"unclear\"],\"comment\":[\"unclear TRE-relations\"],\"sentence\":[\"As c-jun is a direct target of the TCF4/beta-catenin complex\",\"the control of tcf4 expression by JNK/c-Jun leads to a positive feedback loop that connects JNK and Wnt signalling.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is nitric oxide synthase (iNOS) ---{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[\"TG is nitric oxide synthase (iNOS) \"],\"sentence\":[\"Although no consensus PPAR gamma:RXR-responsive element in the promoter regions of the inducible isoform of nitric oxide synthase (iNOS) and TNF-alpha genes was found\",\"PPAR gamma:RXR may interfere with NF-kappa B and AP-1 transcriptional activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is nitric oxide synthase (iNOS) ---{\"tf associated gene name\":[\"PPARG\"],\"tg associated gene name\":[\"AP1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"p\"],\"comment\":[\"TG is nitric oxide synthase (iNOS) \"],\"sentence\":[\"Although no consensus PPAR gamma:RXR-responsive element in the promoter regions of the inducible isoform of nitric oxide synthase (iNOS) and TNF-alpha genes was found\",\"PPAR gamma:RXR may interfere with NF-kappa B and AP-1 transcriptional activity.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE|no TG named---{\"tf associated gene name\":[\"KLF11\"],\"tg associated gene name\":[\"CBX5\"],\"tf text\":[\"KLF11\"],\"tg text\":[\"HP1\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TG named\"],\"sentence\":[\"KLF11 also recruits HP1?? and its histone methyltransferase\",\"SUV39H1\",\"to promoters to limit KLF11-mediated gene activation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE|no TG named---{\"tf associated gene name\":[\"KLF11\"],\"tg associated gene name\":[\"CBX5\"],\"tf text\":[\"KLF11\"],\"tg text\":[\"HP1\"],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\",\"no TG named\"],\"sentence\":[\"KLF11 also recruits HP1?? and its histone methyltransferase\",\"SUV39H1\",\"to promoters to limit KLF11-mediated gene activation.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ADAM-problem|TG is LHbeta---{\"tf associated gene name\":[\"NR5A1\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"ADAM-problem\",\"TG is LHbeta\"],\"sentence\":[\"GnRH stimulation of the LHbeta promoter requires interactions between a complex distal response element containing two specificity protein-1 (Sp1) binding sites and a CArG box\",\"and a proximal element with two bipartite binding sites for steroidogenic factor-1 and early growth response protein-1 (Egr-1).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "ADAM-problem|TG is LHbeta---{\"tf associated gene name\":[\"NR5A1\"],\"tg associated gene name\":[\"ADAM2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"ADAM-problem\",\"TG is LHbeta\"],\"sentence\":[\"GnRH stimulation of the LHbeta promoter requires interactions between a complex distal response element containing two specificity protein-1 (Sp1) binding sites and a CArG box\",\"and a proximal element with two bipartite binding sites for steroidogenic factor-1 and early growth response protein-1 (Egr-1).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is PB|uncertain normalization---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"SPINT2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TG is PB\",\"uncertain normalization\"],\"sentence\":[\"To characterize the determinants of chromatin specificity of the PB promoter with the AR we initially investigated the in vitro interactions of recombinant AR DNA binding domain (AR-DBD) with reconstituted nucleosomes incorporating the proximal PB promoter (nucleotides -268 to -76).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is PB|uncertain normalization---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"SPINT2\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u?\"],\"negation\":[],\"not true\":[\"n?\"],\"comment\":[\"TG is PB\",\"uncertain normalization\"],\"sentence\":[\"To characterize the determinants of chromatin specificity of the PB promoter with the AR we initially investigated the in vitro interactions of recombinant AR DNA binding domain (AR-DBD) with reconstituted nucleosomes incorporating the proximal PB promoter (nucleotides -268 to -76).\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is SMAD3---{\"tf associated gene name\":[\"SMAD2\"],\"tg associated gene name\":[\"GSC\"],\"tf text\":[\"Smad2\"],\"tg text\":[\"gsc\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF is SMAD3\"],\"sentence\":[\"Smad3 is closely related to Smad2 but suppresses activation of the gsc promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is SMAD3---{\"tf associated gene name\":[\"SMAD2\"],\"tg associated gene name\":[\"GSC\"],\"tf text\":[\"Smad2\"],\"tg text\":[\"gsc\"],\"1\":[],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TF is SMAD3\"],\"sentence\":[\"Smad3 is closely related to Smad2 but suppresses activation of the gsc promoter.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not clear TRE---{\"tf associated gene name\":[\"ETV6\"],\"tg associated gene name\":[\"EPOR\"],\"tf text\":[\"ETV6\"],\"tg text\":[\"EPOR\"],\"1\":[],\"negation\":[],\"not true\":[\"f?\"],\"comment\":[\"not clear TRE\"],\"sentence\":[\"We further reveal that murine and human ETV6-RUNX1(+) cells expressing EPOR mRNA have EPO ligand binding activity that correlates with an increased cell survival through activation of the JAK2-STAT5 pathway and up-regulation of antiapoptotic BCL-XL.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not clear TRE---{\"tf associated gene name\":[\"ETV6\"],\"tg associated gene name\":[\"EPOR\"],\"tf text\":[\"ETV6\"],\"tg text\":[\"EPOR\"],\"1\":[],\"negation\":[],\"not true\":[\"f?\"],\"comment\":[\"not clear TRE\"],\"sentence\":[\"We further reveal that murine and human ETV6-RUNX1(+) cells expressing EPOR mRNA have EPO ligand binding activity that correlates with an increased cell survival through activation of the JAK2-STAT5 pathway and up-regulation of antiapoptotic BCL-XL.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"NR2C2\"],\"tg associated gene name\":[\"IKBKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"We show herein that TGF-beta repressed nuclear factor-kappaB (NF-kappaB) activity in normal NMuMG cells\",\"but activated this transcription factor in their malignant counterparts\",\"4T1 cells\",\"by inducing assembly of TGF-beta-activated kinase 1 (TAK1)-binding protein 1 (TAB1):I kappaB kinase beta (IKK beta) complexes\",\"which led to the stimulation of a TAK1:IKK beta:p65 pathway.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"NR2C2\"],\"tg associated gene name\":[\"IKBKB\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"negation\":[],\"not true\":[\"f\"],\"comment\":[\"not TRE\"],\"sentence\":[\"We show herein that TGF-beta repressed nuclear factor-kappaB (NF-kappaB) activity in normal NMuMG cells\",\"but activated this transcription factor in their malignant counterparts\",\"4T1 cells\",\"by inducing assembly of TGF-beta-activated kinase 1 (TAK1)-binding protein 1 (TAB1):I kappaB kinase beta (IKK beta) complexes\",\"which led to the stimulation of a TAK1:IKK beta:p65 pathway.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Pdx1; effect on MAFA is indirect---{\"tf associated gene name\":[\"FOXA2\"],\"tg associated gene name\":[\"MAFA\"],\"tf text\":[\"FoxA2\"],\"tg text\":[\"MafA\"],\"1\":[\"a-ind\"],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is Pdx1; effect on MAFA is indirect\"],\"sentence\":[\"FoxO1 inhibits beta cell proliferation through suppression of Pdx1 by competing with FoxA2 and protects against beta cell failure induced by oxidative stress through NeuroD and MafA induction.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Pdx1; effect on MAFA is indirect---{\"tf associated gene name\":[\"FOXA2\"],\"tg associated gene name\":[\"MAFA\"],\"tf text\":[\"FoxA2\"],\"tg text\":[\"MafA\"],\"1\":[\"a-ind\"],\"negation\":[],\"not true\":[\"ni\"],\"comment\":[\"TG is Pdx1; effect on MAFA is indirect\"],\"sentence\":[\"FoxO1 inhibits beta cell proliferation through suppression of Pdx1 by competing with FoxA2 and protects against beta cell failure induced by oxidative stress through NeuroD and MafA induction.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p21(waf1/cdi1)---{\"tf associated gene name\":[\"CUX1\"],\"tg associated gene name\":[\"EHMT2\"],\"tf text\":[\"CDP/cut\"],\"tg text\":[\"G9a\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is p21(waf1/cdi1)\"],\"sentence\":[\"The endogenous level of p21(waf1/cdi1) expression is repressed through CDP/cut and mediated by HKMT activity of G9a.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is p21(waf1/cdi1)---{\"tf associated gene name\":[\"CUX1\"],\"tg associated gene name\":[\"EHMT2\"],\"tf text\":[\"CDP/cut\"],\"tg text\":[\"G9a\"],\"1\":[],\"negation\":[],\"not true\":[\"n\"],\"comment\":[\"TG is p21(waf1/cdi1)\"],\"sentence\":[\"The endogenous level of p21(waf1/cdi1) expression is repressed through CDP/cut and mediated by HKMT activity of G9a.\"],\"_\":[\"_\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG uncertain---{\"tf associated gene name\":[\"USF1\"],\"tg associated gene name\":[\"AVP\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"not true\":[\"n?\"],\"comment\":[\"TG uncertain\"],\"sentence\":[\"LPS increased NF-kappaB and USF binding to the ADH promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TG uncertain---{\"tf associated gene name\":[\"USF1\"],\"tg associated gene name\":[\"AVP\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u\"],\"not true\":[\"n?\"],\"comment\":[\"TG uncertain\"],\"sentence\":[\"LPS increased NF-kappaB and USF binding to the ADH promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "random IRF6---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NFY\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[\"random IRF6\"],\"sentence\":[\"Transcriptional regulation of the murine C5a receptor gene: NF-Y is required for basal and LPS induced expression in macrophages and endothelial cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "random IRF6---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NFY\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[\"random IRF6\"],\"sentence\":[\"Transcriptional regulation of the murine C5a receptor gene: NF-Y is required for basal and LPS induced expression in macrophages and endothelial cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no clear regulation ---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"KLK3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[\"no clear regulation \"],\"sentence\":[\"We examined the role of AKT in AR activity and PSA expression by creating stable PC3(AR)(2) cell lines transfected with a PI3K-Ras-effector loop mutant.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no clear regulation ---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"KLK3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"not true\":[\"f\"],\"comment\":[\"no clear regulation \"],\"sentence\":[\"We examined the role of AKT in AR activity and PSA expression by creating stable PC3(AR)(2) cell lines transfected with a PI3K-Ras-effector loop mutant.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"SGCG\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"The human folate receptor (hFR) type gamma gene is driven by a TATA-less promoter that uses a canonical Sp1 element for basal transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"SGCG\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[],\"not true\":[\"n\"],\"comment\":[\"TG\"],\"sentence\":[\"The human folate receptor (hFR) type gamma gene is driven by a TATA-less promoter that uses a canonical Sp1 element for basal transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG---{\"tf associated gene name\":[\"ELF1\"],\"tg associated gene name\":[\"KRT14\"],\"tf text\":[\"Elf-1\"],\"tg text\":[\"EBS4\"],\"1\":[],\"not true\":[\"ni\"],\"comment\":[\"no TG\"],\"sentence\":[\"This activation is only partly affected by mutation of EBS4\",\"and a mutant promoter that binds Ets-1\",\"but not Elf-1\",\"at the EBS4 site responds to PMA/ionomycin as efficiently as the wild-type.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TG---{\"tf associated gene name\":[\"ELF1\"],\"tg associated gene name\":[\"KRT14\"],\"tf text\":[\"Elf-1\"],\"tg text\":[\"EBS4\"],\"1\":[],\"not true\":[\"ni\"],\"comment\":[\"no TG\"],\"sentence\":[\"This activation is only partly affected by mutation of EBS4\",\"and a mutant promoter that binds Ets-1\",\"but not Elf-1\",\"at the EBS4 site responds to PMA/ionomycin as efficiently as the wild-type.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is VEGF---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is VEGF\"],\"sentence\":[\"Taken together\",\"these results demonstrate that PGE(2) plays an important role in regulating uterine and thus placental vascular development\",\"acting through its receptors EP2 and EP4\",\"provoking protein kinase A activation of AP2 and Sp1 as well as acetylation of histone H3 to transactivate the VEGF promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is VEGF---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"HIST1H3F\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG is VEGF\"],\"sentence\":[\"Taken together\",\"these results demonstrate that PGE(2) plays an important role in regulating uterine and thus placental vascular development\",\"acting through its receptors EP2 and EP4\",\"provoking protein kinase A activation of AP2 and Sp1 as well as acetylation of histone H3 to transactivate the VEGF promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TRE-sentence---{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"DKK1\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"DKK1\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TRE-sentence\"],\"sentence\":[\"We detected altered expression of >1\",\"000 genes\",\"including targets regulated directly by OCT4 either positively (NANOG\",\"SOX2\",\"REX1\",\"LEFTB\",\"LEFTA/EBAF DPPA4\",\"THY1\",\"and TDGF1) or negatively (CDX2\",\"EOMES\",\"BMP4\",\"TBX18\",\"Brachyury [T]\",\"DKK1\",\"HLX1\",\"GATA6\",\"ID2\",\"and DLX5)\",\"as well as targets for the OCT4-associated stem cell regulators SOX2 and NANOG.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TRE-sentence---{\"tf associated gene name\":[\"CDX2\"],\"tg associated gene name\":[\"DKK1\"],\"tf text\":[\"CDX2\"],\"tg text\":[\"DKK1\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TRE-sentence\"],\"sentence\":[\"We detected altered expression of >1\",\"000 genes\",\"including targets regulated directly by OCT4 either positively (NANOG\",\"SOX2\",\"REX1\",\"LEFTB\",\"LEFTA/EBAF DPPA4\",\"THY1\",\"and TDGF1) or negatively (CDX2\",\"EOMES\",\"BMP4\",\"TBX18\",\"Brachyury [T]\",\"DKK1\",\"HLX1\",\"GATA6\",\"ID2\",\"and DLX5)\",\"as well as targets for the OCT4-associated stem cell regulators SOX2 and NANOG.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG wrongly normalized---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"SAMHD1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG wrongly normalized\"],\"sentence\":[\"IFN-gamma caused phosphorylation of CREB\",\"and overexpression of CREB increased IFN-gamma-induced BAFF promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG wrongly normalized---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"SAMHD1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG wrongly normalized\"],\"sentence\":[\"IFN-gamma caused phosphorylation of CREB\",\"and overexpression of CREB increased IFN-gamma-induced BAFF promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is vascular cell adhesion molecule-1---{\"tf associated gene name\":[\"NFATC1\"],\"tg associated gene name\":[\"F2\"],\"tf text\":[\"NFATc\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is vascular cell adhesion molecule-1\"],\"sentence\":[\"Using a combination of transient transfections\",\"electrophoretic mobility shift assays\",\"and chromatin immunoprecipitation\",\"thrombin was shown to induce time-dependent coordinate binding of RelA and NFATc to a tandem NF-kappaB element in the upstream promoter region of vascular cell adhesion molecule-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is vascular cell adhesion molecule-1---{\"tf associated gene name\":[\"NFATC1\"],\"tg associated gene name\":[\"F2\"],\"tf text\":[\"NFATc\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is vascular cell adhesion molecule-1\"],\"sentence\":[\"Using a combination of transient transfections\",\"electrophoretic mobility shift assays\",\"and chromatin immunoprecipitation\",\"thrombin was shown to induce time-dependent coordinate binding of RelA and NFATc to a tandem NF-kappaB element in the upstream promoter region of vascular cell adhesion molecule-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is CBF-1 ---{\"tf associated gene name\":[\"FOXN1\"],\"tg associated gene name\":[\"PTEN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is CBF-1 \"],\"sentence\":[\"Our data revealed that AKT1 mRNA and protein expression were induced by doxorubicin (a chemotherapeutic agent); the doxorubicin-induced AKT1 expression and activation increased the binding of NF-kappaB on Notch1 DNA promoter and then promoted the Notch1 transcription and expression; enhanced expression of Notch1 further upregulated PTEN expression through CBF-1 binding to PTEN DNA promoter; and inhibition of AKT1 expression and activity sensitized the gastric cancer cell to doxorubicin treatment in cultured gastric cancer cell lines and xenograft nude mice gastric cancer model.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is CBF-1 ---{\"tf associated gene name\":[\"FOXN1\"],\"tg associated gene name\":[\"PTEN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is CBF-1 \"],\"sentence\":[\"Our data revealed that AKT1 mRNA and protein expression were induced by doxorubicin (a chemotherapeutic agent); the doxorubicin-induced AKT1 expression and activation increased the binding of NF-kappaB on Notch1 DNA promoter and then promoted the Notch1 transcription and expression; enhanced expression of Notch1 further upregulated PTEN expression through CBF-1 binding to PTEN DNA promoter; and inhibition of AKT1 expression and activity sensitized the gastric cancer cell to doxorubicin treatment in cultured gastric cancer cell lines and xenograft nude mice gastric cancer model.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs not named---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"FKBP4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TGs not named\"],\"sentence\":[\"FKBP52 (-/-) uteri showed a normal growth response to estradiol\",\"and unaltered expression of genes controlled by ER and PR-B.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TGs not named---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"FKBP4\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TGs not named\"],\"sentence\":[\"FKBP52 (-/-) uteri showed a normal growth response to estradiol\",\"and unaltered expression of genes controlled by ER and PR-B.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"PSEN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"PS1/PS2 act as endoplasmic reticulum (ER) Ca????? leak channels that facilitate passive Ca????? leak across ER membrane.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"PSEN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"PS1/PS2 act as endoplasmic reticulum (ER) Ca????? leak channels that facilitate passive Ca????? leak across ER membrane.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF named---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TF named\"],\"sentence\":[\"Inducible nitric-oxide synthase (iNOS) is an important signaling protein involved in the regulation of biological processes (e.g. vasodilation\",\"inflammation) and is subject to transcriptional regulation by cytokines and lipopolysaccharide (LPS).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TF named---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"NOS2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"no TF named\"],\"sentence\":[\"Inducible nitric-oxide synthase (iNOS) is an important signaling protein involved in the regulation of biological processes (e.g. vasodilation\",\"inflammation) and is subject to transcriptional regulation by cytokines and lipopolysaccharide (LPS).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"NF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"The progesterone receptor (PR) binds to a cluster of five hormone responsive elements (HREs) and activates the promoter by synergistic interactions with the ubiquitous transcription factor\",\"nuclear factor 1 (NF1).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG not named---{\"tf associated gene name\":[\"PGR\"],\"tg associated gene name\":[\"NF1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"The progesterone receptor (PR) binds to a cluster of five hormone responsive elements (HREs) and activates the promoter by synergistic interactions with the ubiquitous transcription factor\",\"nuclear factor 1 (NF1).\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is BK2---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"EDNRA\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is BK2\"],\"sentence\":[\"These results demonstrate that the rat BK2 promoter is a target for p53-mediated activation and suggest a new physiological role for p53 in the regulation of G protein-coupled receptor gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is BK2---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"EDNRA\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is BK2\"],\"sentence\":[\"These results demonstrate that the rat BK2 promoter is a target for p53-mediated activation and suggest a new physiological role for p53 in the regulation of G protein-coupled receptor gene expression.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is prostate-specific antigen---{\"tf associated gene name\":[\"CUX1\"],\"tg associated gene name\":[\"NTRK1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is prostate-specific antigen\"],\"sentence\":[\"To determine if endogenous NGF is required for the normal developmental increase in p75 and TrkA expression that occurs in sensory neurons shortly after they innervate their targets\",\"we used quantitative RT/PCR to compare the levels of p75 and trkA mRNAs in the trigeminal ganglia of normal mouse embryos and embryos that are homozygous for a null mutation in the NGF gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is prostate-specific antigen---{\"tf associated gene name\":[\"CUX1\"],\"tg associated gene name\":[\"NTRK1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is prostate-specific antigen\"],\"sentence\":[\"To determine if endogenous NGF is required for the normal developmental increase in p75 and TrkA expression that occurs in sensory neurons shortly after they innervate their targets\",\"we used quantitative RT/PCR to compare the levels of p75 and trkA mRNAs in the trigeminal ganglia of normal mouse embryos and embryos that are homozygous for a null mutation in the NGF gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"CDX1\"],\"tg associated gene name\":[\"LBX1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"The Drosophila caudal homeobox gene is required for definition of the anteroposterior axis and for gut development\",\"and CDX1 and CDX2\",\"human homologs\",\"play crucial roles in the regulation of cell proliferation and differentiation in the intestine.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not GRE/TRE---{\"tf associated gene name\":[\"CDX1\"],\"tg associated gene name\":[\"LBX1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not GRE/TRE\"],\"sentence\":[\"The Drosophila caudal homeobox gene is required for definition of the anteroposterior axis and for gut development\",\"and CDX1 and CDX2\",\"human homologs\",\"play crucial roles in the regulation of cell proliferation and differentiation in the intestine.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"TBP\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[\"TBP\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Not only does TAF-172 activate pol III transcription\",\"but it also inhibits the binding of TBP to the TATA box\",\"thereby repressing pol II transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG not named---{\"tf associated gene name\":[\"TBP\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[\"TBP\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Not only does TAF-172 activate pol III transcription\",\"but it also inhibits the binding of TBP to the TATA box\",\"thereby repressing pol II transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p21 (likely CDKN1A)---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"GTF2A1\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is p21 (likely CDKN1A)\"],\"sentence\":[\"The results of the present study suggest that the upstream TATA-less promoter of p21 needs TFIIA and TFIIA-reactive TLP for p53-dependent transcriptional enhancement.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is p21 (likely CDKN1A)---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"GTF2A1\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is p21 (likely CDKN1A)\"],\"sentence\":[\"The results of the present study suggest that the upstream TATA-less promoter of p21 needs TFIIA and TFIIA-reactive TLP for p53-dependent transcriptional enhancement.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is E-cadherin---{\"tf associated gene name\":[\"ZEB1\"],\"tg associated gene name\":[\"CD44\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E-cadherin\"],\"sentence\":[\"Further studies indicated that Cr(VI) uses multiple mechanisms to repress E-cadherin expression\",\"including activation of E-cadherin repressors such as Slug\",\"ZEB1\",\"KLF8 and enhancement the binding of HDAC1 in E-cadherin gene promoter\",\"but DNA methylation is not responsible for the loss of E-cadherin.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is E-cadherin---{\"tf associated gene name\":[\"ZEB1\"],\"tg associated gene name\":[\"CD44\"],\"tf text\":[\"ZEB1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E-cadherin\"],\"sentence\":[\"Further studies indicated that Cr(VI) uses multiple mechanisms to repress E-cadherin expression\",\"including activation of E-cadherin repressors such as Slug\",\"ZEB1\",\"KLF8 and enhancement the binding of HDAC1 in E-cadherin gene promoter\",\"but DNA methylation is not responsible for the loss of E-cadherin.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "GRE|but TF not named---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"CIITA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"GRE\",\"but TF not named\"],\"sentence\":[\"Transient transfection assays demonstrated that the CIITA type III promoter is active in class II(-) L1210 cells\",\"despite the fact that the endogenous gene is not expressed\",\"which suggests that these cells have all of the transacting factors necessary for CIITA transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "GRE|but TF not named---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"CIITA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"GRE\",\"but TF not named\"],\"sentence\":[\"Transient transfection assays demonstrated that the CIITA type III promoter is active in class II(-) L1210 cells\",\"despite the fact that the endogenous gene is not expressed\",\"which suggests that these cells have all of the transacting factors necessary for CIITA transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is EVI1|TG not named---{\"tf associated gene name\":[\"RUNX1\"],\"tg associated gene name\":[\"MIR124-3\"],\"tf text\":[\"EVI1\"],\"tg text\":[\"miR-124-3\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF is EVI1\",\"TG not named\"],\"sentence\":[\"The mechanisms by which EVI1 transforms normal cells are unknown\",\"but we showed recently that EVI1 indirectly upregulates self-renewal and cell-cycling genes by inappropriate methylation of CpG dinucleotides in the regulatory regions of microRNA-124-3 (miR-124-3)\",\"leading to the repression of this small gene that controls normal differentiation and cell cycling of somatic cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is EVI1|TG not named---{\"tf associated gene name\":[\"RUNX1\"],\"tg associated gene name\":[\"MIR124-3\"],\"tf text\":[\"EVI1\"],\"tg text\":[\"miR-124-3\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TF is EVI1\",\"TG not named\"],\"sentence\":[\"The mechanisms by which EVI1 transforms normal cells are unknown\",\"but we showed recently that EVI1 indirectly upregulates self-renewal and cell-cycling genes by inappropriate methylation of CpG dinucleotides in the regulatory regions of microRNA-124-3 (miR-124-3)\",\"leading to the repression of this small gene that controls normal differentiation and cell cycling of somatic cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "is GRE/TRE---{\"tf associated gene name\":[\"NFAT5\"],\"tg associated gene name\":[\"ATM\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"is GRE/TRE\"],\"sentence\":[\"Additionally\",\"antibody to ATM kinase supershifts TonEBP/OREBP bound to its cognate ORE/TonE DNA element.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "is GRE/TRE---{\"tf associated gene name\":[\"NFAT5\"],\"tg associated gene name\":[\"ATM\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"is GRE/TRE\"],\"sentence\":[\"Additionally\",\"antibody to ATM kinase supershifts TonEBP/OREBP bound to its cognate ORE/TonE DNA element.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG named (I think)---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"DLST\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"no TG named (I think)\"],\"sentence\":[\"However\",\"when delta5ERalpha was cotransfected with normal ERalpha\",\"both basal and E2-stimulated EREtk81Luc reporter activation were increased approximately 500% over levels observed when cells were transfected with ERalpha alone.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TG named (I think)---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"DLST\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"no TG named (I think)\"],\"sentence\":[\"However\",\"when delta5ERalpha was cotransfected with normal ERalpha\",\"both basal and E2-stimulated EREtk81Luc reporter activation were increased approximately 500% over levels observed when cells were transfected with ERalpha alone.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TF named; TGs are COL1A1 and COL1A2---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"ADRA1D\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TF named; TGs are COL1A1 and COL1A2\"],\"sentence\":[\"Increased transactivation of the COL1A2 promoter assessed by betaGal activity was shown in vivo and by transfection with deletion constructs for the collagen alpha1(I) promoter (COL1A1) and COL1A2 promoters in vitro.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TF named; TGs are COL1A1 and COL1A2---{\"tf associated gene name\":[\"TCF3\"],\"tg associated gene name\":[\"ADRA1D\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"no TF named; TGs are COL1A1 and COL1A2\"],\"sentence\":[\"Increased transactivation of the COL1A2 promoter assessed by betaGal activity was shown in vivo and by transfection with deletion constructs for the collagen alpha1(I) promoter (COL1A1) and COL1A2 promoters in vitro.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "MSX1 is TF; TBX22 is TG---{\"tf associated gene name\":[\"TBX22\"],\"tg associated gene name\":[\"MSX1\"],\"tf text\":[\"Tbx22\"],\"tg text\":[\"MSX1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[\"x\"],\"comment\":[\"MSX1 is TF; TBX22 is TG\"],\"sentence\":[\"We suggest that these species-specific differences may result at least in part from differences in the spatiotemporal patterns of BMP activity\",\"but we exclude a direct repression of Tbx22 by the BMP-inducible transcriptional repressor MSX1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[true],\"QC\":[null]}\n",
      "MSX1 is TF; TBX22 is TG---{\"tf associated gene name\":[\"TBX22\"],\"tg associated gene name\":[\"MSX1\"],\"tf text\":[\"Tbx22\"],\"tg text\":[\"MSX1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[\"x\"],\"comment\":[\"MSX1 is TF; TBX22 is TG\"],\"sentence\":[\"We suggest that these species-specific differences may result at least in part from differences in the spatiotemporal patterns of BMP activity\",\"but we exclude a direct repression of Tbx22 by the BMP-inducible transcriptional repressor MSX1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[true],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is chicken histone H5 ---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\" ni\",\"s?\"],\"negation\":[],\"comment\":[\"TG is chicken histone H5 \"],\"sentence\":[\"In vitro DNAase I footprinting and gel mobility shift assays have shown that the activities of several nuclear factors (GATA-1\",\"Sp1) that bind to the promoter and downstream enhancer regions of the chicken histone H5 gene are reduced in mature erythrocytes relative to those in immature erythrocytes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is chicken histone H5 ---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"FGFR1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\" ni\",\"s?\"],\"negation\":[],\"comment\":[\"TG is chicken histone H5 \"],\"sentence\":[\"In vitro DNAase I footprinting and gel mobility shift assays have shown that the activities of several nuclear factors (GATA-1\",\"Sp1) that bind to the promoter and downstream enhancer regions of the chicken histone H5 gene are reduced in mature erythrocytes relative to those in immature erythrocytes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is TGIF---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"IL10\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"regulator is TGIF\"],\"sentence\":[\"Decreased TGIF expression resulted in reduced proliferation and differentiation and lower expression of CEBPbeta\",\"CEBPepsilon\",\"PU.1 and RUNX1\",\"key myeloid transcription factors.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "regulator is TGIF---{\"tf associated gene name\":[\"SPI1\"],\"tg associated gene name\":[\"IL10\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"regulator is TGIF\"],\"sentence\":[\"Decreased TGIF expression resulted in reduced proliferation and differentiation and lower expression of CEBPbeta\",\"CEBPepsilon\",\"PU.1 and RUNX1\",\"key myeloid transcription factors.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "no TG named---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"CD8A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"In contrast to CD8(+)Des(+) T cells activated in vivo with APCs\",\"which express high AP-1 and high NF-kappa B transcriptional activity\",\"the unresponsive CD8(+)Des(+) T cells expressed no AP-1 and only weak NF-kappa B transcriptional activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "no TG named---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"CD8A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"no TG named\"],\"sentence\":[\"In contrast to CD8(+)Des(+) T cells activated in vivo with APCs\",\"which express high AP-1 and high NF-kappa B transcriptional activity\",\"the unresponsive CD8(+)Des(+) T cells expressed no AP-1 and only weak NF-kappa B transcriptional activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is CHOP---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"IL8\"],\"tf text\":[],\"tg text\":[\"IL-8/activator protein 1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is CHOP\"],\"sentence\":[\"Transfection experiments with mutants showed both that the CHOP transactivation domain is essential for IL-8 transcription and that the IL-8/activator protein 1 (AP-1) promoter mutated in NF-kappaB and NF-IL-6\",\"but not in the AP-1 site\",\"harbors essential CHOP-responsive elements.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is CHOP---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"IL8\"],\"tf text\":[],\"tg text\":[\"IL-8/activator protein 1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF is CHOP\"],\"sentence\":[\"Transfection experiments with mutants showed both that the CHOP transactivation domain is essential for IL-8 transcription and that the IL-8/activator protein 1 (AP-1) promoter mutated in NF-kappaB and NF-IL-6\",\"but not in the AP-1 site\",\"harbors essential CHOP-responsive elements.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "regulator is BCA---{\"tf associated gene name\":[\"ATF3\"],\"tg associated gene name\":[\"GADD45A\"],\"tf text\":[],\"tg text\":[\"GADD45A\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"regulator is BCA\"],\"sentence\":[\"BCA upregulates proapoptotic genes such as activating transcription factor 3 (ATF3)\",\"NSAID-activated gene 1 protein (NAG-1)\",\"and growth arrest and DNA-damage-inducible protein alpha (GADD45A) in prostate cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "regulator is BCA---{\"tf associated gene name\":[\"ATF3\"],\"tg associated gene name\":[\"GADD45A\"],\"tf text\":[],\"tg text\":[\"GADD45A\"],\"1\":[],\"0\":[\"c\"],\"negation\":[],\"comment\":[\"regulator is BCA\"],\"sentence\":[\"BCA upregulates proapoptotic genes such as activating transcription factor 3 (ATF3)\",\"NSAID-activated gene 1 protein (NAG-1)\",\"and growth arrest and DNA-damage-inducible protein alpha (GADD45A) in prostate cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p21/waf1 (CDKN1A)---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"RAF1\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is p21/waf1 (CDKN1A)\"],\"sentence\":[\"We observed that transcriptional repression of the p21/waf1 gene by NS5A is p53 dependent by using p53 wild-type (+/+) and null (-/-) cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is p21/waf1 (CDKN1A)---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"RAF1\"],\"tf text\":[\"p53\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is p21/waf1 (CDKN1A)\"],\"sentence\":[\"We observed that transcriptional repression of the p21/waf1 gene by NS5A is p53 dependent by using p53 wild-type (+/+) and null (-/-) cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is HTLV-1---{\"tf associated gene name\":[\"BCL3\"],\"tg associated gene name\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\",\"s?\"],\"negation\":[],\"comment\":[\"TG is HTLV-1\"],\"sentence\":[\"Interestingly\",\"we found that Bcl3 formed a stable complex with Tax and that this complex potently inhibited Tax-dependent HTLV-1 transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is HTLV-1---{\"tf associated gene name\":[\"BCL3\"],\"tg associated gene name\":[\"CNTN2\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\",\"s?\"],\"negation\":[],\"comment\":[\"TG is HTLV-1\"],\"sentence\":[\"Interestingly\",\"we found that Bcl3 formed a stable complex with Tax and that this complex potently inhibited Tax-dependent HTLV-1 transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE; regulator is non-TF HFH-8; TG is P-selectin; ---{\"tf associated gene name\":[\"FOXF1\"],\"tg associated gene name\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE; regulator is non-TF HFH-8; TG is P-selectin; \"],\"sentence\":[\"Cotransfection assays with one of these target promoters\",\"P-selectin\",\"demonstrated that HFH-8 expression was required for IL-6 stimulation of P-selectin promoter activity and suggest that HFH-8 is involved in mediating its cell-specific transcriptional activation in response to cytokines.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE; regulator is non-TF HFH-8; TG is P-selectin; ---{\"tf associated gene name\":[\"FOXF1\"],\"tg associated gene name\":[\"CEL\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE; regulator is non-TF HFH-8; TG is P-selectin; \"],\"sentence\":[\"Cotransfection assays with one of these target promoters\",\"P-selectin\",\"demonstrated that HFH-8 expression was required for IL-6 stimulation of P-selectin promoter activity and suggest that HFH-8 is involved in mediating its cell-specific transcriptional activation in response to cytokines.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"GCG\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"cAMP-responsive element-binding protein (CREB)-regulated transcription coactivator 2 (CRTC2) regulates transcription of gluconeogenic genes by specifying targets for the transcription factor CREB in response to glucagon.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG not named---{\"tf associated gene name\":[\"CREB1\"],\"tg associated gene name\":[\"GCG\"],\"tf text\":[\"CREB\"],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"cAMP-responsive element-binding protein (CREB)-regulated transcription coactivator 2 (CRTC2) regulates transcription of gluconeogenic genes by specifying targets for the transcription factor CREB in response to glucagon.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is PPARbeta/delta---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"SIRT1\"],\"tf text\":[],\"tg text\":[\"SIRT1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is PPARbeta/delta\"],\"sentence\":[\"In contrast\",\"expression of PPARbeta/delta (PPARdelta markedly increased the 5' -promoter activity of SIRT1 gene\",\"which was further amplified by the addition of GW501516\",\"a selective PPARdelta agonist.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is PPARbeta/delta---{\"tf associated gene name\":[\"YY1\"],\"tg associated gene name\":[\"SIRT1\"],\"tf text\":[],\"tg text\":[\"SIRT1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is PPARbeta/delta\"],\"sentence\":[\"In contrast\",\"expression of PPARbeta/delta (PPARdelta markedly increased the 5' -promoter activity of SIRT1 gene\",\"which was further amplified by the addition of GW501516\",\"a selective PPARdelta agonist.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG not named---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"NELFE\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Electrophoretic mobility shift assay of nuclear protein extracts from cells treated with 12(R)-HETrE with specific radiolabeled oligonucleotides corresponding to known transcriptional binding sites\",\"including AP-1\",\"AP-2\",\"SP1\",\"TRE\",\"NF kappa B\",\"TFIID\",\"OKT1\",\"CREB\",\"CTF/NF1\",\"and GRE demonstrated a markedly rapid and specific increase in the binding activity of NF kappa B and to a lesser extent\",\"AP-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG not named---{\"tf associated gene name\":[\"NFKB\"],\"tg associated gene name\":[\"NELFE\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG not named\"],\"sentence\":[\"Electrophoretic mobility shift assay of nuclear protein extracts from cells treated with 12(R)-HETrE with specific radiolabeled oligonucleotides corresponding to known transcriptional binding sites\",\"including AP-1\",\"AP-2\",\"SP1\",\"TRE\",\"NF kappa B\",\"TFIID\",\"OKT1\",\"CREB\",\"CTF/NF1\",\"and GRE demonstrated a markedly rapid and specific increase in the binding activity of NF kappa B and to a lesser extent\",\"AP-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF? (regulator)  is CIITA|TG is IL-4---{\"tf associated gene name\":[\"HAND2\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF? (regulator)  is CIITA\",\"TG is IL-4\"],\"sentence\":[\"These results imply that CIITA is not a direct Th1-specific repressor of the IL-4 gene and that tight control over the expression of CIITA and MHC-II is required to maintain the normal balance between Th1 and Th2 responses.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF? (regulator)  is CIITA|TG is IL-4---{\"tf associated gene name\":[\"HAND2\"],\"tg associated gene name\":[\"CD74\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TF? (regulator)  is CIITA\",\"TG is IL-4\"],\"sentence\":[\"These results imply that CIITA is not a direct Th1-specific repressor of the IL-4 gene and that tight control over the expression of CIITA and MHC-II is required to maintain the normal balance between Th1 and Th2 responses.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is human biotin transporter hSMVT---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"ABCC3\"],\"tf text\":[\"AP-2\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is human biotin transporter hSMVT\"],\"sentence\":[\"Regulation of the human biotin transporter hSMVT promoter by KLF-4 and AP-2: confirmation of promoter activity in vivo.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is human biotin transporter hSMVT---{\"tf associated gene name\":[\"TFAP2A\"],\"tg associated gene name\":[\"ABCC3\"],\"tf text\":[\"AP-2\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is human biotin transporter hSMVT\"],\"sentence\":[\"Regulation of the human biotin transporter hSMVT promoter by KLF-4 and AP-2: confirmation of promoter activity in vivo.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is E-cadherin (CDH1)---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[\"ERalpha\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E-cadherin (CDH1)\"],\"sentence\":[\"Therefore\",\"our results suggest a novel role for the ERalpha that plays the dual role of ligand-independent activator and ligand-dependent repressor of E-cadherin in breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is E-cadherin (CDH1)---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[\"ERalpha\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E-cadherin (CDH1)\"],\"sentence\":[\"Therefore\",\"our results suggest a novel role for the ERalpha that plays the dual role of ligand-independent activator and ligand-dependent repressor of E-cadherin in breast cancer cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Ig heavy chain germline Ig??---{\"tf associated gene name\":[\"SFPQ\"],\"tg associated gene name\":[\"HDAC1\"],\"tf text\":[],\"tg text\":[\"histone deacetylase 1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is Ig heavy chain germline Ig??\"],\"sentence\":[\"PSF recruited histone deacetylase 1 (HDAC1) to the STAT6 transcription complex\",\"which resulted in reduction of H3 acetylation at the promoter regions of Ig heavy chain germline Ig?? and inhibition of STAT6-mediated transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Ig heavy chain germline Ig??---{\"tf associated gene name\":[\"SFPQ\"],\"tg associated gene name\":[\"HDAC1\"],\"tf text\":[],\"tg text\":[\"histone deacetylase 1\"],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"TG is Ig heavy chain germline Ig??\"],\"sentence\":[\"PSF recruited histone deacetylase 1 (HDAC1) to the STAT6 transcription complex\",\"which resulted in reduction of H3 acetylation at the promoter regions of Ig heavy chain germline Ig?? and inhibition of STAT6-mediated transcription.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is TERT---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"HDAC2\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"HDAC2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is TERT\"],\"sentence\":[\"By interacting with Sp1\",\"HMGA2 interferes with the recruitment of HDAC2 to the hTERT proximal promoter\",\"enhancing localized histone H3-K9 acetylation and thereby stimulating hTERT expression and telomerase activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is TERT---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"HDAC2\"],\"tf text\":[\"Sp1\"],\"tg text\":[\"HDAC2\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is TERT\"],\"sentence\":[\"By interacting with Sp1\",\"HMGA2 interferes with the recruitment of HDAC2 to the hTERT proximal promoter\",\"enhancing localized histone H3-K9 acetylation and thereby stimulating hTERT expression and telomerase activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unsure about normalization of TG MTP---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"SLC40A1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"unsure about normalization of TG MTP\"],\"sentence\":[\"Different HNF-1 proteins synergistically enhance MTP promoter activity along with HNF-4?? by binding to different cis elements.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "unsure about normalization of TG MTP---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"SLC40A1\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"unsure about normalization of TG MTP\"],\"sentence\":[\"Different HNF-1 proteins synergistically enhance MTP promoter activity along with HNF-4?? by binding to different cis elements.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is TGF-beta type II receptor---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"TCF3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is TGF-beta type II receptor\"],\"sentence\":[\"Rap1 reverses transcriptional repression of TGF-beta type II receptor by a mechanism involving AP-1 in the human pancreatic cancer cell line\",\"UK Pan-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is TGF-beta type II receptor---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"TCF3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is TGF-beta type II receptor\"],\"sentence\":[\"Rap1 reverses transcriptional repression of TGF-beta type II receptor by a mechanism involving AP-1 in the human pancreatic cancer cell line\",\"UK Pan-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is EBV promoter|most likely \"epstein bar virus\" promoter---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"FOS\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\",\"s?\"],\"negation\":[],\"comment\":[\"\\\"TG\\\" is EBV promoter\",\"most likely \\\"epstein bar virus\\\" promoter\"],\"sentence\":[\"One of them (ZRE-M) overlaps with a consensus TPA responsive element (TRE) defined as an AP-1/c-jun/c-fos binding site and is located in an EBV promoter controlling the expression of the post-transcriptional activator EB2.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is EBV promoter|most likely \"epstein bar virus\" promoter---{\"tf associated gene name\":[\"JUN\"],\"tg associated gene name\":[\"FOS\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\",\"s?\"],\"negation\":[],\"comment\":[\"\\\"TG\\\" is EBV promoter\",\"most likely \\\"epstein bar virus\\\" promoter\"],\"sentence\":[\"One of them (ZRE-M) overlaps with a consensus TPA responsive element (TRE) defined as an AP-1/c-jun/c-fos binding site and is located in an EBV promoter controlling the expression of the post-transcriptional activator EB2.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is Mecp2---{\"tf associated gene name\":[\"TFCP2\"],\"tg associated gene name\":[\"BDNF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Mecp2\"],\"sentence\":[\"Therefore\",\"the present study used Mecp2 null mice to further define the role of MeCP2 in regulation of BDNF expression and neural function\",\"focusing on NG neurons and respiratory control.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is Mecp2---{\"tf associated gene name\":[\"TFCP2\"],\"tg associated gene name\":[\"BDNF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is Mecp2\"],\"sentence\":[\"Therefore\",\"the present study used Mecp2 null mice to further define the role of MeCP2 in regulation of BDNF expression and neural function\",\"focusing on NG neurons and respiratory control.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG should prbably be IFNA1---{\"tf associated gene name\":[\"IRF3\"],\"tg associated gene name\":[\"IFN1@\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG should prbably be IFNA1\"],\"sentence\":[\"The presence of vIRF-3 in the enhanceosome assembled on the IFNA promoters increases binding of IRF-3\",\"IRF-7\",\"and acetylated histone H3 to this promoter region.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TG should prbably be IFNA1---{\"tf associated gene name\":[\"IRF3\"],\"tg associated gene name\":[\"IFN1@\"],\"tf text\":[],\"tg text\":[],\"1\":[\"u \"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG should prbably be IFNA1\"],\"sentence\":[\"The presence of vIRF-3 in the enhanceosome assembled on the IFNA promoters increases binding of IRF-3\",\"IRF-7\",\"and acetylated histone H3 to this promoter region.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "unclear whether AP2 is regulated by HNF4a via transcription of PPI---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"HNF4A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"unclear whether AP2 is regulated by HNF4a via transcription of PPI\"],\"sentence\":[\"Transactivation of AP2 was upregulated by HNF4?? in HeLa cells without endogenous HNF4??.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "unclear whether AP2 is regulated by HNF4a via transcription of PPI---{\"tf associated gene name\":[\"HNF4A\"],\"tg associated gene name\":[\"HNF4A\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"unclear whether AP2 is regulated by HNF4a via transcription of PPI\"],\"sentence\":[\"Transactivation of AP2 was upregulated by HNF4?? in HeLa cells without endogenous HNF4??.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not SSRP1 (but p53 or p63)---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"ADA\"],\"tf text\":[],\"tg text\":[\"ADA\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not SSRP1 (but p53 or p63)\"],\"sentence\":[\"In fact p53 upregulation determined an increase in p21\",\"which in turn mediated the cell cycle arrest\",\"while the downregulation of DeltaNp63 determined a marked decrease in ADA transcript.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF not SSRP1 (but p53 or p63)---{\"tf associated gene name\":[\"SSRP1\"],\"tg associated gene name\":[\"ADA\"],\"tf text\":[],\"tg text\":[\"ADA\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not SSRP1 (but p53 or p63)\"],\"sentence\":[\"In fact p53 upregulation determined an increase in p21\",\"which in turn mediated the cell cycle arrest\",\"while the downregulation of DeltaNp63 determined a marked decrease in ADA transcript.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "noTG named---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"STAT3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"noTG named\"],\"sentence\":[\"Our results show that both Stat3 and NF-kB target genes are induced by the involution switch while Stat5 target genes are distinct from Stat3 induced genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "noTG named---{\"tf associated gene name\":[\"STAT3\"],\"tg associated gene name\":[\"STAT3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"noTG named\"],\"sentence\":[\"Our results show that both Stat3 and NF-kB target genes are induced by the involution switch while Stat5 target genes are distinct from Stat3 induced genes.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain about exact variant TG MYHxx---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"MYH13\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about exact variant TG MYHxx\"],\"sentence\":[\"Here\",\"we show that stimulation of beta-ARs with isoproterenol in neonate rat ventricular myocytes causes a \\\"fetal\\\" response in the relative activities of the human cardiac fetal and/or adult gene promoters that includes repression of the human and rat alpha-myosin heavy chain (alpha-MyHC) promoters with simultaneous activation of the human atrial natriuretic peptide (ANP) and rat beta-MyHC promoters.\"],\"Valid\":[false],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}\n",
      "uncertain about exact variant TG MYHxx---{\"tf associated gene name\":[\"AR\"],\"tg associated gene name\":[\"MYH13\"],\"tf text\":[],\"tg text\":[],\"1\":[\"r\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about exact variant TG MYHxx\"],\"sentence\":[\"Here\",\"we show that stimulation of beta-ARs with isoproterenol in neonate rat ventricular myocytes causes a \\\"fetal\\\" response in the relative activities of the human cardiac fetal and/or adult gene promoters that includes repression of the human and rat alpha-myosin heavy chain (alpha-MyHC) promoters with simultaneous activation of the human atrial natriuretic peptide (ANP) and rat beta-MyHC promoters.\"],\"Valid\":[false],\"Sign\":[\"-\"],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "uncertain about TG (CSE) normalization---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CTH\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[\"u\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about TG (CSE) normalization\"],\"sentence\":[\"Sp1 was the critical transcription factor in regulation of CSE expression via the mutation of transcription factor binding sites on the promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "uncertain about TG (CSE) normalization---{\"tf associated gene name\":[\"SP1\"],\"tg associated gene name\":[\"CTH\"],\"tf text\":[\"Sp1\"],\"tg text\":[],\"1\":[\"u\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"uncertain about TG (CSE) normalization\"],\"sentence\":[\"Sp1 was the critical transcription factor in regulation of CSE expression via the mutation of transcription factor binding sites on the promoter.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "ER-negative not sufficient evidence that it is the lack of this TF that causes TG changes (unlike|e.g. \" TF-knockdown\")---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"\\\"ER-negative\\\" not sufficient evidence that it is the lack of this TF that causes TG changes (unlike\",\"e.g. \\\" TF-knockdown\\\")\"],\"sentence\":[\"Also\",\"we evaluated the effects of this CR extract on the transcriptional regulation of genes involved in cell cycle progression in the ER-negative cell lines 293T and T-47D and we found that this extract markedly inhibited the luciferase activity driven by the cyclin D1 promoter and increased the transcriptional activity of the p21 gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "ER-negative not sufficient evidence that it is the lack of this TF that causes TG changes (unlike|e.g. \" TF-knockdown\")---{\"tf associated gene name\":[\"ESR1\"],\"tg associated gene name\":[\"CCND1\"],\"tf text\":[],\"tg text\":[\"cyclin D1\"],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"\\\"ER-negative\\\" not sufficient evidence that it is the lack of this TF that causes TG changes (unlike\",\"e.g. \\\" TF-knockdown\\\")\"],\"sentence\":[\"Also\",\"we evaluated the effects of this CR extract on the transcriptional regulation of genes involved in cell cycle progression in the ER-negative cell lines 293T and T-47D and we found that this extract markedly inhibited the luciferase activity driven by the cyclin D1 promoter and increased the transcriptional activity of the p21 gene promoter.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "non-mammalian?---{\"tf associated gene name\":[\"NR1I2\"],\"tg associated gene name\":[\"FASLG\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\",\"s?\"],\"negation\":[],\"comment\":[\"non-mammalian?\"],\"sentence\":[\"We also demonstrated binding of MgrA and Rot to the spa promoter\",\"and that rot could stimulate spa transcription even in a sarS mutant.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "non-mammalian?---{\"tf associated gene name\":[\"NR1I2\"],\"tg associated gene name\":[\"FASLG\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\",\"s?\"],\"negation\":[],\"comment\":[\"non-mammalian?\"],\"sentence\":[\"We also demonstrated binding of MgrA and Rot to the spa promoter\",\"and that rot could stimulate spa transcription even in a sarS mutant.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TGs are ??GSU and GnRHR---{\"tf associated gene name\":[\"MSX1\"],\"tg associated gene name\":[\"MSX1\"],\"tf text\":[\"Msx1\"],\"tg text\":[\"Msx1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TGs are ??GSU and GnRHR\"],\"sentence\":[\"The expression levels of both ??GSU and GnRHR in Msx1-null mice at e18.5 are higher compared with wild type\",\"further confirming a role for Msx1 in the repression of ??GSU and GnRHR.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TGs are ??GSU and GnRHR---{\"tf associated gene name\":[\"MSX1\"],\"tg associated gene name\":[\"MSX1\"],\"tf text\":[\"Msx1\"],\"tg text\":[\"Msx1\"],\"1\":[],\"0\":[\"t\"],\"negation\":[],\"comment\":[\"TGs are ??GSU and GnRHR\"],\"sentence\":[\"The expression levels of both ??GSU and GnRHR in Msx1-null mice at e18.5 are higher compared with wild type\",\"further confirming a role for Msx1 in the repression of ??GSU and GnRHR.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "none of the suggested TF/TG in sentence---{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"CALCA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"none of the suggested TF/TG in sentence\"],\"sentence\":[\"COX-1 gene promoter deletion and mutation studies indicate a role of Sp transcription factors in CT-induced COX-1 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "none of the suggested TF/TG in sentence---{\"tf associated gene name\":[\"PAX3\"],\"tg associated gene name\":[\"CALCA\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"ni\"],\"negation\":[],\"comment\":[\"none of the suggested TF/TG in sentence\"],\"sentence\":[\"COX-1 gene promoter deletion and mutation studies indicate a role of Sp transcription factors in CT-induced COX-1 gene.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF is repressor of AP-1|JDP2---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is repressor of AP-1\",\"JDP2\"],\"sentence\":[\"We show here that a repressor of AP-1\",\"JDP2 (Jun dimerization protein 2)\",\"inhibits the transactivation of the c-jun gene by ATF-2 and p300\",\"by recruitment of a histone deacetylase complex\",\"thereby repressing the retinoic acid (RA)-induced transcription of the c-jun gene and inhibiting the RA-mediated differentiation of F9 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF is repressor of AP-1|JDP2---{\"tf associated gene name\":[\"AP1\"],\"tg associated gene name\":[\"JUN\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF is repressor of AP-1\",\"JDP2\"],\"sentence\":[\"We show here that a repressor of AP-1\",\"JDP2 (Jun dimerization protein 2)\",\"inhibits the transactivation of the c-jun gene by ATF-2 and p300\",\"by recruitment of a histone deacetylase complex\",\"thereby repressing the retinoic acid (RA)-induced transcription of the c-jun gene and inhibiting the RA-mediated differentiation of F9 cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "not TRE---{\"tf associated gene name\":[\"DDIT3\"],\"tg associated gene name\":[\"PIK3C3\"],\"tf text\":[\"CHOP\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"This is the first evidence that the insulin-like growth factor I/phosphatidylinositol 3-kinase/mammalian target of rapamycin pathway is required for gene regulation by amino acid deprivation and that this pathway is involved in the induction of CHOP by both amino acid deficiency and oxidative stress by hydrogen peroxide.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "not TRE---{\"tf associated gene name\":[\"DDIT3\"],\"tg associated gene name\":[\"PIK3C3\"],\"tf text\":[\"CHOP\"],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"not TRE\"],\"sentence\":[\"This is the first evidence that the insulin-like growth factor I/phosphatidylinositol 3-kinase/mammalian target of rapamycin pathway is required for gene regulation by amino acid deprivation and that this pathway is involved in the induction of CHOP by both amino acid deficiency and oxidative stress by hydrogen peroxide.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Ad12 E2 (adenovirus promoter); TFs are mammalian---{\"tf associated gene name\":[\"ATF1\"],\"tg associated gene name\":[\"DLST\"],\"tf text\":[\"ATF-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Ad12 E2 (adenovirus promoter); TFs are mammalian\"],\"sentence\":[\"Moreover\",\"the Ad12 E2 promoter is inducible by protein kinase A and repressed by either a dominant-negative cAMP-response element-binding protein (CREB) mutant or the highly specific protein kinase A inhibitor protein underscoring the participation of CREB-1/ATF-1 in promoter activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Ad12 E2 (adenovirus promoter); TFs are mammalian---{\"tf associated gene name\":[\"ATF1\"],\"tg associated gene name\":[\"DLST\"],\"tf text\":[\"ATF-1\"],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is Ad12 E2 (adenovirus promoter); TFs are mammalian\"],\"sentence\":[\"Moreover\",\"the Ad12 E2 promoter is inducible by protein kinase A and repressed by either a dominant-negative cAMP-response element-binding protein (CREB) mutant or the highly specific protein kinase A inhibitor protein underscoring the participation of CREB-1/ATF-1 in promoter activation.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is p21(WAF1) (CDKN1A)---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"HDAC1\"],\"tf text\":[\"p53\"],\"tg text\":[\"HDAC1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is p21(WAF1) (CDKN1A)\"],\"sentence\":[\"In HCT116 p53(+)/(+) cells\",\"we found (i) remarkably increased acetylated H4 around both p21(WAF1) promoter regions\",\"especially at the Sp1 site; (ii) increased acetylation of p53 at lysines 320 and 382;(iii) displacement of HDAC1 from the Sp1 site\",\"thus inhibiting its repression effect and increasing p53 binding.p53 seems to trigger H4-acetylation around the p21(WAF1) promoter because there was nearly no H4 acetylation in HCT116 p53(-)/(-) cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is p21(WAF1) (CDKN1A)---{\"tf associated gene name\":[\"TP53\"],\"tg associated gene name\":[\"HDAC1\"],\"tf text\":[\"p53\"],\"tg text\":[\"HDAC1\"],\"1\":[],\"0\":[\"p\"],\"negation\":[],\"comment\":[\"TG is p21(WAF1) (CDKN1A)\"],\"sentence\":[\"In HCT116 p53(+)/(+) cells\",\"we found (i) remarkably increased acetylated H4 around both p21(WAF1) promoter regions\",\"especially at the Sp1 site; (ii) increased acetylation of p53 at lysines 320 and 382;(iii) displacement of HDAC1 from the Sp1 site\",\"thus inhibiting its repression effect and increasing p53 binding.p53 seems to trigger H4-acetylation around the p21(WAF1) promoter because there was nearly no H4 acetylation in HCT116 p53(-)/(-) cells.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TRE? is p300 histone acetyltransferase/KAT7 DbTF?---{\"tf associated gene name\":[\"KAT7\"],\"tg associated gene name\":[\"MMP1\"],\"tf text\":[],\"tg text\":[\"MMP-1\"],\"1\":[\"u\"],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TRE? is p300 histone acetyltransferase/KAT7 DbTF?\"],\"sentence\":[\"The role of p300 histone acetyltransferase in UV-induced histone modifications and MMP-1 gene transcription.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TRE? is p300 histone acetyltransferase/KAT7 DbTF?---{\"tf associated gene name\":[\"KAT7\"],\"tg associated gene name\":[\"MMP1\"],\"tf text\":[],\"tg text\":[\"MMP-1\"],\"1\":[\"u\"],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"TRE? is p300 histone acetyltransferase/KAT7 DbTF?\"],\"sentence\":[\"The role of p300 histone acetyltransferase in UV-induced histone modifications and MMP-1 gene transcription.\"],\"Valid\":[false],\"Sign\":[\"?\"],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is P15INK4b (CDKN2B)---{\"tf associated gene name\":[\"ZBTB17\"],\"tg associated gene name\":[\"TMPRSS11A\"],\"tf text\":[\"Miz-1\"],\"tg text\":[\"ECRG1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is P15INK4b (CDKN2B)\"],\"sentence\":[\"Induction of G1 cell cycle arrest and P15INK4b expression by ECRG1 through interaction with Miz-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is P15INK4b (CDKN2B)---{\"tf associated gene name\":[\"ZBTB17\"],\"tg associated gene name\":[\"TMPRSS11A\"],\"tf text\":[\"Miz-1\"],\"tg text\":[\"ECRG1\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is P15INK4b (CDKN2B)\"],\"sentence\":[\"Induction of G1 cell cycle arrest and P15INK4b expression by ECRG1 through interaction with Miz-1.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is E-cadherin---{\"tf associated gene name\":[\"LEF1\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E-cadherin\"],\"sentence\":[\"We now show that modest overexpression of ILK in intestinal epithelial cells as well as in mammary epithelial cells results in an invasive phenotype concomitant with a down-regulation of E-cadherin expression\",\"translocation of beta-catenin to the nucleus\",\"formation of a complex between beta-catenin and the high mobility group transcription factor\",\"LEF-1\",\"and transcriptional activation by this LEF-1/beta-catenin complex.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is E-cadherin---{\"tf associated gene name\":[\"LEF1\"],\"tg associated gene name\":[\"CDH17\"],\"tf text\":[],\"tg text\":[],\"1\":[\"a\"],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is E-cadherin\"],\"sentence\":[\"We now show that modest overexpression of ILK in intestinal epithelial cells as well as in mammary epithelial cells results in an invasive phenotype concomitant with a down-regulation of E-cadherin expression\",\"translocation of beta-catenin to the nucleus\",\"formation of a complex between beta-catenin and the high mobility group transcription factor\",\"LEF-1\",\"and transcriptional activation by this LEF-1/beta-catenin complex.\"],\"Valid\":[false],\"Sign\":[\"+\"],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is  SP-B---{\"tf associated gene name\":[\"FOXA1\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is  SP-B\"],\"sentence\":[\"Taken together these data indicated that in NCI-H441 cells 1) TNF-alpha inhibition of SP-B promoter activity may be caused by decreased binding activities of TTF-1 and HNF-3 elements\",\"2) the decreased binding activities of TTF-1 and HNF-3 alpha are not due to decreased nuclear levels of the proteins\",\"and 3) okadaic acid-sensitive phosphatases may be involved in mediating TNF-alpha inhibition of SP-B promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is  SP-B---{\"tf associated gene name\":[\"FOXA1\"],\"tg associated gene name\":[\"TNF\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is  SP-B\"],\"sentence\":[\"Taken together these data indicated that in NCI-H441 cells 1) TNF-alpha inhibition of SP-B promoter activity may be caused by decreased binding activities of TTF-1 and HNF-3 elements\",\"2) the decreased binding activities of TTF-1 and HNF-3 alpha are not due to decreased nuclear levels of the proteins\",\"and 3) okadaic acid-sensitive phosphatases may be involved in mediating TNF-alpha inhibition of SP-B promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "CRX promoter controls NRF ectopic expression---{\"tf associated gene name\":[\"NRL\"],\"tg associated gene name\":[\"CRX\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"CRX promoter controls NRF ectopic expression\"],\"sentence\":[\"To examine the role of NRL in cell fate determination\",\"we generated transgenic mice that express Nrl under the control of Crx promoter in postmitotic photoreceptor precursors of WT and Nrl-/- retina.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "CRX promoter controls NRF ectopic expression---{\"tf associated gene name\":[\"NRL\"],\"tg associated gene name\":[\"CRX\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"f\"],\"negation\":[],\"comment\":[\"CRX promoter controls NRF ectopic expression\"],\"sentence\":[\"To examine the role of NRL in cell fate determination\",\"we generated transgenic mice that express Nrl under the control of Crx promoter in postmitotic photoreceptor precursors of WT and Nrl-/- retina.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF not named---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"APLN\"],\"tf text\":[],\"tg text\":[\"apelin\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not named\"],\"sentence\":[\"Transient transfection experiments showed that LPS\",\"IL-6\",\"or IFN-gamma increased apelin expression by stimulation of apelin promoter activity\",\"and blockade of Jak/Stat signaling abolished elevations in apelin promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF not named---{\"tf associated gene name\":[\"IRF6\"],\"tg associated gene name\":[\"APLN\"],\"tf text\":[],\"tg text\":[\"apelin\"],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF not named\"],\"sentence\":[\"Transient transfection experiments showed that LPS\",\"IL-6\",\"or IFN-gamma increased apelin expression by stimulation of apelin promoter activity\",\"and blockade of Jak/Stat signaling abolished elevations in apelin promoter activity.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is Gbp2---{\"tf associated gene name\":[\"IRF7\"],\"tg associated gene name\":[\"TBK1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is Gbp2\"],\"sentence\":[\"IRF7-mediated control of the Gbp2 gene required the presence and basal activity of the S/T kinase TANK-binding kinase 1 (TBK1)\",\"whereas the binding of IRF7 to the Gbp2 promoter did not.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is Gbp2---{\"tf associated gene name\":[\"IRF7\"],\"tg associated gene name\":[\"TBK1\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"up\"],\"negation\":[],\"comment\":[\"TG is Gbp2\"],\"sentence\":[\"IRF7-mediated control of the Gbp2 gene required the presence and basal activity of the S/T kinase TANK-binding kinase 1 (TBK1)\",\"whereas the binding of IRF7 to the Gbp2 promoter did not.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TG is HLA-DRA---{\"tf associated gene name\":[\"POU2F2\"],\"tg associated gene name\":[\"SLC26A3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is HLA-DRA\"],\"sentence\":[\"Here we show that the lymphoid-specific transcription factor Oct-2A plays a critical role in HLA-DRA gene expression in class II-positive B cell lines\",\"and that the high mobility group protein (HMG) I/Y binds to multiple sites within the DRA promoter\",\"including the Oct-2A binding site.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TG is HLA-DRA---{\"tf associated gene name\":[\"POU2F2\"],\"tg associated gene name\":[\"SLC26A3\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TG is HLA-DRA\"],\"sentence\":[\"Here we show that the lymphoid-specific transcription factor Oct-2A plays a critical role in HLA-DRA gene expression in class II-positive B cell lines\",\"and that the high mobility group protein (HMG) I/Y binds to multiple sites within the DRA promoter\",\"including the Oct-2A binding site.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n",
      "JSON decode failed: Expecting value: line 1 column 1 (char 0)\n",
      "TF and TG (DmC/EBP) likely to be (the same) CEBP-family member (unclear which) - i.e. autoregulation---{\"tf associated gene name\":[\"CEBPG\"],\"tg associated gene name\":[\"EBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF and TG (DmC/EBP) likely to be (the same) CEBP-family member (unclear which) - i.e. autoregulation\"],\"sentence\":[\"Among the DNA sequences that DmC/EBP bound with high affinity was a conserved site within the promoter of the DmC/EBP gene itself.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}\n",
      "TF and TG (DmC/EBP) likely to be (the same) CEBP-family member (unclear which) - i.e. autoregulation---{\"tf associated gene name\":[\"CEBPG\"],\"tg associated gene name\":[\"EBP\"],\"tf text\":[],\"tg text\":[],\"1\":[],\"0\":[\"n\"],\"negation\":[],\"comment\":[\"TF and TG (DmC/EBP) likely to be (the same) CEBP-family member (unclear which) - i.e. autoregulation\"],\"sentence\":[\"Among the DNA sequences that DmC/EBP bound with high affinity was a conserved site within the promoter of the DmC/EBP gene itself.\"],\"Valid\":[false],\"Sign\":[null],\"Negation\":[null],\"QC\":[null]}---\n"
     ]
    }
   ],
   "source": [
    "# TODO - Delete this\n",
    "import json\n",
    "\n",
    "all_dicts = []\n",
    "\n",
    "for c in NTNU_extended[\"Comment\"].to_list():\n",
    "    # split into pieces if multiple JSON objects are concatenated\n",
    "    parts = c.split(\"---|---\")\n",
    "    for p in parts:\n",
    "        p = p.strip(\"-\")  # just in case there are dangling dashes\n",
    "        if p:  # skip empty strings\n",
    "            try:\n",
    "                all_dicts.append(json.loads(p))\n",
    "            except json.JSONDecodeError as e:\n",
    "                print(\"JSON decode failed:\", e)\n",
    "                print(p)\n",
    "                print(c)  # preview\n",
    "                break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a607eeb5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\"transcription factor (associated gene name)\":[\"ARNTL\"],\"target gene (associated gene name)\":[\"CRY1\"],\"valid\":[\"TRUE\"],\"sign\":[],\"final_negation_220621\":[],\"qc\":[\"TRUE\"],\"sentence\":[\"CLOCK and ARNTL are transcriptional activators that regulate Per and Cry gene expression.\"],\"Valid\":[true],\"Sign\":[null],\"Negation\":[null],\"QC\":[true]}\n"
     ]
    },
    {
     "ename": "ValueError",
     "evalue": "malformed node or string on line 1: <ast.Name object at 0x7f332048fc10>",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[13], line 12\u001b[0m\n\u001b[1;32m     10\u001b[0m     \u001b[38;5;28mprint\u001b[39m(inner)\n\u001b[1;32m     11\u001b[0m     \u001b[38;5;66;03m# now safely evaluate the dict-like string\u001b[39;00m\n\u001b[0;32m---> 12\u001b[0m     d \u001b[38;5;241m=\u001b[39m \u001b[43mast\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mliteral_eval\u001b[49m\u001b[43m(\u001b[49m\u001b[43minner\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     13\u001b[0m     dicts\u001b[38;5;241m.\u001b[39mappend(d)\n\u001b[1;32m     15\u001b[0m \u001b[38;5;28mprint\u001b[39m(dicts)\n",
      "File \u001b[0;32m/usr/lib/python3.10/ast.py:110\u001b[0m, in \u001b[0;36mliteral_eval\u001b[0;34m(node_or_string)\u001b[0m\n\u001b[1;32m    108\u001b[0m                 \u001b[38;5;28;01mreturn\u001b[39;00m left \u001b[38;5;241m-\u001b[39m right\n\u001b[1;32m    109\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m _convert_signed_num(node)\n\u001b[0;32m--> 110\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43m_convert\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnode_or_string\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/usr/lib/python3.10/ast.py:99\u001b[0m, in \u001b[0;36mliteral_eval.<locals>._convert\u001b[0;34m(node)\u001b[0m\n\u001b[1;32m     97\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mlen\u001b[39m(node\u001b[38;5;241m.\u001b[39mkeys) \u001b[38;5;241m!=\u001b[39m \u001b[38;5;28mlen\u001b[39m(node\u001b[38;5;241m.\u001b[39mvalues):\n\u001b[1;32m     98\u001b[0m         _raise_malformed_node(node)\n\u001b[0;32m---> 99\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mdict\u001b[39;49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mzip\u001b[39;49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mmap\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43m_convert\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnode\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mkeys\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    100\u001b[0m \u001b[43m                    \u001b[49m\u001b[38;5;28;43mmap\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43m_convert\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnode\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mvalues\u001b[49m\u001b[43m)\u001b[49m\u001b[43m)\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    101\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(node, BinOp) \u001b[38;5;129;01mand\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(node\u001b[38;5;241m.\u001b[39mop, (Add, Sub)):\n\u001b[1;32m    102\u001b[0m     left \u001b[38;5;241m=\u001b[39m _convert_signed_num(node\u001b[38;5;241m.\u001b[39mleft)\n",
      "File \u001b[0;32m/usr/lib/python3.10/ast.py:90\u001b[0m, in \u001b[0;36mliteral_eval.<locals>._convert\u001b[0;34m(node)\u001b[0m\n\u001b[1;32m     88\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mtuple\u001b[39m(\u001b[38;5;28mmap\u001b[39m(_convert, node\u001b[38;5;241m.\u001b[39melts))\n\u001b[1;32m     89\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(node, List):\n\u001b[0;32m---> 90\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mlist\u001b[39;49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mmap\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43m_convert\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnode\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43melts\u001b[49m\u001b[43m)\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     91\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(node, Set):\n\u001b[1;32m     92\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mset\u001b[39m(\u001b[38;5;28mmap\u001b[39m(_convert, node\u001b[38;5;241m.\u001b[39melts))\n",
      "File \u001b[0;32m/usr/lib/python3.10/ast.py:109\u001b[0m, in \u001b[0;36mliteral_eval.<locals>._convert\u001b[0;34m(node)\u001b[0m\n\u001b[1;32m    107\u001b[0m         \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    108\u001b[0m             \u001b[38;5;28;01mreturn\u001b[39;00m left \u001b[38;5;241m-\u001b[39m right\n\u001b[0;32m--> 109\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43m_convert_signed_num\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnode\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/usr/lib/python3.10/ast.py:83\u001b[0m, in \u001b[0;36mliteral_eval.<locals>._convert_signed_num\u001b[0;34m(node)\u001b[0m\n\u001b[1;32m     81\u001b[0m     \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m     82\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;241m-\u001b[39m operand\n\u001b[0;32m---> 83\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43m_convert_num\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnode\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m/usr/lib/python3.10/ast.py:74\u001b[0m, in \u001b[0;36mliteral_eval.<locals>._convert_num\u001b[0;34m(node)\u001b[0m\n\u001b[1;32m     72\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m_convert_num\u001b[39m(node):\n\u001b[1;32m     73\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(node, Constant) \u001b[38;5;129;01mor\u001b[39;00m \u001b[38;5;28mtype\u001b[39m(node\u001b[38;5;241m.\u001b[39mvalue) \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;129;01min\u001b[39;00m (\u001b[38;5;28mint\u001b[39m, \u001b[38;5;28mfloat\u001b[39m, \u001b[38;5;28mcomplex\u001b[39m):\n\u001b[0;32m---> 74\u001b[0m         \u001b[43m_raise_malformed_node\u001b[49m\u001b[43m(\u001b[49m\u001b[43mnode\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     75\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m node\u001b[38;5;241m.\u001b[39mvalue\n",
      "File \u001b[0;32m/usr/lib/python3.10/ast.py:71\u001b[0m, in \u001b[0;36mliteral_eval.<locals>._raise_malformed_node\u001b[0;34m(node)\u001b[0m\n\u001b[1;32m     69\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m lno \u001b[38;5;241m:=\u001b[39m \u001b[38;5;28mgetattr\u001b[39m(node, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mlineno\u001b[39m\u001b[38;5;124m'\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m):\n\u001b[1;32m     70\u001b[0m     msg \u001b[38;5;241m+\u001b[39m\u001b[38;5;241m=\u001b[39m \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m on line \u001b[39m\u001b[38;5;132;01m{\u001b[39;00mlno\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m'\u001b[39m\n\u001b[0;32m---> 71\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mValueError\u001b[39;00m(msg \u001b[38;5;241m+\u001b[39m \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00mnode\u001b[38;5;132;01m!r}\u001b[39;00m\u001b[38;5;124m'\u001b[39m)\n",
      "\u001b[0;31mValueError\u001b[0m: malformed node or string on line 1: <ast.Name object at 0x7f332048fc10>"
     ]
    }
   ],
   "source": [
    "import ast\n",
    "\n",
    "# Get the comments\n",
    "comments = [json.loads(c.strip(\"-\")) for c in NTNU_extended[\"Comment\"].to_list()]\n",
    "# Clean and convert to dicts\n",
    "dicts = []\n",
    "for c in comments:\n",
    "    # remove leading/trailing ---\n",
    "    inner = c.strip(\"-\")\n",
    "    print(inner)\n",
    "    # now safely evaluate the dict-like string\n",
    "    d = ast.literal_eval(inner)\n",
    "    dicts.append(d)\n",
    "\n",
    "print(dicts)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c473683f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "Not NaN sentences:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Sentence\n",
       "Sentence    28439\n",
       "NaN         18934\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Counts per column"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Valid\n",
       "true     23869\n",
       "false     4570\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Sign\n",
       "NaN    13164\n",
       "+       9840\n",
       "-       5360\n",
       "+_-       75\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Negation\n",
       "NaN      25142\n",
       "false     2801\n",
       "true       496\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "QC\n",
       "true             23972\n",
       "false             4465\n",
       "Contradiction        2\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "Source file\n",
       "Curation_stack_201221_qc-ed_060522.tsv                                                5597\n",
       "repressive_curated_May_2023.xlsx                                                      4874\n",
       "curated_sentences_compiled_270121.xlsx|Consensus_for_Miguel_080921.xlsx               4015\n",
       "Curated_TRI-sentences_round2_compiled_090521.xlsx|Consensus_for_Miguel_080921.xlsx    3530\n",
       "unsigned_curated_subset_May_2023.xlsx                                                 2827\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# TODO - Move elsewhere if needed. General analysis of the NTNU extended dataset.\n",
    "\n",
    "md(\"Not NaN sentences:\")\n",
    "display(NTNU_extended['Sentence'].apply(lambda x: \"NaN\" if x is np.NaN else \"Sentence\").value_counts(dropna=False))\n",
    "\n",
    "# Only keep not NaN sentences\n",
    "m = NTNU_extended['Sentence'].notna()\n",
    "final_NTNU_df = NTNU_extended[m].copy()\n",
    "\n",
    "md(\"Counts per column\")\n",
    "for column in ['Valid', 'Sign', 'Negation', 'QC', 'Source file']:\n",
    "    display(final_NTNU_df[column].value_counts(dropna=False).head())"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "23338d94",
   "metadata": {},
   "source": [
    "The NTNU extended dataset did not keep the original TF/TG mentions in the text (only their normalizations). Only those sentences that are marked as negations, and where the exact mention of the normalized TF and TG are present in the text, are considered."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "47d26da1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 496 sentences marked as negations.\n",
      "From those, both normalized TF&TG are present in 28 rows.\n"
     ]
    }
   ],
   "source": [
    "# Extract negations from NTNU\n",
    "m = (NTNU_extended['Negation'] == 'true') & ~NTNU_extended['Sentence'].isna()\n",
    "NTNU_negations = NTNU_extended[m].copy()\n",
    "NTNU_negations['TF'] = NTNU_negations['#TRI'].str.split(':').str[2]\n",
    "NTNU_negations['TG'] = NTNU_negations['#TRI'].str.split(':').str[3]\n",
    "\n",
    "m = NTNU_negations.apply(lambda row: (row['TF'] in row['Sentence']) & (row['TG'] in row['Sentence']), axis=1)\n",
    "\n",
    "print(f\"There are {len(NTNU_negations)} sentences marked as negations.\")\n",
    "print(f\"From those, both normalized TF&TG are present in {m.sum()} rows.\")\n",
    "#NTNU_negations[m].to_csv('negations.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b980cc58",
   "metadata": {},
   "source": [
    "In most cases, the sentence contains more than 1 mention of the TF or TG, making the process of swapping the TF/TG for their `[TF]` and `[TG]` tokens difficult to automate. It has been done manually, and saved in `negations_NTNU.tsv`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "ba56f724",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare the negations in the format of the updated data\n",
    "negations_to_add = negations_to_add[['#TRI', 'Sentence', 'TF', 'TG']]\n",
    "negations_to_add.rename(columns={\"#TRI\": \"#TRI ID\"}, inplace=True)\n",
    "negations_to_add['Label'] = 'false'\n",
    "negations_to_add['Type'] = np.nan\n",
    "negations_to_add['validated?'] = True\n",
    "\n",
    "# Combine it with the updated data\n",
    "training_data = pd.concat([training_data, negations_to_add], axis=0)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".general_env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
