{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "bac2a78d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# IMPORTS\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import os\n",
    "import json\n",
    "from IPython.display import display, HTML\n",
    "import matplotlib.pyplot as plt\n",
    "import matplotlib.ticker as mtick\n",
    "from matplotlib_venn import venn3\n",
    "import matplotlib.colors as mcolors\n",
    "\n",
    "import upsetplot\n",
    "import warnings\n",
    "import re\n",
    "from analysis import *\n",
    "\n",
    "from Bio import Entrez\n",
    "# *Always* tell NCBI who you are\n",
    "Entrez.email = \"example24@gmail.com\"\n",
    "\n",
    "## Custom functions\n",
    "import sys\n",
    "\n",
    "sys.path.append('../scripts/common/')\n",
    "from notebook_utils import h3, h4, h5, md\n",
    "from analysis import load_config\n",
    "pd.set_option('display.max_colwidth', 20)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "74326f24",
   "metadata": {},
   "outputs": [],
   "source": [
    "# LOAD DATA\n",
    "config = load_config()\n",
    "\n",
    "ExTRI2_df = pd.read_csv(config['paper_ExTRI2_p'], sep=\"\\t\", dtype=str, keep_default_na=False)\n",
    "collectri = pd.read_csv(\"../data/paper_tables/CollecTRI2.tsv.gz\", sep='\\t', dtype=\"str\", skiprows=1)\n",
    "collectri_regulome = pd.read_csv(\"../data/paper_tables/CollecTRI2-regulome.tsv.gz\", sep='\\t', dtype=\"str\", skiprows=1)\n",
    "all_TFs_df = pd.read_csv(\"tables/all_TFs.tsv\", sep='\\t', dtype=\"str\")\n",
    "all_orthologs_df = pd.read_csv('../data/postprocessing/tables/orthologs_final.tsv', sep=\"\\t\", dtype='str').set_index('Gene_ID')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "d80e5256",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "225927 222387\n"
     ]
    }
   ],
   "source": [
    "# TODO - For the final version ensure ExTRI2 & CollecTRI[ExTRI] have the same content\n",
    "ExTRI2_TRIs = ExTRI2_df[['TF_human_symbol', 'TG_human_symbol']].drop_duplicates()\n",
    "ExTRI2_collectri = collectri[collectri['[ExTRI2] present'].notna()]\n",
    "\n",
    "print(len(ExTRI2_TRIs), len(ExTRI2_collectri))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "c6d062fb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene ID</th>\n",
       "      <th>TF type</th>\n",
       "      <th>Symbol</th>\n",
       "      <th>TaxID</th>\n",
       "      <th>GO:0003700</th>\n",
       "      <th>GO:0003712</th>\n",
       "      <th>GO:0001098</th>\n",
       "      <th>GO:0002039</th>\n",
       "      <th>GO:0008134</th>\n",
       "      <th>GO:0042393</th>\n",
       "      <th>...</th>\n",
       "      <th>tcof_cotf_human.present</th>\n",
       "      <th>tcof_cotf_mouse.present</th>\n",
       "      <th>TFclass_human</th>\n",
       "      <th>TFclass_mouse</th>\n",
       "      <th>TFclass_rat</th>\n",
       "      <th>updated TF type</th>\n",
       "      <th>In ExTRI</th>\n",
       "      <th>human_gene_ID</th>\n",
       "      <th>human_symbol</th>\n",
       "      <th>hgnc_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6177</th>\n",
       "      <td>573</td>\n",
       "      <td>coTF</td>\n",
       "      <td>BAG1</td>\n",
       "      <td>9606</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>573</td>\n",
       "      <td>BAG1</td>\n",
       "      <td>HGNC:937</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows × 29 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Gene ID TF type Symbol TaxID GO:0003700 GO:0003712 GO:0001098 GO:0002039  \\\n",
       "6177     573    coTF   BAG1  9606        NaN        NaN        NaN        NaN   \n",
       "\n",
       "     GO:0008134 GO:0042393  ... tcof_cotf_human.present  \\\n",
       "6177        NaN        NaN  ...                  NaN      \n",
       "\n",
       "     tcof_cotf_mouse.present TFclass_human TFclass_mouse TFclass_rat  \\\n",
       "6177                  NaN              NaN           NaN         NaN   \n",
       "\n",
       "     updated TF type In ExTRI human_gene_ID human_symbol   hgnc_id  \n",
       "6177             NaN     True           573         BAG1  HGNC:937  \n",
       "\n",
       "[1 rows x 29 columns]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# TODO - Remove this. It is just to show why the 573 ID is no longer in ExTRI2:\n",
    "# \"updated_TF_type\" is NaN, as it has no source, so we remove it\n",
    "all_TFs_df[all_TFs_df['Gene ID'] == '573']\n",
    "# TODO - I remember at some point I had 1000 less dbTFs than before. Recheck why. Ensure I didn't lose a lot of TFs in the process. \n",
    "# But ExTRI2 did not lose many sentences so... idk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8ad488fa",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<style type=\"text/css\">\n",
       "</style>\n",
       "<table id=\"T_106b1\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th id=\"T_106b1_level0_col0\" class=\"col_heading level0 col0\" >TF</th>\n",
       "      <th id=\"T_106b1_level0_col1\" class=\"col_heading level0 col1\" >TRIs in ExTRI</th>\n",
       "      <th id=\"T_106b1_level0_col2\" class=\"col_heading level0 col2\" >TRIs in collecTRI</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td id=\"T_106b1_row0_col0\" class=\"data row0 col0\" >NFKB2</td>\n",
       "      <td id=\"T_106b1_row0_col1\" class=\"data row0 col1\" >147</td>\n",
       "      <td id=\"T_106b1_row0_col2\" class=\"data row0 col2\" >166</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td id=\"T_106b1_row1_col0\" class=\"data row1 col0\" >RELB</td>\n",
       "      <td id=\"T_106b1_row1_col1\" class=\"data row1 col1\" >186</td>\n",
       "      <td id=\"T_106b1_row1_col2\" class=\"data row1 col2\" >194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td id=\"T_106b1_row2_col0\" class=\"data row2 col0\" >RELA</td>\n",
       "      <td id=\"T_106b1_row2_col1\" class=\"data row2 col1\" >1055</td>\n",
       "      <td id=\"T_106b1_row2_col2\" class=\"data row2 col2\" >1190</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td id=\"T_106b1_row3_col0\" class=\"data row3 col0\" >NFKB1</td>\n",
       "      <td id=\"T_106b1_row3_col1\" class=\"data row3 col1\" >385</td>\n",
       "      <td id=\"T_106b1_row3_col2\" class=\"data row3 col2\" >615</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td id=\"T_106b1_row4_col0\" class=\"data row4 col0\" >JUN</td>\n",
       "      <td id=\"T_106b1_row4_col1\" class=\"data row4 col1\" >928</td>\n",
       "      <td id=\"T_106b1_row4_col2\" class=\"data row4 col2\" >1101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td id=\"T_106b1_row5_col0\" class=\"data row5 col0\" >NFKB</td>\n",
       "      <td id=\"T_106b1_row5_col1\" class=\"data row5 col1\" >2747</td>\n",
       "      <td id=\"T_106b1_row5_col2\" class=\"data row5 col2\" >0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td id=\"T_106b1_row6_col0\" class=\"data row6 col0\" >AP1</td>\n",
       "      <td id=\"T_106b1_row6_col1\" class=\"data row6 col1\" >1218</td>\n",
       "      <td id=\"T_106b1_row6_col2\" class=\"data row6 col2\" >0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n"
      ],
      "text/plain": [
       "<pandas.io.formats.style.Styler at 0x7f7861030490>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# TODO - Something weird is happening with the AP1/NFKB families in collectri. I did this to check.\n",
    "# Remove once the problem has been identified :)\n",
    "weird_TFs = ['NFKB2', 'RELB', 'RELA', 'NFKB1', 'JUN', 'NFKB', 'AP1']\n",
    "\n",
    "# --- Unique TF–TG pairs in ExTRI2 ---\n",
    "unique_pairs = ExTRI2_df[['TF_human_symbol', 'TG_human_symbol']].drop_duplicates()\n",
    "\n",
    "# Count how many unique pairs each TF appears in\n",
    "extri_counts = (\n",
    "    unique_pairs['TF_human_symbol']\n",
    "    .value_counts()\n",
    "    .reindex(weird_TFs, fill_value=0)\n",
    "    .rename('TRIs In ExTRI2')\n",
    ")\n",
    "\n",
    "# --- Collectri counts (already unique TF–TG pairs) ---\n",
    "collectri_counts = (\n",
    "    collectri['Transcription Factor (Associated Gene Name)']\n",
    "    .value_counts()\n",
    "    .reindex(weird_TFs, fill_value=0)\n",
    "    .rename('TRIs in collecTRI')\n",
    ")\n",
    "\n",
    "# --- Combine into one summary table ---\n",
    "weird_summary = pd.concat([extri_counts, collectri_counts], axis=1).astype(int)\n",
    "weird_summary.index.name = 'TF'\n",
    "display(weird_summary.reset_index(drop=False).style.hide(axis=\"index\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "65dd2afd",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#TF:TG</th>\n",
       "      <th>Transcription Factor (Associated Gene Name)</th>\n",
       "      <th>Target Gene (Associated Gene Name)</th>\n",
       "      <th>[ExTRI2] present</th>\n",
       "      <th>[ExTRI2] PMID</th>\n",
       "      <th>[ExTRI2] Sign</th>\n",
       "      <th>[ExTRI2] Transcription Factor Type</th>\n",
       "      <th>[HTRI] present</th>\n",
       "      <th>[HTRI] Technique</th>\n",
       "      <th>[HTRI] PMID</th>\n",
       "      <th>...</th>\n",
       "      <th>[DoRothEA_A] PMID</th>\n",
       "      <th>[DoRothEA_A] Directed</th>\n",
       "      <th>[DoRothEA_A] Effect</th>\n",
       "      <th>Lambert</th>\n",
       "      <th>Lovering</th>\n",
       "      <th>GO:0003700</th>\n",
       "      <th>GO:0140223</th>\n",
       "      <th>GO:0003712</th>\n",
       "      <th>TFClass</th>\n",
       "      <th>Auto-regulation</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>MAPK3:BGLAP</td>\n",
       "      <td>MAPK3</td>\n",
       "      <td>BGLAP</td>\n",
       "      <td>ExTRI2</td>\n",
       "      <td>25017640|2937262...</td>\n",
       "      <td>UP|UP|UP|UNKNOWN...</td>\n",
       "      <td>coTF candidate;c...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>MAPK1:BGLAP</td>\n",
       "      <td>MAPK1</td>\n",
       "      <td>BGLAP</td>\n",
       "      <td>ExTRI2</td>\n",
       "      <td>25017640|2937262...</td>\n",
       "      <td>UP|UP|UP|UP|UNKN...</td>\n",
       "      <td>coTF candidate;c...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>MAPK1:BAX</td>\n",
       "      <td>MAPK1</td>\n",
       "      <td>BAX</td>\n",
       "      <td>ExTRI2</td>\n",
       "      <td>11761456|2965252...</td>\n",
       "      <td>UNKNOWN|UNKNOWN|...</td>\n",
       "      <td>coTF candidate;c...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>MAPK3:BMP2</td>\n",
       "      <td>MAPK3</td>\n",
       "      <td>BMP2</td>\n",
       "      <td>ExTRI2</td>\n",
       "      <td>25017640|3183177...</td>\n",
       "      <td>UP|UP|UP|UP|UNKN...</td>\n",
       "      <td>coTF candidate;c...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>MAPK1:BMP2</td>\n",
       "      <td>MAPK1</td>\n",
       "      <td>BMP2</td>\n",
       "      <td>ExTRI2</td>\n",
       "      <td>25017640|3183177...</td>\n",
       "      <td>UP|UP|UP|UP|UP|U...</td>\n",
       "      <td>coTF candidate;c...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 56 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          #TF:TG Transcription Factor (Associated Gene Name)  \\\n",
       "127  MAPK3:BGLAP                MAPK3                          \n",
       "128  MAPK1:BGLAP                MAPK1                          \n",
       "152    MAPK1:BAX                MAPK1                          \n",
       "183   MAPK3:BMP2                MAPK3                          \n",
       "184   MAPK1:BMP2                MAPK1                          \n",
       "\n",
       "    Target Gene (Associated Gene Name) [ExTRI2] present        [ExTRI2] PMID  \\\n",
       "127                BGLAP                         ExTRI2  25017640|2937262...   \n",
       "128                BGLAP                         ExTRI2  25017640|2937262...   \n",
       "152                  BAX                         ExTRI2  11761456|2965252...   \n",
       "183                 BMP2                         ExTRI2  25017640|3183177...   \n",
       "184                 BMP2                         ExTRI2  25017640|3183177...   \n",
       "\n",
       "           [ExTRI2] Sign [ExTRI2] Transcription Factor Type [HTRI] present  \\\n",
       "127  UP|UP|UP|UNKNOWN...  coTF candidate;c...                          NaN   \n",
       "128  UP|UP|UP|UP|UNKN...  coTF candidate;c...                          NaN   \n",
       "152  UNKNOWN|UNKNOWN|...  coTF candidate;c...                          NaN   \n",
       "183  UP|UP|UP|UP|UNKN...  coTF candidate;c...                          NaN   \n",
       "184  UP|UP|UP|UP|UP|U...  coTF candidate;c...                          NaN   \n",
       "\n",
       "    [HTRI] Technique [HTRI] PMID  ... [DoRothEA_A] PMID [DoRothEA_A] Directed  \\\n",
       "127              NaN         NaN  ...               NaN                  NaN    \n",
       "128              NaN         NaN  ...               NaN                  NaN    \n",
       "152              NaN         NaN  ...               NaN                  NaN    \n",
       "183              NaN         NaN  ...               NaN                  NaN    \n",
       "184              NaN         NaN  ...               NaN                  NaN    \n",
       "\n",
       "    [DoRothEA_A] Effect Lambert Lovering GO:0003700 GO:0140223 GO:0003712  \\\n",
       "127                 NaN     NaN      NaN        NaN        NaN        NaN   \n",
       "128                 NaN     NaN      NaN        NaN        NaN        NaN   \n",
       "152                 NaN     NaN      NaN        NaN        NaN        NaN   \n",
       "183                 NaN     NaN      NaN        NaN        NaN        NaN   \n",
       "184                 NaN     NaN      NaN        NaN        NaN        NaN   \n",
       "\n",
       "    TFClass Auto-regulation  \n",
       "127     NaN             NaN  \n",
       "128     NaN             NaN  \n",
       "152     NaN             NaN  \n",
       "183     NaN             NaN  \n",
       "184     NaN             NaN  \n",
       "\n",
       "[5 rows x 56 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "[ExTRI2] Transcription Factor Type\n",
       "dbTF                                   104659\n",
       "coTF candidate                          80132\n",
       "coTF                                    28189\n",
       "NaN                                     11635\n",
       "coTF candidate;coTF candidate            3436\n",
       "                                        ...  \n",
       "coTF candidate;-;-;coTF candidate;-         2\n",
       "coTF;coTF candidate;dbTF                    2\n",
       "dbTF;dbTF;coTF candidate                    2\n",
       "-;coTF candidate;coTF                       1\n",
       "coTF;dbTF;coTF candidate                    1\n",
       "Name: count, Length: 89, dtype: int64"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# TODO - collecTRI's extri TF type is the rodent one, not the human one.\n",
    "# Is it worth for me to fix it? Is it a useful field, really?\n",
    "m = collectri['[ExTRI2] Transcription Factor Type'].str.contains(';', na=False)\n",
    "display(collectri[m].head())\n",
    "collectri['[ExTRI2] Transcription Factor Type'].value_counts(dropna=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "7f51f660",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 280 human genes with non-uppercase symbols. \n",
      "Of these, 277 are ORFs (e.g., ['C9orf72', 'C19orf12', 'C19orf33']).\n",
      "Non-ORFs:\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>NCBI GeneID</th>\n",
       "      <th>Symbol</th>\n",
       "      <th>Gene Type</th>\n",
       "      <th>Taxonomic Name</th>\n",
       "      <th>Description</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>183794</th>\n",
       "      <td>222029</td>\n",
       "      <td>DKFZp434L192</td>\n",
       "      <td>ncRNA</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>uncharacterized ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184134</th>\n",
       "      <td>259265</td>\n",
       "      <td>bA255A11.4</td>\n",
       "      <td>PSEUDO</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>melanoma antigen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187485</th>\n",
       "      <td>401282</td>\n",
       "      <td>DKFZp451B082</td>\n",
       "      <td>ncRNA</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>uncharacterized ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       NCBI GeneID        Symbol Gene Type Taxonomic Name          Description\n",
       "183794      222029  DKFZp434L192     ncRNA   Homo sapiens  uncharacterized ...\n",
       "184134      259265    bA255A11.4    PSEUDO   Homo sapiens  melanoma antigen...\n",
       "187485      401282  DKFZp451B082     ncRNA   Homo sapiens  uncharacterized ..."
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "In CollecTRI, we only have orfs\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Transcription Factor (Associated Gene Name)\n",
       "NaN        155\n",
       "C6orf89      2\n",
       "Name: count, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# SOLVED - THERE AREHUMAN GENES WITHOUT UPPERCASE\n",
    "def load_all_human_genes():\n",
    "    # Load all human TGs\n",
    "    # downloaded from: https://www.ncbi.nlm.nih.gov/datasets/gene/taxon/9606/\n",
    "    all_human_genes = pd.read_csv(config['all_human_TGs_p'], sep='\\t', header=0, dtype=str)\n",
    "    all_human_genes = all_human_genes[['NCBI GeneID', 'Symbol', 'Gene Type', 'Taxonomic Name', 'Description']]\n",
    "    all_human_genes.loc[all_human_genes['Gene Type'].isna(), 'Gene Type'] = 'nan'\n",
    "\n",
    "    # Add AP1 and NFKB complexes\n",
    "    all_human_genes = pd.concat([all_human_genes, pd.DataFrame({\n",
    "        'NCBI GeneID': ['Complex:AP1', 'Complex:NFKB'], \n",
    "        'Symbol': ['AP1', 'NFKB'],\n",
    "        'Description': ['', ''],\n",
    "        'Taxonomic Name': ['Homo sapiens', 'Homo sapiens'],\n",
    "        'Gene Type': ['PROTEIN_CODING', 'PROTEIN_CODING'],\n",
    "        })], ignore_index=True)\n",
    "    \n",
    "    # Ensure the data is expected: all human, no duplicate IDs\n",
    "    assert all_human_genes['Taxonomic Name'].nunique() == 1\n",
    "    assert all_human_genes['Taxonomic Name'].unique()[0] == 'Homo sapiens'\n",
    "    assert (all_human_genes['NCBI GeneID'].duplicated()).sum() == 0 \n",
    "\n",
    "    return all_human_genes\n",
    "all_human_genes = load_all_human_genes()\n",
    "tf_col = 'Transcription Factor (Associated Gene Name)'\n",
    "\n",
    "# Show human genes with non-uppercase symbols\n",
    "m_lowercase = all_human_genes['Symbol'] != all_human_genes['Symbol'].str.upper()\n",
    "m_orfs = m_lowercase & all_human_genes['Symbol'].str.contains('orf')\n",
    "\n",
    "print(f\"There are {m_lowercase.sum()} human genes with non-uppercase symbols. \"\n",
    "      f\"\\nOf these, {m_orfs.sum()} are ORFs (e.g., {all_human_genes[m_orfs]['Symbol'][:3].tolist()}).\"\n",
    "      \"\\nNon-ORFs:\")\n",
    "display(all_human_genes[m_lowercase & ~m_orfs])\n",
    "\n",
    "# Show if there are others in collecTRI: it is only the orf one!\n",
    "m = (collectri[tf_col].str.upper() != collectri[tf_col])\n",
    "print(\"In CollecTRI, we only have orfs\")\n",
    "display(collectri[m][tf_col].value_counts(dropna=False))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d88dca35",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ExTRI2 mv\n",
      "{'Non valid': 21320730, 'Valid': 1149192}\n",
      "Raw ExTRI2 mv\n",
      "{'Non valid': 21320730, 'Valid': 1149192}\n"
     ]
    },
    {
     "ename": "",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31mThe Kernel crashed while executing code in the current cell or a previous cell. \n",
      "\u001b[1;31mPlease review the code in the cell(s) to identify a possible cause of the failure. \n",
      "\u001b[1;31mClick <a href='https://aka.ms/vscodeJupyterKernelCrash'>here</a> for more info. \n",
      "\u001b[1;31mView Jupyter <a href='command:jupyter.viewOutput'>log</a> for further details."
     ]
    }
   ],
   "source": [
    "# SOLVED - HOW TO ACCUMULATE VALUES FROM A BIG FILE\n",
    "from collections import Counter\n",
    "\n",
    "print(\"ExTRI2 mv\")\n",
    "d = {'Non valid': 0, 'Valid': 0}\n",
    "for chunk in pd.read_csv('../../ExTRI2_final_resource.mv.tsv.gz', sep='\\t', header=1, chunksize=1000000, keep_default_na=False):\n",
    "    for v in d:\n",
    "        d[v] += chunk[chunk['Valid'] == v].shape[0]\n",
    "print(d)\n",
    "\n",
    "print(\"Raw ExTRI2 mv\")\n",
    "d = {'Non valid': 0, 'Valid': 0}\n",
    "for chunk in pd.read_csv('../results/ExTRI2.tsv', sep='\\t', header=1, chunksize=1000000, keep_default_na=False):\n",
    "    for v in d:\n",
    "        d[v] += chunk[chunk['Valid'] == v].shape[0]\n",
    "print(d)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "943da9a1",
   "metadata": {},
   "outputs": [],
   "source": [
    "# COMPARISON OF DIFFERENT VERSIONS OF EXTRI\n",
    "old_ExTRI2_df = pd.read_csv('../../ExTRI2-06be616/results/ExTRI2_final_resource.tsv', sep=\"\\t\", dtype=str, keep_default_na=False)\n",
    "results_ExTRI2_df = pd.read_csv('../results/ExTRI2.tsv', sep=\"\\t\", dtype=str, keep_default_na=False)\n",
    "\n",
    "m = (\n",
    "    (ExTRI2_df['TF_human_symbol'] == 'None') | (ExTRI2_df['TG_human_symbol'] == 'None') |\n",
    "    (ExTRI2_df['TF_human_symbol'] == '') | (ExTRI2_df['TG_human_symbol'] == '') |\n",
    "    (ExTRI2_df['TF_human_symbol'].str.contains(';')) | (ExTRI2_df['TG_human_symbol'].str.contains(';'))\n",
    ")\n",
    "print('paper ExTRI', len(ExTRI2_df), len(ExTRI2_df[~m]), len(ExTRI2_df[~m]['TF_human_symbol'].unique()))\n",
    "\n",
    "m = (\n",
    "    (results_ExTRI2_df['TF_human_symbol'] == 'None') | (results_ExTRI2_df['TG_human_symbol'] == 'None') |\n",
    "    (results_ExTRI2_df['TF_human_symbol'] == '') | (results_ExTRI2_df['TG_human_symbol'] == '') |\n",
    "    (results_ExTRI2_df['TF_human_symbol'].str.contains(';')) | (results_ExTRI2_df['TG_human_symbol'].str.contains(';'))\n",
    ")\n",
    "print('results ExTRI', len(results_ExTRI2_df), len(results_ExTRI2_df[~m]), len(results_ExTRI2_df[~m]['TF_human_symbol'].unique()))\n",
    "\n",
    "m = (\n",
    "    (old_ExTRI2_df['TF_human_symbol'] == 'None') | (old_ExTRI2_df['TG_human_symbol'] == 'None') |\n",
    "    (old_ExTRI2_df['TF_human_symbol'] == '') | (old_ExTRI2_df['TG_human_symbol'] == '') |\n",
    "    (old_ExTRI2_df['TF_human_symbol'].str.contains(';')) | (old_ExTRI2_df['TG_human_symbol'].str.contains(';'))\n",
    ")\n",
    "print('old ExTRI', len(old_ExTRI2_df), len(old_ExTRI2_df[~m]), len(old_ExTRI2_df[~m]['TF_human_symbol'].unique()))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "929a378f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TFs without updated TF type: 0\n",
      "TFs in ExTRI2 without updated TF type:   0\n",
      "ExTRI2 sentences with TFs without updated TF type: 0 0.0\n",
      "18 TFs in ExTRI2 are not present in the orthologs table (0.3%), 4 human ones\n"
     ]
    }
   ],
   "source": [
    "# TODO - Ensure all these things print things that make sense\n",
    "# TFs checks (they were in get_NCBI_TF_IDs.ipynb)\n",
    "geneIDs_in_ExTRI2 = {id for col in ['TF Id', 'TG Id'] for ids in ExTRI2_df[col].unique() for id in ids.split(';')}\n",
    "all_TFs_df[\"In ExTRI2\"] = all_TFs_df['Gene ID'].isin(geneIDs_in_ExTRI2)\n",
    "\n",
    "print(\"TFs without updated TF type:\", (all_TFs_df['updated TF type'] == '').sum())\n",
    "print(\"TFs in ExTRI2 without updated TF type:  \", (all_TFs_df[all_TFs_df['In ExTRI2']]['updated TF type'] == '').sum())\n",
    "m = all_TFs_df['In ExTRI2'] & (all_TFs_df['updated TF type'] == '')\n",
    "print(\"ExTRI2 sentences with TFs without updated TF type:\", ExTRI2_df['TF Id'].isin(all_TFs_df[m]['Gene ID']).sum(), (ExTRI2_df['TF Id'].isin(all_TFs_df[m]['Gene ID'])).sum() / len(ExTRI2_df))\n",
    "\n",
    "m_missing = (all_TFs_df['In ExTRI2'] & ~all_TFs_df['Gene ID'].isin(all_orthologs_df.index))\n",
    "m_missing_human = m_missing & (all_TFs_df['TaxID'] == '9606')\n",
    "print(f\"{m_missing.sum()} TFs in ExTRI2 are not present in the orthologs table ({m_missing.sum() / all_TFs_df['In ExTRI2'].sum():.1%}), {(m_missing_human).sum()} human ones\")\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".general_env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
